{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import mgclient # memgraph client\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "```\n",
    "docker run -p 7687:7687 -p 7444:7444 -p 3000:3000 --name memgraph -e MEMGRAPH=\"--query-execution-timeout-sec=3600 --memory-limit=30720\" memgraph/memgraph-platform\n",
    "```"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Data Comparisons "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## `Memgraph`"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Connect to `memgraph` host**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "conn = mgclient.connect(host=\"localhost\", port=7687)   # connect to memgraph\n",
    "conn.autocommit = True  # autocommit mode is required for DDL queries\n",
    "cursor = conn.cursor() # cursor is used to execute queries"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "---"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "from controller.metakg import MetaKG\n",
    "from model import ConsolidatedMetaKGDoc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "index = ConsolidatedMetaKGDoc.Index.name"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Index on the name property of the Entity node\n",
    "q=\"\"\"\n",
    "CREATE INDEX ON :Entity(name);\n",
    "\"\"\"\n",
    "\n",
    "cursor.execute(q) # execute query"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This graph database reflects the `ConsolidatedMetaKG` Index with node enitities reflecting the `subject` and `object`, and the paths reflecting the pairs corresponding `predicate`.\n",
    "\n",
    "This database would be built once."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[1;32m/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb Cell 12\u001b[0m line \u001b[0;36m1\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=11'>12</a>\u001b[0m query \u001b[39m=\u001b[39m \u001b[39m\"\"\"\u001b[39m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=12'>13</a>\u001b[0m \u001b[39mMERGE (s:Entity \u001b[39m\u001b[39m{\u001b[39m\u001b[39mname: $subject})\u001b[39m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=13'>14</a>\u001b[0m \u001b[39mMERGE (o:Entity \u001b[39m\u001b[39m{\u001b[39m\u001b[39mname: $object})\u001b[39m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=14'>15</a>\u001b[0m \u001b[39mMERGE (s)-[r:RELATED_TO \u001b[39m\u001b[39m{\u001b[39m\u001b[39mid: $unique_id}]->(o)\u001b[39m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=15'>16</a>\u001b[0m \u001b[39mSET r.predicate = $predicate, r.api = $api_data\u001b[39m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=16'>17</a>\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"\u001b[39m\n\u001b[1;32m     <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=17'>18</a>\u001b[0m params \u001b[39m=\u001b[39m {\u001b[39m'\u001b[39m\u001b[39msubject\u001b[39m\u001b[39m'\u001b[39m: subject, \u001b[39m'\u001b[39m\u001b[39mobject\u001b[39m\u001b[39m'\u001b[39m: \u001b[39mobject\u001b[39m, \u001b[39m'\u001b[39m\u001b[39munique_id\u001b[39m\u001b[39m'\u001b[39m: unique_id, \u001b[39m'\u001b[39m\u001b[39mpredicate\u001b[39m\u001b[39m'\u001b[39m: predicate, \u001b[39m'\u001b[39m\u001b[39mapi_data\u001b[39m\u001b[39m'\u001b[39m: api_data}\n\u001b[0;32m---> <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X13sZmlsZQ%3D%3D?line=18'>19</a>\u001b[0m cursor\u001b[39m.\u001b[39;49mexecute(query, params)\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "for edge in MetaKG.get_all_via_scan(size=1000, index=index):\n",
    "    subject = edge['_source']['subject']\n",
    "    object = edge['_source']['object']\n",
    "    predicate = edge['_source']['predicate']\n",
    "    api_data = edge['_source']['api']  # Additional data as properties\n",
    "\n",
    "    # Construct a unique identifier for each relationship\n",
    "    # This is just an example - modify it according to your data structure\n",
    "    unique_id = f\"{subject}-{predicate}-{object}\"\n",
    "    \n",
    "    # Cypher query to create nodes with a static relationship\n",
    "    query = \"\"\"\n",
    "    MERGE (s:Entity {name: $subject})\n",
    "    MERGE (o:Entity {name: $object})\n",
    "    MERGE (s)-[r:RELATED_TO {id: $unique_id}]->(o)\n",
    "    SET r.predicate = $predicate, r.api = $api_data\n",
    "    \"\"\"\n",
    "    params = {'subject': subject, 'object': object, 'unique_id': unique_id, 'predicate': predicate, 'api_data': api_data}\n",
    "    cursor.execute(query, params)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "---"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..1]-(end:Entity {name: 'Publication'})\n",
    "RETURN p;\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1\n"
     ]
    }
   ],
   "source": [
    "cursor.execute(query)\n",
    "\n",
    "# Display the results\n",
    "results = cursor.fetchall()\n",
    "print(len(results))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..2]-(end:Entity {name: 'Publication'})\n",
    "RETURN p;\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "593\n"
     ]
    }
   ],
   "source": [
    "cursor.execute(query)\n",
    "\n",
    "# Display the results\n",
    "results = cursor.fetchall()\n",
    "print(len(results))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "query=\"\"\"\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..2]->(end:Entity {name: 'Publication'})\n",
    "UNWIND relationships(p) AS r\n",
    "WITH start, end, collect(r.predicate) AS predicates\n",
    "RETURN start, end, predicates;\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "ename": "DatabaseError",
     "evalue": "Transaction was asked to abort because of transaction timeout.",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mDatabaseError\u001b[0m                             Traceback (most recent call last)",
      "\u001b[1;32m/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb Cell 11\u001b[0m line \u001b[0;36m1\n\u001b[0;32m----> <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X16sZmlsZQ%3D%3D?line=0'>1</a>\u001b[0m cursor\u001b[39m.\u001b[39;49mexecute(query)\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X16sZmlsZQ%3D%3D?line=2'>3</a>\u001b[0m \u001b[39m# Display the results\u001b[39;00m\n\u001b[1;32m      <a href='vscode-notebook-cell:/Users/nacosta/Documents/smartAPI/path_finder2.0/smartAPI/src/memgraph_compare.ipynb#X16sZmlsZQ%3D%3D?line=3'>4</a>\u001b[0m results \u001b[39m=\u001b[39m cursor\u001b[39m.\u001b[39mfetchall()\n",
      "\u001b[0;31mDatabaseError\u001b[0m: Transaction was asked to abort because of transaction timeout."
     ]
    }
   ],
   "source": [
    "cursor.execute(query)\n",
    "\n",
    "# Display the results\n",
    "results = cursor.fetchall()\n",
    "print(len(results))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "`PROFILE MATCH` query "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "q=\"\"\"\n",
    "PROFILE MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..3]->(end:Entity {name: 'Publication'})\n",
    "RETURN p\n",
    "LIMIT 100;\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('* Limit', 101, '  0.003494 %', '  0.020170 ms')\n",
      "('* Produce {p}', 100, '  0.037991 %', '  0.219289 ms')\n",
      "('* ConstructNamedPath', 100, '  0.027923 %', '  0.161174 ms')\n",
      "('* Filter (start :Entity), {start.name}', 100, ' 67.900171 %', ' 391.930952 ms')\n",
      "('* ExpandVariable (end)<-[anon1:RELATED_TO]-(start)', 1548889, ' 32.029290 %', ' 184.878330 ms')\n",
      "('* ScanAllByLabelPropertyValue (end :Entity {name})', 1, '  0.001127 %', '  0.006506 ms')\n",
      "('* Once', 1, '  0.000003 %', '  0.000019 ms')\n"
     ]
    }
   ],
   "source": [
    "cursor.execute(q)\n",
    "\n",
    "# Fetch and display the results\n",
    "results = cursor.fetchall()\n",
    "\n",
    "# Printing each result\n",
    "for result in results:\n",
    "    print(result)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The provided Cypher query is performing the following operations in a Neo4j graph database:\n",
    "```\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..3]->(end:Entity {name: 'Publication'})\n",
    "RETURN p\n",
    "LIMIT 100;\n",
    "```\n",
    "1. **MATCH Clause**: \n",
    "   - `MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..3]->(end:Entity {name: 'Publication'})`\n",
    "   - This part of the query is looking for a specific pattern in the graph. It defines a variable `p` that represents a path between two nodes (labeled as `Entity`). \n",
    "   - The first node (`start`) is an `Entity` with a `name` property equal to `'InformationResource'`.\n",
    "   - The last node (`end`) is an `Entity` with a `name` property equal to `'Publication'`.\n",
    "   - The `[:RELATED_TO*..3]->` segment specifies the type of relationships to follow (`RELATED_TO`) and the depth of the path. It indicates that the query should find paths where `start` and `end` are connected by a sequence of up to 3 `RELATED_TO` relationships. The `->` indicates the direction of the relationship is from `start` to `end`.\n",
    "\n",
    "2. **RETURN Clause**:\n",
    "   - `RETURN p`\n",
    "   - This clause specifies that the query should return the paths that it finds. Each path `p` includes the start node, the end node, and all nodes and relationships between them that match the specified pattern.\n",
    "\n",
    "3. **LIMIT Clause**:\n",
    "   - `LIMIT 100`\n",
    "   - This part of the query limits the number of paths returned to 100. Without this limit, the query could potentially return a very large number of paths, especially in a large or densely connected graph, which might be more data than is needed and could impact performance.\n",
    "\n",
    "In summary, the query is searching for up to 100 instances of a path in the graph where nodes of type `Entity` and with specific `name` properties are connected by a chain of up to 3 `RELATED_TO` relationships. The query is particularly useful for finding specific connections and relationships in a graph, especially when looking for indirect relationships (connections not immediately obvious) between two entities."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cab9590>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cabb390>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c940ed0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c942a30>], relationships=[<mgclient.Relationship(start_id=226, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-subclass_of-Publication', 'predicate': 'subclass_of'}) at 0x10c8377f0>, <mgclient.Relationship(start_id=121, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'Publication-correlated_with-SmallMolecule', 'predicate': 'correlated_with'}) at 0x10c836bf0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10c8357f0>]) at 0x10ca016d0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cad6490>, <mgclient.Node(id=132, labels={'Entity'}, properties={'name': 'InformationContentEntity'}) at 0x10cad5920>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cad5230>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cad5fe0>], relationships=[<mgclient.Relationship(start_id=226, end_id=132, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-subclass_of-InformationContentEntity', 'predicate': 'subclass_of'}) at 0x1046362b0>, <mgclient.Relationship(start_id=132, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'InformationContentEntity-correlated_with-SmallMolecule', 'predicate': 'correlated_with'}) at 0x10c83da30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10c83e570>]) at 0x10ca01650>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cad5ad0>, <mgclient.Node(id=132, labels={'Entity'}, properties={'name': 'InformationContentEntity'}) at 0x10cb44ab0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cb47150>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cb45f50>], relationships=[<mgclient.Relationship(start_id=226, end_id=132, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-related_to-InformationContentEntity', 'predicate': 'related_to'}) at 0x10c80a4f0>, <mgclient.Relationship(start_id=132, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'InformationContentEntity-correlated_with-SmallMolecule', 'predicate': 'correlated_with'}) at 0x10c825670>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10c827330>]) at 0x10ca020d0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cb44c60>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10cb44960>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cb44510>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10caa7f30>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10c81ecf0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-ameliorates-SmallMolecule', 'predicate': 'ameliorates'}) at 0x1046e7b30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x104674770>]) at 0x10ca03950>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10caa5ef0>, <mgclient.Node(id=145, labels={'Entity'}, properties={'name': 'DiseaseOrPhenotypicFeature'}) at 0x10caa7480>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10caa7d20>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10caa4420>], relationships=[<mgclient.Relationship(start_id=226, end_id=145, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-DiseaseOrPhenotypicFeature', 'predicate': 'associated_with'}) at 0x10cc217f0>, <mgclient.Relationship(start_id=145, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'DiseaseOrPhenotypicFeature-related_to-SmallMolecule', 'predicate': 'related_to'}) at 0x10cc559b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cc54530>]) at 0x10c992c90>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10caa5830>, <mgclient.Node(id=145, labels={'Entity'}, properties={'name': 'DiseaseOrPhenotypicFeature'}) at 0x10caa5d40>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10caa46c0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca44690>], relationships=[<mgclient.Relationship(start_id=226, end_id=145, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-DiseaseOrPhenotypicFeature', 'predicate': 'associated_with'}) at 0x10c8182f0>, <mgclient.Relationship(start_id=145, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}], 'id': 'DiseaseOrPhenotypicFeature-is_side_effect_of-SmallMolecule', 'predicate': 'is_side_effect_of'}) at 0x10cc4e1f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10c81a670>]) at 0x10ca02710>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cb297a0>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10cb5a7f0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cb5bab0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cad5740>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10cc2f7f0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"treats\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'treated_by': {'CHEBI': 'subject.CHEBI', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.DRUGBANK,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"treats\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'treated_by': {'DRUGBANK': 'subject.DRUGBANK', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}], 'id': 'PhenotypicFeature-treated_by-SmallMolecule', 'predicate': 'treated_by'}) at 0x10cbe6ff0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cc590b0>]) at 0x10c992f50>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cb010e0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10cb039c0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cb018f0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cb03030>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10c823d70>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}], 'id': 'Disease-is_side_effect_of-SmallMolecule', 'predicate': 'is_side_effect_of'}) at 0x10c823070>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10c822e30>]) at 0x10c9912f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cae96e0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10caeb8d0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cab9860>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cabb420>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10cc4a270>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'aeolus.unii', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'aeolus.indications.meddra_code'}}, 'server': 'https://mychem.info/v1', 'support_batch': True}, 'response_mapping': {'treated_by': {'UNII': 'aeolus.unii'}}}, 'name': 'MyChem.info API', 'provided_by': 'infores:aeolus', 'smartapi': {'id': '8f08d1446e0bb9c2b323713ce83e2bd3', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mychem.info/openapi_full.yml', 'ui': 'https://smart-api.info/ui/8f08d1446e0bb9c2b323713ce83e2bd3'}, 'tags': ['chemical', 'drug', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'chembl.molecule_chembl_id', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'chembl.drug_indications.mesh_id'}}, 'server': 'https://mychem.info/v1', 'support_batch': True}, 'response_mapping': {'treated_by': {'CHEMBL.COMPOUND': 'chembl.molecule_chembl_id'}}}, 'name': 'MyChem.info API', 'provided_by': 'infores:chembl', 'smartapi': {'id': '8f08d1446e0bb9c2b323713ce83e2bd3', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mychem.info/openapi_full.yml', 'ui': 'https://smart-api.info/ui/8f08d1446e0bb9c2b323713ce83e2bd3'}, 'tags': ['chemical', 'drug', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'drugcentral.xrefs.unii', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'drugcentral.drug_use.indication.umls_cui'}}, 'server': 'https://mychem.info/v1', 'support_batch': True}, 'response_mapping': {'treated_by': {'UNII': 'drugcentral.xrefs.unii'}}}, 'name': 'MyChem.info API', 'provided_by': 'infores:drugcentral', 'smartapi': {'id': '8f08d1446e0bb9c2b323713ce83e2bd3', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mychem.info/openapi_full.yml', 'ui': 'https://smart-api.info/ui/8f08d1446e0bb9c2b323713ce83e2bd3'}, 'tags': ['chemical', 'drug', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'drugcentral.xrefs.chembl_id', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'drugcentral.drug_use.off_label_use.umls_cui'}}, 'server': 'https://mychem.info/v1', 'support_batch': True}, 'response_mapping': {'treated_by': {'CHEMBL.COMPOUND': 'drugcentral.xrefs.chembl_id'}}}, 'name': 'MyChem.info API', 'provided_by': 'infores:drugcentral', 'smartapi': {'id': '8f08d1446e0bb9c2b323713ce83e2bd3', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mychem.info/openapi_full.yml', 'ui': 'https://smart-api.info/ui/8f08d1446e0bb9c2b323713ce83e2bd3'}, 'tags': ['chemical', 'drug', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'fda_orphan_drug.pubchem_cid', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'fda_orphan_drug.orphan_designation.umls'}}, 'server': 'https://mychem.info/v1', 'support_batch': True}, 'response_mapping': {'treated_by': {'PUBCHEM.COMPOUND': 'fda_orphan_drug.pubchem_cid'}}}, 'name': 'MyChem.info API', 'provided_by': 'infores:drugcentral', 'smartapi': {'id': '8f08d1446e0bb9c2b323713ce83e2bd3', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mychem.info/openapi_full.yml', 'ui': 'https://smart-api.info/ui/8f08d1446e0bb9c2b323713ce83e2bd3'}, 'tags': ['chemical', 'drug', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'umls', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'is_effective_for.umls'}}, 'server': 'https://biothings.ncats.io/idisk', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'umls'}}}, 'name': 'BioThings IDISK API', 'provided_by': 'infores:idisk', 'smartapi': {'id': '32f36164fabed5d3abe6c2fd899c9418', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/idisk/smartapi.yaml', 'ui': 'https://smart-api.info/ui/32f36164fabed5d3abe6c2fd899c9418'}, 'tags': ['chemical', 'disease', 'umls', 'phenotype', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.intervention.inxight, references.pmid, object.age_use, object.contra, object.intervention.int_class, object.qualscale_reclass, object.rev1_eff_reclass, object.timeframe, subject.clinical_summary, subject.freq_per_birth, subject.pattern_of_inheritance', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'subject.omim'}}, 'server': 'https://biothings.ncats.io/gtrx', 'support_batch': True}, 'response_mapping': {'treated_by': {'UNII': 'object.intervention.inxight', 'clinical_summary': 'subject.clinical_summary', 'condition_incidence': 'subject.freq_per_birth', 'condition_pattern_of_inheritance': 'subject.pattern_of_inheritance', 'intervention_MUST_be_started_within': 'object.timeframe', 'intervention_efficacy': 'object.rev1_eff_reclass', 'intervention_level_of_evidence_available': 'object.qualscale_reclass', 'intervention_may_be_started_in_this_age_group': 'object.age_use', 'is_intervention_contraindicated_in_certain_groups': 'object.contra', 'pubmed': 'references.pmid', 'type_of_intervention': 'object.intervention.int_class'}}}, 'name': 'BioThings GTRx API', 'provided_by': 'infores:gtrx', 'smartapi': {'id': '316eab811fd9ef1097df98bcaa9f7361', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/gtrx/gtrx.yaml', 'ui': 'https://smart-api.info/ui/316eab811fd9ef1097df98bcaa9f7361'}, 'tags': ['association', 'chemical', 'disease', 'metadata', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:nnon AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:vita AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:opco AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:TREATS AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"nnon\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"vita\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"opco\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"TREATS\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'treated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"treats\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'treated_by': {'CHEBI': 'subject.CHEBI', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.DRUGBANK,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"treats\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'treated_by': {'DRUGBANK': 'subject.DRUGBANK', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}], 'id': 'Disease-treated_by-SmallMolecule', 'predicate': 'treated_by'}) at 0x10ccb8bb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10ccba070>]) at 0x10c94e710>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10caa48d0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10caa4a50>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10caa6100>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x104623570>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10ccc01b0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}], 'id': 'Disease-location of-SmallMolecule', 'predicate': 'location of'}) at 0x10ccc0d70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10ccc2230>]) at 0x10c94ded0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x104623fc0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x1046238a0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ee760>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ec4b0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10ccc39f0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-treated_by-SmallMolecule', 'predicate': 'treated_by'}) at 0x10ccc4ef0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10ccc63b0>]) at 0x10c94e6b0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046efc00>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x1046efae0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ef9c0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046efb10>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10ccc84f0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}], 'id': 'PhenotypicFeature-is_assessed_by-SmallMolecule', 'predicate': 'is_assessed_by'}) at 0x10ccc90b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10ccca570>]) at 0x10c94fd30>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ef690>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x1046ef8d0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ec750>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ee7f0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10cccbd30>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'method': 'get', 'params': {'id': '{{ queryInputs | rmPrefix() }}', 'view': 'max'}, 'path': '/data/pathway/{id}', 'path_params': ['id'], 'server': 'https://api.pharmgkb.org/v1'}, 'response_mapping': {'related_to': {'PHARMGKB': 'data.relatedChemicals.id', 'input_name': 'data.name'}}}, 'name': 'PharmGKB REST API', 'smartapi': {'id': 'bde72db681ec0b8f9eeb67bb6b8dd72c', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/pharmgkb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/bde72db681ec0b8f9eeb67bb6b8dd72c'}, 'tags': ['translator', 'auth', 'data', 'report', 'user', 'services'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-related_to-SmallMolecule', 'predicate': 'related_to'}) at 0x10ccd39f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10ccd4ef0>]) at 0x10cbc26d0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ef510>, <mgclient.Node(id=145, labels={'Entity'}, properties={'name': 'DiseaseOrPhenotypicFeature'}) at 0x1046ef540>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ec720>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046edaa0>], relationships=[<mgclient.Relationship(start_id=226, end_id=145, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-DiseaseOrPhenotypicFeature', 'predicate': 'associated_with'}) at 0x10ccd6ff0>, <mgclient.Relationship(start_id=145, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}], 'id': 'DiseaseOrPhenotypicFeature-ameliorates-SmallMolecule', 'predicate': 'ameliorates'}) at 0x10ccd7bb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10ccd90b0>]) at 0x10cbc2490>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ef960>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x1046efb70>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ef570>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ef930>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10ccdb1b0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}], 'id': 'PhenotypicFeature-is_side_effect_of-SmallMolecule', 'predicate': 'is_side_effect_of'}) at 0x10ccdbd30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cce9230>]) at 0x10cbc2210>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046eef40>, <mgclient.Node(id=145, labels={'Entity'}, properties={'name': 'DiseaseOrPhenotypicFeature'}) at 0x1046efc60>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ef660>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ef8a0>], relationships=[<mgclient.Relationship(start_id=226, end_id=145, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-DiseaseOrPhenotypicFeature', 'predicate': 'associated_with'}) at 0x10cceb330>, <mgclient.Relationship(start_id=145, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'DiseaseOrPhenotypicFeature-treated_by-SmallMolecule', 'predicate': 'treated_by'}) at 0x10ccec830>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10ccedcf0>]) at 0x10cbc1630>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ec180>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x1046efcf0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ee940>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ec870>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10ccefe30>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence.transltr.io', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API', 'smartapi': {'id': '71fa2e0f0f1fe1ec67f4ddb719db5ef3', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/main/openapi.yaml', 'ui': 'https://smart-api.info/ui/71fa2e0f0f1fe1ec67f4ddb719db5ef3'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence-trapi-14-p36smkc6hq-uc.a.run.app', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API (TRAPI 1.4.0)', 'smartapi': {'id': '2d7efe7d2dabb473c5856e7da98fda5e', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/TRAPI_1.4/openapi.yaml', 'ui': 'https://smart-api.info/ui/2d7efe7d2dabb473c5856e7da98fda5e'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'associatedWith.mentions.chemicals.mesh, associatedWith.figureUrl,associatedWith.pmc,associatedWith.title', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'associatedWith.mentions.diseases.mesh'}}, 'server': 'https://biothings.ncats.io/pfocr', 'support_batch': True}, 'response_mapping': {'occurs_together_in_literature_with': {'MESH': 'associatedWith.mentions.chemicals.mesh', 'figure_download_url': 'associatedWith.figureUrl', 'figure_title': 'associatedWith.title', 'pmc_reference': 'associatedWith.pmc'}}}, 'name': 'BioThings PFOCR API', 'provided_by': 'infores:pfocr', 'smartapi': {'id': 'edeb26858bd27d0322af93e7a9e08761', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/pfocr/smartapi.yaml', 'ui': 'https://smart-api.info/ui/edeb26858bd27d0322af93e7a9e08761'}, 'tags': ['geneset', 'gene', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'Disease-occurs_together_in_literature_with-SmallMolecule', 'predicate': 'occurs_together_in_literature_with'}) at 0x10ccf79b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cfbceb0>]) at 0x10cbc2410>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ee640>, <mgclient.Node(id=153, labels={'Entity'}, properties={'name': 'NamedThing'}) at 0x1046ec7b0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046efd50>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ef1e0>], relationships=[<mgclient.Relationship(start_id=226, end_id=153, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-NamedThing', 'predicate': 'associated_with'}) at 0x10cfbe670>, <mgclient.Relationship(start_id=153, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence.transltr.io', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API', 'smartapi': {'id': '71fa2e0f0f1fe1ec67f4ddb719db5ef3', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/main/openapi.yaml', 'ui': 'https://smart-api.info/ui/71fa2e0f0f1fe1ec67f4ddb719db5ef3'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence-trapi-14-p36smkc6hq-uc.a.run.app', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API (TRAPI 1.4.0)', 'smartapi': {'id': '2d7efe7d2dabb473c5856e7da98fda5e', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/TRAPI_1.4/openapi.yaml', 'ui': 'https://smart-api.info/ui/2d7efe7d2dabb473c5856e7da98fda5e'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}], 'id': 'NamedThing-occurs_together_in_literature_with-SmallMolecule', 'predicate': 'occurs_together_in_literature_with'}) at 0x10cfbfa30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cfc0f30>]) at 0x10cbc2350>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ef840>, <mgclient.Node(id=153, labels={'Entity'}, properties={'name': 'NamedThing'}) at 0x1046eee50>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ecf60>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046eef10>], relationships=[<mgclient.Relationship(start_id=226, end_id=153, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-subclass_of-NamedThing', 'predicate': 'subclass_of'}) at 0x10cfc26f0>, <mgclient.Relationship(start_id=153, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence.transltr.io', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API', 'smartapi': {'id': '71fa2e0f0f1fe1ec67f4ddb719db5ef3', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/main/openapi.yaml', 'ui': 'https://smart-api.info/ui/71fa2e0f0f1fe1ec67f4ddb719db5ef3'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence-trapi-14-p36smkc6hq-uc.a.run.app', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API (TRAPI 1.4.0)', 'smartapi': {'id': '2d7efe7d2dabb473c5856e7da98fda5e', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/TRAPI_1.4/openapi.yaml', 'ui': 'https://smart-api.info/ui/2d7efe7d2dabb473c5856e7da98fda5e'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}], 'id': 'NamedThing-occurs_together_in_literature_with-SmallMolecule', 'predicate': 'occurs_together_in_literature_with'}) at 0x10cfc3ab0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cfc8fb0>]) at 0x10cbc1cd0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ec090>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x1046ee340>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046efdb0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ed1a0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10cfca770>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence.transltr.io', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API', 'smartapi': {'id': '71fa2e0f0f1fe1ec67f4ddb719db5ef3', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/main/openapi.yaml', 'ui': 'https://smart-api.info/ui/71fa2e0f0f1fe1ec67f4ddb719db5ef3'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence-trapi-14-p36smkc6hq-uc.a.run.app', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API (TRAPI 1.4.0)', 'smartapi': {'id': '2d7efe7d2dabb473c5856e7da98fda5e', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/TRAPI_1.4/openapi.yaml', 'ui': 'https://smart-api.info/ui/2d7efe7d2dabb473c5856e7da98fda5e'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}], 'id': 'Pathway-occurs_together_in_literature_with-SmallMolecule', 'predicate': 'occurs_together_in_literature_with'}) at 0x10cfcbb30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cfd5030>]) at 0x10cbc1670>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046eed60>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x1046ecf90>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ec150>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046ec0f0>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10cfd7130>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence.transltr.io', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API', 'smartapi': {'id': '71fa2e0f0f1fe1ec67f4ddb719db5ef3', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/main/openapi.yaml', 'ui': 'https://smart-api.info/ui/71fa2e0f0f1fe1ec67f4ddb719db5ef3'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cooccurrence-trapi-14-p36smkc6hq-uc.a.run.app', 'support_batch': True}}, 'name': 'Text Mined Cooccurrence API (TRAPI 1.4.0)', 'smartapi': {'id': '2d7efe7d2dabb473c5856e7da98fda5e', 'metadata': 'https://raw.githubusercontent.com/UCDenver-ccp/Translator-TM-Provider-Cooccurrence/TRAPI_1.4/openapi.yaml', 'ui': 'https://smart-api.info/ui/2d7efe7d2dabb473c5856e7da98fda5e'}, 'tags': ['meta_knowledge_graph', 'overlay', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-occurs_together_in_literature_with-SmallMolecule', 'predicate': 'occurs_together_in_literature_with'}) at 0x10cfd8e30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10cfda2f0>]) at 0x10cbc2870>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046ef420>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x1046ef9f0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046ed950>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046eed90>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10cfe0430>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:nnon AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:opco AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"nnon\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"opco\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'PhenotypicFeature-caused_by-SmallMolecule', 'predicate': 'caused_by'}) at 0x10d0121f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d0136b0>]) at 0x10cbc2c70>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c910720>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c913b10>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c910e10>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c912b20>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d0197f0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-has_biomarker-SmallMolecule', 'predicate': 'has_biomarker'}) at 0x10d01a3f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d01b8b0>]) at 0x10cbc2d10>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c912f10>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c9114d0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9108a0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c911d10>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d01d9f0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'PhenotypicFeature-affected_by-SmallMolecule', 'predicate': 'affected_by'}) at 0x10d069bf0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d06b0b0>]) at 0x10cbc3bf0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c910510>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c911980>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c913180>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c911e60>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d06d230>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Disease-causes-SmallMolecule', 'predicate': 'causes'}) at 0x10d06f1b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d0706b0>]) at 0x10cbc3b30>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x104670c60>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x104670a20>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x1046708a0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x101a83d80>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d0727b0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-has_contraindication-SmallMolecule', 'predicate': 'has_contraindication'}) at 0x10d073370>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d078870>]) at 0x10cbc3a10>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca79b00>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10ca786f0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca79320>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca7ad60>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d07a9b0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.DRUGBANK,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"contributes_to\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'contribution_from': {'DRUGBANK': 'subject.DRUGBANK', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"contributes_to\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'contribution_from': {'CHEBI': 'subject.CHEBI', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}], 'id': 'Disease-contribution_from-SmallMolecule', 'predicate': 'contribution_from'}) at 0x10d07dfb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d07f470>]) at 0x10cbc39f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca79c50>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10ca799e0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca797a0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca7ba20>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d0815f0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'drugcentral.xrefs.chembl_id', 'size': 1000}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'drugcentral.drug_use.contraindication.umls_cui'}}, 'server': 'https://mychem.info/v1', 'support_batch': True}, 'response_mapping': {'has_contraindication': {'CHEMBL.COMPOUND': 'drugcentral.xrefs.chembl_id'}}}, 'name': 'MyChem.info API', 'provided_by': 'infores:drugcentral', 'smartapi': {'id': '8f08d1446e0bb9c2b323713ce83e2bd3', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mychem.info/openapi_full.yml', 'ui': 'https://smart-api.info/ui/8f08d1446e0bb9c2b323713ce83e2bd3'}, 'tags': ['chemical', 'drug', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'Disease-has_contraindication-SmallMolecule', 'predicate': 'has_contraindication'}) at 0x10d083fb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d08d4b0>]) at 0x10cbc2fd0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca7bc30>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10ca798c0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca7a520>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca7b630>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d08f5f0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-entity_positively_regulated_by_entity-SmallMolecule', 'predicate': 'entity_positively_regulated_by_entity'}) at 0x10d0941f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d0956b0>]) at 0x10cbc3050>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca8e850>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10ca8ec40>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca8e760>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca8eca0>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d0977b0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-has_real_world_evidence_of_association_with-SmallMolecule', 'predicate': 'has_real_world_evidence_of_association_with'}) at 0x10d0a03b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d0a1870>]) at 0x10cbc30f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca8d2f0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10ca8dec0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca8da40>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca8f0c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d0a39b0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:nnon AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:nnon AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AFFECTS AND subject.semantic_type_abbreviation:opco AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:AUGMENTS AND subject.semantic_type_abbreviation:opco AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"nnon\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"nnon\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AFFECTS\",\"opco\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"AUGMENTS\",\"opco\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'affected_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'Disease-affected_by-SmallMolecule', 'predicate': 'affected_by'}) at 0x10d1096b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d10ab70>]) at 0x10cbc3870>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca8ced0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10ca8cb70>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca8cd50>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cb167c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d10ccf0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-has_biomarker-SmallMolecule', 'predicate': 'has_biomarker'}) at 0x10d10d8f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d10edb0>]) at 0x10cbc3910>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cb16280>, <mgclient.Node(id=168, labels={'Entity'}, properties={'name': 'Behavior'}) at 0x10cb17a50>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cb14c00>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cb16d30>], relationships=[<mgclient.Relationship(start_id=226, end_id=168, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-related_to-Behavior', 'predicate': 'related_to'}) at 0x10d1145b0>, <mgclient.Relationship(start_id=168, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Behavior-related_to-SmallMolecule', 'predicate': 'related_to'}) at 0x10d116eb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d11c3b0>]) at 0x10d124070>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cb17420>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10cb14240>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10cb161f0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10cb16670>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d11e4b0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.DRUGBANK,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"contributes_to\"]\\') }} ],\\n  \"scopes\": [\"object.HP\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'contribution_from': {'DRUGBANK': 'subject.DRUGBANK', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI,association.edge_attributes', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"contributes_to\"]\\') }} ],\\n  \"scopes\": [\"object.HP\", \"association.edge_label\"]\\n}'}, 'server': 'https://biothings.ncats.io/text_mining_targeted_association', 'support_batch': True}, 'response_mapping': {'contribution_from': {'CHEBI': 'subject.CHEBI', 'edge-attributes': 'association.edge_attributes'}}}, 'name': 'Text Mining Targeted Association API', 'smartapi': {'id': '978fe380a147a8641caf72320862697b', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/text_mining/smartapi.yaml', 'ui': 'https://smart-api.info/ui/978fe380a147a8641caf72320862697b'}, 'tags': ['gene', 'chemical', 'disease', 'phenotype', 'association', 'annotation', 'query', 'translator', 'biothings', 'biothings_graph'], 'x-translator': {'component': 'KP', 'team': ['Text Mining Provider', 'Service Provider']}}], 'id': 'PhenotypicFeature-contribution_from-SmallMolecule', 'predicate': 'contribution_from'}) at 0x10d129ab0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d12af70>]) at 0x10d124150>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca5eb20>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10ca5f5d0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca5d860>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca5da10>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d12d0b0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-is_ameliorated_by-SmallMolecule', 'predicate': 'is_ameliorated_by'}) at 0x10d12dc70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d12f130>]) at 0x10d1241f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca5d1a0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10ca5e880>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca5ebb0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca5ccf0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d1392b0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-is_ameliorated_by-SmallMolecule', 'predicate': 'is_ameliorated_by'}) at 0x10d139e70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d13b330>]) at 0x10d124290>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca5d560>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10ca5e520>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90a040>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90a1c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d13d4b0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-decreases_abundance_of-SmallMolecule', 'predicate': 'decreases_abundance_of'}) at 0x10d13e070>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d13f530>]) at 0x10d124330>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90a490>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90aca0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90a520>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90a910>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d1416b0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-increased_likelihood_associated_with-SmallMolecule', 'predicate': 'increased_likelihood_associated_with'}) at 0x10d142270>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d143730>]) at 0x10d1243d0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90a4f0>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c908390>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9082d0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c909da0>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d14d870>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'PhenotypicFeature-has_predisposing_factor-SmallMolecule', 'predicate': 'has_predisposing_factor'}) at 0x10d168df0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d16a2b0>]) at 0x10d124670>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c908210>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c908300>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c908030>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9083c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d1703f0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:DISRUPTS AND subject.semantic_type_abbreviation:nnon AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"DISRUPTS\",\"nnon\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'disrupted_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'PhenotypicFeature-disrupted_by-SmallMolecule', 'predicate': 'disrupted_by'}) at 0x10d19cfb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d19e470>]) at 0x10d1249d0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c908570>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c909ef0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c909f20>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c909fe0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d1a05f0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | replPrefix(\\'NCIT\\') | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.NCIT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | addPrefix(\"SNOMEDCT\") | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.SNOMEDCT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | replPrefix(\\'NCIT\\') | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.NCIT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | addPrefix(\"SNOMEDCT\") | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.SNOMEDCT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | replPrefix(\\'NCIT\\') | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.NCIT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | addPrefix(\"SNOMEDCT\") | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.SNOMEDCT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.MONDO\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | addPrefix(\"SNOMEDCT\") | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"Disease\"]\\') }} ],\\n  \"scopes\": [\"object.SNOMEDCT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Disease-associated_with-SmallMolecule', 'predicate': 'associated_with'}) at 0x10d1b9c70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d1bb130>]) at 0x10d124bd0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9081b0>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c908450>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c908600>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90a070>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d1bd270>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.HP\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | replPrefix(\\'NCIT\\') | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.NCIT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | addPrefix(\"SNOMEDCT\") | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.SNOMEDCT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.HP\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | replPrefix(\\'NCIT\\') | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_increased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.NCIT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.HP\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | replPrefix(\\'NCIT\\') | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.NCIT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.CHEBI, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | addPrefix(\"SNOMEDCT\") | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.SNOMEDCT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'CHEBI': 'subject.CHEBI', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.HP\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.UNII, association.provenance, association.auc_roc,association.p_values, association.feature_coefficient,association.odd_ratio, association.classifier,association.original_predicate,association.provided_date, subject.name,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\\n  \"q\": [ {{ queryInputs | replPrefix(\\'NCIT\\') | wrap( \\'[\"\\' , \\'\",\"ChemicalSubstance\", \"associated_with_decreased_likelihood_of\", \"PhenotypicFeature\"]\\') }} ],\\n  \"scopes\": [\"object.NCIT\", \"subject.type\", \"association.predicate\", \"object.type\"]\\n}'}, 'server': 'https://biothings.ncats.io/multiomics_ehr_risk_kp', 'support_batch': True}, 'response_mapping': {'associated_with': {'UNII': 'subject.UNII', 'auc_roc': 'association.auc_roc', 'biolink:original_predicate': 'association.original_predicate', 'classifier_used': 'association.classifier', 'date_provided': 'association.provided_date', 'feature_coefficient': 'association.feature_coefficient', 'input_name': 'object.name', 'model_url': 'association.provenance', 'odds_ratio': 'association.odd_ratio', 'output_name': 'subject.name', 'p-value': 'association.p_values'}}}, 'name': 'Multiomics EHR Risk KP API', 'smartapi': {'id': 'd86a24f6027ffe778f84ba10a7a1861a', 'metadata': 'https://raw.githubusercontent.com/Hadlock-Lab/clinical_risk_kp/master/ehr_risk_kp.yaml', 'ui': 'https://smart-api.info/ui/d86a24f6027ffe778f84ba10a7a1861a'}, 'tags': ['disease', 'chemical', 'phenotype', 'procedure', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Multiomics Provider', 'Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'PhenotypicFeature-associated_with-SmallMolecule', 'predicate': 'associated_with'}) at 0x10d1d9af0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d1dafb0>]) at 0x10d124db0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90a8b0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90a9a0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90ab80>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90ad30>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d1dd130>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PART_OF AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_part': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PART_OF AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_part': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PART_OF\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_part': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PART_OF\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_part': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Disease-has_part-SmallMolecule', 'predicate': 'has_part'}) at 0x10d1e83f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d1e98b0>]) at 0x10d124ed0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90abb0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90aac0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90aa60>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90aa30>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d1eb9f0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-decreased_likelihood_associated_with-SmallMolecule', 'predicate': 'decreased_likelihood_associated_with'}) at 0x10d1f05f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d1f1ab0>]) at 0x10d124f70>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90a820>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90a7c0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90a700>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90a760>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d1f3bf0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PRODUCES AND object.semantic_type_abbreviation:bacs AND pmid_count:>5 AND subject.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PRODUCES AND object.semantic_type_abbreviation:horm AND pmid_count:>5 AND subject.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PRODUCES AND object.semantic_type_abbreviation:nsba AND pmid_count:>5 AND subject.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PRODUCES AND object.semantic_type_abbreviation:vita AND pmid_count:>5 AND subject.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,pmid,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PRODUCES\",\"bacs\",1,1]\\' ) }} ], \"scopes\": [\"subject.umls\", \"predicate\", \"object.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,pmid,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PRODUCES\",\"horm\",1,1]\\' ) }} ], \"scopes\": [\"subject.umls\", \"predicate\", \"object.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,pmid,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PRODUCES\",\"nsba\",1,1]\\' ) }} ], \"scopes\": [\"subject.umls\", \"predicate\", \"object.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,pmid,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PRODUCES\",\"vita\",1,1]\\' ) }} ], \"scopes\": [\"subject.umls\", \"predicate\", \"object.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'produces': {'UMLS': 'object.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Disease-produces-SmallMolecule', 'predicate': 'produces'}) at 0x10d20e570>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d20fa30>]) at 0x10d125110>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90a580>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90a430>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90a310>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90a370>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d219230>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-interacts_with-SmallMolecule', 'predicate': 'interacts_with'}) at 0x10d21b1b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d21c6b0>]) at 0x10d1251b0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90a160>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90a0d0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90a0a0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c909fb0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d21de70>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-treats-SmallMolecule', 'predicate': 'treats'}) at 0x10d21ea70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d21ff30>]) at 0x10d125230>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9085d0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c908480>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c908330>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c908270>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d225730>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-is_ameliorated_by-SmallMolecule', 'predicate': 'is_ameliorated_by'}) at 0x10d2262f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d2277b0>]) at 0x10d1252b0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c908180>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c9080c0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c908a50>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90b030>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d22cfb0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-regulates-SmallMolecule', 'predicate': 'regulates'}) at 0x10d22ef30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d230430>]) at 0x10d125350>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c908240>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c9082a0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9084b0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c908630>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d231bf0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-causes-SmallMolecule', 'predicate': 'causes'}) at 0x10d233b70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d239070>]) at 0x10d1253f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90a3a0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90a5b0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90a880>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90a3d0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d23a830>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/sri-reference-kg/1.3', 'support_batch': True}}, 'name': 'Sri-reference-kg(Trapi v1.3.0)', 'smartapi': {'id': '00f8ece82e1a9167e89902188c7506ec', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores%3Asri-reference-kg/1.3.0', 'ui': 'https://smart-api.info/ui/00f8ece82e1a9167e89902188c7506ec'}, 'tags': ['translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Standards Reference Implementation Team']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/uberongraph/1.3', 'support_batch': True}}, 'name': 'Automat-uberongraph(Trapi v1.3.0)', 'smartapi': {'id': 'ef9027a7d2246c6540cc7b3ce202d89f', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-uberongraph/1.3.0', 'ui': 'https://smart-api.info/ui/ef9027a7d2246c6540cc7b3ce202d89f'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/hmdb/1.3', 'support_batch': True}}, 'name': 'Automat-hmdb(Trapi v1.3.0)', 'smartapi': {'id': 'ee2d2eae42ca30fe82946d3f42febaa0', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-hmdb/1.3.0', 'ui': 'https://smart-api.info/ui/ee2d2eae42ca30fe82946d3f42febaa0'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/robokopkg/1.3', 'support_batch': True}}, 'name': 'Automat-robokop(Trapi v1.3.0)', 'smartapi': {'id': '123ee14a757d01213707e1f90b01736a', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-robokop/1.3.0', 'ui': 'https://smart-api.info/ui/123ee14a757d01213707e1f90b01736a'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'method': 'get', 'params': {'object1Id': '{{ queryInputs | rmPrefix() }}', 'object2Type': 'Chemical'}, 'path': '/data/connection', 'server': 'https://api.pharmgkb.org/v1'}, 'response_mapping': {'has_participant': {'PHARMGKB': 'data.obj2Id', 'input_name': 'data.obj1Name', 'output_name': 'data.obj2Name', 'pharmgkb_source_of_connection': 'data.relType', 'pubmed': 'data.pmid'}}}, 'name': 'PharmGKB REST API', 'smartapi': {'id': 'bde72db681ec0b8f9eeb67bb6b8dd72c', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/pharmgkb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/bde72db681ec0b8f9eeb67bb6b8dd72c'}, 'tags': ['translator', 'auth', 'data', 'report', 'user', 'services'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-has_participant-SmallMolecule', 'predicate': 'has_participant'}) at 0x10d2450f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d2465b0>]) at 0x10d1254f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90ac10>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c90a6a0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90aaf0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x1046c3c60>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d24c6f0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/sri-reference-kg/1.3', 'support_batch': True}}, 'name': 'Sri-reference-kg(Trapi v1.3.0)', 'smartapi': {'id': '00f8ece82e1a9167e89902188c7506ec', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores%3Asri-reference-kg/1.3.0', 'ui': 'https://smart-api.info/ui/00f8ece82e1a9167e89902188c7506ec'}, 'tags': ['translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Standards Reference Implementation Team']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:vita AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"vita\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'PhenotypicFeature-related_to-SmallMolecule', 'predicate': 'related_to'}) at 0x10d281bf0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d2830b0>]) at 0x10d1258b0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x1046c35a0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x1046c1b30>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90f780>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90f570>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d285230>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/sri-reference-kg/1.3', 'support_batch': True}}, 'name': 'Sri-reference-kg(Trapi v1.3.0)', 'smartapi': {'id': '00f8ece82e1a9167e89902188c7506ec', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores%3Asri-reference-kg/1.3.0', 'ui': 'https://smart-api.info/ui/00f8ece82e1a9167e89902188c7506ec'}, 'tags': ['translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Standards Reference Implementation Team']}}, {'bte': {'query_operation': {'method': 'get', 'params': {'format': 'json', 'inputTermSearchType': 'directAssociations', 'inputTerms': \"{{ queryInputs | replPrefix('MESH') }}\", 'inputType': 'disease', 'report': 'chems_curated'}, 'path': '/tools/batchQuery.go', 'server': 'http://ctdbase.org'}, 'response_mapping': {'related_to': {'MESH': 'data.ChemicalID', 'ctd_chemical_disease_interaction_types': 'data.DirectEvidence', 'pubmed': 'data.PubMedIDs'}}}, 'name': 'CTD API', 'smartapi': {'id': '0212611d1c670f9107baf00b77f0889a', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/CTD/smartapi.yaml', 'ui': 'https://smart-api.info/ui/0212611d1c670f9107baf00b77f0889a'}, 'tags': ['translator', 'ctd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'get', 'params': {'format': 'json', 'inputTermSearchType': 'directAssociations', 'inputTerms': \"{{ queryInputs | replPrefix('OMIM') }}\", 'inputType': 'disease', 'report': 'chems_curated'}, 'path': '/tools/batchQuery.go', 'server': 'http://ctdbase.org'}, 'response_mapping': {'related_to': {'MESH': 'data.ChemicalID', 'ctd_chemical_disease_interaction_types': 'data.DirectEvidence', 'pubmed': 'data.PubMedIDs'}}}, 'name': 'CTD API', 'smartapi': {'id': '0212611d1c670f9107baf00b77f0889a', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/CTD/smartapi.yaml', 'ui': 'https://smart-api.info/ui/0212611d1c670f9107baf00b77f0889a'}, 'tags': ['translator', 'ctd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://molepro-trapi.transltr.io/molepro/trapi/v1.3', 'support_batch': True}}, 'name': 'MolePro', 'smartapi': {'id': '940677a65cae38c9a482e54e5c6794f7', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/molecular_data_provider/MolePro-TRAPI-1.3.yaml', 'ui': 'https://smart-api.info/ui/940677a65cae38c9a482e54e5c6794f7'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Molecular Data Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'ctd.chemical_related_to_disease.mesh_chemical_id, ctd.chemical_related_to_disease.direct_evidence, ctd.chemical_related_to_disease.pubmed'}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'ctd.mesh'}}, 'server': 'https://mydisease.info/v1', 'support_batch': True}, 'response_mapping': {'related_to': {'MESH': 'ctd.chemical_related_to_disease.mesh_chemical_id', 'ctd_evidence_type': 'ctd.chemical_related_to_disease.direct_evidence', 'pubmed': 'ctd.chemical_related_to_disease.pubmed'}}}, 'name': 'MyDisease.info API', 'provided_by': 'infores:ctd', 'smartapi': {'id': '671b45c0301c8624abbd26ae78449ca2', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mydisease.info/smartapi.yaml', 'ui': 'https://smart-api.info/ui/671b45c0301c8624abbd26ae78449ca2'}, 'tags': ['disease', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'ctd.chemical_related_to_disease.mesh_chemical_id, ctd.chemical_related_to_disease.direct_evidence, ctd.chemical_related_to_disease.pubmed'}, 'path': '/query', 'request_body': {'body': {'q': '{{ queryInputs }}', 'scopes': 'ctd.omim'}}, 'server': 'https://mydisease.info/v1', 'support_batch': True}, 'response_mapping': {'related_to': {'MESH': 'ctd.chemical_related_to_disease.mesh_chemical_id', 'ctd_evidence_type': 'ctd.chemical_related_to_disease.direct_evidence', 'pubmed': 'ctd.chemical_related_to_disease.pubmed'}}}, 'name': 'MyDisease.info API', 'provided_by': 'infores:ctd', 'smartapi': {'id': '671b45c0301c8624abbd26ae78449ca2', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/mydisease.info/smartapi.yaml', 'ui': 'https://smart-api.info/ui/671b45c0301c8624abbd26ae78449ca2'}, 'tags': ['disease', 'annotation', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'method': 'get', 'params': {'object1Id': '{{ queryInputs | rmPrefix() }}', 'object2Type': 'Chemical'}, 'path': '/data/connection', 'server': 'https://api.pharmgkb.org/v1'}, 'response_mapping': {'related_to': {'PHARMGKB': 'data.obj2Id', 'input_name': 'data.obj1Name', 'output_name': 'data.obj2Name', 'pharmgkb_source_of_connection': 'data.relType', 'pubmed': 'data.pmid'}}}, 'name': 'PharmGKB REST API', 'smartapi': {'id': 'bde72db681ec0b8f9eeb67bb6b8dd72c', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/pharmgkb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/bde72db681ec0b8f9eeb67bb6b8dd72c'}, 'tags': ['translator', 'auth', 'data', 'report', 'user', 'services'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:vita AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:nnon AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:ASSOCIATED_WITH AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"vita\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"nnon\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"ASSOCIATED_WITH\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'related_to': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Disease-related_to-SmallMolecule', 'predicate': 'related_to'}) at 0x10d2c15b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d2c2a70>]) at 0x10d125cf0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90f6c0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90f6f0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90f720>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90ef10>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d2c8bf0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/sri-reference-kg/1.3', 'support_batch': True}}, 'name': 'Sri-reference-kg(Trapi v1.3.0)', 'smartapi': {'id': '00f8ece82e1a9167e89902188c7506ec', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores%3Asri-reference-kg/1.3.0', 'ui': 'https://smart-api.info/ui/00f8ece82e1a9167e89902188c7506ec'}, 'tags': ['translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Standards Reference Implementation Team']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:nnon AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:opco AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:CAUSES AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"nnon\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"opco\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"CAUSES\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'caused_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'Disease-caused_by-SmallMolecule', 'predicate': 'caused_by'}) at 0x10d2fb1b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3006b0>]) at 0x10d126130>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90ef70>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90efa0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90efd0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90f7e0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d3027f0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/robokopkg/1.3', 'support_batch': True}}, 'name': 'Automat-robokop(Trapi v1.3.0)', 'smartapi': {'id': '123ee14a757d01213707e1f90b01736a', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-robokop/1.3.0', 'ui': 'https://smart-api.info/ui/123ee14a757d01213707e1f90b01736a'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/ctd/1.3', 'support_batch': True}}, 'name': 'Automat-ctd(Trapi v1.3.0)', 'smartapi': {'id': '89aa98098ba4329aa49b43ff8d21ffbb', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-ctd/1.3.0', 'ui': 'https://smart-api.info/ui/89aa98098ba4329aa49b43ff8d21ffbb'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': 'e892e3cea582cd28f4e368e463876d21', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/trapi/cohd/TRAPI14.yaml', 'ui': 'https://smart-api.info/ui/e892e3cea582cd28f4e368e463876d21'}, 'tags': ['meta_knowledge_graph', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd-api.transltr.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': '51c178099fa2dc99b5d8fff8bf9f1a0d', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/master/cohd/TranslatorReasonerAPI.yaml', 'ui': 'https://smart-api.info/ui/51c178099fa2dc99b5d8fff8bf9f1a0d'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'Disease-positively_correlated_with-SmallMolecule', 'predicate': 'positively_correlated_with'}) at 0x10d309fb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d30b470>]) at 0x10d126210>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90f8a0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90f8d0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90f900>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90f9c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d3115f0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/robokopkg/1.3', 'support_batch': True}}, 'name': 'Automat-robokop(Trapi v1.3.0)', 'smartapi': {'id': '123ee14a757d01213707e1f90b01736a', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-robokop/1.3.0', 'ui': 'https://smart-api.info/ui/123ee14a757d01213707e1f90b01736a'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/ctd/1.3', 'support_batch': True}}, 'name': 'Automat-ctd(Trapi v1.3.0)', 'smartapi': {'id': '89aa98098ba4329aa49b43ff8d21ffbb', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-ctd/1.3.0', 'ui': 'https://smart-api.info/ui/89aa98098ba4329aa49b43ff8d21ffbb'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': 'e892e3cea582cd28f4e368e463876d21', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/trapi/cohd/TRAPI14.yaml', 'ui': 'https://smart-api.info/ui/e892e3cea582cd28f4e368e463876d21'}, 'tags': ['meta_knowledge_graph', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd-api.transltr.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': '51c178099fa2dc99b5d8fff8bf9f1a0d', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/master/cohd/TranslatorReasonerAPI.yaml', 'ui': 'https://smart-api.info/ui/51c178099fa2dc99b5d8fff8bf9f1a0d'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'Disease-negatively_correlated_with-SmallMolecule', 'predicate': 'negatively_correlated_with'}) at 0x10d318db0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d31a270>]) at 0x10d1262f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90fa80>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c90fab0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90fc90>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e250>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d3203b0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/robokopkg/1.3', 'support_batch': True}}, 'name': 'Automat-robokop(Trapi v1.3.0)', 'smartapi': {'id': '123ee14a757d01213707e1f90b01736a', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-robokop/1.3.0', 'ui': 'https://smart-api.info/ui/123ee14a757d01213707e1f90b01736a'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/ctd/1.3', 'support_batch': True}}, 'name': 'Automat-ctd(Trapi v1.3.0)', 'smartapi': {'id': '89aa98098ba4329aa49b43ff8d21ffbb', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-ctd/1.3.0', 'ui': 'https://smart-api.info/ui/89aa98098ba4329aa49b43ff8d21ffbb'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': 'e892e3cea582cd28f4e368e463876d21', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/trapi/cohd/TRAPI14.yaml', 'ui': 'https://smart-api.info/ui/e892e3cea582cd28f4e368e463876d21'}, 'tags': ['meta_knowledge_graph', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd-api.transltr.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': '51c178099fa2dc99b5d8fff8bf9f1a0d', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/master/cohd/TranslatorReasonerAPI.yaml', 'ui': 'https://smart-api.info/ui/51c178099fa2dc99b5d8fff8bf9f1a0d'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'PhenotypicFeature-positively_correlated_with-SmallMolecule', 'predicate': 'positively_correlated_with'}) at 0x10d323b30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d32d030>]) at 0x10d1263d0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90f090>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c90f0c0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90f0f0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90f240>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d32f130>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/robokopkg/1.3', 'support_batch': True}}, 'name': 'Automat-robokop(Trapi v1.3.0)', 'smartapi': {'id': '123ee14a757d01213707e1f90b01736a', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-robokop/1.3.0', 'ui': 'https://smart-api.info/ui/123ee14a757d01213707e1f90b01736a'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/ctd/1.3', 'support_batch': True}}, 'name': 'Automat-ctd(Trapi v1.3.0)', 'smartapi': {'id': '89aa98098ba4329aa49b43ff8d21ffbb', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-ctd/1.3.0', 'ui': 'https://smart-api.info/ui/89aa98098ba4329aa49b43ff8d21ffbb'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Ranking Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': 'e892e3cea582cd28f4e368e463876d21', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/trapi/cohd/TRAPI14.yaml', 'ui': 'https://smart-api.info/ui/e892e3cea582cd28f4e368e463876d21'}, 'tags': ['meta_knowledge_graph', 'query', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://cohd-api.transltr.io/api', 'support_batch': True}}, 'name': 'COHD TRAPI', 'smartapi': {'id': '51c178099fa2dc99b5d8fff8bf9f1a0d', 'metadata': 'https://raw.githubusercontent.com/WengLab-InformaticsResearch/cohd_api/master/cohd/TranslatorReasonerAPI.yaml', 'ui': 'https://smart-api.info/ui/51c178099fa2dc99b5d8fff8bf9f1a0d'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Clinical Data Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'PhenotypicFeature-negatively_correlated_with-SmallMolecule', 'predicate': 'negatively_correlated_with'}) at 0x10d3368f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d337db0>]) at 0x10d1264b0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90f3c0>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c90f1b0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90f2d0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90f330>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d339ef0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'PhenotypicFeature-correlated_with-SmallMolecule', 'predicate': 'correlated_with'}) at 0x10d341070>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d342530>]) at 0x10d126570>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90f150>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90f750>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90d0b0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90fe10>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d343cf0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-response_increased_by-SmallMolecule', 'predicate': 'response_increased_by'}) at 0x10d3488f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d349db0>]) at 0x10d1265f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90e1f0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90e340>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90e430>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e8e0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d34b570>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-part_of-SmallMolecule', 'predicate': 'part_of'}) at 0x10d3501b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d351670>]) at 0x10d126670>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90ea00>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90ea30>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90eac0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90eaf0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d352e30>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-has_capability-SmallMolecule', 'predicate': 'has_capability'}) at 0x10d353a30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d358f30>]) at 0x10d1266f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90ec40>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90ecd0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90ed00>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90ed30>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d35a6f0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-participates_in-SmallMolecule', 'predicate': 'participates_in'}) at 0x10d35b2f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d35c7f0>]) at 0x10d126770>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90ee80>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90e550>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90e580>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e5e0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d35dfb0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-contributes_to-SmallMolecule', 'predicate': 'contributes_to'}) at 0x10d35ebb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3680b0>]) at 0x10d1267f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90e6d0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90e730>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90e4c0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e460>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d369870>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-caused_by-SmallMolecule', 'predicate': 'caused_by'}) at 0x10d36a470>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d36b930>]) at 0x10d126870>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90e3a0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90e130>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90e160>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e1c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d36d130>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-enabled_by-SmallMolecule', 'predicate': 'enabled_by'}) at 0x10d36dd30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d36f1f0>]) at 0x10d1268f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90de60>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c90de90>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90de30>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90fb40>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d375370>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-increases_abundance_of-SmallMolecule', 'predicate': 'increases_abundance_of'}) at 0x10d375f30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3773f0>]) at 0x10d126990>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90c2d0>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c90cd20>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90f540>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90f510>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d37d530>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-decreased_likelihood_associated_with-SmallMolecule', 'predicate': 'decreased_likelihood_associated_with'}) at 0x10d37e0f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d37f5b0>]) at 0x10d126a30>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90f480>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90f3f0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90e190>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e790>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d380db0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-has_catalyst-SmallMolecule', 'predicate': 'has_catalyst'}) at 0x10d3819b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d382e70>]) at 0x10d126ab0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90dec0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90d980>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90e610>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90ea60>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d384670>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-decreases_response_to-SmallMolecule', 'predicate': 'decreases_response_to'}) at 0x10d385230>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3866f0>]) at 0x10d126b30>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90db90>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c90e490>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90dda0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e9d0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d387eb0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-associated_with-SmallMolecule', 'predicate': 'associated_with'}) at 0x10d3927f0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d393cb0>]) at 0x10d126bd0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c90df50>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c90e0d0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c90e5b0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c90e220>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d395df0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-increases_abundance_of-SmallMolecule', 'predicate': 'increases_abundance_of'}) at 0x10d3969b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d397e70>]) at 0x10d126c70>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10cb70630>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10cb70060>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca32b80>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca32d60>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d39dff0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Disease-has_participant-SmallMolecule', 'predicate': 'has_participant'}) at 0x10d3a4930>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3a5df0>]) at 0x10d126d30>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca30e10>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10ca33f30>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10ca324f0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10ca33a20>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3a75b0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-has_part-SmallMolecule', 'predicate': 'has_part'}) at 0x10d3ad570>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3aea30>]) at 0x10d126dd0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10ca33c30>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10ca31d70>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9160a0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916040>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3b0230>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-is_exacerbated_by-SmallMolecule', 'predicate': 'is_exacerbated_by'}) at 0x10d3b0df0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3b22b0>]) at 0x10d126e50>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916400>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c9163a0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916370>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9163d0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3b3a70>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-ameliorates-SmallMolecule', 'predicate': 'ameliorates'}) at 0x10d3b86b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3b9b70>]) at 0x10d126ed0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916520>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c916760>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9166d0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9164c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3bb330>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-disrupted_by-SmallMolecule', 'predicate': 'disrupted_by'}) at 0x10d3bbf30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3bd430>]) at 0x10d126f50>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916640>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c916670>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916700>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9149c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3bebf0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-actively_involves-SmallMolecule', 'predicate': 'actively_involves'}) at 0x10d3bf7b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3c4cb0>]) at 0x10d126fd0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c914210>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c914300>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9143f0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9144e0>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d3c6df0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Disease-adverse_event_caused_by-SmallMolecule', 'predicate': 'adverse_event_caused_by'}) at 0x10d3c79b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3cceb0>]) at 0x10d127070>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9148a0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c914d50>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c914e40>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c914f30>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3ce670>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-has_upstream_actor-SmallMolecule', 'predicate': 'has_upstream_actor'}) at 0x10d3cf230>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3d0730>]) at 0x10d1270f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9152f0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c9153e0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9154d0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9155c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3d1ef0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-exacerbates-SmallMolecule', 'predicate': 'exacerbates'}) at 0x10d3d2af0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3d3fb0>]) at 0x10d127170>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c915b60>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c915b90>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c915bc0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c915bf0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3d57b0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-has_active_ingredient-SmallMolecule', 'predicate': 'has_active_ingredient'}) at 0x10d3d6370>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3d7830>]) at 0x10d1271f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c915cb0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c915d10>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c915d40>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c915d70>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3dd030>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-increases_response_to-SmallMolecule', 'predicate': 'increases_response_to'}) at 0x10d3ddbf0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3df0b0>]) at 0x10d127270>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c915e60>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c915e90>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c915ec0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c915ef0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3ec8b0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-primarily_composed_of-SmallMolecule', 'predicate': 'primarily_composed_of'}) at 0x10d3ed470>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3ee930>]) at 0x10d1272f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c915fb0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c915fe0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916100>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916130>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3f0130>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-regulated_by-SmallMolecule', 'predicate': 'regulated_by'}) at 0x10d3f0d30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3f21f0>]) at 0x10d127370>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9161c0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c9161f0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916220>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916250>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3f39b0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-response_affected_by-SmallMolecule', 'predicate': 'response_affected_by'}) at 0x10d3f85b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3f9a70>]) at 0x10d1273f0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916340>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c916790>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9165b0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916550>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d3fb230>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-affects-SmallMolecule', 'predicate': 'affects'}) at 0x10d3fdb70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d3ff030>]) at 0x10d127490>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916820>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c916850>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916880>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9168b0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d400830>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-similar_to-SmallMolecule', 'predicate': 'similar_to'}) at 0x10d401430>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d4028f0>]) at 0x10d127510>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916970>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c9169a0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9169d0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916a00>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d40c0f0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-overlaps-SmallMolecule', 'predicate': 'overlaps'}) at 0x10d40ccf0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d40e1b0>]) at 0x10d127590>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916ac0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c916af0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916b20>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916b50>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d40f970>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-affects_response_to-SmallMolecule', 'predicate': 'affects_response_to'}) at 0x10d410570>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d411a30>]) at 0x10d127610>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916c10>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c916c40>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916c70>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916ca0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d4131f0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-composed_primarily_of-SmallMolecule', 'predicate': 'composed_primarily_of'}) at 0x10d413db0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d4152b0>]) at 0x10d127690>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916d60>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c916d90>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916df0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916e20>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d4173f0>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:lipd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:carb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:antb AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:inch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:strd AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:nsba AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:eico AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:PREDISPOSES AND subject.semantic_type_abbreviation:opco AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"lipd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"carb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"antb\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"inch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"strd\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"nsba\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"eico\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"PREDISPOSES\",\"opco\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'has_predisposing_factor': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'Disease-has_predisposing_factor-SmallMolecule', 'predicate': 'has_predisposing_factor'}) at 0x10d44efb0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d4544b0>]) at 0x10d127a70>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c916ee0>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c916f10>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c916f40>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c916f70>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d455c70>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-has_output-SmallMolecule', 'predicate': 'has_output'}) at 0x10d45c5b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d45da70>]) at 0x10d127b10>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c917030>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c917060>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c917090>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9170c0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d45f230>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-response_decreased_by-SmallMolecule', 'predicate': 'response_decreased_by'}) at 0x10d45fdf0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d4612f0>]) at 0x10d127b90>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c917180>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c9171b0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9171e0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c917210>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d462ab0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'Pathway-disrupts-SmallMolecule', 'predicate': 'disrupts'}) at 0x10d468a70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d469f30>]) at 0x10d127c30>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9172d0>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c914c60>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c914ab0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c917300>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d46c070>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-adverse_event_caused_by-SmallMolecule', 'predicate': 'adverse_event_caused_by'}) at 0x10d46cc30>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d46e0f0>]) at 0x10d127cd0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9173c0>, <mgclient.Node(id=127, labels={'Entity'}, properties={'name': 'Disease'}) at 0x10c9173f0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c917420>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c917450>], relationships=[<mgclient.Relationship(start_id=226, end_id=127, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Disease', 'predicate': 'associated_with'}) at 0x10d474270>, <mgclient.Relationship(start_id=127, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1/team/Service%20Provider', 'support_batch': True}}, 'name': 'Service Provider TRAPI', 'smartapi': {'id': '36f82f05705c317bac17ddae3a0ea2f0', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/service-provider-trapi.yaml', 'ui': 'https://smart-api.info/ui/36f82f05705c317bac17ddae3a0ea2f0'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:COMPLICATES AND subject.semantic_type_abbreviation:orch AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:COMPLICATES AND subject.semantic_type_abbreviation:phsu AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:COMPLICATES AND subject.semantic_type_abbreviation:bacs AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:COMPLICATES AND subject.semantic_type_abbreviation:hops AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'object.umls,predication.pmid,pmid_count,predication_count,subject.umls,subject.name,predicate,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": {{ queryInputs | replPrefix(\\'predicate:COMPLICATES AND subject.semantic_type_abbreviation:horm AND pmid_count:>5 AND object.umls\\')| dump }}, \"scopes\": []}'}, 'server': 'https://biothings.ncats.io/semmeddb2', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pmid_count': 'pmid_count', 'predication_count': 'predication_count', 'pubmed': 'predication.pmid'}}}, 'name': 'BioThings SEMMEDDB v2 API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '5634bdb9a1315f85f595979eaf90a504', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/semmeddb2_specific/semmeddb2_smartapi.yaml', 'ui': 'https://smart-api.info/ui/5634bdb9a1315f85f595979eaf90a504'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"COMPLICATES\",\"orch\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"COMPLICATES\",\"phsu\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"COMPLICATES\",\"bacs\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"COMPLICATES\",\"hops\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}, {'bte': {'query_operation': {'method': 'post', 'params': {'fields': 'subject.umls,pmid,subject.name,predicate,object.umls,object.name', 'size': 1000}, 'path': '/query', 'request_body': {'body': '{\"q\": [ {{ queryInputs | wrap( \\'[\"\\', \\'\",\"COMPLICATES\",\"horm\",1,1]\\' )}}], \"scopes\": [\"object.umls\", \"predicate\", \"subject.semantic_type_abbreviation\", \"subject.novelty\", \"object.novelty\"]}'}, 'server': 'https://biothings.ncats.io/semmeddb', 'support_batch': True}, 'response_mapping': {'is_exacerbated_by': {'UMLS': 'subject.umls', 'biolink:original_object': 'object.umls', 'biolink:original_predicate': 'predicate', 'biolink:original_subject': 'subject.umls', 'original_object_name': 'object.name', 'original_subject_name': 'subject.name', 'pubmed': 'pmid'}}}, 'name': 'BioThings SEMMEDDB API', 'provided_by': 'infores:semmeddb', 'smartapi': {'id': '1d288b3a3caf75d541ffaae3aab386c8', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/semmeddb/smartapi.yaml', 'ui': 'https://smart-api.info/ui/1d288b3a3caf75d541ffaae3aab386c8'}, 'tags': ['disease', 'phenotype', 'gene', 'chemical', 'association', 'query', 'translator', 'biothings', 'ngd'], 'x-translator': {'component': 'KP', 'team': ['Service Provider']}}], 'id': 'Disease-is_exacerbated_by-SmallMolecule', 'predicate': 'is_exacerbated_by'}) at 0x10d486730>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d487bf0>]) at 0x10d127eb0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c917510>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c917540>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c917570>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9175a0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d4893f0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-affected_by-SmallMolecule', 'predicate': 'affected_by'}) at 0x10d489ff0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d48b4b0>]) at 0x10d127f30>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c917660>, <mgclient.Node(id=152, labels={'Entity'}, properties={'name': 'Pathway'}) at 0x10c917690>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c9176c0>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c9176f0>], relationships=[<mgclient.Relationship(start_id=226, end_id=152, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-Pathway', 'predicate': 'associated_with'}) at 0x10d490cb0>, <mgclient.Relationship(start_id=152, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'Pathway-has_substrate-SmallMolecule', 'predicate': 'has_substrate'}) at 0x10d4918b0>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d492d70>]) at 0x10d127fb0>,)\n",
      "(<mgclient.Path(nodes=[<mgclient.Node(id=226, labels={'Entity'}, properties={'name': 'InformationResource'}) at 0x10c9177b0>, <mgclient.Node(id=128, labels={'Entity'}, properties={'name': 'PhenotypicFeature'}) at 0x10c9177e0>, <mgclient.Node(id=114, labels={'Entity'}, properties={'name': 'SmallMolecule'}) at 0x10c917810>, <mgclient.Node(id=121, labels={'Entity'}, properties={'name': 'Publication'}) at 0x10c917840>], relationships=[<mgclient.Relationship(start_id=226, end_id=128, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://kg2.transltr.io/api/rtxkg2/v1.3', 'support_batch': True}}, 'name': 'RTX KG2 - TRAPI 1.3.0', 'smartapi': {'id': 'ccd4a8bb83de81401e9a27f1d8e7f948', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/ccd4a8bb83de81401e9a27f1d8e7f948'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.ncats.io/api/rtxkg2/v1.4', 'support_batch': True}}, 'name': 'RTX KG2', 'smartapi': {'id': 'd6ea093bcb5476331d309623cbece11a', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/NewFmt/code/UI/OpenAPI/python-flask-server/KG2/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/d6ea093bcb5476331d309623cbece11a'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'asyncquery_status', 'translator', 'trapi', 'entity', 'status', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Expander Agent']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://arax.transltr.io/api/arax/v1.3', 'support_batch': True}}, 'name': 'ARAX Translator Reasoner - TRAPI 1.3.0', 'smartapi': {'id': 'e248aefca0f469229e82cca80fbabc89', 'metadata': 'https://raw.githubusercontent.com/RTXteam/RTX/production/code/UI/OpenAPI/python-flask-server/openapi_server/openapi/openapi.yaml', 'ui': 'https://smart-api.info/ui/e248aefca0f469229e82cca80fbabc89'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'response', 'translate', 'entity', 'exampleQuestions', 'PubmedMeshNgd', 'status', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Expander Agent']}}], 'id': 'InformationResource-associated_with-PhenotypicFeature', 'predicate': 'associated_with'}) at 0x10d494eb0>, <mgclient.Relationship(start_id=128, end_id=114, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://explanatory-agent.transltr.io/v1.3/', 'support_batch': True}}, 'name': 'Explanatory Agent API', 'smartapi': {'id': '95b8a4ac17ea779165a709185ac07f23', 'metadata': 'https://raw.githubusercontent.com/NCATSTranslator/Explanatory-Agent/master/Agent/smart-api-1.2.0.yaml', 'ui': 'https://smart-api.info/ui/95b8a4ac17ea779165a709185ac07f23'}, 'tags': ['meta_knowledge_graph', 'query', 'asyncquery', 'translator', '1.3.0', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Explanatory Agent']}}], 'id': 'PhenotypicFeature-increased_likelihood_associated_with-SmallMolecule', 'predicate': 'increased_likelihood_associated_with'}) at 0x10d495a70>, <mgclient.Relationship(start_id=114, end_id=121, type='RELATED_TO', properties={'api': [{'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://automat.transltr.io/icees-kg/1.3', 'support_batch': True}}, 'name': 'Automat-icees-kg(Trapi v1.3.0)', 'smartapi': {'id': '44e7a1147ca8657f50af6bb25982762d', 'metadata': 'https://trapi-openapi.apps.renci.org/trapi/infores:automat-icees-kg/1.3.0', 'ui': 'https://smart-api.info/ui/44e7a1147ca8657f50af6bb25982762d'}, 'tags': ['translator', 'automat', 'trapi', 'bte-trapi'], 'x-translator': {'component': 'KP', 'team': ['Exposures Provider']}}, {'bte': {'query_operation': {'input_separator': ',', 'method': 'post', 'params': None, 'path': '/query', 'path_params': None, 'request_body': None, 'server': 'https://bte.transltr.io/v1', 'support_batch': True}}, 'name': 'BioThings Explorer (BTE) TRAPI', 'smartapi': {'id': 'dc91716f44207d2e1287c727f281d339', 'metadata': 'https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/master/biothings_explorer/smartapi.yaml', 'ui': 'https://smart-api.info/ui/dc91716f44207d2e1287c727f281d339'}, 'tags': ['1.3.0', 'meta_knowledge_graph', 'query', 'asyncquery', 'translator', 'trapi', 'biothings', 'bte-trapi'], 'x-translator': {'component': 'ARA', 'team': ['Exploring Agent']}}], 'id': 'SmallMolecule-correlated_with-Publication', 'predicate': 'correlated_with'}) at 0x10d496f30>]) at 0x10d49c090>,)\n"
     ]
    }
   ],
   "source": [
    "path_query=\"\"\"\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..3]->(end:Entity {name: 'Publication'})\n",
    "RETURN p\n",
    "LIMIT 100;\n",
    "\"\"\"\n",
    "\n",
    "cursor.execute(path_query)\n",
    "\n",
    "# Fetch and display the results\n",
    "results = cursor.fetchall()\n",
    "\n",
    "# Printing each result\n",
    "for result in results:\n",
    "    print(result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "# returns clean rows of nodes and relationships (paths) -- very slow at >= 3 hops ...maybe clean rows post process?\n",
    "q=\"\"\"\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..3]->(end:Entity {name: 'Publication'})\n",
    "UNWIND relationships(p) AS r\n",
    "WITH p, collect(r.predicate) AS predicates\n",
    "RETURN reduce(names = [], n IN nodes(p) | names + n.name) AS nodeNames, predicates\n",
    "LIMIT 10;\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "InformationResource - subclass_of -> Publication - correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - subclass_of -> InformationContentEntity - correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - related_to -> InformationContentEntity - correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - ameliorates -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> DiseaseOrPhenotypicFeature - related_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> DiseaseOrPhenotypicFeature - is_side_effect_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - treated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - is_side_effect_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - treated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - location of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - treated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - is_assessed_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - related_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> DiseaseOrPhenotypicFeature - ameliorates -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - is_side_effect_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> DiseaseOrPhenotypicFeature - treated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - occurs_together_in_literature_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> NamedThing - occurs_together_in_literature_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - subclass_of -> NamedThing - occurs_together_in_literature_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - occurs_together_in_literature_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - occurs_together_in_literature_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - caused_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - has_biomarker -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - affected_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - causes -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - has_contraindication -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - contribution_from -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - has_contraindication -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - entity_positively_regulated_by_entity -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - has_real_world_evidence_of_association_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - affected_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - has_biomarker -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - related_to -> Behavior - related_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - contribution_from -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - is_ameliorated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - is_ameliorated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - decreases_abundance_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - increased_likelihood_associated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - has_predisposing_factor -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - disrupted_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - associated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - associated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - has_part -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - decreased_likelihood_associated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - produces -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - interacts_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - treats -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - is_ameliorated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - regulates -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - causes -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_participant -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - related_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - related_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - caused_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - positively_correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - negatively_correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - positively_correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - negatively_correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - correlated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - response_increased_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - part_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_capability -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - participates_in -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - contributes_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - caused_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - enabled_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - increases_abundance_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - decreased_likelihood_associated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_catalyst -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - decreases_response_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - associated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - increases_abundance_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - has_participant -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_part -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - is_exacerbated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - ameliorates -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - disrupted_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - actively_involves -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - adverse_event_caused_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_upstream_actor -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - exacerbates -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_active_ingredient -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - increases_response_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - primarily_composed_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - regulated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - response_affected_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - affects -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - similar_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - overlaps -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - affects_response_to -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - composed_primarily_of -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - has_predisposing_factor -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_output -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - response_decreased_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - disrupts -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - adverse_event_caused_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Disease - is_exacerbated_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - affected_by -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> Pathway - has_substrate -> SmallMolecule - correlated_with -> Publication\n",
      "\n",
      "InformationResource - associated_with -> PhenotypicFeature - increased_likelihood_associated_with -> SmallMolecule - correlated_with -> Publication\n",
      "\n"
     ]
    }
   ],
   "source": [
    "q=\"\"\"\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..3]->(end:Entity {name: 'Publication'})\n",
    "RETURN p\n",
    "LIMIT 100;\n",
    "\"\"\"\n",
    "\n",
    "cursor.execute(q)\n",
    "\n",
    "# Fetch and display the results\n",
    "results = cursor.fetchall()\n",
    "\n",
    "# Printing each result\n",
    "for result in results:\n",
    "    path = result[0]  # Assuming each result is a path\n",
    "    nodes = path.nodes\n",
    "    relationships = path.relationships\n",
    "\n",
    "    # Iterating through the nodes and relationships\n",
    "    for i in range(len(nodes)):\n",
    "        # Print the node's name\n",
    "        print(nodes[i].properties['name'], end='')\n",
    "\n",
    "        if i < len(relationships):\n",
    "            # Print the relationship's predicate\n",
    "            print(' - ' + relationships[i].properties['predicate'] + ' -> ', end='')\n",
    "\n",
    "    print()  # New line after each path\n",
    "    print()  # New line after each path\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[(346364,)]\n"
     ]
    }
   ],
   "source": [
    "q=\"\"\"\n",
    "MATCH p=(start:Entity {name: 'InformationResource'})-[:RELATED_TO*..3]->(end:Entity {name: 'Publication'})\n",
    "RETURN count(p) AS total;\n",
    "\"\"\"\n",
    "\n",
    "cursor.execute(q)\n",
    "results = cursor.fetchall()\n",
    "\n",
    "print(results)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "---"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## NetworkX"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "from utils.metakg.path_finder import MetaKGPathFinder\n",
    "import networkx.classes.digraph as nx_digraph\n",
    "import networkx as nx\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "subject='InformationResource'\n",
    "object='Publication'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/elasticsearch/connection/base.py:200: ElasticsearchWarning: Elasticsearch built-in security features are not enabled. Without authentication, your cluster could be accessible to anyone. See https://www.elastic.co/guide/en/elasticsearch/reference/7.17/security-minimal-setup.html to enable security.\n",
      "  warnings.warn(message, category=ElasticsearchWarning)\n"
     ]
    }
   ],
   "source": [
    "query_data = {\n",
    "    'q': '*'\n",
    "}\n",
    "pf = MetaKGPathFinder(query_data)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "G = pf.G\n",
    "\n",
    "cutoff=3\n",
    "\n",
    "paths = pf.get_paths(subject=subject, object=object, cutoff=cutoff)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "448"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(paths)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PATH: InformationResource->NamedThing->AnatomicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> AnatomicalEntity\n",
      "NamedThing - {'subclass_of'} -> AnatomicalEntity\n",
      "NamedThing - {'active_in'} -> AnatomicalEntity\n",
      "NamedThing - {'actively_involved_in'} -> AnatomicalEntity\n",
      "NamedThing - {'actively_involves'} -> AnatomicalEntity\n",
      "NamedThing - {'acts_upstream_of'} -> AnatomicalEntity\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> AnatomicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> AnatomicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> AnatomicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> AnatomicalEntity\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> AnatomicalEntity\n",
      "NamedThing - {'affects_activity_of'} -> AnatomicalEntity\n",
      "NamedThing - {'affects'} -> AnatomicalEntity\n",
      "NamedThing - {'affects_transport_of'} -> AnatomicalEntity\n",
      "NamedThing - {'capable_of'} -> AnatomicalEntity\n",
      "NamedThing - {'caused_by'} -> AnatomicalEntity\n",
      "NamedThing - {'causes'} -> AnatomicalEntity\n",
      "NamedThing - {'coexists_with'} -> AnatomicalEntity\n",
      "NamedThing - {'contributes_to'} -> AnatomicalEntity\n",
      "NamedThing - {'derives_from'} -> AnatomicalEntity\n",
      "NamedThing - {'derives_into'} -> AnatomicalEntity\n",
      "NamedThing - {'enabled_by'} -> AnatomicalEntity\n",
      "NamedThing - {'enables'} -> AnatomicalEntity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> AnatomicalEntity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> AnatomicalEntity\n",
      "NamedThing - {'entity_regulates_entity'} -> AnatomicalEntity\n",
      "NamedThing - {'has_input'} -> AnatomicalEntity\n",
      "NamedThing - {'has_output'} -> AnatomicalEntity\n",
      "NamedThing - {'has_part'} -> AnatomicalEntity\n",
      "NamedThing - {'has_participant'} -> AnatomicalEntity\n",
      "NamedThing - {'interacts_with'} -> AnatomicalEntity\n",
      "NamedThing - {'located_in'} -> AnatomicalEntity\n",
      "NamedThing - {'location_of'} -> AnatomicalEntity\n",
      "NamedThing - {'molecularly_interacts_with'} -> AnatomicalEntity\n",
      "NamedThing - {'negatively_regulates'} -> AnatomicalEntity\n",
      "NamedThing - {'occurs_in'} -> AnatomicalEntity\n",
      "NamedThing - {'overlaps'} -> AnatomicalEntity\n",
      "NamedThing - {'participates_in'} -> AnatomicalEntity\n",
      "NamedThing - {'part_of'} -> AnatomicalEntity\n",
      "NamedThing - {'physically_interacts_with'} -> AnatomicalEntity\n",
      "NamedThing - {'positively_regulates'} -> AnatomicalEntity\n",
      "NamedThing - {'preceded_by'} -> AnatomicalEntity\n",
      "NamedThing - {'precedes'} -> AnatomicalEntity\n",
      "NamedThing - {'process_negatively_regulates_process'} -> AnatomicalEntity\n",
      "NamedThing - {'process_positively_regulates_process'} -> AnatomicalEntity\n",
      "NamedThing - {'process_regulates_process'} -> AnatomicalEntity\n",
      "NamedThing - {'produced_by'} -> AnatomicalEntity\n",
      "NamedThing - {'regulates'} -> AnatomicalEntity\n",
      "NamedThing - {'related_to'} -> AnatomicalEntity\n",
      "NamedThing - {'temporally_related_to'} -> AnatomicalEntity\n",
      "NamedThing - {'composed_primarily_of'} -> AnatomicalEntity\n",
      "NamedThing - {'has_upstream_actor'} -> AnatomicalEntity\n",
      "NamedThing - {'exacerbates'} -> AnatomicalEntity\n",
      "NamedThing - {'increases_response_to'} -> AnatomicalEntity\n",
      "NamedThing - {'disrupts'} -> AnatomicalEntity\n",
      "NamedThing - {'affected_by'} -> AnatomicalEntity\n",
      "NamedThing - {'treats'} -> AnatomicalEntity\n",
      "NamedThing - {'is_substrate_of'} -> AnatomicalEntity\n",
      "NamedThing - {'has_capability'} -> AnatomicalEntity\n",
      "NamedThing - {'is_input_of'} -> AnatomicalEntity\n",
      "NamedThing - {'decreases_response_to'} -> AnatomicalEntity\n",
      "NamedThing - {'expresses'} -> AnatomicalEntity\n",
      "NamedThing - {'has_substrate'} -> AnatomicalEntity\n",
      "NamedThing - {'ameliorates'} -> AnatomicalEntity\n",
      "NamedThing - {'has_metabolite'} -> AnatomicalEntity\n",
      "NamedThing - {'genetically_interacts_with'} -> AnatomicalEntity\n",
      "NamedThing - {'has_active_ingredient'} -> AnatomicalEntity\n",
      "NamedThing - {'has_catalyst'} -> AnatomicalEntity\n",
      "NamedThing - {'primarily_composed_of'} -> AnatomicalEntity\n",
      "NamedThing - {'expressed_in'} -> AnatomicalEntity\n",
      "NamedThing - {'associated_with'} -> AnatomicalEntity\n",
      "NamedThing - {'similar_to'} -> AnatomicalEntity\n",
      "NamedThing - {'catalyzes'} -> AnatomicalEntity\n",
      "NamedThing - {'is_output_of'} -> AnatomicalEntity\n",
      "NamedThing - {'affects_response_to'} -> AnatomicalEntity\n",
      "NamedThing - {'gene_product_of'} -> AnatomicalEntity\n",
      "NamedThing - {'is_sequence_variant_of'} -> AnatomicalEntity\n",
      "NamedThing - {'gene_associated_with_condition'} -> AnatomicalEntity\n",
      "NamedThing - {'has_phenotype'} -> AnatomicalEntity\n",
      "NamedThing - {'binds'} -> AnatomicalEntity\n",
      "NamedThing - {'biomarker_for'} -> AnatomicalEntity\n",
      "NamedThing - {'prevents'} -> AnatomicalEntity\n",
      "NamedThing - {'colocalizes_with'} -> AnatomicalEntity\n",
      "NamedThing - {'disease_has_basis_in'} -> AnatomicalEntity\n",
      "NamedThing - {'produces'} -> AnatomicalEntity\n",
      "NamedThing - {'predisposes'} -> AnatomicalEntity\n",
      "NamedThing - {'diagnoses'} -> AnatomicalEntity\n",
      "NamedThing - {'close_match'} -> AnatomicalEntity\n",
      "NamedThing - {'manifestation_of'} -> AnatomicalEntity\n",
      "NamedThing - {'develops_from'} -> AnatomicalEntity\n",
      "NamedThing - {'contraindicated_for'} -> AnatomicalEntity\n",
      "NamedThing - {'disease_has_location'} -> AnatomicalEntity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> AnatomicalEntity\n",
      "AnatomicalEntity - {'develops_from'} -> Publication\n",
      "AnatomicalEntity - {'interacts_with'} -> Publication\n",
      "AnatomicalEntity - {'produces'} -> Publication\n",
      "AnatomicalEntity - {'has_part'} -> Publication\n",
      "AnatomicalEntity - {'diagnoses'} -> Publication\n",
      "AnatomicalEntity - {'related_to'} -> Publication\n",
      "AnatomicalEntity - {'affects'} -> Publication\n",
      "AnatomicalEntity - {'manifestation_of'} -> Publication\n",
      "AnatomicalEntity - {'has_input'} -> Publication\n",
      "AnatomicalEntity - {'occurs_in'} -> Publication\n",
      "AnatomicalEntity - {'coexists_with'} -> Publication\n",
      "AnatomicalEntity - {'close_match'} -> Publication\n",
      "AnatomicalEntity - {'disrupts'} -> Publication\n",
      "AnatomicalEntity - {'located_in'} -> Publication\n",
      "AnatomicalEntity - {'regulates'} -> Publication\n",
      "AnatomicalEntity - {'treats'} -> Publication\n",
      "AnatomicalEntity - {'causes'} -> Publication\n",
      "AnatomicalEntity - {'associated_with'} -> Publication\n",
      "AnatomicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "AnatomicalEntity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->BiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> BiologicalProcess\n",
      "NamedThing - {'subclass_of'} -> BiologicalProcess\n",
      "NamedThing - {'active_in'} -> BiologicalProcess\n",
      "NamedThing - {'actively_involved_in'} -> BiologicalProcess\n",
      "NamedThing - {'actively_involves'} -> BiologicalProcess\n",
      "NamedThing - {'acts_upstream_of'} -> BiologicalProcess\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> BiologicalProcess\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> BiologicalProcess\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> BiologicalProcess\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> BiologicalProcess\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> BiologicalProcess\n",
      "NamedThing - {'affects'} -> BiologicalProcess\n",
      "NamedThing - {'capable_of'} -> BiologicalProcess\n",
      "NamedThing - {'caused_by'} -> BiologicalProcess\n",
      "NamedThing - {'causes'} -> BiologicalProcess\n",
      "NamedThing - {'contributes_to'} -> BiologicalProcess\n",
      "NamedThing - {'derives_from'} -> BiologicalProcess\n",
      "NamedThing - {'derives_into'} -> BiologicalProcess\n",
      "NamedThing - {'enabled_by'} -> BiologicalProcess\n",
      "NamedThing - {'enables'} -> BiologicalProcess\n",
      "NamedThing - {'has_input'} -> BiologicalProcess\n",
      "NamedThing - {'has_output'} -> BiologicalProcess\n",
      "NamedThing - {'has_part'} -> BiologicalProcess\n",
      "NamedThing - {'has_participant'} -> BiologicalProcess\n",
      "NamedThing - {'negatively_regulates'} -> BiologicalProcess\n",
      "NamedThing - {'occurs_in'} -> BiologicalProcess\n",
      "NamedThing - {'overlaps'} -> BiologicalProcess\n",
      "NamedThing - {'participates_in'} -> BiologicalProcess\n",
      "NamedThing - {'part_of'} -> BiologicalProcess\n",
      "NamedThing - {'positively_regulates'} -> BiologicalProcess\n",
      "NamedThing - {'preceded_by'} -> BiologicalProcess\n",
      "NamedThing - {'precedes'} -> BiologicalProcess\n",
      "NamedThing - {'process_negatively_regulates_process'} -> BiologicalProcess\n",
      "NamedThing - {'process_positively_regulates_process'} -> BiologicalProcess\n",
      "NamedThing - {'process_regulates_process'} -> BiologicalProcess\n",
      "NamedThing - {'regulates'} -> BiologicalProcess\n",
      "NamedThing - {'related_to'} -> BiologicalProcess\n",
      "NamedThing - {'temporally_related_to'} -> BiologicalProcess\n",
      "NamedThing - {'is_input_of'} -> BiologicalProcess\n",
      "NamedThing - {'decreases_response_to'} -> BiologicalProcess\n",
      "NamedThing - {'has_substrate'} -> BiologicalProcess\n",
      "NamedThing - {'interacts_with'} -> BiologicalProcess\n",
      "NamedThing - {'ameliorates'} -> BiologicalProcess\n",
      "NamedThing - {'is_substrate_of'} -> BiologicalProcess\n",
      "NamedThing - {'has_capability'} -> BiologicalProcess\n",
      "NamedThing - {'has_catalyst'} -> BiologicalProcess\n",
      "NamedThing - {'associated_with'} -> BiologicalProcess\n",
      "NamedThing - {'catalyzes'} -> BiologicalProcess\n",
      "NamedThing - {'affects_response_to'} -> BiologicalProcess\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> BiologicalProcess\n",
      "NamedThing - {'has_metabolite'} -> BiologicalProcess\n",
      "NamedThing - {'composed_primarily_of'} -> BiologicalProcess\n",
      "NamedThing - {'has_upstream_actor'} -> BiologicalProcess\n",
      "NamedThing - {'affects_transport_of'} -> BiologicalProcess\n",
      "NamedThing - {'exacerbates'} -> BiologicalProcess\n",
      "NamedThing - {'has_active_ingredient'} -> BiologicalProcess\n",
      "NamedThing - {'increases_response_to'} -> BiologicalProcess\n",
      "NamedThing - {'primarily_composed_of'} -> BiologicalProcess\n",
      "NamedThing - {'disrupts'} -> BiologicalProcess\n",
      "NamedThing - {'similar_to'} -> BiologicalProcess\n",
      "NamedThing - {'is_output_of'} -> BiologicalProcess\n",
      "NamedThing - {'affected_by'} -> BiologicalProcess\n",
      "NamedThing - {'treats'} -> BiologicalProcess\n",
      "NamedThing - {'close_match'} -> BiologicalProcess\n",
      "NamedThing - {'manifestation_of'} -> BiologicalProcess\n",
      "NamedThing - {'coexists_with'} -> BiologicalProcess\n",
      "NamedThing - {'contraindicated_for'} -> BiologicalProcess\n",
      "NamedThing - {'physically_interacts_with'} -> BiologicalProcess\n",
      "NamedThing - {'gene_product_of'} -> BiologicalProcess\n",
      "NamedThing - {'directly_physically_interacts_with'} -> BiologicalProcess\n",
      "NamedThing - {'has_phenotype'} -> BiologicalProcess\n",
      "NamedThing - {'binds'} -> BiologicalProcess\n",
      "NamedThing - {'is_sequence_variant_of'} -> BiologicalProcess\n",
      "NamedThing - {'gene_associated_with_condition'} -> BiologicalProcess\n",
      "NamedThing - {'located_in'} -> BiologicalProcess\n",
      "NamedThing - {'colocalizes_with'} -> BiologicalProcess\n",
      "NamedThing - {'produces'} -> BiologicalProcess\n",
      "NamedThing - {'prevents'} -> BiologicalProcess\n",
      "NamedThing - {'diagnoses'} -> BiologicalProcess\n",
      "NamedThing - {'predisposes'} -> BiologicalProcess\n",
      "BiologicalProcess - {'correlated_with'} -> Publication\n",
      "BiologicalProcess - {'disrupts'} -> Publication\n",
      "BiologicalProcess - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalProcess - {'regulates'} -> Publication\n",
      "BiologicalProcess - {'biomarker_for'} -> Publication\n",
      "BiologicalProcess - {'has_part'} -> Publication\n",
      "BiologicalProcess - {'located_in'} -> Publication\n",
      "BiologicalProcess - {'predisposes'} -> Publication\n",
      "BiologicalProcess - {'precedes'} -> Publication\n",
      "BiologicalProcess - {'prevents'} -> Publication\n",
      "BiologicalProcess - {'causes'} -> Publication\n",
      "BiologicalProcess - {'close_match'} -> Publication\n",
      "BiologicalProcess - {'associated_with'} -> Publication\n",
      "BiologicalProcess - {'produces'} -> Publication\n",
      "BiologicalProcess - {'has_participant'} -> Publication\n",
      "BiologicalProcess - {'subclass_of'} -> Publication\n",
      "BiologicalProcess - {'related_to'} -> Publication\n",
      "BiologicalProcess - {'affects'} -> Publication\n",
      "BiologicalProcess - {'treats'} -> Publication\n",
      "BiologicalProcess - {'manifestation_of'} -> Publication\n",
      "BiologicalProcess - {'has_input'} -> Publication\n",
      "BiologicalProcess - {'has_output'} -> Publication\n",
      "BiologicalProcess - {'occurs_in'} -> Publication\n",
      "BiologicalProcess - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Cell->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> Cell\n",
      "NamedThing - {'subclass_of'} -> Cell\n",
      "NamedThing - {'active_in'} -> Cell\n",
      "NamedThing - {'actively_involved_in'} -> Cell\n",
      "NamedThing - {'actively_involves'} -> Cell\n",
      "NamedThing - {'acts_upstream_of'} -> Cell\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> Cell\n",
      "NamedThing - {'affects'} -> Cell\n",
      "NamedThing - {'capable_of'} -> Cell\n",
      "NamedThing - {'causes'} -> Cell\n",
      "NamedThing - {'coexists_with'} -> Cell\n",
      "NamedThing - {'contributes_to'} -> Cell\n",
      "NamedThing - {'derives_from'} -> Cell\n",
      "NamedThing - {'derives_into'} -> Cell\n",
      "NamedThing - {'has_input'} -> Cell\n",
      "NamedThing - {'has_output'} -> Cell\n",
      "NamedThing - {'has_part'} -> Cell\n",
      "NamedThing - {'has_participant'} -> Cell\n",
      "NamedThing - {'occurs_in'} -> Cell\n",
      "NamedThing - {'overlaps'} -> Cell\n",
      "NamedThing - {'participates_in'} -> Cell\n",
      "NamedThing - {'part_of'} -> Cell\n",
      "NamedThing - {'produced_by'} -> Cell\n",
      "NamedThing - {'regulates'} -> Cell\n",
      "NamedThing - {'related_to'} -> Cell\n",
      "NamedThing - {'is_output_of'} -> Cell\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> Cell\n",
      "NamedThing - {'affects_response_to'} -> Cell\n",
      "NamedThing - {'location_of'} -> Cell\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> Cell\n",
      "NamedThing - {'located_in'} -> Cell\n",
      "NamedThing - {'composed_primarily_of'} -> Cell\n",
      "NamedThing - {'preceded_by'} -> Cell\n",
      "NamedThing - {'disrupts'} -> Cell\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> Cell\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> Cell\n",
      "NamedThing - {'affected_by'} -> Cell\n",
      "NamedThing - {'has_substrate'} -> Cell\n",
      "NamedThing - {'interacts_with'} -> Cell\n",
      "NamedThing - {'ameliorates'} -> Cell\n",
      "NamedThing - {'treats'} -> Cell\n",
      "NamedThing - {'is_substrate_of'} -> Cell\n",
      "NamedThing - {'has_capability'} -> Cell\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> Cell\n",
      "NamedThing - {'caused_by'} -> Cell\n",
      "NamedThing - {'enabled_by'} -> Cell\n",
      "NamedThing - {'has_catalyst'} -> Cell\n",
      "NamedThing - {'is_input_of'} -> Cell\n",
      "NamedThing - {'decreases_response_to'} -> Cell\n",
      "NamedThing - {'associated_with'} -> Cell\n",
      "NamedThing - {'enables'} -> Cell\n",
      "NamedThing - {'physically_interacts_with'} -> Cell\n",
      "NamedThing - {'catalyzes'} -> Cell\n",
      "NamedThing - {'expresses'} -> Cell\n",
      "NamedThing - {'precedes'} -> Cell\n",
      "NamedThing - {'has_metabolite'} -> Cell\n",
      "NamedThing - {'genetically_interacts_with'} -> Cell\n",
      "NamedThing - {'has_upstream_actor'} -> Cell\n",
      "NamedThing - {'exacerbates'} -> Cell\n",
      "NamedThing - {'has_active_ingredient'} -> Cell\n",
      "NamedThing - {'increases_response_to'} -> Cell\n",
      "NamedThing - {'primarily_composed_of'} -> Cell\n",
      "NamedThing - {'expressed_in'} -> Cell\n",
      "NamedThing - {'temporally_related_to'} -> Cell\n",
      "NamedThing - {'similar_to'} -> Cell\n",
      "NamedThing - {'disease_has_location'} -> Cell\n",
      "NamedThing - {'correlated_with'} -> Cell\n",
      "NamedThing - {'gene_associated_with_condition'} -> Cell\n",
      "NamedThing - {'has_phenotype'} -> Cell\n",
      "NamedThing - {'biomarker_for'} -> Cell\n",
      "NamedThing - {'prevents'} -> Cell\n",
      "NamedThing - {'produces'} -> Cell\n",
      "NamedThing - {'predisposes'} -> Cell\n",
      "NamedThing - {'diagnoses'} -> Cell\n",
      "NamedThing - {'close_match'} -> Cell\n",
      "NamedThing - {'manifestation_of'} -> Cell\n",
      "Cell - {'subclass_of'} -> Publication\n",
      "Cell - {'has_part'} -> Publication\n",
      "Cell - {'produces'} -> Publication\n",
      "Cell - {'related_to'} -> Publication\n",
      "Cell - {'occurs_in'} -> Publication\n",
      "Cell - {'affects'} -> Publication\n",
      "Cell - {'coexists_with'} -> Publication\n",
      "Cell - {'close_match'} -> Publication\n",
      "Cell - {'located_in'} -> Publication\n",
      "Cell - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->CellularComponent->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> CellularComponent\n",
      "NamedThing - {'subclass_of'} -> CellularComponent\n",
      "NamedThing - {'active_in'} -> CellularComponent\n",
      "NamedThing - {'actively_involved_in'} -> CellularComponent\n",
      "NamedThing - {'actively_involves'} -> CellularComponent\n",
      "NamedThing - {'acts_upstream_of'} -> CellularComponent\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> CellularComponent\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> CellularComponent\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> CellularComponent\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> CellularComponent\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> CellularComponent\n",
      "NamedThing - {'affects_activity_of'} -> CellularComponent\n",
      "NamedThing - {'affects'} -> CellularComponent\n",
      "NamedThing - {'affects_transport_of'} -> CellularComponent\n",
      "NamedThing - {'capable_of'} -> CellularComponent\n",
      "NamedThing - {'caused_by'} -> CellularComponent\n",
      "NamedThing - {'coexists_with'} -> CellularComponent\n",
      "NamedThing - {'enabled_by'} -> CellularComponent\n",
      "NamedThing - {'enables'} -> CellularComponent\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> CellularComponent\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> CellularComponent\n",
      "NamedThing - {'entity_regulates_entity'} -> CellularComponent\n",
      "NamedThing - {'has_input'} -> CellularComponent\n",
      "NamedThing - {'has_output'} -> CellularComponent\n",
      "NamedThing - {'has_part'} -> CellularComponent\n",
      "NamedThing - {'has_participant'} -> CellularComponent\n",
      "NamedThing - {'interacts_with'} -> CellularComponent\n",
      "NamedThing - {'located_in'} -> CellularComponent\n",
      "NamedThing - {'molecularly_interacts_with'} -> CellularComponent\n",
      "NamedThing - {'negatively_regulates'} -> CellularComponent\n",
      "NamedThing - {'occurs_in'} -> CellularComponent\n",
      "NamedThing - {'overlaps'} -> CellularComponent\n",
      "NamedThing - {'participates_in'} -> CellularComponent\n",
      "NamedThing - {'part_of'} -> CellularComponent\n",
      "NamedThing - {'physically_interacts_with'} -> CellularComponent\n",
      "NamedThing - {'positively_regulates'} -> CellularComponent\n",
      "NamedThing - {'preceded_by'} -> CellularComponent\n",
      "NamedThing - {'precedes'} -> CellularComponent\n",
      "NamedThing - {'process_negatively_regulates_process'} -> CellularComponent\n",
      "NamedThing - {'process_positively_regulates_process'} -> CellularComponent\n",
      "NamedThing - {'process_regulates_process'} -> CellularComponent\n",
      "NamedThing - {'regulates'} -> CellularComponent\n",
      "NamedThing - {'related_to'} -> CellularComponent\n",
      "NamedThing - {'temporally_related_to'} -> CellularComponent\n",
      "NamedThing - {'is_output_of'} -> CellularComponent\n",
      "NamedThing - {'affects_response_to'} -> CellularComponent\n",
      "NamedThing - {'composed_primarily_of'} -> CellularComponent\n",
      "NamedThing - {'has_upstream_actor'} -> CellularComponent\n",
      "NamedThing - {'exacerbates'} -> CellularComponent\n",
      "NamedThing - {'increases_response_to'} -> CellularComponent\n",
      "NamedThing - {'disrupts'} -> CellularComponent\n",
      "NamedThing - {'affected_by'} -> CellularComponent\n",
      "NamedThing - {'treats'} -> CellularComponent\n",
      "NamedThing - {'causes'} -> CellularComponent\n",
      "NamedThing - {'is_substrate_of'} -> CellularComponent\n",
      "NamedThing - {'has_capability'} -> CellularComponent\n",
      "NamedThing - {'contributes_to'} -> CellularComponent\n",
      "NamedThing - {'is_input_of'} -> CellularComponent\n",
      "NamedThing - {'decreases_response_to'} -> CellularComponent\n",
      "NamedThing - {'has_substrate'} -> CellularComponent\n",
      "NamedThing - {'ameliorates'} -> CellularComponent\n",
      "NamedThing - {'genetically_interacts_with'} -> CellularComponent\n",
      "NamedThing - {'has_active_ingredient'} -> CellularComponent\n",
      "NamedThing - {'has_catalyst'} -> CellularComponent\n",
      "NamedThing - {'primarily_composed_of'} -> CellularComponent\n",
      "NamedThing - {'expressed_in'} -> CellularComponent\n",
      "NamedThing - {'associated_with'} -> CellularComponent\n",
      "NamedThing - {'similar_to'} -> CellularComponent\n",
      "NamedThing - {'catalyzes'} -> CellularComponent\n",
      "NamedThing - {'manifestation_of'} -> CellularComponent\n",
      "NamedThing - {'close_match'} -> CellularComponent\n",
      "NamedThing - {'derives_from'} -> CellularComponent\n",
      "NamedThing - {'disease_has_location'} -> CellularComponent\n",
      "NamedThing - {'gene_associated_with_condition'} -> CellularComponent\n",
      "NamedThing - {'produces'} -> CellularComponent\n",
      "NamedThing - {'prevents'} -> CellularComponent\n",
      "NamedThing - {'diagnoses'} -> CellularComponent\n",
      "NamedThing - {'colocalizes_with'} -> CellularComponent\n",
      "NamedThing - {'predisposes'} -> CellularComponent\n",
      "CellularComponent - {'subclass_of'} -> Publication\n",
      "CellularComponent - {'produces'} -> Publication\n",
      "CellularComponent - {'has_part'} -> Publication\n",
      "CellularComponent - {'related_to'} -> Publication\n",
      "CellularComponent - {'occurs_in'} -> Publication\n",
      "CellularComponent - {'affects'} -> Publication\n",
      "CellularComponent - {'regulates'} -> Publication\n",
      "CellularComponent - {'disrupts'} -> Publication\n",
      "CellularComponent - {'located_in'} -> Publication\n",
      "CellularComponent - {'associated_with'} -> Publication\n",
      "CellularComponent - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->ChemicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> ChemicalEntity\n",
      "NamedThing - {'is_input_of'} -> ChemicalEntity\n",
      "NamedThing - {'subclass_of'} -> ChemicalEntity\n",
      "NamedThing - {'has_capability'} -> ChemicalEntity\n",
      "NamedThing - {'contributes_to'} -> ChemicalEntity\n",
      "NamedThing - {'decreases_response_to'} -> ChemicalEntity\n",
      "NamedThing - {'associated_with'} -> ChemicalEntity\n",
      "NamedThing - {'expresses'} -> ChemicalEntity\n",
      "NamedThing - {'has_substrate'} -> ChemicalEntity\n",
      "NamedThing - {'ameliorates'} -> ChemicalEntity\n",
      "NamedThing - {'genetically_interacts_with'} -> ChemicalEntity\n",
      "NamedThing - {'actively_involves'} -> ChemicalEntity\n",
      "NamedThing - {'has_active_ingredient'} -> ChemicalEntity\n",
      "NamedThing - {'has_catalyst'} -> ChemicalEntity\n",
      "NamedThing - {'primarily_composed_of'} -> ChemicalEntity\n",
      "NamedThing - {'similar_to'} -> ChemicalEntity\n",
      "NamedThing - {'catalyzes'} -> ChemicalEntity\n",
      "NamedThing - {'is_output_of'} -> ChemicalEntity\n",
      "NamedThing - {'affects_response_to'} -> ChemicalEntity\n",
      "NamedThing - {'composed_primarily_of'} -> ChemicalEntity\n",
      "NamedThing - {'has_upstream_actor'} -> ChemicalEntity\n",
      "NamedThing - {'exacerbates'} -> ChemicalEntity\n",
      "NamedThing - {'increases_response_to'} -> ChemicalEntity\n",
      "NamedThing - {'disrupts'} -> ChemicalEntity\n",
      "NamedThing - {'affected_by'} -> ChemicalEntity\n",
      "NamedThing - {'treats'} -> ChemicalEntity\n",
      "NamedThing - {'causes'} -> ChemicalEntity\n",
      "NamedThing - {'is_substrate_of'} -> ChemicalEntity\n",
      "NamedThing - {'active_in'} -> ChemicalEntity\n",
      "NamedThing - {'actively_involved_in'} -> ChemicalEntity\n",
      "NamedThing - {'acts_upstream_of'} -> ChemicalEntity\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> ChemicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> ChemicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> ChemicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> ChemicalEntity\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> ChemicalEntity\n",
      "NamedThing - {'affects_activity_of'} -> ChemicalEntity\n",
      "NamedThing - {'affects'} -> ChemicalEntity\n",
      "NamedThing - {'affects_transport_of'} -> ChemicalEntity\n",
      "NamedThing - {'capable_of'} -> ChemicalEntity\n",
      "NamedThing - {'caused_by'} -> ChemicalEntity\n",
      "NamedThing - {'enabled_by'} -> ChemicalEntity\n",
      "NamedThing - {'enables'} -> ChemicalEntity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> ChemicalEntity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> ChemicalEntity\n",
      "NamedThing - {'entity_regulates_entity'} -> ChemicalEntity\n",
      "NamedThing - {'has_input'} -> ChemicalEntity\n",
      "NamedThing - {'has_output'} -> ChemicalEntity\n",
      "NamedThing - {'has_part'} -> ChemicalEntity\n",
      "NamedThing - {'has_participant'} -> ChemicalEntity\n",
      "NamedThing - {'interacts_with'} -> ChemicalEntity\n",
      "NamedThing - {'location_of'} -> ChemicalEntity\n",
      "NamedThing - {'molecularly_interacts_with'} -> ChemicalEntity\n",
      "NamedThing - {'negatively_regulates'} -> ChemicalEntity\n",
      "NamedThing - {'occurs_in'} -> ChemicalEntity\n",
      "NamedThing - {'overlaps'} -> ChemicalEntity\n",
      "NamedThing - {'participates_in'} -> ChemicalEntity\n",
      "NamedThing - {'part_of'} -> ChemicalEntity\n",
      "NamedThing - {'physically_interacts_with'} -> ChemicalEntity\n",
      "NamedThing - {'positively_regulates'} -> ChemicalEntity\n",
      "NamedThing - {'preceded_by'} -> ChemicalEntity\n",
      "NamedThing - {'precedes'} -> ChemicalEntity\n",
      "NamedThing - {'process_negatively_regulates_process'} -> ChemicalEntity\n",
      "NamedThing - {'process_positively_regulates_process'} -> ChemicalEntity\n",
      "NamedThing - {'process_regulates_process'} -> ChemicalEntity\n",
      "NamedThing - {'regulates'} -> ChemicalEntity\n",
      "NamedThing - {'related_to'} -> ChemicalEntity\n",
      "NamedThing - {'temporally_related_to'} -> ChemicalEntity\n",
      "NamedThing - {'manifestation_of'} -> ChemicalEntity\n",
      "NamedThing - {'coexists_with'} -> ChemicalEntity\n",
      "NamedThing - {'has_metabolite'} -> ChemicalEntity\n",
      "NamedThing - {'contraindicated_for'} -> ChemicalEntity\n",
      "NamedThing - {'chemically_similar_to'} -> ChemicalEntity\n",
      "NamedThing - {'derives_from'} -> ChemicalEntity\n",
      "NamedThing - {'in_taxon'} -> ChemicalEntity\n",
      "NamedThing - {'disease_has_location'} -> ChemicalEntity\n",
      "NamedThing - {'gene_product_of'} -> ChemicalEntity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> ChemicalEntity\n",
      "NamedThing - {'has_phenotype'} -> ChemicalEntity\n",
      "NamedThing - {'binds'} -> ChemicalEntity\n",
      "NamedThing - {'is_sequence_variant_of'} -> ChemicalEntity\n",
      "NamedThing - {'biomarker_for'} -> ChemicalEntity\n",
      "NamedThing - {'correlated_with'} -> ChemicalEntity\n",
      "NamedThing - {'gene_associated_with_condition'} -> ChemicalEntity\n",
      "NamedThing - {'located_in'} -> ChemicalEntity\n",
      "NamedThing - {'colocalizes_with'} -> ChemicalEntity\n",
      "NamedThing - {'produces'} -> ChemicalEntity\n",
      "NamedThing - {'expressed_in'} -> ChemicalEntity\n",
      "NamedThing - {'prevents'} -> ChemicalEntity\n",
      "NamedThing - {'diagnoses'} -> ChemicalEntity\n",
      "NamedThing - {'predisposes'} -> ChemicalEntity\n",
      "NamedThing - {'same_as'} -> ChemicalEntity\n",
      "NamedThing - {'close_match'} -> ChemicalEntity\n",
      "ChemicalEntity - {'correlated_with'} -> Publication\n",
      "ChemicalEntity - {'disrupts'} -> Publication\n",
      "ChemicalEntity - {'associated_with'} -> Publication\n",
      "ChemicalEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "ChemicalEntity - {'regulates'} -> Publication\n",
      "ChemicalEntity - {'located_in'} -> Publication\n",
      "ChemicalEntity - {'predisposes'} -> Publication\n",
      "ChemicalEntity - {'related_to'} -> Publication\n",
      "ChemicalEntity - {'has_part'} -> Publication\n",
      "ChemicalEntity - {'prevents'} -> Publication\n",
      "ChemicalEntity - {'affects'} -> Publication\n",
      "ChemicalEntity - {'precedes'} -> Publication\n",
      "ChemicalEntity - {'occurs_in'} -> Publication\n",
      "ChemicalEntity - {'has_metabolite'} -> Publication\n",
      "ChemicalEntity - {'close_match'} -> Publication\n",
      "ChemicalEntity - {'subclass_of'} -> Publication\n",
      "ChemicalEntity - {'interacts_with'} -> Publication\n",
      "ChemicalEntity - {'treats'} -> Publication\n",
      "ChemicalEntity - {'physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'has_input'} -> Publication\n",
      "ChemicalEntity - {'causes'} -> Publication\n",
      "ChemicalEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->ChemicalMixture->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> ChemicalMixture\n",
      "NamedThing - {'subclass_of'} -> ChemicalMixture\n",
      "NamedThing - {'has_input'} -> ChemicalMixture\n",
      "NamedThing - {'has_output'} -> ChemicalMixture\n",
      "NamedThing - {'has_participant'} -> ChemicalMixture\n",
      "NamedThing - {'preceded_by'} -> ChemicalMixture\n",
      "NamedThing - {'related_to'} -> ChemicalMixture\n",
      "NamedThing - {'temporally_related_to'} -> ChemicalMixture\n",
      "NamedThing - {'increases_response_to'} -> ChemicalMixture\n",
      "NamedThing - {'has_capability'} -> ChemicalMixture\n",
      "NamedThing - {'participates_in'} -> ChemicalMixture\n",
      "NamedThing - {'enabled_by'} -> ChemicalMixture\n",
      "NamedThing - {'is_input_of'} -> ChemicalMixture\n",
      "NamedThing - {'decreases_response_to'} -> ChemicalMixture\n",
      "NamedThing - {'enables'} -> ChemicalMixture\n",
      "NamedThing - {'has_part'} -> ChemicalMixture\n",
      "NamedThing - {'has_substrate'} -> ChemicalMixture\n",
      "NamedThing - {'precedes'} -> ChemicalMixture\n",
      "NamedThing - {'interacts_with'} -> ChemicalMixture\n",
      "NamedThing - {'ameliorates'} -> ChemicalMixture\n",
      "NamedThing - {'capable_of'} -> ChemicalMixture\n",
      "NamedThing - {'actively_involves'} -> ChemicalMixture\n",
      "NamedThing - {'has_active_ingredient'} -> ChemicalMixture\n",
      "NamedThing - {'has_catalyst'} -> ChemicalMixture\n",
      "NamedThing - {'primarily_composed_of'} -> ChemicalMixture\n",
      "NamedThing - {'actively_involved_in'} -> ChemicalMixture\n",
      "NamedThing - {'associated_with'} -> ChemicalMixture\n",
      "NamedThing - {'affects'} -> ChemicalMixture\n",
      "NamedThing - {'similar_to'} -> ChemicalMixture\n",
      "NamedThing - {'catalyzes'} -> ChemicalMixture\n",
      "NamedThing - {'is_output_of'} -> ChemicalMixture\n",
      "NamedThing - {'overlaps'} -> ChemicalMixture\n",
      "NamedThing - {'affects_response_to'} -> ChemicalMixture\n",
      "NamedThing - {'composed_primarily_of'} -> ChemicalMixture\n",
      "NamedThing - {'has_upstream_actor'} -> ChemicalMixture\n",
      "NamedThing - {'exacerbates'} -> ChemicalMixture\n",
      "NamedThing - {'disrupts'} -> ChemicalMixture\n",
      "NamedThing - {'affected_by'} -> ChemicalMixture\n",
      "NamedThing - {'treats'} -> ChemicalMixture\n",
      "NamedThing - {'regulates'} -> ChemicalMixture\n",
      "NamedThing - {'is_substrate_of'} -> ChemicalMixture\n",
      "NamedThing - {'part_of'} -> ChemicalMixture\n",
      "NamedThing - {'produces'} -> ChemicalMixture\n",
      "NamedThing - {'coexists_with'} -> ChemicalMixture\n",
      "NamedThing - {'close_match'} -> ChemicalMixture\n",
      "NamedThing - {'derives_from'} -> ChemicalMixture\n",
      "ChemicalMixture - {'related_to'} -> Publication\n",
      "ChemicalMixture - {'affects'} -> Publication\n",
      "ChemicalMixture - {'subclass_of'} -> Publication\n",
      "ChemicalMixture - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->ComplexMolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> ComplexMolecularMixture\n",
      "NamedThing - {'affects_response_to'} -> ComplexMolecularMixture\n",
      "NamedThing - {'composed_primarily_of'} -> ComplexMolecularMixture\n",
      "NamedThing - {'affected_by'} -> ComplexMolecularMixture\n",
      "NamedThing - {'interacts_with'} -> ComplexMolecularMixture\n",
      "NamedThing - {'part_of'} -> ComplexMolecularMixture\n",
      "NamedThing - {'participates_in'} -> ComplexMolecularMixture\n",
      "NamedThing - {'has_participant'} -> ComplexMolecularMixture\n",
      "NamedThing - {'decreases_response_to'} -> ComplexMolecularMixture\n",
      "NamedThing - {'associated_with'} -> ComplexMolecularMixture\n",
      "NamedThing - {'has_part'} -> ComplexMolecularMixture\n",
      "NamedThing - {'has_upstream_actor'} -> ComplexMolecularMixture\n",
      "NamedThing - {'has_active_ingredient'} -> ComplexMolecularMixture\n",
      "NamedThing - {'increases_response_to'} -> ComplexMolecularMixture\n",
      "NamedThing - {'primarily_composed_of'} -> ComplexMolecularMixture\n",
      "NamedThing - {'affects'} -> ComplexMolecularMixture\n",
      "NamedThing - {'similar_to'} -> ComplexMolecularMixture\n",
      "NamedThing - {'related_to'} -> ComplexMolecularMixture\n",
      "NamedThing - {'overlaps'} -> ComplexMolecularMixture\n",
      "NamedThing - {'regulates'} -> ComplexMolecularMixture\n",
      "NamedThing - {'subclass_of'} -> ComplexMolecularMixture\n",
      "NamedThing - {'coexists_with'} -> ComplexMolecularMixture\n",
      "NamedThing - {'close_match'} -> ComplexMolecularMixture\n",
      "ComplexMolecularMixture - {'correlated_with'} -> Publication\n",
      "ComplexMolecularMixture - {'subclass_of'} -> Publication\n",
      "ComplexMolecularMixture - {'treats'} -> Publication\n",
      "ComplexMolecularMixture - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Disease->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> Disease\n",
      "NamedThing - {'subclass_of'} -> Disease\n",
      "NamedThing - {'related_to'} -> Disease\n",
      "NamedThing - {'location_of'} -> Disease\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> Disease\n",
      "NamedThing - {'process_positively_regulates_process'} -> Disease\n",
      "NamedThing - {'has_upstream_actor'} -> Disease\n",
      "NamedThing - {'process_negatively_regulates_process'} -> Disease\n",
      "NamedThing - {'exacerbates'} -> Disease\n",
      "NamedThing - {'has_active_ingredient'} -> Disease\n",
      "NamedThing - {'increases_response_to'} -> Disease\n",
      "NamedThing - {'primarily_composed_of'} -> Disease\n",
      "NamedThing - {'temporally_related_to'} -> Disease\n",
      "NamedThing - {'actively_involved_in'} -> Disease\n",
      "NamedThing - {'has_output'} -> Disease\n",
      "NamedThing - {'affects'} -> Disease\n",
      "NamedThing - {'process_regulates_process'} -> Disease\n",
      "NamedThing - {'disrupts'} -> Disease\n",
      "NamedThing - {'similar_to'} -> Disease\n",
      "NamedThing - {'is_output_of'} -> Disease\n",
      "NamedThing - {'affects_response_to'} -> Disease\n",
      "NamedThing - {'treats'} -> Disease\n",
      "NamedThing - {'causes'} -> Disease\n",
      "NamedThing - {'composed_primarily_of'} -> Disease\n",
      "NamedThing - {'enabled_by'} -> Disease\n",
      "NamedThing - {'has_participant'} -> Disease\n",
      "NamedThing - {'is_input_of'} -> Disease\n",
      "NamedThing - {'expresses'} -> Disease\n",
      "NamedThing - {'affected_by'} -> Disease\n",
      "NamedThing - {'has_substrate'} -> Disease\n",
      "NamedThing - {'interacts_with'} -> Disease\n",
      "NamedThing - {'ameliorates'} -> Disease\n",
      "NamedThing - {'regulates'} -> Disease\n",
      "NamedThing - {'is_substrate_of'} -> Disease\n",
      "NamedThing - {'part_of'} -> Disease\n",
      "NamedThing - {'has_capability'} -> Disease\n",
      "NamedThing - {'participates_in'} -> Disease\n",
      "NamedThing - {'capable_of'} -> Disease\n",
      "NamedThing - {'contributes_to'} -> Disease\n",
      "NamedThing - {'caused_by'} -> Disease\n",
      "NamedThing - {'actively_involves'} -> Disease\n",
      "NamedThing - {'has_input'} -> Disease\n",
      "NamedThing - {'has_catalyst'} -> Disease\n",
      "NamedThing - {'decreases_response_to'} -> Disease\n",
      "NamedThing - {'associated_with'} -> Disease\n",
      "NamedThing - {'enables'} -> Disease\n",
      "NamedThing - {'has_part'} -> Disease\n",
      "NamedThing - {'catalyzes'} -> Disease\n",
      "NamedThing - {'precedes'} -> Disease\n",
      "NamedThing - {'overlaps'} -> Disease\n",
      "NamedThing - {'gene_product_of'} -> Disease\n",
      "NamedThing - {'directly_physically_interacts_with'} -> Disease\n",
      "NamedThing - {'has_phenotype'} -> Disease\n",
      "NamedThing - {'binds'} -> Disease\n",
      "NamedThing - {'is_sequence_variant_of'} -> Disease\n",
      "NamedThing - {'biomarker_for'} -> Disease\n",
      "NamedThing - {'correlated_with'} -> Disease\n",
      "NamedThing - {'gene_associated_with_condition'} -> Disease\n",
      "NamedThing - {'located_in'} -> Disease\n",
      "NamedThing - {'colocalizes_with'} -> Disease\n",
      "NamedThing - {'produces'} -> Disease\n",
      "NamedThing - {'expressed_in'} -> Disease\n",
      "NamedThing - {'prevents'} -> Disease\n",
      "NamedThing - {'diagnoses'} -> Disease\n",
      "NamedThing - {'predisposes'} -> Disease\n",
      "NamedThing - {'occurs_in'} -> Disease\n",
      "NamedThing - {'manifestation_of'} -> Disease\n",
      "NamedThing - {'coexists_with'} -> Disease\n",
      "NamedThing - {'close_match'} -> Disease\n",
      "NamedThing - {'contraindicated_for'} -> Disease\n",
      "NamedThing - {'chemically_similar_to'} -> Disease\n",
      "NamedThing - {'physically_interacts_with'} -> Disease\n",
      "NamedThing - {'derives_from'} -> Disease\n",
      "NamedThing - {'disease_has_location'} -> Disease\n",
      "Disease - {'correlated_with'} -> Publication\n",
      "Disease - {'associated_with'} -> Publication\n",
      "Disease - {'gene_associated_with_condition'} -> Publication\n",
      "Disease - {'subclass_of'} -> Publication\n",
      "Disease - {'has_part'} -> Publication\n",
      "Disease - {'interacts_with'} -> Publication\n",
      "Disease - {'produces'} -> Publication\n",
      "Disease - {'predisposes'} -> Publication\n",
      "Disease - {'related_to'} -> Publication\n",
      "Disease - {'prevents'} -> Publication\n",
      "Disease - {'diagnoses'} -> Publication\n",
      "Disease - {'contributes_to'} -> Publication\n",
      "Disease - {'manifestation_of'} -> Publication\n",
      "Disease - {'occurs_in'} -> Publication\n",
      "Disease - {'affects'} -> Publication\n",
      "Disease - {'has_input'} -> Publication\n",
      "Disease - {'exacerbates'} -> Publication\n",
      "Disease - {'coexists_with'} -> Publication\n",
      "Disease - {'close_match'} -> Publication\n",
      "Disease - {'disrupts'} -> Publication\n",
      "Disease - {'regulates'} -> Publication\n",
      "Disease - {'biomarker_for'} -> Publication\n",
      "Disease - {'located_in'} -> Publication\n",
      "Disease - {'treats'} -> Publication\n",
      "Disease - {'causes'} -> Publication\n",
      "Disease - {'disease_has_location'} -> Publication\n",
      "Disease - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->GrossAnatomicalStructure->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'subclass_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'active_in'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'actively_involved_in'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'actively_involves'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'acts_upstream_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'affects'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'capable_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'coexists_with'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_output'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_part'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_participant'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'located_in'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'occurs_in'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'overlaps'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'participates_in'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'part_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'regulates'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'related_to'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'disrupts'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'affected_by'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'produced_by'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'treats'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'causes'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'is_substrate_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_capability'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'contributes_to'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_input'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'preceded_by'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'enabled_by'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'is_input_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'decreases_response_to'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'enables'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'expresses'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_substrate'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'precedes'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'interacts_with'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'ameliorates'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'caused_by'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_active_ingredient'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_catalyst'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'primarily_composed_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'expressed_in'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'temporally_related_to'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'associated_with'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'similar_to'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'catalyzes'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'is_output_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'affects_response_to'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'location_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'composed_primarily_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_upstream_actor'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'exacerbates'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'increases_response_to'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'disease_has_basis_in'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'predisposes'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'close_match'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'manifestation_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'develops_from'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'contraindicated_for'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'physically_interacts_with'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'derives_from'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'disease_has_location'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'directly_physically_interacts_with'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'has_phenotype'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'is_sequence_variant_of'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'biomarker_for'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'gene_associated_with_condition'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'colocalizes_with'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'produces'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'prevents'} -> GrossAnatomicalStructure\n",
      "NamedThing - {'diagnoses'} -> GrossAnatomicalStructure\n",
      "GrossAnatomicalStructure - {'affects'} -> Publication\n",
      "GrossAnatomicalStructure - {'coexists_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'close_match'} -> Publication\n",
      "GrossAnatomicalStructure - {'develops_from'} -> Publication\n",
      "GrossAnatomicalStructure - {'regulates'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_input'} -> Publication\n",
      "GrossAnatomicalStructure - {'associated_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'located_in'} -> Publication\n",
      "GrossAnatomicalStructure - {'subclass_of'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_part'} -> Publication\n",
      "GrossAnatomicalStructure - {'produces'} -> Publication\n",
      "GrossAnatomicalStructure - {'related_to'} -> Publication\n",
      "GrossAnatomicalStructure - {'occurs_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->MolecularActivity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> MolecularActivity\n",
      "NamedThing - {'subclass_of'} -> MolecularActivity\n",
      "NamedThing - {'active_in'} -> MolecularActivity\n",
      "NamedThing - {'actively_involved_in'} -> MolecularActivity\n",
      "NamedThing - {'actively_involves'} -> MolecularActivity\n",
      "NamedThing - {'acts_upstream_of'} -> MolecularActivity\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> MolecularActivity\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> MolecularActivity\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> MolecularActivity\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> MolecularActivity\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> MolecularActivity\n",
      "NamedThing - {'affects_activity_of'} -> MolecularActivity\n",
      "NamedThing - {'affects'} -> MolecularActivity\n",
      "NamedThing - {'affects_transport_of'} -> MolecularActivity\n",
      "NamedThing - {'capable_of'} -> MolecularActivity\n",
      "NamedThing - {'caused_by'} -> MolecularActivity\n",
      "NamedThing - {'contributes_to'} -> MolecularActivity\n",
      "NamedThing - {'enabled_by'} -> MolecularActivity\n",
      "NamedThing - {'enables'} -> MolecularActivity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> MolecularActivity\n",
      "NamedThing - {'entity_regulates_entity'} -> MolecularActivity\n",
      "NamedThing - {'has_input'} -> MolecularActivity\n",
      "NamedThing - {'has_participant'} -> MolecularActivity\n",
      "NamedThing - {'has_part'} -> MolecularActivity\n",
      "NamedThing - {'interacts_with'} -> MolecularActivity\n",
      "NamedThing - {'molecularly_interacts_with'} -> MolecularActivity\n",
      "NamedThing - {'negatively_regulates'} -> MolecularActivity\n",
      "NamedThing - {'occurs_in'} -> MolecularActivity\n",
      "NamedThing - {'overlaps'} -> MolecularActivity\n",
      "NamedThing - {'participates_in'} -> MolecularActivity\n",
      "NamedThing - {'part_of'} -> MolecularActivity\n",
      "NamedThing - {'physically_interacts_with'} -> MolecularActivity\n",
      "NamedThing - {'positively_regulates'} -> MolecularActivity\n",
      "NamedThing - {'preceded_by'} -> MolecularActivity\n",
      "NamedThing - {'precedes'} -> MolecularActivity\n",
      "NamedThing - {'process_negatively_regulates_process'} -> MolecularActivity\n",
      "NamedThing - {'process_positively_regulates_process'} -> MolecularActivity\n",
      "NamedThing - {'process_regulates_process'} -> MolecularActivity\n",
      "NamedThing - {'produces'} -> MolecularActivity\n",
      "NamedThing - {'regulates'} -> MolecularActivity\n",
      "NamedThing - {'related_to'} -> MolecularActivity\n",
      "NamedThing - {'temporally_related_to'} -> MolecularActivity\n",
      "NamedThing - {'is_input_of'} -> MolecularActivity\n",
      "NamedThing - {'decreases_response_to'} -> MolecularActivity\n",
      "NamedThing - {'has_substrate'} -> MolecularActivity\n",
      "NamedThing - {'ameliorates'} -> MolecularActivity\n",
      "NamedThing - {'is_substrate_of'} -> MolecularActivity\n",
      "NamedThing - {'has_capability'} -> MolecularActivity\n",
      "NamedThing - {'has_catalyst'} -> MolecularActivity\n",
      "NamedThing - {'associated_with'} -> MolecularActivity\n",
      "NamedThing - {'similar_to'} -> MolecularActivity\n",
      "NamedThing - {'catalyzes'} -> MolecularActivity\n",
      "NamedThing - {'affects_response_to'} -> MolecularActivity\n",
      "NamedThing - {'composed_primarily_of'} -> MolecularActivity\n",
      "NamedThing - {'genetically_interacts_with'} -> MolecularActivity\n",
      "NamedThing - {'has_upstream_actor'} -> MolecularActivity\n",
      "NamedThing - {'exacerbates'} -> MolecularActivity\n",
      "NamedThing - {'has_active_ingredient'} -> MolecularActivity\n",
      "NamedThing - {'increases_response_to'} -> MolecularActivity\n",
      "NamedThing - {'primarily_composed_of'} -> MolecularActivity\n",
      "NamedThing - {'has_output'} -> MolecularActivity\n",
      "NamedThing - {'disrupts'} -> MolecularActivity\n",
      "NamedThing - {'is_output_of'} -> MolecularActivity\n",
      "NamedThing - {'affected_by'} -> MolecularActivity\n",
      "NamedThing - {'treats'} -> MolecularActivity\n",
      "NamedThing - {'causes'} -> MolecularActivity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> MolecularActivity\n",
      "NamedThing - {'is_sequence_variant_of'} -> MolecularActivity\n",
      "NamedThing - {'gene_associated_with_condition'} -> MolecularActivity\n",
      "NamedThing - {'has_phenotype'} -> MolecularActivity\n",
      "NamedThing - {'located_in'} -> MolecularActivity\n",
      "NamedThing - {'colocalizes_with'} -> MolecularActivity\n",
      "NamedThing - {'manifestation_of'} -> MolecularActivity\n",
      "NamedThing - {'coexists_with'} -> MolecularActivity\n",
      "NamedThing - {'close_match'} -> MolecularActivity\n",
      "NamedThing - {'derives_from'} -> MolecularActivity\n",
      "MolecularActivity - {'occurs_in'} -> Publication\n",
      "MolecularActivity - {'treats'} -> Publication\n",
      "MolecularActivity - {'has_input'} -> Publication\n",
      "MolecularActivity - {'has_output'} -> Publication\n",
      "MolecularActivity - {'coexists_with'} -> Publication\n",
      "MolecularActivity - {'regulates'} -> Publication\n",
      "MolecularActivity - {'disrupts'} -> Publication\n",
      "MolecularActivity - {'associated_with'} -> Publication\n",
      "MolecularActivity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularActivity - {'located_in'} -> Publication\n",
      "MolecularActivity - {'produces'} -> Publication\n",
      "MolecularActivity - {'related_to'} -> Publication\n",
      "MolecularActivity - {'affects'} -> Publication\n",
      "MolecularActivity - {'has_participant'} -> Publication\n",
      "MolecularActivity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->MolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> MolecularMixture\n",
      "NamedThing - {'subclass_of'} -> MolecularMixture\n",
      "NamedThing - {'has_input'} -> MolecularMixture\n",
      "NamedThing - {'has_output'} -> MolecularMixture\n",
      "NamedThing - {'has_participant'} -> MolecularMixture\n",
      "NamedThing - {'preceded_by'} -> MolecularMixture\n",
      "NamedThing - {'related_to'} -> MolecularMixture\n",
      "NamedThing - {'temporally_related_to'} -> MolecularMixture\n",
      "NamedThing - {'composed_primarily_of'} -> MolecularMixture\n",
      "NamedThing - {'has_upstream_actor'} -> MolecularMixture\n",
      "NamedThing - {'affects_transport_of'} -> MolecularMixture\n",
      "NamedThing - {'exacerbates'} -> MolecularMixture\n",
      "NamedThing - {'increases_response_to'} -> MolecularMixture\n",
      "NamedThing - {'disrupts'} -> MolecularMixture\n",
      "NamedThing - {'affected_by'} -> MolecularMixture\n",
      "NamedThing - {'interacts_with'} -> MolecularMixture\n",
      "NamedThing - {'treats'} -> MolecularMixture\n",
      "NamedThing - {'regulates'} -> MolecularMixture\n",
      "NamedThing - {'is_substrate_of'} -> MolecularMixture\n",
      "NamedThing - {'part_of'} -> MolecularMixture\n",
      "NamedThing - {'has_capability'} -> MolecularMixture\n",
      "NamedThing - {'participates_in'} -> MolecularMixture\n",
      "NamedThing - {'enabled_by'} -> MolecularMixture\n",
      "NamedThing - {'is_input_of'} -> MolecularMixture\n",
      "NamedThing - {'decreases_response_to'} -> MolecularMixture\n",
      "NamedThing - {'associated_with'} -> MolecularMixture\n",
      "NamedThing - {'enables'} -> MolecularMixture\n",
      "NamedThing - {'has_part'} -> MolecularMixture\n",
      "NamedThing - {'physically_interacts_with'} -> MolecularMixture\n",
      "NamedThing - {'has_substrate'} -> MolecularMixture\n",
      "NamedThing - {'precedes'} -> MolecularMixture\n",
      "NamedThing - {'ameliorates'} -> MolecularMixture\n",
      "NamedThing - {'affects_activity_of'} -> MolecularMixture\n",
      "NamedThing - {'capable_of'} -> MolecularMixture\n",
      "NamedThing - {'caused_by'} -> MolecularMixture\n",
      "NamedThing - {'genetically_interacts_with'} -> MolecularMixture\n",
      "NamedThing - {'actively_involves'} -> MolecularMixture\n",
      "NamedThing - {'has_active_ingredient'} -> MolecularMixture\n",
      "NamedThing - {'has_catalyst'} -> MolecularMixture\n",
      "NamedThing - {'primarily_composed_of'} -> MolecularMixture\n",
      "NamedThing - {'actively_involved_in'} -> MolecularMixture\n",
      "NamedThing - {'affects'} -> MolecularMixture\n",
      "NamedThing - {'similar_to'} -> MolecularMixture\n",
      "NamedThing - {'catalyzes'} -> MolecularMixture\n",
      "NamedThing - {'is_output_of'} -> MolecularMixture\n",
      "NamedThing - {'overlaps'} -> MolecularMixture\n",
      "NamedThing - {'affects_response_to'} -> MolecularMixture\n",
      "NamedThing - {'close_match'} -> MolecularMixture\n",
      "NamedThing - {'produces'} -> MolecularMixture\n",
      "NamedThing - {'located_in'} -> MolecularMixture\n",
      "NamedThing - {'same_as'} -> MolecularMixture\n",
      "NamedThing - {'chemically_similar_to'} -> MolecularMixture\n",
      "NamedThing - {'coexists_with'} -> MolecularMixture\n",
      "MolecularMixture - {'correlated_with'} -> Publication\n",
      "MolecularMixture - {'treats'} -> Publication\n",
      "MolecularMixture - {'subclass_of'} -> Publication\n",
      "MolecularMixture - {'located_in'} -> Publication\n",
      "MolecularMixture - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->OrganismTaxon->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> OrganismTaxon\n",
      "NamedThing - {'affected_by'} -> OrganismTaxon\n",
      "NamedThing - {'treats'} -> OrganismTaxon\n",
      "NamedThing - {'has_input'} -> OrganismTaxon\n",
      "NamedThing - {'enabled_by'} -> OrganismTaxon\n",
      "NamedThing - {'has_participant'} -> OrganismTaxon\n",
      "NamedThing - {'has_part'} -> OrganismTaxon\n",
      "NamedThing - {'has_substrate'} -> OrganismTaxon\n",
      "NamedThing - {'interacts_with'} -> OrganismTaxon\n",
      "NamedThing - {'ameliorates'} -> OrganismTaxon\n",
      "NamedThing - {'regulates'} -> OrganismTaxon\n",
      "NamedThing - {'part_of'} -> OrganismTaxon\n",
      "NamedThing - {'occurs_in'} -> OrganismTaxon\n",
      "NamedThing - {'has_capability'} -> OrganismTaxon\n",
      "NamedThing - {'participates_in'} -> OrganismTaxon\n",
      "NamedThing - {'actively_involves'} -> OrganismTaxon\n",
      "NamedThing - {'has_catalyst'} -> OrganismTaxon\n",
      "NamedThing - {'expressed_in'} -> OrganismTaxon\n",
      "NamedThing - {'decreases_response_to'} -> OrganismTaxon\n",
      "NamedThing - {'associated_with'} -> OrganismTaxon\n",
      "NamedThing - {'related_to'} -> OrganismTaxon\n",
      "NamedThing - {'overlaps'} -> OrganismTaxon\n",
      "NamedThing - {'affects_response_to'} -> OrganismTaxon\n",
      "NamedThing - {'affects_activity_of'} -> OrganismTaxon\n",
      "NamedThing - {'located_in'} -> OrganismTaxon\n",
      "NamedThing - {'composed_primarily_of'} -> OrganismTaxon\n",
      "NamedThing - {'has_upstream_actor'} -> OrganismTaxon\n",
      "NamedThing - {'affects_transport_of'} -> OrganismTaxon\n",
      "NamedThing - {'exacerbates'} -> OrganismTaxon\n",
      "NamedThing - {'has_active_ingredient'} -> OrganismTaxon\n",
      "NamedThing - {'increases_response_to'} -> OrganismTaxon\n",
      "NamedThing - {'primarily_composed_of'} -> OrganismTaxon\n",
      "NamedThing - {'has_output'} -> OrganismTaxon\n",
      "NamedThing - {'affects'} -> OrganismTaxon\n",
      "NamedThing - {'disrupts'} -> OrganismTaxon\n",
      "NamedThing - {'similar_to'} -> OrganismTaxon\n",
      "NamedThing - {'contraindicated_for'} -> OrganismTaxon\n",
      "NamedThing - {'chemically_similar_to'} -> OrganismTaxon\n",
      "NamedThing - {'physically_interacts_with'} -> OrganismTaxon\n",
      "NamedThing - {'derives_from'} -> OrganismTaxon\n",
      "NamedThing - {'causes'} -> OrganismTaxon\n",
      "NamedThing - {'in_taxon'} -> OrganismTaxon\n",
      "NamedThing - {'disease_has_location'} -> OrganismTaxon\n",
      "NamedThing - {'directly_physically_interacts_with'} -> OrganismTaxon\n",
      "NamedThing - {'gene_product_of'} -> OrganismTaxon\n",
      "NamedThing - {'is_sequence_variant_of'} -> OrganismTaxon\n",
      "NamedThing - {'gene_associated_with_condition'} -> OrganismTaxon\n",
      "NamedThing - {'has_phenotype'} -> OrganismTaxon\n",
      "NamedThing - {'subclass_of'} -> OrganismTaxon\n",
      "NamedThing - {'biomarker_for'} -> OrganismTaxon\n",
      "NamedThing - {'produces'} -> OrganismTaxon\n",
      "NamedThing - {'prevents'} -> OrganismTaxon\n",
      "NamedThing - {'diagnoses'} -> OrganismTaxon\n",
      "NamedThing - {'colocalizes_with'} -> OrganismTaxon\n",
      "NamedThing - {'contributes_to'} -> OrganismTaxon\n",
      "NamedThing - {'predisposes'} -> OrganismTaxon\n",
      "NamedThing - {'precedes'} -> OrganismTaxon\n",
      "NamedThing - {'manifestation_of'} -> OrganismTaxon\n",
      "NamedThing - {'coexists_with'} -> OrganismTaxon\n",
      "NamedThing - {'close_match'} -> OrganismTaxon\n",
      "OrganismTaxon - {'regulates'} -> Publication\n",
      "OrganismTaxon - {'gene_associated_with_condition'} -> Publication\n",
      "OrganismTaxon - {'located_in'} -> Publication\n",
      "OrganismTaxon - {'interacts_with'} -> Publication\n",
      "OrganismTaxon - {'produces'} -> Publication\n",
      "OrganismTaxon - {'related_to'} -> Publication\n",
      "OrganismTaxon - {'has_part'} -> Publication\n",
      "OrganismTaxon - {'predisposes'} -> Publication\n",
      "OrganismTaxon - {'occurs_in'} -> Publication\n",
      "OrganismTaxon - {'affects'} -> Publication\n",
      "OrganismTaxon - {'manifestation_of'} -> Publication\n",
      "OrganismTaxon - {'exacerbates'} -> Publication\n",
      "OrganismTaxon - {'coexists_with'} -> Publication\n",
      "OrganismTaxon - {'close_match'} -> Publication\n",
      "OrganismTaxon - {'subclass_of'} -> Publication\n",
      "OrganismTaxon - {'physically_interacts_with'} -> Publication\n",
      "OrganismTaxon - {'treats'} -> Publication\n",
      "OrganismTaxon - {'has_input'} -> Publication\n",
      "OrganismTaxon - {'causes'} -> Publication\n",
      "OrganismTaxon - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Pathway->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> Pathway\n",
      "NamedThing - {'subclass_of'} -> Pathway\n",
      "NamedThing - {'actively_involved_in'} -> Pathway\n",
      "NamedThing - {'actively_involves'} -> Pathway\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> Pathway\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> Pathway\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> Pathway\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> Pathway\n",
      "NamedThing - {'acts_upstream_of'} -> Pathway\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> Pathway\n",
      "NamedThing - {'affects'} -> Pathway\n",
      "NamedThing - {'capable_of'} -> Pathway\n",
      "NamedThing - {'enables'} -> Pathway\n",
      "NamedThing - {'has_part'} -> Pathway\n",
      "NamedThing - {'negatively_regulates'} -> Pathway\n",
      "NamedThing - {'overlaps'} -> Pathway\n",
      "NamedThing - {'participates_in'} -> Pathway\n",
      "NamedThing - {'part_of'} -> Pathway\n",
      "NamedThing - {'positively_regulates'} -> Pathway\n",
      "NamedThing - {'preceded_by'} -> Pathway\n",
      "NamedThing - {'precedes'} -> Pathway\n",
      "NamedThing - {'process_negatively_regulates_process'} -> Pathway\n",
      "NamedThing - {'process_positively_regulates_process'} -> Pathway\n",
      "NamedThing - {'process_regulates_process'} -> Pathway\n",
      "NamedThing - {'regulates'} -> Pathway\n",
      "NamedThing - {'related_to'} -> Pathway\n",
      "NamedThing - {'temporally_related_to'} -> Pathway\n",
      "NamedThing - {'composed_primarily_of'} -> Pathway\n",
      "NamedThing - {'is_substrate_of'} -> Pathway\n",
      "NamedThing - {'has_capability'} -> Pathway\n",
      "NamedThing - {'contributes_to'} -> Pathway\n",
      "NamedThing - {'caused_by'} -> Pathway\n",
      "NamedThing - {'has_participant'} -> Pathway\n",
      "NamedThing - {'is_input_of'} -> Pathway\n",
      "NamedThing - {'decreases_response_to'} -> Pathway\n",
      "NamedThing - {'associated_with'} -> Pathway\n",
      "NamedThing - {'catalyzes'} -> Pathway\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> Pathway\n",
      "NamedThing - {'has_upstream_actor'} -> Pathway\n",
      "NamedThing - {'exacerbates'} -> Pathway\n",
      "NamedThing - {'has_active_ingredient'} -> Pathway\n",
      "NamedThing - {'increases_response_to'} -> Pathway\n",
      "NamedThing - {'primarily_composed_of'} -> Pathway\n",
      "NamedThing - {'similar_to'} -> Pathway\n",
      "NamedThing - {'is_output_of'} -> Pathway\n",
      "NamedThing - {'affects_response_to'} -> Pathway\n",
      "NamedThing - {'causes'} -> Pathway\n",
      "NamedThing - {'disrupts'} -> Pathway\n",
      "NamedThing - {'affected_by'} -> Pathway\n",
      "NamedThing - {'interacts_with'} -> Pathway\n",
      "NamedThing - {'ameliorates'} -> Pathway\n",
      "NamedThing - {'treats'} -> Pathway\n",
      "NamedThing - {'contraindicated_for'} -> Pathway\n",
      "NamedThing - {'has_input'} -> Pathway\n",
      "NamedThing - {'derives_from'} -> Pathway\n",
      "NamedThing - {'physically_interacts_with'} -> Pathway\n",
      "NamedThing - {'directly_physically_interacts_with'} -> Pathway\n",
      "NamedThing - {'is_sequence_variant_of'} -> Pathway\n",
      "NamedThing - {'correlated_with'} -> Pathway\n",
      "NamedThing - {'gene_associated_with_condition'} -> Pathway\n",
      "NamedThing - {'has_phenotype'} -> Pathway\n",
      "NamedThing - {'located_in'} -> Pathway\n",
      "NamedThing - {'biomarker_for'} -> Pathway\n",
      "NamedThing - {'expressed_in'} -> Pathway\n",
      "NamedThing - {'prevents'} -> Pathway\n",
      "NamedThing - {'colocalizes_with'} -> Pathway\n",
      "NamedThing - {'produces'} -> Pathway\n",
      "NamedThing - {'predisposes'} -> Pathway\n",
      "NamedThing - {'diagnoses'} -> Pathway\n",
      "NamedThing - {'occurs_in'} -> Pathway\n",
      "NamedThing - {'coexists_with'} -> Pathway\n",
      "NamedThing - {'close_match'} -> Pathway\n",
      "NamedThing - {'manifestation_of'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Publication\n",
      "Pathway - {'regulates'} -> Publication\n",
      "Pathway - {'treats'} -> Publication\n",
      "Pathway - {'has_input'} -> Publication\n",
      "Pathway - {'causes'} -> Publication\n",
      "Pathway - {'associated_with'} -> Publication\n",
      "Pathway - {'gene_associated_with_condition'} -> Publication\n",
      "Pathway - {'disrupts'} -> Publication\n",
      "Pathway - {'located_in'} -> Publication\n",
      "Pathway - {'produces'} -> Publication\n",
      "Pathway - {'predisposes'} -> Publication\n",
      "Pathway - {'related_to'} -> Publication\n",
      "Pathway - {'precedes'} -> Publication\n",
      "Pathway - {'occurs_in'} -> Publication\n",
      "Pathway - {'affects'} -> Publication\n",
      "Pathway - {'manifestation_of'} -> Publication\n",
      "Pathway - {'coexists_with'} -> Publication\n",
      "Pathway - {'close_match'} -> Publication\n",
      "Pathway - {'has_participant'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->PhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> PhenotypicFeature\n",
      "NamedThing - {'subclass_of'} -> PhenotypicFeature\n",
      "NamedThing - {'actively_involved_in'} -> PhenotypicFeature\n",
      "NamedThing - {'actively_involves'} -> PhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> PhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of'} -> PhenotypicFeature\n",
      "NamedThing - {'affects'} -> PhenotypicFeature\n",
      "NamedThing - {'capable_of'} -> PhenotypicFeature\n",
      "NamedThing - {'has_part'} -> PhenotypicFeature\n",
      "NamedThing - {'negatively_regulates'} -> PhenotypicFeature\n",
      "NamedThing - {'overlaps'} -> PhenotypicFeature\n",
      "NamedThing - {'participates_in'} -> PhenotypicFeature\n",
      "NamedThing - {'part_of'} -> PhenotypicFeature\n",
      "NamedThing - {'positively_regulates'} -> PhenotypicFeature\n",
      "NamedThing - {'precedes'} -> PhenotypicFeature\n",
      "NamedThing - {'process_negatively_regulates_process'} -> PhenotypicFeature\n",
      "NamedThing - {'process_positively_regulates_process'} -> PhenotypicFeature\n",
      "NamedThing - {'process_regulates_process'} -> PhenotypicFeature\n",
      "NamedThing - {'regulates'} -> PhenotypicFeature\n",
      "NamedThing - {'related_to'} -> PhenotypicFeature\n",
      "NamedThing - {'temporally_related_to'} -> PhenotypicFeature\n",
      "NamedThing - {'preceded_by'} -> PhenotypicFeature\n",
      "NamedThing - {'has_participant'} -> PhenotypicFeature\n",
      "NamedThing - {'is_input_of'} -> PhenotypicFeature\n",
      "NamedThing - {'decreases_response_to'} -> PhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> PhenotypicFeature\n",
      "NamedThing - {'enables'} -> PhenotypicFeature\n",
      "NamedThing - {'expresses'} -> PhenotypicFeature\n",
      "NamedThing - {'interacts_with'} -> PhenotypicFeature\n",
      "NamedThing - {'ameliorates'} -> PhenotypicFeature\n",
      "NamedThing - {'has_active_ingredient'} -> PhenotypicFeature\n",
      "NamedThing - {'primarily_composed_of'} -> PhenotypicFeature\n",
      "NamedThing - {'associated_with'} -> PhenotypicFeature\n",
      "NamedThing - {'similar_to'} -> PhenotypicFeature\n",
      "NamedThing - {'catalyzes'} -> PhenotypicFeature\n",
      "NamedThing - {'affects_response_to'} -> PhenotypicFeature\n",
      "NamedThing - {'location_of'} -> PhenotypicFeature\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> PhenotypicFeature\n",
      "NamedThing - {'composed_primarily_of'} -> PhenotypicFeature\n",
      "NamedThing - {'has_upstream_actor'} -> PhenotypicFeature\n",
      "NamedThing - {'exacerbates'} -> PhenotypicFeature\n",
      "NamedThing - {'increases_response_to'} -> PhenotypicFeature\n",
      "NamedThing - {'disrupts'} -> PhenotypicFeature\n",
      "NamedThing - {'is_output_of'} -> PhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> PhenotypicFeature\n",
      "NamedThing - {'affected_by'} -> PhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> PhenotypicFeature\n",
      "NamedThing - {'treats'} -> PhenotypicFeature\n",
      "NamedThing - {'causes'} -> PhenotypicFeature\n",
      "NamedThing - {'is_substrate_of'} -> PhenotypicFeature\n",
      "NamedThing - {'has_capability'} -> PhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> PhenotypicFeature\n",
      "NamedThing - {'contributes_to'} -> PhenotypicFeature\n",
      "NamedThing - {'correlated_with'} -> PhenotypicFeature\n",
      "NamedThing - {'expressed_in'} -> PhenotypicFeature\n",
      "NamedThing - {'colocalizes_with'} -> PhenotypicFeature\n",
      "NamedThing - {'produces'} -> PhenotypicFeature\n",
      "NamedThing - {'predisposes'} -> PhenotypicFeature\n",
      "NamedThing - {'prevents'} -> PhenotypicFeature\n",
      "NamedThing - {'diagnoses'} -> PhenotypicFeature\n",
      "NamedThing - {'occurs_in'} -> PhenotypicFeature\n",
      "NamedThing - {'close_match'} -> PhenotypicFeature\n",
      "NamedThing - {'manifestation_of'} -> PhenotypicFeature\n",
      "NamedThing - {'coexists_with'} -> PhenotypicFeature\n",
      "NamedThing - {'chemically_similar_to'} -> PhenotypicFeature\n",
      "NamedThing - {'contraindicated_for'} -> PhenotypicFeature\n",
      "NamedThing - {'has_input'} -> PhenotypicFeature\n",
      "NamedThing - {'derives_from'} -> PhenotypicFeature\n",
      "NamedThing - {'physically_interacts_with'} -> PhenotypicFeature\n",
      "NamedThing - {'disease_has_location'} -> PhenotypicFeature\n",
      "NamedThing - {'directly_physically_interacts_with'} -> PhenotypicFeature\n",
      "NamedThing - {'gene_product_of'} -> PhenotypicFeature\n",
      "NamedThing - {'is_sequence_variant_of'} -> PhenotypicFeature\n",
      "NamedThing - {'gene_associated_with_condition'} -> PhenotypicFeature\n",
      "NamedThing - {'has_phenotype'} -> PhenotypicFeature\n",
      "NamedThing - {'located_in'} -> PhenotypicFeature\n",
      "NamedThing - {'binds'} -> PhenotypicFeature\n",
      "NamedThing - {'biomarker_for'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Publication\n",
      "PhenotypicFeature - {'occurs_in'} -> Publication\n",
      "PhenotypicFeature - {'affects'} -> Publication\n",
      "PhenotypicFeature - {'close_match'} -> Publication\n",
      "PhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "PhenotypicFeature - {'exacerbates'} -> Publication\n",
      "PhenotypicFeature - {'coexists_with'} -> Publication\n",
      "PhenotypicFeature - {'subclass_of'} -> Publication\n",
      "PhenotypicFeature - {'regulates'} -> Publication\n",
      "PhenotypicFeature - {'diagnoses'} -> Publication\n",
      "PhenotypicFeature - {'treats'} -> Publication\n",
      "PhenotypicFeature - {'has_input'} -> Publication\n",
      "PhenotypicFeature - {'causes'} -> Publication\n",
      "PhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "PhenotypicFeature - {'disrupts'} -> Publication\n",
      "PhenotypicFeature - {'associated_with'} -> Publication\n",
      "PhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "PhenotypicFeature - {'has_part'} -> Publication\n",
      "PhenotypicFeature - {'located_in'} -> Publication\n",
      "PhenotypicFeature - {'interacts_with'} -> Publication\n",
      "PhenotypicFeature - {'produces'} -> Publication\n",
      "PhenotypicFeature - {'prevents'} -> Publication\n",
      "PhenotypicFeature - {'contributes_to'} -> Publication\n",
      "PhenotypicFeature - {'predisposes'} -> Publication\n",
      "PhenotypicFeature - {'precedes'} -> Publication\n",
      "PhenotypicFeature - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Polypeptide->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> Polypeptide\n",
      "NamedThing - {'subclass_of'} -> Polypeptide\n",
      "NamedThing - {'actively_involved_in'} -> Polypeptide\n",
      "NamedThing - {'affects_activity_of'} -> Polypeptide\n",
      "NamedThing - {'affects'} -> Polypeptide\n",
      "NamedThing - {'affects_transport_of'} -> Polypeptide\n",
      "NamedThing - {'capable_of'} -> Polypeptide\n",
      "NamedThing - {'caused_by'} -> Polypeptide\n",
      "NamedThing - {'enabled_by'} -> Polypeptide\n",
      "NamedThing - {'enables'} -> Polypeptide\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> Polypeptide\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> Polypeptide\n",
      "NamedThing - {'entity_regulates_entity'} -> Polypeptide\n",
      "NamedThing - {'has_input'} -> Polypeptide\n",
      "NamedThing - {'has_output'} -> Polypeptide\n",
      "NamedThing - {'has_participant'} -> Polypeptide\n",
      "NamedThing - {'has_part'} -> Polypeptide\n",
      "NamedThing - {'interacts_with'} -> Polypeptide\n",
      "NamedThing - {'location_of'} -> Polypeptide\n",
      "NamedThing - {'molecularly_interacts_with'} -> Polypeptide\n",
      "NamedThing - {'overlaps'} -> Polypeptide\n",
      "NamedThing - {'participates_in'} -> Polypeptide\n",
      "NamedThing - {'physically_interacts_with'} -> Polypeptide\n",
      "NamedThing - {'preceded_by'} -> Polypeptide\n",
      "NamedThing - {'precedes'} -> Polypeptide\n",
      "NamedThing - {'related_to'} -> Polypeptide\n",
      "NamedThing - {'temporally_related_to'} -> Polypeptide\n",
      "NamedThing - {'has_catalyst'} -> Polypeptide\n",
      "NamedThing - {'is_input_of'} -> Polypeptide\n",
      "NamedThing - {'decreases_response_to'} -> Polypeptide\n",
      "NamedThing - {'associated_with'} -> Polypeptide\n",
      "NamedThing - {'expresses'} -> Polypeptide\n",
      "NamedThing - {'ameliorates'} -> Polypeptide\n",
      "NamedThing - {'genetically_interacts_with'} -> Polypeptide\n",
      "NamedThing - {'actively_involves'} -> Polypeptide\n",
      "NamedThing - {'has_upstream_actor'} -> Polypeptide\n",
      "NamedThing - {'has_active_ingredient'} -> Polypeptide\n",
      "NamedThing - {'primarily_composed_of'} -> Polypeptide\n",
      "NamedThing - {'similar_to'} -> Polypeptide\n",
      "NamedThing - {'catalyzes'} -> Polypeptide\n",
      "NamedThing - {'is_output_of'} -> Polypeptide\n",
      "NamedThing - {'affects_response_to'} -> Polypeptide\n",
      "NamedThing - {'composed_primarily_of'} -> Polypeptide\n",
      "NamedThing - {'exacerbates'} -> Polypeptide\n",
      "NamedThing - {'increases_response_to'} -> Polypeptide\n",
      "NamedThing - {'disrupts'} -> Polypeptide\n",
      "NamedThing - {'affected_by'} -> Polypeptide\n",
      "NamedThing - {'has_substrate'} -> Polypeptide\n",
      "NamedThing - {'treats'} -> Polypeptide\n",
      "NamedThing - {'regulates'} -> Polypeptide\n",
      "NamedThing - {'causes'} -> Polypeptide\n",
      "NamedThing - {'is_substrate_of'} -> Polypeptide\n",
      "NamedThing - {'part_of'} -> Polypeptide\n",
      "NamedThing - {'has_capability'} -> Polypeptide\n",
      "NamedThing - {'contributes_to'} -> Polypeptide\n",
      "NamedThing - {'chemically_similar_to'} -> Polypeptide\n",
      "NamedThing - {'derives_from'} -> Polypeptide\n",
      "NamedThing - {'disease_has_location'} -> Polypeptide\n",
      "NamedThing - {'gene_product_of'} -> Polypeptide\n",
      "NamedThing - {'directly_physically_interacts_with'} -> Polypeptide\n",
      "NamedThing - {'has_phenotype'} -> Polypeptide\n",
      "NamedThing - {'binds'} -> Polypeptide\n",
      "NamedThing - {'is_sequence_variant_of'} -> Polypeptide\n",
      "NamedThing - {'biomarker_for'} -> Polypeptide\n",
      "NamedThing - {'transcribed_from'} -> Polypeptide\n",
      "NamedThing - {'gene_associated_with_condition'} -> Polypeptide\n",
      "NamedThing - {'located_in'} -> Polypeptide\n",
      "NamedThing - {'produces'} -> Polypeptide\n",
      "NamedThing - {'expressed_in'} -> Polypeptide\n",
      "NamedThing - {'diagnoses'} -> Polypeptide\n",
      "NamedThing - {'colocalizes_with'} -> Polypeptide\n",
      "NamedThing - {'occurs_in'} -> Polypeptide\n",
      "NamedThing - {'manifestation_of'} -> Polypeptide\n",
      "NamedThing - {'coexists_with'} -> Polypeptide\n",
      "NamedThing - {'close_match'} -> Polypeptide\n",
      "NamedThing - {'has_metabolite'} -> Polypeptide\n",
      "Polypeptide - {'disrupts'} -> Publication\n",
      "Polypeptide - {'associated_with'} -> Publication\n",
      "Polypeptide - {'gene_associated_with_condition'} -> Publication\n",
      "Polypeptide - {'located_in'} -> Publication\n",
      "Polypeptide - {'regulates'} -> Publication\n",
      "Polypeptide - {'biomarker_for'} -> Publication\n",
      "Polypeptide - {'produces'} -> Publication\n",
      "Polypeptide - {'has_part'} -> Publication\n",
      "Polypeptide - {'physically_interacts_with'} -> Publication\n",
      "Polypeptide - {'colocalizes_with'} -> Publication\n",
      "Polypeptide - {'predisposes'} -> Publication\n",
      "Polypeptide - {'related_to'} -> Publication\n",
      "Polypeptide - {'prevents'} -> Publication\n",
      "Polypeptide - {'precedes'} -> Publication\n",
      "Polypeptide - {'close_match'} -> Publication\n",
      "Polypeptide - {'subclass_of'} -> Publication\n",
      "Polypeptide - {'interacts_with'} -> Publication\n",
      "Polypeptide - {'affects'} -> Publication\n",
      "Polypeptide - {'treats'} -> Publication\n",
      "Polypeptide - {'occurs_in'} -> Publication\n",
      "Polypeptide - {'exacerbates'} -> Publication\n",
      "Polypeptide - {'causes'} -> Publication\n",
      "Polypeptide - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Protein->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> Protein\n",
      "NamedThing - {'subclass_of'} -> Protein\n",
      "NamedThing - {'actively_involved_in'} -> Protein\n",
      "NamedThing - {'affects_activity_of'} -> Protein\n",
      "NamedThing - {'affects'} -> Protein\n",
      "NamedThing - {'affects_transport_of'} -> Protein\n",
      "NamedThing - {'capable_of'} -> Protein\n",
      "NamedThing - {'caused_by'} -> Protein\n",
      "NamedThing - {'enabled_by'} -> Protein\n",
      "NamedThing - {'enables'} -> Protein\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> Protein\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> Protein\n",
      "NamedThing - {'entity_regulates_entity'} -> Protein\n",
      "NamedThing - {'has_input'} -> Protein\n",
      "NamedThing - {'has_output'} -> Protein\n",
      "NamedThing - {'has_participant'} -> Protein\n",
      "NamedThing - {'has_part'} -> Protein\n",
      "NamedThing - {'interacts_with'} -> Protein\n",
      "NamedThing - {'location_of'} -> Protein\n",
      "NamedThing - {'molecularly_interacts_with'} -> Protein\n",
      "NamedThing - {'overlaps'} -> Protein\n",
      "NamedThing - {'participates_in'} -> Protein\n",
      "NamedThing - {'physically_interacts_with'} -> Protein\n",
      "NamedThing - {'preceded_by'} -> Protein\n",
      "NamedThing - {'precedes'} -> Protein\n",
      "NamedThing - {'related_to'} -> Protein\n",
      "NamedThing - {'temporally_related_to'} -> Protein\n",
      "NamedThing - {'negatively_regulates'} -> Protein\n",
      "NamedThing - {'expresses'} -> Protein\n",
      "NamedThing - {'has_substrate'} -> Protein\n",
      "NamedThing - {'ameliorates'} -> Protein\n",
      "NamedThing - {'regulates'} -> Protein\n",
      "NamedThing - {'is_substrate_of'} -> Protein\n",
      "NamedThing - {'part_of'} -> Protein\n",
      "NamedThing - {'has_capability'} -> Protein\n",
      "NamedThing - {'actively_involves'} -> Protein\n",
      "NamedThing - {'has_catalyst'} -> Protein\n",
      "NamedThing - {'associated_with'} -> Protein\n",
      "NamedThing - {'catalyzes'} -> Protein\n",
      "NamedThing - {'affects_response_to'} -> Protein\n",
      "NamedThing - {'process_positively_regulates_process'} -> Protein\n",
      "NamedThing - {'composed_primarily_of'} -> Protein\n",
      "NamedThing - {'genetically_interacts_with'} -> Protein\n",
      "NamedThing - {'has_upstream_actor'} -> Protein\n",
      "NamedThing - {'exacerbates'} -> Protein\n",
      "NamedThing - {'has_active_ingredient'} -> Protein\n",
      "NamedThing - {'increases_response_to'} -> Protein\n",
      "NamedThing - {'primarily_composed_of'} -> Protein\n",
      "NamedThing - {'process_regulates_process'} -> Protein\n",
      "NamedThing - {'disrupts'} -> Protein\n",
      "NamedThing - {'similar_to'} -> Protein\n",
      "NamedThing - {'is_output_of'} -> Protein\n",
      "NamedThing - {'affected_by'} -> Protein\n",
      "NamedThing - {'treats'} -> Protein\n",
      "NamedThing - {'causes'} -> Protein\n",
      "NamedThing - {'positively_regulates'} -> Protein\n",
      "NamedThing - {'contributes_to'} -> Protein\n",
      "NamedThing - {'is_input_of'} -> Protein\n",
      "NamedThing - {'decreases_response_to'} -> Protein\n",
      "NamedThing - {'gene_product_of'} -> Protein\n",
      "NamedThing - {'transcribed_from'} -> Protein\n",
      "NamedThing - {'directly_physically_interacts_with'} -> Protein\n",
      "NamedThing - {'has_phenotype'} -> Protein\n",
      "NamedThing - {'binds'} -> Protein\n",
      "NamedThing - {'is_sequence_variant_of'} -> Protein\n",
      "NamedThing - {'biomarker_for'} -> Protein\n",
      "NamedThing - {'gene_associated_with_condition'} -> Protein\n",
      "NamedThing - {'located_in'} -> Protein\n",
      "NamedThing - {'lacks_part'} -> Protein\n",
      "NamedThing - {'colocalizes_with'} -> Protein\n",
      "NamedThing - {'produces'} -> Protein\n",
      "NamedThing - {'expressed_in'} -> Protein\n",
      "NamedThing - {'prevents'} -> Protein\n",
      "NamedThing - {'diagnoses'} -> Protein\n",
      "NamedThing - {'predisposes'} -> Protein\n",
      "NamedThing - {'occurs_in'} -> Protein\n",
      "NamedThing - {'close_match'} -> Protein\n",
      "NamedThing - {'manifestation_of'} -> Protein\n",
      "NamedThing - {'coexists_with'} -> Protein\n",
      "NamedThing - {'contraindicated_for'} -> Protein\n",
      "NamedThing - {'chemically_similar_to'} -> Protein\n",
      "NamedThing - {'derives_from'} -> Protein\n",
      "NamedThing - {'disease_has_location'} -> Protein\n",
      "NamedThing - {'same_as'} -> Protein\n",
      "Protein - {'correlated_with'} -> Publication\n",
      "Protein - {'gene_product_of'} -> Publication\n",
      "Protein - {'associated_with'} -> Publication\n",
      "Protein - {'gene_associated_with_condition'} -> Publication\n",
      "Protein - {'located_in'} -> Publication\n",
      "Protein - {'regulates'} -> Publication\n",
      "Protein - {'biomarker_for'} -> Publication\n",
      "Protein - {'expressed_in'} -> Publication\n",
      "Protein - {'has_part'} -> Publication\n",
      "Protein - {'colocalizes_with'} -> Publication\n",
      "Protein - {'produces'} -> Publication\n",
      "Protein - {'predisposes'} -> Publication\n",
      "Protein - {'related_to'} -> Publication\n",
      "Protein - {'prevents'} -> Publication\n",
      "Protein - {'occurs_in'} -> Publication\n",
      "Protein - {'precedes'} -> Publication\n",
      "Protein - {'affects'} -> Publication\n",
      "Protein - {'close_match'} -> Publication\n",
      "Protein - {'subclass_of'} -> Publication\n",
      "Protein - {'interacts_with'} -> Publication\n",
      "Protein - {'treats'} -> Publication\n",
      "Protein - {'has_input'} -> Publication\n",
      "Protein - {'causes'} -> Publication\n",
      "Protein - {'physically_interacts_with'} -> Publication\n",
      "Protein - {'exacerbates'} -> Publication\n",
      "Protein - {'coexists_with'} -> Publication\n",
      "Protein - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->SmallMolecule->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'occurs_together_in_literature_with'} -> SmallMolecule\n",
      "NamedThing - {'has_upstream_actor'} -> SmallMolecule\n",
      "NamedThing - {'enabled_by'} -> SmallMolecule\n",
      "NamedThing - {'has_participant'} -> SmallMolecule\n",
      "NamedThing - {'has_catalyst'} -> SmallMolecule\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> SmallMolecule\n",
      "NamedThing - {'is_input_of'} -> SmallMolecule\n",
      "NamedThing - {'decreases_response_to'} -> SmallMolecule\n",
      "NamedThing - {'associated_with'} -> SmallMolecule\n",
      "NamedThing - {'enables'} -> SmallMolecule\n",
      "NamedThing - {'has_part'} -> SmallMolecule\n",
      "NamedThing - {'catalyzes'} -> SmallMolecule\n",
      "NamedThing - {'affects_activity_of'} -> SmallMolecule\n",
      "NamedThing - {'capable_of'} -> SmallMolecule\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> SmallMolecule\n",
      "NamedThing - {'genetically_interacts_with'} -> SmallMolecule\n",
      "NamedThing - {'exacerbates'} -> SmallMolecule\n",
      "NamedThing - {'has_active_ingredient'} -> SmallMolecule\n",
      "NamedThing - {'increases_response_to'} -> SmallMolecule\n",
      "NamedThing - {'primarily_composed_of'} -> SmallMolecule\n",
      "NamedThing - {'actively_involved_in'} -> SmallMolecule\n",
      "NamedThing - {'has_output'} -> SmallMolecule\n",
      "NamedThing - {'affects'} -> SmallMolecule\n",
      "NamedThing - {'similar_to'} -> SmallMolecule\n",
      "NamedThing - {'related_to'} -> SmallMolecule\n",
      "NamedThing - {'is_output_of'} -> SmallMolecule\n",
      "NamedThing - {'overlaps'} -> SmallMolecule\n",
      "NamedThing - {'affects_response_to'} -> SmallMolecule\n",
      "NamedThing - {'causes'} -> SmallMolecule\n",
      "NamedThing - {'composed_primarily_of'} -> SmallMolecule\n",
      "NamedThing - {'affects_transport_of'} -> SmallMolecule\n",
      "NamedThing - {'disrupts'} -> SmallMolecule\n",
      "NamedThing - {'physically_interacts_with'} -> SmallMolecule\n",
      "NamedThing - {'affected_by'} -> SmallMolecule\n",
      "NamedThing - {'has_substrate'} -> SmallMolecule\n",
      "NamedThing - {'molecularly_interacts_with'} -> SmallMolecule\n",
      "NamedThing - {'interacts_with'} -> SmallMolecule\n",
      "NamedThing - {'ameliorates'} -> SmallMolecule\n",
      "NamedThing - {'treats'} -> SmallMolecule\n",
      "NamedThing - {'regulates'} -> SmallMolecule\n",
      "NamedThing - {'is_substrate_of'} -> SmallMolecule\n",
      "NamedThing - {'part_of'} -> SmallMolecule\n",
      "NamedThing - {'has_capability'} -> SmallMolecule\n",
      "NamedThing - {'participates_in'} -> SmallMolecule\n",
      "NamedThing - {'contributes_to'} -> SmallMolecule\n",
      "NamedThing - {'caused_by'} -> SmallMolecule\n",
      "NamedThing - {'actively_involves'} -> SmallMolecule\n",
      "NamedThing - {'has_input'} -> SmallMolecule\n",
      "NamedThing - {'chemically_similar_to'} -> SmallMolecule\n",
      "NamedThing - {'contraindicated_for'} -> SmallMolecule\n",
      "NamedThing - {'derives_from'} -> SmallMolecule\n",
      "NamedThing - {'in_taxon'} -> SmallMolecule\n",
      "NamedThing - {'disease_has_location'} -> SmallMolecule\n",
      "NamedThing - {'directly_physically_interacts_with'} -> SmallMolecule\n",
      "NamedThing - {'binds'} -> SmallMolecule\n",
      "NamedThing - {'gene_product_of'} -> SmallMolecule\n",
      "NamedThing - {'is_sequence_variant_of'} -> SmallMolecule\n",
      "NamedThing - {'correlated_with'} -> SmallMolecule\n",
      "NamedThing - {'gene_associated_with_condition'} -> SmallMolecule\n",
      "NamedThing - {'has_phenotype'} -> SmallMolecule\n",
      "NamedThing - {'located_in'} -> SmallMolecule\n",
      "NamedThing - {'produces'} -> SmallMolecule\n",
      "NamedThing - {'expressed_in'} -> SmallMolecule\n",
      "NamedThing - {'subclass_of'} -> SmallMolecule\n",
      "NamedThing - {'prevents'} -> SmallMolecule\n",
      "NamedThing - {'colocalizes_with'} -> SmallMolecule\n",
      "NamedThing - {'predisposes'} -> SmallMolecule\n",
      "NamedThing - {'diagnoses'} -> SmallMolecule\n",
      "NamedThing - {'occurs_in'} -> SmallMolecule\n",
      "NamedThing - {'precedes'} -> SmallMolecule\n",
      "NamedThing - {'same_as'} -> SmallMolecule\n",
      "NamedThing - {'coexists_with'} -> SmallMolecule\n",
      "NamedThing - {'close_match'} -> SmallMolecule\n",
      "NamedThing - {'manifestation_of'} -> SmallMolecule\n",
      "NamedThing - {'has_metabolite'} -> SmallMolecule\n",
      "SmallMolecule - {'correlated_with'} -> Publication\n",
      "SmallMolecule - {'disrupts'} -> Publication\n",
      "SmallMolecule - {'gene_associated_with_condition'} -> Publication\n",
      "SmallMolecule - {'located_in'} -> Publication\n",
      "SmallMolecule - {'regulates'} -> Publication\n",
      "SmallMolecule - {'has_part'} -> Publication\n",
      "SmallMolecule - {'physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'predisposes'} -> Publication\n",
      "SmallMolecule - {'related_to'} -> Publication\n",
      "SmallMolecule - {'causes'} -> Publication\n",
      "SmallMolecule - {'directly_physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'precedes'} -> Publication\n",
      "SmallMolecule - {'close_match'} -> Publication\n",
      "SmallMolecule - {'associated_with'} -> Publication\n",
      "SmallMolecule - {'has_metabolite'} -> Publication\n",
      "SmallMolecule - {'subclass_of'} -> Publication\n",
      "SmallMolecule - {'interacts_with'} -> Publication\n",
      "SmallMolecule - {'treats'} -> Publication\n",
      "SmallMolecule - {'has_input'} -> Publication\n",
      "SmallMolecule - {'occurs_in'} -> Publication\n",
      "SmallMolecule - {'affects'} -> Publication\n",
      "SmallMolecule - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->CellLine->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'affects'} -> CellLine\n",
      "NamedThing - {'subclass_of'} -> CellLine\n",
      "NamedThing - {'related_to'} -> CellLine\n",
      "NamedThing - {'ameliorates'} -> CellLine\n",
      "NamedThing - {'treats'} -> CellLine\n",
      "NamedThing - {'diagnoses'} -> CellLine\n",
      "NamedThing - {'occurs_in'} -> CellLine\n",
      "NamedThing - {'coexists_with'} -> CellLine\n",
      "NamedThing - {'close_match'} -> CellLine\n",
      "NamedThing - {'has_participant'} -> CellLine\n",
      "NamedThing - {'regulates'} -> CellLine\n",
      "NamedThing - {'physically_interacts_with'} -> CellLine\n",
      "NamedThing - {'has_input'} -> CellLine\n",
      "NamedThing - {'derives_from'} -> CellLine\n",
      "NamedThing - {'disease_has_location'} -> CellLine\n",
      "NamedThing - {'directly_physically_interacts_with'} -> CellLine\n",
      "NamedThing - {'binds'} -> CellLine\n",
      "NamedThing - {'disrupts'} -> CellLine\n",
      "NamedThing - {'gene_product_of'} -> CellLine\n",
      "NamedThing - {'is_sequence_variant_of'} -> CellLine\n",
      "NamedThing - {'associated_with'} -> CellLine\n",
      "NamedThing - {'has_phenotype'} -> CellLine\n",
      "NamedThing - {'located_in'} -> CellLine\n",
      "NamedThing - {'interacts_with'} -> CellLine\n",
      "NamedThing - {'produces'} -> CellLine\n",
      "NamedThing - {'expressed_in'} -> CellLine\n",
      "NamedThing - {'has_part'} -> CellLine\n",
      "CellLine - {'gene_associated_with_condition'} -> Publication\n",
      "CellLine - {'located_in'} -> Publication\n",
      "CellLine - {'produces'} -> Publication\n",
      "CellLine - {'related_to'} -> Publication\n",
      "CellLine - {'affects'} -> Publication\n",
      "CellLine - {'close_match'} -> Publication\n",
      "CellLine - {'subclass_of'} -> Publication\n",
      "CellLine - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Entity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'catalyzes'} -> Entity\n",
      "NamedThing - {'subclass_of'} -> Entity\n",
      "NamedThing - {'active_in'} -> Entity\n",
      "NamedThing - {'actively_involved_in'} -> Entity\n",
      "NamedThing - {'actively_involves'} -> Entity\n",
      "NamedThing - {'acts_upstream_of'} -> Entity\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> Entity\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> Entity\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> Entity\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> Entity\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> Entity\n",
      "NamedThing - {'affects_activity_of'} -> Entity\n",
      "NamedThing - {'affects'} -> Entity\n",
      "NamedThing - {'affects_transport_of'} -> Entity\n",
      "NamedThing - {'capable_of'} -> Entity\n",
      "NamedThing - {'caused_by'} -> Entity\n",
      "NamedThing - {'causes'} -> Entity\n",
      "NamedThing - {'coexists_with'} -> Entity\n",
      "NamedThing - {'contributes_to'} -> Entity\n",
      "NamedThing - {'derives_from'} -> Entity\n",
      "NamedThing - {'derives_into'} -> Entity\n",
      "NamedThing - {'enabled_by'} -> Entity\n",
      "NamedThing - {'enables'} -> Entity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> Entity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> Entity\n",
      "NamedThing - {'entity_regulates_entity'} -> Entity\n",
      "NamedThing - {'has_attribute'} -> Entity\n",
      "NamedThing - {'has_input'} -> Entity\n",
      "NamedThing - {'has_output'} -> Entity\n",
      "NamedThing - {'has_part'} -> Entity\n",
      "NamedThing - {'has_participant'} -> Entity\n",
      "NamedThing - {'interacts_with'} -> Entity\n",
      "NamedThing - {'located_in'} -> Entity\n",
      "NamedThing - {'location_of'} -> Entity\n",
      "NamedThing - {'molecularly_interacts_with'} -> Entity\n",
      "NamedThing - {'negatively_regulates'} -> Entity\n",
      "NamedThing - {'occurs_in'} -> Entity\n",
      "NamedThing - {'overlaps'} -> Entity\n",
      "NamedThing - {'participates_in'} -> Entity\n",
      "NamedThing - {'part_of'} -> Entity\n",
      "NamedThing - {'physically_interacts_with'} -> Entity\n",
      "NamedThing - {'positively_regulates'} -> Entity\n",
      "NamedThing - {'preceded_by'} -> Entity\n",
      "NamedThing - {'precedes'} -> Entity\n",
      "NamedThing - {'process_negatively_regulates_process'} -> Entity\n",
      "NamedThing - {'process_positively_regulates_process'} -> Entity\n",
      "NamedThing - {'process_regulates_process'} -> Entity\n",
      "NamedThing - {'produced_by'} -> Entity\n",
      "NamedThing - {'produces'} -> Entity\n",
      "NamedThing - {'regulates'} -> Entity\n",
      "NamedThing - {'related_to'} -> Entity\n",
      "NamedThing - {'temporally_related_to'} -> Entity\n",
      "NamedThing - {'ameliorates'} -> Entity\n",
      "NamedThing - {'has_catalyst'} -> Entity\n",
      "NamedThing - {'expressed_in'} -> Entity\n",
      "NamedThing - {'decreases_response_to'} -> Entity\n",
      "NamedThing - {'associated_with'} -> Entity\n",
      "NamedThing - {'affects_response_to'} -> Entity\n",
      "NamedThing - {'has_metabolite'} -> Entity\n",
      "NamedThing - {'composed_primarily_of'} -> Entity\n",
      "NamedThing - {'genetically_interacts_with'} -> Entity\n",
      "NamedThing - {'has_upstream_actor'} -> Entity\n",
      "NamedThing - {'exacerbates'} -> Entity\n",
      "NamedThing - {'has_active_ingredient'} -> Entity\n",
      "NamedThing - {'increases_response_to'} -> Entity\n",
      "NamedThing - {'primarily_composed_of'} -> Entity\n",
      "NamedThing - {'disrupts'} -> Entity\n",
      "NamedThing - {'similar_to'} -> Entity\n",
      "NamedThing - {'is_output_of'} -> Entity\n",
      "NamedThing - {'affected_by'} -> Entity\n",
      "NamedThing - {'treats'} -> Entity\n",
      "NamedThing - {'is_input_of'} -> Entity\n",
      "NamedThing - {'expresses'} -> Entity\n",
      "NamedThing - {'has_substrate'} -> Entity\n",
      "NamedThing - {'is_substrate_of'} -> Entity\n",
      "NamedThing - {'has_capability'} -> Entity\n",
      "Entity - {'subclass_of'} -> Publication\n",
      "Entity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->InformationContentEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> InformationContentEntity\n",
      "NamedThing - {'related_to'} -> InformationContentEntity\n",
      "NamedThing - {'primarily_composed_of'} -> InformationContentEntity\n",
      "NamedThing - {'affects'} -> InformationContentEntity\n",
      "NamedThing - {'similar_to'} -> InformationContentEntity\n",
      "NamedThing - {'overlaps'} -> InformationContentEntity\n",
      "NamedThing - {'composed_primarily_of'} -> InformationContentEntity\n",
      "NamedThing - {'has_upstream_actor'} -> InformationContentEntity\n",
      "NamedThing - {'affected_by'} -> InformationContentEntity\n",
      "NamedThing - {'interacts_with'} -> InformationContentEntity\n",
      "NamedThing - {'participates_in'} -> InformationContentEntity\n",
      "NamedThing - {'has_participant'} -> InformationContentEntity\n",
      "NamedThing - {'associated_with'} -> InformationContentEntity\n",
      "NamedThing - {'regulates'} -> InformationContentEntity\n",
      "NamedThing - {'contraindicated_for'} -> InformationContentEntity\n",
      "NamedThing - {'chemically_similar_to'} -> InformationContentEntity\n",
      "NamedThing - {'physically_interacts_with'} -> InformationContentEntity\n",
      "NamedThing - {'treats'} -> InformationContentEntity\n",
      "NamedThing - {'has_input'} -> InformationContentEntity\n",
      "NamedThing - {'derives_from'} -> InformationContentEntity\n",
      "NamedThing - {'causes'} -> InformationContentEntity\n",
      "NamedThing - {'has_output'} -> InformationContentEntity\n",
      "NamedThing - {'disease_has_location'} -> InformationContentEntity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> InformationContentEntity\n",
      "NamedThing - {'actively_involved_in'} -> InformationContentEntity\n",
      "NamedThing - {'binds'} -> InformationContentEntity\n",
      "NamedThing - {'disrupts'} -> InformationContentEntity\n",
      "NamedThing - {'gene_product_of'} -> InformationContentEntity\n",
      "NamedThing - {'is_sequence_variant_of'} -> InformationContentEntity\n",
      "NamedThing - {'correlated_with'} -> InformationContentEntity\n",
      "NamedThing - {'gene_associated_with_condition'} -> InformationContentEntity\n",
      "NamedThing - {'has_phenotype'} -> InformationContentEntity\n",
      "NamedThing - {'located_in'} -> InformationContentEntity\n",
      "NamedThing - {'biomarker_for'} -> InformationContentEntity\n",
      "NamedThing - {'produces'} -> InformationContentEntity\n",
      "NamedThing - {'expressed_in'} -> InformationContentEntity\n",
      "NamedThing - {'has_part'} -> InformationContentEntity\n",
      "NamedThing - {'prevents'} -> InformationContentEntity\n",
      "NamedThing - {'diagnoses'} -> InformationContentEntity\n",
      "NamedThing - {'enables'} -> InformationContentEntity\n",
      "NamedThing - {'colocalizes_with'} -> InformationContentEntity\n",
      "NamedThing - {'contributes_to'} -> InformationContentEntity\n",
      "NamedThing - {'predisposes'} -> InformationContentEntity\n",
      "NamedThing - {'precedes'} -> InformationContentEntity\n",
      "NamedThing - {'occurs_in'} -> InformationContentEntity\n",
      "NamedThing - {'manifestation_of'} -> InformationContentEntity\n",
      "NamedThing - {'exacerbates'} -> InformationContentEntity\n",
      "NamedThing - {'coexists_with'} -> InformationContentEntity\n",
      "NamedThing - {'close_match'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Publication\n",
      "InformationContentEntity - {'subclass_of'} -> Publication\n",
      "InformationContentEntity - {'related_to'} -> Publication\n",
      "InformationContentEntity - {'disrupts'} -> Publication\n",
      "InformationContentEntity - {'associated_with'} -> Publication\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Publication\n",
      "InformationContentEntity - {'regulates'} -> Publication\n",
      "InformationContentEntity - {'biomarker_for'} -> Publication\n",
      "InformationContentEntity - {'produces'} -> Publication\n",
      "InformationContentEntity - {'has_part'} -> Publication\n",
      "InformationContentEntity - {'located_in'} -> Publication\n",
      "InformationContentEntity - {'predisposes'} -> Publication\n",
      "InformationContentEntity - {'prevents'} -> Publication\n",
      "InformationContentEntity - {'affects'} -> Publication\n",
      "InformationContentEntity - {'manifestation_of'} -> Publication\n",
      "InformationContentEntity - {'precedes'} -> Publication\n",
      "InformationContentEntity - {'close_match'} -> Publication\n",
      "InformationContentEntity - {'interacts_with'} -> Publication\n",
      "InformationContentEntity - {'diagnoses'} -> Publication\n",
      "InformationContentEntity - {'treats'} -> Publication\n",
      "InformationContentEntity - {'occurs_in'} -> Publication\n",
      "InformationContentEntity - {'has_input'} -> Publication\n",
      "InformationContentEntity - {'exacerbates'} -> Publication\n",
      "InformationContentEntity - {'causes'} -> Publication\n",
      "InformationContentEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->NoncodingRNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_input'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_output'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_participant'} -> NoncodingRNAProduct\n",
      "NamedThing - {'related_to'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_capability'} -> NoncodingRNAProduct\n",
      "NamedThing - {'participates_in'} -> NoncodingRNAProduct\n",
      "NamedThing - {'enabled_by'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_catalyst'} -> NoncodingRNAProduct\n",
      "NamedThing - {'associated_with'} -> NoncodingRNAProduct\n",
      "NamedThing - {'actively_involves'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_upstream_actor'} -> NoncodingRNAProduct\n",
      "NamedThing - {'primarily_composed_of'} -> NoncodingRNAProduct\n",
      "NamedThing - {'affects'} -> NoncodingRNAProduct\n",
      "NamedThing - {'similar_to'} -> NoncodingRNAProduct\n",
      "NamedThing - {'overlaps'} -> NoncodingRNAProduct\n",
      "NamedThing - {'composed_primarily_of'} -> NoncodingRNAProduct\n",
      "NamedThing - {'affected_by'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_substrate'} -> NoncodingRNAProduct\n",
      "NamedThing - {'interacts_with'} -> NoncodingRNAProduct\n",
      "NamedThing - {'is_sequence_variant_of'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_phenotype'} -> NoncodingRNAProduct\n",
      "NamedThing - {'has_part'} -> NoncodingRNAProduct\n",
      "NamedThing - {'produces'} -> NoncodingRNAProduct\n",
      "NamedThing - {'coexists_with'} -> NoncodingRNAProduct\n",
      "NamedThing - {'close_match'} -> NoncodingRNAProduct\n",
      "NamedThing - {'regulates'} -> NoncodingRNAProduct\n",
      "NamedThing - {'derives_from'} -> NoncodingRNAProduct\n",
      "NamedThing - {'physically_interacts_with'} -> NoncodingRNAProduct\n",
      "NoncodingRNAProduct - {'affects'} -> Publication\n",
      "NoncodingRNAProduct - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Drug->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Drug\n",
      "NamedThing - {'has_input'} -> Drug\n",
      "NamedThing - {'has_output'} -> Drug\n",
      "NamedThing - {'has_participant'} -> Drug\n",
      "NamedThing - {'preceded_by'} -> Drug\n",
      "NamedThing - {'related_to'} -> Drug\n",
      "NamedThing - {'temporally_related_to'} -> Drug\n",
      "NamedThing - {'has_capability'} -> Drug\n",
      "NamedThing - {'actively_involves'} -> Drug\n",
      "NamedThing - {'has_catalyst'} -> Drug\n",
      "NamedThing - {'associated_with'} -> Drug\n",
      "NamedThing - {'affects'} -> Drug\n",
      "NamedThing - {'precedes'} -> Drug\n",
      "NamedThing - {'overlaps'} -> Drug\n",
      "NamedThing - {'composed_primarily_of'} -> Drug\n",
      "NamedThing - {'has_upstream_actor'} -> Drug\n",
      "NamedThing - {'primarily_composed_of'} -> Drug\n",
      "NamedThing - {'similar_to'} -> Drug\n",
      "NamedThing - {'affected_by'} -> Drug\n",
      "NamedThing - {'treats'} -> Drug\n",
      "NamedThing - {'participates_in'} -> Drug\n",
      "NamedThing - {'enabled_by'} -> Drug\n",
      "NamedThing - {'has_substrate'} -> Drug\n",
      "NamedThing - {'interacts_with'} -> Drug\n",
      "NamedThing - {'ameliorates'} -> Drug\n",
      "NamedThing - {'chemically_similar_to'} -> Drug\n",
      "NamedThing - {'contraindicated_for'} -> Drug\n",
      "NamedThing - {'derives_from'} -> Drug\n",
      "NamedThing - {'causes'} -> Drug\n",
      "NamedThing - {'in_taxon'} -> Drug\n",
      "NamedThing - {'physically_interacts_with'} -> Drug\n",
      "NamedThing - {'disease_has_location'} -> Drug\n",
      "NamedThing - {'directly_physically_interacts_with'} -> Drug\n",
      "NamedThing - {'binds'} -> Drug\n",
      "NamedThing - {'disrupts'} -> Drug\n",
      "NamedThing - {'gene_product_of'} -> Drug\n",
      "NamedThing - {'is_sequence_variant_of'} -> Drug\n",
      "NamedThing - {'correlated_with'} -> Drug\n",
      "NamedThing - {'gene_associated_with_condition'} -> Drug\n",
      "NamedThing - {'has_phenotype'} -> Drug\n",
      "NamedThing - {'located_in'} -> Drug\n",
      "NamedThing - {'biomarker_for'} -> Drug\n",
      "NamedThing - {'expressed_in'} -> Drug\n",
      "NamedThing - {'has_part'} -> Drug\n",
      "NamedThing - {'prevents'} -> Drug\n",
      "NamedThing - {'colocalizes_with'} -> Drug\n",
      "NamedThing - {'produces'} -> Drug\n",
      "NamedThing - {'predisposes'} -> Drug\n",
      "NamedThing - {'diagnoses'} -> Drug\n",
      "NamedThing - {'occurs_in'} -> Drug\n",
      "NamedThing - {'exacerbates'} -> Drug\n",
      "NamedThing - {'same_as'} -> Drug\n",
      "NamedThing - {'coexists_with'} -> Drug\n",
      "NamedThing - {'close_match'} -> Drug\n",
      "NamedThing - {'manifestation_of'} -> Drug\n",
      "NamedThing - {'has_metabolite'} -> Drug\n",
      "NamedThing - {'regulates'} -> Drug\n",
      "Drug - {'correlated_with'} -> Publication\n",
      "Drug - {'regulates'} -> Publication\n",
      "Drug - {'treats'} -> Publication\n",
      "Drug - {'has_input'} -> Publication\n",
      "Drug - {'physically_interacts_with'} -> Publication\n",
      "Drug - {'causes'} -> Publication\n",
      "Drug - {'directly_physically_interacts_with'} -> Publication\n",
      "Drug - {'disrupts'} -> Publication\n",
      "Drug - {'associated_with'} -> Publication\n",
      "Drug - {'gene_associated_with_condition'} -> Publication\n",
      "Drug - {'located_in'} -> Publication\n",
      "Drug - {'has_part'} -> Publication\n",
      "Drug - {'prevents'} -> Publication\n",
      "Drug - {'colocalizes_with'} -> Publication\n",
      "Drug - {'interacts_with'} -> Publication\n",
      "Drug - {'predisposes'} -> Publication\n",
      "Drug - {'related_to'} -> Publication\n",
      "Drug - {'occurs_in'} -> Publication\n",
      "Drug - {'precedes'} -> Publication\n",
      "Drug - {'affects'} -> Publication\n",
      "Drug - {'close_match'} -> Publication\n",
      "Drug - {'subclass_of'} -> Publication\n",
      "Drug - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->EnvironmentalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'actively_involved_in'} -> EnvironmentalProcess\n",
      "NamedThing - {'actively_involves'} -> EnvironmentalProcess\n",
      "NamedThing - {'acts_upstream_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> EnvironmentalProcess\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> EnvironmentalProcess\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> EnvironmentalProcess\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> EnvironmentalProcess\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> EnvironmentalProcess\n",
      "NamedThing - {'affects'} -> EnvironmentalProcess\n",
      "NamedThing - {'capable_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'enables'} -> EnvironmentalProcess\n",
      "NamedThing - {'has_part'} -> EnvironmentalProcess\n",
      "NamedThing - {'negatively_regulates'} -> EnvironmentalProcess\n",
      "NamedThing - {'overlaps'} -> EnvironmentalProcess\n",
      "NamedThing - {'participates_in'} -> EnvironmentalProcess\n",
      "NamedThing - {'part_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'positively_regulates'} -> EnvironmentalProcess\n",
      "NamedThing - {'preceded_by'} -> EnvironmentalProcess\n",
      "NamedThing - {'precedes'} -> EnvironmentalProcess\n",
      "NamedThing - {'process_negatively_regulates_process'} -> EnvironmentalProcess\n",
      "NamedThing - {'process_positively_regulates_process'} -> EnvironmentalProcess\n",
      "NamedThing - {'process_regulates_process'} -> EnvironmentalProcess\n",
      "NamedThing - {'regulates'} -> EnvironmentalProcess\n",
      "NamedThing - {'related_to'} -> EnvironmentalProcess\n",
      "NamedThing - {'temporally_related_to'} -> EnvironmentalProcess\n",
      "NamedThing - {'is_input_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'decreases_response_to'} -> EnvironmentalProcess\n",
      "NamedThing - {'interacts_with'} -> EnvironmentalProcess\n",
      "NamedThing - {'ameliorates'} -> EnvironmentalProcess\n",
      "NamedThing - {'is_substrate_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'has_capability'} -> EnvironmentalProcess\n",
      "NamedThing - {'associated_with'} -> EnvironmentalProcess\n",
      "NamedThing - {'similar_to'} -> EnvironmentalProcess\n",
      "NamedThing - {'catalyzes'} -> EnvironmentalProcess\n",
      "NamedThing - {'affects_response_to'} -> EnvironmentalProcess\n",
      "NamedThing - {'composed_primarily_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'has_upstream_actor'} -> EnvironmentalProcess\n",
      "NamedThing - {'exacerbates'} -> EnvironmentalProcess\n",
      "NamedThing - {'has_active_ingredient'} -> EnvironmentalProcess\n",
      "NamedThing - {'increases_response_to'} -> EnvironmentalProcess\n",
      "NamedThing - {'primarily_composed_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'disrupts'} -> EnvironmentalProcess\n",
      "NamedThing - {'is_output_of'} -> EnvironmentalProcess\n",
      "NamedThing - {'affected_by'} -> EnvironmentalProcess\n",
      "NamedThing - {'treats'} -> EnvironmentalProcess\n",
      "NamedThing - {'has_participant'} -> EnvironmentalProcess\n",
      "NamedThing - {'predisposes'} -> EnvironmentalProcess\n",
      "NamedThing - {'prevents'} -> EnvironmentalProcess\n",
      "NamedThing - {'close_match'} -> EnvironmentalProcess\n",
      "NamedThing - {'has_input'} -> EnvironmentalProcess\n",
      "NamedThing - {'causes'} -> EnvironmentalProcess\n",
      "EnvironmentalProcess - {'subclass_of'} -> Publication\n",
      "EnvironmentalProcess - {'related_to'} -> Publication\n",
      "EnvironmentalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Transcript->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Transcript\n",
      "NamedThing - {'affects'} -> Transcript\n",
      "NamedThing - {'affects_transport_of'} -> Transcript\n",
      "NamedThing - {'has_input'} -> Transcript\n",
      "NamedThing - {'has_output'} -> Transcript\n",
      "NamedThing - {'has_participant'} -> Transcript\n",
      "NamedThing - {'related_to'} -> Transcript\n",
      "NamedThing - {'decreases_response_to'} -> Transcript\n",
      "NamedThing - {'enabled_by'} -> Transcript\n",
      "NamedThing - {'associated_with'} -> Transcript\n",
      "NamedThing - {'has_substrate'} -> Transcript\n",
      "NamedThing - {'ameliorates'} -> Transcript\n",
      "NamedThing - {'actively_involves'} -> Transcript\n",
      "NamedThing - {'has_catalyst'} -> Transcript\n",
      "NamedThing - {'primarily_composed_of'} -> Transcript\n",
      "NamedThing - {'similar_to'} -> Transcript\n",
      "NamedThing - {'overlaps'} -> Transcript\n",
      "NamedThing - {'affects_response_to'} -> Transcript\n",
      "NamedThing - {'composed_primarily_of'} -> Transcript\n",
      "NamedThing - {'has_upstream_actor'} -> Transcript\n",
      "NamedThing - {'exacerbates'} -> Transcript\n",
      "NamedThing - {'increases_response_to'} -> Transcript\n",
      "NamedThing - {'disrupts'} -> Transcript\n",
      "NamedThing - {'affected_by'} -> Transcript\n",
      "NamedThing - {'interacts_with'} -> Transcript\n",
      "NamedThing - {'treats'} -> Transcript\n",
      "NamedThing - {'regulates'} -> Transcript\n",
      "NamedThing - {'has_capability'} -> Transcript\n",
      "NamedThing - {'participates_in'} -> Transcript\n",
      "NamedThing - {'derives_from'} -> Transcript\n",
      "NamedThing - {'has_part'} -> Transcript\n",
      "NamedThing - {'located_in'} -> Transcript\n",
      "NamedThing - {'produces'} -> Transcript\n",
      "NamedThing - {'occurs_in'} -> Transcript\n",
      "NamedThing - {'close_match'} -> Transcript\n",
      "NamedThing - {'coexists_with'} -> Transcript\n",
      "Transcript - {'translates_to'} -> Publication\n",
      "Transcript - {'regulates'} -> Publication\n",
      "Transcript - {'transcribed_from'} -> Publication\n",
      "Transcript - {'associated_with'} -> Publication\n",
      "Transcript - {'located_in'} -> Publication\n",
      "Transcript - {'subclass_of'} -> Publication\n",
      "Transcript - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Behavior->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Behavior\n",
      "NamedThing - {'actively_involved_in'} -> Behavior\n",
      "NamedThing - {'actively_involves'} -> Behavior\n",
      "NamedThing - {'acts_upstream_of'} -> Behavior\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> Behavior\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> Behavior\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> Behavior\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> Behavior\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> Behavior\n",
      "NamedThing - {'affects'} -> Behavior\n",
      "NamedThing - {'capable_of'} -> Behavior\n",
      "NamedThing - {'has_part'} -> Behavior\n",
      "NamedThing - {'negatively_regulates'} -> Behavior\n",
      "NamedThing - {'overlaps'} -> Behavior\n",
      "NamedThing - {'participates_in'} -> Behavior\n",
      "NamedThing - {'part_of'} -> Behavior\n",
      "NamedThing - {'positively_regulates'} -> Behavior\n",
      "NamedThing - {'precedes'} -> Behavior\n",
      "NamedThing - {'process_negatively_regulates_process'} -> Behavior\n",
      "NamedThing - {'process_positively_regulates_process'} -> Behavior\n",
      "NamedThing - {'process_regulates_process'} -> Behavior\n",
      "NamedThing - {'regulates'} -> Behavior\n",
      "NamedThing - {'related_to'} -> Behavior\n",
      "NamedThing - {'temporally_related_to'} -> Behavior\n",
      "NamedThing - {'affected_by'} -> Behavior\n",
      "NamedThing - {'interacts_with'} -> Behavior\n",
      "NamedThing - {'treats'} -> Behavior\n",
      "NamedThing - {'is_substrate_of'} -> Behavior\n",
      "NamedThing - {'has_capability'} -> Behavior\n",
      "NamedThing - {'preceded_by'} -> Behavior\n",
      "NamedThing - {'has_participant'} -> Behavior\n",
      "NamedThing - {'is_input_of'} -> Behavior\n",
      "NamedThing - {'decreases_response_to'} -> Behavior\n",
      "NamedThing - {'associated_with'} -> Behavior\n",
      "NamedThing - {'enables'} -> Behavior\n",
      "NamedThing - {'ameliorates'} -> Behavior\n",
      "NamedThing - {'has_active_ingredient'} -> Behavior\n",
      "NamedThing - {'primarily_composed_of'} -> Behavior\n",
      "NamedThing - {'similar_to'} -> Behavior\n",
      "NamedThing - {'catalyzes'} -> Behavior\n",
      "NamedThing - {'is_output_of'} -> Behavior\n",
      "NamedThing - {'affects_response_to'} -> Behavior\n",
      "NamedThing - {'composed_primarily_of'} -> Behavior\n",
      "NamedThing - {'has_upstream_actor'} -> Behavior\n",
      "NamedThing - {'exacerbates'} -> Behavior\n",
      "NamedThing - {'increases_response_to'} -> Behavior\n",
      "NamedThing - {'disrupts'} -> Behavior\n",
      "NamedThing - {'diagnoses'} -> Behavior\n",
      "NamedThing - {'occurs_in'} -> Behavior\n",
      "NamedThing - {'coexists_with'} -> Behavior\n",
      "NamedThing - {'close_match'} -> Behavior\n",
      "NamedThing - {'manifestation_of'} -> Behavior\n",
      "NamedThing - {'has_input'} -> Behavior\n",
      "NamedThing - {'causes'} -> Behavior\n",
      "NamedThing - {'gene_associated_with_condition'} -> Behavior\n",
      "NamedThing - {'has_phenotype'} -> Behavior\n",
      "NamedThing - {'located_in'} -> Behavior\n",
      "NamedThing - {'prevents'} -> Behavior\n",
      "NamedThing - {'predisposes'} -> Behavior\n",
      "Behavior - {'treats'} -> Publication\n",
      "Behavior - {'has_input'} -> Publication\n",
      "Behavior - {'subclass_of'} -> Publication\n",
      "Behavior - {'located_in'} -> Publication\n",
      "Behavior - {'interacts_with'} -> Publication\n",
      "Behavior - {'predisposes'} -> Publication\n",
      "Behavior - {'related_to'} -> Publication\n",
      "Behavior - {'occurs_in'} -> Publication\n",
      "Behavior - {'affects'} -> Publication\n",
      "Behavior - {'coexists_with'} -> Publication\n",
      "Behavior - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->BiologicalProcessOrActivity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'active_in'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'actively_involved_in'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'actively_involves'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'acts_upstream_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'affects_activity_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'affects'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'affects_transport_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'capable_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'caused_by'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'causes'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'contributes_to'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'derives_from'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'derives_into'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'enabled_by'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'enables'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'entity_regulates_entity'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_input'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_output'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_part'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_participant'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'interacts_with'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'molecularly_interacts_with'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'negatively_regulates'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'occurs_in'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'overlaps'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'participates_in'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'part_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'physically_interacts_with'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'positively_regulates'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'preceded_by'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'precedes'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'process_negatively_regulates_process'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'process_positively_regulates_process'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'process_regulates_process'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'produces'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'regulates'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'related_to'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'temporally_related_to'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'similar_to'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'is_output_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'affects_response_to'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'composed_primarily_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'disrupts'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'affected_by'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_substrate'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'ameliorates'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'treats'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'is_substrate_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_capability'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_catalyst'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'is_input_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'decreases_response_to'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'associated_with'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'catalyzes'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_metabolite'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'genetically_interacts_with'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_upstream_actor'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'exacerbates'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'has_active_ingredient'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'increases_response_to'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'primarily_composed_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'manifestation_of'} -> BiologicalProcessOrActivity\n",
      "NamedThing - {'coexists_with'} -> BiologicalProcessOrActivity\n",
      "BiologicalProcessOrActivity - {'occurs_in'} -> Publication\n",
      "BiologicalProcessOrActivity - {'affects'} -> Publication\n",
      "BiologicalProcessOrActivity - {'has_participant'} -> Publication\n",
      "BiologicalProcessOrActivity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->MicroRNA->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> MicroRNA\n",
      "NamedThing - {'related_to'} -> MicroRNA\n",
      "NamedThing - {'regulates'} -> MicroRNA\n",
      "NamedThing - {'has_input'} -> MicroRNA\n",
      "NamedThing - {'is_sequence_variant_of'} -> MicroRNA\n",
      "NamedThing - {'associated_with'} -> MicroRNA\n",
      "NamedThing - {'has_phenotype'} -> MicroRNA\n",
      "NamedThing - {'located_in'} -> MicroRNA\n",
      "NamedThing - {'produces'} -> MicroRNA\n",
      "NamedThing - {'has_part'} -> MicroRNA\n",
      "NamedThing - {'interacts_with'} -> MicroRNA\n",
      "NamedThing - {'occurs_in'} -> MicroRNA\n",
      "NamedThing - {'affects'} -> MicroRNA\n",
      "NamedThing - {'coexists_with'} -> MicroRNA\n",
      "NamedThing - {'close_match'} -> MicroRNA\n",
      "NamedThing - {'has_participant'} -> MicroRNA\n",
      "MicroRNA - {'causes'} -> Publication\n",
      "MicroRNA - {'disrupts'} -> Publication\n",
      "MicroRNA - {'associated_with'} -> Publication\n",
      "MicroRNA - {'gene_associated_with_condition'} -> Publication\n",
      "MicroRNA - {'subclass_of'} -> Publication\n",
      "MicroRNA - {'predisposes'} -> Publication\n",
      "MicroRNA - {'located_in'} -> Publication\n",
      "MicroRNA - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->PathologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> PathologicalProcess\n",
      "NamedThing - {'related_to'} -> PathologicalProcess\n",
      "NamedThing - {'disrupts'} -> PathologicalProcess\n",
      "NamedThing - {'affected_by'} -> PathologicalProcess\n",
      "NamedThing - {'treats'} -> PathologicalProcess\n",
      "NamedThing - {'decreases_response_to'} -> PathologicalProcess\n",
      "NamedThing - {'ameliorates'} -> PathologicalProcess\n",
      "NamedThing - {'regulates'} -> PathologicalProcess\n",
      "NamedThing - {'associated_with'} -> PathologicalProcess\n",
      "NamedThing - {'affects'} -> PathologicalProcess\n",
      "NamedThing - {'affects_response_to'} -> PathologicalProcess\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> PathologicalProcess\n",
      "NamedThing - {'exacerbates'} -> PathologicalProcess\n",
      "NamedThing - {'increases_response_to'} -> PathologicalProcess\n",
      "NamedThing - {'actively_involved_in'} -> PathologicalProcess\n",
      "NamedThing - {'correlated_with'} -> PathologicalProcess\n",
      "NamedThing - {'gene_associated_with_condition'} -> PathologicalProcess\n",
      "NamedThing - {'has_phenotype'} -> PathologicalProcess\n",
      "NamedThing - {'located_in'} -> PathologicalProcess\n",
      "NamedThing - {'biomarker_for'} -> PathologicalProcess\n",
      "NamedThing - {'interacts_with'} -> PathologicalProcess\n",
      "NamedThing - {'produces'} -> PathologicalProcess\n",
      "NamedThing - {'has_part'} -> PathologicalProcess\n",
      "NamedThing - {'prevents'} -> PathologicalProcess\n",
      "NamedThing - {'diagnoses'} -> PathologicalProcess\n",
      "NamedThing - {'enables'} -> PathologicalProcess\n",
      "NamedThing - {'contributes_to'} -> PathologicalProcess\n",
      "NamedThing - {'predisposes'} -> PathologicalProcess\n",
      "NamedThing - {'precedes'} -> PathologicalProcess\n",
      "NamedThing - {'occurs_in'} -> PathologicalProcess\n",
      "NamedThing - {'manifestation_of'} -> PathologicalProcess\n",
      "NamedThing - {'coexists_with'} -> PathologicalProcess\n",
      "NamedThing - {'close_match'} -> PathologicalProcess\n",
      "NamedThing - {'has_participant'} -> PathologicalProcess\n",
      "NamedThing - {'contraindicated_for'} -> PathologicalProcess\n",
      "NamedThing - {'chemically_similar_to'} -> PathologicalProcess\n",
      "NamedThing - {'physically_interacts_with'} -> PathologicalProcess\n",
      "NamedThing - {'has_input'} -> PathologicalProcess\n",
      "NamedThing - {'derives_from'} -> PathologicalProcess\n",
      "NamedThing - {'causes'} -> PathologicalProcess\n",
      "PathologicalProcess - {'manifestation_of'} -> Publication\n",
      "PathologicalProcess - {'occurs_in'} -> Publication\n",
      "PathologicalProcess - {'coexists_with'} -> Publication\n",
      "PathologicalProcess - {'exacerbates'} -> Publication\n",
      "PathologicalProcess - {'disrupts'} -> Publication\n",
      "PathologicalProcess - {'located_in'} -> Publication\n",
      "PathologicalProcess - {'regulates'} -> Publication\n",
      "PathologicalProcess - {'treats'} -> Publication\n",
      "PathologicalProcess - {'precedes'} -> Publication\n",
      "PathologicalProcess - {'causes'} -> Publication\n",
      "PathologicalProcess - {'close_match'} -> Publication\n",
      "PathologicalProcess - {'associated_with'} -> Publication\n",
      "PathologicalProcess - {'subclass_of'} -> Publication\n",
      "PathologicalProcess - {'produces'} -> Publication\n",
      "PathologicalProcess - {'predisposes'} -> Publication\n",
      "PathologicalProcess - {'related_to'} -> Publication\n",
      "PathologicalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->RNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> RNAProduct\n",
      "NamedThing - {'affects'} -> RNAProduct\n",
      "NamedThing - {'affects_transport_of'} -> RNAProduct\n",
      "NamedThing - {'has_input'} -> RNAProduct\n",
      "NamedThing - {'has_output'} -> RNAProduct\n",
      "NamedThing - {'has_participant'} -> RNAProduct\n",
      "NamedThing - {'related_to'} -> RNAProduct\n",
      "NamedThing - {'actively_involves'} -> RNAProduct\n",
      "NamedThing - {'has_upstream_actor'} -> RNAProduct\n",
      "NamedThing - {'exacerbates'} -> RNAProduct\n",
      "NamedThing - {'increases_response_to'} -> RNAProduct\n",
      "NamedThing - {'primarily_composed_of'} -> RNAProduct\n",
      "NamedThing - {'similar_to'} -> RNAProduct\n",
      "NamedThing - {'overlaps'} -> RNAProduct\n",
      "NamedThing - {'affects_response_to'} -> RNAProduct\n",
      "NamedThing - {'composed_primarily_of'} -> RNAProduct\n",
      "NamedThing - {'disrupts'} -> RNAProduct\n",
      "NamedThing - {'affected_by'} -> RNAProduct\n",
      "NamedThing - {'has_substrate'} -> RNAProduct\n",
      "NamedThing - {'interacts_with'} -> RNAProduct\n",
      "NamedThing - {'ameliorates'} -> RNAProduct\n",
      "NamedThing - {'treats'} -> RNAProduct\n",
      "NamedThing - {'regulates'} -> RNAProduct\n",
      "NamedThing - {'has_capability'} -> RNAProduct\n",
      "NamedThing - {'participates_in'} -> RNAProduct\n",
      "NamedThing - {'enabled_by'} -> RNAProduct\n",
      "NamedThing - {'has_catalyst'} -> RNAProduct\n",
      "NamedThing - {'decreases_response_to'} -> RNAProduct\n",
      "NamedThing - {'associated_with'} -> RNAProduct\n",
      "NamedThing - {'is_sequence_variant_of'} -> RNAProduct\n",
      "NamedThing - {'has_phenotype'} -> RNAProduct\n",
      "NamedThing - {'located_in'} -> RNAProduct\n",
      "NamedThing - {'has_part'} -> RNAProduct\n",
      "NamedThing - {'produces'} -> RNAProduct\n",
      "NamedThing - {'occurs_in'} -> RNAProduct\n",
      "NamedThing - {'coexists_with'} -> RNAProduct\n",
      "NamedThing - {'close_match'} -> RNAProduct\n",
      "NamedThing - {'derives_from'} -> RNAProduct\n",
      "NamedThing - {'physically_interacts_with'} -> RNAProduct\n",
      "RNAProduct - {'located_in'} -> Publication\n",
      "RNAProduct - {'regulates'} -> Publication\n",
      "RNAProduct - {'related_to'} -> Publication\n",
      "RNAProduct - {'affects'} -> Publication\n",
      "RNAProduct - {'subclass_of'} -> Publication\n",
      "RNAProduct - {'disrupts'} -> Publication\n",
      "RNAProduct - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Exon->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Exon\n",
      "NamedThing - {'related_to'} -> Exon\n",
      "NamedThing - {'has_part'} -> Exon\n",
      "Exon - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->NucleicAcidEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'actively_involved_in'} -> NucleicAcidEntity\n",
      "NamedThing - {'affects_activity_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'affects'} -> NucleicAcidEntity\n",
      "NamedThing - {'affects_transport_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'capable_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'caused_by'} -> NucleicAcidEntity\n",
      "NamedThing - {'enabled_by'} -> NucleicAcidEntity\n",
      "NamedThing - {'enables'} -> NucleicAcidEntity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> NucleicAcidEntity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> NucleicAcidEntity\n",
      "NamedThing - {'entity_regulates_entity'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_input'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_output'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_participant'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_part'} -> NucleicAcidEntity\n",
      "NamedThing - {'interacts_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'location_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'molecularly_interacts_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'overlaps'} -> NucleicAcidEntity\n",
      "NamedThing - {'participates_in'} -> NucleicAcidEntity\n",
      "NamedThing - {'physically_interacts_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'preceded_by'} -> NucleicAcidEntity\n",
      "NamedThing - {'precedes'} -> NucleicAcidEntity\n",
      "NamedThing - {'related_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'temporally_related_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'expresses'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_substrate'} -> NucleicAcidEntity\n",
      "NamedThing - {'ameliorates'} -> NucleicAcidEntity\n",
      "NamedThing - {'genetically_interacts_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'actively_involves'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_active_ingredient'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_catalyst'} -> NucleicAcidEntity\n",
      "NamedThing - {'primarily_composed_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'similar_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'catalyzes'} -> NucleicAcidEntity\n",
      "NamedThing - {'is_output_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'affects_response_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'composed_primarily_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_upstream_actor'} -> NucleicAcidEntity\n",
      "NamedThing - {'exacerbates'} -> NucleicAcidEntity\n",
      "NamedThing - {'increases_response_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'disrupts'} -> NucleicAcidEntity\n",
      "NamedThing - {'affected_by'} -> NucleicAcidEntity\n",
      "NamedThing - {'treats'} -> NucleicAcidEntity\n",
      "NamedThing - {'regulates'} -> NucleicAcidEntity\n",
      "NamedThing - {'is_substrate_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'part_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_capability'} -> NucleicAcidEntity\n",
      "NamedThing - {'is_input_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'decreases_response_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'associated_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_metabolite'} -> NucleicAcidEntity\n",
      "NamedThing - {'chemically_similar_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'derives_from'} -> NucleicAcidEntity\n",
      "NamedThing - {'causes'} -> NucleicAcidEntity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'gene_product_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'is_sequence_variant_of'} -> NucleicAcidEntity\n",
      "NamedThing - {'gene_associated_with_condition'} -> NucleicAcidEntity\n",
      "NamedThing - {'has_phenotype'} -> NucleicAcidEntity\n",
      "NamedThing - {'located_in'} -> NucleicAcidEntity\n",
      "NamedThing - {'biomarker_for'} -> NucleicAcidEntity\n",
      "NamedThing - {'prevents'} -> NucleicAcidEntity\n",
      "NamedThing - {'colocalizes_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'contributes_to'} -> NucleicAcidEntity\n",
      "NamedThing - {'produces'} -> NucleicAcidEntity\n",
      "NamedThing - {'predisposes'} -> NucleicAcidEntity\n",
      "NamedThing - {'diagnoses'} -> NucleicAcidEntity\n",
      "NamedThing - {'occurs_in'} -> NucleicAcidEntity\n",
      "NamedThing - {'coexists_with'} -> NucleicAcidEntity\n",
      "NamedThing - {'close_match'} -> NucleicAcidEntity\n",
      "NamedThing - {'manifestation_of'} -> NucleicAcidEntity\n",
      "NucleicAcidEntity - {'gene_associated_with_condition'} -> Publication\n",
      "NucleicAcidEntity - {'located_in'} -> Publication\n",
      "NucleicAcidEntity - {'related_to'} -> Publication\n",
      "NucleicAcidEntity - {'close_match'} -> Publication\n",
      "NucleicAcidEntity - {'subclass_of'} -> Publication\n",
      "NucleicAcidEntity - {'interacts_with'} -> Publication\n",
      "NucleicAcidEntity - {'occurs_in'} -> Publication\n",
      "NucleicAcidEntity - {'affects'} -> Publication\n",
      "NucleicAcidEntity - {'treats'} -> Publication\n",
      "NucleicAcidEntity - {'causes'} -> Publication\n",
      "NucleicAcidEntity - {'coexists_with'} -> Publication\n",
      "NucleicAcidEntity - {'disrupts'} -> Publication\n",
      "NucleicAcidEntity - {'associated_with'} -> Publication\n",
      "NucleicAcidEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->LifeStage->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> LifeStage\n",
      "NamedThing - {'temporally_related_to'} -> LifeStage\n",
      "NamedThing - {'related_to'} -> LifeStage\n",
      "NamedThing - {'treats'} -> LifeStage\n",
      "NamedThing - {'located_in'} -> LifeStage\n",
      "NamedThing - {'interacts_with'} -> LifeStage\n",
      "NamedThing - {'produces'} -> LifeStage\n",
      "NamedThing - {'occurs_in'} -> LifeStage\n",
      "NamedThing - {'affects'} -> LifeStage\n",
      "NamedThing - {'close_match'} -> LifeStage\n",
      "LifeStage - {'subclass_of'} -> Publication\n",
      "LifeStage - {'has_part'} -> Publication\n",
      "LifeStage - {'interacts_with'} -> Publication\n",
      "LifeStage - {'produces'} -> Publication\n",
      "LifeStage - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->DiseaseOrPhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'actively_involved_in'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'actively_involves'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'affects'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'capable_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'has_part'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'negatively_regulates'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'overlaps'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'participates_in'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'part_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'positively_regulates'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'precedes'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'process_negatively_regulates_process'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'process_positively_regulates_process'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'process_regulates_process'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'regulates'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'related_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'temporally_related_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'has_upstream_actor'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'exacerbates'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'has_active_ingredient'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'increases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'primarily_composed_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'similar_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'is_output_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'affects_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'treats'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'composed_primarily_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'preceded_by'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'disrupts'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'is_input_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'affected_by'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'ameliorates'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'is_substrate_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'has_capability'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'has_participant'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'decreases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'enables'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'catalyzes'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'coexists_with'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'close_match'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'contraindicated_for'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'chemically_similar_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'physically_interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'has_input'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'derives_from'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'causes'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'disease_has_location'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'directly_physically_interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'binds'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'gene_product_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'is_sequence_variant_of'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'gene_associated_with_condition'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'has_phenotype'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'located_in'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'biomarker_for'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'produces'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'expressed_in'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'prevents'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'colocalizes_with'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'contributes_to'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'predisposes'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'diagnoses'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'occurs_in'} -> DiseaseOrPhenotypicFeature\n",
      "NamedThing - {'manifestation_of'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Gene->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Gene\n",
      "NamedThing - {'actively_involved_in'} -> Gene\n",
      "NamedThing - {'affects_activity_of'} -> Gene\n",
      "NamedThing - {'affects'} -> Gene\n",
      "NamedThing - {'affects_transport_of'} -> Gene\n",
      "NamedThing - {'capable_of'} -> Gene\n",
      "NamedThing - {'caused_by'} -> Gene\n",
      "NamedThing - {'enabled_by'} -> Gene\n",
      "NamedThing - {'enables'} -> Gene\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> Gene\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> Gene\n",
      "NamedThing - {'entity_regulates_entity'} -> Gene\n",
      "NamedThing - {'has_input'} -> Gene\n",
      "NamedThing - {'has_output'} -> Gene\n",
      "NamedThing - {'has_part'} -> Gene\n",
      "NamedThing - {'has_participant'} -> Gene\n",
      "NamedThing - {'interacts_with'} -> Gene\n",
      "NamedThing - {'location_of'} -> Gene\n",
      "NamedThing - {'molecularly_interacts_with'} -> Gene\n",
      "NamedThing - {'overlaps'} -> Gene\n",
      "NamedThing - {'participates_in'} -> Gene\n",
      "NamedThing - {'physically_interacts_with'} -> Gene\n",
      "NamedThing - {'preceded_by'} -> Gene\n",
      "NamedThing - {'precedes'} -> Gene\n",
      "NamedThing - {'related_to'} -> Gene\n",
      "NamedThing - {'temporally_related_to'} -> Gene\n",
      "NamedThing - {'similar_to'} -> Gene\n",
      "NamedThing - {'catalyzes'} -> Gene\n",
      "NamedThing - {'affects_response_to'} -> Gene\n",
      "NamedThing - {'composed_primarily_of'} -> Gene\n",
      "NamedThing - {'genetically_interacts_with'} -> Gene\n",
      "NamedThing - {'has_upstream_actor'} -> Gene\n",
      "NamedThing - {'exacerbates'} -> Gene\n",
      "NamedThing - {'has_active_ingredient'} -> Gene\n",
      "NamedThing - {'increases_response_to'} -> Gene\n",
      "NamedThing - {'primarily_composed_of'} -> Gene\n",
      "NamedThing - {'disrupts'} -> Gene\n",
      "NamedThing - {'is_output_of'} -> Gene\n",
      "NamedThing - {'affected_by'} -> Gene\n",
      "NamedThing - {'treats'} -> Gene\n",
      "NamedThing - {'part_of'} -> Gene\n",
      "NamedThing - {'positively_regulates'} -> Gene\n",
      "NamedThing - {'is_input_of'} -> Gene\n",
      "NamedThing - {'decreases_response_to'} -> Gene\n",
      "NamedThing - {'negatively_regulates'} -> Gene\n",
      "NamedThing - {'expresses'} -> Gene\n",
      "NamedThing - {'has_substrate'} -> Gene\n",
      "NamedThing - {'ameliorates'} -> Gene\n",
      "NamedThing - {'regulates'} -> Gene\n",
      "NamedThing - {'is_substrate_of'} -> Gene\n",
      "NamedThing - {'has_capability'} -> Gene\n",
      "NamedThing - {'actively_involves'} -> Gene\n",
      "NamedThing - {'has_catalyst'} -> Gene\n",
      "NamedThing - {'associated_with'} -> Gene\n",
      "NamedThing - {'has_phenotype'} -> Gene\n",
      "NamedThing - {'binds'} -> Gene\n",
      "NamedThing - {'is_sequence_variant_of'} -> Gene\n",
      "NamedThing - {'transcribed_from'} -> Gene\n",
      "NamedThing - {'gene_associated_with_condition'} -> Gene\n",
      "NamedThing - {'located_in'} -> Gene\n",
      "NamedThing - {'lacks_part'} -> Gene\n",
      "NamedThing - {'biomarker_for'} -> Gene\n",
      "NamedThing - {'produces'} -> Gene\n",
      "NamedThing - {'expressed_in'} -> Gene\n",
      "NamedThing - {'prevents'} -> Gene\n",
      "NamedThing - {'diagnoses'} -> Gene\n",
      "NamedThing - {'colocalizes_with'} -> Gene\n",
      "NamedThing - {'contributes_to'} -> Gene\n",
      "NamedThing - {'predisposes'} -> Gene\n",
      "NamedThing - {'occurs_in'} -> Gene\n",
      "NamedThing - {'manifestation_of'} -> Gene\n",
      "NamedThing - {'coexists_with'} -> Gene\n",
      "NamedThing - {'close_match'} -> Gene\n",
      "NamedThing - {'contraindicated_for'} -> Gene\n",
      "NamedThing - {'chemically_similar_to'} -> Gene\n",
      "NamedThing - {'derives_from'} -> Gene\n",
      "NamedThing - {'causes'} -> Gene\n",
      "NamedThing - {'disease_has_location'} -> Gene\n",
      "NamedThing - {'gene_product_of'} -> Gene\n",
      "NamedThing - {'directly_physically_interacts_with'} -> Gene\n",
      "NamedThing - {'same_as'} -> Gene\n",
      "Gene - {'correlated_with'} -> Publication\n",
      "Gene - {'prevents'} -> Publication\n",
      "Gene - {'treats'} -> Publication\n",
      "Gene - {'has_input'} -> Publication\n",
      "Gene - {'related_to'} -> Publication\n",
      "Gene - {'occurs_in'} -> Publication\n",
      "Gene - {'affects'} -> Publication\n",
      "Gene - {'exacerbates'} -> Publication\n",
      "Gene - {'coexists_with'} -> Publication\n",
      "Gene - {'disrupts'} -> Publication\n",
      "Gene - {'gene_associated_with_condition'} -> Publication\n",
      "Gene - {'located_in'} -> Publication\n",
      "Gene - {'regulates'} -> Publication\n",
      "Gene - {'biomarker_for'} -> Publication\n",
      "Gene - {'expressed_in'} -> Publication\n",
      "Gene - {'physically_interacts_with'} -> Publication\n",
      "Gene - {'predisposes'} -> Publication\n",
      "Gene - {'causes'} -> Publication\n",
      "Gene - {'precedes'} -> Publication\n",
      "Gene - {'gene_product_of'} -> Publication\n",
      "Gene - {'close_match'} -> Publication\n",
      "Gene - {'associated_with'} -> Publication\n",
      "Gene - {'subclass_of'} -> Publication\n",
      "Gene - {'has_part'} -> Publication\n",
      "Gene - {'colocalizes_with'} -> Publication\n",
      "Gene - {'interacts_with'} -> Publication\n",
      "Gene - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->MaterialSample->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> MaterialSample\n",
      "NamedThing - {'related_to'} -> MaterialSample\n",
      "NamedThing - {'occurs_in'} -> MaterialSample\n",
      "NamedThing - {'affects'} -> MaterialSample\n",
      "NamedThing - {'manifestation_of'} -> MaterialSample\n",
      "NamedThing - {'exacerbates'} -> MaterialSample\n",
      "NamedThing - {'coexists_with'} -> MaterialSample\n",
      "NamedThing - {'close_match'} -> MaterialSample\n",
      "NamedThing - {'has_participant'} -> MaterialSample\n",
      "NamedThing - {'regulates'} -> MaterialSample\n",
      "NamedThing - {'treats'} -> MaterialSample\n",
      "NamedThing - {'has_input'} -> MaterialSample\n",
      "NamedThing - {'derives_from'} -> MaterialSample\n",
      "NamedThing - {'causes'} -> MaterialSample\n",
      "NamedThing - {'has_output'} -> MaterialSample\n",
      "NamedThing - {'disrupts'} -> MaterialSample\n",
      "NamedThing - {'associated_with'} -> MaterialSample\n",
      "NamedThing - {'gene_associated_with_condition'} -> MaterialSample\n",
      "NamedThing - {'located_in'} -> MaterialSample\n",
      "NamedThing - {'interacts_with'} -> MaterialSample\n",
      "NamedThing - {'produces'} -> MaterialSample\n",
      "NamedThing - {'has_part'} -> MaterialSample\n",
      "NamedThing - {'diagnoses'} -> MaterialSample\n",
      "NamedThing - {'predisposes'} -> MaterialSample\n",
      "NamedThing - {'precedes'} -> MaterialSample\n",
      "MaterialSample - {'located_in'} -> Publication\n",
      "MaterialSample - {'associated_with'} -> Publication\n",
      "MaterialSample - {'subclass_of'} -> Publication\n",
      "MaterialSample - {'has_part'} -> Publication\n",
      "MaterialSample - {'related_to'} -> Publication\n",
      "MaterialSample - {'occurs_in'} -> Publication\n",
      "MaterialSample - {'affects'} -> Publication\n",
      "MaterialSample - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->PopulationOfIndividualOrganisms->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'related_to'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'associated_with'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'located_in'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'biomarker_for'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'produces'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'has_part'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'prevents'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'interacts_with'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'predisposes'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'diagnoses'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'occurs_in'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'affects'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'coexists_with'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'close_match'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'has_participant'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'regulates'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'treats'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'has_input'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'causes'} -> PopulationOfIndividualOrganisms\n",
      "NamedThing - {'physically_interacts_with'} -> PopulationOfIndividualOrganisms\n",
      "PopulationOfIndividualOrganisms - {'correlated_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_part'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'produces'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'related_to'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'occurs_in'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'affects'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'close_match'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'subclass_of'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'interacts_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'treats'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_input'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Publication\n",
      "NamedThing - {'related_to'} -> Publication\n",
      "NamedThing - {'is_sequence_variant_of'} -> Publication\n",
      "NamedThing - {'associated_with'} -> Publication\n",
      "NamedThing - {'has_phenotype'} -> Publication\n",
      "NamedThing - {'disrupts'} -> Publication\n",
      "NamedThing - {'biomarker_for'} -> Publication\n",
      "NamedThing - {'correlated_with'} -> Publication\n",
      "NamedThing - {'gene_associated_with_condition'} -> Publication\n",
      "NamedThing - {'has_part'} -> Publication\n",
      "NamedThing - {'located_in'} -> Publication\n",
      "NamedThing - {'colocalizes_with'} -> Publication\n",
      "NamedThing - {'interacts_with'} -> Publication\n",
      "NamedThing - {'produces'} -> Publication\n",
      "NamedThing - {'prevents'} -> Publication\n",
      "NamedThing - {'diagnoses'} -> Publication\n",
      "NamedThing - {'predisposes'} -> Publication\n",
      "NamedThing - {'precedes'} -> Publication\n",
      "NamedThing - {'occurs_in'} -> Publication\n",
      "NamedThing - {'affects'} -> Publication\n",
      "NamedThing - {'close_match'} -> Publication\n",
      "NamedThing - {'manifestation_of'} -> Publication\n",
      "NamedThing - {'exacerbates'} -> Publication\n",
      "NamedThing - {'coexists_with'} -> Publication\n",
      "NamedThing - {'regulates'} -> Publication\n",
      "NamedThing - {'has_input'} -> Publication\n",
      "NamedThing - {'physically_interacts_with'} -> Publication\n",
      "NamedThing - {'treats'} -> Publication\n",
      "NamedThing - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->MolecularEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> MolecularEntity\n",
      "NamedThing - {'actively_involved_in'} -> MolecularEntity\n",
      "NamedThing - {'affects_activity_of'} -> MolecularEntity\n",
      "NamedThing - {'affects'} -> MolecularEntity\n",
      "NamedThing - {'affects_transport_of'} -> MolecularEntity\n",
      "NamedThing - {'capable_of'} -> MolecularEntity\n",
      "NamedThing - {'caused_by'} -> MolecularEntity\n",
      "NamedThing - {'enabled_by'} -> MolecularEntity\n",
      "NamedThing - {'enables'} -> MolecularEntity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> MolecularEntity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> MolecularEntity\n",
      "NamedThing - {'entity_regulates_entity'} -> MolecularEntity\n",
      "NamedThing - {'has_input'} -> MolecularEntity\n",
      "NamedThing - {'has_output'} -> MolecularEntity\n",
      "NamedThing - {'has_participant'} -> MolecularEntity\n",
      "NamedThing - {'has_part'} -> MolecularEntity\n",
      "NamedThing - {'interacts_with'} -> MolecularEntity\n",
      "NamedThing - {'location_of'} -> MolecularEntity\n",
      "NamedThing - {'molecularly_interacts_with'} -> MolecularEntity\n",
      "NamedThing - {'overlaps'} -> MolecularEntity\n",
      "NamedThing - {'participates_in'} -> MolecularEntity\n",
      "NamedThing - {'physically_interacts_with'} -> MolecularEntity\n",
      "NamedThing - {'preceded_by'} -> MolecularEntity\n",
      "NamedThing - {'precedes'} -> MolecularEntity\n",
      "NamedThing - {'related_to'} -> MolecularEntity\n",
      "NamedThing - {'temporally_related_to'} -> MolecularEntity\n",
      "NamedThing - {'affects_response_to'} -> MolecularEntity\n",
      "NamedThing - {'composed_primarily_of'} -> MolecularEntity\n",
      "NamedThing - {'genetically_interacts_with'} -> MolecularEntity\n",
      "NamedThing - {'has_upstream_actor'} -> MolecularEntity\n",
      "NamedThing - {'exacerbates'} -> MolecularEntity\n",
      "NamedThing - {'has_active_ingredient'} -> MolecularEntity\n",
      "NamedThing - {'increases_response_to'} -> MolecularEntity\n",
      "NamedThing - {'primarily_composed_of'} -> MolecularEntity\n",
      "NamedThing - {'disrupts'} -> MolecularEntity\n",
      "NamedThing - {'is_output_of'} -> MolecularEntity\n",
      "NamedThing - {'affected_by'} -> MolecularEntity\n",
      "NamedThing - {'treats'} -> MolecularEntity\n",
      "NamedThing - {'is_input_of'} -> MolecularEntity\n",
      "NamedThing - {'decreases_response_to'} -> MolecularEntity\n",
      "NamedThing - {'expresses'} -> MolecularEntity\n",
      "NamedThing - {'has_substrate'} -> MolecularEntity\n",
      "NamedThing - {'ameliorates'} -> MolecularEntity\n",
      "NamedThing - {'regulates'} -> MolecularEntity\n",
      "NamedThing - {'is_substrate_of'} -> MolecularEntity\n",
      "NamedThing - {'part_of'} -> MolecularEntity\n",
      "NamedThing - {'has_capability'} -> MolecularEntity\n",
      "NamedThing - {'actively_involves'} -> MolecularEntity\n",
      "NamedThing - {'has_catalyst'} -> MolecularEntity\n",
      "NamedThing - {'associated_with'} -> MolecularEntity\n",
      "NamedThing - {'similar_to'} -> MolecularEntity\n",
      "NamedThing - {'catalyzes'} -> MolecularEntity\n",
      "NamedThing - {'derives_from'} -> MolecularEntity\n",
      "NamedThing - {'causes'} -> MolecularEntity\n",
      "NamedThing - {'disease_has_location'} -> MolecularEntity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> MolecularEntity\n",
      "NamedThing - {'binds'} -> MolecularEntity\n",
      "NamedThing - {'gene_product_of'} -> MolecularEntity\n",
      "NamedThing - {'is_sequence_variant_of'} -> MolecularEntity\n",
      "NamedThing - {'gene_associated_with_condition'} -> MolecularEntity\n",
      "NamedThing - {'has_phenotype'} -> MolecularEntity\n",
      "NamedThing - {'located_in'} -> MolecularEntity\n",
      "NamedThing - {'biomarker_for'} -> MolecularEntity\n",
      "NamedThing - {'produces'} -> MolecularEntity\n",
      "NamedThing - {'colocalizes_with'} -> MolecularEntity\n",
      "NamedThing - {'contributes_to'} -> MolecularEntity\n",
      "NamedThing - {'diagnoses'} -> MolecularEntity\n",
      "NamedThing - {'occurs_in'} -> MolecularEntity\n",
      "NamedThing - {'same_as'} -> MolecularEntity\n",
      "NamedThing - {'coexists_with'} -> MolecularEntity\n",
      "NamedThing - {'close_match'} -> MolecularEntity\n",
      "NamedThing - {'manifestation_of'} -> MolecularEntity\n",
      "NamedThing - {'has_metabolite'} -> MolecularEntity\n",
      "NamedThing - {'chemically_similar_to'} -> MolecularEntity\n",
      "MolecularEntity - {'causes'} -> Publication\n",
      "MolecularEntity - {'associated_with'} -> Publication\n",
      "MolecularEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "MolecularEntity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularEntity - {'regulates'} -> Publication\n",
      "MolecularEntity - {'disrupts'} -> Publication\n",
      "MolecularEntity - {'located_in'} -> Publication\n",
      "MolecularEntity - {'biomarker_for'} -> Publication\n",
      "MolecularEntity - {'interacts_with'} -> Publication\n",
      "MolecularEntity - {'predisposes'} -> Publication\n",
      "MolecularEntity - {'related_to'} -> Publication\n",
      "MolecularEntity - {'has_part'} -> Publication\n",
      "MolecularEntity - {'occurs_in'} -> Publication\n",
      "MolecularEntity - {'affects'} -> Publication\n",
      "MolecularEntity - {'precedes'} -> Publication\n",
      "MolecularEntity - {'coexists_with'} -> Publication\n",
      "MolecularEntity - {'close_match'} -> Publication\n",
      "MolecularEntity - {'subclass_of'} -> Publication\n",
      "MolecularEntity - {'treats'} -> Publication\n",
      "MolecularEntity - {'physically_interacts_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->OrganismalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> OrganismalEntity\n",
      "NamedThing - {'active_in'} -> OrganismalEntity\n",
      "NamedThing - {'actively_involved_in'} -> OrganismalEntity\n",
      "NamedThing - {'actively_involves'} -> OrganismalEntity\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> OrganismalEntity\n",
      "NamedThing - {'acts_upstream_of'} -> OrganismalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> OrganismalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> OrganismalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> OrganismalEntity\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> OrganismalEntity\n",
      "NamedThing - {'affects_activity_of'} -> OrganismalEntity\n",
      "NamedThing - {'affects'} -> OrganismalEntity\n",
      "NamedThing - {'affects_transport_of'} -> OrganismalEntity\n",
      "NamedThing - {'capable_of'} -> OrganismalEntity\n",
      "NamedThing - {'caused_by'} -> OrganismalEntity\n",
      "NamedThing - {'causes'} -> OrganismalEntity\n",
      "NamedThing - {'coexists_with'} -> OrganismalEntity\n",
      "NamedThing - {'contributes_to'} -> OrganismalEntity\n",
      "NamedThing - {'derives_from'} -> OrganismalEntity\n",
      "NamedThing - {'derives_into'} -> OrganismalEntity\n",
      "NamedThing - {'enabled_by'} -> OrganismalEntity\n",
      "NamedThing - {'enables'} -> OrganismalEntity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> OrganismalEntity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> OrganismalEntity\n",
      "NamedThing - {'entity_regulates_entity'} -> OrganismalEntity\n",
      "NamedThing - {'has_input'} -> OrganismalEntity\n",
      "NamedThing - {'has_output'} -> OrganismalEntity\n",
      "NamedThing - {'has_participant'} -> OrganismalEntity\n",
      "NamedThing - {'has_part'} -> OrganismalEntity\n",
      "NamedThing - {'interacts_with'} -> OrganismalEntity\n",
      "NamedThing - {'located_in'} -> OrganismalEntity\n",
      "NamedThing - {'location_of'} -> OrganismalEntity\n",
      "NamedThing - {'molecularly_interacts_with'} -> OrganismalEntity\n",
      "NamedThing - {'negatively_regulates'} -> OrganismalEntity\n",
      "NamedThing - {'occurs_in'} -> OrganismalEntity\n",
      "NamedThing - {'overlaps'} -> OrganismalEntity\n",
      "NamedThing - {'participates_in'} -> OrganismalEntity\n",
      "NamedThing - {'part_of'} -> OrganismalEntity\n",
      "NamedThing - {'physically_interacts_with'} -> OrganismalEntity\n",
      "NamedThing - {'positively_regulates'} -> OrganismalEntity\n",
      "NamedThing - {'preceded_by'} -> OrganismalEntity\n",
      "NamedThing - {'precedes'} -> OrganismalEntity\n",
      "NamedThing - {'process_negatively_regulates_process'} -> OrganismalEntity\n",
      "NamedThing - {'process_positively_regulates_process'} -> OrganismalEntity\n",
      "NamedThing - {'process_regulates_process'} -> OrganismalEntity\n",
      "NamedThing - {'produced_by'} -> OrganismalEntity\n",
      "NamedThing - {'regulates'} -> OrganismalEntity\n",
      "NamedThing - {'related_to'} -> OrganismalEntity\n",
      "NamedThing - {'temporally_related_to'} -> OrganismalEntity\n",
      "NamedThing - {'is_input_of'} -> OrganismalEntity\n",
      "NamedThing - {'decreases_response_to'} -> OrganismalEntity\n",
      "NamedThing - {'expresses'} -> OrganismalEntity\n",
      "NamedThing - {'has_substrate'} -> OrganismalEntity\n",
      "NamedThing - {'ameliorates'} -> OrganismalEntity\n",
      "NamedThing - {'is_substrate_of'} -> OrganismalEntity\n",
      "NamedThing - {'has_capability'} -> OrganismalEntity\n",
      "NamedThing - {'has_catalyst'} -> OrganismalEntity\n",
      "NamedThing - {'expressed_in'} -> OrganismalEntity\n",
      "NamedThing - {'associated_with'} -> OrganismalEntity\n",
      "NamedThing - {'catalyzes'} -> OrganismalEntity\n",
      "NamedThing - {'affects_response_to'} -> OrganismalEntity\n",
      "NamedThing - {'has_metabolite'} -> OrganismalEntity\n",
      "NamedThing - {'composed_primarily_of'} -> OrganismalEntity\n",
      "NamedThing - {'genetically_interacts_with'} -> OrganismalEntity\n",
      "NamedThing - {'has_upstream_actor'} -> OrganismalEntity\n",
      "NamedThing - {'exacerbates'} -> OrganismalEntity\n",
      "NamedThing - {'has_active_ingredient'} -> OrganismalEntity\n",
      "NamedThing - {'increases_response_to'} -> OrganismalEntity\n",
      "NamedThing - {'primarily_composed_of'} -> OrganismalEntity\n",
      "NamedThing - {'disrupts'} -> OrganismalEntity\n",
      "NamedThing - {'similar_to'} -> OrganismalEntity\n",
      "NamedThing - {'is_output_of'} -> OrganismalEntity\n",
      "NamedThing - {'affected_by'} -> OrganismalEntity\n",
      "NamedThing - {'treats'} -> OrganismalEntity\n",
      "OrganismalEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->BiologicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> BiologicalEntity\n",
      "NamedThing - {'active_in'} -> BiologicalEntity\n",
      "NamedThing - {'actively_involved_in'} -> BiologicalEntity\n",
      "NamedThing - {'actively_involves'} -> BiologicalEntity\n",
      "NamedThing - {'acts_upstream_of'} -> BiologicalEntity\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> BiologicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> BiologicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> BiologicalEntity\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> BiologicalEntity\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> BiologicalEntity\n",
      "NamedThing - {'affects_activity_of'} -> BiologicalEntity\n",
      "NamedThing - {'affects'} -> BiologicalEntity\n",
      "NamedThing - {'affects_transport_of'} -> BiologicalEntity\n",
      "NamedThing - {'capable_of'} -> BiologicalEntity\n",
      "NamedThing - {'caused_by'} -> BiologicalEntity\n",
      "NamedThing - {'causes'} -> BiologicalEntity\n",
      "NamedThing - {'coexists_with'} -> BiologicalEntity\n",
      "NamedThing - {'contributes_to'} -> BiologicalEntity\n",
      "NamedThing - {'derives_from'} -> BiologicalEntity\n",
      "NamedThing - {'derives_into'} -> BiologicalEntity\n",
      "NamedThing - {'enabled_by'} -> BiologicalEntity\n",
      "NamedThing - {'enables'} -> BiologicalEntity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> BiologicalEntity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> BiologicalEntity\n",
      "NamedThing - {'entity_regulates_entity'} -> BiologicalEntity\n",
      "NamedThing - {'has_input'} -> BiologicalEntity\n",
      "NamedThing - {'has_output'} -> BiologicalEntity\n",
      "NamedThing - {'has_part'} -> BiologicalEntity\n",
      "NamedThing - {'has_participant'} -> BiologicalEntity\n",
      "NamedThing - {'interacts_with'} -> BiologicalEntity\n",
      "NamedThing - {'located_in'} -> BiologicalEntity\n",
      "NamedThing - {'location_of'} -> BiologicalEntity\n",
      "NamedThing - {'molecularly_interacts_with'} -> BiologicalEntity\n",
      "NamedThing - {'negatively_regulates'} -> BiologicalEntity\n",
      "NamedThing - {'occurs_in'} -> BiologicalEntity\n",
      "NamedThing - {'overlaps'} -> BiologicalEntity\n",
      "NamedThing - {'participates_in'} -> BiologicalEntity\n",
      "NamedThing - {'part_of'} -> BiologicalEntity\n",
      "NamedThing - {'physically_interacts_with'} -> BiologicalEntity\n",
      "NamedThing - {'positively_regulates'} -> BiologicalEntity\n",
      "NamedThing - {'preceded_by'} -> BiologicalEntity\n",
      "NamedThing - {'precedes'} -> BiologicalEntity\n",
      "NamedThing - {'process_negatively_regulates_process'} -> BiologicalEntity\n",
      "NamedThing - {'process_positively_regulates_process'} -> BiologicalEntity\n",
      "NamedThing - {'process_regulates_process'} -> BiologicalEntity\n",
      "NamedThing - {'produced_by'} -> BiologicalEntity\n",
      "NamedThing - {'produces'} -> BiologicalEntity\n",
      "NamedThing - {'regulates'} -> BiologicalEntity\n",
      "NamedThing - {'related_to'} -> BiologicalEntity\n",
      "NamedThing - {'temporally_related_to'} -> BiologicalEntity\n",
      "NamedThing - {'has_catalyst'} -> BiologicalEntity\n",
      "NamedThing - {'expressed_in'} -> BiologicalEntity\n",
      "NamedThing - {'decreases_response_to'} -> BiologicalEntity\n",
      "NamedThing - {'associated_with'} -> BiologicalEntity\n",
      "NamedThing - {'catalyzes'} -> BiologicalEntity\n",
      "NamedThing - {'affects_response_to'} -> BiologicalEntity\n",
      "NamedThing - {'has_metabolite'} -> BiologicalEntity\n",
      "NamedThing - {'composed_primarily_of'} -> BiologicalEntity\n",
      "NamedThing - {'genetically_interacts_with'} -> BiologicalEntity\n",
      "NamedThing - {'has_upstream_actor'} -> BiologicalEntity\n",
      "NamedThing - {'exacerbates'} -> BiologicalEntity\n",
      "NamedThing - {'has_active_ingredient'} -> BiologicalEntity\n",
      "NamedThing - {'increases_response_to'} -> BiologicalEntity\n",
      "NamedThing - {'primarily_composed_of'} -> BiologicalEntity\n",
      "NamedThing - {'disrupts'} -> BiologicalEntity\n",
      "NamedThing - {'similar_to'} -> BiologicalEntity\n",
      "NamedThing - {'is_output_of'} -> BiologicalEntity\n",
      "NamedThing - {'affected_by'} -> BiologicalEntity\n",
      "NamedThing - {'treats'} -> BiologicalEntity\n",
      "NamedThing - {'is_input_of'} -> BiologicalEntity\n",
      "NamedThing - {'expresses'} -> BiologicalEntity\n",
      "NamedThing - {'has_substrate'} -> BiologicalEntity\n",
      "NamedThing - {'ameliorates'} -> BiologicalEntity\n",
      "NamedThing - {'is_substrate_of'} -> BiologicalEntity\n",
      "NamedThing - {'has_capability'} -> BiologicalEntity\n",
      "NamedThing - {'in_taxon'} -> BiologicalEntity\n",
      "NamedThing - {'disease_has_location'} -> BiologicalEntity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> BiologicalEntity\n",
      "NamedThing - {'gene_product_of'} -> BiologicalEntity\n",
      "NamedThing - {'is_sequence_variant_of'} -> BiologicalEntity\n",
      "NamedThing - {'transcribed_from'} -> BiologicalEntity\n",
      "NamedThing - {'correlated_with'} -> BiologicalEntity\n",
      "NamedThing - {'gene_associated_with_condition'} -> BiologicalEntity\n",
      "NamedThing - {'has_phenotype'} -> BiologicalEntity\n",
      "NamedThing - {'binds'} -> BiologicalEntity\n",
      "NamedThing - {'biomarker_for'} -> BiologicalEntity\n",
      "NamedThing - {'colocalizes_with'} -> BiologicalEntity\n",
      "NamedThing - {'predisposes'} -> BiologicalEntity\n",
      "NamedThing - {'prevents'} -> BiologicalEntity\n",
      "NamedThing - {'diagnoses'} -> BiologicalEntity\n",
      "NamedThing - {'same_as'} -> BiologicalEntity\n",
      "NamedThing - {'close_match'} -> BiologicalEntity\n",
      "NamedThing - {'manifestation_of'} -> BiologicalEntity\n",
      "NamedThing - {'chemically_similar_to'} -> BiologicalEntity\n",
      "NamedThing - {'contraindicated_for'} -> BiologicalEntity\n",
      "BiologicalEntity - {'physically_interacts_with'} -> Publication\n",
      "BiologicalEntity - {'treats'} -> Publication\n",
      "BiologicalEntity - {'has_input'} -> Publication\n",
      "BiologicalEntity - {'causes'} -> Publication\n",
      "BiologicalEntity - {'disrupts'} -> Publication\n",
      "BiologicalEntity - {'is_sequence_variant_of'} -> Publication\n",
      "BiologicalEntity - {'has_phenotype'} -> Publication\n",
      "BiologicalEntity - {'biomarker_for'} -> Publication\n",
      "BiologicalEntity - {'associated_with'} -> Publication\n",
      "BiologicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalEntity - {'has_part'} -> Publication\n",
      "BiologicalEntity - {'located_in'} -> Publication\n",
      "BiologicalEntity - {'colocalizes_with'} -> Publication\n",
      "BiologicalEntity - {'interacts_with'} -> Publication\n",
      "BiologicalEntity - {'produces'} -> Publication\n",
      "BiologicalEntity - {'predisposes'} -> Publication\n",
      "BiologicalEntity - {'related_to'} -> Publication\n",
      "BiologicalEntity - {'prevents'} -> Publication\n",
      "BiologicalEntity - {'manifestation_of'} -> Publication\n",
      "BiologicalEntity - {'precedes'} -> Publication\n",
      "BiologicalEntity - {'occurs_in'} -> Publication\n",
      "BiologicalEntity - {'affects'} -> Publication\n",
      "BiologicalEntity - {'close_match'} -> Publication\n",
      "BiologicalEntity - {'exacerbates'} -> Publication\n",
      "BiologicalEntity - {'coexists_with'} -> Publication\n",
      "BiologicalEntity - {'has_participant'} -> Publication\n",
      "BiologicalEntity - {'subclass_of'} -> Publication\n",
      "BiologicalEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->OrganismAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> OrganismAttribute\n",
      "NamedThing - {'actively_involved_in'} -> OrganismAttribute\n",
      "NamedThing - {'actively_involves'} -> OrganismAttribute\n",
      "NamedThing - {'acts_upstream_of'} -> OrganismAttribute\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> OrganismAttribute\n",
      "NamedThing - {'affects'} -> OrganismAttribute\n",
      "NamedThing - {'capable_of'} -> OrganismAttribute\n",
      "NamedThing - {'has_attribute'} -> OrganismAttribute\n",
      "NamedThing - {'has_part'} -> OrganismAttribute\n",
      "NamedThing - {'negatively_regulates'} -> OrganismAttribute\n",
      "NamedThing - {'overlaps'} -> OrganismAttribute\n",
      "NamedThing - {'participates_in'} -> OrganismAttribute\n",
      "NamedThing - {'part_of'} -> OrganismAttribute\n",
      "NamedThing - {'positively_regulates'} -> OrganismAttribute\n",
      "NamedThing - {'precedes'} -> OrganismAttribute\n",
      "NamedThing - {'process_negatively_regulates_process'} -> OrganismAttribute\n",
      "NamedThing - {'process_positively_regulates_process'} -> OrganismAttribute\n",
      "NamedThing - {'process_regulates_process'} -> OrganismAttribute\n",
      "NamedThing - {'regulates'} -> OrganismAttribute\n",
      "NamedThing - {'related_to'} -> OrganismAttribute\n",
      "NamedThing - {'temporally_related_to'} -> OrganismAttribute\n",
      "NamedThing - {'treats'} -> OrganismAttribute\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> OrganismAttribute\n",
      "NamedThing - {'preceded_by'} -> OrganismAttribute\n",
      "NamedThing - {'has_participant'} -> OrganismAttribute\n",
      "NamedThing - {'is_input_of'} -> OrganismAttribute\n",
      "NamedThing - {'decreases_response_to'} -> OrganismAttribute\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> OrganismAttribute\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> OrganismAttribute\n",
      "NamedThing - {'interacts_with'} -> OrganismAttribute\n",
      "NamedThing - {'ameliorates'} -> OrganismAttribute\n",
      "NamedThing - {'is_substrate_of'} -> OrganismAttribute\n",
      "NamedThing - {'has_capability'} -> OrganismAttribute\n",
      "NamedThing - {'associated_with'} -> OrganismAttribute\n",
      "NamedThing - {'enables'} -> OrganismAttribute\n",
      "NamedThing - {'similar_to'} -> OrganismAttribute\n",
      "NamedThing - {'catalyzes'} -> OrganismAttribute\n",
      "NamedThing - {'affects_response_to'} -> OrganismAttribute\n",
      "NamedThing - {'composed_primarily_of'} -> OrganismAttribute\n",
      "NamedThing - {'has_upstream_actor'} -> OrganismAttribute\n",
      "NamedThing - {'exacerbates'} -> OrganismAttribute\n",
      "NamedThing - {'has_active_ingredient'} -> OrganismAttribute\n",
      "NamedThing - {'increases_response_to'} -> OrganismAttribute\n",
      "NamedThing - {'primarily_composed_of'} -> OrganismAttribute\n",
      "NamedThing - {'disrupts'} -> OrganismAttribute\n",
      "NamedThing - {'is_output_of'} -> OrganismAttribute\n",
      "NamedThing - {'affected_by'} -> OrganismAttribute\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> OrganismAttribute\n",
      "NamedThing - {'located_in'} -> OrganismAttribute\n",
      "NamedThing - {'occurs_in'} -> OrganismAttribute\n",
      "NamedThing - {'causes'} -> OrganismAttribute\n",
      "NamedThing - {'coexists_with'} -> OrganismAttribute\n",
      "NamedThing - {'close_match'} -> OrganismAttribute\n",
      "OrganismAttribute - {'related_to'} -> Publication\n",
      "OrganismAttribute - {'produces'} -> Publication\n",
      "OrganismAttribute - {'has_input'} -> Publication\n",
      "OrganismAttribute - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->PhysicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> PhysicalEntity\n",
      "NamedThing - {'related_to'} -> PhysicalEntity\n",
      "NamedThing - {'causes'} -> PhysicalEntity\n",
      "NamedThing - {'directly_physically_interacts_with'} -> PhysicalEntity\n",
      "NamedThing - {'is_sequence_variant_of'} -> PhysicalEntity\n",
      "NamedThing - {'associated_with'} -> PhysicalEntity\n",
      "NamedThing - {'gene_associated_with_condition'} -> PhysicalEntity\n",
      "NamedThing - {'has_phenotype'} -> PhysicalEntity\n",
      "NamedThing - {'located_in'} -> PhysicalEntity\n",
      "NamedThing - {'disrupts'} -> PhysicalEntity\n",
      "NamedThing - {'expressed_in'} -> PhysicalEntity\n",
      "NamedThing - {'has_part'} -> PhysicalEntity\n",
      "NamedThing - {'prevents'} -> PhysicalEntity\n",
      "NamedThing - {'colocalizes_with'} -> PhysicalEntity\n",
      "NamedThing - {'interacts_with'} -> PhysicalEntity\n",
      "NamedThing - {'produces'} -> PhysicalEntity\n",
      "NamedThing - {'predisposes'} -> PhysicalEntity\n",
      "NamedThing - {'diagnoses'} -> PhysicalEntity\n",
      "NamedThing - {'occurs_in'} -> PhysicalEntity\n",
      "NamedThing - {'exacerbates'} -> PhysicalEntity\n",
      "NamedThing - {'precedes'} -> PhysicalEntity\n",
      "NamedThing - {'affects'} -> PhysicalEntity\n",
      "NamedThing - {'coexists_with'} -> PhysicalEntity\n",
      "NamedThing - {'close_match'} -> PhysicalEntity\n",
      "NamedThing - {'has_participant'} -> PhysicalEntity\n",
      "NamedThing - {'manifestation_of'} -> PhysicalEntity\n",
      "NamedThing - {'regulates'} -> PhysicalEntity\n",
      "NamedThing - {'contraindicated_for'} -> PhysicalEntity\n",
      "NamedThing - {'treats'} -> PhysicalEntity\n",
      "NamedThing - {'has_input'} -> PhysicalEntity\n",
      "NamedThing - {'derives_from'} -> PhysicalEntity\n",
      "NamedThing - {'in_taxon'} -> PhysicalEntity\n",
      "NamedThing - {'physically_interacts_with'} -> PhysicalEntity\n",
      "NamedThing - {'has_output'} -> PhysicalEntity\n",
      "PhysicalEntity - {'subclass_of'} -> Publication\n",
      "PhysicalEntity - {'has_part'} -> Publication\n",
      "PhysicalEntity - {'interacts_with'} -> Publication\n",
      "PhysicalEntity - {'produces'} -> Publication\n",
      "PhysicalEntity - {'predisposes'} -> Publication\n",
      "PhysicalEntity - {'related_to'} -> Publication\n",
      "PhysicalEntity - {'occurs_in'} -> Publication\n",
      "PhysicalEntity - {'affects'} -> Publication\n",
      "PhysicalEntity - {'close_match'} -> Publication\n",
      "PhysicalEntity - {'coexists_with'} -> Publication\n",
      "PhysicalEntity - {'regulates'} -> Publication\n",
      "PhysicalEntity - {'located_in'} -> Publication\n",
      "PhysicalEntity - {'treats'} -> Publication\n",
      "PhysicalEntity - {'has_input'} -> Publication\n",
      "PhysicalEntity - {'causes'} -> Publication\n",
      "PhysicalEntity - {'disrupts'} -> Publication\n",
      "PhysicalEntity - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->GenomicEntity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> GenomicEntity\n",
      "NamedThing - {'actively_involved_in'} -> GenomicEntity\n",
      "NamedThing - {'affects_activity_of'} -> GenomicEntity\n",
      "NamedThing - {'affects'} -> GenomicEntity\n",
      "NamedThing - {'affects_transport_of'} -> GenomicEntity\n",
      "NamedThing - {'capable_of'} -> GenomicEntity\n",
      "NamedThing - {'caused_by'} -> GenomicEntity\n",
      "NamedThing - {'enabled_by'} -> GenomicEntity\n",
      "NamedThing - {'enables'} -> GenomicEntity\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> GenomicEntity\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> GenomicEntity\n",
      "NamedThing - {'entity_regulates_entity'} -> GenomicEntity\n",
      "NamedThing - {'has_input'} -> GenomicEntity\n",
      "NamedThing - {'has_output'} -> GenomicEntity\n",
      "NamedThing - {'has_part'} -> GenomicEntity\n",
      "NamedThing - {'has_participant'} -> GenomicEntity\n",
      "NamedThing - {'interacts_with'} -> GenomicEntity\n",
      "NamedThing - {'location_of'} -> GenomicEntity\n",
      "NamedThing - {'molecularly_interacts_with'} -> GenomicEntity\n",
      "NamedThing - {'overlaps'} -> GenomicEntity\n",
      "NamedThing - {'participates_in'} -> GenomicEntity\n",
      "NamedThing - {'physically_interacts_with'} -> GenomicEntity\n",
      "NamedThing - {'preceded_by'} -> GenomicEntity\n",
      "NamedThing - {'precedes'} -> GenomicEntity\n",
      "NamedThing - {'related_to'} -> GenomicEntity\n",
      "NamedThing - {'temporally_related_to'} -> GenomicEntity\n",
      "NamedThing - {'is_output_of'} -> GenomicEntity\n",
      "NamedThing - {'affected_by'} -> GenomicEntity\n",
      "NamedThing - {'treats'} -> GenomicEntity\n",
      "NamedThing - {'part_of'} -> GenomicEntity\n",
      "NamedThing - {'is_input_of'} -> GenomicEntity\n",
      "NamedThing - {'decreases_response_to'} -> GenomicEntity\n",
      "NamedThing - {'expresses'} -> GenomicEntity\n",
      "NamedThing - {'has_substrate'} -> GenomicEntity\n",
      "NamedThing - {'ameliorates'} -> GenomicEntity\n",
      "NamedThing - {'regulates'} -> GenomicEntity\n",
      "NamedThing - {'is_substrate_of'} -> GenomicEntity\n",
      "NamedThing - {'has_capability'} -> GenomicEntity\n",
      "NamedThing - {'actively_involves'} -> GenomicEntity\n",
      "NamedThing - {'has_active_ingredient'} -> GenomicEntity\n",
      "NamedThing - {'has_catalyst'} -> GenomicEntity\n",
      "NamedThing - {'associated_with'} -> GenomicEntity\n",
      "NamedThing - {'similar_to'} -> GenomicEntity\n",
      "NamedThing - {'catalyzes'} -> GenomicEntity\n",
      "NamedThing - {'affects_response_to'} -> GenomicEntity\n",
      "NamedThing - {'composed_primarily_of'} -> GenomicEntity\n",
      "NamedThing - {'genetically_interacts_with'} -> GenomicEntity\n",
      "NamedThing - {'has_upstream_actor'} -> GenomicEntity\n",
      "NamedThing - {'exacerbates'} -> GenomicEntity\n",
      "NamedThing - {'increases_response_to'} -> GenomicEntity\n",
      "NamedThing - {'primarily_composed_of'} -> GenomicEntity\n",
      "NamedThing - {'disrupts'} -> GenomicEntity\n",
      "NamedThing - {'coexists_with'} -> GenomicEntity\n",
      "NamedThing - {'close_match'} -> GenomicEntity\n",
      "NamedThing - {'produces'} -> GenomicEntity\n",
      "GenomicEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Annotation->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Annotation\n",
      "NamedThing - {'actively_involved_in'} -> Annotation\n",
      "NamedThing - {'actively_involves'} -> Annotation\n",
      "NamedThing - {'acts_upstream_of'} -> Annotation\n",
      "NamedThing - {'acts_upstream_of_or_within_positive_effect'} -> Annotation\n",
      "NamedThing - {'affects'} -> Annotation\n",
      "NamedThing - {'capable_of'} -> Annotation\n",
      "NamedThing - {'has_attribute'} -> Annotation\n",
      "NamedThing - {'has_part'} -> Annotation\n",
      "NamedThing - {'negatively_regulates'} -> Annotation\n",
      "NamedThing - {'overlaps'} -> Annotation\n",
      "NamedThing - {'participates_in'} -> Annotation\n",
      "NamedThing - {'part_of'} -> Annotation\n",
      "NamedThing - {'positively_regulates'} -> Annotation\n",
      "NamedThing - {'precedes'} -> Annotation\n",
      "NamedThing - {'process_negatively_regulates_process'} -> Annotation\n",
      "NamedThing - {'process_positively_regulates_process'} -> Annotation\n",
      "NamedThing - {'process_regulates_process'} -> Annotation\n",
      "NamedThing - {'regulates'} -> Annotation\n",
      "NamedThing - {'related_to'} -> Annotation\n",
      "NamedThing - {'temporally_related_to'} -> Annotation\n",
      "NamedThing - {'preceded_by'} -> Annotation\n",
      "NamedThing - {'disrupts'} -> Annotation\n",
      "NamedThing - {'acts_upstream_of_negative_effect'} -> Annotation\n",
      "NamedThing - {'acts_upstream_of_or_within'} -> Annotation\n",
      "NamedThing - {'affected_by'} -> Annotation\n",
      "NamedThing - {'interacts_with'} -> Annotation\n",
      "NamedThing - {'ameliorates'} -> Annotation\n",
      "NamedThing - {'treats'} -> Annotation\n",
      "NamedThing - {'is_substrate_of'} -> Annotation\n",
      "NamedThing - {'has_capability'} -> Annotation\n",
      "NamedThing - {'acts_upstream_of_or_within_negative_effect'} -> Annotation\n",
      "NamedThing - {'has_participant'} -> Annotation\n",
      "NamedThing - {'is_input_of'} -> Annotation\n",
      "NamedThing - {'decreases_response_to'} -> Annotation\n",
      "NamedThing - {'associated_with'} -> Annotation\n",
      "NamedThing - {'enables'} -> Annotation\n",
      "NamedThing - {'catalyzes'} -> Annotation\n",
      "NamedThing - {'has_upstream_actor'} -> Annotation\n",
      "NamedThing - {'exacerbates'} -> Annotation\n",
      "NamedThing - {'has_active_ingredient'} -> Annotation\n",
      "NamedThing - {'increases_response_to'} -> Annotation\n",
      "NamedThing - {'primarily_composed_of'} -> Annotation\n",
      "NamedThing - {'similar_to'} -> Annotation\n",
      "NamedThing - {'is_output_of'} -> Annotation\n",
      "NamedThing - {'acts_upstream_of_positive_effect'} -> Annotation\n",
      "NamedThing - {'affects_response_to'} -> Annotation\n",
      "NamedThing - {'composed_primarily_of'} -> Annotation\n",
      "Annotation - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->IndividualOrganism->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'has_input'} -> IndividualOrganism\n",
      "NamedThing - {'has_participant'} -> IndividualOrganism\n",
      "NamedThing - {'related_to'} -> IndividualOrganism\n",
      "NamedThing - {'interacts_with'} -> IndividualOrganism\n",
      "NamedThing - {'has_capability'} -> IndividualOrganism\n",
      "NamedThing - {'participates_in'} -> IndividualOrganism\n",
      "NamedThing - {'actively_involves'} -> IndividualOrganism\n",
      "NamedThing - {'has_catalyst'} -> IndividualOrganism\n",
      "NamedThing - {'associated_with'} -> IndividualOrganism\n",
      "NamedThing - {'overlaps'} -> IndividualOrganism\n",
      "NamedThing - {'has_upstream_actor'} -> IndividualOrganism\n",
      "NamedThing - {'primarily_composed_of'} -> IndividualOrganism\n",
      "NamedThing - {'has_output'} -> IndividualOrganism\n",
      "NamedThing - {'affects'} -> IndividualOrganism\n",
      "NamedThing - {'similar_to'} -> IndividualOrganism\n",
      "NamedThing - {'affected_by'} -> IndividualOrganism\n",
      "NamedThing - {'composed_primarily_of'} -> IndividualOrganism\n",
      "NamedThing - {'enabled_by'} -> IndividualOrganism\n",
      "NamedThing - {'has_substrate'} -> IndividualOrganism\n",
      "NamedThing - {'subclass_of'} -> IndividualOrganism\n",
      "NamedThing - {'located_in'} -> IndividualOrganism\n",
      "NamedThing - {'produces'} -> IndividualOrganism\n",
      "NamedThing - {'has_part'} -> IndividualOrganism\n",
      "NamedThing - {'occurs_in'} -> IndividualOrganism\n",
      "NamedThing - {'close_match'} -> IndividualOrganism\n",
      "NamedThing - {'regulates'} -> IndividualOrganism\n",
      "NamedThing - {'treats'} -> IndividualOrganism\n",
      "NamedThing - {'causes'} -> IndividualOrganism\n",
      "NamedThing - {'in_taxon'} -> IndividualOrganism\n",
      "NamedThing - {'disrupts'} -> IndividualOrganism\n",
      "IndividualOrganism - {'close_match'} -> Publication\n",
      "IndividualOrganism - {'subclass_of'} -> Publication\n",
      "IndividualOrganism - {'occurs_in'} -> Publication\n",
      "IndividualOrganism - {'located_in'} -> Publication\n",
      "IndividualOrganism - {'related_to'} -> Publication\n",
      "IndividualOrganism - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->ClinicalAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'primarily_composed_of'} -> ClinicalAttribute\n",
      "NamedThing - {'associated_with'} -> ClinicalAttribute\n",
      "NamedThing - {'affects'} -> ClinicalAttribute\n",
      "NamedThing - {'similar_to'} -> ClinicalAttribute\n",
      "NamedThing - {'related_to'} -> ClinicalAttribute\n",
      "NamedThing - {'overlaps'} -> ClinicalAttribute\n",
      "NamedThing - {'composed_primarily_of'} -> ClinicalAttribute\n",
      "NamedThing - {'has_upstream_actor'} -> ClinicalAttribute\n",
      "NamedThing - {'affected_by'} -> ClinicalAttribute\n",
      "NamedThing - {'participates_in'} -> ClinicalAttribute\n",
      "NamedThing - {'has_participant'} -> ClinicalAttribute\n",
      "NamedThing - {'interacts_with'} -> ClinicalAttribute\n",
      "NamedThing - {'close_match'} -> ClinicalAttribute\n",
      "NamedThing - {'disrupts'} -> ClinicalAttribute\n",
      "NamedThing - {'treats'} -> ClinicalAttribute\n",
      "NamedThing - {'regulates'} -> ClinicalAttribute\n",
      "NamedThing - {'has_part'} -> ClinicalAttribute\n",
      "NamedThing - {'coexists_with'} -> ClinicalAttribute\n",
      "ClinicalAttribute - {'correlated_with'} -> Publication\n",
      "ClinicalAttribute - {'has_input'} -> Publication\n",
      "ClinicalAttribute - {'subclass_of'} -> Publication\n",
      "ClinicalAttribute - {'related_to'} -> Publication\n",
      "ClinicalAttribute - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->PhysiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'entity_positively_regulates_entity'} -> PhysiologicalProcess\n",
      "NamedThing - {'has_upstream_actor'} -> PhysiologicalProcess\n",
      "NamedThing - {'exacerbates'} -> PhysiologicalProcess\n",
      "NamedThing - {'increases_response_to'} -> PhysiologicalProcess\n",
      "NamedThing - {'primarily_composed_of'} -> PhysiologicalProcess\n",
      "NamedThing - {'affects'} -> PhysiologicalProcess\n",
      "NamedThing - {'similar_to'} -> PhysiologicalProcess\n",
      "NamedThing - {'related_to'} -> PhysiologicalProcess\n",
      "NamedThing - {'overlaps'} -> PhysiologicalProcess\n",
      "NamedThing - {'affects_response_to'} -> PhysiologicalProcess\n",
      "NamedThing - {'causes'} -> PhysiologicalProcess\n",
      "NamedThing - {'composed_primarily_of'} -> PhysiologicalProcess\n",
      "NamedThing - {'disrupts'} -> PhysiologicalProcess\n",
      "NamedThing - {'affected_by'} -> PhysiologicalProcess\n",
      "NamedThing - {'interacts_with'} -> PhysiologicalProcess\n",
      "NamedThing - {'ameliorates'} -> PhysiologicalProcess\n",
      "NamedThing - {'treats'} -> PhysiologicalProcess\n",
      "NamedThing - {'regulates'} -> PhysiologicalProcess\n",
      "NamedThing - {'participates_in'} -> PhysiologicalProcess\n",
      "NamedThing - {'contributes_to'} -> PhysiologicalProcess\n",
      "NamedThing - {'caused_by'} -> PhysiologicalProcess\n",
      "NamedThing - {'has_participant'} -> PhysiologicalProcess\n",
      "NamedThing - {'entity_negatively_regulates_entity'} -> PhysiologicalProcess\n",
      "NamedThing - {'decreases_response_to'} -> PhysiologicalProcess\n",
      "NamedThing - {'associated_with'} -> PhysiologicalProcess\n",
      "NamedThing - {'subclass_of'} -> PhysiologicalProcess\n",
      "NamedThing - {'has_part'} -> PhysiologicalProcess\n",
      "NamedThing - {'prevents'} -> PhysiologicalProcess\n",
      "NamedThing - {'enables'} -> PhysiologicalProcess\n",
      "NamedThing - {'predisposes'} -> PhysiologicalProcess\n",
      "NamedThing - {'diagnoses'} -> PhysiologicalProcess\n",
      "NamedThing - {'occurs_in'} -> PhysiologicalProcess\n",
      "NamedThing - {'precedes'} -> PhysiologicalProcess\n",
      "NamedThing - {'coexists_with'} -> PhysiologicalProcess\n",
      "NamedThing - {'close_match'} -> PhysiologicalProcess\n",
      "NamedThing - {'manifestation_of'} -> PhysiologicalProcess\n",
      "NamedThing - {'capable_of'} -> PhysiologicalProcess\n",
      "NamedThing - {'contraindicated_for'} -> PhysiologicalProcess\n",
      "NamedThing - {'has_input'} -> PhysiologicalProcess\n",
      "NamedThing - {'temporally_related_to'} -> PhysiologicalProcess\n",
      "NamedThing - {'actively_involved_in'} -> PhysiologicalProcess\n",
      "NamedThing - {'correlated_with'} -> PhysiologicalProcess\n",
      "NamedThing - {'gene_associated_with_condition'} -> PhysiologicalProcess\n",
      "NamedThing - {'has_phenotype'} -> PhysiologicalProcess\n",
      "NamedThing - {'located_in'} -> PhysiologicalProcess\n",
      "NamedThing - {'produces'} -> PhysiologicalProcess\n",
      "PhysiologicalProcess - {'correlated_with'} -> Publication\n",
      "PhysiologicalProcess - {'has_input'} -> Publication\n",
      "PhysiologicalProcess - {'causes'} -> Publication\n",
      "PhysiologicalProcess - {'has_output'} -> Publication\n",
      "PhysiologicalProcess - {'disrupts'} -> Publication\n",
      "PhysiologicalProcess - {'associated_with'} -> Publication\n",
      "PhysiologicalProcess - {'located_in'} -> Publication\n",
      "PhysiologicalProcess - {'produces'} -> Publication\n",
      "PhysiologicalProcess - {'predisposes'} -> Publication\n",
      "PhysiologicalProcess - {'precedes'} -> Publication\n",
      "PhysiologicalProcess - {'related_to'} -> Publication\n",
      "PhysiologicalProcess - {'occurs_in'} -> Publication\n",
      "PhysiologicalProcess - {'affects'} -> Publication\n",
      "PhysiologicalProcess - {'coexists_with'} -> Publication\n",
      "PhysiologicalProcess - {'close_match'} -> Publication\n",
      "PhysiologicalProcess - {'has_participant'} -> Publication\n",
      "PhysiologicalProcess - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Phenomenon->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'related_to'} -> Phenomenon\n",
      "NamedThing - {'prevents'} -> Phenomenon\n",
      "NamedThing - {'diagnoses'} -> Phenomenon\n",
      "NamedThing - {'predisposes'} -> Phenomenon\n",
      "NamedThing - {'precedes'} -> Phenomenon\n",
      "NamedThing - {'occurs_in'} -> Phenomenon\n",
      "NamedThing - {'affects'} -> Phenomenon\n",
      "NamedThing - {'manifestation_of'} -> Phenomenon\n",
      "NamedThing - {'exacerbates'} -> Phenomenon\n",
      "NamedThing - {'coexists_with'} -> Phenomenon\n",
      "NamedThing - {'close_match'} -> Phenomenon\n",
      "NamedThing - {'has_participant'} -> Phenomenon\n",
      "NamedThing - {'regulates'} -> Phenomenon\n",
      "NamedThing - {'contraindicated_for'} -> Phenomenon\n",
      "NamedThing - {'physically_interacts_with'} -> Phenomenon\n",
      "NamedThing - {'treats'} -> Phenomenon\n",
      "NamedThing - {'has_input'} -> Phenomenon\n",
      "NamedThing - {'derives_from'} -> Phenomenon\n",
      "NamedThing - {'causes'} -> Phenomenon\n",
      "NamedThing - {'disrupts'} -> Phenomenon\n",
      "NamedThing - {'associated_with'} -> Phenomenon\n",
      "NamedThing - {'correlated_with'} -> Phenomenon\n",
      "NamedThing - {'gene_associated_with_condition'} -> Phenomenon\n",
      "NamedThing - {'has_phenotype'} -> Phenomenon\n",
      "NamedThing - {'subclass_of'} -> Phenomenon\n",
      "NamedThing - {'located_in'} -> Phenomenon\n",
      "NamedThing - {'biomarker_for'} -> Phenomenon\n",
      "NamedThing - {'interacts_with'} -> Phenomenon\n",
      "NamedThing - {'produces'} -> Phenomenon\n",
      "NamedThing - {'has_part'} -> Phenomenon\n",
      "Phenomenon - {'associated_with'} -> Publication\n",
      "Phenomenon - {'subclass_of'} -> Publication\n",
      "Phenomenon - {'interacts_with'} -> Publication\n",
      "Phenomenon - {'produces'} -> Publication\n",
      "Phenomenon - {'related_to'} -> Publication\n",
      "Phenomenon - {'has_part'} -> Publication\n",
      "Phenomenon - {'diagnoses'} -> Publication\n",
      "Phenomenon - {'occurs_in'} -> Publication\n",
      "Phenomenon - {'affects'} -> Publication\n",
      "Phenomenon - {'coexists_with'} -> Publication\n",
      "Phenomenon - {'close_match'} -> Publication\n",
      "Phenomenon - {'disrupts'} -> Publication\n",
      "Phenomenon - {'regulates'} -> Publication\n",
      "Phenomenon - {'located_in'} -> Publication\n",
      "Phenomenon - {'has_input'} -> Publication\n",
      "Phenomenon - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Procedure->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'interacts_with'} -> Procedure\n",
      "NamedThing - {'treats'} -> Procedure\n",
      "NamedThing - {'participates_in'} -> Procedure\n",
      "NamedThing - {'preceded_by'} -> Procedure\n",
      "NamedThing - {'has_participant'} -> Procedure\n",
      "NamedThing - {'associated_with'} -> Procedure\n",
      "NamedThing - {'precedes'} -> Procedure\n",
      "NamedThing - {'ameliorates'} -> Procedure\n",
      "NamedThing - {'has_upstream_actor'} -> Procedure\n",
      "NamedThing - {'primarily_composed_of'} -> Procedure\n",
      "NamedThing - {'temporally_related_to'} -> Procedure\n",
      "NamedThing - {'affects'} -> Procedure\n",
      "NamedThing - {'similar_to'} -> Procedure\n",
      "NamedThing - {'related_to'} -> Procedure\n",
      "NamedThing - {'overlaps'} -> Procedure\n",
      "NamedThing - {'composed_primarily_of'} -> Procedure\n",
      "NamedThing - {'affected_by'} -> Procedure\n",
      "NamedThing - {'regulates'} -> Procedure\n",
      "NamedThing - {'contraindicated_for'} -> Procedure\n",
      "NamedThing - {'chemically_similar_to'} -> Procedure\n",
      "NamedThing - {'physically_interacts_with'} -> Procedure\n",
      "NamedThing - {'has_input'} -> Procedure\n",
      "NamedThing - {'derives_from'} -> Procedure\n",
      "NamedThing - {'causes'} -> Procedure\n",
      "NamedThing - {'gene_product_of'} -> Procedure\n",
      "NamedThing - {'has_phenotype'} -> Procedure\n",
      "NamedThing - {'disrupts'} -> Procedure\n",
      "NamedThing - {'biomarker_for'} -> Procedure\n",
      "NamedThing - {'correlated_with'} -> Procedure\n",
      "NamedThing - {'gene_associated_with_condition'} -> Procedure\n",
      "NamedThing - {'subclass_of'} -> Procedure\n",
      "NamedThing - {'has_part'} -> Procedure\n",
      "NamedThing - {'located_in'} -> Procedure\n",
      "NamedThing - {'produces'} -> Procedure\n",
      "NamedThing - {'prevents'} -> Procedure\n",
      "NamedThing - {'diagnoses'} -> Procedure\n",
      "NamedThing - {'contributes_to'} -> Procedure\n",
      "NamedThing - {'predisposes'} -> Procedure\n",
      "NamedThing - {'occurs_in'} -> Procedure\n",
      "NamedThing - {'close_match'} -> Procedure\n",
      "NamedThing - {'manifestation_of'} -> Procedure\n",
      "NamedThing - {'exacerbates'} -> Procedure\n",
      "NamedThing - {'coexists_with'} -> Procedure\n",
      "Procedure - {'correlated_with'} -> Publication\n",
      "Procedure - {'manifestation_of'} -> Publication\n",
      "Procedure - {'close_match'} -> Publication\n",
      "Procedure - {'subclass_of'} -> Publication\n",
      "Procedure - {'interacts_with'} -> Publication\n",
      "Procedure - {'diagnoses'} -> Publication\n",
      "Procedure - {'occurs_in'} -> Publication\n",
      "Procedure - {'treats'} -> Publication\n",
      "Procedure - {'has_input'} -> Publication\n",
      "Procedure - {'exacerbates'} -> Publication\n",
      "Procedure - {'causes'} -> Publication\n",
      "Procedure - {'coexists_with'} -> Publication\n",
      "Procedure - {'disrupts'} -> Publication\n",
      "Procedure - {'associated_with'} -> Publication\n",
      "Procedure - {'regulates'} -> Publication\n",
      "Procedure - {'produces'} -> Publication\n",
      "Procedure - {'located_in'} -> Publication\n",
      "Procedure - {'predisposes'} -> Publication\n",
      "Procedure - {'related_to'} -> Publication\n",
      "Procedure - {'has_part'} -> Publication\n",
      "Procedure - {'prevents'} -> Publication\n",
      "Procedure - {'precedes'} -> Publication\n",
      "Procedure - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->GeneFamily->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'actively_involves'} -> GeneFamily\n",
      "NamedThing - {'has_catalyst'} -> GeneFamily\n",
      "NamedThing - {'associated_with'} -> GeneFamily\n",
      "NamedThing - {'overlaps'} -> GeneFamily\n",
      "NamedThing - {'composed_primarily_of'} -> GeneFamily\n",
      "NamedThing - {'has_upstream_actor'} -> GeneFamily\n",
      "NamedThing - {'has_active_ingredient'} -> GeneFamily\n",
      "NamedThing - {'primarily_composed_of'} -> GeneFamily\n",
      "NamedThing - {'has_output'} -> GeneFamily\n",
      "NamedThing - {'affects'} -> GeneFamily\n",
      "NamedThing - {'similar_to'} -> GeneFamily\n",
      "NamedThing - {'related_to'} -> GeneFamily\n",
      "NamedThing - {'affected_by'} -> GeneFamily\n",
      "NamedThing - {'has_input'} -> GeneFamily\n",
      "NamedThing - {'enabled_by'} -> GeneFamily\n",
      "NamedThing - {'has_participant'} -> GeneFamily\n",
      "NamedThing - {'has_part'} -> GeneFamily\n",
      "NamedThing - {'has_substrate'} -> GeneFamily\n",
      "NamedThing - {'interacts_with'} -> GeneFamily\n",
      "NamedThing - {'part_of'} -> GeneFamily\n",
      "NamedThing - {'has_capability'} -> GeneFamily\n",
      "NamedThing - {'participates_in'} -> GeneFamily\n",
      "NamedThing - {'coexists_with'} -> GeneFamily\n",
      "NamedThing - {'regulates'} -> GeneFamily\n",
      "NamedThing - {'derives_from'} -> GeneFamily\n",
      "NamedThing - {'physically_interacts_with'} -> GeneFamily\n",
      "NamedThing - {'directly_physically_interacts_with'} -> GeneFamily\n",
      "NamedThing - {'is_sequence_variant_of'} -> GeneFamily\n",
      "NamedThing - {'has_phenotype'} -> GeneFamily\n",
      "NamedThing - {'located_in'} -> GeneFamily\n",
      "NamedThing - {'subclass_of'} -> GeneFamily\n",
      "NamedThing - {'colocalizes_with'} -> GeneFamily\n",
      "NamedThing - {'produces'} -> GeneFamily\n",
      "NamedThing - {'occurs_in'} -> GeneFamily\n",
      "NamedThing - {'precedes'} -> GeneFamily\n",
      "NamedThing - {'close_match'} -> GeneFamily\n",
      "GeneFamily - {'treats'} -> Publication\n",
      "GeneFamily - {'causes'} -> Publication\n",
      "GeneFamily - {'associated_with'} -> Publication\n",
      "GeneFamily - {'disrupts'} -> Publication\n",
      "GeneFamily - {'gene_associated_with_condition'} -> Publication\n",
      "GeneFamily - {'located_in'} -> Publication\n",
      "GeneFamily - {'regulates'} -> Publication\n",
      "GeneFamily - {'predisposes'} -> Publication\n",
      "GeneFamily - {'occurs_in'} -> Publication\n",
      "GeneFamily - {'affects'} -> Publication\n",
      "GeneFamily - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Activity->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'ameliorates'} -> Activity\n",
      "NamedThing - {'related_to'} -> Activity\n",
      "NamedThing - {'affects'} -> Activity\n",
      "NamedThing - {'treats'} -> Activity\n",
      "NamedThing - {'regulates'} -> Activity\n",
      "NamedThing - {'contraindicated_for'} -> Activity\n",
      "NamedThing - {'physically_interacts_with'} -> Activity\n",
      "NamedThing - {'has_input'} -> Activity\n",
      "NamedThing - {'causes'} -> Activity\n",
      "NamedThing - {'gene_product_of'} -> Activity\n",
      "NamedThing - {'has_phenotype'} -> Activity\n",
      "NamedThing - {'disrupts'} -> Activity\n",
      "NamedThing - {'is_sequence_variant_of'} -> Activity\n",
      "NamedThing - {'biomarker_for'} -> Activity\n",
      "NamedThing - {'associated_with'} -> Activity\n",
      "NamedThing - {'correlated_with'} -> Activity\n",
      "NamedThing - {'gene_associated_with_condition'} -> Activity\n",
      "NamedThing - {'subclass_of'} -> Activity\n",
      "NamedThing - {'has_part'} -> Activity\n",
      "NamedThing - {'located_in'} -> Activity\n",
      "NamedThing - {'interacts_with'} -> Activity\n",
      "NamedThing - {'produces'} -> Activity\n",
      "NamedThing - {'prevents'} -> Activity\n",
      "NamedThing - {'diagnoses'} -> Activity\n",
      "NamedThing - {'enables'} -> Activity\n",
      "NamedThing - {'predisposes'} -> Activity\n",
      "NamedThing - {'precedes'} -> Activity\n",
      "NamedThing - {'occurs_in'} -> Activity\n",
      "NamedThing - {'close_match'} -> Activity\n",
      "NamedThing - {'manifestation_of'} -> Activity\n",
      "NamedThing - {'exacerbates'} -> Activity\n",
      "NamedThing - {'coexists_with'} -> Activity\n",
      "NamedThing - {'has_participant'} -> Activity\n",
      "Activity - {'correlated_with'} -> Publication\n",
      "Activity - {'causes'} -> Publication\n",
      "Activity - {'disrupts'} -> Publication\n",
      "Activity - {'subclass_of'} -> Publication\n",
      "Activity - {'located_in'} -> Publication\n",
      "Activity - {'interacts_with'} -> Publication\n",
      "Activity - {'produces'} -> Publication\n",
      "Activity - {'has_part'} -> Publication\n",
      "Activity - {'diagnoses'} -> Publication\n",
      "Activity - {'prevents'} -> Publication\n",
      "Activity - {'predisposes'} -> Publication\n",
      "Activity - {'precedes'} -> Publication\n",
      "Activity - {'related_to'} -> Publication\n",
      "Activity - {'occurs_in'} -> Publication\n",
      "Activity - {'affects'} -> Publication\n",
      "Activity - {'coexists_with'} -> Publication\n",
      "Activity - {'close_match'} -> Publication\n",
      "Activity - {'regulates'} -> Publication\n",
      "Activity - {'treats'} -> Publication\n",
      "Activity - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Device->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'has_part'} -> Device\n",
      "NamedThing - {'prevents'} -> Device\n",
      "NamedThing - {'colocalizes_with'} -> Device\n",
      "NamedThing - {'interacts_with'} -> Device\n",
      "NamedThing - {'produces'} -> Device\n",
      "NamedThing - {'predisposes'} -> Device\n",
      "NamedThing - {'related_to'} -> Device\n",
      "NamedThing - {'diagnoses'} -> Device\n",
      "NamedThing - {'occurs_in'} -> Device\n",
      "NamedThing - {'exacerbates'} -> Device\n",
      "NamedThing - {'precedes'} -> Device\n",
      "NamedThing - {'affects'} -> Device\n",
      "NamedThing - {'coexists_with'} -> Device\n",
      "NamedThing - {'close_match'} -> Device\n",
      "NamedThing - {'has_participant'} -> Device\n",
      "NamedThing - {'manifestation_of'} -> Device\n",
      "NamedThing - {'regulates'} -> Device\n",
      "NamedThing - {'contraindicated_for'} -> Device\n",
      "NamedThing - {'treats'} -> Device\n",
      "NamedThing - {'has_input'} -> Device\n",
      "NamedThing - {'derives_from'} -> Device\n",
      "NamedThing - {'physically_interacts_with'} -> Device\n",
      "NamedThing - {'causes'} -> Device\n",
      "NamedThing - {'disease_has_location'} -> Device\n",
      "NamedThing - {'directly_physically_interacts_with'} -> Device\n",
      "NamedThing - {'is_sequence_variant_of'} -> Device\n",
      "NamedThing - {'associated_with'} -> Device\n",
      "NamedThing - {'gene_associated_with_condition'} -> Device\n",
      "NamedThing - {'has_phenotype'} -> Device\n",
      "NamedThing - {'located_in'} -> Device\n",
      "NamedThing - {'disrupts'} -> Device\n",
      "NamedThing - {'expressed_in'} -> Device\n",
      "NamedThing - {'subclass_of'} -> Device\n",
      "Device - {'subclass_of'} -> Publication\n",
      "Device - {'diagnoses'} -> Publication\n",
      "Device - {'related_to'} -> Publication\n",
      "Device - {'occurs_in'} -> Publication\n",
      "Device - {'affects'} -> Publication\n",
      "Device - {'treats'} -> Publication\n",
      "Device - {'has_input'} -> Publication\n",
      "Device - {'coexists_with'} -> Publication\n",
      "Device - {'regulates'} -> Publication\n",
      "Device - {'disrupts'} -> Publication\n",
      "Device - {'biomarker_for'} -> Publication\n",
      "Device - {'gene_associated_with_condition'} -> Publication\n",
      "Device - {'has_part'} -> Publication\n",
      "Device - {'located_in'} -> Publication\n",
      "Device - {'causes'} -> Publication\n",
      "Device - {'precedes'} -> Publication\n",
      "Device - {'close_match'} -> Publication\n",
      "Device - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->ProteinFamily->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'biomarker_for'} -> ProteinFamily\n",
      "NamedThing - {'subclass_of'} -> ProteinFamily\n",
      "NamedThing - {'has_part'} -> ProteinFamily\n",
      "NamedThing - {'colocalizes_with'} -> ProteinFamily\n",
      "NamedThing - {'interacts_with'} -> ProteinFamily\n",
      "NamedThing - {'produces'} -> ProteinFamily\n",
      "NamedThing - {'related_to'} -> ProteinFamily\n",
      "NamedThing - {'occurs_in'} -> ProteinFamily\n",
      "NamedThing - {'affects'} -> ProteinFamily\n",
      "NamedThing - {'close_match'} -> ProteinFamily\n",
      "NamedThing - {'has_participant'} -> ProteinFamily\n",
      "NamedThing - {'coexists_with'} -> ProteinFamily\n",
      "NamedThing - {'regulates'} -> ProteinFamily\n",
      "NamedThing - {'capable_of'} -> ProteinFamily\n",
      "NamedThing - {'has_input'} -> ProteinFamily\n",
      "NamedThing - {'derives_from'} -> ProteinFamily\n",
      "NamedThing - {'physically_interacts_with'} -> ProteinFamily\n",
      "NamedThing - {'directly_physically_interacts_with'} -> ProteinFamily\n",
      "NamedThing - {'gene_product_of'} -> ProteinFamily\n",
      "NamedThing - {'is_sequence_variant_of'} -> ProteinFamily\n",
      "NamedThing - {'associated_with'} -> ProteinFamily\n",
      "NamedThing - {'has_phenotype'} -> ProteinFamily\n",
      "NamedThing - {'located_in'} -> ProteinFamily\n",
      "NamedThing - {'binds'} -> ProteinFamily\n",
      "ProteinFamily - {'associated_with'} -> Publication\n",
      "ProteinFamily - {'subclass_of'} -> Publication\n",
      "ProteinFamily - {'gene_associated_with_condition'} -> Publication\n",
      "ProteinFamily - {'predisposes'} -> Publication\n",
      "ProteinFamily - {'related_to'} -> Publication\n",
      "ProteinFamily - {'affects'} -> Publication\n",
      "ProteinFamily - {'disrupts'} -> Publication\n",
      "ProteinFamily - {'located_in'} -> Publication\n",
      "ProteinFamily - {'regulates'} -> Publication\n",
      "ProteinFamily - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->BehavioralFeature->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'predisposes'} -> BehavioralFeature\n",
      "NamedThing - {'related_to'} -> BehavioralFeature\n",
      "NamedThing - {'diagnoses'} -> BehavioralFeature\n",
      "NamedThing - {'occurs_in'} -> BehavioralFeature\n",
      "NamedThing - {'exacerbates'} -> BehavioralFeature\n",
      "NamedThing - {'precedes'} -> BehavioralFeature\n",
      "NamedThing - {'affects'} -> BehavioralFeature\n",
      "NamedThing - {'coexists_with'} -> BehavioralFeature\n",
      "NamedThing - {'close_match'} -> BehavioralFeature\n",
      "NamedThing - {'manifestation_of'} -> BehavioralFeature\n",
      "NamedThing - {'regulates'} -> BehavioralFeature\n",
      "NamedThing - {'contraindicated_for'} -> BehavioralFeature\n",
      "NamedThing - {'treats'} -> BehavioralFeature\n",
      "NamedThing - {'has_input'} -> BehavioralFeature\n",
      "NamedThing - {'causes'} -> BehavioralFeature\n",
      "NamedThing - {'disrupts'} -> BehavioralFeature\n",
      "NamedThing - {'associated_with'} -> BehavioralFeature\n",
      "NamedThing - {'gene_associated_with_condition'} -> BehavioralFeature\n",
      "NamedThing - {'has_phenotype'} -> BehavioralFeature\n",
      "NamedThing - {'subclass_of'} -> BehavioralFeature\n",
      "NamedThing - {'has_part'} -> BehavioralFeature\n",
      "NamedThing - {'prevents'} -> BehavioralFeature\n",
      "NamedThing - {'contributes_to'} -> BehavioralFeature\n",
      "NamedThing - {'interacts_with'} -> BehavioralFeature\n",
      "BehavioralFeature - {'subclass_of'} -> Publication\n",
      "BehavioralFeature - {'produces'} -> Publication\n",
      "BehavioralFeature - {'related_to'} -> Publication\n",
      "BehavioralFeature - {'occurs_in'} -> Publication\n",
      "BehavioralFeature - {'affects'} -> Publication\n",
      "BehavioralFeature - {'manifestation_of'} -> Publication\n",
      "BehavioralFeature - {'coexists_with'} -> Publication\n",
      "BehavioralFeature - {'disrupts'} -> Publication\n",
      "BehavioralFeature - {'located_in'} -> Publication\n",
      "BehavioralFeature - {'predisposes'} -> Publication\n",
      "BehavioralFeature - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Vitamin->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'physically_interacts_with'} -> Vitamin\n",
      "NamedThing - {'has_input'} -> Vitamin\n",
      "NamedThing - {'derives_from'} -> Vitamin\n",
      "NamedThing - {'associated_with'} -> Vitamin\n",
      "NamedThing - {'subclass_of'} -> Vitamin\n",
      "NamedThing - {'has_part'} -> Vitamin\n",
      "NamedThing - {'interacts_with'} -> Vitamin\n",
      "NamedThing - {'produces'} -> Vitamin\n",
      "NamedThing - {'precedes'} -> Vitamin\n",
      "NamedThing - {'related_to'} -> Vitamin\n",
      "NamedThing - {'close_match'} -> Vitamin\n",
      "NamedThing - {'coexists_with'} -> Vitamin\n",
      "NamedThing - {'has_participant'} -> Vitamin\n",
      "NamedThing - {'regulates'} -> Vitamin\n",
      "Vitamin - {'regulates'} -> Publication\n",
      "Vitamin - {'treats'} -> Publication\n",
      "Vitamin - {'disrupts'} -> Publication\n",
      "Vitamin - {'subclass_of'} -> Publication\n",
      "Vitamin - {'related_to'} -> Publication\n",
      "Vitamin - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->ClinicalIntervention->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'affects'} -> ClinicalIntervention\n",
      "NamedThing - {'close_match'} -> ClinicalIntervention\n",
      "NamedThing - {'has_participant'} -> ClinicalIntervention\n",
      "NamedThing - {'manifestation_of'} -> ClinicalIntervention\n",
      "NamedThing - {'exacerbates'} -> ClinicalIntervention\n",
      "NamedThing - {'coexists_with'} -> ClinicalIntervention\n",
      "NamedThing - {'regulates'} -> ClinicalIntervention\n",
      "NamedThing - {'contraindicated_for'} -> ClinicalIntervention\n",
      "NamedThing - {'treats'} -> ClinicalIntervention\n",
      "NamedThing - {'has_input'} -> ClinicalIntervention\n",
      "NamedThing - {'derives_from'} -> ClinicalIntervention\n",
      "NamedThing - {'physically_interacts_with'} -> ClinicalIntervention\n",
      "NamedThing - {'causes'} -> ClinicalIntervention\n",
      "NamedThing - {'associated_with'} -> ClinicalIntervention\n",
      "NamedThing - {'gene_associated_with_condition'} -> ClinicalIntervention\n",
      "NamedThing - {'has_phenotype'} -> ClinicalIntervention\n",
      "NamedThing - {'located_in'} -> ClinicalIntervention\n",
      "NamedThing - {'disrupts'} -> ClinicalIntervention\n",
      "NamedThing - {'biomarker_for'} -> ClinicalIntervention\n",
      "NamedThing - {'correlated_with'} -> ClinicalIntervention\n",
      "NamedThing - {'subclass_of'} -> ClinicalIntervention\n",
      "NamedThing - {'has_part'} -> ClinicalIntervention\n",
      "NamedThing - {'interacts_with'} -> ClinicalIntervention\n",
      "NamedThing - {'produces'} -> ClinicalIntervention\n",
      "NamedThing - {'predisposes'} -> ClinicalIntervention\n",
      "NamedThing - {'related_to'} -> ClinicalIntervention\n",
      "NamedThing - {'prevents'} -> ClinicalIntervention\n",
      "NamedThing - {'diagnoses'} -> ClinicalIntervention\n",
      "NamedThing - {'occurs_in'} -> ClinicalIntervention\n",
      "NamedThing - {'precedes'} -> ClinicalIntervention\n",
      "ClinicalIntervention - {'correlated_with'} -> Publication\n",
      "ClinicalIntervention - {'regulates'} -> Publication\n",
      "ClinicalIntervention - {'diagnoses'} -> Publication\n",
      "ClinicalIntervention - {'treats'} -> Publication\n",
      "ClinicalIntervention - {'has_input'} -> Publication\n",
      "ClinicalIntervention - {'causes'} -> Publication\n",
      "ClinicalIntervention - {'disrupts'} -> Publication\n",
      "ClinicalIntervention - {'associated_with'} -> Publication\n",
      "ClinicalIntervention - {'has_part'} -> Publication\n",
      "ClinicalIntervention - {'located_in'} -> Publication\n",
      "ClinicalIntervention - {'predisposes'} -> Publication\n",
      "ClinicalIntervention - {'related_to'} -> Publication\n",
      "ClinicalIntervention - {'prevents'} -> Publication\n",
      "ClinicalIntervention - {'manifestation_of'} -> Publication\n",
      "ClinicalIntervention - {'precedes'} -> Publication\n",
      "ClinicalIntervention - {'occurs_in'} -> Publication\n",
      "ClinicalIntervention - {'affects'} -> Publication\n",
      "ClinicalIntervention - {'close_match'} -> Publication\n",
      "ClinicalIntervention - {'exacerbates'} -> Publication\n",
      "ClinicalIntervention - {'coexists_with'} -> Publication\n",
      "ClinicalIntervention - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Food->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'colocalizes_with'} -> Food\n",
      "NamedThing - {'interacts_with'} -> Food\n",
      "NamedThing - {'predisposes'} -> Food\n",
      "NamedThing - {'related_to'} -> Food\n",
      "NamedThing - {'diagnoses'} -> Food\n",
      "NamedThing - {'occurs_in'} -> Food\n",
      "NamedThing - {'affects'} -> Food\n",
      "NamedThing - {'exacerbates'} -> Food\n",
      "NamedThing - {'precedes'} -> Food\n",
      "NamedThing - {'coexists_with'} -> Food\n",
      "NamedThing - {'close_match'} -> Food\n",
      "NamedThing - {'has_participant'} -> Food\n",
      "NamedThing - {'manifestation_of'} -> Food\n",
      "NamedThing - {'regulates'} -> Food\n",
      "NamedThing - {'treats'} -> Food\n",
      "NamedThing - {'has_input'} -> Food\n",
      "NamedThing - {'derives_from'} -> Food\n",
      "NamedThing - {'causes'} -> Food\n",
      "NamedThing - {'in_taxon'} -> Food\n",
      "NamedThing - {'physically_interacts_with'} -> Food\n",
      "NamedThing - {'disrupts'} -> Food\n",
      "NamedThing - {'is_sequence_variant_of'} -> Food\n",
      "NamedThing - {'associated_with'} -> Food\n",
      "NamedThing - {'gene_associated_with_condition'} -> Food\n",
      "NamedThing - {'has_phenotype'} -> Food\n",
      "NamedThing - {'located_in'} -> Food\n",
      "NamedThing - {'produces'} -> Food\n",
      "NamedThing - {'subclass_of'} -> Food\n",
      "NamedThing - {'has_part'} -> Food\n",
      "NamedThing - {'prevents'} -> Food\n",
      "Food - {'regulates'} -> Publication\n",
      "Food - {'associated_with'} -> Publication\n",
      "Food - {'located_in'} -> Publication\n",
      "Food - {'predisposes'} -> Publication\n",
      "Food - {'related_to'} -> Publication\n",
      "Food - {'occurs_in'} -> Publication\n",
      "Food - {'affects'} -> Publication\n",
      "Food - {'precedes'} -> Publication\n",
      "Food - {'subclass_of'} -> Publication\n",
      "Food - {'treats'} -> Publication\n",
      "Food - {'causes'} -> Publication\n",
      "Food - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Cohort->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'related_to'} -> Cohort\n",
      "NamedThing - {'occurs_in'} -> Cohort\n",
      "NamedThing - {'affects'} -> Cohort\n",
      "NamedThing - {'close_match'} -> Cohort\n",
      "NamedThing - {'coexists_with'} -> Cohort\n",
      "NamedThing - {'regulates'} -> Cohort\n",
      "NamedThing - {'physically_interacts_with'} -> Cohort\n",
      "NamedThing - {'treats'} -> Cohort\n",
      "NamedThing - {'has_input'} -> Cohort\n",
      "NamedThing - {'causes'} -> Cohort\n",
      "NamedThing - {'associated_with'} -> Cohort\n",
      "NamedThing - {'has_phenotype'} -> Cohort\n",
      "NamedThing - {'is_sequence_variant_of'} -> Cohort\n",
      "NamedThing - {'correlated_with'} -> Cohort\n",
      "NamedThing - {'subclass_of'} -> Cohort\n",
      "NamedThing - {'has_part'} -> Cohort\n",
      "NamedThing - {'located_in'} -> Cohort\n",
      "NamedThing - {'colocalizes_with'} -> Cohort\n",
      "NamedThing - {'interacts_with'} -> Cohort\n",
      "NamedThing - {'produces'} -> Cohort\n",
      "NamedThing - {'diagnoses'} -> Cohort\n",
      "Cohort - {'diagnoses'} -> Publication\n",
      "Cohort - {'related_to'} -> Publication\n",
      "Cohort - {'treats'} -> Publication\n",
      "Cohort - {'has_input'} -> Publication\n",
      "Cohort - {'occurs_in'} -> Publication\n",
      "Cohort - {'has_part'} -> Publication\n",
      "Cohort - {'close_match'} -> Publication\n",
      "Cohort - {'subclass_of'} -> Publication\n",
      "Cohort - {'interacts_with'} -> Publication\n",
      "Cohort - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->GeographicLocation->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'related_to'} -> GeographicLocation\n",
      "NamedThing - {'diagnoses'} -> GeographicLocation\n",
      "NamedThing - {'occurs_in'} -> GeographicLocation\n",
      "NamedThing - {'affects'} -> GeographicLocation\n",
      "NamedThing - {'coexists_with'} -> GeographicLocation\n",
      "NamedThing - {'close_match'} -> GeographicLocation\n",
      "NamedThing - {'has_participant'} -> GeographicLocation\n",
      "NamedThing - {'regulates'} -> GeographicLocation\n",
      "NamedThing - {'treats'} -> GeographicLocation\n",
      "NamedThing - {'has_input'} -> GeographicLocation\n",
      "NamedThing - {'derives_from'} -> GeographicLocation\n",
      "NamedThing - {'physically_interacts_with'} -> GeographicLocation\n",
      "NamedThing - {'directly_physically_interacts_with'} -> GeographicLocation\n",
      "NamedThing - {'is_sequence_variant_of'} -> GeographicLocation\n",
      "NamedThing - {'associated_with'} -> GeographicLocation\n",
      "NamedThing - {'has_phenotype'} -> GeographicLocation\n",
      "NamedThing - {'located_in'} -> GeographicLocation\n",
      "NamedThing - {'interacts_with'} -> GeographicLocation\n",
      "NamedThing - {'produces'} -> GeographicLocation\n",
      "NamedThing - {'subclass_of'} -> GeographicLocation\n",
      "NamedThing - {'has_part'} -> GeographicLocation\n",
      "NamedThing - {'colocalizes_with'} -> GeographicLocation\n",
      "GeographicLocation - {'subclass_of'} -> Publication\n",
      "GeographicLocation - {'related_to'} -> Publication\n",
      "GeographicLocation - {'gene_associated_with_condition'} -> Publication\n",
      "GeographicLocation - {'located_in'} -> Publication\n",
      "GeographicLocation - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Agent->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'regulates'} -> Agent\n",
      "NamedThing - {'contraindicated_for'} -> Agent\n",
      "NamedThing - {'treats'} -> Agent\n",
      "NamedThing - {'has_input'} -> Agent\n",
      "NamedThing - {'causes'} -> Agent\n",
      "NamedThing - {'disrupts'} -> Agent\n",
      "NamedThing - {'associated_with'} -> Agent\n",
      "NamedThing - {'correlated_with'} -> Agent\n",
      "NamedThing - {'gene_associated_with_condition'} -> Agent\n",
      "NamedThing - {'has_phenotype'} -> Agent\n",
      "NamedThing - {'subclass_of'} -> Agent\n",
      "NamedThing - {'located_in'} -> Agent\n",
      "NamedThing - {'interacts_with'} -> Agent\n",
      "NamedThing - {'produces'} -> Agent\n",
      "NamedThing - {'has_part'} -> Agent\n",
      "NamedThing - {'prevents'} -> Agent\n",
      "NamedThing - {'predisposes'} -> Agent\n",
      "NamedThing - {'related_to'} -> Agent\n",
      "NamedThing - {'diagnoses'} -> Agent\n",
      "NamedThing - {'occurs_in'} -> Agent\n",
      "NamedThing - {'affects'} -> Agent\n",
      "NamedThing - {'manifestation_of'} -> Agent\n",
      "NamedThing - {'exacerbates'} -> Agent\n",
      "NamedThing - {'precedes'} -> Agent\n",
      "NamedThing - {'coexists_with'} -> Agent\n",
      "NamedThing - {'close_match'} -> Agent\n",
      "Agent - {'has_part'} -> Publication\n",
      "Agent - {'interacts_with'} -> Publication\n",
      "Agent - {'produces'} -> Publication\n",
      "Agent - {'predisposes'} -> Publication\n",
      "Agent - {'related_to'} -> Publication\n",
      "Agent - {'prevents'} -> Publication\n",
      "Agent - {'precedes'} -> Publication\n",
      "Agent - {'occurs_in'} -> Publication\n",
      "Agent - {'affects'} -> Publication\n",
      "Agent - {'exacerbates'} -> Publication\n",
      "Agent - {'coexists_with'} -> Publication\n",
      "Agent - {'close_match'} -> Publication\n",
      "Agent - {'subclass_of'} -> Publication\n",
      "Agent - {'regulates'} -> Publication\n",
      "Agent - {'diagnoses'} -> Publication\n",
      "Agent - {'treats'} -> Publication\n",
      "Agent - {'has_input'} -> Publication\n",
      "Agent - {'causes'} -> Publication\n",
      "Agent - {'located_in'} -> Publication\n",
      "Agent - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->ProteinDomain->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'has_input'} -> ProteinDomain\n",
      "NamedThing - {'derives_from'} -> ProteinDomain\n",
      "NamedThing - {'disrupts'} -> ProteinDomain\n",
      "NamedThing - {'associated_with'} -> ProteinDomain\n",
      "NamedThing - {'subclass_of'} -> ProteinDomain\n",
      "NamedThing - {'has_part'} -> ProteinDomain\n",
      "NamedThing - {'located_in'} -> ProteinDomain\n",
      "NamedThing - {'interacts_with'} -> ProteinDomain\n",
      "NamedThing - {'produces'} -> ProteinDomain\n",
      "NamedThing - {'expressed_in'} -> ProteinDomain\n",
      "NamedThing - {'related_to'} -> ProteinDomain\n",
      "NamedThing - {'occurs_in'} -> ProteinDomain\n",
      "NamedThing - {'affects'} -> ProteinDomain\n",
      "NamedThing - {'close_match'} -> ProteinDomain\n",
      "NamedThing - {'coexists_with'} -> ProteinDomain\n",
      "NamedThing - {'regulates'} -> ProteinDomain\n",
      "ProteinDomain - {'associated_with'} -> Publication\n",
      "ProteinDomain - {'located_in'} -> Publication\n",
      "ProteinDomain - {'related_to'} -> Publication\n",
      "ProteinDomain - {'occurs_in'} -> Publication\n",
      "ProteinDomain - {'affects'} -> Publication\n",
      "ProteinDomain - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Event->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'treats'} -> Event\n",
      "NamedThing - {'has_input'} -> Event\n",
      "NamedThing - {'causes'} -> Event\n",
      "NamedThing - {'associated_with'} -> Event\n",
      "NamedThing - {'disrupts'} -> Event\n",
      "NamedThing - {'subclass_of'} -> Event\n",
      "NamedThing - {'has_part'} -> Event\n",
      "NamedThing - {'prevents'} -> Event\n",
      "NamedThing - {'predisposes'} -> Event\n",
      "NamedThing - {'related_to'} -> Event\n",
      "NamedThing - {'diagnoses'} -> Event\n",
      "NamedThing - {'affects'} -> Event\n",
      "NamedThing - {'manifestation_of'} -> Event\n",
      "NamedThing - {'coexists_with'} -> Event\n",
      "NamedThing - {'close_match'} -> Event\n",
      "Event - {'occurs_in'} -> Publication\n",
      "Event - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->Treatment->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'treats'} -> Treatment\n",
      "NamedThing - {'has_input'} -> Treatment\n",
      "NamedThing - {'subclass_of'} -> Treatment\n",
      "NamedThing - {'has_part'} -> Treatment\n",
      "NamedThing - {'interacts_with'} -> Treatment\n",
      "NamedThing - {'contributes_to'} -> Treatment\n",
      "NamedThing - {'precedes'} -> Treatment\n",
      "NamedThing - {'related_to'} -> Treatment\n",
      "NamedThing - {'manifestation_of'} -> Treatment\n",
      "NamedThing - {'coexists_with'} -> Treatment\n",
      "NamedThing - {'close_match'} -> Treatment\n",
      "Treatment - {'subclass_of'} -> Publication\n",
      "Treatment - {'occurs_in'} -> Publication\n",
      "Treatment - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->NamedThing->GeneticInheritance->Publication\n",
      "InformationResource - {'associated_with'} -> NamedThing\n",
      "InformationResource - {'subclass_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> GeneticInheritance\n",
      "NamedThing - {'regulates'} -> GeneticInheritance\n",
      "NamedThing - {'causes'} -> GeneticInheritance\n",
      "NamedThing - {'affects'} -> GeneticInheritance\n",
      "NamedThing - {'coexists_with'} -> GeneticInheritance\n",
      "NamedThing - {'manifestation_of'} -> GeneticInheritance\n",
      "NamedThing - {'related_to'} -> GeneticInheritance\n",
      "NamedThing - {'close_match'} -> GeneticInheritance\n",
      "NamedThing - {'disrupts'} -> GeneticInheritance\n",
      "GeneticInheritance - {'occurs_in'} -> Publication\n",
      "GeneticInheritance - {'has_input'} -> Publication\n",
      "GeneticInheritance - {'located_in'} -> Publication\n",
      "GeneticInheritance - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->SmallMolecule->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'treated_by'} -> SmallMolecule\n",
      "Pathway - {'related_to'} -> SmallMolecule\n",
      "Pathway - {'occurs_together_in_literature_with'} -> SmallMolecule\n",
      "Pathway - {'interacts_with'} -> SmallMolecule\n",
      "Pathway - {'treats'} -> SmallMolecule\n",
      "Pathway - {'is_ameliorated_by'} -> SmallMolecule\n",
      "Pathway - {'regulates'} -> SmallMolecule\n",
      "Pathway - {'causes'} -> SmallMolecule\n",
      "Pathway - {'has_participant'} -> SmallMolecule\n",
      "Pathway - {'response_increased_by'} -> SmallMolecule\n",
      "Pathway - {'part_of'} -> SmallMolecule\n",
      "Pathway - {'has_capability'} -> SmallMolecule\n",
      "Pathway - {'participates_in'} -> SmallMolecule\n",
      "Pathway - {'contributes_to'} -> SmallMolecule\n",
      "Pathway - {'caused_by'} -> SmallMolecule\n",
      "Pathway - {'enabled_by'} -> SmallMolecule\n",
      "Pathway - {'has_catalyst'} -> SmallMolecule\n",
      "Pathway - {'decreases_response_to'} -> SmallMolecule\n",
      "Pathway - {'associated_with'} -> SmallMolecule\n",
      "Pathway - {'has_part'} -> SmallMolecule\n",
      "Pathway - {'is_exacerbated_by'} -> SmallMolecule\n",
      "Pathway - {'ameliorates'} -> SmallMolecule\n",
      "Pathway - {'disrupted_by'} -> SmallMolecule\n",
      "Pathway - {'actively_involves'} -> SmallMolecule\n",
      "Pathway - {'has_upstream_actor'} -> SmallMolecule\n",
      "Pathway - {'exacerbates'} -> SmallMolecule\n",
      "Pathway - {'has_active_ingredient'} -> SmallMolecule\n",
      "Pathway - {'increases_response_to'} -> SmallMolecule\n",
      "Pathway - {'primarily_composed_of'} -> SmallMolecule\n",
      "Pathway - {'regulated_by'} -> SmallMolecule\n",
      "Pathway - {'response_affected_by'} -> SmallMolecule\n",
      "Pathway - {'affects'} -> SmallMolecule\n",
      "Pathway - {'similar_to'} -> SmallMolecule\n",
      "Pathway - {'overlaps'} -> SmallMolecule\n",
      "Pathway - {'affects_response_to'} -> SmallMolecule\n",
      "Pathway - {'composed_primarily_of'} -> SmallMolecule\n",
      "Pathway - {'has_output'} -> SmallMolecule\n",
      "Pathway - {'response_decreased_by'} -> SmallMolecule\n",
      "Pathway - {'disrupts'} -> SmallMolecule\n",
      "Pathway - {'affected_by'} -> SmallMolecule\n",
      "Pathway - {'has_substrate'} -> SmallMolecule\n",
      "Pathway - {'has_input'} -> SmallMolecule\n",
      "Pathway - {'predisposes'} -> SmallMolecule\n",
      "Pathway - {'precedes'} -> SmallMolecule\n",
      "Pathway - {'occurs_in'} -> SmallMolecule\n",
      "Pathway - {'coexists_with'} -> SmallMolecule\n",
      "Pathway - {'manifestation_of'} -> SmallMolecule\n",
      "Pathway - {'close_match'} -> SmallMolecule\n",
      "Pathway - {'subclass_of'} -> SmallMolecule\n",
      "Pathway - {'disease_has_basis_in'} -> SmallMolecule\n",
      "Pathway - {'derives_from'} -> SmallMolecule\n",
      "Pathway - {'in_taxon'} -> SmallMolecule\n",
      "Pathway - {'physically_interacts_with'} -> SmallMolecule\n",
      "Pathway - {'gene_associated_with_condition'} -> SmallMolecule\n",
      "Pathway - {'produces'} -> SmallMolecule\n",
      "Pathway - {'located_in'} -> SmallMolecule\n",
      "Pathway - {'colocalizes_with'} -> SmallMolecule\n",
      "SmallMolecule - {'correlated_with'} -> Publication\n",
      "SmallMolecule - {'disrupts'} -> Publication\n",
      "SmallMolecule - {'gene_associated_with_condition'} -> Publication\n",
      "SmallMolecule - {'located_in'} -> Publication\n",
      "SmallMolecule - {'regulates'} -> Publication\n",
      "SmallMolecule - {'has_part'} -> Publication\n",
      "SmallMolecule - {'physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'predisposes'} -> Publication\n",
      "SmallMolecule - {'related_to'} -> Publication\n",
      "SmallMolecule - {'causes'} -> Publication\n",
      "SmallMolecule - {'directly_physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'precedes'} -> Publication\n",
      "SmallMolecule - {'close_match'} -> Publication\n",
      "SmallMolecule - {'associated_with'} -> Publication\n",
      "SmallMolecule - {'has_metabolite'} -> Publication\n",
      "SmallMolecule - {'subclass_of'} -> Publication\n",
      "SmallMolecule - {'interacts_with'} -> Publication\n",
      "SmallMolecule - {'treats'} -> Publication\n",
      "SmallMolecule - {'has_input'} -> Publication\n",
      "SmallMolecule - {'occurs_in'} -> Publication\n",
      "SmallMolecule - {'affects'} -> Publication\n",
      "SmallMolecule - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Disease->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'genetic_association'} -> Disease\n",
      "Pathway - {'occurs_together_in_literature_with'} -> Disease\n",
      "Pathway - {'causes'} -> Disease\n",
      "Pathway - {'subclass_of'} -> Disease\n",
      "Pathway - {'part_of'} -> Disease\n",
      "Pathway - {'caused_by'} -> Disease\n",
      "Pathway - {'related_to'} -> Disease\n",
      "Pathway - {'participates_in'} -> Disease\n",
      "Pathway - {'disrupted_by'} -> Disease\n",
      "Pathway - {'overlaps'} -> Disease\n",
      "Pathway - {'actively_involved_in'} -> Disease\n",
      "Pathway - {'decreases_response_to'} -> Disease\n",
      "Pathway - {'associated_with'} -> Disease\n",
      "Pathway - {'has_part'} -> Disease\n",
      "Pathway - {'is_exacerbated_by'} -> Disease\n",
      "Pathway - {'precedes'} -> Disease\n",
      "Pathway - {'capable_of'} -> Disease\n",
      "Pathway - {'has_upstream_actor'} -> Disease\n",
      "Pathway - {'treated_by'} -> Disease\n",
      "Pathway - {'process_negatively_regulates_process'} -> Disease\n",
      "Pathway - {'exacerbates'} -> Disease\n",
      "Pathway - {'has_active_ingredient'} -> Disease\n",
      "Pathway - {'increases_response_to'} -> Disease\n",
      "Pathway - {'primarily_composed_of'} -> Disease\n",
      "Pathway - {'regulated_by'} -> Disease\n",
      "Pathway - {'temporally_related_to'} -> Disease\n",
      "Pathway - {'response_affected_by'} -> Disease\n",
      "Pathway - {'affects'} -> Disease\n",
      "Pathway - {'process_regulates_process'} -> Disease\n",
      "Pathway - {'similar_to'} -> Disease\n",
      "Pathway - {'affects_response_to'} -> Disease\n",
      "Pathway - {'condition_associated_with_gene'} -> Disease\n",
      "Pathway - {'process_positively_regulates_process'} -> Disease\n",
      "Pathway - {'composed_primarily_of'} -> Disease\n",
      "Pathway - {'gene_associated_with_condition'} -> Disease\n",
      "Pathway - {'response_decreased_by'} -> Disease\n",
      "Pathway - {'disrupts'} -> Disease\n",
      "Pathway - {'affected_by'} -> Disease\n",
      "Pathway - {'interacts_with'} -> Disease\n",
      "Pathway - {'ameliorates'} -> Disease\n",
      "Pathway - {'treats'} -> Disease\n",
      "Pathway - {'is_ameliorated_by'} -> Disease\n",
      "Pathway - {'regulates'} -> Disease\n",
      "Pathway - {'response_increased_by'} -> Disease\n",
      "Pathway - {'contributes_to'} -> Disease\n",
      "Pathway - {'has_participant'} -> Disease\n",
      "Pathway - {'contraindicated_for'} -> Disease\n",
      "Pathway - {'diagnoses'} -> Disease\n",
      "Pathway - {'has_input'} -> Disease\n",
      "Pathway - {'has_output'} -> Disease\n",
      "Pathway - {'has_phenotype'} -> Disease\n",
      "Pathway - {'biomarker_for'} -> Disease\n",
      "Pathway - {'located_in'} -> Disease\n",
      "Pathway - {'produces'} -> Disease\n",
      "Pathway - {'correlated_with'} -> Disease\n",
      "Pathway - {'predisposes'} -> Disease\n",
      "Pathway - {'prevents'} -> Disease\n",
      "Pathway - {'occurs_in'} -> Disease\n",
      "Pathway - {'manifestation_of'} -> Disease\n",
      "Pathway - {'coexists_with'} -> Disease\n",
      "Pathway - {'close_match'} -> Disease\n",
      "Disease - {'correlated_with'} -> Publication\n",
      "Disease - {'associated_with'} -> Publication\n",
      "Disease - {'gene_associated_with_condition'} -> Publication\n",
      "Disease - {'subclass_of'} -> Publication\n",
      "Disease - {'has_part'} -> Publication\n",
      "Disease - {'interacts_with'} -> Publication\n",
      "Disease - {'produces'} -> Publication\n",
      "Disease - {'predisposes'} -> Publication\n",
      "Disease - {'related_to'} -> Publication\n",
      "Disease - {'prevents'} -> Publication\n",
      "Disease - {'diagnoses'} -> Publication\n",
      "Disease - {'contributes_to'} -> Publication\n",
      "Disease - {'manifestation_of'} -> Publication\n",
      "Disease - {'occurs_in'} -> Publication\n",
      "Disease - {'affects'} -> Publication\n",
      "Disease - {'has_input'} -> Publication\n",
      "Disease - {'exacerbates'} -> Publication\n",
      "Disease - {'coexists_with'} -> Publication\n",
      "Disease - {'close_match'} -> Publication\n",
      "Disease - {'disrupts'} -> Publication\n",
      "Disease - {'regulates'} -> Publication\n",
      "Disease - {'biomarker_for'} -> Publication\n",
      "Disease - {'located_in'} -> Publication\n",
      "Disease - {'treats'} -> Publication\n",
      "Disease - {'causes'} -> Publication\n",
      "Disease - {'disease_has_location'} -> Publication\n",
      "Disease - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->PhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'genetic_association'} -> PhenotypicFeature\n",
      "Pathway - {'occurs_together_in_literature_with'} -> PhenotypicFeature\n",
      "Pathway - {'affected_by'} -> PhenotypicFeature\n",
      "Pathway - {'precedes'} -> PhenotypicFeature\n",
      "Pathway - {'related_to'} -> PhenotypicFeature\n",
      "Pathway - {'temporally_related_to'} -> PhenotypicFeature\n",
      "Pathway - {'treats'} -> PhenotypicFeature\n",
      "Pathway - {'is_ameliorated_by'} -> PhenotypicFeature\n",
      "Pathway - {'causes'} -> PhenotypicFeature\n",
      "Pathway - {'part_of'} -> PhenotypicFeature\n",
      "Pathway - {'positively_regulates'} -> PhenotypicFeature\n",
      "Pathway - {'participates_in'} -> PhenotypicFeature\n",
      "Pathway - {'contributes_to'} -> PhenotypicFeature\n",
      "Pathway - {'preceded_by'} -> PhenotypicFeature\n",
      "Pathway - {'has_participant'} -> PhenotypicFeature\n",
      "Pathway - {'decreases_response_to'} -> PhenotypicFeature\n",
      "Pathway - {'has_part'} -> PhenotypicFeature\n",
      "Pathway - {'is_exacerbated_by'} -> PhenotypicFeature\n",
      "Pathway - {'negatively_regulates'} -> PhenotypicFeature\n",
      "Pathway - {'interacts_with'} -> PhenotypicFeature\n",
      "Pathway - {'ameliorates'} -> PhenotypicFeature\n",
      "Pathway - {'regulates'} -> PhenotypicFeature\n",
      "Pathway - {'response_increased_by'} -> PhenotypicFeature\n",
      "Pathway - {'has_capability'} -> PhenotypicFeature\n",
      "Pathway - {'capable_of'} -> PhenotypicFeature\n",
      "Pathway - {'caused_by'} -> PhenotypicFeature\n",
      "Pathway - {'actively_involves'} -> PhenotypicFeature\n",
      "Pathway - {'actively_involved_in'} -> PhenotypicFeature\n",
      "Pathway - {'associated_with'} -> PhenotypicFeature\n",
      "Pathway - {'affects'} -> PhenotypicFeature\n",
      "Pathway - {'similar_to'} -> PhenotypicFeature\n",
      "Pathway - {'subclass_of'} -> PhenotypicFeature\n",
      "Pathway - {'overlaps'} -> PhenotypicFeature\n",
      "Pathway - {'affects_response_to'} -> PhenotypicFeature\n",
      "Pathway - {'disrupted_by'} -> PhenotypicFeature\n",
      "Pathway - {'process_positively_regulates_process'} -> PhenotypicFeature\n",
      "Pathway - {'composed_primarily_of'} -> PhenotypicFeature\n",
      "Pathway - {'has_upstream_actor'} -> PhenotypicFeature\n",
      "Pathway - {'treated_by'} -> PhenotypicFeature\n",
      "Pathway - {'gene_associated_with_condition'} -> PhenotypicFeature\n",
      "Pathway - {'process_negatively_regulates_process'} -> PhenotypicFeature\n",
      "Pathway - {'exacerbates'} -> PhenotypicFeature\n",
      "Pathway - {'has_active_ingredient'} -> PhenotypicFeature\n",
      "Pathway - {'increases_response_to'} -> PhenotypicFeature\n",
      "Pathway - {'primarily_composed_of'} -> PhenotypicFeature\n",
      "Pathway - {'regulated_by'} -> PhenotypicFeature\n",
      "Pathway - {'response_affected_by'} -> PhenotypicFeature\n",
      "Pathway - {'response_decreased_by'} -> PhenotypicFeature\n",
      "Pathway - {'process_regulates_process'} -> PhenotypicFeature\n",
      "Pathway - {'disrupts'} -> PhenotypicFeature\n",
      "Pathway - {'has_input'} -> PhenotypicFeature\n",
      "Pathway - {'has_output'} -> PhenotypicFeature\n",
      "Pathway - {'has_phenotype'} -> PhenotypicFeature\n",
      "Pathway - {'biomarker_for'} -> PhenotypicFeature\n",
      "Pathway - {'correlated_with'} -> PhenotypicFeature\n",
      "Pathway - {'located_in'} -> PhenotypicFeature\n",
      "Pathway - {'produces'} -> PhenotypicFeature\n",
      "Pathway - {'predisposes'} -> PhenotypicFeature\n",
      "Pathway - {'prevents'} -> PhenotypicFeature\n",
      "Pathway - {'occurs_in'} -> PhenotypicFeature\n",
      "Pathway - {'manifestation_of'} -> PhenotypicFeature\n",
      "Pathway - {'coexists_with'} -> PhenotypicFeature\n",
      "Pathway - {'close_match'} -> PhenotypicFeature\n",
      "Pathway - {'contraindicated_for'} -> PhenotypicFeature\n",
      "Pathway - {'diagnoses'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Publication\n",
      "PhenotypicFeature - {'occurs_in'} -> Publication\n",
      "PhenotypicFeature - {'affects'} -> Publication\n",
      "PhenotypicFeature - {'close_match'} -> Publication\n",
      "PhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "PhenotypicFeature - {'exacerbates'} -> Publication\n",
      "PhenotypicFeature - {'coexists_with'} -> Publication\n",
      "PhenotypicFeature - {'subclass_of'} -> Publication\n",
      "PhenotypicFeature - {'regulates'} -> Publication\n",
      "PhenotypicFeature - {'diagnoses'} -> Publication\n",
      "PhenotypicFeature - {'treats'} -> Publication\n",
      "PhenotypicFeature - {'has_input'} -> Publication\n",
      "PhenotypicFeature - {'causes'} -> Publication\n",
      "PhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "PhenotypicFeature - {'disrupts'} -> Publication\n",
      "PhenotypicFeature - {'associated_with'} -> Publication\n",
      "PhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "PhenotypicFeature - {'has_part'} -> Publication\n",
      "PhenotypicFeature - {'located_in'} -> Publication\n",
      "PhenotypicFeature - {'interacts_with'} -> Publication\n",
      "PhenotypicFeature - {'produces'} -> Publication\n",
      "PhenotypicFeature - {'prevents'} -> Publication\n",
      "PhenotypicFeature - {'contributes_to'} -> Publication\n",
      "PhenotypicFeature - {'predisposes'} -> Publication\n",
      "PhenotypicFeature - {'precedes'} -> Publication\n",
      "PhenotypicFeature - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->AnatomicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> AnatomicalEntity\n",
      "Pathway - {'related_to'} -> AnatomicalEntity\n",
      "Pathway - {'caused_by'} -> AnatomicalEntity\n",
      "Pathway - {'enabled_by'} -> AnatomicalEntity\n",
      "Pathway - {'has_part'} -> AnatomicalEntity\n",
      "Pathway - {'has_participant'} -> AnatomicalEntity\n",
      "Pathway - {'occurs_in'} -> AnatomicalEntity\n",
      "Pathway - {'overlaps'} -> AnatomicalEntity\n",
      "Pathway - {'temporally_related_to'} -> AnatomicalEntity\n",
      "Pathway - {'actively_involved_in'} -> AnatomicalEntity\n",
      "Pathway - {'associated_with'} -> AnatomicalEntity\n",
      "Pathway - {'affects'} -> AnatomicalEntity\n",
      "Pathway - {'similar_to'} -> AnatomicalEntity\n",
      "Pathway - {'subclass_of'} -> AnatomicalEntity\n",
      "Pathway - {'affects_response_to'} -> AnatomicalEntity\n",
      "Pathway - {'process_positively_regulates_process'} -> AnatomicalEntity\n",
      "Pathway - {'composed_primarily_of'} -> AnatomicalEntity\n",
      "Pathway - {'has_upstream_actor'} -> AnatomicalEntity\n",
      "Pathway - {'process_negatively_regulates_process'} -> AnatomicalEntity\n",
      "Pathway - {'exacerbates'} -> AnatomicalEntity\n",
      "Pathway - {'has_active_ingredient'} -> AnatomicalEntity\n",
      "Pathway - {'increases_response_to'} -> AnatomicalEntity\n",
      "Pathway - {'primarily_composed_of'} -> AnatomicalEntity\n",
      "Pathway - {'has_output'} -> AnatomicalEntity\n",
      "Pathway - {'process_regulates_process'} -> AnatomicalEntity\n",
      "Pathway - {'disrupts'} -> AnatomicalEntity\n",
      "Pathway - {'affected_by'} -> AnatomicalEntity\n",
      "Pathway - {'treats'} -> AnatomicalEntity\n",
      "Pathway - {'causes'} -> AnatomicalEntity\n",
      "Pathway - {'part_of'} -> AnatomicalEntity\n",
      "Pathway - {'positively_regulates'} -> AnatomicalEntity\n",
      "Pathway - {'participates_in'} -> AnatomicalEntity\n",
      "Pathway - {'contributes_to'} -> AnatomicalEntity\n",
      "Pathway - {'has_input'} -> AnatomicalEntity\n",
      "Pathway - {'preceded_by'} -> AnatomicalEntity\n",
      "Pathway - {'decreases_response_to'} -> AnatomicalEntity\n",
      "Pathway - {'negatively_regulates'} -> AnatomicalEntity\n",
      "Pathway - {'has_substrate'} -> AnatomicalEntity\n",
      "Pathway - {'precedes'} -> AnatomicalEntity\n",
      "Pathway - {'interacts_with'} -> AnatomicalEntity\n",
      "Pathway - {'ameliorates'} -> AnatomicalEntity\n",
      "Pathway - {'regulates'} -> AnatomicalEntity\n",
      "Pathway - {'has_capability'} -> AnatomicalEntity\n",
      "Pathway - {'capable_of'} -> AnatomicalEntity\n",
      "Pathway - {'actively_involves'} -> AnatomicalEntity\n",
      "Pathway - {'has_catalyst'} -> AnatomicalEntity\n",
      "Pathway - {'derives_from'} -> AnatomicalEntity\n",
      "Pathway - {'disease_has_location'} -> AnatomicalEntity\n",
      "Pathway - {'has_phenotype'} -> AnatomicalEntity\n",
      "Pathway - {'expressed_in'} -> AnatomicalEntity\n",
      "Pathway - {'located_in'} -> AnatomicalEntity\n",
      "Pathway - {'produces'} -> AnatomicalEntity\n",
      "Pathway - {'predisposes'} -> AnatomicalEntity\n",
      "Pathway - {'prevents'} -> AnatomicalEntity\n",
      "Pathway - {'manifestation_of'} -> AnatomicalEntity\n",
      "Pathway - {'coexists_with'} -> AnatomicalEntity\n",
      "Pathway - {'close_match'} -> AnatomicalEntity\n",
      "Pathway - {'same_as'} -> AnatomicalEntity\n",
      "AnatomicalEntity - {'develops_from'} -> Publication\n",
      "AnatomicalEntity - {'interacts_with'} -> Publication\n",
      "AnatomicalEntity - {'produces'} -> Publication\n",
      "AnatomicalEntity - {'has_part'} -> Publication\n",
      "AnatomicalEntity - {'diagnoses'} -> Publication\n",
      "AnatomicalEntity - {'related_to'} -> Publication\n",
      "AnatomicalEntity - {'affects'} -> Publication\n",
      "AnatomicalEntity - {'manifestation_of'} -> Publication\n",
      "AnatomicalEntity - {'has_input'} -> Publication\n",
      "AnatomicalEntity - {'occurs_in'} -> Publication\n",
      "AnatomicalEntity - {'coexists_with'} -> Publication\n",
      "AnatomicalEntity - {'close_match'} -> Publication\n",
      "AnatomicalEntity - {'disrupts'} -> Publication\n",
      "AnatomicalEntity - {'located_in'} -> Publication\n",
      "AnatomicalEntity - {'regulates'} -> Publication\n",
      "AnatomicalEntity - {'treats'} -> Publication\n",
      "AnatomicalEntity - {'causes'} -> Publication\n",
      "AnatomicalEntity - {'associated_with'} -> Publication\n",
      "AnatomicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "AnatomicalEntity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->BiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> BiologicalProcess\n",
      "Pathway - {'subclass_of'} -> BiologicalProcess\n",
      "Pathway - {'regulated_by'} -> BiologicalProcess\n",
      "Pathway - {'superclass_of'} -> BiologicalProcess\n",
      "Pathway - {'caused_by'} -> BiologicalProcess\n",
      "Pathway - {'overlaps'} -> BiologicalProcess\n",
      "Pathway - {'part_of'} -> BiologicalProcess\n",
      "Pathway - {'positively_regulates'} -> BiologicalProcess\n",
      "Pathway - {'preceded_by'} -> BiologicalProcess\n",
      "Pathway - {'precedes'} -> BiologicalProcess\n",
      "Pathway - {'process_negatively_regulates_process'} -> BiologicalProcess\n",
      "Pathway - {'process_positively_regulates_process'} -> BiologicalProcess\n",
      "Pathway - {'process_regulates_process'} -> BiologicalProcess\n",
      "Pathway - {'regulates'} -> BiologicalProcess\n",
      "Pathway - {'related_to'} -> BiologicalProcess\n",
      "Pathway - {'temporally_related_to'} -> BiologicalProcess\n",
      "Pathway - {'affects'} -> BiologicalProcess\n",
      "Pathway - {'has_part'} -> BiologicalProcess\n",
      "Pathway - {'negatively_regulates'} -> BiologicalProcess\n",
      "Pathway - {'causes'} -> BiologicalProcess\n",
      "Pathway - {'composed_primarily_of'} -> BiologicalProcess\n",
      "Pathway - {'enabled_by'} -> BiologicalProcess\n",
      "Pathway - {'is_input_of'} -> BiologicalProcess\n",
      "Pathway - {'affected_by'} -> BiologicalProcess\n",
      "Pathway - {'has_substrate'} -> BiologicalProcess\n",
      "Pathway - {'interacts_with'} -> BiologicalProcess\n",
      "Pathway - {'ameliorates'} -> BiologicalProcess\n",
      "Pathway - {'is_substrate_of'} -> BiologicalProcess\n",
      "Pathway - {'occurs_in'} -> BiologicalProcess\n",
      "Pathway - {'has_capability'} -> BiologicalProcess\n",
      "Pathway - {'participates_in'} -> BiologicalProcess\n",
      "Pathway - {'capable_of'} -> BiologicalProcess\n",
      "Pathway - {'contributes_to'} -> BiologicalProcess\n",
      "Pathway - {'actively_involves'} -> BiologicalProcess\n",
      "Pathway - {'has_input'} -> BiologicalProcess\n",
      "Pathway - {'has_participant'} -> BiologicalProcess\n",
      "Pathway - {'has_catalyst'} -> BiologicalProcess\n",
      "Pathway - {'decreases_response_to'} -> BiologicalProcess\n",
      "Pathway - {'associated_with'} -> BiologicalProcess\n",
      "Pathway - {'enables'} -> BiologicalProcess\n",
      "Pathway - {'catalyzes'} -> BiologicalProcess\n",
      "Pathway - {'has_upstream_actor'} -> BiologicalProcess\n",
      "Pathway - {'exacerbates'} -> BiologicalProcess\n",
      "Pathway - {'has_active_ingredient'} -> BiologicalProcess\n",
      "Pathway - {'increases_response_to'} -> BiologicalProcess\n",
      "Pathway - {'primarily_composed_of'} -> BiologicalProcess\n",
      "Pathway - {'actively_involved_in'} -> BiologicalProcess\n",
      "Pathway - {'has_output'} -> BiologicalProcess\n",
      "Pathway - {'disrupts'} -> BiologicalProcess\n",
      "Pathway - {'similar_to'} -> BiologicalProcess\n",
      "Pathway - {'is_output_of'} -> BiologicalProcess\n",
      "Pathway - {'affects_response_to'} -> BiologicalProcess\n",
      "Pathway - {'treats'} -> BiologicalProcess\n",
      "Pathway - {'gene_associated_with_condition'} -> BiologicalProcess\n",
      "Pathway - {'located_in'} -> BiologicalProcess\n",
      "Pathway - {'colocalizes_with'} -> BiologicalProcess\n",
      "Pathway - {'produces'} -> BiologicalProcess\n",
      "Pathway - {'prevents'} -> BiologicalProcess\n",
      "Pathway - {'predisposes'} -> BiologicalProcess\n",
      "Pathway - {'manifestation_of'} -> BiologicalProcess\n",
      "Pathway - {'coexists_with'} -> BiologicalProcess\n",
      "Pathway - {'close_match'} -> BiologicalProcess\n",
      "Pathway - {'contraindicated_for'} -> BiologicalProcess\n",
      "Pathway - {'physically_interacts_with'} -> BiologicalProcess\n",
      "Pathway - {'same_as'} -> BiologicalProcess\n",
      "BiologicalProcess - {'correlated_with'} -> Publication\n",
      "BiologicalProcess - {'disrupts'} -> Publication\n",
      "BiologicalProcess - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalProcess - {'regulates'} -> Publication\n",
      "BiologicalProcess - {'biomarker_for'} -> Publication\n",
      "BiologicalProcess - {'has_part'} -> Publication\n",
      "BiologicalProcess - {'located_in'} -> Publication\n",
      "BiologicalProcess - {'predisposes'} -> Publication\n",
      "BiologicalProcess - {'precedes'} -> Publication\n",
      "BiologicalProcess - {'prevents'} -> Publication\n",
      "BiologicalProcess - {'causes'} -> Publication\n",
      "BiologicalProcess - {'close_match'} -> Publication\n",
      "BiologicalProcess - {'associated_with'} -> Publication\n",
      "BiologicalProcess - {'produces'} -> Publication\n",
      "BiologicalProcess - {'has_participant'} -> Publication\n",
      "BiologicalProcess - {'subclass_of'} -> Publication\n",
      "BiologicalProcess - {'related_to'} -> Publication\n",
      "BiologicalProcess - {'affects'} -> Publication\n",
      "BiologicalProcess - {'treats'} -> Publication\n",
      "BiologicalProcess - {'manifestation_of'} -> Publication\n",
      "BiologicalProcess - {'has_input'} -> Publication\n",
      "BiologicalProcess - {'has_output'} -> Publication\n",
      "BiologicalProcess - {'occurs_in'} -> Publication\n",
      "BiologicalProcess - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Cell->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> Cell\n",
      "Pathway - {'has_capability'} -> Cell\n",
      "Pathway - {'overlaps'} -> Cell\n",
      "Pathway - {'related_to'} -> Cell\n",
      "Pathway - {'occurs_in'} -> Cell\n",
      "Pathway - {'has_participant'} -> Cell\n",
      "Pathway - {'has_upstream_actor'} -> Cell\n",
      "Pathway - {'has_active_ingredient'} -> Cell\n",
      "Pathway - {'has_catalyst'} -> Cell\n",
      "Pathway - {'primarily_composed_of'} -> Cell\n",
      "Pathway - {'temporally_related_to'} -> Cell\n",
      "Pathway - {'actively_involved_in'} -> Cell\n",
      "Pathway - {'affects'} -> Cell\n",
      "Pathway - {'similar_to'} -> Cell\n",
      "Pathway - {'subclass_of'} -> Cell\n",
      "Pathway - {'affects_response_to'} -> Cell\n",
      "Pathway - {'composed_primarily_of'} -> Cell\n",
      "Pathway - {'exacerbates'} -> Cell\n",
      "Pathway - {'increases_response_to'} -> Cell\n",
      "Pathway - {'has_output'} -> Cell\n",
      "Pathway - {'disrupts'} -> Cell\n",
      "Pathway - {'affected_by'} -> Cell\n",
      "Pathway - {'has_substrate'} -> Cell\n",
      "Pathway - {'interacts_with'} -> Cell\n",
      "Pathway - {'treats'} -> Cell\n",
      "Pathway - {'regulates'} -> Cell\n",
      "Pathway - {'causes'} -> Cell\n",
      "Pathway - {'part_of'} -> Cell\n",
      "Pathway - {'participates_in'} -> Cell\n",
      "Pathway - {'contributes_to'} -> Cell\n",
      "Pathway - {'has_input'} -> Cell\n",
      "Pathway - {'preceded_by'} -> Cell\n",
      "Pathway - {'enabled_by'} -> Cell\n",
      "Pathway - {'decreases_response_to'} -> Cell\n",
      "Pathway - {'associated_with'} -> Cell\n",
      "Pathway - {'has_part'} -> Cell\n",
      "Pathway - {'precedes'} -> Cell\n",
      "Pathway - {'ameliorates'} -> Cell\n",
      "Pathway - {'capable_of'} -> Cell\n",
      "Pathway - {'caused_by'} -> Cell\n",
      "Pathway - {'actively_involves'} -> Cell\n",
      "Pathway - {'located_in'} -> Cell\n",
      "Pathway - {'predisposes'} -> Cell\n",
      "Pathway - {'coexists_with'} -> Cell\n",
      "Pathway - {'manifestation_of'} -> Cell\n",
      "Pathway - {'close_match'} -> Cell\n",
      "Pathway - {'derives_from'} -> Cell\n",
      "Pathway - {'disease_has_location'} -> Cell\n",
      "Pathway - {'produces'} -> Cell\n",
      "Cell - {'subclass_of'} -> Publication\n",
      "Cell - {'has_part'} -> Publication\n",
      "Cell - {'produces'} -> Publication\n",
      "Cell - {'related_to'} -> Publication\n",
      "Cell - {'occurs_in'} -> Publication\n",
      "Cell - {'affects'} -> Publication\n",
      "Cell - {'coexists_with'} -> Publication\n",
      "Cell - {'close_match'} -> Publication\n",
      "Cell - {'located_in'} -> Publication\n",
      "Cell - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->CellularComponent->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> CellularComponent\n",
      "Pathway - {'causes'} -> CellularComponent\n",
      "Pathway - {'has_capability'} -> CellularComponent\n",
      "Pathway - {'overlaps'} -> CellularComponent\n",
      "Pathway - {'related_to'} -> CellularComponent\n",
      "Pathway - {'caused_by'} -> CellularComponent\n",
      "Pathway - {'has_participant'} -> CellularComponent\n",
      "Pathway - {'occurs_in'} -> CellularComponent\n",
      "Pathway - {'preceded_by'} -> CellularComponent\n",
      "Pathway - {'has_catalyst'} -> CellularComponent\n",
      "Pathway - {'temporally_related_to'} -> CellularComponent\n",
      "Pathway - {'actively_involved_in'} -> CellularComponent\n",
      "Pathway - {'associated_with'} -> CellularComponent\n",
      "Pathway - {'affects'} -> CellularComponent\n",
      "Pathway - {'similar_to'} -> CellularComponent\n",
      "Pathway - {'affects_response_to'} -> CellularComponent\n",
      "Pathway - {'process_positively_regulates_process'} -> CellularComponent\n",
      "Pathway - {'composed_primarily_of'} -> CellularComponent\n",
      "Pathway - {'has_upstream_actor'} -> CellularComponent\n",
      "Pathway - {'process_negatively_regulates_process'} -> CellularComponent\n",
      "Pathway - {'exacerbates'} -> CellularComponent\n",
      "Pathway - {'has_active_ingredient'} -> CellularComponent\n",
      "Pathway - {'increases_response_to'} -> CellularComponent\n",
      "Pathway - {'primarily_composed_of'} -> CellularComponent\n",
      "Pathway - {'has_output'} -> CellularComponent\n",
      "Pathway - {'process_regulates_process'} -> CellularComponent\n",
      "Pathway - {'disrupts'} -> CellularComponent\n",
      "Pathway - {'affected_by'} -> CellularComponent\n",
      "Pathway - {'treats'} -> CellularComponent\n",
      "Pathway - {'part_of'} -> CellularComponent\n",
      "Pathway - {'positively_regulates'} -> CellularComponent\n",
      "Pathway - {'participates_in'} -> CellularComponent\n",
      "Pathway - {'contributes_to'} -> CellularComponent\n",
      "Pathway - {'has_input'} -> CellularComponent\n",
      "Pathway - {'enabled_by'} -> CellularComponent\n",
      "Pathway - {'decreases_response_to'} -> CellularComponent\n",
      "Pathway - {'has_part'} -> CellularComponent\n",
      "Pathway - {'negatively_regulates'} -> CellularComponent\n",
      "Pathway - {'has_substrate'} -> CellularComponent\n",
      "Pathway - {'precedes'} -> CellularComponent\n",
      "Pathway - {'interacts_with'} -> CellularComponent\n",
      "Pathway - {'ameliorates'} -> CellularComponent\n",
      "Pathway - {'regulates'} -> CellularComponent\n",
      "Pathway - {'capable_of'} -> CellularComponent\n",
      "Pathway - {'actively_involves'} -> CellularComponent\n",
      "Pathway - {'gene_associated_with_condition'} -> CellularComponent\n",
      "Pathway - {'produces'} -> CellularComponent\n",
      "Pathway - {'expressed_in'} -> CellularComponent\n",
      "Pathway - {'located_in'} -> CellularComponent\n",
      "Pathway - {'predisposes'} -> CellularComponent\n",
      "Pathway - {'manifestation_of'} -> CellularComponent\n",
      "Pathway - {'coexists_with'} -> CellularComponent\n",
      "Pathway - {'close_match'} -> CellularComponent\n",
      "Pathway - {'subclass_of'} -> CellularComponent\n",
      "Pathway - {'same_as'} -> CellularComponent\n",
      "CellularComponent - {'subclass_of'} -> Publication\n",
      "CellularComponent - {'produces'} -> Publication\n",
      "CellularComponent - {'has_part'} -> Publication\n",
      "CellularComponent - {'related_to'} -> Publication\n",
      "CellularComponent - {'occurs_in'} -> Publication\n",
      "CellularComponent - {'affects'} -> Publication\n",
      "CellularComponent - {'regulates'} -> Publication\n",
      "CellularComponent - {'disrupts'} -> Publication\n",
      "CellularComponent - {'located_in'} -> Publication\n",
      "CellularComponent - {'associated_with'} -> Publication\n",
      "CellularComponent - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->ChemicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> ChemicalEntity\n",
      "Pathway - {'causes'} -> ChemicalEntity\n",
      "Pathway - {'overlaps'} -> ChemicalEntity\n",
      "Pathway - {'positively_regulates'} -> ChemicalEntity\n",
      "Pathway - {'preceded_by'} -> ChemicalEntity\n",
      "Pathway - {'precedes'} -> ChemicalEntity\n",
      "Pathway - {'process_positively_regulates_process'} -> ChemicalEntity\n",
      "Pathway - {'process_regulates_process'} -> ChemicalEntity\n",
      "Pathway - {'regulates'} -> ChemicalEntity\n",
      "Pathway - {'related_to'} -> ChemicalEntity\n",
      "Pathway - {'temporally_related_to'} -> ChemicalEntity\n",
      "Pathway - {'affects'} -> ChemicalEntity\n",
      "Pathway - {'caused_by'} -> ChemicalEntity\n",
      "Pathway - {'enabled_by'} -> ChemicalEntity\n",
      "Pathway - {'has_input'} -> ChemicalEntity\n",
      "Pathway - {'has_part'} -> ChemicalEntity\n",
      "Pathway - {'has_participant'} -> ChemicalEntity\n",
      "Pathway - {'associated_with'} -> ChemicalEntity\n",
      "Pathway - {'affects_response_to'} -> ChemicalEntity\n",
      "Pathway - {'disrupted_by'} -> ChemicalEntity\n",
      "Pathway - {'composed_primarily_of'} -> ChemicalEntity\n",
      "Pathway - {'has_upstream_actor'} -> ChemicalEntity\n",
      "Pathway - {'treated_by'} -> ChemicalEntity\n",
      "Pathway - {'process_negatively_regulates_process'} -> ChemicalEntity\n",
      "Pathway - {'exacerbates'} -> ChemicalEntity\n",
      "Pathway - {'has_active_ingredient'} -> ChemicalEntity\n",
      "Pathway - {'increases_response_to'} -> ChemicalEntity\n",
      "Pathway - {'primarily_composed_of'} -> ChemicalEntity\n",
      "Pathway - {'regulated_by'} -> ChemicalEntity\n",
      "Pathway - {'actively_involved_in'} -> ChemicalEntity\n",
      "Pathway - {'has_output'} -> ChemicalEntity\n",
      "Pathway - {'response_affected_by'} -> ChemicalEntity\n",
      "Pathway - {'response_decreased_by'} -> ChemicalEntity\n",
      "Pathway - {'disrupts'} -> ChemicalEntity\n",
      "Pathway - {'similar_to'} -> ChemicalEntity\n",
      "Pathway - {'affected_by'} -> ChemicalEntity\n",
      "Pathway - {'treats'} -> ChemicalEntity\n",
      "Pathway - {'is_ameliorated_by'} -> ChemicalEntity\n",
      "Pathway - {'participates_in'} -> ChemicalEntity\n",
      "Pathway - {'contributes_to'} -> ChemicalEntity\n",
      "Pathway - {'decreases_response_to'} -> ChemicalEntity\n",
      "Pathway - {'is_exacerbated_by'} -> ChemicalEntity\n",
      "Pathway - {'negatively_regulates'} -> ChemicalEntity\n",
      "Pathway - {'has_substrate'} -> ChemicalEntity\n",
      "Pathway - {'interacts_with'} -> ChemicalEntity\n",
      "Pathway - {'ameliorates'} -> ChemicalEntity\n",
      "Pathway - {'response_increased_by'} -> ChemicalEntity\n",
      "Pathway - {'part_of'} -> ChemicalEntity\n",
      "Pathway - {'occurs_in'} -> ChemicalEntity\n",
      "Pathway - {'has_capability'} -> ChemicalEntity\n",
      "Pathway - {'capable_of'} -> ChemicalEntity\n",
      "Pathway - {'actively_involves'} -> ChemicalEntity\n",
      "Pathway - {'has_catalyst'} -> ChemicalEntity\n",
      "Pathway - {'gene_associated_with_condition'} -> ChemicalEntity\n",
      "Pathway - {'located_in'} -> ChemicalEntity\n",
      "Pathway - {'colocalizes_with'} -> ChemicalEntity\n",
      "Pathway - {'produces'} -> ChemicalEntity\n",
      "Pathway - {'predisposes'} -> ChemicalEntity\n",
      "Pathway - {'manifestation_of'} -> ChemicalEntity\n",
      "Pathway - {'coexists_with'} -> ChemicalEntity\n",
      "Pathway - {'close_match'} -> ChemicalEntity\n",
      "Pathway - {'subclass_of'} -> ChemicalEntity\n",
      "Pathway - {'disease_has_basis_in'} -> ChemicalEntity\n",
      "Pathway - {'derives_from'} -> ChemicalEntity\n",
      "Pathway - {'physically_interacts_with'} -> ChemicalEntity\n",
      "ChemicalEntity - {'correlated_with'} -> Publication\n",
      "ChemicalEntity - {'disrupts'} -> Publication\n",
      "ChemicalEntity - {'associated_with'} -> Publication\n",
      "ChemicalEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "ChemicalEntity - {'regulates'} -> Publication\n",
      "ChemicalEntity - {'located_in'} -> Publication\n",
      "ChemicalEntity - {'predisposes'} -> Publication\n",
      "ChemicalEntity - {'related_to'} -> Publication\n",
      "ChemicalEntity - {'has_part'} -> Publication\n",
      "ChemicalEntity - {'prevents'} -> Publication\n",
      "ChemicalEntity - {'affects'} -> Publication\n",
      "ChemicalEntity - {'precedes'} -> Publication\n",
      "ChemicalEntity - {'occurs_in'} -> Publication\n",
      "ChemicalEntity - {'has_metabolite'} -> Publication\n",
      "ChemicalEntity - {'close_match'} -> Publication\n",
      "ChemicalEntity - {'subclass_of'} -> Publication\n",
      "ChemicalEntity - {'interacts_with'} -> Publication\n",
      "ChemicalEntity - {'treats'} -> Publication\n",
      "ChemicalEntity - {'physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'has_input'} -> Publication\n",
      "ChemicalEntity - {'causes'} -> Publication\n",
      "ChemicalEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->ChemicalMixture->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> ChemicalMixture\n",
      "Pathway - {'affected_by'} -> ChemicalMixture\n",
      "Pathway - {'causes'} -> ChemicalMixture\n",
      "Pathway - {'part_of'} -> ChemicalMixture\n",
      "Pathway - {'participates_in'} -> ChemicalMixture\n",
      "Pathway - {'contributes_to'} -> ChemicalMixture\n",
      "Pathway - {'has_input'} -> ChemicalMixture\n",
      "Pathway - {'preceded_by'} -> ChemicalMixture\n",
      "Pathway - {'enabled_by'} -> ChemicalMixture\n",
      "Pathway - {'has_participant'} -> ChemicalMixture\n",
      "Pathway - {'has_part'} -> ChemicalMixture\n",
      "Pathway - {'has_substrate'} -> ChemicalMixture\n",
      "Pathway - {'precedes'} -> ChemicalMixture\n",
      "Pathway - {'interacts_with'} -> ChemicalMixture\n",
      "Pathway - {'has_capability'} -> ChemicalMixture\n",
      "Pathway - {'caused_by'} -> ChemicalMixture\n",
      "Pathway - {'actively_involves'} -> ChemicalMixture\n",
      "Pathway - {'has_catalyst'} -> ChemicalMixture\n",
      "Pathway - {'temporally_related_to'} -> ChemicalMixture\n",
      "Pathway - {'associated_with'} -> ChemicalMixture\n",
      "Pathway - {'affects'} -> ChemicalMixture\n",
      "Pathway - {'similar_to'} -> ChemicalMixture\n",
      "Pathway - {'related_to'} -> ChemicalMixture\n",
      "Pathway - {'overlaps'} -> ChemicalMixture\n",
      "Pathway - {'composed_primarily_of'} -> ChemicalMixture\n",
      "Pathway - {'has_upstream_actor'} -> ChemicalMixture\n",
      "Pathway - {'has_active_ingredient'} -> ChemicalMixture\n",
      "Pathway - {'primarily_composed_of'} -> ChemicalMixture\n",
      "Pathway - {'has_output'} -> ChemicalMixture\n",
      "Pathway - {'subclass_of'} -> ChemicalMixture\n",
      "Pathway - {'regulates'} -> ChemicalMixture\n",
      "Pathway - {'produces'} -> ChemicalMixture\n",
      "Pathway - {'coexists_with'} -> ChemicalMixture\n",
      "ChemicalMixture - {'related_to'} -> Publication\n",
      "ChemicalMixture - {'affects'} -> Publication\n",
      "ChemicalMixture - {'subclass_of'} -> Publication\n",
      "ChemicalMixture - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->ComplexMolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> ComplexMolecularMixture\n",
      "Pathway - {'has_participant'} -> ComplexMolecularMixture\n",
      "Pathway - {'has_part'} -> ComplexMolecularMixture\n",
      "Pathway - {'is_exacerbated_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'interacts_with'} -> ComplexMolecularMixture\n",
      "Pathway - {'has_active_ingredient'} -> ComplexMolecularMixture\n",
      "Pathway - {'primarily_composed_of'} -> ComplexMolecularMixture\n",
      "Pathway - {'response_affected_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'associated_with'} -> ComplexMolecularMixture\n",
      "Pathway - {'affects'} -> ComplexMolecularMixture\n",
      "Pathway - {'similar_to'} -> ComplexMolecularMixture\n",
      "Pathway - {'related_to'} -> ComplexMolecularMixture\n",
      "Pathway - {'overlaps'} -> ComplexMolecularMixture\n",
      "Pathway - {'disrupted_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'composed_primarily_of'} -> ComplexMolecularMixture\n",
      "Pathway - {'has_upstream_actor'} -> ComplexMolecularMixture\n",
      "Pathway - {'treated_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'regulated_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'response_decreased_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'affected_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'is_ameliorated_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'response_increased_by'} -> ComplexMolecularMixture\n",
      "Pathway - {'part_of'} -> ComplexMolecularMixture\n",
      "Pathway - {'participates_in'} -> ComplexMolecularMixture\n",
      "ComplexMolecularMixture - {'correlated_with'} -> Publication\n",
      "ComplexMolecularMixture - {'subclass_of'} -> Publication\n",
      "ComplexMolecularMixture - {'treats'} -> Publication\n",
      "ComplexMolecularMixture - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->GrossAnatomicalStructure->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> GrossAnatomicalStructure\n",
      "Pathway - {'related_to'} -> GrossAnatomicalStructure\n",
      "Pathway - {'occurs_in'} -> GrossAnatomicalStructure\n",
      "Pathway - {'composed_primarily_of'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_upstream_actor'} -> GrossAnatomicalStructure\n",
      "Pathway - {'exacerbates'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_active_ingredient'} -> GrossAnatomicalStructure\n",
      "Pathway - {'increases_response_to'} -> GrossAnatomicalStructure\n",
      "Pathway - {'primarily_composed_of'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_output'} -> GrossAnatomicalStructure\n",
      "Pathway - {'disrupts'} -> GrossAnatomicalStructure\n",
      "Pathway - {'affected_by'} -> GrossAnatomicalStructure\n",
      "Pathway - {'treats'} -> GrossAnatomicalStructure\n",
      "Pathway - {'causes'} -> GrossAnatomicalStructure\n",
      "Pathway - {'part_of'} -> GrossAnatomicalStructure\n",
      "Pathway - {'participates_in'} -> GrossAnatomicalStructure\n",
      "Pathway - {'contributes_to'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_input'} -> GrossAnatomicalStructure\n",
      "Pathway - {'preceded_by'} -> GrossAnatomicalStructure\n",
      "Pathway - {'enabled_by'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_participant'} -> GrossAnatomicalStructure\n",
      "Pathway - {'decreases_response_to'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_part'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_substrate'} -> GrossAnatomicalStructure\n",
      "Pathway - {'precedes'} -> GrossAnatomicalStructure\n",
      "Pathway - {'interacts_with'} -> GrossAnatomicalStructure\n",
      "Pathway - {'ameliorates'} -> GrossAnatomicalStructure\n",
      "Pathway - {'regulates'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_capability'} -> GrossAnatomicalStructure\n",
      "Pathway - {'capable_of'} -> GrossAnatomicalStructure\n",
      "Pathway - {'caused_by'} -> GrossAnatomicalStructure\n",
      "Pathway - {'actively_involves'} -> GrossAnatomicalStructure\n",
      "Pathway - {'has_catalyst'} -> GrossAnatomicalStructure\n",
      "Pathway - {'temporally_related_to'} -> GrossAnatomicalStructure\n",
      "Pathway - {'actively_involved_in'} -> GrossAnatomicalStructure\n",
      "Pathway - {'associated_with'} -> GrossAnatomicalStructure\n",
      "Pathway - {'affects'} -> GrossAnatomicalStructure\n",
      "Pathway - {'similar_to'} -> GrossAnatomicalStructure\n",
      "Pathway - {'subclass_of'} -> GrossAnatomicalStructure\n",
      "Pathway - {'overlaps'} -> GrossAnatomicalStructure\n",
      "Pathway - {'affects_response_to'} -> GrossAnatomicalStructure\n",
      "Pathway - {'disease_has_location'} -> GrossAnatomicalStructure\n",
      "Pathway - {'located_in'} -> GrossAnatomicalStructure\n",
      "Pathway - {'produces'} -> GrossAnatomicalStructure\n",
      "Pathway - {'expressed_in'} -> GrossAnatomicalStructure\n",
      "Pathway - {'predisposes'} -> GrossAnatomicalStructure\n",
      "Pathway - {'manifestation_of'} -> GrossAnatomicalStructure\n",
      "Pathway - {'coexists_with'} -> GrossAnatomicalStructure\n",
      "GrossAnatomicalStructure - {'affects'} -> Publication\n",
      "GrossAnatomicalStructure - {'coexists_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'close_match'} -> Publication\n",
      "GrossAnatomicalStructure - {'develops_from'} -> Publication\n",
      "GrossAnatomicalStructure - {'regulates'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_input'} -> Publication\n",
      "GrossAnatomicalStructure - {'associated_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'located_in'} -> Publication\n",
      "GrossAnatomicalStructure - {'subclass_of'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_part'} -> Publication\n",
      "GrossAnatomicalStructure - {'produces'} -> Publication\n",
      "GrossAnatomicalStructure - {'related_to'} -> Publication\n",
      "GrossAnatomicalStructure - {'occurs_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->MolecularActivity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> MolecularActivity\n",
      "Pathway - {'overlaps'} -> MolecularActivity\n",
      "Pathway - {'positively_regulates'} -> MolecularActivity\n",
      "Pathway - {'preceded_by'} -> MolecularActivity\n",
      "Pathway - {'precedes'} -> MolecularActivity\n",
      "Pathway - {'process_negatively_regulates_process'} -> MolecularActivity\n",
      "Pathway - {'process_positively_regulates_process'} -> MolecularActivity\n",
      "Pathway - {'process_regulates_process'} -> MolecularActivity\n",
      "Pathway - {'regulates'} -> MolecularActivity\n",
      "Pathway - {'related_to'} -> MolecularActivity\n",
      "Pathway - {'temporally_related_to'} -> MolecularActivity\n",
      "Pathway - {'affects'} -> MolecularActivity\n",
      "Pathway - {'caused_by'} -> MolecularActivity\n",
      "Pathway - {'has_participant'} -> MolecularActivity\n",
      "Pathway - {'has_part'} -> MolecularActivity\n",
      "Pathway - {'negatively_regulates'} -> MolecularActivity\n",
      "Pathway - {'contributes_to'} -> MolecularActivity\n",
      "Pathway - {'has_input'} -> MolecularActivity\n",
      "Pathway - {'enabled_by'} -> MolecularActivity\n",
      "Pathway - {'decreases_response_to'} -> MolecularActivity\n",
      "Pathway - {'is_exacerbated_by'} -> MolecularActivity\n",
      "Pathway - {'has_substrate'} -> MolecularActivity\n",
      "Pathway - {'interacts_with'} -> MolecularActivity\n",
      "Pathway - {'ameliorates'} -> MolecularActivity\n",
      "Pathway - {'response_increased_by'} -> MolecularActivity\n",
      "Pathway - {'part_of'} -> MolecularActivity\n",
      "Pathway - {'occurs_in'} -> MolecularActivity\n",
      "Pathway - {'has_capability'} -> MolecularActivity\n",
      "Pathway - {'capable_of'} -> MolecularActivity\n",
      "Pathway - {'actively_involves'} -> MolecularActivity\n",
      "Pathway - {'has_catalyst'} -> MolecularActivity\n",
      "Pathway - {'associated_with'} -> MolecularActivity\n",
      "Pathway - {'affects_response_to'} -> MolecularActivity\n",
      "Pathway - {'disrupted_by'} -> MolecularActivity\n",
      "Pathway - {'composed_primarily_of'} -> MolecularActivity\n",
      "Pathway - {'has_upstream_actor'} -> MolecularActivity\n",
      "Pathway - {'treated_by'} -> MolecularActivity\n",
      "Pathway - {'exacerbates'} -> MolecularActivity\n",
      "Pathway - {'has_active_ingredient'} -> MolecularActivity\n",
      "Pathway - {'increases_response_to'} -> MolecularActivity\n",
      "Pathway - {'primarily_composed_of'} -> MolecularActivity\n",
      "Pathway - {'regulated_by'} -> MolecularActivity\n",
      "Pathway - {'actively_involved_in'} -> MolecularActivity\n",
      "Pathway - {'has_output'} -> MolecularActivity\n",
      "Pathway - {'response_affected_by'} -> MolecularActivity\n",
      "Pathway - {'response_decreased_by'} -> MolecularActivity\n",
      "Pathway - {'disrupts'} -> MolecularActivity\n",
      "Pathway - {'similar_to'} -> MolecularActivity\n",
      "Pathway - {'affected_by'} -> MolecularActivity\n",
      "Pathway - {'treats'} -> MolecularActivity\n",
      "Pathway - {'is_ameliorated_by'} -> MolecularActivity\n",
      "Pathway - {'causes'} -> MolecularActivity\n",
      "Pathway - {'participates_in'} -> MolecularActivity\n",
      "Pathway - {'coexists_with'} -> MolecularActivity\n",
      "Pathway - {'close_match'} -> MolecularActivity\n",
      "Pathway - {'subclass_of'} -> MolecularActivity\n",
      "Pathway - {'catalyzes'} -> MolecularActivity\n",
      "Pathway - {'produces'} -> MolecularActivity\n",
      "Pathway - {'enables'} -> MolecularActivity\n",
      "Pathway - {'manifestation_of'} -> MolecularActivity\n",
      "Pathway - {'same_as'} -> MolecularActivity\n",
      "MolecularActivity - {'occurs_in'} -> Publication\n",
      "MolecularActivity - {'treats'} -> Publication\n",
      "MolecularActivity - {'has_input'} -> Publication\n",
      "MolecularActivity - {'has_output'} -> Publication\n",
      "MolecularActivity - {'coexists_with'} -> Publication\n",
      "MolecularActivity - {'regulates'} -> Publication\n",
      "MolecularActivity - {'disrupts'} -> Publication\n",
      "MolecularActivity - {'associated_with'} -> Publication\n",
      "MolecularActivity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularActivity - {'located_in'} -> Publication\n",
      "MolecularActivity - {'produces'} -> Publication\n",
      "MolecularActivity - {'related_to'} -> Publication\n",
      "MolecularActivity - {'affects'} -> Publication\n",
      "MolecularActivity - {'has_participant'} -> Publication\n",
      "MolecularActivity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->MolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> MolecularMixture\n",
      "Pathway - {'has_input'} -> MolecularMixture\n",
      "Pathway - {'has_participant'} -> MolecularMixture\n",
      "Pathway - {'temporally_related_to'} -> MolecularMixture\n",
      "Pathway - {'response_affected_by'} -> MolecularMixture\n",
      "Pathway - {'associated_with'} -> MolecularMixture\n",
      "Pathway - {'affects'} -> MolecularMixture\n",
      "Pathway - {'similar_to'} -> MolecularMixture\n",
      "Pathway - {'related_to'} -> MolecularMixture\n",
      "Pathway - {'overlaps'} -> MolecularMixture\n",
      "Pathway - {'disrupted_by'} -> MolecularMixture\n",
      "Pathway - {'composed_primarily_of'} -> MolecularMixture\n",
      "Pathway - {'has_upstream_actor'} -> MolecularMixture\n",
      "Pathway - {'treated_by'} -> MolecularMixture\n",
      "Pathway - {'has_active_ingredient'} -> MolecularMixture\n",
      "Pathway - {'primarily_composed_of'} -> MolecularMixture\n",
      "Pathway - {'regulated_by'} -> MolecularMixture\n",
      "Pathway - {'has_output'} -> MolecularMixture\n",
      "Pathway - {'response_decreased_by'} -> MolecularMixture\n",
      "Pathway - {'affected_by'} -> MolecularMixture\n",
      "Pathway - {'is_ameliorated_by'} -> MolecularMixture\n",
      "Pathway - {'causes'} -> MolecularMixture\n",
      "Pathway - {'response_increased_by'} -> MolecularMixture\n",
      "Pathway - {'part_of'} -> MolecularMixture\n",
      "Pathway - {'participates_in'} -> MolecularMixture\n",
      "Pathway - {'contributes_to'} -> MolecularMixture\n",
      "Pathway - {'preceded_by'} -> MolecularMixture\n",
      "Pathway - {'enabled_by'} -> MolecularMixture\n",
      "Pathway - {'has_part'} -> MolecularMixture\n",
      "Pathway - {'is_exacerbated_by'} -> MolecularMixture\n",
      "Pathway - {'has_substrate'} -> MolecularMixture\n",
      "Pathway - {'precedes'} -> MolecularMixture\n",
      "Pathway - {'interacts_with'} -> MolecularMixture\n",
      "Pathway - {'has_capability'} -> MolecularMixture\n",
      "Pathway - {'caused_by'} -> MolecularMixture\n",
      "Pathway - {'actively_involves'} -> MolecularMixture\n",
      "Pathway - {'has_catalyst'} -> MolecularMixture\n",
      "Pathway - {'located_in'} -> MolecularMixture\n",
      "MolecularMixture - {'correlated_with'} -> Publication\n",
      "MolecularMixture - {'treats'} -> Publication\n",
      "MolecularMixture - {'subclass_of'} -> Publication\n",
      "MolecularMixture - {'located_in'} -> Publication\n",
      "MolecularMixture - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->NamedThing->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> NamedThing\n",
      "Pathway - {'disrupts'} -> NamedThing\n",
      "Pathway - {'subclass_of'} -> NamedThing\n",
      "Pathway - {'overlaps'} -> NamedThing\n",
      "Pathway - {'part_of'} -> NamedThing\n",
      "Pathway - {'positively_regulates'} -> NamedThing\n",
      "Pathway - {'preceded_by'} -> NamedThing\n",
      "Pathway - {'precedes'} -> NamedThing\n",
      "Pathway - {'process_negatively_regulates_process'} -> NamedThing\n",
      "Pathway - {'process_positively_regulates_process'} -> NamedThing\n",
      "Pathway - {'process_regulates_process'} -> NamedThing\n",
      "Pathway - {'regulates'} -> NamedThing\n",
      "Pathway - {'related_to'} -> NamedThing\n",
      "Pathway - {'temporally_related_to'} -> NamedThing\n",
      "Pathway - {'affects'} -> NamedThing\n",
      "Pathway - {'caused_by'} -> NamedThing\n",
      "Pathway - {'enabled_by'} -> NamedThing\n",
      "Pathway - {'has_input'} -> NamedThing\n",
      "Pathway - {'has_participant'} -> NamedThing\n",
      "Pathway - {'has_part'} -> NamedThing\n",
      "Pathway - {'negatively_regulates'} -> NamedThing\n",
      "Pathway - {'occurs_in'} -> NamedThing\n",
      "Pathway - {'has_upstream_actor'} -> NamedThing\n",
      "Pathway - {'exacerbates'} -> NamedThing\n",
      "Pathway - {'increases_response_to'} -> NamedThing\n",
      "Pathway - {'has_output'} -> NamedThing\n",
      "Pathway - {'affected_by'} -> NamedThing\n",
      "Pathway - {'interacts_with'} -> NamedThing\n",
      "Pathway - {'treats'} -> NamedThing\n",
      "Pathway - {'causes'} -> NamedThing\n",
      "Pathway - {'has_capability'} -> NamedThing\n",
      "Pathway - {'participates_in'} -> NamedThing\n",
      "Pathway - {'contributes_to'} -> NamedThing\n",
      "Pathway - {'decreases_response_to'} -> NamedThing\n",
      "Pathway - {'has_substrate'} -> NamedThing\n",
      "Pathway - {'ameliorates'} -> NamedThing\n",
      "Pathway - {'capable_of'} -> NamedThing\n",
      "Pathway - {'actively_involves'} -> NamedThing\n",
      "Pathway - {'has_active_ingredient'} -> NamedThing\n",
      "Pathway - {'has_catalyst'} -> NamedThing\n",
      "Pathway - {'primarily_composed_of'} -> NamedThing\n",
      "Pathway - {'actively_involved_in'} -> NamedThing\n",
      "Pathway - {'associated_with'} -> NamedThing\n",
      "Pathway - {'similar_to'} -> NamedThing\n",
      "Pathway - {'affects_response_to'} -> NamedThing\n",
      "Pathway - {'composed_primarily_of'} -> NamedThing\n",
      "Pathway - {'coexists_with'} -> NamedThing\n",
      "Pathway - {'manifestation_of'} -> NamedThing\n",
      "Pathway - {'close_match'} -> NamedThing\n",
      "Pathway - {'contraindicated_for'} -> NamedThing\n",
      "Pathway - {'diagnoses'} -> NamedThing\n",
      "Pathway - {'derives_from'} -> NamedThing\n",
      "Pathway - {'in_taxon'} -> NamedThing\n",
      "Pathway - {'physically_interacts_with'} -> NamedThing\n",
      "Pathway - {'disease_has_location'} -> NamedThing\n",
      "Pathway - {'directly_physically_interacts_with'} -> NamedThing\n",
      "Pathway - {'gene_associated_with_condition'} -> NamedThing\n",
      "Pathway - {'has_phenotype'} -> NamedThing\n",
      "Pathway - {'biomarker_for'} -> NamedThing\n",
      "Pathway - {'produces'} -> NamedThing\n",
      "Pathway - {'correlated_with'} -> NamedThing\n",
      "Pathway - {'located_in'} -> NamedThing\n",
      "Pathway - {'colocalizes_with'} -> NamedThing\n",
      "Pathway - {'predisposes'} -> NamedThing\n",
      "Pathway - {'prevents'} -> NamedThing\n",
      "Pathway - {'same_as'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Publication\n",
      "NamedThing - {'related_to'} -> Publication\n",
      "NamedThing - {'is_sequence_variant_of'} -> Publication\n",
      "NamedThing - {'associated_with'} -> Publication\n",
      "NamedThing - {'has_phenotype'} -> Publication\n",
      "NamedThing - {'disrupts'} -> Publication\n",
      "NamedThing - {'biomarker_for'} -> Publication\n",
      "NamedThing - {'correlated_with'} -> Publication\n",
      "NamedThing - {'gene_associated_with_condition'} -> Publication\n",
      "NamedThing - {'has_part'} -> Publication\n",
      "NamedThing - {'located_in'} -> Publication\n",
      "NamedThing - {'colocalizes_with'} -> Publication\n",
      "NamedThing - {'interacts_with'} -> Publication\n",
      "NamedThing - {'produces'} -> Publication\n",
      "NamedThing - {'prevents'} -> Publication\n",
      "NamedThing - {'diagnoses'} -> Publication\n",
      "NamedThing - {'predisposes'} -> Publication\n",
      "NamedThing - {'precedes'} -> Publication\n",
      "NamedThing - {'occurs_in'} -> Publication\n",
      "NamedThing - {'affects'} -> Publication\n",
      "NamedThing - {'close_match'} -> Publication\n",
      "NamedThing - {'manifestation_of'} -> Publication\n",
      "NamedThing - {'exacerbates'} -> Publication\n",
      "NamedThing - {'coexists_with'} -> Publication\n",
      "NamedThing - {'regulates'} -> Publication\n",
      "NamedThing - {'has_input'} -> Publication\n",
      "NamedThing - {'physically_interacts_with'} -> Publication\n",
      "NamedThing - {'treats'} -> Publication\n",
      "NamedThing - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->OrganismTaxon->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> OrganismTaxon\n",
      "Pathway - {'in_taxon'} -> OrganismTaxon\n",
      "Pathway - {'related_to'} -> OrganismTaxon\n",
      "Pathway - {'occurs_in'} -> OrganismTaxon\n",
      "Pathway - {'caused_by'} -> OrganismTaxon\n",
      "Pathway - {'disrupted_by'} -> OrganismTaxon\n",
      "Pathway - {'has_upstream_actor'} -> OrganismTaxon\n",
      "Pathway - {'has_active_ingredient'} -> OrganismTaxon\n",
      "Pathway - {'primarily_composed_of'} -> OrganismTaxon\n",
      "Pathway - {'regulated_by'} -> OrganismTaxon\n",
      "Pathway - {'response_affected_by'} -> OrganismTaxon\n",
      "Pathway - {'affects'} -> OrganismTaxon\n",
      "Pathway - {'similar_to'} -> OrganismTaxon\n",
      "Pathway - {'subclass_of'} -> OrganismTaxon\n",
      "Pathway - {'overlaps'} -> OrganismTaxon\n",
      "Pathway - {'causes'} -> OrganismTaxon\n",
      "Pathway - {'composed_primarily_of'} -> OrganismTaxon\n",
      "Pathway - {'treated_by'} -> OrganismTaxon\n",
      "Pathway - {'response_decreased_by'} -> OrganismTaxon\n",
      "Pathway - {'affected_by'} -> OrganismTaxon\n",
      "Pathway - {'interacts_with'} -> OrganismTaxon\n",
      "Pathway - {'treats'} -> OrganismTaxon\n",
      "Pathway - {'is_ameliorated_by'} -> OrganismTaxon\n",
      "Pathway - {'response_increased_by'} -> OrganismTaxon\n",
      "Pathway - {'part_of'} -> OrganismTaxon\n",
      "Pathway - {'participates_in'} -> OrganismTaxon\n",
      "Pathway - {'contributes_to'} -> OrganismTaxon\n",
      "Pathway - {'has_participant'} -> OrganismTaxon\n",
      "Pathway - {'associated_with'} -> OrganismTaxon\n",
      "Pathway - {'has_part'} -> OrganismTaxon\n",
      "Pathway - {'is_exacerbated_by'} -> OrganismTaxon\n",
      "Pathway - {'ameliorates'} -> OrganismTaxon\n",
      "Pathway - {'gene_associated_with_condition'} -> OrganismTaxon\n",
      "Pathway - {'produces'} -> OrganismTaxon\n",
      "Pathway - {'located_in'} -> OrganismTaxon\n",
      "Pathway - {'predisposes'} -> OrganismTaxon\n",
      "Pathway - {'precedes'} -> OrganismTaxon\n",
      "Pathway - {'prevents'} -> OrganismTaxon\n",
      "Pathway - {'manifestation_of'} -> OrganismTaxon\n",
      "Pathway - {'coexists_with'} -> OrganismTaxon\n",
      "Pathway - {'close_match'} -> OrganismTaxon\n",
      "Pathway - {'exacerbates'} -> OrganismTaxon\n",
      "Pathway - {'regulates'} -> OrganismTaxon\n",
      "Pathway - {'diagnoses'} -> OrganismTaxon\n",
      "Pathway - {'has_input'} -> OrganismTaxon\n",
      "Pathway - {'physically_interacts_with'} -> OrganismTaxon\n",
      "Pathway - {'has_output'} -> OrganismTaxon\n",
      "Pathway - {'has_phenotype'} -> OrganismTaxon\n",
      "Pathway - {'disrupts'} -> OrganismTaxon\n",
      "OrganismTaxon - {'regulates'} -> Publication\n",
      "OrganismTaxon - {'gene_associated_with_condition'} -> Publication\n",
      "OrganismTaxon - {'located_in'} -> Publication\n",
      "OrganismTaxon - {'interacts_with'} -> Publication\n",
      "OrganismTaxon - {'produces'} -> Publication\n",
      "OrganismTaxon - {'related_to'} -> Publication\n",
      "OrganismTaxon - {'has_part'} -> Publication\n",
      "OrganismTaxon - {'predisposes'} -> Publication\n",
      "OrganismTaxon - {'occurs_in'} -> Publication\n",
      "OrganismTaxon - {'affects'} -> Publication\n",
      "OrganismTaxon - {'manifestation_of'} -> Publication\n",
      "OrganismTaxon - {'exacerbates'} -> Publication\n",
      "OrganismTaxon - {'coexists_with'} -> Publication\n",
      "OrganismTaxon - {'close_match'} -> Publication\n",
      "OrganismTaxon - {'subclass_of'} -> Publication\n",
      "OrganismTaxon - {'physically_interacts_with'} -> Publication\n",
      "OrganismTaxon - {'treats'} -> Publication\n",
      "OrganismTaxon - {'has_input'} -> Publication\n",
      "OrganismTaxon - {'causes'} -> Publication\n",
      "OrganismTaxon - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Polypeptide->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> Polypeptide\n",
      "Pathway - {'preceded_by'} -> Polypeptide\n",
      "Pathway - {'related_to'} -> Polypeptide\n",
      "Pathway - {'temporally_related_to'} -> Polypeptide\n",
      "Pathway - {'caused_by'} -> Polypeptide\n",
      "Pathway - {'enabled_by'} -> Polypeptide\n",
      "Pathway - {'has_participant'} -> Polypeptide\n",
      "Pathway - {'has_input'} -> Polypeptide\n",
      "Pathway - {'exacerbates'} -> Polypeptide\n",
      "Pathway - {'treats'} -> Polypeptide\n",
      "Pathway - {'is_ameliorated_by'} -> Polypeptide\n",
      "Pathway - {'regulates'} -> Polypeptide\n",
      "Pathway - {'causes'} -> Polypeptide\n",
      "Pathway - {'response_increased_by'} -> Polypeptide\n",
      "Pathway - {'part_of'} -> Polypeptide\n",
      "Pathway - {'has_capability'} -> Polypeptide\n",
      "Pathway - {'participates_in'} -> Polypeptide\n",
      "Pathway - {'contributes_to'} -> Polypeptide\n",
      "Pathway - {'decreases_response_to'} -> Polypeptide\n",
      "Pathway - {'has_part'} -> Polypeptide\n",
      "Pathway - {'is_exacerbated_by'} -> Polypeptide\n",
      "Pathway - {'has_substrate'} -> Polypeptide\n",
      "Pathway - {'precedes'} -> Polypeptide\n",
      "Pathway - {'interacts_with'} -> Polypeptide\n",
      "Pathway - {'ameliorates'} -> Polypeptide\n",
      "Pathway - {'actively_involves'} -> Polypeptide\n",
      "Pathway - {'has_active_ingredient'} -> Polypeptide\n",
      "Pathway - {'has_catalyst'} -> Polypeptide\n",
      "Pathway - {'primarily_composed_of'} -> Polypeptide\n",
      "Pathway - {'response_affected_by'} -> Polypeptide\n",
      "Pathway - {'associated_with'} -> Polypeptide\n",
      "Pathway - {'affects'} -> Polypeptide\n",
      "Pathway - {'similar_to'} -> Polypeptide\n",
      "Pathway - {'overlaps'} -> Polypeptide\n",
      "Pathway - {'affects_response_to'} -> Polypeptide\n",
      "Pathway - {'disrupted_by'} -> Polypeptide\n",
      "Pathway - {'composed_primarily_of'} -> Polypeptide\n",
      "Pathway - {'has_upstream_actor'} -> Polypeptide\n",
      "Pathway - {'treated_by'} -> Polypeptide\n",
      "Pathway - {'increases_response_to'} -> Polypeptide\n",
      "Pathway - {'regulated_by'} -> Polypeptide\n",
      "Pathway - {'has_output'} -> Polypeptide\n",
      "Pathway - {'response_decreased_by'} -> Polypeptide\n",
      "Pathway - {'disrupts'} -> Polypeptide\n",
      "Pathway - {'affected_by'} -> Polypeptide\n",
      "Pathway - {'located_in'} -> Polypeptide\n",
      "Pathway - {'occurs_in'} -> Polypeptide\n",
      "Pathway - {'coexists_with'} -> Polypeptide\n",
      "Pathway - {'close_match'} -> Polypeptide\n",
      "Pathway - {'subclass_of'} -> Polypeptide\n",
      "Pathway - {'capable_of'} -> Polypeptide\n",
      "Pathway - {'derives_from'} -> Polypeptide\n",
      "Pathway - {'physically_interacts_with'} -> Polypeptide\n",
      "Pathway - {'directly_physically_interacts_with'} -> Polypeptide\n",
      "Pathway - {'actively_involved_in'} -> Polypeptide\n",
      "Pathway - {'biomarker_for'} -> Polypeptide\n",
      "Pathway - {'colocalizes_with'} -> Polypeptide\n",
      "Pathway - {'produces'} -> Polypeptide\n",
      "Polypeptide - {'disrupts'} -> Publication\n",
      "Polypeptide - {'associated_with'} -> Publication\n",
      "Polypeptide - {'gene_associated_with_condition'} -> Publication\n",
      "Polypeptide - {'located_in'} -> Publication\n",
      "Polypeptide - {'regulates'} -> Publication\n",
      "Polypeptide - {'biomarker_for'} -> Publication\n",
      "Polypeptide - {'produces'} -> Publication\n",
      "Polypeptide - {'has_part'} -> Publication\n",
      "Polypeptide - {'physically_interacts_with'} -> Publication\n",
      "Polypeptide - {'colocalizes_with'} -> Publication\n",
      "Polypeptide - {'predisposes'} -> Publication\n",
      "Polypeptide - {'related_to'} -> Publication\n",
      "Polypeptide - {'prevents'} -> Publication\n",
      "Polypeptide - {'precedes'} -> Publication\n",
      "Polypeptide - {'close_match'} -> Publication\n",
      "Polypeptide - {'subclass_of'} -> Publication\n",
      "Polypeptide - {'interacts_with'} -> Publication\n",
      "Polypeptide - {'affects'} -> Publication\n",
      "Polypeptide - {'treats'} -> Publication\n",
      "Polypeptide - {'occurs_in'} -> Publication\n",
      "Polypeptide - {'exacerbates'} -> Publication\n",
      "Polypeptide - {'causes'} -> Publication\n",
      "Polypeptide - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Protein->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_together_in_literature_with'} -> Protein\n",
      "Pathway - {'affected_by'} -> Protein\n",
      "Pathway - {'has_negative_upstream_or_within_actor'} -> Protein\n",
      "Pathway - {'has_upstream_or_within_actor'} -> Protein\n",
      "Pathway - {'has_positive_upstream_or_within_actor'} -> Protein\n",
      "Pathway - {'actively_involves'} -> Protein\n",
      "Pathway - {'actively_involved_in'} -> Protein\n",
      "Pathway - {'has_capability'} -> Protein\n",
      "Pathway - {'has_upstream_actor'} -> Protein\n",
      "Pathway - {'causes'} -> Protein\n",
      "Pathway - {'overlaps'} -> Protein\n",
      "Pathway - {'preceded_by'} -> Protein\n",
      "Pathway - {'related_to'} -> Protein\n",
      "Pathway - {'temporally_related_to'} -> Protein\n",
      "Pathway - {'caused_by'} -> Protein\n",
      "Pathway - {'enabled_by'} -> Protein\n",
      "Pathway - {'has_participant'} -> Protein\n",
      "Pathway - {'treated_by'} -> Protein\n",
      "Pathway - {'gene_associated_with_condition'} -> Protein\n",
      "Pathway - {'has_output'} -> Protein\n",
      "Pathway - {'disrupts'} -> Protein\n",
      "Pathway - {'has_substrate'} -> Protein\n",
      "Pathway - {'interacts_with'} -> Protein\n",
      "Pathway - {'ameliorates'} -> Protein\n",
      "Pathway - {'treats'} -> Protein\n",
      "Pathway - {'is_ameliorated_by'} -> Protein\n",
      "Pathway - {'regulates'} -> Protein\n",
      "Pathway - {'response_increased_by'} -> Protein\n",
      "Pathway - {'part_of'} -> Protein\n",
      "Pathway - {'positively_regulates'} -> Protein\n",
      "Pathway - {'participates_in'} -> Protein\n",
      "Pathway - {'contributes_to'} -> Protein\n",
      "Pathway - {'has_input'} -> Protein\n",
      "Pathway - {'has_catalyst'} -> Protein\n",
      "Pathway - {'decreases_response_to'} -> Protein\n",
      "Pathway - {'associated_with'} -> Protein\n",
      "Pathway - {'has_part'} -> Protein\n",
      "Pathway - {'precedes'} -> Protein\n",
      "Pathway - {'genetic_association'} -> Protein\n",
      "Pathway - {'exacerbates'} -> Protein\n",
      "Pathway - {'has_active_ingredient'} -> Protein\n",
      "Pathway - {'increases_response_to'} -> Protein\n",
      "Pathway - {'primarily_composed_of'} -> Protein\n",
      "Pathway - {'response_affected_by'} -> Protein\n",
      "Pathway - {'affects'} -> Protein\n",
      "Pathway - {'process_regulates_process'} -> Protein\n",
      "Pathway - {'similar_to'} -> Protein\n",
      "Pathway - {'affects_response_to'} -> Protein\n",
      "Pathway - {'condition_associated_with_gene'} -> Protein\n",
      "Pathway - {'process_positively_regulates_process'} -> Protein\n",
      "Pathway - {'composed_primarily_of'} -> Protein\n",
      "Pathway - {'capable_of'} -> Protein\n",
      "Pathway - {'derives_from'} -> Protein\n",
      "Pathway - {'physically_interacts_with'} -> Protein\n",
      "Pathway - {'gene_product_of'} -> Protein\n",
      "Pathway - {'directly_physically_interacts_with'} -> Protein\n",
      "Pathway - {'biomarker_for'} -> Protein\n",
      "Pathway - {'located_in'} -> Protein\n",
      "Pathway - {'colocalizes_with'} -> Protein\n",
      "Pathway - {'produces'} -> Protein\n",
      "Pathway - {'prevents'} -> Protein\n",
      "Pathway - {'predisposes'} -> Protein\n",
      "Pathway - {'occurs_in'} -> Protein\n",
      "Pathway - {'manifestation_of'} -> Protein\n",
      "Pathway - {'coexists_with'} -> Protein\n",
      "Pathway - {'close_match'} -> Protein\n",
      "Pathway - {'subclass_of'} -> Protein\n",
      "Protein - {'correlated_with'} -> Publication\n",
      "Protein - {'gene_product_of'} -> Publication\n",
      "Protein - {'associated_with'} -> Publication\n",
      "Protein - {'gene_associated_with_condition'} -> Publication\n",
      "Protein - {'located_in'} -> Publication\n",
      "Protein - {'regulates'} -> Publication\n",
      "Protein - {'biomarker_for'} -> Publication\n",
      "Protein - {'expressed_in'} -> Publication\n",
      "Protein - {'has_part'} -> Publication\n",
      "Protein - {'colocalizes_with'} -> Publication\n",
      "Protein - {'produces'} -> Publication\n",
      "Protein - {'predisposes'} -> Publication\n",
      "Protein - {'related_to'} -> Publication\n",
      "Protein - {'prevents'} -> Publication\n",
      "Protein - {'occurs_in'} -> Publication\n",
      "Protein - {'precedes'} -> Publication\n",
      "Protein - {'affects'} -> Publication\n",
      "Protein - {'close_match'} -> Publication\n",
      "Protein - {'subclass_of'} -> Publication\n",
      "Protein - {'interacts_with'} -> Publication\n",
      "Protein - {'treats'} -> Publication\n",
      "Protein - {'has_input'} -> Publication\n",
      "Protein - {'causes'} -> Publication\n",
      "Protein - {'physically_interacts_with'} -> Publication\n",
      "Protein - {'exacerbates'} -> Publication\n",
      "Protein - {'coexists_with'} -> Publication\n",
      "Protein - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Transcript->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'treats'} -> Transcript\n",
      "Pathway - {'affected_by'} -> Transcript\n",
      "Pathway - {'participates_in'} -> Transcript\n",
      "Pathway - {'has_input'} -> Transcript\n",
      "Pathway - {'enabled_by'} -> Transcript\n",
      "Pathway - {'related_to'} -> Transcript\n",
      "Pathway - {'has_participant'} -> Transcript\n",
      "Pathway - {'decreases_response_to'} -> Transcript\n",
      "Pathway - {'has_substrate'} -> Transcript\n",
      "Pathway - {'interacts_with'} -> Transcript\n",
      "Pathway - {'ameliorates'} -> Transcript\n",
      "Pathway - {'regulates'} -> Transcript\n",
      "Pathway - {'has_capability'} -> Transcript\n",
      "Pathway - {'actively_involves'} -> Transcript\n",
      "Pathway - {'has_catalyst'} -> Transcript\n",
      "Pathway - {'associated_with'} -> Transcript\n",
      "Pathway - {'affects'} -> Transcript\n",
      "Pathway - {'overlaps'} -> Transcript\n",
      "Pathway - {'affects_response_to'} -> Transcript\n",
      "Pathway - {'composed_primarily_of'} -> Transcript\n",
      "Pathway - {'has_upstream_actor'} -> Transcript\n",
      "Pathway - {'exacerbates'} -> Transcript\n",
      "Pathway - {'increases_response_to'} -> Transcript\n",
      "Pathway - {'primarily_composed_of'} -> Transcript\n",
      "Pathway - {'has_output'} -> Transcript\n",
      "Pathway - {'disrupts'} -> Transcript\n",
      "Pathway - {'similar_to'} -> Transcript\n",
      "Pathway - {'coexists_with'} -> Transcript\n",
      "Pathway - {'has_part'} -> Transcript\n",
      "Pathway - {'produces'} -> Transcript\n",
      "Transcript - {'translates_to'} -> Publication\n",
      "Transcript - {'regulates'} -> Publication\n",
      "Transcript - {'transcribed_from'} -> Publication\n",
      "Transcript - {'associated_with'} -> Publication\n",
      "Transcript - {'located_in'} -> Publication\n",
      "Transcript - {'subclass_of'} -> Publication\n",
      "Transcript - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->PhysiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'treats'} -> PhysiologicalProcess\n",
      "Pathway - {'has_participant'} -> PhysiologicalProcess\n",
      "Pathway - {'decreases_response_to'} -> PhysiologicalProcess\n",
      "Pathway - {'associated_with'} -> PhysiologicalProcess\n",
      "Pathway - {'is_exacerbated_by'} -> PhysiologicalProcess\n",
      "Pathway - {'ameliorates'} -> PhysiologicalProcess\n",
      "Pathway - {'disrupted_by'} -> PhysiologicalProcess\n",
      "Pathway - {'caused_by'} -> PhysiologicalProcess\n",
      "Pathway - {'has_upstream_actor'} -> PhysiologicalProcess\n",
      "Pathway - {'primarily_composed_of'} -> PhysiologicalProcess\n",
      "Pathway - {'regulated_by'} -> PhysiologicalProcess\n",
      "Pathway - {'response_affected_by'} -> PhysiologicalProcess\n",
      "Pathway - {'affects'} -> PhysiologicalProcess\n",
      "Pathway - {'similar_to'} -> PhysiologicalProcess\n",
      "Pathway - {'related_to'} -> PhysiologicalProcess\n",
      "Pathway - {'overlaps'} -> PhysiologicalProcess\n",
      "Pathway - {'affects_response_to'} -> PhysiologicalProcess\n",
      "Pathway - {'composed_primarily_of'} -> PhysiologicalProcess\n",
      "Pathway - {'treated_by'} -> PhysiologicalProcess\n",
      "Pathway - {'exacerbates'} -> PhysiologicalProcess\n",
      "Pathway - {'increases_response_to'} -> PhysiologicalProcess\n",
      "Pathway - {'response_decreased_by'} -> PhysiologicalProcess\n",
      "Pathway - {'disrupts'} -> PhysiologicalProcess\n",
      "Pathway - {'affected_by'} -> PhysiologicalProcess\n",
      "Pathway - {'interacts_with'} -> PhysiologicalProcess\n",
      "Pathway - {'is_ameliorated_by'} -> PhysiologicalProcess\n",
      "Pathway - {'regulates'} -> PhysiologicalProcess\n",
      "Pathway - {'causes'} -> PhysiologicalProcess\n",
      "Pathway - {'response_increased_by'} -> PhysiologicalProcess\n",
      "Pathway - {'participates_in'} -> PhysiologicalProcess\n",
      "Pathway - {'contributes_to'} -> PhysiologicalProcess\n",
      "Pathway - {'temporally_related_to'} -> PhysiologicalProcess\n",
      "Pathway - {'gene_associated_with_condition'} -> PhysiologicalProcess\n",
      "Pathway - {'has_part'} -> PhysiologicalProcess\n",
      "Pathway - {'enables'} -> PhysiologicalProcess\n",
      "Pathway - {'predisposes'} -> PhysiologicalProcess\n",
      "Pathway - {'prevents'} -> PhysiologicalProcess\n",
      "Pathway - {'precedes'} -> PhysiologicalProcess\n",
      "Pathway - {'occurs_in'} -> PhysiologicalProcess\n",
      "Pathway - {'coexists_with'} -> PhysiologicalProcess\n",
      "Pathway - {'manifestation_of'} -> PhysiologicalProcess\n",
      "Pathway - {'close_match'} -> PhysiologicalProcess\n",
      "Pathway - {'subclass_of'} -> PhysiologicalProcess\n",
      "Pathway - {'capable_of'} -> PhysiologicalProcess\n",
      "Pathway - {'contraindicated_for'} -> PhysiologicalProcess\n",
      "Pathway - {'diagnoses'} -> PhysiologicalProcess\n",
      "Pathway - {'same_as'} -> PhysiologicalProcess\n",
      "PhysiologicalProcess - {'correlated_with'} -> Publication\n",
      "PhysiologicalProcess - {'has_input'} -> Publication\n",
      "PhysiologicalProcess - {'causes'} -> Publication\n",
      "PhysiologicalProcess - {'has_output'} -> Publication\n",
      "PhysiologicalProcess - {'disrupts'} -> Publication\n",
      "PhysiologicalProcess - {'associated_with'} -> Publication\n",
      "PhysiologicalProcess - {'located_in'} -> Publication\n",
      "PhysiologicalProcess - {'produces'} -> Publication\n",
      "PhysiologicalProcess - {'predisposes'} -> Publication\n",
      "PhysiologicalProcess - {'precedes'} -> Publication\n",
      "PhysiologicalProcess - {'related_to'} -> Publication\n",
      "PhysiologicalProcess - {'occurs_in'} -> Publication\n",
      "PhysiologicalProcess - {'affects'} -> Publication\n",
      "PhysiologicalProcess - {'coexists_with'} -> Publication\n",
      "PhysiologicalProcess - {'close_match'} -> Publication\n",
      "PhysiologicalProcess - {'has_participant'} -> Publication\n",
      "PhysiologicalProcess - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Procedure->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'treats'} -> Procedure\n",
      "Pathway - {'affected_by'} -> Procedure\n",
      "Pathway - {'interacts_with'} -> Procedure\n",
      "Pathway - {'is_ameliorated_by'} -> Procedure\n",
      "Pathway - {'causes'} -> Procedure\n",
      "Pathway - {'participates_in'} -> Procedure\n",
      "Pathway - {'contributes_to'} -> Procedure\n",
      "Pathway - {'preceded_by'} -> Procedure\n",
      "Pathway - {'has_participant'} -> Procedure\n",
      "Pathway - {'associated_with'} -> Procedure\n",
      "Pathway - {'precedes'} -> Procedure\n",
      "Pathway - {'ameliorates'} -> Procedure\n",
      "Pathway - {'caused_by'} -> Procedure\n",
      "Pathway - {'primarily_composed_of'} -> Procedure\n",
      "Pathway - {'temporally_related_to'} -> Procedure\n",
      "Pathway - {'affects'} -> Procedure\n",
      "Pathway - {'similar_to'} -> Procedure\n",
      "Pathway - {'related_to'} -> Procedure\n",
      "Pathway - {'overlaps'} -> Procedure\n",
      "Pathway - {'disrupted_by'} -> Procedure\n",
      "Pathway - {'composed_primarily_of'} -> Procedure\n",
      "Pathway - {'has_upstream_actor'} -> Procedure\n",
      "Pathway - {'treated_by'} -> Procedure\n",
      "Pathway - {'predisposes'} -> Procedure\n",
      "Pathway - {'occurs_in'} -> Procedure\n",
      "Pathway - {'manifestation_of'} -> Procedure\n",
      "Pathway - {'exacerbates'} -> Procedure\n",
      "Pathway - {'coexists_with'} -> Procedure\n",
      "Pathway - {'close_match'} -> Procedure\n",
      "Pathway - {'subclass_of'} -> Procedure\n",
      "Pathway - {'regulates'} -> Procedure\n",
      "Pathway - {'contraindicated_for'} -> Procedure\n",
      "Pathway - {'has_input'} -> Procedure\n",
      "Pathway - {'physically_interacts_with'} -> Procedure\n",
      "Pathway - {'has_output'} -> Procedure\n",
      "Pathway - {'has_phenotype'} -> Procedure\n",
      "Pathway - {'disrupts'} -> Procedure\n",
      "Pathway - {'biomarker_for'} -> Procedure\n",
      "Pathway - {'gene_associated_with_condition'} -> Procedure\n",
      "Pathway - {'has_part'} -> Procedure\n",
      "Pathway - {'located_in'} -> Procedure\n",
      "Pathway - {'produces'} -> Procedure\n",
      "Pathway - {'prevents'} -> Procedure\n",
      "Procedure - {'correlated_with'} -> Publication\n",
      "Procedure - {'manifestation_of'} -> Publication\n",
      "Procedure - {'close_match'} -> Publication\n",
      "Procedure - {'subclass_of'} -> Publication\n",
      "Procedure - {'interacts_with'} -> Publication\n",
      "Procedure - {'diagnoses'} -> Publication\n",
      "Procedure - {'occurs_in'} -> Publication\n",
      "Procedure - {'treats'} -> Publication\n",
      "Procedure - {'has_input'} -> Publication\n",
      "Procedure - {'exacerbates'} -> Publication\n",
      "Procedure - {'causes'} -> Publication\n",
      "Procedure - {'coexists_with'} -> Publication\n",
      "Procedure - {'disrupts'} -> Publication\n",
      "Procedure - {'associated_with'} -> Publication\n",
      "Procedure - {'regulates'} -> Publication\n",
      "Procedure - {'produces'} -> Publication\n",
      "Procedure - {'located_in'} -> Publication\n",
      "Procedure - {'predisposes'} -> Publication\n",
      "Procedure - {'related_to'} -> Publication\n",
      "Procedure - {'has_part'} -> Publication\n",
      "Procedure - {'prevents'} -> Publication\n",
      "Procedure - {'precedes'} -> Publication\n",
      "Procedure - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Drug->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'enabled_by'} -> Drug\n",
      "Pathway - {'ameliorates'} -> Drug\n",
      "Pathway - {'has_participant'} -> Drug\n",
      "Pathway - {'affected_by'} -> Drug\n",
      "Pathway - {'has_substrate'} -> Drug\n",
      "Pathway - {'interacts_with'} -> Drug\n",
      "Pathway - {'has_capability'} -> Drug\n",
      "Pathway - {'participates_in'} -> Drug\n",
      "Pathway - {'caused_by'} -> Drug\n",
      "Pathway - {'actively_involves'} -> Drug\n",
      "Pathway - {'has_catalyst'} -> Drug\n",
      "Pathway - {'associated_with'} -> Drug\n",
      "Pathway - {'precedes'} -> Drug\n",
      "Pathway - {'overlaps'} -> Drug\n",
      "Pathway - {'disrupted_by'} -> Drug\n",
      "Pathway - {'has_upstream_actor'} -> Drug\n",
      "Pathway - {'primarily_composed_of'} -> Drug\n",
      "Pathway - {'temporally_related_to'} -> Drug\n",
      "Pathway - {'has_output'} -> Drug\n",
      "Pathway - {'affects'} -> Drug\n",
      "Pathway - {'similar_to'} -> Drug\n",
      "Pathway - {'related_to'} -> Drug\n",
      "Pathway - {'treats'} -> Drug\n",
      "Pathway - {'composed_primarily_of'} -> Drug\n",
      "Pathway - {'has_input'} -> Drug\n",
      "Pathway - {'preceded_by'} -> Drug\n",
      "Pathway - {'predisposes'} -> Drug\n",
      "Pathway - {'occurs_in'} -> Drug\n",
      "Pathway - {'coexists_with'} -> Drug\n",
      "Pathway - {'manifestation_of'} -> Drug\n",
      "Pathway - {'close_match'} -> Drug\n",
      "Pathway - {'subclass_of'} -> Drug\n",
      "Pathway - {'regulates'} -> Drug\n",
      "Pathway - {'diagnoses'} -> Drug\n",
      "Pathway - {'derives_from'} -> Drug\n",
      "Pathway - {'physically_interacts_with'} -> Drug\n",
      "Pathway - {'causes'} -> Drug\n",
      "Pathway - {'disrupts'} -> Drug\n",
      "Pathway - {'gene_associated_with_condition'} -> Drug\n",
      "Pathway - {'produces'} -> Drug\n",
      "Pathway - {'has_part'} -> Drug\n",
      "Pathway - {'located_in'} -> Drug\n",
      "Drug - {'correlated_with'} -> Publication\n",
      "Drug - {'regulates'} -> Publication\n",
      "Drug - {'treats'} -> Publication\n",
      "Drug - {'has_input'} -> Publication\n",
      "Drug - {'physically_interacts_with'} -> Publication\n",
      "Drug - {'causes'} -> Publication\n",
      "Drug - {'directly_physically_interacts_with'} -> Publication\n",
      "Drug - {'disrupts'} -> Publication\n",
      "Drug - {'associated_with'} -> Publication\n",
      "Drug - {'gene_associated_with_condition'} -> Publication\n",
      "Drug - {'located_in'} -> Publication\n",
      "Drug - {'has_part'} -> Publication\n",
      "Drug - {'prevents'} -> Publication\n",
      "Drug - {'colocalizes_with'} -> Publication\n",
      "Drug - {'interacts_with'} -> Publication\n",
      "Drug - {'predisposes'} -> Publication\n",
      "Drug - {'related_to'} -> Publication\n",
      "Drug - {'occurs_in'} -> Publication\n",
      "Drug - {'precedes'} -> Publication\n",
      "Drug - {'affects'} -> Publication\n",
      "Drug - {'close_match'} -> Publication\n",
      "Drug - {'subclass_of'} -> Publication\n",
      "Drug - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->GeneFamily->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> GeneFamily\n",
      "Pathway - {'actively_involves'} -> GeneFamily\n",
      "Pathway - {'has_participant'} -> GeneFamily\n",
      "Pathway - {'overlaps'} -> GeneFamily\n",
      "Pathway - {'affected_by'} -> GeneFamily\n",
      "Pathway - {'has_substrate'} -> GeneFamily\n",
      "Pathway - {'interacts_with'} -> GeneFamily\n",
      "Pathway - {'has_capability'} -> GeneFamily\n",
      "Pathway - {'participates_in'} -> GeneFamily\n",
      "Pathway - {'has_input'} -> GeneFamily\n",
      "Pathway - {'enabled_by'} -> GeneFamily\n",
      "Pathway - {'has_catalyst'} -> GeneFamily\n",
      "Pathway - {'associated_with'} -> GeneFamily\n",
      "Pathway - {'has_part'} -> GeneFamily\n",
      "Pathway - {'has_upstream_actor'} -> GeneFamily\n",
      "Pathway - {'has_active_ingredient'} -> GeneFamily\n",
      "Pathway - {'primarily_composed_of'} -> GeneFamily\n",
      "Pathway - {'has_output'} -> GeneFamily\n",
      "Pathway - {'affects'} -> GeneFamily\n",
      "Pathway - {'similar_to'} -> GeneFamily\n",
      "Pathway - {'composed_primarily_of'} -> GeneFamily\n",
      "Pathway - {'located_in'} -> GeneFamily\n",
      "Pathway - {'produces'} -> GeneFamily\n",
      "Pathway - {'occurs_in'} -> GeneFamily\n",
      "Pathway - {'coexists_with'} -> GeneFamily\n",
      "Pathway - {'close_match'} -> GeneFamily\n",
      "Pathway - {'regulates'} -> GeneFamily\n",
      "GeneFamily - {'treats'} -> Publication\n",
      "GeneFamily - {'causes'} -> Publication\n",
      "GeneFamily - {'associated_with'} -> Publication\n",
      "GeneFamily - {'disrupts'} -> Publication\n",
      "GeneFamily - {'gene_associated_with_condition'} -> Publication\n",
      "GeneFamily - {'located_in'} -> Publication\n",
      "GeneFamily - {'regulates'} -> Publication\n",
      "GeneFamily - {'predisposes'} -> Publication\n",
      "GeneFamily - {'occurs_in'} -> Publication\n",
      "GeneFamily - {'affects'} -> Publication\n",
      "GeneFamily - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Gene->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'has_part'} -> Gene\n",
      "Pathway - {'affected_by'} -> Gene\n",
      "Pathway - {'actively_involves'} -> Gene\n",
      "Pathway - {'has_positive_upstream_or_within_actor'} -> Gene\n",
      "Pathway - {'has_positive_upstream_actor'} -> Gene\n",
      "Pathway - {'has_negative_upstream_or_within_actor'} -> Gene\n",
      "Pathway - {'has_upstream_actor'} -> Gene\n",
      "Pathway - {'has_upstream_or_within_actor'} -> Gene\n",
      "Pathway - {'has_negative_upstream_actor'} -> Gene\n",
      "Pathway - {'related_to'} -> Gene\n",
      "Pathway - {'caused_by'} -> Gene\n",
      "Pathway - {'enabled_by'} -> Gene\n",
      "Pathway - {'has_participant'} -> Gene\n",
      "Pathway - {'composed_primarily_of'} -> Gene\n",
      "Pathway - {'treated_by'} -> Gene\n",
      "Pathway - {'gene_associated_with_condition'} -> Gene\n",
      "Pathway - {'exacerbates'} -> Gene\n",
      "Pathway - {'increases_response_to'} -> Gene\n",
      "Pathway - {'has_output'} -> Gene\n",
      "Pathway - {'response_decreased_by'} -> Gene\n",
      "Pathway - {'disrupts'} -> Gene\n",
      "Pathway - {'has_substrate'} -> Gene\n",
      "Pathway - {'interacts_with'} -> Gene\n",
      "Pathway - {'treats'} -> Gene\n",
      "Pathway - {'condition_associated_with_gene'} -> Gene\n",
      "Pathway - {'is_ameliorated_by'} -> Gene\n",
      "Pathway - {'regulates'} -> Gene\n",
      "Pathway - {'causes'} -> Gene\n",
      "Pathway - {'response_increased_by'} -> Gene\n",
      "Pathway - {'part_of'} -> Gene\n",
      "Pathway - {'has_capability'} -> Gene\n",
      "Pathway - {'positively_regulates'} -> Gene\n",
      "Pathway - {'participates_in'} -> Gene\n",
      "Pathway - {'contributes_to'} -> Gene\n",
      "Pathway - {'has_input'} -> Gene\n",
      "Pathway - {'preceded_by'} -> Gene\n",
      "Pathway - {'has_catalyst'} -> Gene\n",
      "Pathway - {'decreases_response_to'} -> Gene\n",
      "Pathway - {'associated_with'} -> Gene\n",
      "Pathway - {'is_exacerbated_by'} -> Gene\n",
      "Pathway - {'precedes'} -> Gene\n",
      "Pathway - {'ameliorates'} -> Gene\n",
      "Pathway - {'genetic_association'} -> Gene\n",
      "Pathway - {'disrupted_by'} -> Gene\n",
      "Pathway - {'has_active_ingredient'} -> Gene\n",
      "Pathway - {'primarily_composed_of'} -> Gene\n",
      "Pathway - {'regulated_by'} -> Gene\n",
      "Pathway - {'temporally_related_to'} -> Gene\n",
      "Pathway - {'response_affected_by'} -> Gene\n",
      "Pathway - {'affects'} -> Gene\n",
      "Pathway - {'similar_to'} -> Gene\n",
      "Pathway - {'overlaps'} -> Gene\n",
      "Pathway - {'affects_response_to'} -> Gene\n",
      "Pathway - {'subclass_of'} -> Gene\n",
      "Pathway - {'capable_of'} -> Gene\n",
      "Pathway - {'derives_from'} -> Gene\n",
      "Pathway - {'physically_interacts_with'} -> Gene\n",
      "Pathway - {'gene_product_of'} -> Gene\n",
      "Pathway - {'directly_physically_interacts_with'} -> Gene\n",
      "Pathway - {'biomarker_for'} -> Gene\n",
      "Pathway - {'produces'} -> Gene\n",
      "Pathway - {'located_in'} -> Gene\n",
      "Pathway - {'colocalizes_with'} -> Gene\n",
      "Pathway - {'predisposes'} -> Gene\n",
      "Pathway - {'prevents'} -> Gene\n",
      "Pathway - {'occurs_in'} -> Gene\n",
      "Pathway - {'manifestation_of'} -> Gene\n",
      "Pathway - {'coexists_with'} -> Gene\n",
      "Pathway - {'close_match'} -> Gene\n",
      "Gene - {'correlated_with'} -> Publication\n",
      "Gene - {'prevents'} -> Publication\n",
      "Gene - {'treats'} -> Publication\n",
      "Gene - {'has_input'} -> Publication\n",
      "Gene - {'related_to'} -> Publication\n",
      "Gene - {'occurs_in'} -> Publication\n",
      "Gene - {'affects'} -> Publication\n",
      "Gene - {'exacerbates'} -> Publication\n",
      "Gene - {'coexists_with'} -> Publication\n",
      "Gene - {'disrupts'} -> Publication\n",
      "Gene - {'gene_associated_with_condition'} -> Publication\n",
      "Gene - {'located_in'} -> Publication\n",
      "Gene - {'regulates'} -> Publication\n",
      "Gene - {'biomarker_for'} -> Publication\n",
      "Gene - {'expressed_in'} -> Publication\n",
      "Gene - {'physically_interacts_with'} -> Publication\n",
      "Gene - {'predisposes'} -> Publication\n",
      "Gene - {'causes'} -> Publication\n",
      "Gene - {'precedes'} -> Publication\n",
      "Gene - {'gene_product_of'} -> Publication\n",
      "Gene - {'close_match'} -> Publication\n",
      "Gene - {'associated_with'} -> Publication\n",
      "Gene - {'subclass_of'} -> Publication\n",
      "Gene - {'has_part'} -> Publication\n",
      "Gene - {'colocalizes_with'} -> Publication\n",
      "Gene - {'interacts_with'} -> Publication\n",
      "Gene - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->MolecularEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'has_input'} -> MolecularEntity\n",
      "Pathway - {'preceded_by'} -> MolecularEntity\n",
      "Pathway - {'related_to'} -> MolecularEntity\n",
      "Pathway - {'temporally_related_to'} -> MolecularEntity\n",
      "Pathway - {'caused_by'} -> MolecularEntity\n",
      "Pathway - {'enabled_by'} -> MolecularEntity\n",
      "Pathway - {'has_participant'} -> MolecularEntity\n",
      "Pathway - {'ameliorates'} -> MolecularEntity\n",
      "Pathway - {'response_decreased_by'} -> MolecularEntity\n",
      "Pathway - {'disrupts'} -> MolecularEntity\n",
      "Pathway - {'affected_by'} -> MolecularEntity\n",
      "Pathway - {'has_substrate'} -> MolecularEntity\n",
      "Pathway - {'interacts_with'} -> MolecularEntity\n",
      "Pathway - {'treats'} -> MolecularEntity\n",
      "Pathway - {'is_ameliorated_by'} -> MolecularEntity\n",
      "Pathway - {'regulates'} -> MolecularEntity\n",
      "Pathway - {'response_increased_by'} -> MolecularEntity\n",
      "Pathway - {'part_of'} -> MolecularEntity\n",
      "Pathway - {'has_capability'} -> MolecularEntity\n",
      "Pathway - {'participates_in'} -> MolecularEntity\n",
      "Pathway - {'has_catalyst'} -> MolecularEntity\n",
      "Pathway - {'decreases_response_to'} -> MolecularEntity\n",
      "Pathway - {'associated_with'} -> MolecularEntity\n",
      "Pathway - {'has_part'} -> MolecularEntity\n",
      "Pathway - {'is_exacerbated_by'} -> MolecularEntity\n",
      "Pathway - {'precedes'} -> MolecularEntity\n",
      "Pathway - {'disrupted_by'} -> MolecularEntity\n",
      "Pathway - {'actively_involves'} -> MolecularEntity\n",
      "Pathway - {'has_upstream_actor'} -> MolecularEntity\n",
      "Pathway - {'exacerbates'} -> MolecularEntity\n",
      "Pathway - {'has_active_ingredient'} -> MolecularEntity\n",
      "Pathway - {'increases_response_to'} -> MolecularEntity\n",
      "Pathway - {'primarily_composed_of'} -> MolecularEntity\n",
      "Pathway - {'regulated_by'} -> MolecularEntity\n",
      "Pathway - {'has_output'} -> MolecularEntity\n",
      "Pathway - {'response_affected_by'} -> MolecularEntity\n",
      "Pathway - {'affects'} -> MolecularEntity\n",
      "Pathway - {'similar_to'} -> MolecularEntity\n",
      "Pathway - {'overlaps'} -> MolecularEntity\n",
      "Pathway - {'affects_response_to'} -> MolecularEntity\n",
      "Pathway - {'composed_primarily_of'} -> MolecularEntity\n",
      "Pathway - {'treated_by'} -> MolecularEntity\n",
      "Pathway - {'occurs_in'} -> MolecularEntity\n",
      "Pathway - {'coexists_with'} -> MolecularEntity\n",
      "Pathway - {'close_match'} -> MolecularEntity\n",
      "Pathway - {'subclass_of'} -> MolecularEntity\n",
      "Pathway - {'disease_has_basis_in'} -> MolecularEntity\n",
      "Pathway - {'derives_from'} -> MolecularEntity\n",
      "Pathway - {'physically_interacts_with'} -> MolecularEntity\n",
      "Pathway - {'biomarker_for'} -> MolecularEntity\n",
      "Pathway - {'produces'} -> MolecularEntity\n",
      "Pathway - {'located_in'} -> MolecularEntity\n",
      "Pathway - {'colocalizes_with'} -> MolecularEntity\n",
      "MolecularEntity - {'causes'} -> Publication\n",
      "MolecularEntity - {'associated_with'} -> Publication\n",
      "MolecularEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "MolecularEntity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularEntity - {'regulates'} -> Publication\n",
      "MolecularEntity - {'disrupts'} -> Publication\n",
      "MolecularEntity - {'located_in'} -> Publication\n",
      "MolecularEntity - {'biomarker_for'} -> Publication\n",
      "MolecularEntity - {'interacts_with'} -> Publication\n",
      "MolecularEntity - {'predisposes'} -> Publication\n",
      "MolecularEntity - {'related_to'} -> Publication\n",
      "MolecularEntity - {'has_part'} -> Publication\n",
      "MolecularEntity - {'occurs_in'} -> Publication\n",
      "MolecularEntity - {'affects'} -> Publication\n",
      "MolecularEntity - {'precedes'} -> Publication\n",
      "MolecularEntity - {'coexists_with'} -> Publication\n",
      "MolecularEntity - {'close_match'} -> Publication\n",
      "MolecularEntity - {'subclass_of'} -> Publication\n",
      "MolecularEntity - {'treats'} -> Publication\n",
      "MolecularEntity - {'physically_interacts_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->BiologicalProcessOrActivity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'overlaps'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'part_of'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'positively_regulates'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'preceded_by'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'precedes'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'process_negatively_regulates_process'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'process_positively_regulates_process'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'process_regulates_process'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'regulates'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'related_to'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'temporally_related_to'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'affects'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'caused_by'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_part'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_participant'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'negatively_regulates'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'capable_of'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'actively_involves'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_active_ingredient'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_catalyst'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'primarily_composed_of'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'actively_involved_in'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'associated_with'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'similar_to'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'affects_response_to'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'composed_primarily_of'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_upstream_actor'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'exacerbates'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'increases_response_to'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_output'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'disrupts'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'affected_by'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'treats'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'causes'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'participates_in'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'contributes_to'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_input'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'enabled_by'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'decreases_response_to'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_substrate'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'interacts_with'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'ameliorates'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'occurs_in'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'has_capability'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'enables'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'coexists_with'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'manifestation_of'} -> BiologicalProcessOrActivity\n",
      "Pathway - {'same_as'} -> BiologicalProcessOrActivity\n",
      "BiologicalProcessOrActivity - {'occurs_in'} -> Publication\n",
      "BiologicalProcessOrActivity - {'affects'} -> Publication\n",
      "BiologicalProcessOrActivity - {'has_participant'} -> Publication\n",
      "BiologicalProcessOrActivity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->BiologicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> BiologicalEntity\n",
      "Pathway - {'part_of'} -> BiologicalEntity\n",
      "Pathway - {'positively_regulates'} -> BiologicalEntity\n",
      "Pathway - {'preceded_by'} -> BiologicalEntity\n",
      "Pathway - {'precedes'} -> BiologicalEntity\n",
      "Pathway - {'process_negatively_regulates_process'} -> BiologicalEntity\n",
      "Pathway - {'process_positively_regulates_process'} -> BiologicalEntity\n",
      "Pathway - {'process_regulates_process'} -> BiologicalEntity\n",
      "Pathway - {'regulates'} -> BiologicalEntity\n",
      "Pathway - {'related_to'} -> BiologicalEntity\n",
      "Pathway - {'temporally_related_to'} -> BiologicalEntity\n",
      "Pathway - {'affects'} -> BiologicalEntity\n",
      "Pathway - {'caused_by'} -> BiologicalEntity\n",
      "Pathway - {'enabled_by'} -> BiologicalEntity\n",
      "Pathway - {'has_part'} -> BiologicalEntity\n",
      "Pathway - {'has_participant'} -> BiologicalEntity\n",
      "Pathway - {'negatively_regulates'} -> BiologicalEntity\n",
      "Pathway - {'occurs_in'} -> BiologicalEntity\n",
      "Pathway - {'overlaps'} -> BiologicalEntity\n",
      "Pathway - {'affected_by'} -> BiologicalEntity\n",
      "Pathway - {'has_substrate'} -> BiologicalEntity\n",
      "Pathway - {'interacts_with'} -> BiologicalEntity\n",
      "Pathway - {'treats'} -> BiologicalEntity\n",
      "Pathway - {'causes'} -> BiologicalEntity\n",
      "Pathway - {'has_capability'} -> BiologicalEntity\n",
      "Pathway - {'participates_in'} -> BiologicalEntity\n",
      "Pathway - {'contributes_to'} -> BiologicalEntity\n",
      "Pathway - {'has_input'} -> BiologicalEntity\n",
      "Pathway - {'has_catalyst'} -> BiologicalEntity\n",
      "Pathway - {'decreases_response_to'} -> BiologicalEntity\n",
      "Pathway - {'associated_with'} -> BiologicalEntity\n",
      "Pathway - {'ameliorates'} -> BiologicalEntity\n",
      "Pathway - {'capable_of'} -> BiologicalEntity\n",
      "Pathway - {'actively_involves'} -> BiologicalEntity\n",
      "Pathway - {'has_upstream_actor'} -> BiologicalEntity\n",
      "Pathway - {'exacerbates'} -> BiologicalEntity\n",
      "Pathway - {'has_active_ingredient'} -> BiologicalEntity\n",
      "Pathway - {'primarily_composed_of'} -> BiologicalEntity\n",
      "Pathway - {'actively_involved_in'} -> BiologicalEntity\n",
      "Pathway - {'similar_to'} -> BiologicalEntity\n",
      "Pathway - {'affects_response_to'} -> BiologicalEntity\n",
      "Pathway - {'composed_primarily_of'} -> BiologicalEntity\n",
      "Pathway - {'increases_response_to'} -> BiologicalEntity\n",
      "Pathway - {'has_output'} -> BiologicalEntity\n",
      "Pathway - {'disrupts'} -> BiologicalEntity\n",
      "Pathway - {'manifestation_of'} -> BiologicalEntity\n",
      "Pathway - {'coexists_with'} -> BiologicalEntity\n",
      "Pathway - {'close_match'} -> BiologicalEntity\n",
      "Pathway - {'contraindicated_for'} -> BiologicalEntity\n",
      "Pathway - {'diagnoses'} -> BiologicalEntity\n",
      "Pathway - {'disease_has_basis_in'} -> BiologicalEntity\n",
      "Pathway - {'derives_from'} -> BiologicalEntity\n",
      "Pathway - {'physically_interacts_with'} -> BiologicalEntity\n",
      "Pathway - {'directly_physically_interacts_with'} -> BiologicalEntity\n",
      "Pathway - {'gene_product_of'} -> BiologicalEntity\n",
      "Pathway - {'gene_associated_with_condition'} -> BiologicalEntity\n",
      "Pathway - {'has_phenotype'} -> BiologicalEntity\n",
      "Pathway - {'biomarker_for'} -> BiologicalEntity\n",
      "Pathway - {'located_in'} -> BiologicalEntity\n",
      "Pathway - {'colocalizes_with'} -> BiologicalEntity\n",
      "Pathway - {'produces'} -> BiologicalEntity\n",
      "Pathway - {'predisposes'} -> BiologicalEntity\n",
      "Pathway - {'prevents'} -> BiologicalEntity\n",
      "BiologicalEntity - {'physically_interacts_with'} -> Publication\n",
      "BiologicalEntity - {'treats'} -> Publication\n",
      "BiologicalEntity - {'has_input'} -> Publication\n",
      "BiologicalEntity - {'causes'} -> Publication\n",
      "BiologicalEntity - {'disrupts'} -> Publication\n",
      "BiologicalEntity - {'is_sequence_variant_of'} -> Publication\n",
      "BiologicalEntity - {'has_phenotype'} -> Publication\n",
      "BiologicalEntity - {'biomarker_for'} -> Publication\n",
      "BiologicalEntity - {'associated_with'} -> Publication\n",
      "BiologicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalEntity - {'has_part'} -> Publication\n",
      "BiologicalEntity - {'located_in'} -> Publication\n",
      "BiologicalEntity - {'colocalizes_with'} -> Publication\n",
      "BiologicalEntity - {'interacts_with'} -> Publication\n",
      "BiologicalEntity - {'produces'} -> Publication\n",
      "BiologicalEntity - {'predisposes'} -> Publication\n",
      "BiologicalEntity - {'related_to'} -> Publication\n",
      "BiologicalEntity - {'prevents'} -> Publication\n",
      "BiologicalEntity - {'manifestation_of'} -> Publication\n",
      "BiologicalEntity - {'precedes'} -> Publication\n",
      "BiologicalEntity - {'occurs_in'} -> Publication\n",
      "BiologicalEntity - {'affects'} -> Publication\n",
      "BiologicalEntity - {'close_match'} -> Publication\n",
      "BiologicalEntity - {'exacerbates'} -> Publication\n",
      "BiologicalEntity - {'coexists_with'} -> Publication\n",
      "BiologicalEntity - {'has_participant'} -> Publication\n",
      "BiologicalEntity - {'subclass_of'} -> Publication\n",
      "BiologicalEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Entity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Entity\n",
      "Pathway - {'overlaps'} -> Entity\n",
      "Pathway - {'part_of'} -> Entity\n",
      "Pathway - {'positively_regulates'} -> Entity\n",
      "Pathway - {'preceded_by'} -> Entity\n",
      "Pathway - {'precedes'} -> Entity\n",
      "Pathway - {'process_negatively_regulates_process'} -> Entity\n",
      "Pathway - {'process_positively_regulates_process'} -> Entity\n",
      "Pathway - {'process_regulates_process'} -> Entity\n",
      "Pathway - {'regulates'} -> Entity\n",
      "Pathway - {'related_to'} -> Entity\n",
      "Pathway - {'temporally_related_to'} -> Entity\n",
      "Pathway - {'affects'} -> Entity\n",
      "Pathway - {'caused_by'} -> Entity\n",
      "Pathway - {'enabled_by'} -> Entity\n",
      "Pathway - {'has_input'} -> Entity\n",
      "Pathway - {'has_part'} -> Entity\n",
      "Pathway - {'has_participant'} -> Entity\n",
      "Pathway - {'negatively_regulates'} -> Entity\n",
      "Pathway - {'occurs_in'} -> Entity\n",
      "Pathway - {'affected_by'} -> Entity\n",
      "Pathway - {'has_substrate'} -> Entity\n",
      "Pathway - {'interacts_with'} -> Entity\n",
      "Pathway - {'ameliorates'} -> Entity\n",
      "Pathway - {'has_capability'} -> Entity\n",
      "Pathway - {'participates_in'} -> Entity\n",
      "Pathway - {'capable_of'} -> Entity\n",
      "Pathway - {'contributes_to'} -> Entity\n",
      "Pathway - {'actively_involves'} -> Entity\n",
      "Pathway - {'has_catalyst'} -> Entity\n",
      "Pathway - {'decreases_response_to'} -> Entity\n",
      "Pathway - {'associated_with'} -> Entity\n",
      "Pathway - {'has_upstream_actor'} -> Entity\n",
      "Pathway - {'exacerbates'} -> Entity\n",
      "Pathway - {'has_active_ingredient'} -> Entity\n",
      "Pathway - {'increases_response_to'} -> Entity\n",
      "Pathway - {'primarily_composed_of'} -> Entity\n",
      "Pathway - {'actively_involved_in'} -> Entity\n",
      "Pathway - {'has_output'} -> Entity\n",
      "Pathway - {'similar_to'} -> Entity\n",
      "Pathway - {'affects_response_to'} -> Entity\n",
      "Pathway - {'treats'} -> Entity\n",
      "Pathway - {'causes'} -> Entity\n",
      "Pathway - {'composed_primarily_of'} -> Entity\n",
      "Pathway - {'disrupts'} -> Entity\n",
      "Entity - {'subclass_of'} -> Publication\n",
      "Entity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->EnvironmentalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'overlaps'} -> EnvironmentalProcess\n",
      "Pathway - {'part_of'} -> EnvironmentalProcess\n",
      "Pathway - {'related_to'} -> EnvironmentalProcess\n",
      "Pathway - {'has_part'} -> EnvironmentalProcess\n",
      "Pathway - {'composed_primarily_of'} -> EnvironmentalProcess\n",
      "Pathway - {'preceded_by'} -> EnvironmentalProcess\n",
      "Pathway - {'has_participant'} -> EnvironmentalProcess\n",
      "Pathway - {'negatively_regulates'} -> EnvironmentalProcess\n",
      "Pathway - {'interacts_with'} -> EnvironmentalProcess\n",
      "Pathway - {'ameliorates'} -> EnvironmentalProcess\n",
      "Pathway - {'regulates'} -> EnvironmentalProcess\n",
      "Pathway - {'has_capability'} -> EnvironmentalProcess\n",
      "Pathway - {'positively_regulates'} -> EnvironmentalProcess\n",
      "Pathway - {'participates_in'} -> EnvironmentalProcess\n",
      "Pathway - {'capable_of'} -> EnvironmentalProcess\n",
      "Pathway - {'actively_involves'} -> EnvironmentalProcess\n",
      "Pathway - {'decreases_response_to'} -> EnvironmentalProcess\n",
      "Pathway - {'associated_with'} -> EnvironmentalProcess\n",
      "Pathway - {'precedes'} -> EnvironmentalProcess\n",
      "Pathway - {'affects_response_to'} -> EnvironmentalProcess\n",
      "Pathway - {'process_positively_regulates_process'} -> EnvironmentalProcess\n",
      "Pathway - {'has_upstream_actor'} -> EnvironmentalProcess\n",
      "Pathway - {'process_negatively_regulates_process'} -> EnvironmentalProcess\n",
      "Pathway - {'exacerbates'} -> EnvironmentalProcess\n",
      "Pathway - {'has_active_ingredient'} -> EnvironmentalProcess\n",
      "Pathway - {'increases_response_to'} -> EnvironmentalProcess\n",
      "Pathway - {'primarily_composed_of'} -> EnvironmentalProcess\n",
      "Pathway - {'temporally_related_to'} -> EnvironmentalProcess\n",
      "Pathway - {'actively_involved_in'} -> EnvironmentalProcess\n",
      "Pathway - {'affects'} -> EnvironmentalProcess\n",
      "Pathway - {'process_regulates_process'} -> EnvironmentalProcess\n",
      "Pathway - {'disrupts'} -> EnvironmentalProcess\n",
      "Pathway - {'similar_to'} -> EnvironmentalProcess\n",
      "Pathway - {'affected_by'} -> EnvironmentalProcess\n",
      "Pathway - {'treats'} -> EnvironmentalProcess\n",
      "EnvironmentalProcess - {'subclass_of'} -> Publication\n",
      "EnvironmentalProcess - {'related_to'} -> Publication\n",
      "EnvironmentalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->OrganismalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'overlaps'} -> OrganismalEntity\n",
      "Pathway - {'related_to'} -> OrganismalEntity\n",
      "Pathway - {'caused_by'} -> OrganismalEntity\n",
      "Pathway - {'enabled_by'} -> OrganismalEntity\n",
      "Pathway - {'has_participant'} -> OrganismalEntity\n",
      "Pathway - {'has_part'} -> OrganismalEntity\n",
      "Pathway - {'occurs_in'} -> OrganismalEntity\n",
      "Pathway - {'affects_response_to'} -> OrganismalEntity\n",
      "Pathway - {'treats'} -> OrganismalEntity\n",
      "Pathway - {'causes'} -> OrganismalEntity\n",
      "Pathway - {'composed_primarily_of'} -> OrganismalEntity\n",
      "Pathway - {'preceded_by'} -> OrganismalEntity\n",
      "Pathway - {'negatively_regulates'} -> OrganismalEntity\n",
      "Pathway - {'affected_by'} -> OrganismalEntity\n",
      "Pathway - {'has_substrate'} -> OrganismalEntity\n",
      "Pathway - {'interacts_with'} -> OrganismalEntity\n",
      "Pathway - {'ameliorates'} -> OrganismalEntity\n",
      "Pathway - {'regulates'} -> OrganismalEntity\n",
      "Pathway - {'part_of'} -> OrganismalEntity\n",
      "Pathway - {'has_capability'} -> OrganismalEntity\n",
      "Pathway - {'positively_regulates'} -> OrganismalEntity\n",
      "Pathway - {'participates_in'} -> OrganismalEntity\n",
      "Pathway - {'capable_of'} -> OrganismalEntity\n",
      "Pathway - {'contributes_to'} -> OrganismalEntity\n",
      "Pathway - {'actively_involves'} -> OrganismalEntity\n",
      "Pathway - {'has_input'} -> OrganismalEntity\n",
      "Pathway - {'has_catalyst'} -> OrganismalEntity\n",
      "Pathway - {'decreases_response_to'} -> OrganismalEntity\n",
      "Pathway - {'associated_with'} -> OrganismalEntity\n",
      "Pathway - {'precedes'} -> OrganismalEntity\n",
      "Pathway - {'process_positively_regulates_process'} -> OrganismalEntity\n",
      "Pathway - {'has_upstream_actor'} -> OrganismalEntity\n",
      "Pathway - {'process_negatively_regulates_process'} -> OrganismalEntity\n",
      "Pathway - {'exacerbates'} -> OrganismalEntity\n",
      "Pathway - {'has_active_ingredient'} -> OrganismalEntity\n",
      "Pathway - {'increases_response_to'} -> OrganismalEntity\n",
      "Pathway - {'primarily_composed_of'} -> OrganismalEntity\n",
      "Pathway - {'temporally_related_to'} -> OrganismalEntity\n",
      "Pathway - {'actively_involved_in'} -> OrganismalEntity\n",
      "Pathway - {'has_output'} -> OrganismalEntity\n",
      "Pathway - {'affects'} -> OrganismalEntity\n",
      "Pathway - {'process_regulates_process'} -> OrganismalEntity\n",
      "Pathway - {'disrupts'} -> OrganismalEntity\n",
      "Pathway - {'similar_to'} -> OrganismalEntity\n",
      "OrganismalEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Behavior->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'part_of'} -> Behavior\n",
      "Pathway - {'precedes'} -> Behavior\n",
      "Pathway - {'related_to'} -> Behavior\n",
      "Pathway - {'temporally_related_to'} -> Behavior\n",
      "Pathway - {'overlaps'} -> Behavior\n",
      "Pathway - {'process_positively_regulates_process'} -> Behavior\n",
      "Pathway - {'composed_primarily_of'} -> Behavior\n",
      "Pathway - {'has_upstream_actor'} -> Behavior\n",
      "Pathway - {'exacerbates'} -> Behavior\n",
      "Pathway - {'has_active_ingredient'} -> Behavior\n",
      "Pathway - {'increases_response_to'} -> Behavior\n",
      "Pathway - {'primarily_composed_of'} -> Behavior\n",
      "Pathway - {'disrupts'} -> Behavior\n",
      "Pathway - {'affected_by'} -> Behavior\n",
      "Pathway - {'treats'} -> Behavior\n",
      "Pathway - {'causes'} -> Behavior\n",
      "Pathway - {'positively_regulates'} -> Behavior\n",
      "Pathway - {'participates_in'} -> Behavior\n",
      "Pathway - {'contributes_to'} -> Behavior\n",
      "Pathway - {'preceded_by'} -> Behavior\n",
      "Pathway - {'has_participant'} -> Behavior\n",
      "Pathway - {'decreases_response_to'} -> Behavior\n",
      "Pathway - {'has_part'} -> Behavior\n",
      "Pathway - {'negatively_regulates'} -> Behavior\n",
      "Pathway - {'interacts_with'} -> Behavior\n",
      "Pathway - {'ameliorates'} -> Behavior\n",
      "Pathway - {'regulates'} -> Behavior\n",
      "Pathway - {'has_capability'} -> Behavior\n",
      "Pathway - {'actively_involves'} -> Behavior\n",
      "Pathway - {'process_negatively_regulates_process'} -> Behavior\n",
      "Pathway - {'associated_with'} -> Behavior\n",
      "Pathway - {'affects'} -> Behavior\n",
      "Pathway - {'process_regulates_process'} -> Behavior\n",
      "Pathway - {'similar_to'} -> Behavior\n",
      "Pathway - {'affects_response_to'} -> Behavior\n",
      "Pathway - {'gene_associated_with_condition'} -> Behavior\n",
      "Pathway - {'has_phenotype'} -> Behavior\n",
      "Pathway - {'predisposes'} -> Behavior\n",
      "Pathway - {'occurs_in'} -> Behavior\n",
      "Pathway - {'coexists_with'} -> Behavior\n",
      "Pathway - {'manifestation_of'} -> Behavior\n",
      "Pathway - {'subclass_of'} -> Behavior\n",
      "Pathway - {'close_match'} -> Behavior\n",
      "Behavior - {'treats'} -> Publication\n",
      "Behavior - {'has_input'} -> Publication\n",
      "Behavior - {'subclass_of'} -> Publication\n",
      "Behavior - {'located_in'} -> Publication\n",
      "Behavior - {'interacts_with'} -> Publication\n",
      "Behavior - {'predisposes'} -> Publication\n",
      "Behavior - {'related_to'} -> Publication\n",
      "Behavior - {'occurs_in'} -> Publication\n",
      "Behavior - {'affects'} -> Publication\n",
      "Behavior - {'coexists_with'} -> Publication\n",
      "Behavior - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Annotation->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'precedes'} -> Annotation\n",
      "Pathway - {'related_to'} -> Annotation\n",
      "Pathway - {'temporally_related_to'} -> Annotation\n",
      "Pathway - {'overlaps'} -> Annotation\n",
      "Pathway - {'affects_response_to'} -> Annotation\n",
      "Pathway - {'process_positively_regulates_process'} -> Annotation\n",
      "Pathway - {'composed_primarily_of'} -> Annotation\n",
      "Pathway - {'exacerbates'} -> Annotation\n",
      "Pathway - {'increases_response_to'} -> Annotation\n",
      "Pathway - {'disrupts'} -> Annotation\n",
      "Pathway - {'affected_by'} -> Annotation\n",
      "Pathway - {'interacts_with'} -> Annotation\n",
      "Pathway - {'treats'} -> Annotation\n",
      "Pathway - {'regulates'} -> Annotation\n",
      "Pathway - {'part_of'} -> Annotation\n",
      "Pathway - {'has_capability'} -> Annotation\n",
      "Pathway - {'positively_regulates'} -> Annotation\n",
      "Pathway - {'participates_in'} -> Annotation\n",
      "Pathway - {'preceded_by'} -> Annotation\n",
      "Pathway - {'has_participant'} -> Annotation\n",
      "Pathway - {'decreases_response_to'} -> Annotation\n",
      "Pathway - {'associated_with'} -> Annotation\n",
      "Pathway - {'has_part'} -> Annotation\n",
      "Pathway - {'negatively_regulates'} -> Annotation\n",
      "Pathway - {'ameliorates'} -> Annotation\n",
      "Pathway - {'capable_of'} -> Annotation\n",
      "Pathway - {'actively_involves'} -> Annotation\n",
      "Pathway - {'has_upstream_actor'} -> Annotation\n",
      "Pathway - {'process_negatively_regulates_process'} -> Annotation\n",
      "Pathway - {'has_active_ingredient'} -> Annotation\n",
      "Pathway - {'primarily_composed_of'} -> Annotation\n",
      "Pathway - {'actively_involved_in'} -> Annotation\n",
      "Pathway - {'affects'} -> Annotation\n",
      "Pathway - {'process_regulates_process'} -> Annotation\n",
      "Pathway - {'similar_to'} -> Annotation\n",
      "Annotation - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->DiseaseOrPhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'precedes'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'related_to'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'temporally_related_to'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_participant'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'decreases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_part'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'negatively_regulates'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'capable_of'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'actively_involves'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_upstream_actor'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'process_negatively_regulates_process'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'exacerbates'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_active_ingredient'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'increases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'primarily_composed_of'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'actively_involved_in'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'affects'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'process_regulates_process'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'similar_to'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'overlaps'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'affects_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'process_positively_regulates_process'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'composed_primarily_of'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'preceded_by'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'disrupts'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'affected_by'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'ameliorates'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'treats'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'regulates'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'part_of'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_capability'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'positively_regulates'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'participates_in'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'gene_associated_with_condition'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_phenotype'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'biomarker_for'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'produces'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'located_in'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'predisposes'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'prevents'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'manifestation_of'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'occurs_in'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'coexists_with'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'close_match'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'subclass_of'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'contraindicated_for'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'diagnoses'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_input'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'causes'} -> DiseaseOrPhenotypicFeature\n",
      "Pathway - {'has_output'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->OrganismAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'precedes'} -> OrganismAttribute\n",
      "Pathway - {'related_to'} -> OrganismAttribute\n",
      "Pathway - {'temporally_related_to'} -> OrganismAttribute\n",
      "Pathway - {'part_of'} -> OrganismAttribute\n",
      "Pathway - {'primarily_composed_of'} -> OrganismAttribute\n",
      "Pathway - {'actively_involved_in'} -> OrganismAttribute\n",
      "Pathway - {'process_regulates_process'} -> OrganismAttribute\n",
      "Pathway - {'disrupts'} -> OrganismAttribute\n",
      "Pathway - {'similar_to'} -> OrganismAttribute\n",
      "Pathway - {'affected_by'} -> OrganismAttribute\n",
      "Pathway - {'treats'} -> OrganismAttribute\n",
      "Pathway - {'positively_regulates'} -> OrganismAttribute\n",
      "Pathway - {'participates_in'} -> OrganismAttribute\n",
      "Pathway - {'preceded_by'} -> OrganismAttribute\n",
      "Pathway - {'has_participant'} -> OrganismAttribute\n",
      "Pathway - {'decreases_response_to'} -> OrganismAttribute\n",
      "Pathway - {'has_part'} -> OrganismAttribute\n",
      "Pathway - {'negatively_regulates'} -> OrganismAttribute\n",
      "Pathway - {'interacts_with'} -> OrganismAttribute\n",
      "Pathway - {'ameliorates'} -> OrganismAttribute\n",
      "Pathway - {'regulates'} -> OrganismAttribute\n",
      "Pathway - {'has_capability'} -> OrganismAttribute\n",
      "Pathway - {'capable_of'} -> OrganismAttribute\n",
      "Pathway - {'actively_involves'} -> OrganismAttribute\n",
      "Pathway - {'associated_with'} -> OrganismAttribute\n",
      "Pathway - {'affects'} -> OrganismAttribute\n",
      "Pathway - {'overlaps'} -> OrganismAttribute\n",
      "Pathway - {'affects_response_to'} -> OrganismAttribute\n",
      "Pathway - {'process_positively_regulates_process'} -> OrganismAttribute\n",
      "Pathway - {'composed_primarily_of'} -> OrganismAttribute\n",
      "Pathway - {'has_upstream_actor'} -> OrganismAttribute\n",
      "Pathway - {'process_negatively_regulates_process'} -> OrganismAttribute\n",
      "Pathway - {'exacerbates'} -> OrganismAttribute\n",
      "Pathway - {'has_active_ingredient'} -> OrganismAttribute\n",
      "Pathway - {'increases_response_to'} -> OrganismAttribute\n",
      "Pathway - {'occurs_in'} -> OrganismAttribute\n",
      "OrganismAttribute - {'related_to'} -> Publication\n",
      "OrganismAttribute - {'produces'} -> Publication\n",
      "OrganismAttribute - {'has_input'} -> Publication\n",
      "OrganismAttribute - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->GenomicEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> GenomicEntity\n",
      "Pathway - {'caused_by'} -> GenomicEntity\n",
      "Pathway - {'enabled_by'} -> GenomicEntity\n",
      "Pathway - {'has_participant'} -> GenomicEntity\n",
      "Pathway - {'composed_primarily_of'} -> GenomicEntity\n",
      "Pathway - {'has_upstream_actor'} -> GenomicEntity\n",
      "Pathway - {'exacerbates'} -> GenomicEntity\n",
      "Pathway - {'has_active_ingredient'} -> GenomicEntity\n",
      "Pathway - {'increases_response_to'} -> GenomicEntity\n",
      "Pathway - {'primarily_composed_of'} -> GenomicEntity\n",
      "Pathway - {'temporally_related_to'} -> GenomicEntity\n",
      "Pathway - {'has_output'} -> GenomicEntity\n",
      "Pathway - {'disrupts'} -> GenomicEntity\n",
      "Pathway - {'affected_by'} -> GenomicEntity\n",
      "Pathway - {'treats'} -> GenomicEntity\n",
      "Pathway - {'participates_in'} -> GenomicEntity\n",
      "Pathway - {'has_input'} -> GenomicEntity\n",
      "Pathway - {'preceded_by'} -> GenomicEntity\n",
      "Pathway - {'decreases_response_to'} -> GenomicEntity\n",
      "Pathway - {'has_part'} -> GenomicEntity\n",
      "Pathway - {'has_substrate'} -> GenomicEntity\n",
      "Pathway - {'interacts_with'} -> GenomicEntity\n",
      "Pathway - {'ameliorates'} -> GenomicEntity\n",
      "Pathway - {'regulates'} -> GenomicEntity\n",
      "Pathway - {'part_of'} -> GenomicEntity\n",
      "Pathway - {'has_capability'} -> GenomicEntity\n",
      "Pathway - {'actively_involves'} -> GenomicEntity\n",
      "Pathway - {'has_catalyst'} -> GenomicEntity\n",
      "Pathway - {'associated_with'} -> GenomicEntity\n",
      "Pathway - {'affects'} -> GenomicEntity\n",
      "Pathway - {'similar_to'} -> GenomicEntity\n",
      "Pathway - {'precedes'} -> GenomicEntity\n",
      "Pathway - {'overlaps'} -> GenomicEntity\n",
      "Pathway - {'affects_response_to'} -> GenomicEntity\n",
      "GenomicEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->NucleicAcidEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> NucleicAcidEntity\n",
      "Pathway - {'caused_by'} -> NucleicAcidEntity\n",
      "Pathway - {'enabled_by'} -> NucleicAcidEntity\n",
      "Pathway - {'has_participant'} -> NucleicAcidEntity\n",
      "Pathway - {'has_input'} -> NucleicAcidEntity\n",
      "Pathway - {'preceded_by'} -> NucleicAcidEntity\n",
      "Pathway - {'decreases_response_to'} -> NucleicAcidEntity\n",
      "Pathway - {'has_part'} -> NucleicAcidEntity\n",
      "Pathway - {'has_substrate'} -> NucleicAcidEntity\n",
      "Pathway - {'precedes'} -> NucleicAcidEntity\n",
      "Pathway - {'interacts_with'} -> NucleicAcidEntity\n",
      "Pathway - {'ameliorates'} -> NucleicAcidEntity\n",
      "Pathway - {'regulates'} -> NucleicAcidEntity\n",
      "Pathway - {'has_capability'} -> NucleicAcidEntity\n",
      "Pathway - {'actively_involves'} -> NucleicAcidEntity\n",
      "Pathway - {'has_catalyst'} -> NucleicAcidEntity\n",
      "Pathway - {'temporally_related_to'} -> NucleicAcidEntity\n",
      "Pathway - {'associated_with'} -> NucleicAcidEntity\n",
      "Pathway - {'affects'} -> NucleicAcidEntity\n",
      "Pathway - {'similar_to'} -> NucleicAcidEntity\n",
      "Pathway - {'overlaps'} -> NucleicAcidEntity\n",
      "Pathway - {'affects_response_to'} -> NucleicAcidEntity\n",
      "Pathway - {'composed_primarily_of'} -> NucleicAcidEntity\n",
      "Pathway - {'has_upstream_actor'} -> NucleicAcidEntity\n",
      "Pathway - {'exacerbates'} -> NucleicAcidEntity\n",
      "Pathway - {'has_active_ingredient'} -> NucleicAcidEntity\n",
      "Pathway - {'increases_response_to'} -> NucleicAcidEntity\n",
      "Pathway - {'primarily_composed_of'} -> NucleicAcidEntity\n",
      "Pathway - {'has_output'} -> NucleicAcidEntity\n",
      "Pathway - {'disrupts'} -> NucleicAcidEntity\n",
      "Pathway - {'affected_by'} -> NucleicAcidEntity\n",
      "Pathway - {'treats'} -> NucleicAcidEntity\n",
      "Pathway - {'part_of'} -> NucleicAcidEntity\n",
      "Pathway - {'participates_in'} -> NucleicAcidEntity\n",
      "Pathway - {'located_in'} -> NucleicAcidEntity\n",
      "Pathway - {'produces'} -> NucleicAcidEntity\n",
      "Pathway - {'predisposes'} -> NucleicAcidEntity\n",
      "Pathway - {'occurs_in'} -> NucleicAcidEntity\n",
      "Pathway - {'coexists_with'} -> NucleicAcidEntity\n",
      "Pathway - {'manifestation_of'} -> NucleicAcidEntity\n",
      "Pathway - {'subclass_of'} -> NucleicAcidEntity\n",
      "Pathway - {'physically_interacts_with'} -> NucleicAcidEntity\n",
      "Pathway - {'causes'} -> NucleicAcidEntity\n",
      "NucleicAcidEntity - {'gene_associated_with_condition'} -> Publication\n",
      "NucleicAcidEntity - {'located_in'} -> Publication\n",
      "NucleicAcidEntity - {'related_to'} -> Publication\n",
      "NucleicAcidEntity - {'close_match'} -> Publication\n",
      "NucleicAcidEntity - {'subclass_of'} -> Publication\n",
      "NucleicAcidEntity - {'interacts_with'} -> Publication\n",
      "NucleicAcidEntity - {'occurs_in'} -> Publication\n",
      "NucleicAcidEntity - {'affects'} -> Publication\n",
      "NucleicAcidEntity - {'treats'} -> Publication\n",
      "NucleicAcidEntity - {'causes'} -> Publication\n",
      "NucleicAcidEntity - {'coexists_with'} -> Publication\n",
      "NucleicAcidEntity - {'disrupts'} -> Publication\n",
      "NucleicAcidEntity - {'associated_with'} -> Publication\n",
      "NucleicAcidEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->RNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> RNAProduct\n",
      "Pathway - {'has_participant'} -> RNAProduct\n",
      "Pathway - {'has_input'} -> RNAProduct\n",
      "Pathway - {'enabled_by'} -> RNAProduct\n",
      "Pathway - {'has_catalyst'} -> RNAProduct\n",
      "Pathway - {'decreases_response_to'} -> RNAProduct\n",
      "Pathway - {'associated_with'} -> RNAProduct\n",
      "Pathway - {'ameliorates'} -> RNAProduct\n",
      "Pathway - {'actively_involves'} -> RNAProduct\n",
      "Pathway - {'has_upstream_actor'} -> RNAProduct\n",
      "Pathway - {'primarily_composed_of'} -> RNAProduct\n",
      "Pathway - {'affects'} -> RNAProduct\n",
      "Pathway - {'similar_to'} -> RNAProduct\n",
      "Pathway - {'overlaps'} -> RNAProduct\n",
      "Pathway - {'affects_response_to'} -> RNAProduct\n",
      "Pathway - {'composed_primarily_of'} -> RNAProduct\n",
      "Pathway - {'exacerbates'} -> RNAProduct\n",
      "Pathway - {'increases_response_to'} -> RNAProduct\n",
      "Pathway - {'has_output'} -> RNAProduct\n",
      "Pathway - {'disrupts'} -> RNAProduct\n",
      "Pathway - {'affected_by'} -> RNAProduct\n",
      "Pathway - {'has_substrate'} -> RNAProduct\n",
      "Pathway - {'interacts_with'} -> RNAProduct\n",
      "Pathway - {'treats'} -> RNAProduct\n",
      "Pathway - {'regulates'} -> RNAProduct\n",
      "Pathway - {'has_capability'} -> RNAProduct\n",
      "Pathway - {'participates_in'} -> RNAProduct\n",
      "Pathway - {'subclass_of'} -> RNAProduct\n",
      "Pathway - {'has_part'} -> RNAProduct\n",
      "Pathway - {'produces'} -> RNAProduct\n",
      "Pathway - {'occurs_in'} -> RNAProduct\n",
      "Pathway - {'coexists_with'} -> RNAProduct\n",
      "RNAProduct - {'located_in'} -> Publication\n",
      "RNAProduct - {'regulates'} -> Publication\n",
      "RNAProduct - {'related_to'} -> Publication\n",
      "RNAProduct - {'affects'} -> Publication\n",
      "RNAProduct - {'subclass_of'} -> Publication\n",
      "RNAProduct - {'disrupts'} -> Publication\n",
      "RNAProduct - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->ClinicalAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'interacts_with'} -> ClinicalAttribute\n",
      "Pathway - {'associated_with'} -> ClinicalAttribute\n",
      "Pathway - {'affects'} -> ClinicalAttribute\n",
      "Pathway - {'similar_to'} -> ClinicalAttribute\n",
      "Pathway - {'related_to'} -> ClinicalAttribute\n",
      "Pathway - {'overlaps'} -> ClinicalAttribute\n",
      "Pathway - {'composed_primarily_of'} -> ClinicalAttribute\n",
      "Pathway - {'has_upstream_actor'} -> ClinicalAttribute\n",
      "Pathway - {'primarily_composed_of'} -> ClinicalAttribute\n",
      "Pathway - {'affected_by'} -> ClinicalAttribute\n",
      "Pathway - {'participates_in'} -> ClinicalAttribute\n",
      "Pathway - {'has_participant'} -> ClinicalAttribute\n",
      "Pathway - {'precedes'} -> ClinicalAttribute\n",
      "Pathway - {'coexists_with'} -> ClinicalAttribute\n",
      "ClinicalAttribute - {'correlated_with'} -> Publication\n",
      "ClinicalAttribute - {'has_input'} -> Publication\n",
      "ClinicalAttribute - {'subclass_of'} -> Publication\n",
      "ClinicalAttribute - {'related_to'} -> Publication\n",
      "ClinicalAttribute - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->NoncodingRNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'has_catalyst'} -> NoncodingRNAProduct\n",
      "Pathway - {'primarily_composed_of'} -> NoncodingRNAProduct\n",
      "Pathway - {'associated_with'} -> NoncodingRNAProduct\n",
      "Pathway - {'affects'} -> NoncodingRNAProduct\n",
      "Pathway - {'similar_to'} -> NoncodingRNAProduct\n",
      "Pathway - {'related_to'} -> NoncodingRNAProduct\n",
      "Pathway - {'overlaps'} -> NoncodingRNAProduct\n",
      "Pathway - {'composed_primarily_of'} -> NoncodingRNAProduct\n",
      "Pathway - {'has_upstream_actor'} -> NoncodingRNAProduct\n",
      "Pathway - {'has_output'} -> NoncodingRNAProduct\n",
      "Pathway - {'affected_by'} -> NoncodingRNAProduct\n",
      "Pathway - {'participates_in'} -> NoncodingRNAProduct\n",
      "Pathway - {'has_input'} -> NoncodingRNAProduct\n",
      "Pathway - {'enabled_by'} -> NoncodingRNAProduct\n",
      "Pathway - {'has_participant'} -> NoncodingRNAProduct\n",
      "Pathway - {'has_substrate'} -> NoncodingRNAProduct\n",
      "Pathway - {'interacts_with'} -> NoncodingRNAProduct\n",
      "Pathway - {'has_capability'} -> NoncodingRNAProduct\n",
      "Pathway - {'actively_involves'} -> NoncodingRNAProduct\n",
      "Pathway - {'produces'} -> NoncodingRNAProduct\n",
      "Pathway - {'occurs_in'} -> NoncodingRNAProduct\n",
      "NoncodingRNAProduct - {'affects'} -> Publication\n",
      "NoncodingRNAProduct - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Device->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> Device\n",
      "Pathway - {'has_participant'} -> Device\n",
      "Pathway - {'causes'} -> Device\n",
      "Pathway - {'has_output'} -> Device\n",
      "Pathway - {'disrupts'} -> Device\n",
      "Pathway - {'has_part'} -> Device\n",
      "Pathway - {'located_in'} -> Device\n",
      "Pathway - {'colocalizes_with'} -> Device\n",
      "Pathway - {'interacts_with'} -> Device\n",
      "Pathway - {'produces'} -> Device\n",
      "Pathway - {'predisposes'} -> Device\n",
      "Pathway - {'occurs_in'} -> Device\n",
      "Pathway - {'affects'} -> Device\n",
      "Pathway - {'coexists_with'} -> Device\n",
      "Pathway - {'subclass_of'} -> Device\n",
      "Pathway - {'regulates'} -> Device\n",
      "Pathway - {'has_input'} -> Device\n",
      "Device - {'subclass_of'} -> Publication\n",
      "Device - {'diagnoses'} -> Publication\n",
      "Device - {'related_to'} -> Publication\n",
      "Device - {'occurs_in'} -> Publication\n",
      "Device - {'affects'} -> Publication\n",
      "Device - {'treats'} -> Publication\n",
      "Device - {'has_input'} -> Publication\n",
      "Device - {'coexists_with'} -> Publication\n",
      "Device - {'regulates'} -> Publication\n",
      "Device - {'disrupts'} -> Publication\n",
      "Device - {'biomarker_for'} -> Publication\n",
      "Device - {'gene_associated_with_condition'} -> Publication\n",
      "Device - {'has_part'} -> Publication\n",
      "Device - {'located_in'} -> Publication\n",
      "Device - {'causes'} -> Publication\n",
      "Device - {'precedes'} -> Publication\n",
      "Device - {'close_match'} -> Publication\n",
      "Device - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->InformationContentEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'has_upstream_actor'} -> InformationContentEntity\n",
      "Pathway - {'primarily_composed_of'} -> InformationContentEntity\n",
      "Pathway - {'affects'} -> InformationContentEntity\n",
      "Pathway - {'similar_to'} -> InformationContentEntity\n",
      "Pathway - {'affected_by'} -> InformationContentEntity\n",
      "Pathway - {'causes'} -> InformationContentEntity\n",
      "Pathway - {'participates_in'} -> InformationContentEntity\n",
      "Pathway - {'contributes_to'} -> InformationContentEntity\n",
      "Pathway - {'has_participant'} -> InformationContentEntity\n",
      "Pathway - {'interacts_with'} -> InformationContentEntity\n",
      "Pathway - {'associated_with'} -> InformationContentEntity\n",
      "Pathway - {'related_to'} -> InformationContentEntity\n",
      "Pathway - {'overlaps'} -> InformationContentEntity\n",
      "Pathway - {'composed_primarily_of'} -> InformationContentEntity\n",
      "Pathway - {'gene_associated_with_condition'} -> InformationContentEntity\n",
      "Pathway - {'produces'} -> InformationContentEntity\n",
      "Pathway - {'has_part'} -> InformationContentEntity\n",
      "Pathway - {'located_in'} -> InformationContentEntity\n",
      "Pathway - {'colocalizes_with'} -> InformationContentEntity\n",
      "Pathway - {'predisposes'} -> InformationContentEntity\n",
      "Pathway - {'precedes'} -> InformationContentEntity\n",
      "Pathway - {'prevents'} -> InformationContentEntity\n",
      "Pathway - {'manifestation_of'} -> InformationContentEntity\n",
      "Pathway - {'occurs_in'} -> InformationContentEntity\n",
      "Pathway - {'coexists_with'} -> InformationContentEntity\n",
      "Pathway - {'close_match'} -> InformationContentEntity\n",
      "Pathway - {'exacerbates'} -> InformationContentEntity\n",
      "Pathway - {'subclass_of'} -> InformationContentEntity\n",
      "Pathway - {'contraindicated_for'} -> InformationContentEntity\n",
      "Pathway - {'regulates'} -> InformationContentEntity\n",
      "Pathway - {'diagnoses'} -> InformationContentEntity\n",
      "Pathway - {'treats'} -> InformationContentEntity\n",
      "Pathway - {'has_input'} -> InformationContentEntity\n",
      "Pathway - {'physically_interacts_with'} -> InformationContentEntity\n",
      "Pathway - {'has_output'} -> InformationContentEntity\n",
      "Pathway - {'disease_has_location'} -> InformationContentEntity\n",
      "Pathway - {'has_phenotype'} -> InformationContentEntity\n",
      "Pathway - {'disrupts'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Publication\n",
      "InformationContentEntity - {'subclass_of'} -> Publication\n",
      "InformationContentEntity - {'related_to'} -> Publication\n",
      "InformationContentEntity - {'disrupts'} -> Publication\n",
      "InformationContentEntity - {'associated_with'} -> Publication\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Publication\n",
      "InformationContentEntity - {'regulates'} -> Publication\n",
      "InformationContentEntity - {'biomarker_for'} -> Publication\n",
      "InformationContentEntity - {'produces'} -> Publication\n",
      "InformationContentEntity - {'has_part'} -> Publication\n",
      "InformationContentEntity - {'located_in'} -> Publication\n",
      "InformationContentEntity - {'predisposes'} -> Publication\n",
      "InformationContentEntity - {'prevents'} -> Publication\n",
      "InformationContentEntity - {'affects'} -> Publication\n",
      "InformationContentEntity - {'manifestation_of'} -> Publication\n",
      "InformationContentEntity - {'precedes'} -> Publication\n",
      "InformationContentEntity - {'close_match'} -> Publication\n",
      "InformationContentEntity - {'interacts_with'} -> Publication\n",
      "InformationContentEntity - {'diagnoses'} -> Publication\n",
      "InformationContentEntity - {'treats'} -> Publication\n",
      "InformationContentEntity - {'occurs_in'} -> Publication\n",
      "InformationContentEntity - {'has_input'} -> Publication\n",
      "InformationContentEntity - {'exacerbates'} -> Publication\n",
      "InformationContentEntity - {'causes'} -> Publication\n",
      "InformationContentEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Phenomenon->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> Phenomenon\n",
      "Pathway - {'causes'} -> Phenomenon\n",
      "Pathway - {'contributes_to'} -> Phenomenon\n",
      "Pathway - {'has_output'} -> Phenomenon\n",
      "Pathway - {'has_phenotype'} -> Phenomenon\n",
      "Pathway - {'disrupts'} -> Phenomenon\n",
      "Pathway - {'associated_with'} -> Phenomenon\n",
      "Pathway - {'gene_associated_with_condition'} -> Phenomenon\n",
      "Pathway - {'biomarker_for'} -> Phenomenon\n",
      "Pathway - {'interacts_with'} -> Phenomenon\n",
      "Pathway - {'produces'} -> Phenomenon\n",
      "Pathway - {'has_part'} -> Phenomenon\n",
      "Pathway - {'located_in'} -> Phenomenon\n",
      "Pathway - {'predisposes'} -> Phenomenon\n",
      "Pathway - {'precedes'} -> Phenomenon\n",
      "Pathway - {'prevents'} -> Phenomenon\n",
      "Pathway - {'affects'} -> Phenomenon\n",
      "Pathway - {'manifestation_of'} -> Phenomenon\n",
      "Pathway - {'occurs_in'} -> Phenomenon\n",
      "Pathway - {'coexists_with'} -> Phenomenon\n",
      "Pathway - {'close_match'} -> Phenomenon\n",
      "Pathway - {'exacerbates'} -> Phenomenon\n",
      "Pathway - {'has_participant'} -> Phenomenon\n",
      "Pathway - {'subclass_of'} -> Phenomenon\n",
      "Pathway - {'capable_of'} -> Phenomenon\n",
      "Pathway - {'regulates'} -> Phenomenon\n",
      "Pathway - {'treats'} -> Phenomenon\n",
      "Pathway - {'has_input'} -> Phenomenon\n",
      "Phenomenon - {'associated_with'} -> Publication\n",
      "Phenomenon - {'subclass_of'} -> Publication\n",
      "Phenomenon - {'interacts_with'} -> Publication\n",
      "Phenomenon - {'produces'} -> Publication\n",
      "Phenomenon - {'related_to'} -> Publication\n",
      "Phenomenon - {'has_part'} -> Publication\n",
      "Phenomenon - {'diagnoses'} -> Publication\n",
      "Phenomenon - {'occurs_in'} -> Publication\n",
      "Phenomenon - {'affects'} -> Publication\n",
      "Phenomenon - {'coexists_with'} -> Publication\n",
      "Phenomenon - {'close_match'} -> Publication\n",
      "Phenomenon - {'disrupts'} -> Publication\n",
      "Phenomenon - {'regulates'} -> Publication\n",
      "Phenomenon - {'located_in'} -> Publication\n",
      "Phenomenon - {'has_input'} -> Publication\n",
      "Phenomenon - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->IndividualOrganism->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'affected_by'} -> IndividualOrganism\n",
      "Pathway - {'has_substrate'} -> IndividualOrganism\n",
      "Pathway - {'interacts_with'} -> IndividualOrganism\n",
      "Pathway - {'has_capability'} -> IndividualOrganism\n",
      "Pathway - {'participates_in'} -> IndividualOrganism\n",
      "Pathway - {'actively_involves'} -> IndividualOrganism\n",
      "Pathway - {'has_input'} -> IndividualOrganism\n",
      "Pathway - {'enabled_by'} -> IndividualOrganism\n",
      "Pathway - {'has_participant'} -> IndividualOrganism\n",
      "Pathway - {'has_catalyst'} -> IndividualOrganism\n",
      "Pathway - {'associated_with'} -> IndividualOrganism\n",
      "Pathway - {'has_upstream_actor'} -> IndividualOrganism\n",
      "Pathway - {'primarily_composed_of'} -> IndividualOrganism\n",
      "Pathway - {'has_output'} -> IndividualOrganism\n",
      "Pathway - {'affects'} -> IndividualOrganism\n",
      "Pathway - {'similar_to'} -> IndividualOrganism\n",
      "Pathway - {'related_to'} -> IndividualOrganism\n",
      "Pathway - {'overlaps'} -> IndividualOrganism\n",
      "Pathway - {'composed_primarily_of'} -> IndividualOrganism\n",
      "Pathway - {'regulates'} -> IndividualOrganism\n",
      "Pathway - {'in_taxon'} -> IndividualOrganism\n",
      "Pathway - {'has_part'} -> IndividualOrganism\n",
      "Pathway - {'located_in'} -> IndividualOrganism\n",
      "Pathway - {'occurs_in'} -> IndividualOrganism\n",
      "Pathway - {'close_match'} -> IndividualOrganism\n",
      "Pathway - {'subclass_of'} -> IndividualOrganism\n",
      "IndividualOrganism - {'close_match'} -> Publication\n",
      "IndividualOrganism - {'subclass_of'} -> Publication\n",
      "IndividualOrganism - {'occurs_in'} -> Publication\n",
      "IndividualOrganism - {'located_in'} -> Publication\n",
      "IndividualOrganism - {'related_to'} -> Publication\n",
      "IndividualOrganism - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Treatment->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'composed_primarily_of'} -> Treatment\n",
      "Pathway - {'has_upstream_actor'} -> Treatment\n",
      "Pathway - {'primarily_composed_of'} -> Treatment\n",
      "Pathway - {'similar_to'} -> Treatment\n",
      "Pathway - {'affected_by'} -> Treatment\n",
      "Pathway - {'participates_in'} -> Treatment\n",
      "Pathway - {'has_participant'} -> Treatment\n",
      "Pathway - {'interacts_with'} -> Treatment\n",
      "Pathway - {'associated_with'} -> Treatment\n",
      "Pathway - {'affects'} -> Treatment\n",
      "Pathway - {'related_to'} -> Treatment\n",
      "Pathway - {'overlaps'} -> Treatment\n",
      "Pathway - {'manifestation_of'} -> Treatment\n",
      "Pathway - {'subclass_of'} -> Treatment\n",
      "Treatment - {'subclass_of'} -> Publication\n",
      "Treatment - {'occurs_in'} -> Publication\n",
      "Treatment - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->PathologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'affects_response_to'} -> PathologicalProcess\n",
      "Pathway - {'disrupted_by'} -> PathologicalProcess\n",
      "Pathway - {'exacerbates'} -> PathologicalProcess\n",
      "Pathway - {'increases_response_to'} -> PathologicalProcess\n",
      "Pathway - {'affects'} -> PathologicalProcess\n",
      "Pathway - {'disrupts'} -> PathologicalProcess\n",
      "Pathway - {'affected_by'} -> PathologicalProcess\n",
      "Pathway - {'treats'} -> PathologicalProcess\n",
      "Pathway - {'causes'} -> PathologicalProcess\n",
      "Pathway - {'ameliorates'} -> PathologicalProcess\n",
      "Pathway - {'regulates'} -> PathologicalProcess\n",
      "Pathway - {'contributes_to'} -> PathologicalProcess\n",
      "Pathway - {'caused_by'} -> PathologicalProcess\n",
      "Pathway - {'decreases_response_to'} -> PathologicalProcess\n",
      "Pathway - {'associated_with'} -> PathologicalProcess\n",
      "Pathway - {'related_to'} -> PathologicalProcess\n",
      "Pathway - {'has_participant'} -> PathologicalProcess\n",
      "Pathway - {'subclass_of'} -> PathologicalProcess\n",
      "Pathway - {'contraindicated_for'} -> PathologicalProcess\n",
      "Pathway - {'diagnoses'} -> PathologicalProcess\n",
      "Pathway - {'has_output'} -> PathologicalProcess\n",
      "Pathway - {'actively_involved_in'} -> PathologicalProcess\n",
      "Pathway - {'gene_associated_with_condition'} -> PathologicalProcess\n",
      "Pathway - {'has_phenotype'} -> PathologicalProcess\n",
      "Pathway - {'interacts_with'} -> PathologicalProcess\n",
      "Pathway - {'produces'} -> PathologicalProcess\n",
      "Pathway - {'has_part'} -> PathologicalProcess\n",
      "Pathway - {'located_in'} -> PathologicalProcess\n",
      "Pathway - {'predisposes'} -> PathologicalProcess\n",
      "Pathway - {'precedes'} -> PathologicalProcess\n",
      "Pathway - {'prevents'} -> PathologicalProcess\n",
      "Pathway - {'manifestation_of'} -> PathologicalProcess\n",
      "Pathway - {'occurs_in'} -> PathologicalProcess\n",
      "Pathway - {'coexists_with'} -> PathologicalProcess\n",
      "Pathway - {'close_match'} -> PathologicalProcess\n",
      "PathologicalProcess - {'manifestation_of'} -> Publication\n",
      "PathologicalProcess - {'occurs_in'} -> Publication\n",
      "PathologicalProcess - {'coexists_with'} -> Publication\n",
      "PathologicalProcess - {'exacerbates'} -> Publication\n",
      "PathologicalProcess - {'disrupts'} -> Publication\n",
      "PathologicalProcess - {'located_in'} -> Publication\n",
      "PathologicalProcess - {'regulates'} -> Publication\n",
      "PathologicalProcess - {'treats'} -> Publication\n",
      "PathologicalProcess - {'precedes'} -> Publication\n",
      "PathologicalProcess - {'causes'} -> Publication\n",
      "PathologicalProcess - {'close_match'} -> Publication\n",
      "PathologicalProcess - {'associated_with'} -> Publication\n",
      "PathologicalProcess - {'subclass_of'} -> Publication\n",
      "PathologicalProcess - {'produces'} -> Publication\n",
      "PathologicalProcess - {'predisposes'} -> Publication\n",
      "PathologicalProcess - {'related_to'} -> Publication\n",
      "PathologicalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->CellLine->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'ameliorates'} -> CellLine\n",
      "Pathway - {'treats'} -> CellLine\n",
      "Pathway - {'affects'} -> CellLine\n",
      "Pathway - {'related_to'} -> CellLine\n",
      "Pathway - {'causes'} -> CellLine\n",
      "Pathway - {'has_output'} -> CellLine\n",
      "Pathway - {'disease_has_location'} -> CellLine\n",
      "Pathway - {'disrupts'} -> CellLine\n",
      "Pathway - {'interacts_with'} -> CellLine\n",
      "Pathway - {'has_part'} -> CellLine\n",
      "Pathway - {'located_in'} -> CellLine\n",
      "Pathway - {'occurs_in'} -> CellLine\n",
      "Pathway - {'close_match'} -> CellLine\n",
      "Pathway - {'has_participant'} -> CellLine\n",
      "Pathway - {'subclass_of'} -> CellLine\n",
      "Pathway - {'regulates'} -> CellLine\n",
      "Pathway - {'has_input'} -> CellLine\n",
      "Pathway - {'derives_from'} -> CellLine\n",
      "CellLine - {'gene_associated_with_condition'} -> Publication\n",
      "CellLine - {'located_in'} -> Publication\n",
      "CellLine - {'produces'} -> Publication\n",
      "CellLine - {'related_to'} -> Publication\n",
      "CellLine - {'affects'} -> Publication\n",
      "CellLine - {'close_match'} -> Publication\n",
      "CellLine - {'subclass_of'} -> Publication\n",
      "CellLine - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Activity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> Activity\n",
      "Pathway - {'treats'} -> Activity\n",
      "Pathway - {'ameliorates'} -> Activity\n",
      "Pathway - {'has_participant'} -> Activity\n",
      "Pathway - {'affects'} -> Activity\n",
      "Pathway - {'produces'} -> Activity\n",
      "Pathway - {'prevents'} -> Activity\n",
      "Pathway - {'predisposes'} -> Activity\n",
      "Pathway - {'precedes'} -> Activity\n",
      "Pathway - {'occurs_in'} -> Activity\n",
      "Pathway - {'manifestation_of'} -> Activity\n",
      "Pathway - {'exacerbates'} -> Activity\n",
      "Pathway - {'coexists_with'} -> Activity\n",
      "Pathway - {'close_match'} -> Activity\n",
      "Pathway - {'subclass_of'} -> Activity\n",
      "Pathway - {'regulates'} -> Activity\n",
      "Pathway - {'contraindicated_for'} -> Activity\n",
      "Pathway - {'diagnoses'} -> Activity\n",
      "Pathway - {'has_input'} -> Activity\n",
      "Pathway - {'causes'} -> Activity\n",
      "Pathway - {'associated_with'} -> Activity\n",
      "Pathway - {'has_phenotype'} -> Activity\n",
      "Pathway - {'disrupts'} -> Activity\n",
      "Pathway - {'gene_associated_with_condition'} -> Activity\n",
      "Pathway - {'has_part'} -> Activity\n",
      "Pathway - {'located_in'} -> Activity\n",
      "Activity - {'correlated_with'} -> Publication\n",
      "Activity - {'causes'} -> Publication\n",
      "Activity - {'disrupts'} -> Publication\n",
      "Activity - {'subclass_of'} -> Publication\n",
      "Activity - {'located_in'} -> Publication\n",
      "Activity - {'interacts_with'} -> Publication\n",
      "Activity - {'produces'} -> Publication\n",
      "Activity - {'has_part'} -> Publication\n",
      "Activity - {'diagnoses'} -> Publication\n",
      "Activity - {'prevents'} -> Publication\n",
      "Activity - {'predisposes'} -> Publication\n",
      "Activity - {'precedes'} -> Publication\n",
      "Activity - {'related_to'} -> Publication\n",
      "Activity - {'occurs_in'} -> Publication\n",
      "Activity - {'affects'} -> Publication\n",
      "Activity - {'coexists_with'} -> Publication\n",
      "Activity - {'close_match'} -> Publication\n",
      "Activity - {'regulates'} -> Publication\n",
      "Activity - {'treats'} -> Publication\n",
      "Activity - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Food->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'affected_by'} -> Food\n",
      "Pathway - {'caused_by'} -> Food\n",
      "Pathway - {'disrupted_by'} -> Food\n",
      "Pathway - {'related_to'} -> Food\n",
      "Pathway - {'causes'} -> Food\n",
      "Pathway - {'associated_with'} -> Food\n",
      "Pathway - {'produces'} -> Food\n",
      "Pathway - {'has_part'} -> Food\n",
      "Pathway - {'located_in'} -> Food\n",
      "Pathway - {'interacts_with'} -> Food\n",
      "Pathway - {'affects'} -> Food\n",
      "Pathway - {'precedes'} -> Food\n",
      "Pathway - {'occurs_in'} -> Food\n",
      "Pathway - {'coexists_with'} -> Food\n",
      "Pathway - {'has_participant'} -> Food\n",
      "Pathway - {'subclass_of'} -> Food\n",
      "Pathway - {'regulates'} -> Food\n",
      "Pathway - {'physically_interacts_with'} -> Food\n",
      "Food - {'regulates'} -> Publication\n",
      "Food - {'associated_with'} -> Publication\n",
      "Food - {'located_in'} -> Publication\n",
      "Food - {'predisposes'} -> Publication\n",
      "Food - {'related_to'} -> Publication\n",
      "Food - {'occurs_in'} -> Publication\n",
      "Food - {'affects'} -> Publication\n",
      "Food - {'precedes'} -> Publication\n",
      "Food - {'subclass_of'} -> Publication\n",
      "Food - {'treats'} -> Publication\n",
      "Food - {'causes'} -> Publication\n",
      "Food - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Agent->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'interacts_with'} -> Agent\n",
      "Pathway - {'has_part'} -> Agent\n",
      "Pathway - {'located_in'} -> Agent\n",
      "Pathway - {'predisposes'} -> Agent\n",
      "Pathway - {'related_to'} -> Agent\n",
      "Pathway - {'prevents'} -> Agent\n",
      "Pathway - {'affects'} -> Agent\n",
      "Pathway - {'manifestation_of'} -> Agent\n",
      "Pathway - {'precedes'} -> Agent\n",
      "Pathway - {'occurs_in'} -> Agent\n",
      "Pathway - {'coexists_with'} -> Agent\n",
      "Pathway - {'exacerbates'} -> Agent\n",
      "Pathway - {'subclass_of'} -> Agent\n",
      "Pathway - {'regulates'} -> Agent\n",
      "Pathway - {'treats'} -> Agent\n",
      "Pathway - {'causes'} -> Agent\n",
      "Pathway - {'disrupts'} -> Agent\n",
      "Pathway - {'associated_with'} -> Agent\n",
      "Pathway - {'gene_associated_with_condition'} -> Agent\n",
      "Agent - {'has_part'} -> Publication\n",
      "Agent - {'interacts_with'} -> Publication\n",
      "Agent - {'produces'} -> Publication\n",
      "Agent - {'predisposes'} -> Publication\n",
      "Agent - {'related_to'} -> Publication\n",
      "Agent - {'prevents'} -> Publication\n",
      "Agent - {'precedes'} -> Publication\n",
      "Agent - {'occurs_in'} -> Publication\n",
      "Agent - {'affects'} -> Publication\n",
      "Agent - {'exacerbates'} -> Publication\n",
      "Agent - {'coexists_with'} -> Publication\n",
      "Agent - {'close_match'} -> Publication\n",
      "Agent - {'subclass_of'} -> Publication\n",
      "Agent - {'regulates'} -> Publication\n",
      "Agent - {'diagnoses'} -> Publication\n",
      "Agent - {'treats'} -> Publication\n",
      "Agent - {'has_input'} -> Publication\n",
      "Agent - {'causes'} -> Publication\n",
      "Agent - {'located_in'} -> Publication\n",
      "Agent - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->LifeStage->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> LifeStage\n",
      "Pathway - {'occurs_in'} -> LifeStage\n",
      "Pathway - {'affects'} -> LifeStage\n",
      "Pathway - {'treats'} -> LifeStage\n",
      "Pathway - {'located_in'} -> LifeStage\n",
      "LifeStage - {'subclass_of'} -> Publication\n",
      "LifeStage - {'has_part'} -> Publication\n",
      "LifeStage - {'interacts_with'} -> Publication\n",
      "LifeStage - {'produces'} -> Publication\n",
      "LifeStage - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->MicroRNA->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'produces'} -> MicroRNA\n",
      "Pathway - {'has_part'} -> MicroRNA\n",
      "Pathway - {'interacts_with'} -> MicroRNA\n",
      "Pathway - {'affects'} -> MicroRNA\n",
      "Pathway - {'coexists_with'} -> MicroRNA\n",
      "Pathway - {'regulates'} -> MicroRNA\n",
      "Pathway - {'associated_with'} -> MicroRNA\n",
      "MicroRNA - {'causes'} -> Publication\n",
      "MicroRNA - {'disrupts'} -> Publication\n",
      "MicroRNA - {'associated_with'} -> Publication\n",
      "MicroRNA - {'gene_associated_with_condition'} -> Publication\n",
      "MicroRNA - {'subclass_of'} -> Publication\n",
      "MicroRNA - {'predisposes'} -> Publication\n",
      "MicroRNA - {'located_in'} -> Publication\n",
      "MicroRNA - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Vitamin->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'related_to'} -> Vitamin\n",
      "Pathway - {'coexists_with'} -> Vitamin\n",
      "Pathway - {'has_participant'} -> Vitamin\n",
      "Pathway - {'regulates'} -> Vitamin\n",
      "Pathway - {'associated_with'} -> Vitamin\n",
      "Pathway - {'interacts_with'} -> Vitamin\n",
      "Pathway - {'produces'} -> Vitamin\n",
      "Vitamin - {'regulates'} -> Publication\n",
      "Vitamin - {'treats'} -> Publication\n",
      "Vitamin - {'disrupts'} -> Publication\n",
      "Vitamin - {'subclass_of'} -> Publication\n",
      "Vitamin - {'related_to'} -> Publication\n",
      "Vitamin - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Publication\n",
      "Pathway - {'regulates'} -> Publication\n",
      "Pathway - {'treats'} -> Publication\n",
      "Pathway - {'has_input'} -> Publication\n",
      "Pathway - {'causes'} -> Publication\n",
      "Pathway - {'associated_with'} -> Publication\n",
      "Pathway - {'gene_associated_with_condition'} -> Publication\n",
      "Pathway - {'disrupts'} -> Publication\n",
      "Pathway - {'located_in'} -> Publication\n",
      "Pathway - {'produces'} -> Publication\n",
      "Pathway - {'predisposes'} -> Publication\n",
      "Pathway - {'related_to'} -> Publication\n",
      "Pathway - {'precedes'} -> Publication\n",
      "Pathway - {'occurs_in'} -> Publication\n",
      "Pathway - {'affects'} -> Publication\n",
      "Pathway - {'manifestation_of'} -> Publication\n",
      "Pathway - {'coexists_with'} -> Publication\n",
      "Pathway - {'close_match'} -> Publication\n",
      "Pathway - {'has_participant'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->ClinicalIntervention->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'associated_with'} -> ClinicalIntervention\n",
      "Pathway - {'gene_associated_with_condition'} -> ClinicalIntervention\n",
      "Pathway - {'has_phenotype'} -> ClinicalIntervention\n",
      "Pathway - {'disrupts'} -> ClinicalIntervention\n",
      "Pathway - {'biomarker_for'} -> ClinicalIntervention\n",
      "Pathway - {'has_part'} -> ClinicalIntervention\n",
      "Pathway - {'located_in'} -> ClinicalIntervention\n",
      "Pathway - {'interacts_with'} -> ClinicalIntervention\n",
      "Pathway - {'produces'} -> ClinicalIntervention\n",
      "Pathway - {'predisposes'} -> ClinicalIntervention\n",
      "Pathway - {'related_to'} -> ClinicalIntervention\n",
      "Pathway - {'precedes'} -> ClinicalIntervention\n",
      "Pathway - {'occurs_in'} -> ClinicalIntervention\n",
      "Pathway - {'affects'} -> ClinicalIntervention\n",
      "Pathway - {'manifestation_of'} -> ClinicalIntervention\n",
      "Pathway - {'coexists_with'} -> ClinicalIntervention\n",
      "Pathway - {'close_match'} -> ClinicalIntervention\n",
      "Pathway - {'has_participant'} -> ClinicalIntervention\n",
      "Pathway - {'subclass_of'} -> ClinicalIntervention\n",
      "Pathway - {'regulates'} -> ClinicalIntervention\n",
      "Pathway - {'contraindicated_for'} -> ClinicalIntervention\n",
      "Pathway - {'has_input'} -> ClinicalIntervention\n",
      "Pathway - {'causes'} -> ClinicalIntervention\n",
      "ClinicalIntervention - {'correlated_with'} -> Publication\n",
      "ClinicalIntervention - {'regulates'} -> Publication\n",
      "ClinicalIntervention - {'diagnoses'} -> Publication\n",
      "ClinicalIntervention - {'treats'} -> Publication\n",
      "ClinicalIntervention - {'has_input'} -> Publication\n",
      "ClinicalIntervention - {'causes'} -> Publication\n",
      "ClinicalIntervention - {'disrupts'} -> Publication\n",
      "ClinicalIntervention - {'associated_with'} -> Publication\n",
      "ClinicalIntervention - {'has_part'} -> Publication\n",
      "ClinicalIntervention - {'located_in'} -> Publication\n",
      "ClinicalIntervention - {'predisposes'} -> Publication\n",
      "ClinicalIntervention - {'related_to'} -> Publication\n",
      "ClinicalIntervention - {'prevents'} -> Publication\n",
      "ClinicalIntervention - {'manifestation_of'} -> Publication\n",
      "ClinicalIntervention - {'precedes'} -> Publication\n",
      "ClinicalIntervention - {'occurs_in'} -> Publication\n",
      "ClinicalIntervention - {'affects'} -> Publication\n",
      "ClinicalIntervention - {'close_match'} -> Publication\n",
      "ClinicalIntervention - {'exacerbates'} -> Publication\n",
      "ClinicalIntervention - {'coexists_with'} -> Publication\n",
      "ClinicalIntervention - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->BehavioralFeature->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'causes'} -> BehavioralFeature\n",
      "Pathway - {'disrupts'} -> BehavioralFeature\n",
      "Pathway - {'associated_with'} -> BehavioralFeature\n",
      "Pathway - {'gene_associated_with_condition'} -> BehavioralFeature\n",
      "Pathway - {'has_phenotype'} -> BehavioralFeature\n",
      "Pathway - {'predisposes'} -> BehavioralFeature\n",
      "Pathway - {'related_to'} -> BehavioralFeature\n",
      "Pathway - {'prevents'} -> BehavioralFeature\n",
      "Pathway - {'affects'} -> BehavioralFeature\n",
      "Pathway - {'precedes'} -> BehavioralFeature\n",
      "Pathway - {'coexists_with'} -> BehavioralFeature\n",
      "Pathway - {'manifestation_of'} -> BehavioralFeature\n",
      "Pathway - {'exacerbates'} -> BehavioralFeature\n",
      "Pathway - {'close_match'} -> BehavioralFeature\n",
      "Pathway - {'subclass_of'} -> BehavioralFeature\n",
      "Pathway - {'regulates'} -> BehavioralFeature\n",
      "Pathway - {'contraindicated_for'} -> BehavioralFeature\n",
      "Pathway - {'treats'} -> BehavioralFeature\n",
      "BehavioralFeature - {'subclass_of'} -> Publication\n",
      "BehavioralFeature - {'produces'} -> Publication\n",
      "BehavioralFeature - {'related_to'} -> Publication\n",
      "BehavioralFeature - {'occurs_in'} -> Publication\n",
      "BehavioralFeature - {'affects'} -> Publication\n",
      "BehavioralFeature - {'manifestation_of'} -> Publication\n",
      "BehavioralFeature - {'coexists_with'} -> Publication\n",
      "BehavioralFeature - {'disrupts'} -> Publication\n",
      "BehavioralFeature - {'located_in'} -> Publication\n",
      "BehavioralFeature - {'predisposes'} -> Publication\n",
      "BehavioralFeature - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->ProteinFamily->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'has_input'} -> ProteinFamily\n",
      "Pathway - {'associated_with'} -> ProteinFamily\n",
      "Pathway - {'biomarker_for'} -> ProteinFamily\n",
      "Pathway - {'has_part'} -> ProteinFamily\n",
      "Pathway - {'located_in'} -> ProteinFamily\n",
      "Pathway - {'interacts_with'} -> ProteinFamily\n",
      "Pathway - {'produces'} -> ProteinFamily\n",
      "Pathway - {'related_to'} -> ProteinFamily\n",
      "Pathway - {'occurs_in'} -> ProteinFamily\n",
      "Pathway - {'affects'} -> ProteinFamily\n",
      "Pathway - {'coexists_with'} -> ProteinFamily\n",
      "Pathway - {'close_match'} -> ProteinFamily\n",
      "Pathway - {'has_participant'} -> ProteinFamily\n",
      "Pathway - {'capable_of'} -> ProteinFamily\n",
      "Pathway - {'regulates'} -> ProteinFamily\n",
      "ProteinFamily - {'associated_with'} -> Publication\n",
      "ProteinFamily - {'subclass_of'} -> Publication\n",
      "ProteinFamily - {'gene_associated_with_condition'} -> Publication\n",
      "ProteinFamily - {'predisposes'} -> Publication\n",
      "ProteinFamily - {'related_to'} -> Publication\n",
      "ProteinFamily - {'affects'} -> Publication\n",
      "ProteinFamily - {'disrupts'} -> Publication\n",
      "ProteinFamily - {'located_in'} -> Publication\n",
      "ProteinFamily - {'regulates'} -> Publication\n",
      "ProteinFamily - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->PhysicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'has_participant'} -> PhysicalEntity\n",
      "Pathway - {'subclass_of'} -> PhysicalEntity\n",
      "Pathway - {'regulates'} -> PhysicalEntity\n",
      "Pathway - {'contraindicated_for'} -> PhysicalEntity\n",
      "Pathway - {'treats'} -> PhysicalEntity\n",
      "Pathway - {'has_input'} -> PhysicalEntity\n",
      "Pathway - {'in_taxon'} -> PhysicalEntity\n",
      "Pathway - {'causes'} -> PhysicalEntity\n",
      "Pathway - {'has_output'} -> PhysicalEntity\n",
      "Pathway - {'associated_with'} -> PhysicalEntity\n",
      "Pathway - {'gene_associated_with_condition'} -> PhysicalEntity\n",
      "Pathway - {'has_part'} -> PhysicalEntity\n",
      "Pathway - {'located_in'} -> PhysicalEntity\n",
      "Pathway - {'interacts_with'} -> PhysicalEntity\n",
      "Pathway - {'produces'} -> PhysicalEntity\n",
      "Pathway - {'predisposes'} -> PhysicalEntity\n",
      "Pathway - {'related_to'} -> PhysicalEntity\n",
      "Pathway - {'precedes'} -> PhysicalEntity\n",
      "Pathway - {'occurs_in'} -> PhysicalEntity\n",
      "Pathway - {'affects'} -> PhysicalEntity\n",
      "Pathway - {'coexists_with'} -> PhysicalEntity\n",
      "Pathway - {'manifestation_of'} -> PhysicalEntity\n",
      "Pathway - {'exacerbates'} -> PhysicalEntity\n",
      "PhysicalEntity - {'subclass_of'} -> Publication\n",
      "PhysicalEntity - {'has_part'} -> Publication\n",
      "PhysicalEntity - {'interacts_with'} -> Publication\n",
      "PhysicalEntity - {'produces'} -> Publication\n",
      "PhysicalEntity - {'predisposes'} -> Publication\n",
      "PhysicalEntity - {'related_to'} -> Publication\n",
      "PhysicalEntity - {'occurs_in'} -> Publication\n",
      "PhysicalEntity - {'affects'} -> Publication\n",
      "PhysicalEntity - {'close_match'} -> Publication\n",
      "PhysicalEntity - {'coexists_with'} -> Publication\n",
      "PhysicalEntity - {'regulates'} -> Publication\n",
      "PhysicalEntity - {'located_in'} -> Publication\n",
      "PhysicalEntity - {'treats'} -> Publication\n",
      "PhysicalEntity - {'has_input'} -> Publication\n",
      "PhysicalEntity - {'causes'} -> Publication\n",
      "PhysicalEntity - {'disrupts'} -> Publication\n",
      "PhysicalEntity - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Cohort->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'associated_with'} -> Cohort\n",
      "Pathway - {'located_in'} -> Cohort\n",
      "Pathway - {'colocalizes_with'} -> Cohort\n",
      "Pathway - {'interacts_with'} -> Cohort\n",
      "Pathway - {'produces'} -> Cohort\n",
      "Pathway - {'related_to'} -> Cohort\n",
      "Pathway - {'occurs_in'} -> Cohort\n",
      "Pathway - {'affects'} -> Cohort\n",
      "Pathway - {'coexists_with'} -> Cohort\n",
      "Pathway - {'has_participant'} -> Cohort\n",
      "Pathway - {'subclass_of'} -> Cohort\n",
      "Pathway - {'treats'} -> Cohort\n",
      "Cohort - {'diagnoses'} -> Publication\n",
      "Cohort - {'related_to'} -> Publication\n",
      "Cohort - {'treats'} -> Publication\n",
      "Cohort - {'has_input'} -> Publication\n",
      "Cohort - {'occurs_in'} -> Publication\n",
      "Cohort - {'has_part'} -> Publication\n",
      "Cohort - {'close_match'} -> Publication\n",
      "Cohort - {'subclass_of'} -> Publication\n",
      "Cohort - {'interacts_with'} -> Publication\n",
      "Cohort - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->PopulationOfIndividualOrganisms->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'treats'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'associated_with'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'located_in'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'related_to'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'affects'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'occurs_in'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'coexists_with'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'has_participant'} -> PopulationOfIndividualOrganisms\n",
      "Pathway - {'subclass_of'} -> PopulationOfIndividualOrganisms\n",
      "PopulationOfIndividualOrganisms - {'correlated_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_part'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'produces'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'related_to'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'occurs_in'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'affects'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'close_match'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'subclass_of'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'interacts_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'treats'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_input'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->MaterialSample->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'disrupts'} -> MaterialSample\n",
      "Pathway - {'associated_with'} -> MaterialSample\n",
      "Pathway - {'interacts_with'} -> MaterialSample\n",
      "Pathway - {'produces'} -> MaterialSample\n",
      "Pathway - {'has_part'} -> MaterialSample\n",
      "Pathway - {'located_in'} -> MaterialSample\n",
      "Pathway - {'predisposes'} -> MaterialSample\n",
      "Pathway - {'precedes'} -> MaterialSample\n",
      "Pathway - {'affects'} -> MaterialSample\n",
      "Pathway - {'occurs_in'} -> MaterialSample\n",
      "Pathway - {'coexists_with'} -> MaterialSample\n",
      "Pathway - {'subclass_of'} -> MaterialSample\n",
      "Pathway - {'regulates'} -> MaterialSample\n",
      "Pathway - {'causes'} -> MaterialSample\n",
      "MaterialSample - {'located_in'} -> Publication\n",
      "MaterialSample - {'associated_with'} -> Publication\n",
      "MaterialSample - {'subclass_of'} -> Publication\n",
      "MaterialSample - {'has_part'} -> Publication\n",
      "MaterialSample - {'related_to'} -> Publication\n",
      "MaterialSample - {'occurs_in'} -> Publication\n",
      "MaterialSample - {'affects'} -> Publication\n",
      "MaterialSample - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->ProteinDomain->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'occurs_in'} -> ProteinDomain\n",
      "Pathway - {'affects'} -> ProteinDomain\n",
      "Pathway - {'coexists_with'} -> ProteinDomain\n",
      "Pathway - {'regulates'} -> ProteinDomain\n",
      "Pathway - {'associated_with'} -> ProteinDomain\n",
      "Pathway - {'has_part'} -> ProteinDomain\n",
      "Pathway - {'located_in'} -> ProteinDomain\n",
      "Pathway - {'interacts_with'} -> ProteinDomain\n",
      "Pathway - {'produces'} -> ProteinDomain\n",
      "ProteinDomain - {'associated_with'} -> Publication\n",
      "ProteinDomain - {'located_in'} -> Publication\n",
      "ProteinDomain - {'related_to'} -> Publication\n",
      "ProteinDomain - {'occurs_in'} -> Publication\n",
      "ProteinDomain - {'affects'} -> Publication\n",
      "ProteinDomain - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->GeographicLocation->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'associated_with'} -> GeographicLocation\n",
      "Pathway - {'interacts_with'} -> GeographicLocation\n",
      "Pathway - {'produces'} -> GeographicLocation\n",
      "Pathway - {'has_part'} -> GeographicLocation\n",
      "Pathway - {'affects'} -> GeographicLocation\n",
      "Pathway - {'has_participant'} -> GeographicLocation\n",
      "GeographicLocation - {'subclass_of'} -> Publication\n",
      "GeographicLocation - {'related_to'} -> Publication\n",
      "GeographicLocation - {'gene_associated_with_condition'} -> Publication\n",
      "GeographicLocation - {'located_in'} -> Publication\n",
      "GeographicLocation - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->Event->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'coexists_with'} -> Event\n",
      "Pathway - {'manifestation_of'} -> Event\n",
      "Event - {'occurs_in'} -> Publication\n",
      "Event - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Pathway->GeneticInheritance->Publication\n",
      "InformationResource - {'associated_with'} -> Pathway\n",
      "Pathway - {'coexists_with'} -> GeneticInheritance\n",
      "Pathway - {'regulates'} -> GeneticInheritance\n",
      "Pathway - {'causes'} -> GeneticInheritance\n",
      "Pathway - {'affects'} -> GeneticInheritance\n",
      "Pathway - {'close_match'} -> GeneticInheritance\n",
      "Pathway - {'precedes'} -> GeneticInheritance\n",
      "GeneticInheritance - {'occurs_in'} -> Publication\n",
      "GeneticInheritance - {'has_input'} -> Publication\n",
      "GeneticInheritance - {'located_in'} -> Publication\n",
      "GeneticInheritance - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->SmallMolecule->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'is_side_effect_of'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'treated_by'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'is_ameliorated_by'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_basis_in'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'chemically_similar_to'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'in_taxon'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> SmallMolecule\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> SmallMolecule\n",
      "SmallMolecule - {'correlated_with'} -> Publication\n",
      "SmallMolecule - {'disrupts'} -> Publication\n",
      "SmallMolecule - {'gene_associated_with_condition'} -> Publication\n",
      "SmallMolecule - {'located_in'} -> Publication\n",
      "SmallMolecule - {'regulates'} -> Publication\n",
      "SmallMolecule - {'has_part'} -> Publication\n",
      "SmallMolecule - {'physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'predisposes'} -> Publication\n",
      "SmallMolecule - {'related_to'} -> Publication\n",
      "SmallMolecule - {'causes'} -> Publication\n",
      "SmallMolecule - {'directly_physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'precedes'} -> Publication\n",
      "SmallMolecule - {'close_match'} -> Publication\n",
      "SmallMolecule - {'associated_with'} -> Publication\n",
      "SmallMolecule - {'has_metabolite'} -> Publication\n",
      "SmallMolecule - {'subclass_of'} -> Publication\n",
      "SmallMolecule - {'interacts_with'} -> Publication\n",
      "SmallMolecule - {'treats'} -> Publication\n",
      "SmallMolecule - {'has_input'} -> Publication\n",
      "SmallMolecule - {'occurs_in'} -> Publication\n",
      "SmallMolecule - {'affects'} -> Publication\n",
      "SmallMolecule - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Protein->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'treated_by'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'coexpressed_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'negatively_correlated_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_biomarker'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'positively_correlated_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'occurs_together_in_literature_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'capable_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'gene_product_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Protein\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> Protein\n",
      "Protein - {'correlated_with'} -> Publication\n",
      "Protein - {'gene_product_of'} -> Publication\n",
      "Protein - {'associated_with'} -> Publication\n",
      "Protein - {'gene_associated_with_condition'} -> Publication\n",
      "Protein - {'located_in'} -> Publication\n",
      "Protein - {'regulates'} -> Publication\n",
      "Protein - {'biomarker_for'} -> Publication\n",
      "Protein - {'expressed_in'} -> Publication\n",
      "Protein - {'has_part'} -> Publication\n",
      "Protein - {'colocalizes_with'} -> Publication\n",
      "Protein - {'produces'} -> Publication\n",
      "Protein - {'predisposes'} -> Publication\n",
      "Protein - {'related_to'} -> Publication\n",
      "Protein - {'prevents'} -> Publication\n",
      "Protein - {'occurs_in'} -> Publication\n",
      "Protein - {'precedes'} -> Publication\n",
      "Protein - {'affects'} -> Publication\n",
      "Protein - {'close_match'} -> Publication\n",
      "Protein - {'subclass_of'} -> Publication\n",
      "Protein - {'interacts_with'} -> Publication\n",
      "Protein - {'treats'} -> Publication\n",
      "Protein - {'has_input'} -> Publication\n",
      "Protein - {'causes'} -> Publication\n",
      "Protein - {'physically_interacts_with'} -> Publication\n",
      "Protein - {'exacerbates'} -> Publication\n",
      "Protein - {'coexists_with'} -> Publication\n",
      "Protein - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->ChemicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'treated_by'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_together_in_literature_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_resistance_to'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexpressed_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'negatively_correlated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'decreased_likelihood_associated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_sensitivity_to'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_increased_likelihood_of'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'resistance_associated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'is_ameliorated_by'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'sensitivity_associated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'positively_correlated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_biomarker'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'increased_likelihood_associated_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'in_taxon'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_basis_in'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'chemically_similar_to'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> ChemicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> ChemicalEntity\n",
      "ChemicalEntity - {'correlated_with'} -> Publication\n",
      "ChemicalEntity - {'disrupts'} -> Publication\n",
      "ChemicalEntity - {'associated_with'} -> Publication\n",
      "ChemicalEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "ChemicalEntity - {'regulates'} -> Publication\n",
      "ChemicalEntity - {'located_in'} -> Publication\n",
      "ChemicalEntity - {'predisposes'} -> Publication\n",
      "ChemicalEntity - {'related_to'} -> Publication\n",
      "ChemicalEntity - {'has_part'} -> Publication\n",
      "ChemicalEntity - {'prevents'} -> Publication\n",
      "ChemicalEntity - {'affects'} -> Publication\n",
      "ChemicalEntity - {'precedes'} -> Publication\n",
      "ChemicalEntity - {'occurs_in'} -> Publication\n",
      "ChemicalEntity - {'has_metabolite'} -> Publication\n",
      "ChemicalEntity - {'close_match'} -> Publication\n",
      "ChemicalEntity - {'subclass_of'} -> Publication\n",
      "ChemicalEntity - {'interacts_with'} -> Publication\n",
      "ChemicalEntity - {'treats'} -> Publication\n",
      "ChemicalEntity - {'physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'has_input'} -> Publication\n",
      "ChemicalEntity - {'causes'} -> Publication\n",
      "ChemicalEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->MolecularEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'treated_by'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_basis_in'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'chemically_similar_to'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> MolecularEntity\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> MolecularEntity\n",
      "MolecularEntity - {'causes'} -> Publication\n",
      "MolecularEntity - {'associated_with'} -> Publication\n",
      "MolecularEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "MolecularEntity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularEntity - {'regulates'} -> Publication\n",
      "MolecularEntity - {'disrupts'} -> Publication\n",
      "MolecularEntity - {'located_in'} -> Publication\n",
      "MolecularEntity - {'biomarker_for'} -> Publication\n",
      "MolecularEntity - {'interacts_with'} -> Publication\n",
      "MolecularEntity - {'predisposes'} -> Publication\n",
      "MolecularEntity - {'related_to'} -> Publication\n",
      "MolecularEntity - {'has_part'} -> Publication\n",
      "MolecularEntity - {'occurs_in'} -> Publication\n",
      "MolecularEntity - {'affects'} -> Publication\n",
      "MolecularEntity - {'precedes'} -> Publication\n",
      "MolecularEntity - {'coexists_with'} -> Publication\n",
      "MolecularEntity - {'close_match'} -> Publication\n",
      "MolecularEntity - {'subclass_of'} -> Publication\n",
      "MolecularEntity - {'treats'} -> Publication\n",
      "MolecularEntity - {'physically_interacts_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Disease->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'positively_correlated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'increased_likelihood_associated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'occurs_together_in_literature_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_resistance_to'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'coexpressed_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'negatively_correlated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'treated_by'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_sensitivity_to'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'resistance_associated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'is_ameliorated_by'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'sensitivity_associated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'has_biomarker'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'decreased_likelihood_associated_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_increased_likelihood_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'gene_product_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Disease\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Disease\n",
      "Disease - {'correlated_with'} -> Publication\n",
      "Disease - {'associated_with'} -> Publication\n",
      "Disease - {'gene_associated_with_condition'} -> Publication\n",
      "Disease - {'subclass_of'} -> Publication\n",
      "Disease - {'has_part'} -> Publication\n",
      "Disease - {'interacts_with'} -> Publication\n",
      "Disease - {'produces'} -> Publication\n",
      "Disease - {'predisposes'} -> Publication\n",
      "Disease - {'related_to'} -> Publication\n",
      "Disease - {'prevents'} -> Publication\n",
      "Disease - {'diagnoses'} -> Publication\n",
      "Disease - {'contributes_to'} -> Publication\n",
      "Disease - {'manifestation_of'} -> Publication\n",
      "Disease - {'occurs_in'} -> Publication\n",
      "Disease - {'affects'} -> Publication\n",
      "Disease - {'has_input'} -> Publication\n",
      "Disease - {'exacerbates'} -> Publication\n",
      "Disease - {'coexists_with'} -> Publication\n",
      "Disease - {'close_match'} -> Publication\n",
      "Disease - {'disrupts'} -> Publication\n",
      "Disease - {'regulates'} -> Publication\n",
      "Disease - {'biomarker_for'} -> Publication\n",
      "Disease - {'located_in'} -> Publication\n",
      "Disease - {'treats'} -> Publication\n",
      "Disease - {'causes'} -> Publication\n",
      "Disease - {'disease_has_location'} -> Publication\n",
      "Disease - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->PhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'is_ameliorated_by'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_biomarker'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'decreased_likelihood_associated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_increased_likelihood_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'resistance_associated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'sensitivity_associated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'positively_correlated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'increased_likelihood_associated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'negatively_correlated_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'occurs_together_in_literature_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'treated_by'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_sensitivity_to'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_resistance_to'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'coexpressed_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'gene_product_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> PhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Publication\n",
      "PhenotypicFeature - {'occurs_in'} -> Publication\n",
      "PhenotypicFeature - {'affects'} -> Publication\n",
      "PhenotypicFeature - {'close_match'} -> Publication\n",
      "PhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "PhenotypicFeature - {'exacerbates'} -> Publication\n",
      "PhenotypicFeature - {'coexists_with'} -> Publication\n",
      "PhenotypicFeature - {'subclass_of'} -> Publication\n",
      "PhenotypicFeature - {'regulates'} -> Publication\n",
      "PhenotypicFeature - {'diagnoses'} -> Publication\n",
      "PhenotypicFeature - {'treats'} -> Publication\n",
      "PhenotypicFeature - {'has_input'} -> Publication\n",
      "PhenotypicFeature - {'causes'} -> Publication\n",
      "PhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "PhenotypicFeature - {'disrupts'} -> Publication\n",
      "PhenotypicFeature - {'associated_with'} -> Publication\n",
      "PhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "PhenotypicFeature - {'has_part'} -> Publication\n",
      "PhenotypicFeature - {'located_in'} -> Publication\n",
      "PhenotypicFeature - {'interacts_with'} -> Publication\n",
      "PhenotypicFeature - {'produces'} -> Publication\n",
      "PhenotypicFeature - {'prevents'} -> Publication\n",
      "PhenotypicFeature - {'contributes_to'} -> Publication\n",
      "PhenotypicFeature - {'predisposes'} -> Publication\n",
      "PhenotypicFeature - {'precedes'} -> Publication\n",
      "PhenotypicFeature - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->OrganismAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> OrganismAttribute\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> OrganismAttribute\n",
      "OrganismAttribute - {'related_to'} -> Publication\n",
      "OrganismAttribute - {'produces'} -> Publication\n",
      "OrganismAttribute - {'has_input'} -> Publication\n",
      "OrganismAttribute - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->BiologicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_basis_in'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'chemically_similar_to'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'in_taxon'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> BiologicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'gene_product_of'} -> BiologicalEntity\n",
      "BiologicalEntity - {'physically_interacts_with'} -> Publication\n",
      "BiologicalEntity - {'treats'} -> Publication\n",
      "BiologicalEntity - {'has_input'} -> Publication\n",
      "BiologicalEntity - {'causes'} -> Publication\n",
      "BiologicalEntity - {'disrupts'} -> Publication\n",
      "BiologicalEntity - {'is_sequence_variant_of'} -> Publication\n",
      "BiologicalEntity - {'has_phenotype'} -> Publication\n",
      "BiologicalEntity - {'biomarker_for'} -> Publication\n",
      "BiologicalEntity - {'associated_with'} -> Publication\n",
      "BiologicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalEntity - {'has_part'} -> Publication\n",
      "BiologicalEntity - {'located_in'} -> Publication\n",
      "BiologicalEntity - {'colocalizes_with'} -> Publication\n",
      "BiologicalEntity - {'interacts_with'} -> Publication\n",
      "BiologicalEntity - {'produces'} -> Publication\n",
      "BiologicalEntity - {'predisposes'} -> Publication\n",
      "BiologicalEntity - {'related_to'} -> Publication\n",
      "BiologicalEntity - {'prevents'} -> Publication\n",
      "BiologicalEntity - {'manifestation_of'} -> Publication\n",
      "BiologicalEntity - {'precedes'} -> Publication\n",
      "BiologicalEntity - {'occurs_in'} -> Publication\n",
      "BiologicalEntity - {'affects'} -> Publication\n",
      "BiologicalEntity - {'close_match'} -> Publication\n",
      "BiologicalEntity - {'exacerbates'} -> Publication\n",
      "BiologicalEntity - {'coexists_with'} -> Publication\n",
      "BiologicalEntity - {'has_participant'} -> Publication\n",
      "BiologicalEntity - {'subclass_of'} -> Publication\n",
      "BiologicalEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Annotation->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Annotation\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Annotation\n",
      "Annotation - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->NamedThing->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'in_taxon'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> NamedThing\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Publication\n",
      "NamedThing - {'related_to'} -> Publication\n",
      "NamedThing - {'is_sequence_variant_of'} -> Publication\n",
      "NamedThing - {'associated_with'} -> Publication\n",
      "NamedThing - {'has_phenotype'} -> Publication\n",
      "NamedThing - {'disrupts'} -> Publication\n",
      "NamedThing - {'biomarker_for'} -> Publication\n",
      "NamedThing - {'correlated_with'} -> Publication\n",
      "NamedThing - {'gene_associated_with_condition'} -> Publication\n",
      "NamedThing - {'has_part'} -> Publication\n",
      "NamedThing - {'located_in'} -> Publication\n",
      "NamedThing - {'colocalizes_with'} -> Publication\n",
      "NamedThing - {'interacts_with'} -> Publication\n",
      "NamedThing - {'produces'} -> Publication\n",
      "NamedThing - {'prevents'} -> Publication\n",
      "NamedThing - {'diagnoses'} -> Publication\n",
      "NamedThing - {'predisposes'} -> Publication\n",
      "NamedThing - {'precedes'} -> Publication\n",
      "NamedThing - {'occurs_in'} -> Publication\n",
      "NamedThing - {'affects'} -> Publication\n",
      "NamedThing - {'close_match'} -> Publication\n",
      "NamedThing - {'manifestation_of'} -> Publication\n",
      "NamedThing - {'exacerbates'} -> Publication\n",
      "NamedThing - {'coexists_with'} -> Publication\n",
      "NamedThing - {'regulates'} -> Publication\n",
      "NamedThing - {'has_input'} -> Publication\n",
      "NamedThing - {'physically_interacts_with'} -> Publication\n",
      "NamedThing - {'treats'} -> Publication\n",
      "NamedThing - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Entity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Entity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Entity\n",
      "Entity - {'subclass_of'} -> Publication\n",
      "Entity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->AnatomicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_basis_in'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'expressed_in'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'develops_from'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> AnatomicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> AnatomicalEntity\n",
      "AnatomicalEntity - {'develops_from'} -> Publication\n",
      "AnatomicalEntity - {'interacts_with'} -> Publication\n",
      "AnatomicalEntity - {'produces'} -> Publication\n",
      "AnatomicalEntity - {'has_part'} -> Publication\n",
      "AnatomicalEntity - {'diagnoses'} -> Publication\n",
      "AnatomicalEntity - {'related_to'} -> Publication\n",
      "AnatomicalEntity - {'affects'} -> Publication\n",
      "AnatomicalEntity - {'manifestation_of'} -> Publication\n",
      "AnatomicalEntity - {'has_input'} -> Publication\n",
      "AnatomicalEntity - {'occurs_in'} -> Publication\n",
      "AnatomicalEntity - {'coexists_with'} -> Publication\n",
      "AnatomicalEntity - {'close_match'} -> Publication\n",
      "AnatomicalEntity - {'disrupts'} -> Publication\n",
      "AnatomicalEntity - {'located_in'} -> Publication\n",
      "AnatomicalEntity - {'regulates'} -> Publication\n",
      "AnatomicalEntity - {'treats'} -> Publication\n",
      "AnatomicalEntity - {'causes'} -> Publication\n",
      "AnatomicalEntity - {'associated_with'} -> Publication\n",
      "AnatomicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "AnatomicalEntity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Gene->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_biomarker'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'positively_correlated_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'occurs_together_in_literature_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'coexpressed_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'negatively_correlated_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'capable_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'gene_product_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Gene\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Gene\n",
      "Gene - {'correlated_with'} -> Publication\n",
      "Gene - {'prevents'} -> Publication\n",
      "Gene - {'treats'} -> Publication\n",
      "Gene - {'has_input'} -> Publication\n",
      "Gene - {'related_to'} -> Publication\n",
      "Gene - {'occurs_in'} -> Publication\n",
      "Gene - {'affects'} -> Publication\n",
      "Gene - {'exacerbates'} -> Publication\n",
      "Gene - {'coexists_with'} -> Publication\n",
      "Gene - {'disrupts'} -> Publication\n",
      "Gene - {'gene_associated_with_condition'} -> Publication\n",
      "Gene - {'located_in'} -> Publication\n",
      "Gene - {'regulates'} -> Publication\n",
      "Gene - {'biomarker_for'} -> Publication\n",
      "Gene - {'expressed_in'} -> Publication\n",
      "Gene - {'physically_interacts_with'} -> Publication\n",
      "Gene - {'predisposes'} -> Publication\n",
      "Gene - {'causes'} -> Publication\n",
      "Gene - {'precedes'} -> Publication\n",
      "Gene - {'gene_product_of'} -> Publication\n",
      "Gene - {'close_match'} -> Publication\n",
      "Gene - {'associated_with'} -> Publication\n",
      "Gene - {'subclass_of'} -> Publication\n",
      "Gene - {'has_part'} -> Publication\n",
      "Gene - {'colocalizes_with'} -> Publication\n",
      "Gene - {'interacts_with'} -> Publication\n",
      "Gene - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->GenomicEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> GenomicEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> GenomicEntity\n",
      "GenomicEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->NucleicAcidEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> NucleicAcidEntity\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> NucleicAcidEntity\n",
      "NucleicAcidEntity - {'gene_associated_with_condition'} -> Publication\n",
      "NucleicAcidEntity - {'located_in'} -> Publication\n",
      "NucleicAcidEntity - {'related_to'} -> Publication\n",
      "NucleicAcidEntity - {'close_match'} -> Publication\n",
      "NucleicAcidEntity - {'subclass_of'} -> Publication\n",
      "NucleicAcidEntity - {'interacts_with'} -> Publication\n",
      "NucleicAcidEntity - {'occurs_in'} -> Publication\n",
      "NucleicAcidEntity - {'affects'} -> Publication\n",
      "NucleicAcidEntity - {'treats'} -> Publication\n",
      "NucleicAcidEntity - {'causes'} -> Publication\n",
      "NucleicAcidEntity - {'coexists_with'} -> Publication\n",
      "NucleicAcidEntity - {'disrupts'} -> Publication\n",
      "NucleicAcidEntity - {'associated_with'} -> Publication\n",
      "NucleicAcidEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->OrganismalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> OrganismalEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> OrganismalEntity\n",
      "OrganismalEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Polypeptide->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Polypeptide\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> Polypeptide\n",
      "Polypeptide - {'disrupts'} -> Publication\n",
      "Polypeptide - {'associated_with'} -> Publication\n",
      "Polypeptide - {'gene_associated_with_condition'} -> Publication\n",
      "Polypeptide - {'located_in'} -> Publication\n",
      "Polypeptide - {'regulates'} -> Publication\n",
      "Polypeptide - {'biomarker_for'} -> Publication\n",
      "Polypeptide - {'produces'} -> Publication\n",
      "Polypeptide - {'has_part'} -> Publication\n",
      "Polypeptide - {'physically_interacts_with'} -> Publication\n",
      "Polypeptide - {'colocalizes_with'} -> Publication\n",
      "Polypeptide - {'predisposes'} -> Publication\n",
      "Polypeptide - {'related_to'} -> Publication\n",
      "Polypeptide - {'prevents'} -> Publication\n",
      "Polypeptide - {'precedes'} -> Publication\n",
      "Polypeptide - {'close_match'} -> Publication\n",
      "Polypeptide - {'subclass_of'} -> Publication\n",
      "Polypeptide - {'interacts_with'} -> Publication\n",
      "Polypeptide - {'affects'} -> Publication\n",
      "Polypeptide - {'treats'} -> Publication\n",
      "Polypeptide - {'occurs_in'} -> Publication\n",
      "Polypeptide - {'exacerbates'} -> Publication\n",
      "Polypeptide - {'causes'} -> Publication\n",
      "Polypeptide - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->BiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'enables'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> BiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> BiologicalProcess\n",
      "BiologicalProcess - {'correlated_with'} -> Publication\n",
      "BiologicalProcess - {'disrupts'} -> Publication\n",
      "BiologicalProcess - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalProcess - {'regulates'} -> Publication\n",
      "BiologicalProcess - {'biomarker_for'} -> Publication\n",
      "BiologicalProcess - {'has_part'} -> Publication\n",
      "BiologicalProcess - {'located_in'} -> Publication\n",
      "BiologicalProcess - {'predisposes'} -> Publication\n",
      "BiologicalProcess - {'precedes'} -> Publication\n",
      "BiologicalProcess - {'prevents'} -> Publication\n",
      "BiologicalProcess - {'causes'} -> Publication\n",
      "BiologicalProcess - {'close_match'} -> Publication\n",
      "BiologicalProcess - {'associated_with'} -> Publication\n",
      "BiologicalProcess - {'produces'} -> Publication\n",
      "BiologicalProcess - {'has_participant'} -> Publication\n",
      "BiologicalProcess - {'subclass_of'} -> Publication\n",
      "BiologicalProcess - {'related_to'} -> Publication\n",
      "BiologicalProcess - {'affects'} -> Publication\n",
      "BiologicalProcess - {'treats'} -> Publication\n",
      "BiologicalProcess - {'manifestation_of'} -> Publication\n",
      "BiologicalProcess - {'has_input'} -> Publication\n",
      "BiologicalProcess - {'has_output'} -> Publication\n",
      "BiologicalProcess - {'occurs_in'} -> Publication\n",
      "BiologicalProcess - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->BiologicalProcessOrActivity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'enables'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> BiologicalProcessOrActivity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> BiologicalProcessOrActivity\n",
      "BiologicalProcessOrActivity - {'occurs_in'} -> Publication\n",
      "BiologicalProcessOrActivity - {'affects'} -> Publication\n",
      "BiologicalProcessOrActivity - {'has_participant'} -> Publication\n",
      "BiologicalProcessOrActivity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->MolecularActivity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'enables'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> MolecularActivity\n",
      "DiseaseOrPhenotypicFeature - {'catalyzes'} -> MolecularActivity\n",
      "MolecularActivity - {'occurs_in'} -> Publication\n",
      "MolecularActivity - {'treats'} -> Publication\n",
      "MolecularActivity - {'has_input'} -> Publication\n",
      "MolecularActivity - {'has_output'} -> Publication\n",
      "MolecularActivity - {'coexists_with'} -> Publication\n",
      "MolecularActivity - {'regulates'} -> Publication\n",
      "MolecularActivity - {'disrupts'} -> Publication\n",
      "MolecularActivity - {'associated_with'} -> Publication\n",
      "MolecularActivity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularActivity - {'located_in'} -> Publication\n",
      "MolecularActivity - {'produces'} -> Publication\n",
      "MolecularActivity - {'related_to'} -> Publication\n",
      "MolecularActivity - {'affects'} -> Publication\n",
      "MolecularActivity - {'has_participant'} -> Publication\n",
      "MolecularActivity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Pathway->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Pathway\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Publication\n",
      "Pathway - {'regulates'} -> Publication\n",
      "Pathway - {'treats'} -> Publication\n",
      "Pathway - {'has_input'} -> Publication\n",
      "Pathway - {'causes'} -> Publication\n",
      "Pathway - {'associated_with'} -> Publication\n",
      "Pathway - {'gene_associated_with_condition'} -> Publication\n",
      "Pathway - {'disrupts'} -> Publication\n",
      "Pathway - {'located_in'} -> Publication\n",
      "Pathway - {'produces'} -> Publication\n",
      "Pathway - {'predisposes'} -> Publication\n",
      "Pathway - {'related_to'} -> Publication\n",
      "Pathway - {'precedes'} -> Publication\n",
      "Pathway - {'occurs_in'} -> Publication\n",
      "Pathway - {'affects'} -> Publication\n",
      "Pathway - {'manifestation_of'} -> Publication\n",
      "Pathway - {'coexists_with'} -> Publication\n",
      "Pathway - {'close_match'} -> Publication\n",
      "Pathway - {'has_participant'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->CellularComponent->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'positively_correlated_with'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'expressed_in'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> CellularComponent\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> CellularComponent\n",
      "CellularComponent - {'subclass_of'} -> Publication\n",
      "CellularComponent - {'produces'} -> Publication\n",
      "CellularComponent - {'has_part'} -> Publication\n",
      "CellularComponent - {'related_to'} -> Publication\n",
      "CellularComponent - {'occurs_in'} -> Publication\n",
      "CellularComponent - {'affects'} -> Publication\n",
      "CellularComponent - {'regulates'} -> Publication\n",
      "CellularComponent - {'disrupts'} -> Publication\n",
      "CellularComponent - {'located_in'} -> Publication\n",
      "CellularComponent - {'associated_with'} -> Publication\n",
      "CellularComponent - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->CellLine->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'expressed_in'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> CellLine\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> CellLine\n",
      "CellLine - {'gene_associated_with_condition'} -> Publication\n",
      "CellLine - {'located_in'} -> Publication\n",
      "CellLine - {'produces'} -> Publication\n",
      "CellLine - {'related_to'} -> Publication\n",
      "CellLine - {'affects'} -> Publication\n",
      "CellLine - {'close_match'} -> Publication\n",
      "CellLine - {'subclass_of'} -> Publication\n",
      "CellLine - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Transcript->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Transcript\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Transcript\n",
      "Transcript - {'translates_to'} -> Publication\n",
      "Transcript - {'regulates'} -> Publication\n",
      "Transcript - {'transcribed_from'} -> Publication\n",
      "Transcript - {'associated_with'} -> Publication\n",
      "Transcript - {'located_in'} -> Publication\n",
      "Transcript - {'subclass_of'} -> Publication\n",
      "Transcript - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->ClinicalAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> ClinicalAttribute\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> ClinicalAttribute\n",
      "ClinicalAttribute - {'correlated_with'} -> Publication\n",
      "ClinicalAttribute - {'has_input'} -> Publication\n",
      "ClinicalAttribute - {'subclass_of'} -> Publication\n",
      "ClinicalAttribute - {'related_to'} -> Publication\n",
      "ClinicalAttribute - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->PhysiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'gene_product_of'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'enables'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> PhysiologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> PhysiologicalProcess\n",
      "PhysiologicalProcess - {'correlated_with'} -> Publication\n",
      "PhysiologicalProcess - {'has_input'} -> Publication\n",
      "PhysiologicalProcess - {'causes'} -> Publication\n",
      "PhysiologicalProcess - {'has_output'} -> Publication\n",
      "PhysiologicalProcess - {'disrupts'} -> Publication\n",
      "PhysiologicalProcess - {'associated_with'} -> Publication\n",
      "PhysiologicalProcess - {'located_in'} -> Publication\n",
      "PhysiologicalProcess - {'produces'} -> Publication\n",
      "PhysiologicalProcess - {'predisposes'} -> Publication\n",
      "PhysiologicalProcess - {'precedes'} -> Publication\n",
      "PhysiologicalProcess - {'related_to'} -> Publication\n",
      "PhysiologicalProcess - {'occurs_in'} -> Publication\n",
      "PhysiologicalProcess - {'affects'} -> Publication\n",
      "PhysiologicalProcess - {'coexists_with'} -> Publication\n",
      "PhysiologicalProcess - {'close_match'} -> Publication\n",
      "PhysiologicalProcess - {'has_participant'} -> Publication\n",
      "PhysiologicalProcess - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->ComplexMolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> ComplexMolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> ComplexMolecularMixture\n",
      "ComplexMolecularMixture - {'correlated_with'} -> Publication\n",
      "ComplexMolecularMixture - {'subclass_of'} -> Publication\n",
      "ComplexMolecularMixture - {'treats'} -> Publication\n",
      "ComplexMolecularMixture - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->NoncodingRNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> NoncodingRNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> NoncodingRNAProduct\n",
      "NoncodingRNAProduct - {'affects'} -> Publication\n",
      "NoncodingRNAProduct - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->RNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> RNAProduct\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> RNAProduct\n",
      "RNAProduct - {'located_in'} -> Publication\n",
      "RNAProduct - {'regulates'} -> Publication\n",
      "RNAProduct - {'related_to'} -> Publication\n",
      "RNAProduct - {'affects'} -> Publication\n",
      "RNAProduct - {'subclass_of'} -> Publication\n",
      "RNAProduct - {'disrupts'} -> Publication\n",
      "RNAProduct - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->EnvironmentalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> EnvironmentalProcess\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> EnvironmentalProcess\n",
      "EnvironmentalProcess - {'subclass_of'} -> Publication\n",
      "EnvironmentalProcess - {'related_to'} -> Publication\n",
      "EnvironmentalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->InformationContentEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> InformationContentEntity\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Publication\n",
      "InformationContentEntity - {'subclass_of'} -> Publication\n",
      "InformationContentEntity - {'related_to'} -> Publication\n",
      "InformationContentEntity - {'disrupts'} -> Publication\n",
      "InformationContentEntity - {'associated_with'} -> Publication\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Publication\n",
      "InformationContentEntity - {'regulates'} -> Publication\n",
      "InformationContentEntity - {'biomarker_for'} -> Publication\n",
      "InformationContentEntity - {'produces'} -> Publication\n",
      "InformationContentEntity - {'has_part'} -> Publication\n",
      "InformationContentEntity - {'located_in'} -> Publication\n",
      "InformationContentEntity - {'predisposes'} -> Publication\n",
      "InformationContentEntity - {'prevents'} -> Publication\n",
      "InformationContentEntity - {'affects'} -> Publication\n",
      "InformationContentEntity - {'manifestation_of'} -> Publication\n",
      "InformationContentEntity - {'precedes'} -> Publication\n",
      "InformationContentEntity - {'close_match'} -> Publication\n",
      "InformationContentEntity - {'interacts_with'} -> Publication\n",
      "InformationContentEntity - {'diagnoses'} -> Publication\n",
      "InformationContentEntity - {'treats'} -> Publication\n",
      "InformationContentEntity - {'occurs_in'} -> Publication\n",
      "InformationContentEntity - {'has_input'} -> Publication\n",
      "InformationContentEntity - {'exacerbates'} -> Publication\n",
      "InformationContentEntity - {'causes'} -> Publication\n",
      "InformationContentEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Phenomenon->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Phenomenon\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Phenomenon\n",
      "Phenomenon - {'associated_with'} -> Publication\n",
      "Phenomenon - {'subclass_of'} -> Publication\n",
      "Phenomenon - {'interacts_with'} -> Publication\n",
      "Phenomenon - {'produces'} -> Publication\n",
      "Phenomenon - {'related_to'} -> Publication\n",
      "Phenomenon - {'has_part'} -> Publication\n",
      "Phenomenon - {'diagnoses'} -> Publication\n",
      "Phenomenon - {'occurs_in'} -> Publication\n",
      "Phenomenon - {'affects'} -> Publication\n",
      "Phenomenon - {'coexists_with'} -> Publication\n",
      "Phenomenon - {'close_match'} -> Publication\n",
      "Phenomenon - {'disrupts'} -> Publication\n",
      "Phenomenon - {'regulates'} -> Publication\n",
      "Phenomenon - {'located_in'} -> Publication\n",
      "Phenomenon - {'has_input'} -> Publication\n",
      "Phenomenon - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Procedure->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Procedure\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Procedure\n",
      "Procedure - {'correlated_with'} -> Publication\n",
      "Procedure - {'manifestation_of'} -> Publication\n",
      "Procedure - {'close_match'} -> Publication\n",
      "Procedure - {'subclass_of'} -> Publication\n",
      "Procedure - {'interacts_with'} -> Publication\n",
      "Procedure - {'diagnoses'} -> Publication\n",
      "Procedure - {'occurs_in'} -> Publication\n",
      "Procedure - {'treats'} -> Publication\n",
      "Procedure - {'has_input'} -> Publication\n",
      "Procedure - {'exacerbates'} -> Publication\n",
      "Procedure - {'causes'} -> Publication\n",
      "Procedure - {'coexists_with'} -> Publication\n",
      "Procedure - {'disrupts'} -> Publication\n",
      "Procedure - {'associated_with'} -> Publication\n",
      "Procedure - {'regulates'} -> Publication\n",
      "Procedure - {'produces'} -> Publication\n",
      "Procedure - {'located_in'} -> Publication\n",
      "Procedure - {'predisposes'} -> Publication\n",
      "Procedure - {'related_to'} -> Publication\n",
      "Procedure - {'has_part'} -> Publication\n",
      "Procedure - {'prevents'} -> Publication\n",
      "Procedure - {'precedes'} -> Publication\n",
      "Procedure - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->IndividualOrganism->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> IndividualOrganism\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> IndividualOrganism\n",
      "IndividualOrganism - {'close_match'} -> Publication\n",
      "IndividualOrganism - {'subclass_of'} -> Publication\n",
      "IndividualOrganism - {'occurs_in'} -> Publication\n",
      "IndividualOrganism - {'located_in'} -> Publication\n",
      "IndividualOrganism - {'related_to'} -> Publication\n",
      "IndividualOrganism - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Behavior->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Behavior\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Behavior\n",
      "Behavior - {'treats'} -> Publication\n",
      "Behavior - {'has_input'} -> Publication\n",
      "Behavior - {'subclass_of'} -> Publication\n",
      "Behavior - {'located_in'} -> Publication\n",
      "Behavior - {'interacts_with'} -> Publication\n",
      "Behavior - {'predisposes'} -> Publication\n",
      "Behavior - {'related_to'} -> Publication\n",
      "Behavior - {'occurs_in'} -> Publication\n",
      "Behavior - {'affects'} -> Publication\n",
      "Behavior - {'coexists_with'} -> Publication\n",
      "Behavior - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->GrossAnatomicalStructure->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'caused_by'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'expressed_in'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'develops_from'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_basis_in'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> GrossAnatomicalStructure\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> GrossAnatomicalStructure\n",
      "GrossAnatomicalStructure - {'affects'} -> Publication\n",
      "GrossAnatomicalStructure - {'coexists_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'close_match'} -> Publication\n",
      "GrossAnatomicalStructure - {'develops_from'} -> Publication\n",
      "GrossAnatomicalStructure - {'regulates'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_input'} -> Publication\n",
      "GrossAnatomicalStructure - {'associated_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'located_in'} -> Publication\n",
      "GrossAnatomicalStructure - {'subclass_of'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_part'} -> Publication\n",
      "GrossAnatomicalStructure - {'produces'} -> Publication\n",
      "GrossAnatomicalStructure - {'related_to'} -> Publication\n",
      "GrossAnatomicalStructure - {'occurs_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->PathologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'increased_likelihood_associated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'occurs_together_in_literature_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_resistance_to'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'coexpressed_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'negatively_correlated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_sensitivity_to'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with_increased_likelihood_of'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'resistance_associated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'sensitivity_associated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'positively_correlated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_biomarker'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'decreased_likelihood_associated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> PathologicalProcess\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> PathologicalProcess\n",
      "PathologicalProcess - {'manifestation_of'} -> Publication\n",
      "PathologicalProcess - {'occurs_in'} -> Publication\n",
      "PathologicalProcess - {'coexists_with'} -> Publication\n",
      "PathologicalProcess - {'exacerbates'} -> Publication\n",
      "PathologicalProcess - {'disrupts'} -> Publication\n",
      "PathologicalProcess - {'located_in'} -> Publication\n",
      "PathologicalProcess - {'regulates'} -> Publication\n",
      "PathologicalProcess - {'treats'} -> Publication\n",
      "PathologicalProcess - {'precedes'} -> Publication\n",
      "PathologicalProcess - {'causes'} -> Publication\n",
      "PathologicalProcess - {'close_match'} -> Publication\n",
      "PathologicalProcess - {'associated_with'} -> Publication\n",
      "PathologicalProcess - {'subclass_of'} -> Publication\n",
      "PathologicalProcess - {'produces'} -> Publication\n",
      "PathologicalProcess - {'predisposes'} -> Publication\n",
      "PathologicalProcess - {'related_to'} -> Publication\n",
      "PathologicalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->OrganismTaxon->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'in_taxon'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> OrganismTaxon\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> OrganismTaxon\n",
      "OrganismTaxon - {'regulates'} -> Publication\n",
      "OrganismTaxon - {'gene_associated_with_condition'} -> Publication\n",
      "OrganismTaxon - {'located_in'} -> Publication\n",
      "OrganismTaxon - {'interacts_with'} -> Publication\n",
      "OrganismTaxon - {'produces'} -> Publication\n",
      "OrganismTaxon - {'related_to'} -> Publication\n",
      "OrganismTaxon - {'has_part'} -> Publication\n",
      "OrganismTaxon - {'predisposes'} -> Publication\n",
      "OrganismTaxon - {'occurs_in'} -> Publication\n",
      "OrganismTaxon - {'affects'} -> Publication\n",
      "OrganismTaxon - {'manifestation_of'} -> Publication\n",
      "OrganismTaxon - {'exacerbates'} -> Publication\n",
      "OrganismTaxon - {'coexists_with'} -> Publication\n",
      "OrganismTaxon - {'close_match'} -> Publication\n",
      "OrganismTaxon - {'subclass_of'} -> Publication\n",
      "OrganismTaxon - {'physically_interacts_with'} -> Publication\n",
      "OrganismTaxon - {'treats'} -> Publication\n",
      "OrganismTaxon - {'has_input'} -> Publication\n",
      "OrganismTaxon - {'causes'} -> Publication\n",
      "OrganismTaxon - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->MolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'treated_by'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'is_ameliorated_by'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> MolecularMixture\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> MolecularMixture\n",
      "MolecularMixture - {'correlated_with'} -> Publication\n",
      "MolecularMixture - {'treats'} -> Publication\n",
      "MolecularMixture - {'subclass_of'} -> Publication\n",
      "MolecularMixture - {'located_in'} -> Publication\n",
      "MolecularMixture - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->GeneFamily->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'is_sequence_variant_of'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> GeneFamily\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> GeneFamily\n",
      "GeneFamily - {'treats'} -> Publication\n",
      "GeneFamily - {'causes'} -> Publication\n",
      "GeneFamily - {'associated_with'} -> Publication\n",
      "GeneFamily - {'disrupts'} -> Publication\n",
      "GeneFamily - {'gene_associated_with_condition'} -> Publication\n",
      "GeneFamily - {'located_in'} -> Publication\n",
      "GeneFamily - {'regulates'} -> Publication\n",
      "GeneFamily - {'predisposes'} -> Publication\n",
      "GeneFamily - {'occurs_in'} -> Publication\n",
      "GeneFamily - {'affects'} -> Publication\n",
      "GeneFamily - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Cell->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_active_ingredient'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'part_of'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'expressed_in'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Cell\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Cell\n",
      "Cell - {'subclass_of'} -> Publication\n",
      "Cell - {'has_part'} -> Publication\n",
      "Cell - {'produces'} -> Publication\n",
      "Cell - {'related_to'} -> Publication\n",
      "Cell - {'occurs_in'} -> Publication\n",
      "Cell - {'affects'} -> Publication\n",
      "Cell - {'coexists_with'} -> Publication\n",
      "Cell - {'close_match'} -> Publication\n",
      "Cell - {'located_in'} -> Publication\n",
      "Cell - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->ChemicalMixture->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> ChemicalMixture\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> ChemicalMixture\n",
      "ChemicalMixture - {'related_to'} -> Publication\n",
      "ChemicalMixture - {'affects'} -> Publication\n",
      "ChemicalMixture - {'subclass_of'} -> Publication\n",
      "ChemicalMixture - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Activity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'actively_involved_in'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Activity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Activity\n",
      "Activity - {'correlated_with'} -> Publication\n",
      "Activity - {'causes'} -> Publication\n",
      "Activity - {'disrupts'} -> Publication\n",
      "Activity - {'subclass_of'} -> Publication\n",
      "Activity - {'located_in'} -> Publication\n",
      "Activity - {'interacts_with'} -> Publication\n",
      "Activity - {'produces'} -> Publication\n",
      "Activity - {'has_part'} -> Publication\n",
      "Activity - {'diagnoses'} -> Publication\n",
      "Activity - {'prevents'} -> Publication\n",
      "Activity - {'predisposes'} -> Publication\n",
      "Activity - {'precedes'} -> Publication\n",
      "Activity - {'related_to'} -> Publication\n",
      "Activity - {'occurs_in'} -> Publication\n",
      "Activity - {'affects'} -> Publication\n",
      "Activity - {'coexists_with'} -> Publication\n",
      "Activity - {'close_match'} -> Publication\n",
      "Activity - {'regulates'} -> Publication\n",
      "Activity - {'treats'} -> Publication\n",
      "Activity - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Drug->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affected_by'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_substrate'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'ameliorates'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_capability'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'participates_in'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'enabled_by'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_catalyst'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'actively_involves'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_upstream_actor'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'primarily_composed_of'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'temporally_related_to'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'similar_to'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'overlaps'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'composed_primarily_of'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'preceded_by'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'correlated_with'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'colocalizes_with'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'chemically_similar_to'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Drug\n",
      "DiseaseOrPhenotypicFeature - {'directly_physically_interacts_with'} -> Drug\n",
      "Drug - {'correlated_with'} -> Publication\n",
      "Drug - {'regulates'} -> Publication\n",
      "Drug - {'treats'} -> Publication\n",
      "Drug - {'has_input'} -> Publication\n",
      "Drug - {'physically_interacts_with'} -> Publication\n",
      "Drug - {'causes'} -> Publication\n",
      "Drug - {'directly_physically_interacts_with'} -> Publication\n",
      "Drug - {'disrupts'} -> Publication\n",
      "Drug - {'associated_with'} -> Publication\n",
      "Drug - {'gene_associated_with_condition'} -> Publication\n",
      "Drug - {'located_in'} -> Publication\n",
      "Drug - {'has_part'} -> Publication\n",
      "Drug - {'prevents'} -> Publication\n",
      "Drug - {'colocalizes_with'} -> Publication\n",
      "Drug - {'interacts_with'} -> Publication\n",
      "Drug - {'predisposes'} -> Publication\n",
      "Drug - {'related_to'} -> Publication\n",
      "Drug - {'occurs_in'} -> Publication\n",
      "Drug - {'precedes'} -> Publication\n",
      "Drug - {'affects'} -> Publication\n",
      "Drug - {'close_match'} -> Publication\n",
      "Drug - {'subclass_of'} -> Publication\n",
      "Drug - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->PhysicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> PhysicalEntity\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> PhysicalEntity\n",
      "PhysicalEntity - {'subclass_of'} -> Publication\n",
      "PhysicalEntity - {'has_part'} -> Publication\n",
      "PhysicalEntity - {'interacts_with'} -> Publication\n",
      "PhysicalEntity - {'produces'} -> Publication\n",
      "PhysicalEntity - {'predisposes'} -> Publication\n",
      "PhysicalEntity - {'related_to'} -> Publication\n",
      "PhysicalEntity - {'occurs_in'} -> Publication\n",
      "PhysicalEntity - {'affects'} -> Publication\n",
      "PhysicalEntity - {'close_match'} -> Publication\n",
      "PhysicalEntity - {'coexists_with'} -> Publication\n",
      "PhysicalEntity - {'regulates'} -> Publication\n",
      "PhysicalEntity - {'located_in'} -> Publication\n",
      "PhysicalEntity - {'treats'} -> Publication\n",
      "PhysicalEntity - {'has_input'} -> Publication\n",
      "PhysicalEntity - {'causes'} -> Publication\n",
      "PhysicalEntity - {'disrupts'} -> Publication\n",
      "PhysicalEntity - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Agent->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Agent\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Agent\n",
      "Agent - {'has_part'} -> Publication\n",
      "Agent - {'interacts_with'} -> Publication\n",
      "Agent - {'produces'} -> Publication\n",
      "Agent - {'predisposes'} -> Publication\n",
      "Agent - {'related_to'} -> Publication\n",
      "Agent - {'prevents'} -> Publication\n",
      "Agent - {'precedes'} -> Publication\n",
      "Agent - {'occurs_in'} -> Publication\n",
      "Agent - {'affects'} -> Publication\n",
      "Agent - {'exacerbates'} -> Publication\n",
      "Agent - {'coexists_with'} -> Publication\n",
      "Agent - {'close_match'} -> Publication\n",
      "Agent - {'subclass_of'} -> Publication\n",
      "Agent - {'regulates'} -> Publication\n",
      "Agent - {'diagnoses'} -> Publication\n",
      "Agent - {'treats'} -> Publication\n",
      "Agent - {'has_input'} -> Publication\n",
      "Agent - {'causes'} -> Publication\n",
      "Agent - {'located_in'} -> Publication\n",
      "Agent - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Device->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Device\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Device\n",
      "Device - {'subclass_of'} -> Publication\n",
      "Device - {'diagnoses'} -> Publication\n",
      "Device - {'related_to'} -> Publication\n",
      "Device - {'occurs_in'} -> Publication\n",
      "Device - {'affects'} -> Publication\n",
      "Device - {'treats'} -> Publication\n",
      "Device - {'has_input'} -> Publication\n",
      "Device - {'coexists_with'} -> Publication\n",
      "Device - {'regulates'} -> Publication\n",
      "Device - {'disrupts'} -> Publication\n",
      "Device - {'biomarker_for'} -> Publication\n",
      "Device - {'gene_associated_with_condition'} -> Publication\n",
      "Device - {'has_part'} -> Publication\n",
      "Device - {'located_in'} -> Publication\n",
      "Device - {'causes'} -> Publication\n",
      "Device - {'precedes'} -> Publication\n",
      "Device - {'close_match'} -> Publication\n",
      "Device - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Vitamin->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Vitamin\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Vitamin\n",
      "Vitamin - {'regulates'} -> Publication\n",
      "Vitamin - {'treats'} -> Publication\n",
      "Vitamin - {'disrupts'} -> Publication\n",
      "Vitamin - {'subclass_of'} -> Publication\n",
      "Vitamin - {'related_to'} -> Publication\n",
      "Vitamin - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->PopulationOfIndividualOrganisms->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> PopulationOfIndividualOrganisms\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> PopulationOfIndividualOrganisms\n",
      "PopulationOfIndividualOrganisms - {'correlated_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_part'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'produces'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'related_to'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'occurs_in'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'affects'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'close_match'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'subclass_of'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'interacts_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'treats'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_input'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->ClinicalIntervention->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> ClinicalIntervention\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> ClinicalIntervention\n",
      "ClinicalIntervention - {'correlated_with'} -> Publication\n",
      "ClinicalIntervention - {'regulates'} -> Publication\n",
      "ClinicalIntervention - {'diagnoses'} -> Publication\n",
      "ClinicalIntervention - {'treats'} -> Publication\n",
      "ClinicalIntervention - {'has_input'} -> Publication\n",
      "ClinicalIntervention - {'causes'} -> Publication\n",
      "ClinicalIntervention - {'disrupts'} -> Publication\n",
      "ClinicalIntervention - {'associated_with'} -> Publication\n",
      "ClinicalIntervention - {'has_part'} -> Publication\n",
      "ClinicalIntervention - {'located_in'} -> Publication\n",
      "ClinicalIntervention - {'predisposes'} -> Publication\n",
      "ClinicalIntervention - {'related_to'} -> Publication\n",
      "ClinicalIntervention - {'prevents'} -> Publication\n",
      "ClinicalIntervention - {'manifestation_of'} -> Publication\n",
      "ClinicalIntervention - {'precedes'} -> Publication\n",
      "ClinicalIntervention - {'occurs_in'} -> Publication\n",
      "ClinicalIntervention - {'affects'} -> Publication\n",
      "ClinicalIntervention - {'close_match'} -> Publication\n",
      "ClinicalIntervention - {'exacerbates'} -> Publication\n",
      "ClinicalIntervention - {'coexists_with'} -> Publication\n",
      "ClinicalIntervention - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->LifeStage->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> LifeStage\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> LifeStage\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> LifeStage\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> LifeStage\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> LifeStage\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> LifeStage\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> LifeStage\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> LifeStage\n",
      "LifeStage - {'subclass_of'} -> Publication\n",
      "LifeStage - {'has_part'} -> Publication\n",
      "LifeStage - {'interacts_with'} -> Publication\n",
      "LifeStage - {'produces'} -> Publication\n",
      "LifeStage - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->MaterialSample->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> MaterialSample\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> MaterialSample\n",
      "MaterialSample - {'located_in'} -> Publication\n",
      "MaterialSample - {'associated_with'} -> Publication\n",
      "MaterialSample - {'subclass_of'} -> Publication\n",
      "MaterialSample - {'has_part'} -> Publication\n",
      "MaterialSample - {'related_to'} -> Publication\n",
      "MaterialSample - {'occurs_in'} -> Publication\n",
      "MaterialSample - {'affects'} -> Publication\n",
      "MaterialSample - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->ProteinFamily->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> ProteinFamily\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> ProteinFamily\n",
      "ProteinFamily - {'associated_with'} -> Publication\n",
      "ProteinFamily - {'subclass_of'} -> Publication\n",
      "ProteinFamily - {'gene_associated_with_condition'} -> Publication\n",
      "ProteinFamily - {'predisposes'} -> Publication\n",
      "ProteinFamily - {'related_to'} -> Publication\n",
      "ProteinFamily - {'affects'} -> Publication\n",
      "ProteinFamily - {'disrupts'} -> Publication\n",
      "ProteinFamily - {'located_in'} -> Publication\n",
      "ProteinFamily - {'regulates'} -> Publication\n",
      "ProteinFamily - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->BehavioralFeature->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'contraindicated_for'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> BehavioralFeature\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> BehavioralFeature\n",
      "BehavioralFeature - {'subclass_of'} -> Publication\n",
      "BehavioralFeature - {'produces'} -> Publication\n",
      "BehavioralFeature - {'related_to'} -> Publication\n",
      "BehavioralFeature - {'occurs_in'} -> Publication\n",
      "BehavioralFeature - {'affects'} -> Publication\n",
      "BehavioralFeature - {'manifestation_of'} -> Publication\n",
      "BehavioralFeature - {'coexists_with'} -> Publication\n",
      "BehavioralFeature - {'disrupts'} -> Publication\n",
      "BehavioralFeature - {'located_in'} -> Publication\n",
      "BehavioralFeature - {'predisposes'} -> Publication\n",
      "BehavioralFeature - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->ProteinDomain->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> ProteinDomain\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> ProteinDomain\n",
      "ProteinDomain - {'associated_with'} -> Publication\n",
      "ProteinDomain - {'located_in'} -> Publication\n",
      "ProteinDomain - {'related_to'} -> Publication\n",
      "ProteinDomain - {'occurs_in'} -> Publication\n",
      "ProteinDomain - {'affects'} -> Publication\n",
      "ProteinDomain - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Cohort->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Cohort\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Cohort\n",
      "Cohort - {'diagnoses'} -> Publication\n",
      "Cohort - {'related_to'} -> Publication\n",
      "Cohort - {'treats'} -> Publication\n",
      "Cohort - {'has_input'} -> Publication\n",
      "Cohort - {'occurs_in'} -> Publication\n",
      "Cohort - {'has_part'} -> Publication\n",
      "Cohort - {'close_match'} -> Publication\n",
      "Cohort - {'subclass_of'} -> Publication\n",
      "Cohort - {'interacts_with'} -> Publication\n",
      "Cohort - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Food->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'has_phenotype'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'physically_interacts_with'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'has_output'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> Food\n",
      "DiseaseOrPhenotypicFeature - {'derives_from'} -> Food\n",
      "Food - {'regulates'} -> Publication\n",
      "Food - {'associated_with'} -> Publication\n",
      "Food - {'located_in'} -> Publication\n",
      "Food - {'predisposes'} -> Publication\n",
      "Food - {'related_to'} -> Publication\n",
      "Food - {'occurs_in'} -> Publication\n",
      "Food - {'affects'} -> Publication\n",
      "Food - {'precedes'} -> Publication\n",
      "Food - {'subclass_of'} -> Publication\n",
      "Food - {'treats'} -> Publication\n",
      "Food - {'causes'} -> Publication\n",
      "Food - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->MicroRNA->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> MicroRNA\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> MicroRNA\n",
      "MicroRNA - {'causes'} -> Publication\n",
      "MicroRNA - {'disrupts'} -> Publication\n",
      "MicroRNA - {'associated_with'} -> Publication\n",
      "MicroRNA - {'gene_associated_with_condition'} -> Publication\n",
      "MicroRNA - {'subclass_of'} -> Publication\n",
      "MicroRNA - {'predisposes'} -> Publication\n",
      "MicroRNA - {'located_in'} -> Publication\n",
      "MicroRNA - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Treatment->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Treatment\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Treatment\n",
      "Treatment - {'subclass_of'} -> Publication\n",
      "Treatment - {'occurs_in'} -> Publication\n",
      "Treatment - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->GeographicLocation->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> GeographicLocation\n",
      "DiseaseOrPhenotypicFeature - {'has_participant'} -> GeographicLocation\n",
      "GeographicLocation - {'subclass_of'} -> Publication\n",
      "GeographicLocation - {'related_to'} -> Publication\n",
      "GeographicLocation - {'gene_associated_with_condition'} -> Publication\n",
      "GeographicLocation - {'located_in'} -> Publication\n",
      "GeographicLocation - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->Event->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Event\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Event\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Event\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Event\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Event\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Event\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Event\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Event\n",
      "Event - {'occurs_in'} -> Publication\n",
      "Event - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->DiseaseOrPhenotypicFeature->GeneticInheritance->Publication\n",
      "InformationResource - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> GeneticInheritance\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> GeneticInheritance\n",
      "GeneticInheritance - {'occurs_in'} -> Publication\n",
      "GeneticInheritance - {'has_input'} -> Publication\n",
      "GeneticInheritance - {'located_in'} -> Publication\n",
      "GeneticInheritance - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->SmallMolecule->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> SmallMolecule\n",
      "InformationContentEntity - {'caused_by'} -> SmallMolecule\n",
      "InformationContentEntity - {'related_to'} -> SmallMolecule\n",
      "InformationContentEntity - {'has_upstream_actor'} -> SmallMolecule\n",
      "InformationContentEntity - {'primarily_composed_of'} -> SmallMolecule\n",
      "InformationContentEntity - {'affects'} -> SmallMolecule\n",
      "InformationContentEntity - {'similar_to'} -> SmallMolecule\n",
      "InformationContentEntity - {'overlaps'} -> SmallMolecule\n",
      "InformationContentEntity - {'composed_primarily_of'} -> SmallMolecule\n",
      "InformationContentEntity - {'affected_by'} -> SmallMolecule\n",
      "InformationContentEntity - {'interacts_with'} -> SmallMolecule\n",
      "InformationContentEntity - {'participates_in'} -> SmallMolecule\n",
      "InformationContentEntity - {'has_participant'} -> SmallMolecule\n",
      "InformationContentEntity - {'associated_with'} -> SmallMolecule\n",
      "InformationContentEntity - {'treats'} -> SmallMolecule\n",
      "InformationContentEntity - {'has_input'} -> SmallMolecule\n",
      "InformationContentEntity - {'derives_from'} -> SmallMolecule\n",
      "InformationContentEntity - {'in_taxon'} -> SmallMolecule\n",
      "InformationContentEntity - {'physically_interacts_with'} -> SmallMolecule\n",
      "InformationContentEntity - {'has_output'} -> SmallMolecule\n",
      "InformationContentEntity - {'occurs_in'} -> SmallMolecule\n",
      "InformationContentEntity - {'causes'} -> SmallMolecule\n",
      "InformationContentEntity - {'coexists_with'} -> SmallMolecule\n",
      "InformationContentEntity - {'actively_involved_in'} -> SmallMolecule\n",
      "InformationContentEntity - {'disrupts'} -> SmallMolecule\n",
      "InformationContentEntity - {'chemically_similar_to'} -> SmallMolecule\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> SmallMolecule\n",
      "InformationContentEntity - {'located_in'} -> SmallMolecule\n",
      "InformationContentEntity - {'regulates'} -> SmallMolecule\n",
      "InformationContentEntity - {'produces'} -> SmallMolecule\n",
      "InformationContentEntity - {'expressed_in'} -> SmallMolecule\n",
      "InformationContentEntity - {'has_part'} -> SmallMolecule\n",
      "InformationContentEntity - {'prevents'} -> SmallMolecule\n",
      "InformationContentEntity - {'colocalizes_with'} -> SmallMolecule\n",
      "InformationContentEntity - {'predisposes'} -> SmallMolecule\n",
      "InformationContentEntity - {'manifestation_of'} -> SmallMolecule\n",
      "InformationContentEntity - {'precedes'} -> SmallMolecule\n",
      "InformationContentEntity - {'close_match'} -> SmallMolecule\n",
      "InformationContentEntity - {'subclass_of'} -> SmallMolecule\n",
      "InformationContentEntity - {'diagnoses'} -> SmallMolecule\n",
      "SmallMolecule - {'correlated_with'} -> Publication\n",
      "SmallMolecule - {'disrupts'} -> Publication\n",
      "SmallMolecule - {'gene_associated_with_condition'} -> Publication\n",
      "SmallMolecule - {'located_in'} -> Publication\n",
      "SmallMolecule - {'regulates'} -> Publication\n",
      "SmallMolecule - {'has_part'} -> Publication\n",
      "SmallMolecule - {'physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'predisposes'} -> Publication\n",
      "SmallMolecule - {'related_to'} -> Publication\n",
      "SmallMolecule - {'causes'} -> Publication\n",
      "SmallMolecule - {'directly_physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'precedes'} -> Publication\n",
      "SmallMolecule - {'close_match'} -> Publication\n",
      "SmallMolecule - {'associated_with'} -> Publication\n",
      "SmallMolecule - {'has_metabolite'} -> Publication\n",
      "SmallMolecule - {'subclass_of'} -> Publication\n",
      "SmallMolecule - {'interacts_with'} -> Publication\n",
      "SmallMolecule - {'treats'} -> Publication\n",
      "SmallMolecule - {'has_input'} -> Publication\n",
      "SmallMolecule - {'occurs_in'} -> Publication\n",
      "SmallMolecule - {'affects'} -> Publication\n",
      "SmallMolecule - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Drug->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Drug\n",
      "InformationContentEntity - {'associated_with'} -> Drug\n",
      "InformationContentEntity - {'affects'} -> Drug\n",
      "InformationContentEntity - {'related_to'} -> Drug\n",
      "InformationContentEntity - {'diagnoses'} -> Drug\n",
      "InformationContentEntity - {'treats'} -> Drug\n",
      "InformationContentEntity - {'has_input'} -> Drug\n",
      "InformationContentEntity - {'derives_from'} -> Drug\n",
      "InformationContentEntity - {'in_taxon'} -> Drug\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Drug\n",
      "InformationContentEntity - {'has_output'} -> Drug\n",
      "InformationContentEntity - {'occurs_in'} -> Drug\n",
      "InformationContentEntity - {'exacerbates'} -> Drug\n",
      "InformationContentEntity - {'causes'} -> Drug\n",
      "InformationContentEntity - {'coexists_with'} -> Drug\n",
      "InformationContentEntity - {'disrupts'} -> Drug\n",
      "InformationContentEntity - {'chemically_similar_to'} -> Drug\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Drug\n",
      "InformationContentEntity - {'regulates'} -> Drug\n",
      "InformationContentEntity - {'produces'} -> Drug\n",
      "InformationContentEntity - {'has_part'} -> Drug\n",
      "InformationContentEntity - {'located_in'} -> Drug\n",
      "InformationContentEntity - {'colocalizes_with'} -> Drug\n",
      "InformationContentEntity - {'predisposes'} -> Drug\n",
      "InformationContentEntity - {'prevents'} -> Drug\n",
      "InformationContentEntity - {'manifestation_of'} -> Drug\n",
      "InformationContentEntity - {'precedes'} -> Drug\n",
      "InformationContentEntity - {'close_match'} -> Drug\n",
      "InformationContentEntity - {'has_participant'} -> Drug\n",
      "InformationContentEntity - {'subclass_of'} -> Drug\n",
      "InformationContentEntity - {'interacts_with'} -> Drug\n",
      "Drug - {'correlated_with'} -> Publication\n",
      "Drug - {'regulates'} -> Publication\n",
      "Drug - {'treats'} -> Publication\n",
      "Drug - {'has_input'} -> Publication\n",
      "Drug - {'physically_interacts_with'} -> Publication\n",
      "Drug - {'causes'} -> Publication\n",
      "Drug - {'directly_physically_interacts_with'} -> Publication\n",
      "Drug - {'disrupts'} -> Publication\n",
      "Drug - {'associated_with'} -> Publication\n",
      "Drug - {'gene_associated_with_condition'} -> Publication\n",
      "Drug - {'located_in'} -> Publication\n",
      "Drug - {'has_part'} -> Publication\n",
      "Drug - {'prevents'} -> Publication\n",
      "Drug - {'colocalizes_with'} -> Publication\n",
      "Drug - {'interacts_with'} -> Publication\n",
      "Drug - {'predisposes'} -> Publication\n",
      "Drug - {'related_to'} -> Publication\n",
      "Drug - {'occurs_in'} -> Publication\n",
      "Drug - {'precedes'} -> Publication\n",
      "Drug - {'affects'} -> Publication\n",
      "Drug - {'close_match'} -> Publication\n",
      "Drug - {'subclass_of'} -> Publication\n",
      "Drug - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ChemicalExposure->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> ChemicalExposure\n",
      "ChemicalExposure - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Disease->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Disease\n",
      "InformationContentEntity - {'affects'} -> Disease\n",
      "InformationContentEntity - {'interacts_with'} -> Disease\n",
      "InformationContentEntity - {'participates_in'} -> Disease\n",
      "InformationContentEntity - {'caused_by'} -> Disease\n",
      "InformationContentEntity - {'associated_with'} -> Disease\n",
      "InformationContentEntity - {'related_to'} -> Disease\n",
      "InformationContentEntity - {'overlaps'} -> Disease\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Disease\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Disease\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Disease\n",
      "InformationContentEntity - {'similar_to'} -> Disease\n",
      "InformationContentEntity - {'affected_by'} -> Disease\n",
      "InformationContentEntity - {'has_participant'} -> Disease\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Disease\n",
      "InformationContentEntity - {'has_phenotype'} -> Disease\n",
      "InformationContentEntity - {'regulates'} -> Disease\n",
      "InformationContentEntity - {'disrupts'} -> Disease\n",
      "InformationContentEntity - {'located_in'} -> Disease\n",
      "InformationContentEntity - {'biomarker_for'} -> Disease\n",
      "InformationContentEntity - {'contributes_to'} -> Disease\n",
      "InformationContentEntity - {'produces'} -> Disease\n",
      "InformationContentEntity - {'predisposes'} -> Disease\n",
      "InformationContentEntity - {'has_part'} -> Disease\n",
      "InformationContentEntity - {'prevents'} -> Disease\n",
      "InformationContentEntity - {'manifestation_of'} -> Disease\n",
      "InformationContentEntity - {'precedes'} -> Disease\n",
      "InformationContentEntity - {'close_match'} -> Disease\n",
      "InformationContentEntity - {'subclass_of'} -> Disease\n",
      "InformationContentEntity - {'contraindicated_for'} -> Disease\n",
      "InformationContentEntity - {'diagnoses'} -> Disease\n",
      "InformationContentEntity - {'treats'} -> Disease\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Disease\n",
      "InformationContentEntity - {'occurs_in'} -> Disease\n",
      "InformationContentEntity - {'has_input'} -> Disease\n",
      "InformationContentEntity - {'derives_from'} -> Disease\n",
      "InformationContentEntity - {'exacerbates'} -> Disease\n",
      "InformationContentEntity - {'causes'} -> Disease\n",
      "InformationContentEntity - {'coexists_with'} -> Disease\n",
      "InformationContentEntity - {'actively_involved_in'} -> Disease\n",
      "Disease - {'correlated_with'} -> Publication\n",
      "Disease - {'associated_with'} -> Publication\n",
      "Disease - {'gene_associated_with_condition'} -> Publication\n",
      "Disease - {'subclass_of'} -> Publication\n",
      "Disease - {'has_part'} -> Publication\n",
      "Disease - {'interacts_with'} -> Publication\n",
      "Disease - {'produces'} -> Publication\n",
      "Disease - {'predisposes'} -> Publication\n",
      "Disease - {'related_to'} -> Publication\n",
      "Disease - {'prevents'} -> Publication\n",
      "Disease - {'diagnoses'} -> Publication\n",
      "Disease - {'contributes_to'} -> Publication\n",
      "Disease - {'manifestation_of'} -> Publication\n",
      "Disease - {'occurs_in'} -> Publication\n",
      "Disease - {'affects'} -> Publication\n",
      "Disease - {'has_input'} -> Publication\n",
      "Disease - {'exacerbates'} -> Publication\n",
      "Disease - {'coexists_with'} -> Publication\n",
      "Disease - {'close_match'} -> Publication\n",
      "Disease - {'disrupts'} -> Publication\n",
      "Disease - {'regulates'} -> Publication\n",
      "Disease - {'biomarker_for'} -> Publication\n",
      "Disease - {'located_in'} -> Publication\n",
      "Disease - {'treats'} -> Publication\n",
      "Disease - {'causes'} -> Publication\n",
      "Disease - {'disease_has_location'} -> Publication\n",
      "Disease - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->PhenotypicFeature->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'associated_with'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'affects'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'similar_to'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'related_to'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'overlaps'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'composed_primarily_of'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'has_upstream_actor'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'primarily_composed_of'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'affected_by'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'participates_in'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'has_participant'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'interacts_with'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'has_part'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'located_in'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'contributes_to'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'predisposes'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'prevents'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'manifestation_of'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'precedes'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'close_match'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'subclass_of'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'contraindicated_for'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'diagnoses'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'treats'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'physically_interacts_with'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'occurs_in'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'has_input'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'derives_from'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'exacerbates'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'causes'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'coexists_with'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'actively_involved_in'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'disrupts'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'has_phenotype'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'regulates'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'biomarker_for'} -> PhenotypicFeature\n",
      "InformationContentEntity - {'produces'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Publication\n",
      "PhenotypicFeature - {'occurs_in'} -> Publication\n",
      "PhenotypicFeature - {'affects'} -> Publication\n",
      "PhenotypicFeature - {'close_match'} -> Publication\n",
      "PhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "PhenotypicFeature - {'exacerbates'} -> Publication\n",
      "PhenotypicFeature - {'coexists_with'} -> Publication\n",
      "PhenotypicFeature - {'subclass_of'} -> Publication\n",
      "PhenotypicFeature - {'regulates'} -> Publication\n",
      "PhenotypicFeature - {'diagnoses'} -> Publication\n",
      "PhenotypicFeature - {'treats'} -> Publication\n",
      "PhenotypicFeature - {'has_input'} -> Publication\n",
      "PhenotypicFeature - {'causes'} -> Publication\n",
      "PhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "PhenotypicFeature - {'disrupts'} -> Publication\n",
      "PhenotypicFeature - {'associated_with'} -> Publication\n",
      "PhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "PhenotypicFeature - {'has_part'} -> Publication\n",
      "PhenotypicFeature - {'located_in'} -> Publication\n",
      "PhenotypicFeature - {'interacts_with'} -> Publication\n",
      "PhenotypicFeature - {'produces'} -> Publication\n",
      "PhenotypicFeature - {'prevents'} -> Publication\n",
      "PhenotypicFeature - {'contributes_to'} -> Publication\n",
      "PhenotypicFeature - {'predisposes'} -> Publication\n",
      "PhenotypicFeature - {'precedes'} -> Publication\n",
      "PhenotypicFeature - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->EnvironmentalExposure->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> EnvironmentalExposure\n",
      "EnvironmentalExposure - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Activity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Activity\n",
      "InformationContentEntity - {'overlaps'} -> Activity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Activity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Activity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Activity\n",
      "InformationContentEntity - {'similar_to'} -> Activity\n",
      "InformationContentEntity - {'affected_by'} -> Activity\n",
      "InformationContentEntity - {'participates_in'} -> Activity\n",
      "InformationContentEntity - {'has_participant'} -> Activity\n",
      "InformationContentEntity - {'interacts_with'} -> Activity\n",
      "InformationContentEntity - {'associated_with'} -> Activity\n",
      "InformationContentEntity - {'affects'} -> Activity\n",
      "InformationContentEntity - {'related_to'} -> Activity\n",
      "InformationContentEntity - {'diagnoses'} -> Activity\n",
      "InformationContentEntity - {'treats'} -> Activity\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Activity\n",
      "InformationContentEntity - {'occurs_in'} -> Activity\n",
      "InformationContentEntity - {'has_input'} -> Activity\n",
      "InformationContentEntity - {'exacerbates'} -> Activity\n",
      "InformationContentEntity - {'causes'} -> Activity\n",
      "InformationContentEntity - {'coexists_with'} -> Activity\n",
      "InformationContentEntity - {'disrupts'} -> Activity\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Activity\n",
      "InformationContentEntity - {'regulates'} -> Activity\n",
      "InformationContentEntity - {'produces'} -> Activity\n",
      "InformationContentEntity - {'located_in'} -> Activity\n",
      "InformationContentEntity - {'biomarker_for'} -> Activity\n",
      "InformationContentEntity - {'predisposes'} -> Activity\n",
      "InformationContentEntity - {'has_part'} -> Activity\n",
      "InformationContentEntity - {'prevents'} -> Activity\n",
      "InformationContentEntity - {'precedes'} -> Activity\n",
      "InformationContentEntity - {'manifestation_of'} -> Activity\n",
      "InformationContentEntity - {'close_match'} -> Activity\n",
      "InformationContentEntity - {'subclass_of'} -> Activity\n",
      "Activity - {'correlated_with'} -> Publication\n",
      "Activity - {'causes'} -> Publication\n",
      "Activity - {'disrupts'} -> Publication\n",
      "Activity - {'subclass_of'} -> Publication\n",
      "Activity - {'located_in'} -> Publication\n",
      "Activity - {'interacts_with'} -> Publication\n",
      "Activity - {'produces'} -> Publication\n",
      "Activity - {'has_part'} -> Publication\n",
      "Activity - {'diagnoses'} -> Publication\n",
      "Activity - {'prevents'} -> Publication\n",
      "Activity - {'predisposes'} -> Publication\n",
      "Activity - {'precedes'} -> Publication\n",
      "Activity - {'related_to'} -> Publication\n",
      "Activity - {'occurs_in'} -> Publication\n",
      "Activity - {'affects'} -> Publication\n",
      "Activity - {'coexists_with'} -> Publication\n",
      "Activity - {'close_match'} -> Publication\n",
      "Activity - {'regulates'} -> Publication\n",
      "Activity - {'treats'} -> Publication\n",
      "Activity - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ClinicalIntervention->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'close_match'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'subclass_of'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'interacts_with'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'diagnoses'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'treats'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'physically_interacts_with'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'occurs_in'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'has_input'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'derives_from'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'exacerbates'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'causes'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'coexists_with'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'disrupts'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'associated_with'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'regulates'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'biomarker_for'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'produces'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'has_part'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'located_in'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'predisposes'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'related_to'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'prevents'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'affects'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'manifestation_of'} -> ClinicalIntervention\n",
      "InformationContentEntity - {'precedes'} -> ClinicalIntervention\n",
      "ClinicalIntervention - {'correlated_with'} -> Publication\n",
      "ClinicalIntervention - {'regulates'} -> Publication\n",
      "ClinicalIntervention - {'diagnoses'} -> Publication\n",
      "ClinicalIntervention - {'treats'} -> Publication\n",
      "ClinicalIntervention - {'has_input'} -> Publication\n",
      "ClinicalIntervention - {'causes'} -> Publication\n",
      "ClinicalIntervention - {'disrupts'} -> Publication\n",
      "ClinicalIntervention - {'associated_with'} -> Publication\n",
      "ClinicalIntervention - {'has_part'} -> Publication\n",
      "ClinicalIntervention - {'located_in'} -> Publication\n",
      "ClinicalIntervention - {'predisposes'} -> Publication\n",
      "ClinicalIntervention - {'related_to'} -> Publication\n",
      "ClinicalIntervention - {'prevents'} -> Publication\n",
      "ClinicalIntervention - {'manifestation_of'} -> Publication\n",
      "ClinicalIntervention - {'precedes'} -> Publication\n",
      "ClinicalIntervention - {'occurs_in'} -> Publication\n",
      "ClinicalIntervention - {'affects'} -> Publication\n",
      "ClinicalIntervention - {'close_match'} -> Publication\n",
      "ClinicalIntervention - {'exacerbates'} -> Publication\n",
      "ClinicalIntervention - {'coexists_with'} -> Publication\n",
      "ClinicalIntervention - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->PopulationOfIndividualOrganisms->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'located_in'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'produces'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'has_part'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'predisposes'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'related_to'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'affects'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'close_match'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'subclass_of'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'interacts_with'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'diagnoses'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'treats'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'has_input'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'occurs_in'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'coexists_with'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'disrupts'} -> PopulationOfIndividualOrganisms\n",
      "InformationContentEntity - {'associated_with'} -> PopulationOfIndividualOrganisms\n",
      "PopulationOfIndividualOrganisms - {'correlated_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_part'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'produces'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'related_to'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'occurs_in'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'affects'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'close_match'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'subclass_of'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'interacts_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'treats'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_input'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ChemicalEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> ChemicalEntity\n",
      "InformationContentEntity - {'related_to'} -> ChemicalEntity\n",
      "InformationContentEntity - {'caused_by'} -> ChemicalEntity\n",
      "InformationContentEntity - {'interacts_with'} -> ChemicalEntity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> ChemicalEntity\n",
      "InformationContentEntity - {'associated_with'} -> ChemicalEntity\n",
      "InformationContentEntity - {'affects'} -> ChemicalEntity\n",
      "InformationContentEntity - {'similar_to'} -> ChemicalEntity\n",
      "InformationContentEntity - {'overlaps'} -> ChemicalEntity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> ChemicalEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> ChemicalEntity\n",
      "InformationContentEntity - {'affected_by'} -> ChemicalEntity\n",
      "InformationContentEntity - {'participates_in'} -> ChemicalEntity\n",
      "InformationContentEntity - {'has_participant'} -> ChemicalEntity\n",
      "InformationContentEntity - {'close_match'} -> ChemicalEntity\n",
      "InformationContentEntity - {'subclass_of'} -> ChemicalEntity\n",
      "InformationContentEntity - {'diagnoses'} -> ChemicalEntity\n",
      "InformationContentEntity - {'treats'} -> ChemicalEntity\n",
      "InformationContentEntity - {'physically_interacts_with'} -> ChemicalEntity\n",
      "InformationContentEntity - {'occurs_in'} -> ChemicalEntity\n",
      "InformationContentEntity - {'has_input'} -> ChemicalEntity\n",
      "InformationContentEntity - {'derives_from'} -> ChemicalEntity\n",
      "InformationContentEntity - {'exacerbates'} -> ChemicalEntity\n",
      "InformationContentEntity - {'causes'} -> ChemicalEntity\n",
      "InformationContentEntity - {'has_output'} -> ChemicalEntity\n",
      "InformationContentEntity - {'coexists_with'} -> ChemicalEntity\n",
      "InformationContentEntity - {'actively_involved_in'} -> ChemicalEntity\n",
      "InformationContentEntity - {'disrupts'} -> ChemicalEntity\n",
      "InformationContentEntity - {'chemically_similar_to'} -> ChemicalEntity\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> ChemicalEntity\n",
      "InformationContentEntity - {'regulates'} -> ChemicalEntity\n",
      "InformationContentEntity - {'produces'} -> ChemicalEntity\n",
      "InformationContentEntity - {'expressed_in'} -> ChemicalEntity\n",
      "InformationContentEntity - {'located_in'} -> ChemicalEntity\n",
      "InformationContentEntity - {'predisposes'} -> ChemicalEntity\n",
      "InformationContentEntity - {'has_part'} -> ChemicalEntity\n",
      "InformationContentEntity - {'prevents'} -> ChemicalEntity\n",
      "InformationContentEntity - {'colocalizes_with'} -> ChemicalEntity\n",
      "InformationContentEntity - {'precedes'} -> ChemicalEntity\n",
      "InformationContentEntity - {'manifestation_of'} -> ChemicalEntity\n",
      "ChemicalEntity - {'correlated_with'} -> Publication\n",
      "ChemicalEntity - {'disrupts'} -> Publication\n",
      "ChemicalEntity - {'associated_with'} -> Publication\n",
      "ChemicalEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "ChemicalEntity - {'regulates'} -> Publication\n",
      "ChemicalEntity - {'located_in'} -> Publication\n",
      "ChemicalEntity - {'predisposes'} -> Publication\n",
      "ChemicalEntity - {'related_to'} -> Publication\n",
      "ChemicalEntity - {'has_part'} -> Publication\n",
      "ChemicalEntity - {'prevents'} -> Publication\n",
      "ChemicalEntity - {'affects'} -> Publication\n",
      "ChemicalEntity - {'precedes'} -> Publication\n",
      "ChemicalEntity - {'occurs_in'} -> Publication\n",
      "ChemicalEntity - {'has_metabolite'} -> Publication\n",
      "ChemicalEntity - {'close_match'} -> Publication\n",
      "ChemicalEntity - {'subclass_of'} -> Publication\n",
      "ChemicalEntity - {'interacts_with'} -> Publication\n",
      "ChemicalEntity - {'treats'} -> Publication\n",
      "ChemicalEntity - {'physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'has_input'} -> Publication\n",
      "ChemicalEntity - {'causes'} -> Publication\n",
      "ChemicalEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ComplexMolecularMixture->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'affected_by'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'interacts_with'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'participates_in'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'has_participant'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'associated_with'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'has_upstream_actor'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'primarily_composed_of'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'affects'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'similar_to'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'related_to'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'overlaps'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'composed_primarily_of'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'produces'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'subclass_of'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'has_part'} -> ComplexMolecularMixture\n",
      "InformationContentEntity - {'has_input'} -> ComplexMolecularMixture\n",
      "ComplexMolecularMixture - {'correlated_with'} -> Publication\n",
      "ComplexMolecularMixture - {'subclass_of'} -> Publication\n",
      "ComplexMolecularMixture - {'treats'} -> Publication\n",
      "ComplexMolecularMixture - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->BiologicalProcess->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> BiologicalProcess\n",
      "InformationContentEntity - {'caused_by'} -> BiologicalProcess\n",
      "InformationContentEntity - {'has_participant'} -> BiologicalProcess\n",
      "InformationContentEntity - {'associated_with'} -> BiologicalProcess\n",
      "InformationContentEntity - {'overlaps'} -> BiologicalProcess\n",
      "InformationContentEntity - {'has_upstream_actor'} -> BiologicalProcess\n",
      "InformationContentEntity - {'primarily_composed_of'} -> BiologicalProcess\n",
      "InformationContentEntity - {'affects'} -> BiologicalProcess\n",
      "InformationContentEntity - {'similar_to'} -> BiologicalProcess\n",
      "InformationContentEntity - {'related_to'} -> BiologicalProcess\n",
      "InformationContentEntity - {'composed_primarily_of'} -> BiologicalProcess\n",
      "InformationContentEntity - {'affected_by'} -> BiologicalProcess\n",
      "InformationContentEntity - {'interacts_with'} -> BiologicalProcess\n",
      "InformationContentEntity - {'participates_in'} -> BiologicalProcess\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> BiologicalProcess\n",
      "InformationContentEntity - {'close_match'} -> BiologicalProcess\n",
      "InformationContentEntity - {'subclass_of'} -> BiologicalProcess\n",
      "InformationContentEntity - {'enables'} -> BiologicalProcess\n",
      "InformationContentEntity - {'diagnoses'} -> BiologicalProcess\n",
      "InformationContentEntity - {'treats'} -> BiologicalProcess\n",
      "InformationContentEntity - {'physically_interacts_with'} -> BiologicalProcess\n",
      "InformationContentEntity - {'occurs_in'} -> BiologicalProcess\n",
      "InformationContentEntity - {'has_input'} -> BiologicalProcess\n",
      "InformationContentEntity - {'derives_from'} -> BiologicalProcess\n",
      "InformationContentEntity - {'exacerbates'} -> BiologicalProcess\n",
      "InformationContentEntity - {'causes'} -> BiologicalProcess\n",
      "InformationContentEntity - {'coexists_with'} -> BiologicalProcess\n",
      "InformationContentEntity - {'disrupts'} -> BiologicalProcess\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> BiologicalProcess\n",
      "InformationContentEntity - {'capable_of'} -> BiologicalProcess\n",
      "InformationContentEntity - {'regulates'} -> BiologicalProcess\n",
      "InformationContentEntity - {'produces'} -> BiologicalProcess\n",
      "InformationContentEntity - {'has_part'} -> BiologicalProcess\n",
      "InformationContentEntity - {'located_in'} -> BiologicalProcess\n",
      "InformationContentEntity - {'predisposes'} -> BiologicalProcess\n",
      "InformationContentEntity - {'prevents'} -> BiologicalProcess\n",
      "InformationContentEntity - {'temporally_related_to'} -> BiologicalProcess\n",
      "InformationContentEntity - {'colocalizes_with'} -> BiologicalProcess\n",
      "InformationContentEntity - {'manifestation_of'} -> BiologicalProcess\n",
      "InformationContentEntity - {'precedes'} -> BiologicalProcess\n",
      "BiologicalProcess - {'correlated_with'} -> Publication\n",
      "BiologicalProcess - {'disrupts'} -> Publication\n",
      "BiologicalProcess - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalProcess - {'regulates'} -> Publication\n",
      "BiologicalProcess - {'biomarker_for'} -> Publication\n",
      "BiologicalProcess - {'has_part'} -> Publication\n",
      "BiologicalProcess - {'located_in'} -> Publication\n",
      "BiologicalProcess - {'predisposes'} -> Publication\n",
      "BiologicalProcess - {'precedes'} -> Publication\n",
      "BiologicalProcess - {'prevents'} -> Publication\n",
      "BiologicalProcess - {'causes'} -> Publication\n",
      "BiologicalProcess - {'close_match'} -> Publication\n",
      "BiologicalProcess - {'associated_with'} -> Publication\n",
      "BiologicalProcess - {'produces'} -> Publication\n",
      "BiologicalProcess - {'has_participant'} -> Publication\n",
      "BiologicalProcess - {'subclass_of'} -> Publication\n",
      "BiologicalProcess - {'related_to'} -> Publication\n",
      "BiologicalProcess - {'affects'} -> Publication\n",
      "BiologicalProcess - {'treats'} -> Publication\n",
      "BiologicalProcess - {'manifestation_of'} -> Publication\n",
      "BiologicalProcess - {'has_input'} -> Publication\n",
      "BiologicalProcess - {'has_output'} -> Publication\n",
      "BiologicalProcess - {'occurs_in'} -> Publication\n",
      "BiologicalProcess - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->PhysiologicalProcess->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'affects'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'similar_to'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'related_to'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'overlaps'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'composed_primarily_of'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'affected_by'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'interacts_with'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'participates_in'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'has_participant'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'associated_with'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'has_upstream_actor'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'primarily_composed_of'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'temporally_related_to'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'contributes_to'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'predisposes'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'prevents'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'manifestation_of'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'precedes'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'close_match'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'subclass_of'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'diagnoses'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'enables'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'treats'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'has_input'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'occurs_in'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'exacerbates'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'causes'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'coexists_with'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'disrupts'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'capable_of'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'located_in'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'regulates'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'produces'} -> PhysiologicalProcess\n",
      "InformationContentEntity - {'has_part'} -> PhysiologicalProcess\n",
      "PhysiologicalProcess - {'correlated_with'} -> Publication\n",
      "PhysiologicalProcess - {'has_input'} -> Publication\n",
      "PhysiologicalProcess - {'causes'} -> Publication\n",
      "PhysiologicalProcess - {'has_output'} -> Publication\n",
      "PhysiologicalProcess - {'disrupts'} -> Publication\n",
      "PhysiologicalProcess - {'associated_with'} -> Publication\n",
      "PhysiologicalProcess - {'located_in'} -> Publication\n",
      "PhysiologicalProcess - {'produces'} -> Publication\n",
      "PhysiologicalProcess - {'predisposes'} -> Publication\n",
      "PhysiologicalProcess - {'precedes'} -> Publication\n",
      "PhysiologicalProcess - {'related_to'} -> Publication\n",
      "PhysiologicalProcess - {'occurs_in'} -> Publication\n",
      "PhysiologicalProcess - {'affects'} -> Publication\n",
      "PhysiologicalProcess - {'coexists_with'} -> Publication\n",
      "PhysiologicalProcess - {'close_match'} -> Publication\n",
      "PhysiologicalProcess - {'has_participant'} -> Publication\n",
      "PhysiologicalProcess - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Protein->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Protein\n",
      "InformationContentEntity - {'associated_with'} -> Protein\n",
      "InformationContentEntity - {'overlaps'} -> Protein\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Protein\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Protein\n",
      "InformationContentEntity - {'affects'} -> Protein\n",
      "InformationContentEntity - {'similar_to'} -> Protein\n",
      "InformationContentEntity - {'related_to'} -> Protein\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Protein\n",
      "InformationContentEntity - {'has_participant'} -> Protein\n",
      "InformationContentEntity - {'affected_by'} -> Protein\n",
      "InformationContentEntity - {'interacts_with'} -> Protein\n",
      "InformationContentEntity - {'participates_in'} -> Protein\n",
      "InformationContentEntity - {'caused_by'} -> Protein\n",
      "InformationContentEntity - {'disrupts'} -> Protein\n",
      "InformationContentEntity - {'gene_product_of'} -> Protein\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Protein\n",
      "InformationContentEntity - {'capable_of'} -> Protein\n",
      "InformationContentEntity - {'regulates'} -> Protein\n",
      "InformationContentEntity - {'produces'} -> Protein\n",
      "InformationContentEntity - {'located_in'} -> Protein\n",
      "InformationContentEntity - {'biomarker_for'} -> Protein\n",
      "InformationContentEntity - {'predisposes'} -> Protein\n",
      "InformationContentEntity - {'has_part'} -> Protein\n",
      "InformationContentEntity - {'prevents'} -> Protein\n",
      "InformationContentEntity - {'colocalizes_with'} -> Protein\n",
      "InformationContentEntity - {'precedes'} -> Protein\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> Protein\n",
      "InformationContentEntity - {'manifestation_of'} -> Protein\n",
      "InformationContentEntity - {'close_match'} -> Protein\n",
      "InformationContentEntity - {'subclass_of'} -> Protein\n",
      "InformationContentEntity - {'diagnoses'} -> Protein\n",
      "InformationContentEntity - {'treats'} -> Protein\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Protein\n",
      "InformationContentEntity - {'occurs_in'} -> Protein\n",
      "InformationContentEntity - {'has_input'} -> Protein\n",
      "InformationContentEntity - {'derives_from'} -> Protein\n",
      "InformationContentEntity - {'exacerbates'} -> Protein\n",
      "InformationContentEntity - {'causes'} -> Protein\n",
      "InformationContentEntity - {'has_output'} -> Protein\n",
      "InformationContentEntity - {'coexists_with'} -> Protein\n",
      "Protein - {'correlated_with'} -> Publication\n",
      "Protein - {'gene_product_of'} -> Publication\n",
      "Protein - {'associated_with'} -> Publication\n",
      "Protein - {'gene_associated_with_condition'} -> Publication\n",
      "Protein - {'located_in'} -> Publication\n",
      "Protein - {'regulates'} -> Publication\n",
      "Protein - {'biomarker_for'} -> Publication\n",
      "Protein - {'expressed_in'} -> Publication\n",
      "Protein - {'has_part'} -> Publication\n",
      "Protein - {'colocalizes_with'} -> Publication\n",
      "Protein - {'produces'} -> Publication\n",
      "Protein - {'predisposes'} -> Publication\n",
      "Protein - {'related_to'} -> Publication\n",
      "Protein - {'prevents'} -> Publication\n",
      "Protein - {'occurs_in'} -> Publication\n",
      "Protein - {'precedes'} -> Publication\n",
      "Protein - {'affects'} -> Publication\n",
      "Protein - {'close_match'} -> Publication\n",
      "Protein - {'subclass_of'} -> Publication\n",
      "Protein - {'interacts_with'} -> Publication\n",
      "Protein - {'treats'} -> Publication\n",
      "Protein - {'has_input'} -> Publication\n",
      "Protein - {'causes'} -> Publication\n",
      "Protein - {'physically_interacts_with'} -> Publication\n",
      "Protein - {'exacerbates'} -> Publication\n",
      "Protein - {'coexists_with'} -> Publication\n",
      "Protein - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Device->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Device\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Device\n",
      "InformationContentEntity - {'affected_by'} -> Device\n",
      "InformationContentEntity - {'interacts_with'} -> Device\n",
      "InformationContentEntity - {'participates_in'} -> Device\n",
      "InformationContentEntity - {'has_participant'} -> Device\n",
      "InformationContentEntity - {'associated_with'} -> Device\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Device\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Device\n",
      "InformationContentEntity - {'affects'} -> Device\n",
      "InformationContentEntity - {'similar_to'} -> Device\n",
      "InformationContentEntity - {'related_to'} -> Device\n",
      "InformationContentEntity - {'overlaps'} -> Device\n",
      "InformationContentEntity - {'predisposes'} -> Device\n",
      "InformationContentEntity - {'precedes'} -> Device\n",
      "InformationContentEntity - {'close_match'} -> Device\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> Device\n",
      "InformationContentEntity - {'subclass_of'} -> Device\n",
      "InformationContentEntity - {'diagnoses'} -> Device\n",
      "InformationContentEntity - {'treats'} -> Device\n",
      "InformationContentEntity - {'occurs_in'} -> Device\n",
      "InformationContentEntity - {'has_input'} -> Device\n",
      "InformationContentEntity - {'derives_from'} -> Device\n",
      "InformationContentEntity - {'exacerbates'} -> Device\n",
      "InformationContentEntity - {'causes'} -> Device\n",
      "InformationContentEntity - {'coexists_with'} -> Device\n",
      "InformationContentEntity - {'disrupts'} -> Device\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Device\n",
      "InformationContentEntity - {'regulates'} -> Device\n",
      "InformationContentEntity - {'produces'} -> Device\n",
      "InformationContentEntity - {'has_part'} -> Device\n",
      "InformationContentEntity - {'located_in'} -> Device\n",
      "Device - {'subclass_of'} -> Publication\n",
      "Device - {'diagnoses'} -> Publication\n",
      "Device - {'related_to'} -> Publication\n",
      "Device - {'occurs_in'} -> Publication\n",
      "Device - {'affects'} -> Publication\n",
      "Device - {'treats'} -> Publication\n",
      "Device - {'has_input'} -> Publication\n",
      "Device - {'coexists_with'} -> Publication\n",
      "Device - {'regulates'} -> Publication\n",
      "Device - {'disrupts'} -> Publication\n",
      "Device - {'biomarker_for'} -> Publication\n",
      "Device - {'gene_associated_with_condition'} -> Publication\n",
      "Device - {'has_part'} -> Publication\n",
      "Device - {'located_in'} -> Publication\n",
      "Device - {'causes'} -> Publication\n",
      "Device - {'precedes'} -> Publication\n",
      "Device - {'close_match'} -> Publication\n",
      "Device - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Gene->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Gene\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Gene\n",
      "InformationContentEntity - {'affects'} -> Gene\n",
      "InformationContentEntity - {'similar_to'} -> Gene\n",
      "InformationContentEntity - {'affected_by'} -> Gene\n",
      "InformationContentEntity - {'has_participant'} -> Gene\n",
      "InformationContentEntity - {'interacts_with'} -> Gene\n",
      "InformationContentEntity - {'participates_in'} -> Gene\n",
      "InformationContentEntity - {'caused_by'} -> Gene\n",
      "InformationContentEntity - {'associated_with'} -> Gene\n",
      "InformationContentEntity - {'related_to'} -> Gene\n",
      "InformationContentEntity - {'overlaps'} -> Gene\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Gene\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Gene\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Gene\n",
      "InformationContentEntity - {'capable_of'} -> Gene\n",
      "InformationContentEntity - {'regulates'} -> Gene\n",
      "InformationContentEntity - {'disrupts'} -> Gene\n",
      "InformationContentEntity - {'located_in'} -> Gene\n",
      "InformationContentEntity - {'biomarker_for'} -> Gene\n",
      "InformationContentEntity - {'produces'} -> Gene\n",
      "InformationContentEntity - {'predisposes'} -> Gene\n",
      "InformationContentEntity - {'has_part'} -> Gene\n",
      "InformationContentEntity - {'prevents'} -> Gene\n",
      "InformationContentEntity - {'colocalizes_with'} -> Gene\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> Gene\n",
      "InformationContentEntity - {'manifestation_of'} -> Gene\n",
      "InformationContentEntity - {'precedes'} -> Gene\n",
      "InformationContentEntity - {'close_match'} -> Gene\n",
      "InformationContentEntity - {'subclass_of'} -> Gene\n",
      "InformationContentEntity - {'diagnoses'} -> Gene\n",
      "InformationContentEntity - {'treats'} -> Gene\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Gene\n",
      "InformationContentEntity - {'occurs_in'} -> Gene\n",
      "InformationContentEntity - {'has_input'} -> Gene\n",
      "InformationContentEntity - {'derives_from'} -> Gene\n",
      "InformationContentEntity - {'exacerbates'} -> Gene\n",
      "InformationContentEntity - {'causes'} -> Gene\n",
      "InformationContentEntity - {'has_output'} -> Gene\n",
      "InformationContentEntity - {'coexists_with'} -> Gene\n",
      "InformationContentEntity - {'gene_product_of'} -> Gene\n",
      "Gene - {'correlated_with'} -> Publication\n",
      "Gene - {'prevents'} -> Publication\n",
      "Gene - {'treats'} -> Publication\n",
      "Gene - {'has_input'} -> Publication\n",
      "Gene - {'related_to'} -> Publication\n",
      "Gene - {'occurs_in'} -> Publication\n",
      "Gene - {'affects'} -> Publication\n",
      "Gene - {'exacerbates'} -> Publication\n",
      "Gene - {'coexists_with'} -> Publication\n",
      "Gene - {'disrupts'} -> Publication\n",
      "Gene - {'gene_associated_with_condition'} -> Publication\n",
      "Gene - {'located_in'} -> Publication\n",
      "Gene - {'regulates'} -> Publication\n",
      "Gene - {'biomarker_for'} -> Publication\n",
      "Gene - {'expressed_in'} -> Publication\n",
      "Gene - {'physically_interacts_with'} -> Publication\n",
      "Gene - {'predisposes'} -> Publication\n",
      "Gene - {'causes'} -> Publication\n",
      "Gene - {'precedes'} -> Publication\n",
      "Gene - {'gene_product_of'} -> Publication\n",
      "Gene - {'close_match'} -> Publication\n",
      "Gene - {'associated_with'} -> Publication\n",
      "Gene - {'subclass_of'} -> Publication\n",
      "Gene - {'has_part'} -> Publication\n",
      "Gene - {'colocalizes_with'} -> Publication\n",
      "Gene - {'interacts_with'} -> Publication\n",
      "Gene - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Procedure->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Procedure\n",
      "InformationContentEntity - {'associated_with'} -> Procedure\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Procedure\n",
      "InformationContentEntity - {'affects'} -> Procedure\n",
      "InformationContentEntity - {'similar_to'} -> Procedure\n",
      "InformationContentEntity - {'related_to'} -> Procedure\n",
      "InformationContentEntity - {'overlaps'} -> Procedure\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Procedure\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Procedure\n",
      "InformationContentEntity - {'affected_by'} -> Procedure\n",
      "InformationContentEntity - {'interacts_with'} -> Procedure\n",
      "InformationContentEntity - {'participates_in'} -> Procedure\n",
      "InformationContentEntity - {'has_participant'} -> Procedure\n",
      "InformationContentEntity - {'treats'} -> Procedure\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Procedure\n",
      "InformationContentEntity - {'occurs_in'} -> Procedure\n",
      "InformationContentEntity - {'has_input'} -> Procedure\n",
      "InformationContentEntity - {'derives_from'} -> Procedure\n",
      "InformationContentEntity - {'exacerbates'} -> Procedure\n",
      "InformationContentEntity - {'causes'} -> Procedure\n",
      "InformationContentEntity - {'coexists_with'} -> Procedure\n",
      "InformationContentEntity - {'disrupts'} -> Procedure\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Procedure\n",
      "InformationContentEntity - {'regulates'} -> Procedure\n",
      "InformationContentEntity - {'produces'} -> Procedure\n",
      "InformationContentEntity - {'has_part'} -> Procedure\n",
      "InformationContentEntity - {'located_in'} -> Procedure\n",
      "InformationContentEntity - {'biomarker_for'} -> Procedure\n",
      "InformationContentEntity - {'predisposes'} -> Procedure\n",
      "InformationContentEntity - {'prevents'} -> Procedure\n",
      "InformationContentEntity - {'manifestation_of'} -> Procedure\n",
      "InformationContentEntity - {'precedes'} -> Procedure\n",
      "InformationContentEntity - {'close_match'} -> Procedure\n",
      "InformationContentEntity - {'subclass_of'} -> Procedure\n",
      "InformationContentEntity - {'diagnoses'} -> Procedure\n",
      "Procedure - {'correlated_with'} -> Publication\n",
      "Procedure - {'manifestation_of'} -> Publication\n",
      "Procedure - {'close_match'} -> Publication\n",
      "Procedure - {'subclass_of'} -> Publication\n",
      "Procedure - {'interacts_with'} -> Publication\n",
      "Procedure - {'diagnoses'} -> Publication\n",
      "Procedure - {'occurs_in'} -> Publication\n",
      "Procedure - {'treats'} -> Publication\n",
      "Procedure - {'has_input'} -> Publication\n",
      "Procedure - {'exacerbates'} -> Publication\n",
      "Procedure - {'causes'} -> Publication\n",
      "Procedure - {'coexists_with'} -> Publication\n",
      "Procedure - {'disrupts'} -> Publication\n",
      "Procedure - {'associated_with'} -> Publication\n",
      "Procedure - {'regulates'} -> Publication\n",
      "Procedure - {'produces'} -> Publication\n",
      "Procedure - {'located_in'} -> Publication\n",
      "Procedure - {'predisposes'} -> Publication\n",
      "Procedure - {'related_to'} -> Publication\n",
      "Procedure - {'has_part'} -> Publication\n",
      "Procedure - {'prevents'} -> Publication\n",
      "Procedure - {'precedes'} -> Publication\n",
      "Procedure - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Publication\n",
      "InformationContentEntity - {'subclass_of'} -> Publication\n",
      "InformationContentEntity - {'related_to'} -> Publication\n",
      "InformationContentEntity - {'disrupts'} -> Publication\n",
      "InformationContentEntity - {'associated_with'} -> Publication\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Publication\n",
      "InformationContentEntity - {'regulates'} -> Publication\n",
      "InformationContentEntity - {'biomarker_for'} -> Publication\n",
      "InformationContentEntity - {'produces'} -> Publication\n",
      "InformationContentEntity - {'has_part'} -> Publication\n",
      "InformationContentEntity - {'located_in'} -> Publication\n",
      "InformationContentEntity - {'predisposes'} -> Publication\n",
      "InformationContentEntity - {'prevents'} -> Publication\n",
      "InformationContentEntity - {'affects'} -> Publication\n",
      "InformationContentEntity - {'manifestation_of'} -> Publication\n",
      "InformationContentEntity - {'precedes'} -> Publication\n",
      "InformationContentEntity - {'close_match'} -> Publication\n",
      "InformationContentEntity - {'interacts_with'} -> Publication\n",
      "InformationContentEntity - {'diagnoses'} -> Publication\n",
      "InformationContentEntity - {'treats'} -> Publication\n",
      "InformationContentEntity - {'occurs_in'} -> Publication\n",
      "InformationContentEntity - {'has_input'} -> Publication\n",
      "InformationContentEntity - {'exacerbates'} -> Publication\n",
      "InformationContentEntity - {'causes'} -> Publication\n",
      "InformationContentEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ClinicalAttribute->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'composed_primarily_of'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'has_upstream_actor'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'primarily_composed_of'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'affected_by'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'participates_in'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'has_participant'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'interacts_with'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'associated_with'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'affects'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'similar_to'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'related_to'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'overlaps'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'causes'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'located_in'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'coexists_with'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'close_match'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'diagnoses'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'subclass_of'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'treats'} -> ClinicalAttribute\n",
      "InformationContentEntity - {'predisposes'} -> ClinicalAttribute\n",
      "ClinicalAttribute - {'correlated_with'} -> Publication\n",
      "ClinicalAttribute - {'has_input'} -> Publication\n",
      "ClinicalAttribute - {'subclass_of'} -> Publication\n",
      "ClinicalAttribute - {'related_to'} -> Publication\n",
      "ClinicalAttribute - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Annotation->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Annotation\n",
      "InformationContentEntity - {'participates_in'} -> Annotation\n",
      "InformationContentEntity - {'has_participant'} -> Annotation\n",
      "InformationContentEntity - {'associated_with'} -> Annotation\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Annotation\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Annotation\n",
      "InformationContentEntity - {'affects'} -> Annotation\n",
      "InformationContentEntity - {'similar_to'} -> Annotation\n",
      "InformationContentEntity - {'related_to'} -> Annotation\n",
      "InformationContentEntity - {'overlaps'} -> Annotation\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Annotation\n",
      "InformationContentEntity - {'affected_by'} -> Annotation\n",
      "InformationContentEntity - {'interacts_with'} -> Annotation\n",
      "Annotation - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->MolecularMixture->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> MolecularMixture\n",
      "InformationContentEntity - {'caused_by'} -> MolecularMixture\n",
      "InformationContentEntity - {'has_upstream_actor'} -> MolecularMixture\n",
      "InformationContentEntity - {'primarily_composed_of'} -> MolecularMixture\n",
      "InformationContentEntity - {'affects'} -> MolecularMixture\n",
      "InformationContentEntity - {'similar_to'} -> MolecularMixture\n",
      "InformationContentEntity - {'related_to'} -> MolecularMixture\n",
      "InformationContentEntity - {'overlaps'} -> MolecularMixture\n",
      "InformationContentEntity - {'composed_primarily_of'} -> MolecularMixture\n",
      "InformationContentEntity - {'affected_by'} -> MolecularMixture\n",
      "InformationContentEntity - {'interacts_with'} -> MolecularMixture\n",
      "InformationContentEntity - {'participates_in'} -> MolecularMixture\n",
      "InformationContentEntity - {'has_participant'} -> MolecularMixture\n",
      "InformationContentEntity - {'associated_with'} -> MolecularMixture\n",
      "InformationContentEntity - {'located_in'} -> MolecularMixture\n",
      "InformationContentEntity - {'produces'} -> MolecularMixture\n",
      "InformationContentEntity - {'subclass_of'} -> MolecularMixture\n",
      "InformationContentEntity - {'occurs_in'} -> MolecularMixture\n",
      "InformationContentEntity - {'has_part'} -> MolecularMixture\n",
      "InformationContentEntity - {'has_input'} -> MolecularMixture\n",
      "MolecularMixture - {'correlated_with'} -> Publication\n",
      "MolecularMixture - {'treats'} -> Publication\n",
      "MolecularMixture - {'subclass_of'} -> Publication\n",
      "MolecularMixture - {'located_in'} -> Publication\n",
      "MolecularMixture - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->NamedThing->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'subclass_of'} -> NamedThing\n",
      "InformationContentEntity - {'related_to'} -> NamedThing\n",
      "InformationContentEntity - {'primarily_composed_of'} -> NamedThing\n",
      "InformationContentEntity - {'affects'} -> NamedThing\n",
      "InformationContentEntity - {'similar_to'} -> NamedThing\n",
      "InformationContentEntity - {'overlaps'} -> NamedThing\n",
      "InformationContentEntity - {'composed_primarily_of'} -> NamedThing\n",
      "InformationContentEntity - {'affected_by'} -> NamedThing\n",
      "InformationContentEntity - {'interacts_with'} -> NamedThing\n",
      "InformationContentEntity - {'participates_in'} -> NamedThing\n",
      "InformationContentEntity - {'caused_by'} -> NamedThing\n",
      "InformationContentEntity - {'has_participant'} -> NamedThing\n",
      "InformationContentEntity - {'associated_with'} -> NamedThing\n",
      "InformationContentEntity - {'has_upstream_actor'} -> NamedThing\n",
      "InformationContentEntity - {'treats'} -> NamedThing\n",
      "InformationContentEntity - {'has_input'} -> NamedThing\n",
      "InformationContentEntity - {'derives_from'} -> NamedThing\n",
      "InformationContentEntity - {'in_taxon'} -> NamedThing\n",
      "InformationContentEntity - {'physically_interacts_with'} -> NamedThing\n",
      "InformationContentEntity - {'has_output'} -> NamedThing\n",
      "InformationContentEntity - {'occurs_in'} -> NamedThing\n",
      "InformationContentEntity - {'exacerbates'} -> NamedThing\n",
      "InformationContentEntity - {'causes'} -> NamedThing\n",
      "InformationContentEntity - {'gene_product_of'} -> NamedThing\n",
      "InformationContentEntity - {'coexists_with'} -> NamedThing\n",
      "InformationContentEntity - {'actively_involved_in'} -> NamedThing\n",
      "InformationContentEntity - {'has_phenotype'} -> NamedThing\n",
      "InformationContentEntity - {'disrupts'} -> NamedThing\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> NamedThing\n",
      "InformationContentEntity - {'capable_of'} -> NamedThing\n",
      "InformationContentEntity - {'located_in'} -> NamedThing\n",
      "InformationContentEntity - {'regulates'} -> NamedThing\n",
      "InformationContentEntity - {'biomarker_for'} -> NamedThing\n",
      "InformationContentEntity - {'produces'} -> NamedThing\n",
      "InformationContentEntity - {'has_part'} -> NamedThing\n",
      "InformationContentEntity - {'temporally_related_to'} -> NamedThing\n",
      "InformationContentEntity - {'colocalizes_with'} -> NamedThing\n",
      "InformationContentEntity - {'predisposes'} -> NamedThing\n",
      "InformationContentEntity - {'prevents'} -> NamedThing\n",
      "InformationContentEntity - {'manifestation_of'} -> NamedThing\n",
      "InformationContentEntity - {'precedes'} -> NamedThing\n",
      "InformationContentEntity - {'close_match'} -> NamedThing\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> NamedThing\n",
      "InformationContentEntity - {'correlated_with'} -> NamedThing\n",
      "InformationContentEntity - {'contraindicated_for'} -> NamedThing\n",
      "InformationContentEntity - {'diagnoses'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Publication\n",
      "NamedThing - {'related_to'} -> Publication\n",
      "NamedThing - {'is_sequence_variant_of'} -> Publication\n",
      "NamedThing - {'associated_with'} -> Publication\n",
      "NamedThing - {'has_phenotype'} -> Publication\n",
      "NamedThing - {'disrupts'} -> Publication\n",
      "NamedThing - {'biomarker_for'} -> Publication\n",
      "NamedThing - {'correlated_with'} -> Publication\n",
      "NamedThing - {'gene_associated_with_condition'} -> Publication\n",
      "NamedThing - {'has_part'} -> Publication\n",
      "NamedThing - {'located_in'} -> Publication\n",
      "NamedThing - {'colocalizes_with'} -> Publication\n",
      "NamedThing - {'interacts_with'} -> Publication\n",
      "NamedThing - {'produces'} -> Publication\n",
      "NamedThing - {'prevents'} -> Publication\n",
      "NamedThing - {'diagnoses'} -> Publication\n",
      "NamedThing - {'predisposes'} -> Publication\n",
      "NamedThing - {'precedes'} -> Publication\n",
      "NamedThing - {'occurs_in'} -> Publication\n",
      "NamedThing - {'affects'} -> Publication\n",
      "NamedThing - {'close_match'} -> Publication\n",
      "NamedThing - {'manifestation_of'} -> Publication\n",
      "NamedThing - {'exacerbates'} -> Publication\n",
      "NamedThing - {'coexists_with'} -> Publication\n",
      "NamedThing - {'regulates'} -> Publication\n",
      "NamedThing - {'has_input'} -> Publication\n",
      "NamedThing - {'physically_interacts_with'} -> Publication\n",
      "NamedThing - {'treats'} -> Publication\n",
      "NamedThing - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Entity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'subclass_of'} -> Entity\n",
      "InformationContentEntity - {'related_to'} -> Entity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Entity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Entity\n",
      "InformationContentEntity - {'affects'} -> Entity\n",
      "InformationContentEntity - {'similar_to'} -> Entity\n",
      "InformationContentEntity - {'overlaps'} -> Entity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Entity\n",
      "InformationContentEntity - {'affected_by'} -> Entity\n",
      "InformationContentEntity - {'interacts_with'} -> Entity\n",
      "InformationContentEntity - {'participates_in'} -> Entity\n",
      "InformationContentEntity - {'caused_by'} -> Entity\n",
      "InformationContentEntity - {'has_participant'} -> Entity\n",
      "InformationContentEntity - {'associated_with'} -> Entity\n",
      "Entity - {'subclass_of'} -> Publication\n",
      "Entity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->MolecularActivity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'associated_with'} -> MolecularActivity\n",
      "InformationContentEntity - {'affects'} -> MolecularActivity\n",
      "InformationContentEntity - {'related_to'} -> MolecularActivity\n",
      "InformationContentEntity - {'overlaps'} -> MolecularActivity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> MolecularActivity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> MolecularActivity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> MolecularActivity\n",
      "InformationContentEntity - {'similar_to'} -> MolecularActivity\n",
      "InformationContentEntity - {'affected_by'} -> MolecularActivity\n",
      "InformationContentEntity - {'participates_in'} -> MolecularActivity\n",
      "InformationContentEntity - {'has_participant'} -> MolecularActivity\n",
      "InformationContentEntity - {'interacts_with'} -> MolecularActivity\n",
      "InformationContentEntity - {'caused_by'} -> MolecularActivity\n",
      "InformationContentEntity - {'occurs_in'} -> MolecularActivity\n",
      "InformationContentEntity - {'causes'} -> MolecularActivity\n",
      "InformationContentEntity - {'coexists_with'} -> MolecularActivity\n",
      "InformationContentEntity - {'disrupts'} -> MolecularActivity\n",
      "InformationContentEntity - {'catalyzes'} -> MolecularActivity\n",
      "InformationContentEntity - {'located_in'} -> MolecularActivity\n",
      "InformationContentEntity - {'regulates'} -> MolecularActivity\n",
      "InformationContentEntity - {'produces'} -> MolecularActivity\n",
      "InformationContentEntity - {'has_part'} -> MolecularActivity\n",
      "InformationContentEntity - {'contributes_to'} -> MolecularActivity\n",
      "InformationContentEntity - {'manifestation_of'} -> MolecularActivity\n",
      "InformationContentEntity - {'precedes'} -> MolecularActivity\n",
      "InformationContentEntity - {'close_match'} -> MolecularActivity\n",
      "InformationContentEntity - {'subclass_of'} -> MolecularActivity\n",
      "InformationContentEntity - {'enables'} -> MolecularActivity\n",
      "InformationContentEntity - {'has_input'} -> MolecularActivity\n",
      "InformationContentEntity - {'derives_from'} -> MolecularActivity\n",
      "MolecularActivity - {'occurs_in'} -> Publication\n",
      "MolecularActivity - {'treats'} -> Publication\n",
      "MolecularActivity - {'has_input'} -> Publication\n",
      "MolecularActivity - {'has_output'} -> Publication\n",
      "MolecularActivity - {'coexists_with'} -> Publication\n",
      "MolecularActivity - {'regulates'} -> Publication\n",
      "MolecularActivity - {'disrupts'} -> Publication\n",
      "MolecularActivity - {'associated_with'} -> Publication\n",
      "MolecularActivity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularActivity - {'located_in'} -> Publication\n",
      "MolecularActivity - {'produces'} -> Publication\n",
      "MolecularActivity - {'related_to'} -> Publication\n",
      "MolecularActivity - {'affects'} -> Publication\n",
      "MolecularActivity - {'has_participant'} -> Publication\n",
      "MolecularActivity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Behavior->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'has_participant'} -> Behavior\n",
      "InformationContentEntity - {'interacts_with'} -> Behavior\n",
      "InformationContentEntity - {'associated_with'} -> Behavior\n",
      "InformationContentEntity - {'affects'} -> Behavior\n",
      "InformationContentEntity - {'similar_to'} -> Behavior\n",
      "InformationContentEntity - {'related_to'} -> Behavior\n",
      "InformationContentEntity - {'overlaps'} -> Behavior\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Behavior\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Behavior\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Behavior\n",
      "InformationContentEntity - {'affected_by'} -> Behavior\n",
      "InformationContentEntity - {'participates_in'} -> Behavior\n",
      "InformationContentEntity - {'manifestation_of'} -> Behavior\n",
      "InformationContentEntity - {'precedes'} -> Behavior\n",
      "InformationContentEntity - {'close_match'} -> Behavior\n",
      "InformationContentEntity - {'subclass_of'} -> Behavior\n",
      "InformationContentEntity - {'diagnoses'} -> Behavior\n",
      "InformationContentEntity - {'treats'} -> Behavior\n",
      "InformationContentEntity - {'occurs_in'} -> Behavior\n",
      "InformationContentEntity - {'has_input'} -> Behavior\n",
      "InformationContentEntity - {'exacerbates'} -> Behavior\n",
      "InformationContentEntity - {'causes'} -> Behavior\n",
      "InformationContentEntity - {'coexists_with'} -> Behavior\n",
      "InformationContentEntity - {'disrupts'} -> Behavior\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Behavior\n",
      "InformationContentEntity - {'regulates'} -> Behavior\n",
      "InformationContentEntity - {'has_part'} -> Behavior\n",
      "InformationContentEntity - {'predisposes'} -> Behavior\n",
      "InformationContentEntity - {'prevents'} -> Behavior\n",
      "Behavior - {'treats'} -> Publication\n",
      "Behavior - {'has_input'} -> Publication\n",
      "Behavior - {'subclass_of'} -> Publication\n",
      "Behavior - {'located_in'} -> Publication\n",
      "Behavior - {'interacts_with'} -> Publication\n",
      "Behavior - {'predisposes'} -> Publication\n",
      "Behavior - {'related_to'} -> Publication\n",
      "Behavior - {'occurs_in'} -> Publication\n",
      "Behavior - {'affects'} -> Publication\n",
      "Behavior - {'coexists_with'} -> Publication\n",
      "Behavior - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->CellularComponent->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'affected_by'} -> CellularComponent\n",
      "InformationContentEntity - {'participates_in'} -> CellularComponent\n",
      "InformationContentEntity - {'has_participant'} -> CellularComponent\n",
      "InformationContentEntity - {'interacts_with'} -> CellularComponent\n",
      "InformationContentEntity - {'associated_with'} -> CellularComponent\n",
      "InformationContentEntity - {'affects'} -> CellularComponent\n",
      "InformationContentEntity - {'similar_to'} -> CellularComponent\n",
      "InformationContentEntity - {'related_to'} -> CellularComponent\n",
      "InformationContentEntity - {'overlaps'} -> CellularComponent\n",
      "InformationContentEntity - {'composed_primarily_of'} -> CellularComponent\n",
      "InformationContentEntity - {'has_upstream_actor'} -> CellularComponent\n",
      "InformationContentEntity - {'primarily_composed_of'} -> CellularComponent\n",
      "InformationContentEntity - {'manifestation_of'} -> CellularComponent\n",
      "InformationContentEntity - {'precedes'} -> CellularComponent\n",
      "InformationContentEntity - {'close_match'} -> CellularComponent\n",
      "InformationContentEntity - {'subclass_of'} -> CellularComponent\n",
      "InformationContentEntity - {'diagnoses'} -> CellularComponent\n",
      "InformationContentEntity - {'treats'} -> CellularComponent\n",
      "InformationContentEntity - {'occurs_in'} -> CellularComponent\n",
      "InformationContentEntity - {'has_input'} -> CellularComponent\n",
      "InformationContentEntity - {'derives_from'} -> CellularComponent\n",
      "InformationContentEntity - {'causes'} -> CellularComponent\n",
      "InformationContentEntity - {'coexists_with'} -> CellularComponent\n",
      "InformationContentEntity - {'actively_involved_in'} -> CellularComponent\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> CellularComponent\n",
      "InformationContentEntity - {'regulates'} -> CellularComponent\n",
      "InformationContentEntity - {'disrupts'} -> CellularComponent\n",
      "InformationContentEntity - {'expressed_in'} -> CellularComponent\n",
      "InformationContentEntity - {'located_in'} -> CellularComponent\n",
      "InformationContentEntity - {'produces'} -> CellularComponent\n",
      "InformationContentEntity - {'predisposes'} -> CellularComponent\n",
      "InformationContentEntity - {'has_part'} -> CellularComponent\n",
      "InformationContentEntity - {'prevents'} -> CellularComponent\n",
      "InformationContentEntity - {'colocalizes_with'} -> CellularComponent\n",
      "CellularComponent - {'subclass_of'} -> Publication\n",
      "CellularComponent - {'produces'} -> Publication\n",
      "CellularComponent - {'has_part'} -> Publication\n",
      "CellularComponent - {'related_to'} -> Publication\n",
      "CellularComponent - {'occurs_in'} -> Publication\n",
      "CellularComponent - {'affects'} -> Publication\n",
      "CellularComponent - {'regulates'} -> Publication\n",
      "CellularComponent - {'disrupts'} -> Publication\n",
      "CellularComponent - {'located_in'} -> Publication\n",
      "CellularComponent - {'associated_with'} -> Publication\n",
      "CellularComponent - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->MolecularEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'similar_to'} -> MolecularEntity\n",
      "InformationContentEntity - {'affected_by'} -> MolecularEntity\n",
      "InformationContentEntity - {'has_participant'} -> MolecularEntity\n",
      "InformationContentEntity - {'interacts_with'} -> MolecularEntity\n",
      "InformationContentEntity - {'participates_in'} -> MolecularEntity\n",
      "InformationContentEntity - {'caused_by'} -> MolecularEntity\n",
      "InformationContentEntity - {'associated_with'} -> MolecularEntity\n",
      "InformationContentEntity - {'related_to'} -> MolecularEntity\n",
      "InformationContentEntity - {'overlaps'} -> MolecularEntity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> MolecularEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> MolecularEntity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> MolecularEntity\n",
      "InformationContentEntity - {'affects'} -> MolecularEntity\n",
      "InformationContentEntity - {'treats'} -> MolecularEntity\n",
      "InformationContentEntity - {'has_input'} -> MolecularEntity\n",
      "InformationContentEntity - {'derives_from'} -> MolecularEntity\n",
      "InformationContentEntity - {'physically_interacts_with'} -> MolecularEntity\n",
      "InformationContentEntity - {'has_output'} -> MolecularEntity\n",
      "InformationContentEntity - {'occurs_in'} -> MolecularEntity\n",
      "InformationContentEntity - {'causes'} -> MolecularEntity\n",
      "InformationContentEntity - {'coexists_with'} -> MolecularEntity\n",
      "InformationContentEntity - {'disrupts'} -> MolecularEntity\n",
      "InformationContentEntity - {'chemically_similar_to'} -> MolecularEntity\n",
      "InformationContentEntity - {'capable_of'} -> MolecularEntity\n",
      "InformationContentEntity - {'located_in'} -> MolecularEntity\n",
      "InformationContentEntity - {'regulates'} -> MolecularEntity\n",
      "InformationContentEntity - {'produces'} -> MolecularEntity\n",
      "InformationContentEntity - {'has_part'} -> MolecularEntity\n",
      "InformationContentEntity - {'colocalizes_with'} -> MolecularEntity\n",
      "InformationContentEntity - {'precedes'} -> MolecularEntity\n",
      "InformationContentEntity - {'close_match'} -> MolecularEntity\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> MolecularEntity\n",
      "InformationContentEntity - {'subclass_of'} -> MolecularEntity\n",
      "InformationContentEntity - {'diagnoses'} -> MolecularEntity\n",
      "MolecularEntity - {'causes'} -> Publication\n",
      "MolecularEntity - {'associated_with'} -> Publication\n",
      "MolecularEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "MolecularEntity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularEntity - {'regulates'} -> Publication\n",
      "MolecularEntity - {'disrupts'} -> Publication\n",
      "MolecularEntity - {'located_in'} -> Publication\n",
      "MolecularEntity - {'biomarker_for'} -> Publication\n",
      "MolecularEntity - {'interacts_with'} -> Publication\n",
      "MolecularEntity - {'predisposes'} -> Publication\n",
      "MolecularEntity - {'related_to'} -> Publication\n",
      "MolecularEntity - {'has_part'} -> Publication\n",
      "MolecularEntity - {'occurs_in'} -> Publication\n",
      "MolecularEntity - {'affects'} -> Publication\n",
      "MolecularEntity - {'precedes'} -> Publication\n",
      "MolecularEntity - {'coexists_with'} -> Publication\n",
      "MolecularEntity - {'close_match'} -> Publication\n",
      "MolecularEntity - {'subclass_of'} -> Publication\n",
      "MolecularEntity - {'treats'} -> Publication\n",
      "MolecularEntity - {'physically_interacts_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->GrossAnatomicalStructure->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'has_participant'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'interacts_with'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'associated_with'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'affects'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'similar_to'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'related_to'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'overlaps'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'composed_primarily_of'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'has_upstream_actor'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'primarily_composed_of'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'affected_by'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'participates_in'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'precedes'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'manifestation_of'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'close_match'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'subclass_of'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'develops_from'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'diagnoses'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'treats'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'physically_interacts_with'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'occurs_in'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'has_input'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'derives_from'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'causes'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'coexists_with'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'actively_involved_in'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'disrupts'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'regulates'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'produces'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'expressed_in'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'located_in'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'predisposes'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'has_part'} -> GrossAnatomicalStructure\n",
      "InformationContentEntity - {'prevents'} -> GrossAnatomicalStructure\n",
      "GrossAnatomicalStructure - {'affects'} -> Publication\n",
      "GrossAnatomicalStructure - {'coexists_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'close_match'} -> Publication\n",
      "GrossAnatomicalStructure - {'develops_from'} -> Publication\n",
      "GrossAnatomicalStructure - {'regulates'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_input'} -> Publication\n",
      "GrossAnatomicalStructure - {'associated_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'located_in'} -> Publication\n",
      "GrossAnatomicalStructure - {'subclass_of'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_part'} -> Publication\n",
      "GrossAnatomicalStructure - {'produces'} -> Publication\n",
      "GrossAnatomicalStructure - {'related_to'} -> Publication\n",
      "GrossAnatomicalStructure - {'occurs_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->PathologicalProcess->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'associated_with'} -> PathologicalProcess\n",
      "InformationContentEntity - {'related_to'} -> PathologicalProcess\n",
      "InformationContentEntity - {'affects'} -> PathologicalProcess\n",
      "InformationContentEntity - {'exacerbates'} -> PathologicalProcess\n",
      "InformationContentEntity - {'coexists_with'} -> PathologicalProcess\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> PathologicalProcess\n",
      "InformationContentEntity - {'disrupts'} -> PathologicalProcess\n",
      "InformationContentEntity - {'located_in'} -> PathologicalProcess\n",
      "InformationContentEntity - {'regulates'} -> PathologicalProcess\n",
      "InformationContentEntity - {'produces'} -> PathologicalProcess\n",
      "InformationContentEntity - {'predisposes'} -> PathologicalProcess\n",
      "InformationContentEntity - {'has_part'} -> PathologicalProcess\n",
      "InformationContentEntity - {'prevents'} -> PathologicalProcess\n",
      "InformationContentEntity - {'manifestation_of'} -> PathologicalProcess\n",
      "InformationContentEntity - {'precedes'} -> PathologicalProcess\n",
      "InformationContentEntity - {'close_match'} -> PathologicalProcess\n",
      "InformationContentEntity - {'has_participant'} -> PathologicalProcess\n",
      "InformationContentEntity - {'subclass_of'} -> PathologicalProcess\n",
      "InformationContentEntity - {'interacts_with'} -> PathologicalProcess\n",
      "InformationContentEntity - {'contraindicated_for'} -> PathologicalProcess\n",
      "InformationContentEntity - {'diagnoses'} -> PathologicalProcess\n",
      "InformationContentEntity - {'treats'} -> PathologicalProcess\n",
      "InformationContentEntity - {'occurs_in'} -> PathologicalProcess\n",
      "InformationContentEntity - {'has_input'} -> PathologicalProcess\n",
      "InformationContentEntity - {'derives_from'} -> PathologicalProcess\n",
      "InformationContentEntity - {'causes'} -> PathologicalProcess\n",
      "PathologicalProcess - {'manifestation_of'} -> Publication\n",
      "PathologicalProcess - {'occurs_in'} -> Publication\n",
      "PathologicalProcess - {'coexists_with'} -> Publication\n",
      "PathologicalProcess - {'exacerbates'} -> Publication\n",
      "PathologicalProcess - {'disrupts'} -> Publication\n",
      "PathologicalProcess - {'located_in'} -> Publication\n",
      "PathologicalProcess - {'regulates'} -> Publication\n",
      "PathologicalProcess - {'treats'} -> Publication\n",
      "PathologicalProcess - {'precedes'} -> Publication\n",
      "PathologicalProcess - {'causes'} -> Publication\n",
      "PathologicalProcess - {'close_match'} -> Publication\n",
      "PathologicalProcess - {'associated_with'} -> Publication\n",
      "PathologicalProcess - {'subclass_of'} -> Publication\n",
      "PathologicalProcess - {'produces'} -> Publication\n",
      "PathologicalProcess - {'predisposes'} -> Publication\n",
      "PathologicalProcess - {'related_to'} -> Publication\n",
      "PathologicalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->GenomicEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'has_participant'} -> GenomicEntity\n",
      "InformationContentEntity - {'interacts_with'} -> GenomicEntity\n",
      "InformationContentEntity - {'caused_by'} -> GenomicEntity\n",
      "InformationContentEntity - {'associated_with'} -> GenomicEntity\n",
      "InformationContentEntity - {'affects'} -> GenomicEntity\n",
      "InformationContentEntity - {'similar_to'} -> GenomicEntity\n",
      "InformationContentEntity - {'related_to'} -> GenomicEntity\n",
      "InformationContentEntity - {'overlaps'} -> GenomicEntity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> GenomicEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> GenomicEntity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> GenomicEntity\n",
      "InformationContentEntity - {'affected_by'} -> GenomicEntity\n",
      "InformationContentEntity - {'participates_in'} -> GenomicEntity\n",
      "InformationContentEntity - {'produces'} -> GenomicEntity\n",
      "InformationContentEntity - {'has_part'} -> GenomicEntity\n",
      "InformationContentEntity - {'has_input'} -> GenomicEntity\n",
      "GenomicEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->CellLine->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'affects'} -> CellLine\n",
      "InformationContentEntity - {'related_to'} -> CellLine\n",
      "InformationContentEntity - {'subclass_of'} -> CellLine\n",
      "InformationContentEntity - {'interacts_with'} -> CellLine\n",
      "InformationContentEntity - {'diagnoses'} -> CellLine\n",
      "InformationContentEntity - {'treats'} -> CellLine\n",
      "InformationContentEntity - {'has_input'} -> CellLine\n",
      "InformationContentEntity - {'derives_from'} -> CellLine\n",
      "InformationContentEntity - {'occurs_in'} -> CellLine\n",
      "InformationContentEntity - {'associated_with'} -> CellLine\n",
      "InformationContentEntity - {'coexists_with'} -> CellLine\n",
      "InformationContentEntity - {'disrupts'} -> CellLine\n",
      "InformationContentEntity - {'located_in'} -> CellLine\n",
      "InformationContentEntity - {'regulates'} -> CellLine\n",
      "InformationContentEntity - {'produces'} -> CellLine\n",
      "InformationContentEntity - {'has_part'} -> CellLine\n",
      "CellLine - {'gene_associated_with_condition'} -> Publication\n",
      "CellLine - {'located_in'} -> Publication\n",
      "CellLine - {'produces'} -> Publication\n",
      "CellLine - {'related_to'} -> Publication\n",
      "CellLine - {'affects'} -> Publication\n",
      "CellLine - {'close_match'} -> Publication\n",
      "CellLine - {'subclass_of'} -> Publication\n",
      "CellLine - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ChemicalMixture->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'interacts_with'} -> ChemicalMixture\n",
      "InformationContentEntity - {'associated_with'} -> ChemicalMixture\n",
      "InformationContentEntity - {'affects'} -> ChemicalMixture\n",
      "InformationContentEntity - {'similar_to'} -> ChemicalMixture\n",
      "InformationContentEntity - {'related_to'} -> ChemicalMixture\n",
      "InformationContentEntity - {'overlaps'} -> ChemicalMixture\n",
      "InformationContentEntity - {'composed_primarily_of'} -> ChemicalMixture\n",
      "InformationContentEntity - {'has_upstream_actor'} -> ChemicalMixture\n",
      "InformationContentEntity - {'primarily_composed_of'} -> ChemicalMixture\n",
      "InformationContentEntity - {'affected_by'} -> ChemicalMixture\n",
      "InformationContentEntity - {'participates_in'} -> ChemicalMixture\n",
      "InformationContentEntity - {'has_participant'} -> ChemicalMixture\n",
      "InformationContentEntity - {'regulates'} -> ChemicalMixture\n",
      "InformationContentEntity - {'produces'} -> ChemicalMixture\n",
      "InformationContentEntity - {'subclass_of'} -> ChemicalMixture\n",
      "InformationContentEntity - {'has_input'} -> ChemicalMixture\n",
      "InformationContentEntity - {'derives_from'} -> ChemicalMixture\n",
      "InformationContentEntity - {'coexists_with'} -> ChemicalMixture\n",
      "ChemicalMixture - {'related_to'} -> Publication\n",
      "ChemicalMixture - {'affects'} -> Publication\n",
      "ChemicalMixture - {'subclass_of'} -> Publication\n",
      "ChemicalMixture - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->NucleicAcidEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'associated_with'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'affects'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'similar_to'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'related_to'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'overlaps'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'affected_by'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'participates_in'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'has_participant'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'interacts_with'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'caused_by'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'subclass_of'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'diagnoses'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'treats'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'physically_interacts_with'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'has_output'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'occurs_in'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'has_input'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'derives_from'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'causes'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'coexists_with'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'disrupts'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'regulates'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'produces'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'has_part'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'located_in'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'predisposes'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'manifestation_of'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'precedes'} -> NucleicAcidEntity\n",
      "InformationContentEntity - {'close_match'} -> NucleicAcidEntity\n",
      "NucleicAcidEntity - {'gene_associated_with_condition'} -> Publication\n",
      "NucleicAcidEntity - {'located_in'} -> Publication\n",
      "NucleicAcidEntity - {'related_to'} -> Publication\n",
      "NucleicAcidEntity - {'close_match'} -> Publication\n",
      "NucleicAcidEntity - {'subclass_of'} -> Publication\n",
      "NucleicAcidEntity - {'interacts_with'} -> Publication\n",
      "NucleicAcidEntity - {'occurs_in'} -> Publication\n",
      "NucleicAcidEntity - {'affects'} -> Publication\n",
      "NucleicAcidEntity - {'treats'} -> Publication\n",
      "NucleicAcidEntity - {'causes'} -> Publication\n",
      "NucleicAcidEntity - {'coexists_with'} -> Publication\n",
      "NucleicAcidEntity - {'disrupts'} -> Publication\n",
      "NucleicAcidEntity - {'associated_with'} -> Publication\n",
      "NucleicAcidEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Pathway->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Pathway\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Pathway\n",
      "InformationContentEntity - {'affects'} -> Pathway\n",
      "InformationContentEntity - {'similar_to'} -> Pathway\n",
      "InformationContentEntity - {'related_to'} -> Pathway\n",
      "InformationContentEntity - {'overlaps'} -> Pathway\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Pathway\n",
      "InformationContentEntity - {'affected_by'} -> Pathway\n",
      "InformationContentEntity - {'interacts_with'} -> Pathway\n",
      "InformationContentEntity - {'participates_in'} -> Pathway\n",
      "InformationContentEntity - {'caused_by'} -> Pathway\n",
      "InformationContentEntity - {'has_participant'} -> Pathway\n",
      "InformationContentEntity - {'associated_with'} -> Pathway\n",
      "InformationContentEntity - {'diagnoses'} -> Pathway\n",
      "InformationContentEntity - {'treats'} -> Pathway\n",
      "InformationContentEntity - {'occurs_in'} -> Pathway\n",
      "InformationContentEntity - {'has_input'} -> Pathway\n",
      "InformationContentEntity - {'derives_from'} -> Pathway\n",
      "InformationContentEntity - {'exacerbates'} -> Pathway\n",
      "InformationContentEntity - {'causes'} -> Pathway\n",
      "InformationContentEntity - {'coexists_with'} -> Pathway\n",
      "InformationContentEntity - {'disrupts'} -> Pathway\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Pathway\n",
      "InformationContentEntity - {'regulates'} -> Pathway\n",
      "InformationContentEntity - {'produces'} -> Pathway\n",
      "InformationContentEntity - {'expressed_in'} -> Pathway\n",
      "InformationContentEntity - {'has_part'} -> Pathway\n",
      "InformationContentEntity - {'located_in'} -> Pathway\n",
      "InformationContentEntity - {'colocalizes_with'} -> Pathway\n",
      "InformationContentEntity - {'predisposes'} -> Pathway\n",
      "InformationContentEntity - {'prevents'} -> Pathway\n",
      "InformationContentEntity - {'temporally_related_to'} -> Pathway\n",
      "InformationContentEntity - {'manifestation_of'} -> Pathway\n",
      "InformationContentEntity - {'precedes'} -> Pathway\n",
      "InformationContentEntity - {'close_match'} -> Pathway\n",
      "InformationContentEntity - {'subclass_of'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Publication\n",
      "Pathway - {'regulates'} -> Publication\n",
      "Pathway - {'treats'} -> Publication\n",
      "Pathway - {'has_input'} -> Publication\n",
      "Pathway - {'causes'} -> Publication\n",
      "Pathway - {'associated_with'} -> Publication\n",
      "Pathway - {'gene_associated_with_condition'} -> Publication\n",
      "Pathway - {'disrupts'} -> Publication\n",
      "Pathway - {'located_in'} -> Publication\n",
      "Pathway - {'produces'} -> Publication\n",
      "Pathway - {'predisposes'} -> Publication\n",
      "Pathway - {'related_to'} -> Publication\n",
      "Pathway - {'precedes'} -> Publication\n",
      "Pathway - {'occurs_in'} -> Publication\n",
      "Pathway - {'affects'} -> Publication\n",
      "Pathway - {'manifestation_of'} -> Publication\n",
      "Pathway - {'coexists_with'} -> Publication\n",
      "Pathway - {'close_match'} -> Publication\n",
      "Pathway - {'has_participant'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->DiseaseOrPhenotypicFeature->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'participates_in'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'overlaps'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'has_upstream_actor'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'primarily_composed_of'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'affects'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'similar_to'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'related_to'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'affected_by'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'composed_primarily_of'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'has_participant'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'subclass_of'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'contraindicated_for'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'diagnoses'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'treats'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'has_input'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'derives_from'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'physically_interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'occurs_in'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'exacerbates'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'causes'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'coexists_with'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'actively_involved_in'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'has_phenotype'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'disrupts'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'located_in'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'regulates'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'biomarker_for'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'produces'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'has_part'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'prevents'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'contributes_to'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'predisposes'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'manifestation_of'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'precedes'} -> DiseaseOrPhenotypicFeature\n",
      "InformationContentEntity - {'close_match'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->AnatomicalEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'affected_by'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'participates_in'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'has_participant'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'interacts_with'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'caused_by'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'associated_with'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'affects'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'similar_to'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'related_to'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'overlaps'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'disrupts'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'regulates'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'produces'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'expressed_in'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'has_part'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'located_in'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'colocalizes_with'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'predisposes'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'prevents'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'temporally_related_to'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'manifestation_of'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'precedes'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'close_match'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'develops_from'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'subclass_of'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'diagnoses'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'treats'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'physically_interacts_with'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'occurs_in'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'has_input'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'derives_from'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'exacerbates'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'causes'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'has_output'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'coexists_with'} -> AnatomicalEntity\n",
      "InformationContentEntity - {'actively_involved_in'} -> AnatomicalEntity\n",
      "AnatomicalEntity - {'develops_from'} -> Publication\n",
      "AnatomicalEntity - {'interacts_with'} -> Publication\n",
      "AnatomicalEntity - {'produces'} -> Publication\n",
      "AnatomicalEntity - {'has_part'} -> Publication\n",
      "AnatomicalEntity - {'diagnoses'} -> Publication\n",
      "AnatomicalEntity - {'related_to'} -> Publication\n",
      "AnatomicalEntity - {'affects'} -> Publication\n",
      "AnatomicalEntity - {'manifestation_of'} -> Publication\n",
      "AnatomicalEntity - {'has_input'} -> Publication\n",
      "AnatomicalEntity - {'occurs_in'} -> Publication\n",
      "AnatomicalEntity - {'coexists_with'} -> Publication\n",
      "AnatomicalEntity - {'close_match'} -> Publication\n",
      "AnatomicalEntity - {'disrupts'} -> Publication\n",
      "AnatomicalEntity - {'located_in'} -> Publication\n",
      "AnatomicalEntity - {'regulates'} -> Publication\n",
      "AnatomicalEntity - {'treats'} -> Publication\n",
      "AnatomicalEntity - {'causes'} -> Publication\n",
      "AnatomicalEntity - {'associated_with'} -> Publication\n",
      "AnatomicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "AnatomicalEntity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Polypeptide->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'caused_by'} -> Polypeptide\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Polypeptide\n",
      "InformationContentEntity - {'associated_with'} -> Polypeptide\n",
      "InformationContentEntity - {'affects'} -> Polypeptide\n",
      "InformationContentEntity - {'similar_to'} -> Polypeptide\n",
      "InformationContentEntity - {'related_to'} -> Polypeptide\n",
      "InformationContentEntity - {'overlaps'} -> Polypeptide\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Polypeptide\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Polypeptide\n",
      "InformationContentEntity - {'affected_by'} -> Polypeptide\n",
      "InformationContentEntity - {'participates_in'} -> Polypeptide\n",
      "InformationContentEntity - {'has_participant'} -> Polypeptide\n",
      "InformationContentEntity - {'interacts_with'} -> Polypeptide\n",
      "InformationContentEntity - {'diagnoses'} -> Polypeptide\n",
      "InformationContentEntity - {'treats'} -> Polypeptide\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Polypeptide\n",
      "InformationContentEntity - {'occurs_in'} -> Polypeptide\n",
      "InformationContentEntity - {'has_input'} -> Polypeptide\n",
      "InformationContentEntity - {'derives_from'} -> Polypeptide\n",
      "InformationContentEntity - {'causes'} -> Polypeptide\n",
      "InformationContentEntity - {'coexists_with'} -> Polypeptide\n",
      "InformationContentEntity - {'actively_involved_in'} -> Polypeptide\n",
      "InformationContentEntity - {'gene_product_of'} -> Polypeptide\n",
      "InformationContentEntity - {'capable_of'} -> Polypeptide\n",
      "InformationContentEntity - {'regulates'} -> Polypeptide\n",
      "InformationContentEntity - {'disrupts'} -> Polypeptide\n",
      "InformationContentEntity - {'chemically_similar_to'} -> Polypeptide\n",
      "InformationContentEntity - {'located_in'} -> Polypeptide\n",
      "InformationContentEntity - {'biomarker_for'} -> Polypeptide\n",
      "InformationContentEntity - {'produces'} -> Polypeptide\n",
      "InformationContentEntity - {'has_part'} -> Polypeptide\n",
      "InformationContentEntity - {'colocalizes_with'} -> Polypeptide\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> Polypeptide\n",
      "InformationContentEntity - {'precedes'} -> Polypeptide\n",
      "InformationContentEntity - {'close_match'} -> Polypeptide\n",
      "InformationContentEntity - {'subclass_of'} -> Polypeptide\n",
      "Polypeptide - {'disrupts'} -> Publication\n",
      "Polypeptide - {'associated_with'} -> Publication\n",
      "Polypeptide - {'gene_associated_with_condition'} -> Publication\n",
      "Polypeptide - {'located_in'} -> Publication\n",
      "Polypeptide - {'regulates'} -> Publication\n",
      "Polypeptide - {'biomarker_for'} -> Publication\n",
      "Polypeptide - {'produces'} -> Publication\n",
      "Polypeptide - {'has_part'} -> Publication\n",
      "Polypeptide - {'physically_interacts_with'} -> Publication\n",
      "Polypeptide - {'colocalizes_with'} -> Publication\n",
      "Polypeptide - {'predisposes'} -> Publication\n",
      "Polypeptide - {'related_to'} -> Publication\n",
      "Polypeptide - {'prevents'} -> Publication\n",
      "Polypeptide - {'precedes'} -> Publication\n",
      "Polypeptide - {'close_match'} -> Publication\n",
      "Polypeptide - {'subclass_of'} -> Publication\n",
      "Polypeptide - {'interacts_with'} -> Publication\n",
      "Polypeptide - {'affects'} -> Publication\n",
      "Polypeptide - {'treats'} -> Publication\n",
      "Polypeptide - {'occurs_in'} -> Publication\n",
      "Polypeptide - {'exacerbates'} -> Publication\n",
      "Polypeptide - {'causes'} -> Publication\n",
      "Polypeptide - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->EnvironmentalProcess->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'associated_with'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'overlaps'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'has_upstream_actor'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'primarily_composed_of'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'affects'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'similar_to'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'related_to'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'affected_by'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'composed_primarily_of'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'has_participant'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'interacts_with'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'participates_in'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'disrupts'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'regulates'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'close_match'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'subclass_of'} -> EnvironmentalProcess\n",
      "InformationContentEntity - {'treats'} -> EnvironmentalProcess\n",
      "EnvironmentalProcess - {'subclass_of'} -> Publication\n",
      "EnvironmentalProcess - {'related_to'} -> Publication\n",
      "EnvironmentalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->BiologicalProcessOrActivity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'affected_by'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'interacts_with'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'participates_in'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'caused_by'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'has_participant'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'associated_with'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'affects'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'similar_to'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'related_to'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'overlaps'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'subclass_of'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'enables'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'causes'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'coexists_with'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'disrupts'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'regulates'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'manifestation_of'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'precedes'} -> BiologicalProcessOrActivity\n",
      "InformationContentEntity - {'close_match'} -> BiologicalProcessOrActivity\n",
      "BiologicalProcessOrActivity - {'occurs_in'} -> Publication\n",
      "BiologicalProcessOrActivity - {'affects'} -> Publication\n",
      "BiologicalProcessOrActivity - {'has_participant'} -> Publication\n",
      "BiologicalProcessOrActivity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->OrganismalEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'interacts_with'} -> OrganismalEntity\n",
      "InformationContentEntity - {'participates_in'} -> OrganismalEntity\n",
      "InformationContentEntity - {'caused_by'} -> OrganismalEntity\n",
      "InformationContentEntity - {'has_participant'} -> OrganismalEntity\n",
      "InformationContentEntity - {'associated_with'} -> OrganismalEntity\n",
      "InformationContentEntity - {'overlaps'} -> OrganismalEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> OrganismalEntity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> OrganismalEntity\n",
      "InformationContentEntity - {'affects'} -> OrganismalEntity\n",
      "InformationContentEntity - {'similar_to'} -> OrganismalEntity\n",
      "InformationContentEntity - {'related_to'} -> OrganismalEntity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> OrganismalEntity\n",
      "InformationContentEntity - {'affected_by'} -> OrganismalEntity\n",
      "OrganismalEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->BiologicalEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'has_upstream_actor'} -> BiologicalEntity\n",
      "InformationContentEntity - {'primarily_composed_of'} -> BiologicalEntity\n",
      "InformationContentEntity - {'affects'} -> BiologicalEntity\n",
      "InformationContentEntity - {'similar_to'} -> BiologicalEntity\n",
      "InformationContentEntity - {'related_to'} -> BiologicalEntity\n",
      "InformationContentEntity - {'overlaps'} -> BiologicalEntity\n",
      "InformationContentEntity - {'composed_primarily_of'} -> BiologicalEntity\n",
      "InformationContentEntity - {'affected_by'} -> BiologicalEntity\n",
      "InformationContentEntity - {'interacts_with'} -> BiologicalEntity\n",
      "InformationContentEntity - {'participates_in'} -> BiologicalEntity\n",
      "InformationContentEntity - {'caused_by'} -> BiologicalEntity\n",
      "InformationContentEntity - {'has_participant'} -> BiologicalEntity\n",
      "InformationContentEntity - {'associated_with'} -> BiologicalEntity\n",
      "InformationContentEntity - {'in_taxon'} -> BiologicalEntity\n",
      "InformationContentEntity - {'physically_interacts_with'} -> BiologicalEntity\n",
      "InformationContentEntity - {'occurs_in'} -> BiologicalEntity\n",
      "InformationContentEntity - {'has_input'} -> BiologicalEntity\n",
      "InformationContentEntity - {'derives_from'} -> BiologicalEntity\n",
      "InformationContentEntity - {'exacerbates'} -> BiologicalEntity\n",
      "InformationContentEntity - {'causes'} -> BiologicalEntity\n",
      "InformationContentEntity - {'has_output'} -> BiologicalEntity\n",
      "InformationContentEntity - {'coexists_with'} -> BiologicalEntity\n",
      "InformationContentEntity - {'disrupts'} -> BiologicalEntity\n",
      "InformationContentEntity - {'gene_product_of'} -> BiologicalEntity\n",
      "InformationContentEntity - {'chemically_similar_to'} -> BiologicalEntity\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> BiologicalEntity\n",
      "InformationContentEntity - {'regulates'} -> BiologicalEntity\n",
      "InformationContentEntity - {'biomarker_for'} -> BiologicalEntity\n",
      "InformationContentEntity - {'produces'} -> BiologicalEntity\n",
      "InformationContentEntity - {'has_part'} -> BiologicalEntity\n",
      "InformationContentEntity - {'located_in'} -> BiologicalEntity\n",
      "InformationContentEntity - {'colocalizes_with'} -> BiologicalEntity\n",
      "InformationContentEntity - {'predisposes'} -> BiologicalEntity\n",
      "InformationContentEntity - {'prevents'} -> BiologicalEntity\n",
      "InformationContentEntity - {'manifestation_of'} -> BiologicalEntity\n",
      "InformationContentEntity - {'precedes'} -> BiologicalEntity\n",
      "InformationContentEntity - {'close_match'} -> BiologicalEntity\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> BiologicalEntity\n",
      "InformationContentEntity - {'subclass_of'} -> BiologicalEntity\n",
      "InformationContentEntity - {'contraindicated_for'} -> BiologicalEntity\n",
      "InformationContentEntity - {'diagnoses'} -> BiologicalEntity\n",
      "InformationContentEntity - {'treats'} -> BiologicalEntity\n",
      "BiologicalEntity - {'physically_interacts_with'} -> Publication\n",
      "BiologicalEntity - {'treats'} -> Publication\n",
      "BiologicalEntity - {'has_input'} -> Publication\n",
      "BiologicalEntity - {'causes'} -> Publication\n",
      "BiologicalEntity - {'disrupts'} -> Publication\n",
      "BiologicalEntity - {'is_sequence_variant_of'} -> Publication\n",
      "BiologicalEntity - {'has_phenotype'} -> Publication\n",
      "BiologicalEntity - {'biomarker_for'} -> Publication\n",
      "BiologicalEntity - {'associated_with'} -> Publication\n",
      "BiologicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalEntity - {'has_part'} -> Publication\n",
      "BiologicalEntity - {'located_in'} -> Publication\n",
      "BiologicalEntity - {'colocalizes_with'} -> Publication\n",
      "BiologicalEntity - {'interacts_with'} -> Publication\n",
      "BiologicalEntity - {'produces'} -> Publication\n",
      "BiologicalEntity - {'predisposes'} -> Publication\n",
      "BiologicalEntity - {'related_to'} -> Publication\n",
      "BiologicalEntity - {'prevents'} -> Publication\n",
      "BiologicalEntity - {'manifestation_of'} -> Publication\n",
      "BiologicalEntity - {'precedes'} -> Publication\n",
      "BiologicalEntity - {'occurs_in'} -> Publication\n",
      "BiologicalEntity - {'affects'} -> Publication\n",
      "BiologicalEntity - {'close_match'} -> Publication\n",
      "BiologicalEntity - {'exacerbates'} -> Publication\n",
      "BiologicalEntity - {'coexists_with'} -> Publication\n",
      "BiologicalEntity - {'has_participant'} -> Publication\n",
      "BiologicalEntity - {'subclass_of'} -> Publication\n",
      "BiologicalEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->OrganismTaxon->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'affects'} -> OrganismTaxon\n",
      "InformationContentEntity - {'related_to'} -> OrganismTaxon\n",
      "InformationContentEntity - {'subclass_of'} -> OrganismTaxon\n",
      "InformationContentEntity - {'develops_from'} -> OrganismTaxon\n",
      "InformationContentEntity - {'interacts_with'} -> OrganismTaxon\n",
      "InformationContentEntity - {'diagnoses'} -> OrganismTaxon\n",
      "InformationContentEntity - {'treats'} -> OrganismTaxon\n",
      "InformationContentEntity - {'occurs_in'} -> OrganismTaxon\n",
      "InformationContentEntity - {'has_input'} -> OrganismTaxon\n",
      "InformationContentEntity - {'derives_from'} -> OrganismTaxon\n",
      "InformationContentEntity - {'exacerbates'} -> OrganismTaxon\n",
      "InformationContentEntity - {'in_taxon'} -> OrganismTaxon\n",
      "InformationContentEntity - {'causes'} -> OrganismTaxon\n",
      "InformationContentEntity - {'physically_interacts_with'} -> OrganismTaxon\n",
      "InformationContentEntity - {'associated_with'} -> OrganismTaxon\n",
      "InformationContentEntity - {'coexists_with'} -> OrganismTaxon\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> OrganismTaxon\n",
      "InformationContentEntity - {'disrupts'} -> OrganismTaxon\n",
      "InformationContentEntity - {'located_in'} -> OrganismTaxon\n",
      "InformationContentEntity - {'regulates'} -> OrganismTaxon\n",
      "InformationContentEntity - {'produces'} -> OrganismTaxon\n",
      "InformationContentEntity - {'predisposes'} -> OrganismTaxon\n",
      "InformationContentEntity - {'has_part'} -> OrganismTaxon\n",
      "InformationContentEntity - {'prevents'} -> OrganismTaxon\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> OrganismTaxon\n",
      "InformationContentEntity - {'manifestation_of'} -> OrganismTaxon\n",
      "InformationContentEntity - {'precedes'} -> OrganismTaxon\n",
      "InformationContentEntity - {'close_match'} -> OrganismTaxon\n",
      "InformationContentEntity - {'has_participant'} -> OrganismTaxon\n",
      "OrganismTaxon - {'regulates'} -> Publication\n",
      "OrganismTaxon - {'gene_associated_with_condition'} -> Publication\n",
      "OrganismTaxon - {'located_in'} -> Publication\n",
      "OrganismTaxon - {'interacts_with'} -> Publication\n",
      "OrganismTaxon - {'produces'} -> Publication\n",
      "OrganismTaxon - {'related_to'} -> Publication\n",
      "OrganismTaxon - {'has_part'} -> Publication\n",
      "OrganismTaxon - {'predisposes'} -> Publication\n",
      "OrganismTaxon - {'occurs_in'} -> Publication\n",
      "OrganismTaxon - {'affects'} -> Publication\n",
      "OrganismTaxon - {'manifestation_of'} -> Publication\n",
      "OrganismTaxon - {'exacerbates'} -> Publication\n",
      "OrganismTaxon - {'coexists_with'} -> Publication\n",
      "OrganismTaxon - {'close_match'} -> Publication\n",
      "OrganismTaxon - {'subclass_of'} -> Publication\n",
      "OrganismTaxon - {'physically_interacts_with'} -> Publication\n",
      "OrganismTaxon - {'treats'} -> Publication\n",
      "OrganismTaxon - {'has_input'} -> Publication\n",
      "OrganismTaxon - {'causes'} -> Publication\n",
      "OrganismTaxon - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Cell->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'associated_with'} -> Cell\n",
      "InformationContentEntity - {'has_upstream_actor'} -> Cell\n",
      "InformationContentEntity - {'primarily_composed_of'} -> Cell\n",
      "InformationContentEntity - {'affects'} -> Cell\n",
      "InformationContentEntity - {'similar_to'} -> Cell\n",
      "InformationContentEntity - {'related_to'} -> Cell\n",
      "InformationContentEntity - {'overlaps'} -> Cell\n",
      "InformationContentEntity - {'composed_primarily_of'} -> Cell\n",
      "InformationContentEntity - {'affected_by'} -> Cell\n",
      "InformationContentEntity - {'interacts_with'} -> Cell\n",
      "InformationContentEntity - {'participates_in'} -> Cell\n",
      "InformationContentEntity - {'has_participant'} -> Cell\n",
      "InformationContentEntity - {'diagnoses'} -> Cell\n",
      "InformationContentEntity - {'treats'} -> Cell\n",
      "InformationContentEntity - {'has_input'} -> Cell\n",
      "InformationContentEntity - {'derives_from'} -> Cell\n",
      "InformationContentEntity - {'occurs_in'} -> Cell\n",
      "InformationContentEntity - {'causes'} -> Cell\n",
      "InformationContentEntity - {'coexists_with'} -> Cell\n",
      "InformationContentEntity - {'disrupts'} -> Cell\n",
      "InformationContentEntity - {'located_in'} -> Cell\n",
      "InformationContentEntity - {'regulates'} -> Cell\n",
      "InformationContentEntity - {'produces'} -> Cell\n",
      "InformationContentEntity - {'expressed_in'} -> Cell\n",
      "InformationContentEntity - {'has_part'} -> Cell\n",
      "InformationContentEntity - {'predisposes'} -> Cell\n",
      "InformationContentEntity - {'prevents'} -> Cell\n",
      "InformationContentEntity - {'precedes'} -> Cell\n",
      "InformationContentEntity - {'close_match'} -> Cell\n",
      "InformationContentEntity - {'subclass_of'} -> Cell\n",
      "Cell - {'subclass_of'} -> Publication\n",
      "Cell - {'has_part'} -> Publication\n",
      "Cell - {'produces'} -> Publication\n",
      "Cell - {'related_to'} -> Publication\n",
      "Cell - {'occurs_in'} -> Publication\n",
      "Cell - {'affects'} -> Publication\n",
      "Cell - {'coexists_with'} -> Publication\n",
      "Cell - {'close_match'} -> Publication\n",
      "Cell - {'located_in'} -> Publication\n",
      "Cell - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->OrganismAttribute->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'interacts_with'} -> OrganismAttribute\n",
      "InformationContentEntity - {'associated_with'} -> OrganismAttribute\n",
      "InformationContentEntity - {'affects'} -> OrganismAttribute\n",
      "InformationContentEntity - {'related_to'} -> OrganismAttribute\n",
      "InformationContentEntity - {'overlaps'} -> OrganismAttribute\n",
      "InformationContentEntity - {'composed_primarily_of'} -> OrganismAttribute\n",
      "InformationContentEntity - {'has_upstream_actor'} -> OrganismAttribute\n",
      "InformationContentEntity - {'primarily_composed_of'} -> OrganismAttribute\n",
      "InformationContentEntity - {'similar_to'} -> OrganismAttribute\n",
      "InformationContentEntity - {'affected_by'} -> OrganismAttribute\n",
      "InformationContentEntity - {'participates_in'} -> OrganismAttribute\n",
      "InformationContentEntity - {'has_participant'} -> OrganismAttribute\n",
      "InformationContentEntity - {'close_match'} -> OrganismAttribute\n",
      "InformationContentEntity - {'subclass_of'} -> OrganismAttribute\n",
      "InformationContentEntity - {'treats'} -> OrganismAttribute\n",
      "InformationContentEntity - {'occurs_in'} -> OrganismAttribute\n",
      "InformationContentEntity - {'predisposes'} -> OrganismAttribute\n",
      "InformationContentEntity - {'located_in'} -> OrganismAttribute\n",
      "InformationContentEntity - {'coexists_with'} -> OrganismAttribute\n",
      "OrganismAttribute - {'related_to'} -> Publication\n",
      "OrganismAttribute - {'produces'} -> Publication\n",
      "OrganismAttribute - {'has_input'} -> Publication\n",
      "OrganismAttribute - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Phenomenon->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'related_to'} -> Phenomenon\n",
      "InformationContentEntity - {'affects'} -> Phenomenon\n",
      "InformationContentEntity - {'manifestation_of'} -> Phenomenon\n",
      "InformationContentEntity - {'precedes'} -> Phenomenon\n",
      "InformationContentEntity - {'close_match'} -> Phenomenon\n",
      "InformationContentEntity - {'subclass_of'} -> Phenomenon\n",
      "InformationContentEntity - {'interacts_with'} -> Phenomenon\n",
      "InformationContentEntity - {'contraindicated_for'} -> Phenomenon\n",
      "InformationContentEntity - {'diagnoses'} -> Phenomenon\n",
      "InformationContentEntity - {'treats'} -> Phenomenon\n",
      "InformationContentEntity - {'occurs_in'} -> Phenomenon\n",
      "InformationContentEntity - {'has_input'} -> Phenomenon\n",
      "InformationContentEntity - {'derives_from'} -> Phenomenon\n",
      "InformationContentEntity - {'exacerbates'} -> Phenomenon\n",
      "InformationContentEntity - {'causes'} -> Phenomenon\n",
      "InformationContentEntity - {'associated_with'} -> Phenomenon\n",
      "InformationContentEntity - {'coexists_with'} -> Phenomenon\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Phenomenon\n",
      "InformationContentEntity - {'has_phenotype'} -> Phenomenon\n",
      "InformationContentEntity - {'disrupts'} -> Phenomenon\n",
      "InformationContentEntity - {'located_in'} -> Phenomenon\n",
      "InformationContentEntity - {'regulates'} -> Phenomenon\n",
      "InformationContentEntity - {'biomarker_for'} -> Phenomenon\n",
      "InformationContentEntity - {'produces'} -> Phenomenon\n",
      "InformationContentEntity - {'predisposes'} -> Phenomenon\n",
      "InformationContentEntity - {'has_part'} -> Phenomenon\n",
      "InformationContentEntity - {'prevents'} -> Phenomenon\n",
      "Phenomenon - {'associated_with'} -> Publication\n",
      "Phenomenon - {'subclass_of'} -> Publication\n",
      "Phenomenon - {'interacts_with'} -> Publication\n",
      "Phenomenon - {'produces'} -> Publication\n",
      "Phenomenon - {'related_to'} -> Publication\n",
      "Phenomenon - {'has_part'} -> Publication\n",
      "Phenomenon - {'diagnoses'} -> Publication\n",
      "Phenomenon - {'occurs_in'} -> Publication\n",
      "Phenomenon - {'affects'} -> Publication\n",
      "Phenomenon - {'coexists_with'} -> Publication\n",
      "Phenomenon - {'close_match'} -> Publication\n",
      "Phenomenon - {'disrupts'} -> Publication\n",
      "Phenomenon - {'regulates'} -> Publication\n",
      "Phenomenon - {'located_in'} -> Publication\n",
      "Phenomenon - {'has_input'} -> Publication\n",
      "Phenomenon - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->BehavioralFeature->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'predisposes'} -> BehavioralFeature\n",
      "InformationContentEntity - {'related_to'} -> BehavioralFeature\n",
      "InformationContentEntity - {'prevents'} -> BehavioralFeature\n",
      "InformationContentEntity - {'affects'} -> BehavioralFeature\n",
      "InformationContentEntity - {'manifestation_of'} -> BehavioralFeature\n",
      "InformationContentEntity - {'precedes'} -> BehavioralFeature\n",
      "InformationContentEntity - {'close_match'} -> BehavioralFeature\n",
      "InformationContentEntity - {'subclass_of'} -> BehavioralFeature\n",
      "InformationContentEntity - {'interacts_with'} -> BehavioralFeature\n",
      "InformationContentEntity - {'diagnoses'} -> BehavioralFeature\n",
      "InformationContentEntity - {'treats'} -> BehavioralFeature\n",
      "InformationContentEntity - {'exacerbates'} -> BehavioralFeature\n",
      "InformationContentEntity - {'causes'} -> BehavioralFeature\n",
      "InformationContentEntity - {'coexists_with'} -> BehavioralFeature\n",
      "InformationContentEntity - {'disrupts'} -> BehavioralFeature\n",
      "InformationContentEntity - {'associated_with'} -> BehavioralFeature\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> BehavioralFeature\n",
      "InformationContentEntity - {'regulates'} -> BehavioralFeature\n",
      "BehavioralFeature - {'subclass_of'} -> Publication\n",
      "BehavioralFeature - {'produces'} -> Publication\n",
      "BehavioralFeature - {'related_to'} -> Publication\n",
      "BehavioralFeature - {'occurs_in'} -> Publication\n",
      "BehavioralFeature - {'affects'} -> Publication\n",
      "BehavioralFeature - {'manifestation_of'} -> Publication\n",
      "BehavioralFeature - {'coexists_with'} -> Publication\n",
      "BehavioralFeature - {'disrupts'} -> Publication\n",
      "BehavioralFeature - {'located_in'} -> Publication\n",
      "BehavioralFeature - {'predisposes'} -> Publication\n",
      "BehavioralFeature - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Agent->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'interacts_with'} -> Agent\n",
      "InformationContentEntity - {'diagnoses'} -> Agent\n",
      "InformationContentEntity - {'treats'} -> Agent\n",
      "InformationContentEntity - {'occurs_in'} -> Agent\n",
      "InformationContentEntity - {'has_input'} -> Agent\n",
      "InformationContentEntity - {'derives_from'} -> Agent\n",
      "InformationContentEntity - {'exacerbates'} -> Agent\n",
      "InformationContentEntity - {'causes'} -> Agent\n",
      "InformationContentEntity - {'associated_with'} -> Agent\n",
      "InformationContentEntity - {'coexists_with'} -> Agent\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Agent\n",
      "InformationContentEntity - {'disrupts'} -> Agent\n",
      "InformationContentEntity - {'located_in'} -> Agent\n",
      "InformationContentEntity - {'regulates'} -> Agent\n",
      "InformationContentEntity - {'produces'} -> Agent\n",
      "InformationContentEntity - {'predisposes'} -> Agent\n",
      "InformationContentEntity - {'related_to'} -> Agent\n",
      "InformationContentEntity - {'has_part'} -> Agent\n",
      "InformationContentEntity - {'prevents'} -> Agent\n",
      "InformationContentEntity - {'affects'} -> Agent\n",
      "InformationContentEntity - {'manifestation_of'} -> Agent\n",
      "InformationContentEntity - {'precedes'} -> Agent\n",
      "InformationContentEntity - {'close_match'} -> Agent\n",
      "InformationContentEntity - {'subclass_of'} -> Agent\n",
      "Agent - {'has_part'} -> Publication\n",
      "Agent - {'interacts_with'} -> Publication\n",
      "Agent - {'produces'} -> Publication\n",
      "Agent - {'predisposes'} -> Publication\n",
      "Agent - {'related_to'} -> Publication\n",
      "Agent - {'prevents'} -> Publication\n",
      "Agent - {'precedes'} -> Publication\n",
      "Agent - {'occurs_in'} -> Publication\n",
      "Agent - {'affects'} -> Publication\n",
      "Agent - {'exacerbates'} -> Publication\n",
      "Agent - {'coexists_with'} -> Publication\n",
      "Agent - {'close_match'} -> Publication\n",
      "Agent - {'subclass_of'} -> Publication\n",
      "Agent - {'regulates'} -> Publication\n",
      "Agent - {'diagnoses'} -> Publication\n",
      "Agent - {'treats'} -> Publication\n",
      "Agent - {'has_input'} -> Publication\n",
      "Agent - {'causes'} -> Publication\n",
      "Agent - {'located_in'} -> Publication\n",
      "Agent - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->MaterialSample->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'affects'} -> MaterialSample\n",
      "InformationContentEntity - {'close_match'} -> MaterialSample\n",
      "InformationContentEntity - {'subclass_of'} -> MaterialSample\n",
      "InformationContentEntity - {'interacts_with'} -> MaterialSample\n",
      "InformationContentEntity - {'diagnoses'} -> MaterialSample\n",
      "InformationContentEntity - {'treats'} -> MaterialSample\n",
      "InformationContentEntity - {'occurs_in'} -> MaterialSample\n",
      "InformationContentEntity - {'has_input'} -> MaterialSample\n",
      "InformationContentEntity - {'derives_from'} -> MaterialSample\n",
      "InformationContentEntity - {'causes'} -> MaterialSample\n",
      "InformationContentEntity - {'associated_with'} -> MaterialSample\n",
      "InformationContentEntity - {'coexists_with'} -> MaterialSample\n",
      "InformationContentEntity - {'disrupts'} -> MaterialSample\n",
      "InformationContentEntity - {'located_in'} -> MaterialSample\n",
      "InformationContentEntity - {'regulates'} -> MaterialSample\n",
      "InformationContentEntity - {'produces'} -> MaterialSample\n",
      "InformationContentEntity - {'predisposes'} -> MaterialSample\n",
      "InformationContentEntity - {'related_to'} -> MaterialSample\n",
      "InformationContentEntity - {'has_part'} -> MaterialSample\n",
      "MaterialSample - {'located_in'} -> Publication\n",
      "MaterialSample - {'associated_with'} -> Publication\n",
      "MaterialSample - {'subclass_of'} -> Publication\n",
      "MaterialSample - {'has_part'} -> Publication\n",
      "MaterialSample - {'related_to'} -> Publication\n",
      "MaterialSample - {'occurs_in'} -> Publication\n",
      "MaterialSample - {'affects'} -> Publication\n",
      "MaterialSample - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->MicroRNA->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'interacts_with'} -> MicroRNA\n",
      "InformationContentEntity - {'has_input'} -> MicroRNA\n",
      "InformationContentEntity - {'derives_from'} -> MicroRNA\n",
      "InformationContentEntity - {'occurs_in'} -> MicroRNA\n",
      "InformationContentEntity - {'coexists_with'} -> MicroRNA\n",
      "InformationContentEntity - {'associated_with'} -> MicroRNA\n",
      "InformationContentEntity - {'located_in'} -> MicroRNA\n",
      "InformationContentEntity - {'regulates'} -> MicroRNA\n",
      "InformationContentEntity - {'produces'} -> MicroRNA\n",
      "InformationContentEntity - {'has_part'} -> MicroRNA\n",
      "InformationContentEntity - {'related_to'} -> MicroRNA\n",
      "InformationContentEntity - {'affects'} -> MicroRNA\n",
      "InformationContentEntity - {'has_participant'} -> MicroRNA\n",
      "MicroRNA - {'causes'} -> Publication\n",
      "MicroRNA - {'disrupts'} -> Publication\n",
      "MicroRNA - {'associated_with'} -> Publication\n",
      "MicroRNA - {'gene_associated_with_condition'} -> Publication\n",
      "MicroRNA - {'subclass_of'} -> Publication\n",
      "MicroRNA - {'predisposes'} -> Publication\n",
      "MicroRNA - {'located_in'} -> Publication\n",
      "MicroRNA - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->PhysicalEntity->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'causes'} -> PhysicalEntity\n",
      "InformationContentEntity - {'coexists_with'} -> PhysicalEntity\n",
      "InformationContentEntity - {'disrupts'} -> PhysicalEntity\n",
      "InformationContentEntity - {'associated_with'} -> PhysicalEntity\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> PhysicalEntity\n",
      "InformationContentEntity - {'regulates'} -> PhysicalEntity\n",
      "InformationContentEntity - {'produces'} -> PhysicalEntity\n",
      "InformationContentEntity - {'has_part'} -> PhysicalEntity\n",
      "InformationContentEntity - {'located_in'} -> PhysicalEntity\n",
      "InformationContentEntity - {'predisposes'} -> PhysicalEntity\n",
      "InformationContentEntity - {'related_to'} -> PhysicalEntity\n",
      "InformationContentEntity - {'prevents'} -> PhysicalEntity\n",
      "InformationContentEntity - {'affects'} -> PhysicalEntity\n",
      "InformationContentEntity - {'manifestation_of'} -> PhysicalEntity\n",
      "InformationContentEntity - {'precedes'} -> PhysicalEntity\n",
      "InformationContentEntity - {'close_match'} -> PhysicalEntity\n",
      "InformationContentEntity - {'interacts_with'} -> PhysicalEntity\n",
      "InformationContentEntity - {'has_participant'} -> PhysicalEntity\n",
      "InformationContentEntity - {'subclass_of'} -> PhysicalEntity\n",
      "InformationContentEntity - {'diagnoses'} -> PhysicalEntity\n",
      "InformationContentEntity - {'treats'} -> PhysicalEntity\n",
      "InformationContentEntity - {'has_input'} -> PhysicalEntity\n",
      "InformationContentEntity - {'occurs_in'} -> PhysicalEntity\n",
      "InformationContentEntity - {'derives_from'} -> PhysicalEntity\n",
      "InformationContentEntity - {'exacerbates'} -> PhysicalEntity\n",
      "PhysicalEntity - {'subclass_of'} -> Publication\n",
      "PhysicalEntity - {'has_part'} -> Publication\n",
      "PhysicalEntity - {'interacts_with'} -> Publication\n",
      "PhysicalEntity - {'produces'} -> Publication\n",
      "PhysicalEntity - {'predisposes'} -> Publication\n",
      "PhysicalEntity - {'related_to'} -> Publication\n",
      "PhysicalEntity - {'occurs_in'} -> Publication\n",
      "PhysicalEntity - {'affects'} -> Publication\n",
      "PhysicalEntity - {'close_match'} -> Publication\n",
      "PhysicalEntity - {'coexists_with'} -> Publication\n",
      "PhysicalEntity - {'regulates'} -> Publication\n",
      "PhysicalEntity - {'located_in'} -> Publication\n",
      "PhysicalEntity - {'treats'} -> Publication\n",
      "PhysicalEntity - {'has_input'} -> Publication\n",
      "PhysicalEntity - {'causes'} -> Publication\n",
      "PhysicalEntity - {'disrupts'} -> Publication\n",
      "PhysicalEntity - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Vitamin->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'physically_interacts_with'} -> Vitamin\n",
      "InformationContentEntity - {'has_input'} -> Vitamin\n",
      "InformationContentEntity - {'derives_from'} -> Vitamin\n",
      "InformationContentEntity - {'coexists_with'} -> Vitamin\n",
      "InformationContentEntity - {'regulates'} -> Vitamin\n",
      "InformationContentEntity - {'produces'} -> Vitamin\n",
      "InformationContentEntity - {'has_part'} -> Vitamin\n",
      "InformationContentEntity - {'related_to'} -> Vitamin\n",
      "InformationContentEntity - {'precedes'} -> Vitamin\n",
      "InformationContentEntity - {'close_match'} -> Vitamin\n",
      "InformationContentEntity - {'subclass_of'} -> Vitamin\n",
      "InformationContentEntity - {'interacts_with'} -> Vitamin\n",
      "Vitamin - {'regulates'} -> Publication\n",
      "Vitamin - {'treats'} -> Publication\n",
      "Vitamin - {'disrupts'} -> Publication\n",
      "Vitamin - {'subclass_of'} -> Publication\n",
      "Vitamin - {'related_to'} -> Publication\n",
      "Vitamin - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ProteinFamily->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'produces'} -> ProteinFamily\n",
      "InformationContentEntity - {'has_part'} -> ProteinFamily\n",
      "InformationContentEntity - {'located_in'} -> ProteinFamily\n",
      "InformationContentEntity - {'related_to'} -> ProteinFamily\n",
      "InformationContentEntity - {'affects'} -> ProteinFamily\n",
      "InformationContentEntity - {'close_match'} -> ProteinFamily\n",
      "InformationContentEntity - {'has_participant'} -> ProteinFamily\n",
      "InformationContentEntity - {'subclass_of'} -> ProteinFamily\n",
      "InformationContentEntity - {'interacts_with'} -> ProteinFamily\n",
      "InformationContentEntity - {'physically_interacts_with'} -> ProteinFamily\n",
      "InformationContentEntity - {'occurs_in'} -> ProteinFamily\n",
      "InformationContentEntity - {'has_input'} -> ProteinFamily\n",
      "InformationContentEntity - {'derives_from'} -> ProteinFamily\n",
      "InformationContentEntity - {'coexists_with'} -> ProteinFamily\n",
      "InformationContentEntity - {'associated_with'} -> ProteinFamily\n",
      "InformationContentEntity - {'capable_of'} -> ProteinFamily\n",
      "InformationContentEntity - {'regulates'} -> ProteinFamily\n",
      "ProteinFamily - {'associated_with'} -> Publication\n",
      "ProteinFamily - {'subclass_of'} -> Publication\n",
      "ProteinFamily - {'gene_associated_with_condition'} -> Publication\n",
      "ProteinFamily - {'predisposes'} -> Publication\n",
      "ProteinFamily - {'related_to'} -> Publication\n",
      "ProteinFamily - {'affects'} -> Publication\n",
      "ProteinFamily - {'disrupts'} -> Publication\n",
      "ProteinFamily - {'located_in'} -> Publication\n",
      "ProteinFamily - {'regulates'} -> Publication\n",
      "ProteinFamily - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Event->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'subclass_of'} -> Event\n",
      "InformationContentEntity - {'treats'} -> Event\n",
      "InformationContentEntity - {'causes'} -> Event\n",
      "InformationContentEntity - {'coexists_with'} -> Event\n",
      "InformationContentEntity - {'disrupts'} -> Event\n",
      "InformationContentEntity - {'associated_with'} -> Event\n",
      "InformationContentEntity - {'related_to'} -> Event\n",
      "InformationContentEntity - {'close_match'} -> Event\n",
      "Event - {'occurs_in'} -> Publication\n",
      "Event - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->RNAProduct->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'coexists_with'} -> RNAProduct\n",
      "InformationContentEntity - {'regulates'} -> RNAProduct\n",
      "InformationContentEntity - {'produces'} -> RNAProduct\n",
      "InformationContentEntity - {'has_part'} -> RNAProduct\n",
      "InformationContentEntity - {'located_in'} -> RNAProduct\n",
      "InformationContentEntity - {'related_to'} -> RNAProduct\n",
      "InformationContentEntity - {'affects'} -> RNAProduct\n",
      "InformationContentEntity - {'close_match'} -> RNAProduct\n",
      "InformationContentEntity - {'interacts_with'} -> RNAProduct\n",
      "InformationContentEntity - {'subclass_of'} -> RNAProduct\n",
      "InformationContentEntity - {'has_input'} -> RNAProduct\n",
      "InformationContentEntity - {'derives_from'} -> RNAProduct\n",
      "InformationContentEntity - {'occurs_in'} -> RNAProduct\n",
      "RNAProduct - {'located_in'} -> Publication\n",
      "RNAProduct - {'regulates'} -> Publication\n",
      "RNAProduct - {'related_to'} -> Publication\n",
      "RNAProduct - {'affects'} -> Publication\n",
      "RNAProduct - {'subclass_of'} -> Publication\n",
      "RNAProduct - {'disrupts'} -> Publication\n",
      "RNAProduct - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->GeographicLocation->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'related_to'} -> GeographicLocation\n",
      "InformationContentEntity - {'affects'} -> GeographicLocation\n",
      "InformationContentEntity - {'close_match'} -> GeographicLocation\n",
      "InformationContentEntity - {'has_participant'} -> GeographicLocation\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> GeographicLocation\n",
      "InformationContentEntity - {'subclass_of'} -> GeographicLocation\n",
      "InformationContentEntity - {'interacts_with'} -> GeographicLocation\n",
      "InformationContentEntity - {'has_input'} -> GeographicLocation\n",
      "InformationContentEntity - {'physically_interacts_with'} -> GeographicLocation\n",
      "InformationContentEntity - {'occurs_in'} -> GeographicLocation\n",
      "InformationContentEntity - {'associated_with'} -> GeographicLocation\n",
      "InformationContentEntity - {'coexists_with'} -> GeographicLocation\n",
      "InformationContentEntity - {'disrupts'} -> GeographicLocation\n",
      "InformationContentEntity - {'located_in'} -> GeographicLocation\n",
      "InformationContentEntity - {'regulates'} -> GeographicLocation\n",
      "InformationContentEntity - {'produces'} -> GeographicLocation\n",
      "InformationContentEntity - {'has_part'} -> GeographicLocation\n",
      "GeographicLocation - {'subclass_of'} -> Publication\n",
      "GeographicLocation - {'related_to'} -> Publication\n",
      "GeographicLocation - {'gene_associated_with_condition'} -> Publication\n",
      "GeographicLocation - {'located_in'} -> Publication\n",
      "GeographicLocation - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->LifeStage->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'occurs_in'} -> LifeStage\n",
      "InformationContentEntity - {'located_in'} -> LifeStage\n",
      "InformationContentEntity - {'produces'} -> LifeStage\n",
      "InformationContentEntity - {'related_to'} -> LifeStage\n",
      "InformationContentEntity - {'affects'} -> LifeStage\n",
      "InformationContentEntity - {'close_match'} -> LifeStage\n",
      "InformationContentEntity - {'subclass_of'} -> LifeStage\n",
      "InformationContentEntity - {'interacts_with'} -> LifeStage\n",
      "InformationContentEntity - {'treats'} -> LifeStage\n",
      "LifeStage - {'subclass_of'} -> Publication\n",
      "LifeStage - {'has_part'} -> Publication\n",
      "LifeStage - {'interacts_with'} -> Publication\n",
      "LifeStage - {'produces'} -> Publication\n",
      "LifeStage - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->GeneFamily->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'interacts_with'} -> GeneFamily\n",
      "InformationContentEntity - {'has_participant'} -> GeneFamily\n",
      "InformationContentEntity - {'subclass_of'} -> GeneFamily\n",
      "InformationContentEntity - {'occurs_in'} -> GeneFamily\n",
      "InformationContentEntity - {'has_input'} -> GeneFamily\n",
      "InformationContentEntity - {'derives_from'} -> GeneFamily\n",
      "InformationContentEntity - {'coexists_with'} -> GeneFamily\n",
      "InformationContentEntity - {'associated_with'} -> GeneFamily\n",
      "InformationContentEntity - {'regulates'} -> GeneFamily\n",
      "InformationContentEntity - {'produces'} -> GeneFamily\n",
      "InformationContentEntity - {'has_part'} -> GeneFamily\n",
      "InformationContentEntity - {'located_in'} -> GeneFamily\n",
      "InformationContentEntity - {'related_to'} -> GeneFamily\n",
      "InformationContentEntity - {'affects'} -> GeneFamily\n",
      "InformationContentEntity - {'precedes'} -> GeneFamily\n",
      "InformationContentEntity - {'close_match'} -> GeneFamily\n",
      "InformationContentEntity - {'overlaps'} -> GeneFamily\n",
      "InformationContentEntity - {'directly_physically_interacts_with'} -> GeneFamily\n",
      "GeneFamily - {'treats'} -> Publication\n",
      "GeneFamily - {'causes'} -> Publication\n",
      "GeneFamily - {'associated_with'} -> Publication\n",
      "GeneFamily - {'disrupts'} -> Publication\n",
      "GeneFamily - {'gene_associated_with_condition'} -> Publication\n",
      "GeneFamily - {'located_in'} -> Publication\n",
      "GeneFamily - {'regulates'} -> Publication\n",
      "GeneFamily - {'predisposes'} -> Publication\n",
      "GeneFamily - {'occurs_in'} -> Publication\n",
      "GeneFamily - {'affects'} -> Publication\n",
      "GeneFamily - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->IndividualOrganism->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'located_in'} -> IndividualOrganism\n",
      "InformationContentEntity - {'regulates'} -> IndividualOrganism\n",
      "InformationContentEntity - {'related_to'} -> IndividualOrganism\n",
      "InformationContentEntity - {'has_part'} -> IndividualOrganism\n",
      "InformationContentEntity - {'affects'} -> IndividualOrganism\n",
      "InformationContentEntity - {'close_match'} -> IndividualOrganism\n",
      "InformationContentEntity - {'subclass_of'} -> IndividualOrganism\n",
      "InformationContentEntity - {'interacts_with'} -> IndividualOrganism\n",
      "InformationContentEntity - {'treats'} -> IndividualOrganism\n",
      "InformationContentEntity - {'occurs_in'} -> IndividualOrganism\n",
      "InformationContentEntity - {'has_input'} -> IndividualOrganism\n",
      "InformationContentEntity - {'in_taxon'} -> IndividualOrganism\n",
      "InformationContentEntity - {'associated_with'} -> IndividualOrganism\n",
      "InformationContentEntity - {'disrupts'} -> IndividualOrganism\n",
      "IndividualOrganism - {'close_match'} -> Publication\n",
      "IndividualOrganism - {'subclass_of'} -> Publication\n",
      "IndividualOrganism - {'occurs_in'} -> Publication\n",
      "IndividualOrganism - {'located_in'} -> Publication\n",
      "IndividualOrganism - {'related_to'} -> Publication\n",
      "IndividualOrganism - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Treatment->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'treats'} -> Treatment\n",
      "InformationContentEntity - {'has_input'} -> Treatment\n",
      "InformationContentEntity - {'causes'} -> Treatment\n",
      "InformationContentEntity - {'coexists_with'} -> Treatment\n",
      "InformationContentEntity - {'related_to'} -> Treatment\n",
      "InformationContentEntity - {'precedes'} -> Treatment\n",
      "InformationContentEntity - {'subclass_of'} -> Treatment\n",
      "Treatment - {'subclass_of'} -> Publication\n",
      "Treatment - {'occurs_in'} -> Publication\n",
      "Treatment - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Food->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'related_to'} -> Food\n",
      "InformationContentEntity - {'affects'} -> Food\n",
      "InformationContentEntity - {'precedes'} -> Food\n",
      "InformationContentEntity - {'close_match'} -> Food\n",
      "InformationContentEntity - {'subclass_of'} -> Food\n",
      "InformationContentEntity - {'interacts_with'} -> Food\n",
      "InformationContentEntity - {'diagnoses'} -> Food\n",
      "InformationContentEntity - {'treats'} -> Food\n",
      "InformationContentEntity - {'has_input'} -> Food\n",
      "InformationContentEntity - {'derives_from'} -> Food\n",
      "InformationContentEntity - {'has_output'} -> Food\n",
      "InformationContentEntity - {'occurs_in'} -> Food\n",
      "InformationContentEntity - {'causes'} -> Food\n",
      "InformationContentEntity - {'coexists_with'} -> Food\n",
      "InformationContentEntity - {'disrupts'} -> Food\n",
      "InformationContentEntity - {'associated_with'} -> Food\n",
      "InformationContentEntity - {'located_in'} -> Food\n",
      "InformationContentEntity - {'regulates'} -> Food\n",
      "InformationContentEntity - {'produces'} -> Food\n",
      "InformationContentEntity - {'has_part'} -> Food\n",
      "Food - {'regulates'} -> Publication\n",
      "Food - {'associated_with'} -> Publication\n",
      "Food - {'located_in'} -> Publication\n",
      "Food - {'predisposes'} -> Publication\n",
      "Food - {'related_to'} -> Publication\n",
      "Food - {'occurs_in'} -> Publication\n",
      "Food - {'affects'} -> Publication\n",
      "Food - {'precedes'} -> Publication\n",
      "Food - {'subclass_of'} -> Publication\n",
      "Food - {'treats'} -> Publication\n",
      "Food - {'causes'} -> Publication\n",
      "Food - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->NoncodingRNAProduct->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'regulates'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'produces'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'has_part'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'located_in'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'related_to'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'affects'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'close_match'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'has_participant'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'subclass_of'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'interacts_with'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'derives_from'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'occurs_in'} -> NoncodingRNAProduct\n",
      "InformationContentEntity - {'coexists_with'} -> NoncodingRNAProduct\n",
      "NoncodingRNAProduct - {'affects'} -> Publication\n",
      "NoncodingRNAProduct - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->ProteinDomain->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'occurs_in'} -> ProteinDomain\n",
      "InformationContentEntity - {'has_input'} -> ProteinDomain\n",
      "InformationContentEntity - {'derives_from'} -> ProteinDomain\n",
      "InformationContentEntity - {'coexists_with'} -> ProteinDomain\n",
      "InformationContentEntity - {'disrupts'} -> ProteinDomain\n",
      "InformationContentEntity - {'associated_with'} -> ProteinDomain\n",
      "InformationContentEntity - {'regulates'} -> ProteinDomain\n",
      "InformationContentEntity - {'produces'} -> ProteinDomain\n",
      "InformationContentEntity - {'located_in'} -> ProteinDomain\n",
      "InformationContentEntity - {'related_to'} -> ProteinDomain\n",
      "InformationContentEntity - {'has_part'} -> ProteinDomain\n",
      "InformationContentEntity - {'affects'} -> ProteinDomain\n",
      "InformationContentEntity - {'close_match'} -> ProteinDomain\n",
      "InformationContentEntity - {'subclass_of'} -> ProteinDomain\n",
      "InformationContentEntity - {'interacts_with'} -> ProteinDomain\n",
      "ProteinDomain - {'associated_with'} -> Publication\n",
      "ProteinDomain - {'located_in'} -> Publication\n",
      "ProteinDomain - {'related_to'} -> Publication\n",
      "ProteinDomain - {'occurs_in'} -> Publication\n",
      "ProteinDomain - {'affects'} -> Publication\n",
      "ProteinDomain - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Transcript->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'occurs_in'} -> Transcript\n",
      "InformationContentEntity - {'has_input'} -> Transcript\n",
      "InformationContentEntity - {'derives_from'} -> Transcript\n",
      "InformationContentEntity - {'coexists_with'} -> Transcript\n",
      "InformationContentEntity - {'regulates'} -> Transcript\n",
      "InformationContentEntity - {'produces'} -> Transcript\n",
      "InformationContentEntity - {'related_to'} -> Transcript\n",
      "InformationContentEntity - {'has_part'} -> Transcript\n",
      "InformationContentEntity - {'affects'} -> Transcript\n",
      "InformationContentEntity - {'close_match'} -> Transcript\n",
      "InformationContentEntity - {'has_participant'} -> Transcript\n",
      "InformationContentEntity - {'interacts_with'} -> Transcript\n",
      "Transcript - {'translates_to'} -> Publication\n",
      "Transcript - {'regulates'} -> Publication\n",
      "Transcript - {'transcribed_from'} -> Publication\n",
      "Transcript - {'associated_with'} -> Publication\n",
      "Transcript - {'located_in'} -> Publication\n",
      "Transcript - {'subclass_of'} -> Publication\n",
      "Transcript - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Cohort->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'close_match'} -> Cohort\n",
      "InformationContentEntity - {'subclass_of'} -> Cohort\n",
      "InformationContentEntity - {'interacts_with'} -> Cohort\n",
      "InformationContentEntity - {'treats'} -> Cohort\n",
      "InformationContentEntity - {'occurs_in'} -> Cohort\n",
      "InformationContentEntity - {'has_input'} -> Cohort\n",
      "InformationContentEntity - {'coexists_with'} -> Cohort\n",
      "InformationContentEntity - {'associated_with'} -> Cohort\n",
      "InformationContentEntity - {'produces'} -> Cohort\n",
      "InformationContentEntity - {'located_in'} -> Cohort\n",
      "InformationContentEntity - {'predisposes'} -> Cohort\n",
      "InformationContentEntity - {'related_to'} -> Cohort\n",
      "InformationContentEntity - {'affects'} -> Cohort\n",
      "Cohort - {'diagnoses'} -> Publication\n",
      "Cohort - {'related_to'} -> Publication\n",
      "Cohort - {'treats'} -> Publication\n",
      "Cohort - {'has_input'} -> Publication\n",
      "Cohort - {'occurs_in'} -> Publication\n",
      "Cohort - {'has_part'} -> Publication\n",
      "Cohort - {'close_match'} -> Publication\n",
      "Cohort - {'subclass_of'} -> Publication\n",
      "Cohort - {'interacts_with'} -> Publication\n",
      "Cohort - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->Exon->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'has_part'} -> Exon\n",
      "Exon - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->InformationContentEntity->GeneticInheritance->Publication\n",
      "InformationResource - {'subclass_of'} -> InformationContentEntity\n",
      "InformationResource - {'related_to'} -> InformationContentEntity\n",
      "InformationContentEntity - {'manifestation_of'} -> GeneticInheritance\n",
      "InformationContentEntity - {'subclass_of'} -> GeneticInheritance\n",
      "InformationContentEntity - {'related_to'} -> GeneticInheritance\n",
      "InformationContentEntity - {'precedes'} -> GeneticInheritance\n",
      "InformationContentEntity - {'has_input'} -> GeneticInheritance\n",
      "InformationContentEntity - {'affects'} -> GeneticInheritance\n",
      "InformationContentEntity - {'associated_with'} -> GeneticInheritance\n",
      "InformationContentEntity - {'coexists_with'} -> GeneticInheritance\n",
      "GeneticInheritance - {'occurs_in'} -> Publication\n",
      "GeneticInheritance - {'has_input'} -> Publication\n",
      "GeneticInheritance - {'located_in'} -> Publication\n",
      "GeneticInheritance - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Disease->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> Disease\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> Disease\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> Disease\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> Disease\n",
      "PhenotypicFeature - {'superclass_of'} -> Disease\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> Disease\n",
      "PhenotypicFeature - {'overlaps'} -> Disease\n",
      "PhenotypicFeature - {'part_of'} -> Disease\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> Disease\n",
      "PhenotypicFeature - {'caused_by'} -> Disease\n",
      "PhenotypicFeature - {'causes'} -> Disease\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> Disease\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> Disease\n",
      "PhenotypicFeature - {'related_to'} -> Disease\n",
      "PhenotypicFeature - {'phenotype_of'} -> Disease\n",
      "PhenotypicFeature - {'mode_of_inheritance_of'} -> Disease\n",
      "PhenotypicFeature - {'associated_with'} -> Disease\n",
      "PhenotypicFeature - {'correlated_with'} -> Disease\n",
      "PhenotypicFeature - {'manifestation_of'} -> Disease\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> Disease\n",
      "PhenotypicFeature - {'diagnoses'} -> Disease\n",
      "PhenotypicFeature - {'coexists_with'} -> Disease\n",
      "PhenotypicFeature - {'predisposes'} -> Disease\n",
      "PhenotypicFeature - {'affected_by'} -> Disease\n",
      "PhenotypicFeature - {'has_real_world_evidence_of_association_with'} -> Disease\n",
      "PhenotypicFeature - {'subclass_of'} -> Disease\n",
      "PhenotypicFeature - {'has_phenotype'} -> Disease\n",
      "PhenotypicFeature - {'has_part'} -> Disease\n",
      "PhenotypicFeature - {'affects'} -> Disease\n",
      "PhenotypicFeature - {'ameliorates'} -> Disease\n",
      "PhenotypicFeature - {'affects_response_to'} -> Disease\n",
      "PhenotypicFeature - {'treats'} -> Disease\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Disease\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Disease\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> Disease\n",
      "PhenotypicFeature - {'response_decreased_by'} -> Disease\n",
      "PhenotypicFeature - {'disrupts'} -> Disease\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> Disease\n",
      "PhenotypicFeature - {'interacts_with'} -> Disease\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> Disease\n",
      "PhenotypicFeature - {'regulates'} -> Disease\n",
      "PhenotypicFeature - {'has_predisposing_factor'} -> Disease\n",
      "PhenotypicFeature - {'response_increased_by'} -> Disease\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> Disease\n",
      "PhenotypicFeature - {'has_biomarker'} -> Disease\n",
      "PhenotypicFeature - {'participates_in'} -> Disease\n",
      "PhenotypicFeature - {'contributes_to'} -> Disease\n",
      "PhenotypicFeature - {'has_participant'} -> Disease\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Disease\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> Disease\n",
      "PhenotypicFeature - {'precedes'} -> Disease\n",
      "PhenotypicFeature - {'biomarker_for'} -> Disease\n",
      "PhenotypicFeature - {'genetic_association'} -> Disease\n",
      "PhenotypicFeature - {'disrupted_by'} -> Disease\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Disease\n",
      "PhenotypicFeature - {'treated_by'} -> Disease\n",
      "PhenotypicFeature - {'exacerbates'} -> Disease\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Disease\n",
      "PhenotypicFeature - {'increases_response_to'} -> Disease\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Disease\n",
      "PhenotypicFeature - {'regulated_by'} -> Disease\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Disease\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> Disease\n",
      "PhenotypicFeature - {'response_affected_by'} -> Disease\n",
      "PhenotypicFeature - {'coexpressed_with'} -> Disease\n",
      "PhenotypicFeature - {'similar_to'} -> Disease\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> Disease\n",
      "PhenotypicFeature - {'close_match'} -> Disease\n",
      "PhenotypicFeature - {'contraindicated_for'} -> Disease\n",
      "PhenotypicFeature - {'has_input'} -> Disease\n",
      "PhenotypicFeature - {'derives_from'} -> Disease\n",
      "PhenotypicFeature - {'disease_has_location'} -> Disease\n",
      "PhenotypicFeature - {'gene_product_of'} -> Disease\n",
      "PhenotypicFeature - {'actively_involved_in'} -> Disease\n",
      "PhenotypicFeature - {'located_in'} -> Disease\n",
      "PhenotypicFeature - {'produces'} -> Disease\n",
      "PhenotypicFeature - {'prevents'} -> Disease\n",
      "PhenotypicFeature - {'occurs_in'} -> Disease\n",
      "Disease - {'correlated_with'} -> Publication\n",
      "Disease - {'associated_with'} -> Publication\n",
      "Disease - {'gene_associated_with_condition'} -> Publication\n",
      "Disease - {'subclass_of'} -> Publication\n",
      "Disease - {'has_part'} -> Publication\n",
      "Disease - {'interacts_with'} -> Publication\n",
      "Disease - {'produces'} -> Publication\n",
      "Disease - {'predisposes'} -> Publication\n",
      "Disease - {'related_to'} -> Publication\n",
      "Disease - {'prevents'} -> Publication\n",
      "Disease - {'diagnoses'} -> Publication\n",
      "Disease - {'contributes_to'} -> Publication\n",
      "Disease - {'manifestation_of'} -> Publication\n",
      "Disease - {'occurs_in'} -> Publication\n",
      "Disease - {'affects'} -> Publication\n",
      "Disease - {'has_input'} -> Publication\n",
      "Disease - {'exacerbates'} -> Publication\n",
      "Disease - {'coexists_with'} -> Publication\n",
      "Disease - {'close_match'} -> Publication\n",
      "Disease - {'disrupts'} -> Publication\n",
      "Disease - {'regulates'} -> Publication\n",
      "Disease - {'biomarker_for'} -> Publication\n",
      "Disease - {'located_in'} -> Publication\n",
      "Disease - {'treats'} -> Publication\n",
      "Disease - {'causes'} -> Publication\n",
      "Disease - {'disease_has_location'} -> Publication\n",
      "Disease - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Drug->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> Drug\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> Drug\n",
      "PhenotypicFeature - {'correlated_with'} -> Drug\n",
      "PhenotypicFeature - {'related_to'} -> Drug\n",
      "PhenotypicFeature - {'precedes'} -> Drug\n",
      "PhenotypicFeature - {'overlaps'} -> Drug\n",
      "PhenotypicFeature - {'disrupted_by'} -> Drug\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Drug\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Drug\n",
      "PhenotypicFeature - {'treated_by'} -> Drug\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Drug\n",
      "PhenotypicFeature - {'regulated_by'} -> Drug\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Drug\n",
      "PhenotypicFeature - {'has_output'} -> Drug\n",
      "PhenotypicFeature - {'response_affected_by'} -> Drug\n",
      "PhenotypicFeature - {'affects'} -> Drug\n",
      "PhenotypicFeature - {'similar_to'} -> Drug\n",
      "PhenotypicFeature - {'affected_by'} -> Drug\n",
      "PhenotypicFeature - {'treats'} -> Drug\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> Drug\n",
      "PhenotypicFeature - {'participates_in'} -> Drug\n",
      "PhenotypicFeature - {'has_input'} -> Drug\n",
      "PhenotypicFeature - {'preceded_by'} -> Drug\n",
      "PhenotypicFeature - {'enabled_by'} -> Drug\n",
      "PhenotypicFeature - {'has_participant'} -> Drug\n",
      "PhenotypicFeature - {'response_decreased_by'} -> Drug\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> Drug\n",
      "PhenotypicFeature - {'has_substrate'} -> Drug\n",
      "PhenotypicFeature - {'interacts_with'} -> Drug\n",
      "PhenotypicFeature - {'ameliorates'} -> Drug\n",
      "PhenotypicFeature - {'response_increased_by'} -> Drug\n",
      "PhenotypicFeature - {'has_capability'} -> Drug\n",
      "PhenotypicFeature - {'caused_by'} -> Drug\n",
      "PhenotypicFeature - {'actively_involves'} -> Drug\n",
      "PhenotypicFeature - {'has_catalyst'} -> Drug\n",
      "PhenotypicFeature - {'associated_with'} -> Drug\n",
      "PhenotypicFeature - {'has_phenotype'} -> Drug\n",
      "PhenotypicFeature - {'located_in'} -> Drug\n",
      "PhenotypicFeature - {'biomarker_for'} -> Drug\n",
      "PhenotypicFeature - {'has_part'} -> Drug\n",
      "PhenotypicFeature - {'prevents'} -> Drug\n",
      "PhenotypicFeature - {'colocalizes_with'} -> Drug\n",
      "PhenotypicFeature - {'contributes_to'} -> Drug\n",
      "PhenotypicFeature - {'produces'} -> Drug\n",
      "PhenotypicFeature - {'predisposes'} -> Drug\n",
      "PhenotypicFeature - {'occurs_in'} -> Drug\n",
      "PhenotypicFeature - {'exacerbates'} -> Drug\n",
      "PhenotypicFeature - {'coexists_with'} -> Drug\n",
      "PhenotypicFeature - {'close_match'} -> Drug\n",
      "PhenotypicFeature - {'manifestation_of'} -> Drug\n",
      "PhenotypicFeature - {'regulates'} -> Drug\n",
      "PhenotypicFeature - {'subclass_of'} -> Drug\n",
      "PhenotypicFeature - {'chemically_similar_to'} -> Drug\n",
      "PhenotypicFeature - {'diagnoses'} -> Drug\n",
      "PhenotypicFeature - {'derives_from'} -> Drug\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Drug\n",
      "PhenotypicFeature - {'causes'} -> Drug\n",
      "PhenotypicFeature - {'disease_has_location'} -> Drug\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> Drug\n",
      "PhenotypicFeature - {'disrupts'} -> Drug\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Drug\n",
      "Drug - {'correlated_with'} -> Publication\n",
      "Drug - {'regulates'} -> Publication\n",
      "Drug - {'treats'} -> Publication\n",
      "Drug - {'has_input'} -> Publication\n",
      "Drug - {'physically_interacts_with'} -> Publication\n",
      "Drug - {'causes'} -> Publication\n",
      "Drug - {'directly_physically_interacts_with'} -> Publication\n",
      "Drug - {'disrupts'} -> Publication\n",
      "Drug - {'associated_with'} -> Publication\n",
      "Drug - {'gene_associated_with_condition'} -> Publication\n",
      "Drug - {'located_in'} -> Publication\n",
      "Drug - {'has_part'} -> Publication\n",
      "Drug - {'prevents'} -> Publication\n",
      "Drug - {'colocalizes_with'} -> Publication\n",
      "Drug - {'interacts_with'} -> Publication\n",
      "Drug - {'predisposes'} -> Publication\n",
      "Drug - {'related_to'} -> Publication\n",
      "Drug - {'occurs_in'} -> Publication\n",
      "Drug - {'precedes'} -> Publication\n",
      "Drug - {'affects'} -> Publication\n",
      "Drug - {'close_match'} -> Publication\n",
      "Drug - {'subclass_of'} -> Publication\n",
      "Drug - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Procedure->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> Procedure\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> Procedure\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> Procedure\n",
      "PhenotypicFeature - {'treated_by'} -> Procedure\n",
      "PhenotypicFeature - {'correlated_with'} -> Procedure\n",
      "PhenotypicFeature - {'is_diagnosed_by'} -> Procedure\n",
      "PhenotypicFeature - {'caused_by'} -> Procedure\n",
      "PhenotypicFeature - {'prevented_by'} -> Procedure\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> Procedure\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> Procedure\n",
      "PhenotypicFeature - {'has_predisposing_factor'} -> Procedure\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> Procedure\n",
      "PhenotypicFeature - {'affected_by'} -> Procedure\n",
      "PhenotypicFeature - {'treats'} -> Procedure\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> Procedure\n",
      "PhenotypicFeature - {'response_increased_by'} -> Procedure\n",
      "PhenotypicFeature - {'participates_in'} -> Procedure\n",
      "PhenotypicFeature - {'has_participant'} -> Procedure\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> Procedure\n",
      "PhenotypicFeature - {'interacts_with'} -> Procedure\n",
      "PhenotypicFeature - {'ameliorates'} -> Procedure\n",
      "PhenotypicFeature - {'response_affected_by'} -> Procedure\n",
      "PhenotypicFeature - {'associated_with'} -> Procedure\n",
      "PhenotypicFeature - {'affects'} -> Procedure\n",
      "PhenotypicFeature - {'similar_to'} -> Procedure\n",
      "PhenotypicFeature - {'related_to'} -> Procedure\n",
      "PhenotypicFeature - {'overlaps'} -> Procedure\n",
      "PhenotypicFeature - {'disrupted_by'} -> Procedure\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Procedure\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Procedure\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Procedure\n",
      "PhenotypicFeature - {'regulated_by'} -> Procedure\n",
      "PhenotypicFeature - {'response_decreased_by'} -> Procedure\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Procedure\n",
      "PhenotypicFeature - {'has_part'} -> Procedure\n",
      "PhenotypicFeature - {'located_in'} -> Procedure\n",
      "PhenotypicFeature - {'produces'} -> Procedure\n",
      "PhenotypicFeature - {'prevents'} -> Procedure\n",
      "PhenotypicFeature - {'contributes_to'} -> Procedure\n",
      "PhenotypicFeature - {'predisposes'} -> Procedure\n",
      "PhenotypicFeature - {'precedes'} -> Procedure\n",
      "PhenotypicFeature - {'occurs_in'} -> Procedure\n",
      "PhenotypicFeature - {'close_match'} -> Procedure\n",
      "PhenotypicFeature - {'manifestation_of'} -> Procedure\n",
      "PhenotypicFeature - {'exacerbates'} -> Procedure\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> Procedure\n",
      "PhenotypicFeature - {'coexists_with'} -> Procedure\n",
      "PhenotypicFeature - {'subclass_of'} -> Procedure\n",
      "PhenotypicFeature - {'regulates'} -> Procedure\n",
      "PhenotypicFeature - {'diagnoses'} -> Procedure\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Procedure\n",
      "PhenotypicFeature - {'has_input'} -> Procedure\n",
      "PhenotypicFeature - {'derives_from'} -> Procedure\n",
      "PhenotypicFeature - {'causes'} -> Procedure\n",
      "PhenotypicFeature - {'disrupts'} -> Procedure\n",
      "PhenotypicFeature - {'has_phenotype'} -> Procedure\n",
      "PhenotypicFeature - {'biomarker_for'} -> Procedure\n",
      "Procedure - {'correlated_with'} -> Publication\n",
      "Procedure - {'manifestation_of'} -> Publication\n",
      "Procedure - {'close_match'} -> Publication\n",
      "Procedure - {'subclass_of'} -> Publication\n",
      "Procedure - {'interacts_with'} -> Publication\n",
      "Procedure - {'diagnoses'} -> Publication\n",
      "Procedure - {'occurs_in'} -> Publication\n",
      "Procedure - {'treats'} -> Publication\n",
      "Procedure - {'has_input'} -> Publication\n",
      "Procedure - {'exacerbates'} -> Publication\n",
      "Procedure - {'causes'} -> Publication\n",
      "Procedure - {'coexists_with'} -> Publication\n",
      "Procedure - {'disrupts'} -> Publication\n",
      "Procedure - {'associated_with'} -> Publication\n",
      "Procedure - {'regulates'} -> Publication\n",
      "Procedure - {'produces'} -> Publication\n",
      "Procedure - {'located_in'} -> Publication\n",
      "Procedure - {'predisposes'} -> Publication\n",
      "Procedure - {'related_to'} -> Publication\n",
      "Procedure - {'has_part'} -> Publication\n",
      "Procedure - {'prevents'} -> Publication\n",
      "Procedure - {'precedes'} -> Publication\n",
      "Procedure - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->SmallMolecule->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'ameliorates'} -> SmallMolecule\n",
      "PhenotypicFeature - {'treated_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'is_assessed_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'is_side_effect_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'caused_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_biomarker'} -> SmallMolecule\n",
      "PhenotypicFeature - {'affected_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_contraindication'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_real_world_evidence_of_association_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'contribution_from'} -> SmallMolecule\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_predisposing_factor'} -> SmallMolecule\n",
      "PhenotypicFeature - {'disrupted_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'associated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'related_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'correlated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'adverse_event_caused_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'entity_positively_regulated_by_entity'} -> SmallMolecule\n",
      "PhenotypicFeature - {'prevented_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'affects'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_part'} -> SmallMolecule\n",
      "PhenotypicFeature - {'contributes_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_participant'} -> SmallMolecule\n",
      "PhenotypicFeature - {'decreases_response_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> SmallMolecule\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'regulated_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'response_affected_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'coexpressed_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'similar_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'subclass_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'overlaps'} -> SmallMolecule\n",
      "PhenotypicFeature - {'biomarker_for'} -> SmallMolecule\n",
      "PhenotypicFeature - {'affects_response_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'exacerbates'} -> SmallMolecule\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'increases_response_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'response_decreased_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'disrupts'} -> SmallMolecule\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'interacts_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'treats'} -> SmallMolecule\n",
      "PhenotypicFeature - {'regulates'} -> SmallMolecule\n",
      "PhenotypicFeature - {'causes'} -> SmallMolecule\n",
      "PhenotypicFeature - {'response_increased_by'} -> SmallMolecule\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'part_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'participates_in'} -> SmallMolecule\n",
      "PhenotypicFeature - {'located_in'} -> SmallMolecule\n",
      "PhenotypicFeature - {'produces'} -> SmallMolecule\n",
      "PhenotypicFeature - {'prevents'} -> SmallMolecule\n",
      "PhenotypicFeature - {'colocalizes_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'predisposes'} -> SmallMolecule\n",
      "PhenotypicFeature - {'occurs_in'} -> SmallMolecule\n",
      "PhenotypicFeature - {'precedes'} -> SmallMolecule\n",
      "PhenotypicFeature - {'coexists_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'close_match'} -> SmallMolecule\n",
      "PhenotypicFeature - {'manifestation_of'} -> SmallMolecule\n",
      "PhenotypicFeature - {'chemically_similar_to'} -> SmallMolecule\n",
      "PhenotypicFeature - {'disease_has_basis_in'} -> SmallMolecule\n",
      "PhenotypicFeature - {'diagnoses'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_input'} -> SmallMolecule\n",
      "PhenotypicFeature - {'derives_from'} -> SmallMolecule\n",
      "PhenotypicFeature - {'in_taxon'} -> SmallMolecule\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_output'} -> SmallMolecule\n",
      "PhenotypicFeature - {'disease_has_location'} -> SmallMolecule\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> SmallMolecule\n",
      "PhenotypicFeature - {'actively_involved_in'} -> SmallMolecule\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> SmallMolecule\n",
      "PhenotypicFeature - {'has_phenotype'} -> SmallMolecule\n",
      "PhenotypicFeature - {'adverse_event_of'} -> SmallMolecule\n",
      "SmallMolecule - {'correlated_with'} -> Publication\n",
      "SmallMolecule - {'disrupts'} -> Publication\n",
      "SmallMolecule - {'gene_associated_with_condition'} -> Publication\n",
      "SmallMolecule - {'located_in'} -> Publication\n",
      "SmallMolecule - {'regulates'} -> Publication\n",
      "SmallMolecule - {'has_part'} -> Publication\n",
      "SmallMolecule - {'physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'predisposes'} -> Publication\n",
      "SmallMolecule - {'related_to'} -> Publication\n",
      "SmallMolecule - {'causes'} -> Publication\n",
      "SmallMolecule - {'directly_physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'precedes'} -> Publication\n",
      "SmallMolecule - {'close_match'} -> Publication\n",
      "SmallMolecule - {'associated_with'} -> Publication\n",
      "SmallMolecule - {'has_metabolite'} -> Publication\n",
      "SmallMolecule - {'subclass_of'} -> Publication\n",
      "SmallMolecule - {'interacts_with'} -> Publication\n",
      "SmallMolecule - {'treats'} -> Publication\n",
      "SmallMolecule - {'has_input'} -> Publication\n",
      "SmallMolecule - {'occurs_in'} -> Publication\n",
      "SmallMolecule - {'affects'} -> Publication\n",
      "SmallMolecule - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ChemicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'treated_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'is_assessed_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'contribution_from'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'affected_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'subclass_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_biomarker'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_contraindication'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_predisposing_factor'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_real_world_evidence_of_association_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'affects'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'correlated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'associated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'caused_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'related_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'entity_positively_regulated_by_entity'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'treats'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'causes'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'part_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'participates_in'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'contributes_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_input'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'preceded_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'enabled_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_participant'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_part'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_substrate'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'precedes'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'ameliorates'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'regulates'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'response_increased_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_capability'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'actively_involves'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_catalyst'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'coexpressed_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'similar_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'overlaps'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'biomarker_for'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'affects_response_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'disrupted_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'increases_response_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'regulated_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_output'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'response_affected_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'response_decreased_by'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'disrupts'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'disease_has_location'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'has_phenotype'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'located_in'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'colocalizes_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'produces'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'prevents'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'predisposes'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'close_match'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'manifestation_of'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'coexists_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'diagnoses'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'chemically_similar_to'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'disease_has_basis_in'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'derives_from'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'in_taxon'} -> ChemicalEntity\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> ChemicalEntity\n",
      "ChemicalEntity - {'correlated_with'} -> Publication\n",
      "ChemicalEntity - {'disrupts'} -> Publication\n",
      "ChemicalEntity - {'associated_with'} -> Publication\n",
      "ChemicalEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "ChemicalEntity - {'regulates'} -> Publication\n",
      "ChemicalEntity - {'located_in'} -> Publication\n",
      "ChemicalEntity - {'predisposes'} -> Publication\n",
      "ChemicalEntity - {'related_to'} -> Publication\n",
      "ChemicalEntity - {'has_part'} -> Publication\n",
      "ChemicalEntity - {'prevents'} -> Publication\n",
      "ChemicalEntity - {'affects'} -> Publication\n",
      "ChemicalEntity - {'precedes'} -> Publication\n",
      "ChemicalEntity - {'occurs_in'} -> Publication\n",
      "ChemicalEntity - {'has_metabolite'} -> Publication\n",
      "ChemicalEntity - {'close_match'} -> Publication\n",
      "ChemicalEntity - {'subclass_of'} -> Publication\n",
      "ChemicalEntity - {'interacts_with'} -> Publication\n",
      "ChemicalEntity - {'treats'} -> Publication\n",
      "ChemicalEntity - {'physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'has_input'} -> Publication\n",
      "ChemicalEntity - {'causes'} -> Publication\n",
      "ChemicalEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Cell->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'treated_by'} -> Cell\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> Cell\n",
      "PhenotypicFeature - {'related_to'} -> Cell\n",
      "PhenotypicFeature - {'affects'} -> Cell\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> Cell\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> Cell\n",
      "PhenotypicFeature - {'disrupts'} -> Cell\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> Cell\n",
      "PhenotypicFeature - {'correlated_with'} -> Cell\n",
      "PhenotypicFeature - {'affected_by'} -> Cell\n",
      "PhenotypicFeature - {'has_substrate'} -> Cell\n",
      "PhenotypicFeature - {'interacts_with'} -> Cell\n",
      "PhenotypicFeature - {'treats'} -> Cell\n",
      "PhenotypicFeature - {'regulates'} -> Cell\n",
      "PhenotypicFeature - {'causes'} -> Cell\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> Cell\n",
      "PhenotypicFeature - {'part_of'} -> Cell\n",
      "PhenotypicFeature - {'has_capability'} -> Cell\n",
      "PhenotypicFeature - {'has_biomarker'} -> Cell\n",
      "PhenotypicFeature - {'participates_in'} -> Cell\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> Cell\n",
      "PhenotypicFeature - {'contributes_to'} -> Cell\n",
      "PhenotypicFeature - {'caused_by'} -> Cell\n",
      "PhenotypicFeature - {'has_input'} -> Cell\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> Cell\n",
      "PhenotypicFeature - {'enabled_by'} -> Cell\n",
      "PhenotypicFeature - {'has_participant'} -> Cell\n",
      "PhenotypicFeature - {'has_catalyst'} -> Cell\n",
      "PhenotypicFeature - {'superclass_of'} -> Cell\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Cell\n",
      "PhenotypicFeature - {'associated_with'} -> Cell\n",
      "PhenotypicFeature - {'has_part'} -> Cell\n",
      "PhenotypicFeature - {'precedes'} -> Cell\n",
      "PhenotypicFeature - {'ameliorates'} -> Cell\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> Cell\n",
      "PhenotypicFeature - {'genetic_association'} -> Cell\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> Cell\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> Cell\n",
      "PhenotypicFeature - {'actively_involves'} -> Cell\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Cell\n",
      "PhenotypicFeature - {'exacerbates'} -> Cell\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Cell\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Cell\n",
      "PhenotypicFeature - {'expressed_in'} -> Cell\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Cell\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> Cell\n",
      "PhenotypicFeature - {'coexpressed_with'} -> Cell\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> Cell\n",
      "PhenotypicFeature - {'similar_to'} -> Cell\n",
      "PhenotypicFeature - {'subclass_of'} -> Cell\n",
      "PhenotypicFeature - {'overlaps'} -> Cell\n",
      "PhenotypicFeature - {'biomarker_for'} -> Cell\n",
      "PhenotypicFeature - {'contribution_from'} -> Cell\n",
      "PhenotypicFeature - {'affects_response_to'} -> Cell\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> Cell\n",
      "PhenotypicFeature - {'located_in'} -> Cell\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Cell\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Cell\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> Cell\n",
      "PhenotypicFeature - {'increases_response_to'} -> Cell\n",
      "PhenotypicFeature - {'preceded_by'} -> Cell\n",
      "PhenotypicFeature - {'has_output'} -> Cell\n",
      "PhenotypicFeature - {'has_phenotype'} -> Cell\n",
      "PhenotypicFeature - {'prevents'} -> Cell\n",
      "PhenotypicFeature - {'produces'} -> Cell\n",
      "PhenotypicFeature - {'predisposes'} -> Cell\n",
      "PhenotypicFeature - {'occurs_in'} -> Cell\n",
      "PhenotypicFeature - {'coexists_with'} -> Cell\n",
      "PhenotypicFeature - {'close_match'} -> Cell\n",
      "PhenotypicFeature - {'manifestation_of'} -> Cell\n",
      "PhenotypicFeature - {'diagnoses'} -> Cell\n",
      "PhenotypicFeature - {'derives_from'} -> Cell\n",
      "PhenotypicFeature - {'disease_has_location'} -> Cell\n",
      "Cell - {'subclass_of'} -> Publication\n",
      "Cell - {'has_part'} -> Publication\n",
      "Cell - {'produces'} -> Publication\n",
      "Cell - {'related_to'} -> Publication\n",
      "Cell - {'occurs_in'} -> Publication\n",
      "Cell - {'affects'} -> Publication\n",
      "Cell - {'coexists_with'} -> Publication\n",
      "Cell - {'close_match'} -> Publication\n",
      "Cell - {'located_in'} -> Publication\n",
      "Cell - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Protein->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'treated_by'} -> Protein\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> Protein\n",
      "PhenotypicFeature - {'phenotype_of'} -> Protein\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> Protein\n",
      "PhenotypicFeature - {'contribution_from'} -> Protein\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> Protein\n",
      "PhenotypicFeature - {'affects'} -> Protein\n",
      "PhenotypicFeature - {'genetic_association'} -> Protein\n",
      "PhenotypicFeature - {'correlated_with'} -> Protein\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> Protein\n",
      "PhenotypicFeature - {'caused_by'} -> Protein\n",
      "PhenotypicFeature - {'related_to'} -> Protein\n",
      "PhenotypicFeature - {'preceded_by'} -> Protein\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> Protein\n",
      "PhenotypicFeature - {'response_decreased_by'} -> Protein\n",
      "PhenotypicFeature - {'disrupts'} -> Protein\n",
      "PhenotypicFeature - {'superclass_of'} -> Protein\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> Protein\n",
      "PhenotypicFeature - {'affected_by'} -> Protein\n",
      "PhenotypicFeature - {'has_substrate'} -> Protein\n",
      "PhenotypicFeature - {'interacts_with'} -> Protein\n",
      "PhenotypicFeature - {'ameliorates'} -> Protein\n",
      "PhenotypicFeature - {'treats'} -> Protein\n",
      "PhenotypicFeature - {'regulates'} -> Protein\n",
      "PhenotypicFeature - {'response_increased_by'} -> Protein\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> Protein\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> Protein\n",
      "PhenotypicFeature - {'part_of'} -> Protein\n",
      "PhenotypicFeature - {'has_capability'} -> Protein\n",
      "PhenotypicFeature - {'has_biomarker'} -> Protein\n",
      "PhenotypicFeature - {'participates_in'} -> Protein\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> Protein\n",
      "PhenotypicFeature - {'contributes_to'} -> Protein\n",
      "PhenotypicFeature - {'actively_involves'} -> Protein\n",
      "PhenotypicFeature - {'has_input'} -> Protein\n",
      "PhenotypicFeature - {'enabled_by'} -> Protein\n",
      "PhenotypicFeature - {'has_participant'} -> Protein\n",
      "PhenotypicFeature - {'has_catalyst'} -> Protein\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Protein\n",
      "PhenotypicFeature - {'associated_with'} -> Protein\n",
      "PhenotypicFeature - {'has_part'} -> Protein\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> Protein\n",
      "PhenotypicFeature - {'precedes'} -> Protein\n",
      "PhenotypicFeature - {'biomarker_for'} -> Protein\n",
      "PhenotypicFeature - {'disrupted_by'} -> Protein\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> Protein\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> Protein\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Protein\n",
      "PhenotypicFeature - {'exacerbates'} -> Protein\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Protein\n",
      "PhenotypicFeature - {'increases_response_to'} -> Protein\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Protein\n",
      "PhenotypicFeature - {'regulated_by'} -> Protein\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Protein\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> Protein\n",
      "PhenotypicFeature - {'has_output'} -> Protein\n",
      "PhenotypicFeature - {'response_affected_by'} -> Protein\n",
      "PhenotypicFeature - {'coexpressed_with'} -> Protein\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> Protein\n",
      "PhenotypicFeature - {'similar_to'} -> Protein\n",
      "PhenotypicFeature - {'subclass_of'} -> Protein\n",
      "PhenotypicFeature - {'overlaps'} -> Protein\n",
      "PhenotypicFeature - {'entity_positively_regulated_by_entity'} -> Protein\n",
      "PhenotypicFeature - {'affects_response_to'} -> Protein\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> Protein\n",
      "PhenotypicFeature - {'causes'} -> Protein\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Protein\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Protein\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> Protein\n",
      "PhenotypicFeature - {'manifestation_of'} -> Protein\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> Protein\n",
      "PhenotypicFeature - {'coexists_with'} -> Protein\n",
      "PhenotypicFeature - {'diagnoses'} -> Protein\n",
      "PhenotypicFeature - {'capable_of'} -> Protein\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Protein\n",
      "PhenotypicFeature - {'derives_from'} -> Protein\n",
      "PhenotypicFeature - {'disease_has_location'} -> Protein\n",
      "PhenotypicFeature - {'gene_product_of'} -> Protein\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> Protein\n",
      "PhenotypicFeature - {'has_phenotype'} -> Protein\n",
      "PhenotypicFeature - {'located_in'} -> Protein\n",
      "PhenotypicFeature - {'colocalizes_with'} -> Protein\n",
      "PhenotypicFeature - {'produces'} -> Protein\n",
      "PhenotypicFeature - {'prevents'} -> Protein\n",
      "PhenotypicFeature - {'predisposes'} -> Protein\n",
      "PhenotypicFeature - {'occurs_in'} -> Protein\n",
      "PhenotypicFeature - {'close_match'} -> Protein\n",
      "Protein - {'correlated_with'} -> Publication\n",
      "Protein - {'gene_product_of'} -> Publication\n",
      "Protein - {'associated_with'} -> Publication\n",
      "Protein - {'gene_associated_with_condition'} -> Publication\n",
      "Protein - {'located_in'} -> Publication\n",
      "Protein - {'regulates'} -> Publication\n",
      "Protein - {'biomarker_for'} -> Publication\n",
      "Protein - {'expressed_in'} -> Publication\n",
      "Protein - {'has_part'} -> Publication\n",
      "Protein - {'colocalizes_with'} -> Publication\n",
      "Protein - {'produces'} -> Publication\n",
      "Protein - {'predisposes'} -> Publication\n",
      "Protein - {'related_to'} -> Publication\n",
      "Protein - {'prevents'} -> Publication\n",
      "Protein - {'occurs_in'} -> Publication\n",
      "Protein - {'precedes'} -> Publication\n",
      "Protein - {'affects'} -> Publication\n",
      "Protein - {'close_match'} -> Publication\n",
      "Protein - {'subclass_of'} -> Publication\n",
      "Protein - {'interacts_with'} -> Publication\n",
      "Protein - {'treats'} -> Publication\n",
      "Protein - {'has_input'} -> Publication\n",
      "Protein - {'causes'} -> Publication\n",
      "Protein - {'physically_interacts_with'} -> Publication\n",
      "Protein - {'exacerbates'} -> Publication\n",
      "Protein - {'coexists_with'} -> Publication\n",
      "Protein - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->MolecularEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'is_assessed_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'treated_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'affects'} -> MolecularEntity\n",
      "PhenotypicFeature - {'caused_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'related_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_part'} -> MolecularEntity\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'precedes'} -> MolecularEntity\n",
      "PhenotypicFeature - {'disrupted_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> MolecularEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> MolecularEntity\n",
      "PhenotypicFeature - {'increases_response_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> MolecularEntity\n",
      "PhenotypicFeature - {'regulated_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_output'} -> MolecularEntity\n",
      "PhenotypicFeature - {'response_affected_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'similar_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'overlaps'} -> MolecularEntity\n",
      "PhenotypicFeature - {'affects_response_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'treats'} -> MolecularEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> MolecularEntity\n",
      "PhenotypicFeature - {'preceded_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'enabled_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'response_decreased_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'disrupts'} -> MolecularEntity\n",
      "PhenotypicFeature - {'affected_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_substrate'} -> MolecularEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> MolecularEntity\n",
      "PhenotypicFeature - {'ameliorates'} -> MolecularEntity\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'regulates'} -> MolecularEntity\n",
      "PhenotypicFeature - {'response_increased_by'} -> MolecularEntity\n",
      "PhenotypicFeature - {'part_of'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_capability'} -> MolecularEntity\n",
      "PhenotypicFeature - {'participates_in'} -> MolecularEntity\n",
      "PhenotypicFeature - {'actively_involves'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_input'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_participant'} -> MolecularEntity\n",
      "PhenotypicFeature - {'has_catalyst'} -> MolecularEntity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'associated_with'} -> MolecularEntity\n",
      "PhenotypicFeature - {'biomarker_for'} -> MolecularEntity\n",
      "PhenotypicFeature - {'produces'} -> MolecularEntity\n",
      "PhenotypicFeature - {'colocalizes_with'} -> MolecularEntity\n",
      "PhenotypicFeature - {'contributes_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'predisposes'} -> MolecularEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> MolecularEntity\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> MolecularEntity\n",
      "PhenotypicFeature - {'coexists_with'} -> MolecularEntity\n",
      "PhenotypicFeature - {'close_match'} -> MolecularEntity\n",
      "PhenotypicFeature - {'manifestation_of'} -> MolecularEntity\n",
      "PhenotypicFeature - {'subclass_of'} -> MolecularEntity\n",
      "PhenotypicFeature - {'chemically_similar_to'} -> MolecularEntity\n",
      "PhenotypicFeature - {'disease_has_basis_in'} -> MolecularEntity\n",
      "PhenotypicFeature - {'diagnoses'} -> MolecularEntity\n",
      "PhenotypicFeature - {'derives_from'} -> MolecularEntity\n",
      "PhenotypicFeature - {'causes'} -> MolecularEntity\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> MolecularEntity\n",
      "PhenotypicFeature - {'disease_has_location'} -> MolecularEntity\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> MolecularEntity\n",
      "PhenotypicFeature - {'located_in'} -> MolecularEntity\n",
      "MolecularEntity - {'causes'} -> Publication\n",
      "MolecularEntity - {'associated_with'} -> Publication\n",
      "MolecularEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "MolecularEntity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularEntity - {'regulates'} -> Publication\n",
      "MolecularEntity - {'disrupts'} -> Publication\n",
      "MolecularEntity - {'located_in'} -> Publication\n",
      "MolecularEntity - {'biomarker_for'} -> Publication\n",
      "MolecularEntity - {'interacts_with'} -> Publication\n",
      "MolecularEntity - {'predisposes'} -> Publication\n",
      "MolecularEntity - {'related_to'} -> Publication\n",
      "MolecularEntity - {'has_part'} -> Publication\n",
      "MolecularEntity - {'occurs_in'} -> Publication\n",
      "MolecularEntity - {'affects'} -> Publication\n",
      "MolecularEntity - {'precedes'} -> Publication\n",
      "MolecularEntity - {'coexists_with'} -> Publication\n",
      "MolecularEntity - {'close_match'} -> Publication\n",
      "MolecularEntity - {'subclass_of'} -> Publication\n",
      "MolecularEntity - {'treats'} -> Publication\n",
      "MolecularEntity - {'physically_interacts_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Gene->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'has_participant'} -> Gene\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> Gene\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> Gene\n",
      "PhenotypicFeature - {'entity_positively_regulated_by_entity'} -> Gene\n",
      "PhenotypicFeature - {'phenotype_of'} -> Gene\n",
      "PhenotypicFeature - {'mode_of_inheritance_of'} -> Gene\n",
      "PhenotypicFeature - {'affects'} -> Gene\n",
      "PhenotypicFeature - {'correlated_with'} -> Gene\n",
      "PhenotypicFeature - {'associated_with'} -> Gene\n",
      "PhenotypicFeature - {'affected_by'} -> Gene\n",
      "PhenotypicFeature - {'disrupted_by'} -> Gene\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> Gene\n",
      "PhenotypicFeature - {'has_predisposing_factor'} -> Gene\n",
      "PhenotypicFeature - {'contribution_from'} -> Gene\n",
      "PhenotypicFeature - {'caused_by'} -> Gene\n",
      "PhenotypicFeature - {'related_to'} -> Gene\n",
      "PhenotypicFeature - {'genetic_association'} -> Gene\n",
      "PhenotypicFeature - {'has_catalyst'} -> Gene\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Gene\n",
      "PhenotypicFeature - {'has_part'} -> Gene\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> Gene\n",
      "PhenotypicFeature - {'precedes'} -> Gene\n",
      "PhenotypicFeature - {'overlaps'} -> Gene\n",
      "PhenotypicFeature - {'biomarker_for'} -> Gene\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> Gene\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> Gene\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Gene\n",
      "PhenotypicFeature - {'treated_by'} -> Gene\n",
      "PhenotypicFeature - {'exacerbates'} -> Gene\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Gene\n",
      "PhenotypicFeature - {'increases_response_to'} -> Gene\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Gene\n",
      "PhenotypicFeature - {'regulated_by'} -> Gene\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Gene\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> Gene\n",
      "PhenotypicFeature - {'has_output'} -> Gene\n",
      "PhenotypicFeature - {'response_affected_by'} -> Gene\n",
      "PhenotypicFeature - {'coexpressed_with'} -> Gene\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> Gene\n",
      "PhenotypicFeature - {'disrupts'} -> Gene\n",
      "PhenotypicFeature - {'similar_to'} -> Gene\n",
      "PhenotypicFeature - {'affects_response_to'} -> Gene\n",
      "PhenotypicFeature - {'treats'} -> Gene\n",
      "PhenotypicFeature - {'causes'} -> Gene\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Gene\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> Gene\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Gene\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> Gene\n",
      "PhenotypicFeature - {'preceded_by'} -> Gene\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> Gene\n",
      "PhenotypicFeature - {'enabled_by'} -> Gene\n",
      "PhenotypicFeature - {'response_decreased_by'} -> Gene\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> Gene\n",
      "PhenotypicFeature - {'has_substrate'} -> Gene\n",
      "PhenotypicFeature - {'interacts_with'} -> Gene\n",
      "PhenotypicFeature - {'ameliorates'} -> Gene\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> Gene\n",
      "PhenotypicFeature - {'regulates'} -> Gene\n",
      "PhenotypicFeature - {'response_increased_by'} -> Gene\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> Gene\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> Gene\n",
      "PhenotypicFeature - {'part_of'} -> Gene\n",
      "PhenotypicFeature - {'has_capability'} -> Gene\n",
      "PhenotypicFeature - {'has_biomarker'} -> Gene\n",
      "PhenotypicFeature - {'participates_in'} -> Gene\n",
      "PhenotypicFeature - {'contributes_to'} -> Gene\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> Gene\n",
      "PhenotypicFeature - {'actively_involves'} -> Gene\n",
      "PhenotypicFeature - {'has_input'} -> Gene\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> Gene\n",
      "PhenotypicFeature - {'coexists_with'} -> Gene\n",
      "PhenotypicFeature - {'close_match'} -> Gene\n",
      "PhenotypicFeature - {'subclass_of'} -> Gene\n",
      "PhenotypicFeature - {'capable_of'} -> Gene\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Gene\n",
      "PhenotypicFeature - {'diagnoses'} -> Gene\n",
      "PhenotypicFeature - {'derives_from'} -> Gene\n",
      "PhenotypicFeature - {'disease_has_location'} -> Gene\n",
      "PhenotypicFeature - {'gene_product_of'} -> Gene\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> Gene\n",
      "PhenotypicFeature - {'has_phenotype'} -> Gene\n",
      "PhenotypicFeature - {'located_in'} -> Gene\n",
      "PhenotypicFeature - {'produces'} -> Gene\n",
      "PhenotypicFeature - {'prevents'} -> Gene\n",
      "PhenotypicFeature - {'colocalizes_with'} -> Gene\n",
      "PhenotypicFeature - {'predisposes'} -> Gene\n",
      "PhenotypicFeature - {'occurs_in'} -> Gene\n",
      "PhenotypicFeature - {'manifestation_of'} -> Gene\n",
      "Gene - {'correlated_with'} -> Publication\n",
      "Gene - {'prevents'} -> Publication\n",
      "Gene - {'treats'} -> Publication\n",
      "Gene - {'has_input'} -> Publication\n",
      "Gene - {'related_to'} -> Publication\n",
      "Gene - {'occurs_in'} -> Publication\n",
      "Gene - {'affects'} -> Publication\n",
      "Gene - {'exacerbates'} -> Publication\n",
      "Gene - {'coexists_with'} -> Publication\n",
      "Gene - {'disrupts'} -> Publication\n",
      "Gene - {'gene_associated_with_condition'} -> Publication\n",
      "Gene - {'located_in'} -> Publication\n",
      "Gene - {'regulates'} -> Publication\n",
      "Gene - {'biomarker_for'} -> Publication\n",
      "Gene - {'expressed_in'} -> Publication\n",
      "Gene - {'physically_interacts_with'} -> Publication\n",
      "Gene - {'predisposes'} -> Publication\n",
      "Gene - {'causes'} -> Publication\n",
      "Gene - {'precedes'} -> Publication\n",
      "Gene - {'gene_product_of'} -> Publication\n",
      "Gene - {'close_match'} -> Publication\n",
      "Gene - {'associated_with'} -> Publication\n",
      "Gene - {'subclass_of'} -> Publication\n",
      "Gene - {'has_part'} -> Publication\n",
      "Gene - {'colocalizes_with'} -> Publication\n",
      "Gene - {'interacts_with'} -> Publication\n",
      "Gene - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Pathway->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'genetic_association'} -> Pathway\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> Pathway\n",
      "PhenotypicFeature - {'affects'} -> Pathway\n",
      "PhenotypicFeature - {'preceded_by'} -> Pathway\n",
      "PhenotypicFeature - {'related_to'} -> Pathway\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Pathway\n",
      "PhenotypicFeature - {'affected_by'} -> Pathway\n",
      "PhenotypicFeature - {'interacts_with'} -> Pathway\n",
      "PhenotypicFeature - {'ameliorates'} -> Pathway\n",
      "PhenotypicFeature - {'treats'} -> Pathway\n",
      "PhenotypicFeature - {'regulates'} -> Pathway\n",
      "PhenotypicFeature - {'part_of'} -> Pathway\n",
      "PhenotypicFeature - {'has_capability'} -> Pathway\n",
      "PhenotypicFeature - {'participates_in'} -> Pathway\n",
      "PhenotypicFeature - {'contributes_to'} -> Pathway\n",
      "PhenotypicFeature - {'caused_by'} -> Pathway\n",
      "PhenotypicFeature - {'has_participant'} -> Pathway\n",
      "PhenotypicFeature - {'superclass_of'} -> Pathway\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Pathway\n",
      "PhenotypicFeature - {'associated_with'} -> Pathway\n",
      "PhenotypicFeature - {'has_part'} -> Pathway\n",
      "PhenotypicFeature - {'precedes'} -> Pathway\n",
      "PhenotypicFeature - {'actively_involves'} -> Pathway\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Pathway\n",
      "PhenotypicFeature - {'exacerbates'} -> Pathway\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Pathway\n",
      "PhenotypicFeature - {'increases_response_to'} -> Pathway\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Pathway\n",
      "PhenotypicFeature - {'similar_to'} -> Pathway\n",
      "PhenotypicFeature - {'overlaps'} -> Pathway\n",
      "PhenotypicFeature - {'affects_response_to'} -> Pathway\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> Pathway\n",
      "PhenotypicFeature - {'causes'} -> Pathway\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Pathway\n",
      "PhenotypicFeature - {'disrupts'} -> Pathway\n",
      "PhenotypicFeature - {'correlated_with'} -> Pathway\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Pathway\n",
      "PhenotypicFeature - {'has_phenotype'} -> Pathway\n",
      "PhenotypicFeature - {'located_in'} -> Pathway\n",
      "PhenotypicFeature - {'biomarker_for'} -> Pathway\n",
      "PhenotypicFeature - {'prevents'} -> Pathway\n",
      "PhenotypicFeature - {'colocalizes_with'} -> Pathway\n",
      "PhenotypicFeature - {'produces'} -> Pathway\n",
      "PhenotypicFeature - {'predisposes'} -> Pathway\n",
      "PhenotypicFeature - {'occurs_in'} -> Pathway\n",
      "PhenotypicFeature - {'coexists_with'} -> Pathway\n",
      "PhenotypicFeature - {'close_match'} -> Pathway\n",
      "PhenotypicFeature - {'manifestation_of'} -> Pathway\n",
      "PhenotypicFeature - {'subclass_of'} -> Pathway\n",
      "PhenotypicFeature - {'contraindicated_for'} -> Pathway\n",
      "PhenotypicFeature - {'diagnoses'} -> Pathway\n",
      "PhenotypicFeature - {'has_input'} -> Pathway\n",
      "PhenotypicFeature - {'disease_has_location'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Publication\n",
      "Pathway - {'regulates'} -> Publication\n",
      "Pathway - {'treats'} -> Publication\n",
      "Pathway - {'has_input'} -> Publication\n",
      "Pathway - {'causes'} -> Publication\n",
      "Pathway - {'associated_with'} -> Publication\n",
      "Pathway - {'gene_associated_with_condition'} -> Publication\n",
      "Pathway - {'disrupts'} -> Publication\n",
      "Pathway - {'located_in'} -> Publication\n",
      "Pathway - {'produces'} -> Publication\n",
      "Pathway - {'predisposes'} -> Publication\n",
      "Pathway - {'related_to'} -> Publication\n",
      "Pathway - {'precedes'} -> Publication\n",
      "Pathway - {'occurs_in'} -> Publication\n",
      "Pathway - {'affects'} -> Publication\n",
      "Pathway - {'manifestation_of'} -> Publication\n",
      "Pathway - {'coexists_with'} -> Publication\n",
      "Pathway - {'close_match'} -> Publication\n",
      "Pathway - {'has_participant'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->AnatomicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'superclass_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'subclass_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'located_in'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'affects'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'contribution_from'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'caused_by'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'related_to'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'precedes'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'overlaps'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'biomarker_for'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'affects_response_to'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'genetic_association'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'treated_by'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'increases_response_to'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_output'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'disrupts'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'affected_by'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'treats'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'causes'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_biomarker'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'participates_in'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'contributes_to'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_input'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'preceded_by'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'enabled_by'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_participant'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_part'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'correlated_with'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_substrate'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'ameliorates'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'regulates'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'part_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_capability'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'actively_involves'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_catalyst'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'expressed_in'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'associated_with'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'similar_to'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'close_match'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'manifestation_of'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'coexists_with'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'develops_from'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'diagnoses'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'disease_has_basis_in'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'derives_from'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'disease_has_location'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'actively_involved_in'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'has_phenotype'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'colocalizes_with'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'produces'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'predisposes'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'prevents'} -> AnatomicalEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> AnatomicalEntity\n",
      "AnatomicalEntity - {'develops_from'} -> Publication\n",
      "AnatomicalEntity - {'interacts_with'} -> Publication\n",
      "AnatomicalEntity - {'produces'} -> Publication\n",
      "AnatomicalEntity - {'has_part'} -> Publication\n",
      "AnatomicalEntity - {'diagnoses'} -> Publication\n",
      "AnatomicalEntity - {'related_to'} -> Publication\n",
      "AnatomicalEntity - {'affects'} -> Publication\n",
      "AnatomicalEntity - {'manifestation_of'} -> Publication\n",
      "AnatomicalEntity - {'has_input'} -> Publication\n",
      "AnatomicalEntity - {'occurs_in'} -> Publication\n",
      "AnatomicalEntity - {'coexists_with'} -> Publication\n",
      "AnatomicalEntity - {'close_match'} -> Publication\n",
      "AnatomicalEntity - {'disrupts'} -> Publication\n",
      "AnatomicalEntity - {'located_in'} -> Publication\n",
      "AnatomicalEntity - {'regulates'} -> Publication\n",
      "AnatomicalEntity - {'treats'} -> Publication\n",
      "AnatomicalEntity - {'causes'} -> Publication\n",
      "AnatomicalEntity - {'associated_with'} -> Publication\n",
      "AnatomicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "AnatomicalEntity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->BiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'affects'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'subclass_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'correlated_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_real_world_evidence_of_association_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'preceded_by'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'related_to'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'temporally_related_to'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'disrupts'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'superclass_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'affected_by'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_substrate'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'interacts_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'ameliorates'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'treats'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'regulates'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'part_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_capability'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'participates_in'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'contributes_to'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'caused_by'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'actively_involves'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_input'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'enabled_by'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_participant'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_catalyst'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'decreases_response_to'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'associated_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_part'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'precedes'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'exacerbates'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'increases_response_to'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_output'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'similar_to'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'overlaps'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'affects_response_to'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'causes'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'has_phenotype'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'located_in'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'colocalizes_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'produces'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'prevents'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'enables'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'predisposes'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'occurs_in'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'close_match'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'manifestation_of'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'coexists_with'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'diagnoses'} -> BiologicalProcess\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> BiologicalProcess\n",
      "BiologicalProcess - {'correlated_with'} -> Publication\n",
      "BiologicalProcess - {'disrupts'} -> Publication\n",
      "BiologicalProcess - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalProcess - {'regulates'} -> Publication\n",
      "BiologicalProcess - {'biomarker_for'} -> Publication\n",
      "BiologicalProcess - {'has_part'} -> Publication\n",
      "BiologicalProcess - {'located_in'} -> Publication\n",
      "BiologicalProcess - {'predisposes'} -> Publication\n",
      "BiologicalProcess - {'precedes'} -> Publication\n",
      "BiologicalProcess - {'prevents'} -> Publication\n",
      "BiologicalProcess - {'causes'} -> Publication\n",
      "BiologicalProcess - {'close_match'} -> Publication\n",
      "BiologicalProcess - {'associated_with'} -> Publication\n",
      "BiologicalProcess - {'produces'} -> Publication\n",
      "BiologicalProcess - {'has_participant'} -> Publication\n",
      "BiologicalProcess - {'subclass_of'} -> Publication\n",
      "BiologicalProcess - {'related_to'} -> Publication\n",
      "BiologicalProcess - {'affects'} -> Publication\n",
      "BiologicalProcess - {'treats'} -> Publication\n",
      "BiologicalProcess - {'manifestation_of'} -> Publication\n",
      "BiologicalProcess - {'has_input'} -> Publication\n",
      "BiologicalProcess - {'has_output'} -> Publication\n",
      "BiologicalProcess - {'occurs_in'} -> Publication\n",
      "BiologicalProcess - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->CellularComponent->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> CellularComponent\n",
      "PhenotypicFeature - {'related_to'} -> CellularComponent\n",
      "PhenotypicFeature - {'affects'} -> CellularComponent\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'expressed_in'} -> CellularComponent\n",
      "PhenotypicFeature - {'associated_with'} -> CellularComponent\n",
      "PhenotypicFeature - {'similar_to'} -> CellularComponent\n",
      "PhenotypicFeature - {'precedes'} -> CellularComponent\n",
      "PhenotypicFeature - {'subclass_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'overlaps'} -> CellularComponent\n",
      "PhenotypicFeature - {'biomarker_for'} -> CellularComponent\n",
      "PhenotypicFeature - {'affects_response_to'} -> CellularComponent\n",
      "PhenotypicFeature - {'genetic_association'} -> CellularComponent\n",
      "PhenotypicFeature - {'located_in'} -> CellularComponent\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> CellularComponent\n",
      "PhenotypicFeature - {'exacerbates'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> CellularComponent\n",
      "PhenotypicFeature - {'increases_response_to'} -> CellularComponent\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'temporally_related_to'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_output'} -> CellularComponent\n",
      "PhenotypicFeature - {'disrupts'} -> CellularComponent\n",
      "PhenotypicFeature - {'affected_by'} -> CellularComponent\n",
      "PhenotypicFeature - {'treats'} -> CellularComponent\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> CellularComponent\n",
      "PhenotypicFeature - {'causes'} -> CellularComponent\n",
      "PhenotypicFeature - {'participates_in'} -> CellularComponent\n",
      "PhenotypicFeature - {'contributes_to'} -> CellularComponent\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_input'} -> CellularComponent\n",
      "PhenotypicFeature - {'preceded_by'} -> CellularComponent\n",
      "PhenotypicFeature - {'enabled_by'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_participant'} -> CellularComponent\n",
      "PhenotypicFeature - {'superclass_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'decreases_response_to'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_part'} -> CellularComponent\n",
      "PhenotypicFeature - {'correlated_with'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_substrate'} -> CellularComponent\n",
      "PhenotypicFeature - {'interacts_with'} -> CellularComponent\n",
      "PhenotypicFeature - {'ameliorates'} -> CellularComponent\n",
      "PhenotypicFeature - {'regulates'} -> CellularComponent\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> CellularComponent\n",
      "PhenotypicFeature - {'part_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_capability'} -> CellularComponent\n",
      "PhenotypicFeature - {'caused_by'} -> CellularComponent\n",
      "PhenotypicFeature - {'actively_involves'} -> CellularComponent\n",
      "PhenotypicFeature - {'has_catalyst'} -> CellularComponent\n",
      "PhenotypicFeature - {'disease_has_location'} -> CellularComponent\n",
      "PhenotypicFeature - {'actively_involved_in'} -> CellularComponent\n",
      "PhenotypicFeature - {'produces'} -> CellularComponent\n",
      "PhenotypicFeature - {'prevents'} -> CellularComponent\n",
      "PhenotypicFeature - {'colocalizes_with'} -> CellularComponent\n",
      "PhenotypicFeature - {'predisposes'} -> CellularComponent\n",
      "PhenotypicFeature - {'occurs_in'} -> CellularComponent\n",
      "PhenotypicFeature - {'manifestation_of'} -> CellularComponent\n",
      "PhenotypicFeature - {'coexists_with'} -> CellularComponent\n",
      "PhenotypicFeature - {'close_match'} -> CellularComponent\n",
      "PhenotypicFeature - {'diagnoses'} -> CellularComponent\n",
      "CellularComponent - {'subclass_of'} -> Publication\n",
      "CellularComponent - {'produces'} -> Publication\n",
      "CellularComponent - {'has_part'} -> Publication\n",
      "CellularComponent - {'related_to'} -> Publication\n",
      "CellularComponent - {'occurs_in'} -> Publication\n",
      "CellularComponent - {'affects'} -> Publication\n",
      "CellularComponent - {'regulates'} -> Publication\n",
      "CellularComponent - {'disrupts'} -> Publication\n",
      "CellularComponent - {'located_in'} -> Publication\n",
      "CellularComponent - {'associated_with'} -> Publication\n",
      "CellularComponent - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ChemicalMixture->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'related_to'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_real_world_evidence_of_association_with'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'affects'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'affected_by'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'treats'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'causes'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'participates_in'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'contributes_to'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_input'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'preceded_by'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'enabled_by'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_participant'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'decreases_response_to'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_substrate'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'interacts_with'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'ameliorates'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'regulates'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_capability'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'caused_by'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'actively_involves'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_catalyst'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'associated_with'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'precedes'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'subclass_of'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'overlaps'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'affects_response_to'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'exacerbates'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'increases_response_to'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'temporally_related_to'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'has_output'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'disrupts'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'similar_to'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'produces'} -> ChemicalMixture\n",
      "PhenotypicFeature - {'coexists_with'} -> ChemicalMixture\n",
      "ChemicalMixture - {'related_to'} -> Publication\n",
      "ChemicalMixture - {'affects'} -> Publication\n",
      "ChemicalMixture - {'subclass_of'} -> Publication\n",
      "ChemicalMixture - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ComplexMolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'affected_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'has_biomarker'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'treated_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'contribution_from'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'correlated_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'has_real_world_evidence_of_association_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'affects'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'similar_to'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'related_to'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'overlaps'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'affects_response_to'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'response_decreased_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'interacts_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'response_increased_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'participates_in'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'has_participant'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'decreases_response_to'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'associated_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'disrupted_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'increases_response_to'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'regulated_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'response_affected_by'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'regulates'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'close_match'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'has_input'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'coexists_with'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'subclass_of'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'has_part'} -> ComplexMolecularMixture\n",
      "PhenotypicFeature - {'produces'} -> ComplexMolecularMixture\n",
      "ComplexMolecularMixture - {'correlated_with'} -> Publication\n",
      "ComplexMolecularMixture - {'subclass_of'} -> Publication\n",
      "ComplexMolecularMixture - {'treats'} -> Publication\n",
      "ComplexMolecularMixture - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->GrossAnatomicalStructure->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'part_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'located_in'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'related_to'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'affects'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'exacerbates'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'increases_response_to'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'temporally_related_to'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_output'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'disrupts'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'similar_to'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'affected_by'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'treats'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'causes'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'participates_in'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'contributes_to'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_input'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'preceded_by'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'enabled_by'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_participant'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'superclass_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'decreases_response_to'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_part'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_substrate'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'interacts_with'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'ameliorates'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'regulates'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_capability'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'caused_by'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'actively_involves'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_catalyst'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'expressed_in'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'associated_with'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'precedes'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'subclass_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'overlaps'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'affects_response_to'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'derives_from'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'disease_has_location'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'actively_involved_in'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'has_phenotype'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'biomarker_for'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'produces'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'prevents'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'predisposes'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'occurs_in'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'close_match'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'manifestation_of'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'coexists_with'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'develops_from'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'diagnoses'} -> GrossAnatomicalStructure\n",
      "PhenotypicFeature - {'disease_has_basis_in'} -> GrossAnatomicalStructure\n",
      "GrossAnatomicalStructure - {'affects'} -> Publication\n",
      "GrossAnatomicalStructure - {'coexists_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'close_match'} -> Publication\n",
      "GrossAnatomicalStructure - {'develops_from'} -> Publication\n",
      "GrossAnatomicalStructure - {'regulates'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_input'} -> Publication\n",
      "GrossAnatomicalStructure - {'associated_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'located_in'} -> Publication\n",
      "GrossAnatomicalStructure - {'subclass_of'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_part'} -> Publication\n",
      "GrossAnatomicalStructure - {'produces'} -> Publication\n",
      "GrossAnatomicalStructure - {'related_to'} -> Publication\n",
      "GrossAnatomicalStructure - {'occurs_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->MolecularActivity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'affects'} -> MolecularActivity\n",
      "PhenotypicFeature - {'preceded_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'related_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_input'} -> MolecularActivity\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> MolecularActivity\n",
      "PhenotypicFeature - {'enabled_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_participant'} -> MolecularActivity\n",
      "PhenotypicFeature - {'response_decreased_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_part'} -> MolecularActivity\n",
      "PhenotypicFeature - {'correlated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_substrate'} -> MolecularActivity\n",
      "PhenotypicFeature - {'interacts_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'ameliorates'} -> MolecularActivity\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'regulates'} -> MolecularActivity\n",
      "PhenotypicFeature - {'response_increased_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'part_of'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_capability'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_biomarker'} -> MolecularActivity\n",
      "PhenotypicFeature - {'participates_in'} -> MolecularActivity\n",
      "PhenotypicFeature - {'contributes_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'caused_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'actively_involves'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_catalyst'} -> MolecularActivity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'associated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'precedes'} -> MolecularActivity\n",
      "PhenotypicFeature - {'overlaps'} -> MolecularActivity\n",
      "PhenotypicFeature - {'biomarker_for'} -> MolecularActivity\n",
      "PhenotypicFeature - {'affects_response_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'disrupted_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> MolecularActivity\n",
      "PhenotypicFeature - {'treated_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'exacerbates'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> MolecularActivity\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'increases_response_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> MolecularActivity\n",
      "PhenotypicFeature - {'regulated_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'has_output'} -> MolecularActivity\n",
      "PhenotypicFeature - {'response_affected_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'coexpressed_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'coexists_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'disrupts'} -> MolecularActivity\n",
      "PhenotypicFeature - {'similar_to'} -> MolecularActivity\n",
      "PhenotypicFeature - {'affected_by'} -> MolecularActivity\n",
      "PhenotypicFeature - {'treats'} -> MolecularActivity\n",
      "PhenotypicFeature - {'causes'} -> MolecularActivity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> MolecularActivity\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> MolecularActivity\n",
      "PhenotypicFeature - {'occurs_in'} -> MolecularActivity\n",
      "PhenotypicFeature - {'close_match'} -> MolecularActivity\n",
      "PhenotypicFeature - {'manifestation_of'} -> MolecularActivity\n",
      "PhenotypicFeature - {'subclass_of'} -> MolecularActivity\n",
      "PhenotypicFeature - {'catalyzes'} -> MolecularActivity\n",
      "PhenotypicFeature - {'located_in'} -> MolecularActivity\n",
      "PhenotypicFeature - {'produces'} -> MolecularActivity\n",
      "PhenotypicFeature - {'enables'} -> MolecularActivity\n",
      "MolecularActivity - {'occurs_in'} -> Publication\n",
      "MolecularActivity - {'treats'} -> Publication\n",
      "MolecularActivity - {'has_input'} -> Publication\n",
      "MolecularActivity - {'has_output'} -> Publication\n",
      "MolecularActivity - {'coexists_with'} -> Publication\n",
      "MolecularActivity - {'regulates'} -> Publication\n",
      "MolecularActivity - {'disrupts'} -> Publication\n",
      "MolecularActivity - {'associated_with'} -> Publication\n",
      "MolecularActivity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularActivity - {'located_in'} -> Publication\n",
      "MolecularActivity - {'produces'} -> Publication\n",
      "MolecularActivity - {'related_to'} -> Publication\n",
      "MolecularActivity - {'affects'} -> Publication\n",
      "MolecularActivity - {'has_participant'} -> Publication\n",
      "MolecularActivity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->MolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'treated_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'subclass_of'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_biomarker'} -> MolecularMixture\n",
      "PhenotypicFeature - {'caused_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_contraindication'} -> MolecularMixture\n",
      "PhenotypicFeature - {'contribution_from'} -> MolecularMixture\n",
      "PhenotypicFeature - {'affected_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> MolecularMixture\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'correlated_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'related_to'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_real_world_evidence_of_association_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'affects'} -> MolecularMixture\n",
      "PhenotypicFeature - {'interacts_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'treats'} -> MolecularMixture\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> MolecularMixture\n",
      "PhenotypicFeature - {'regulates'} -> MolecularMixture\n",
      "PhenotypicFeature - {'causes'} -> MolecularMixture\n",
      "PhenotypicFeature - {'response_increased_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_capability'} -> MolecularMixture\n",
      "PhenotypicFeature - {'participates_in'} -> MolecularMixture\n",
      "PhenotypicFeature - {'contributes_to'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_input'} -> MolecularMixture\n",
      "PhenotypicFeature - {'preceded_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'enabled_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_participant'} -> MolecularMixture\n",
      "PhenotypicFeature - {'decreases_response_to'} -> MolecularMixture\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_substrate'} -> MolecularMixture\n",
      "PhenotypicFeature - {'precedes'} -> MolecularMixture\n",
      "PhenotypicFeature - {'ameliorates'} -> MolecularMixture\n",
      "PhenotypicFeature - {'actively_involves'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_catalyst'} -> MolecularMixture\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> MolecularMixture\n",
      "PhenotypicFeature - {'temporally_related_to'} -> MolecularMixture\n",
      "PhenotypicFeature - {'response_affected_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'coexpressed_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'associated_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'similar_to'} -> MolecularMixture\n",
      "PhenotypicFeature - {'overlaps'} -> MolecularMixture\n",
      "PhenotypicFeature - {'biomarker_for'} -> MolecularMixture\n",
      "PhenotypicFeature - {'affects_response_to'} -> MolecularMixture\n",
      "PhenotypicFeature - {'genetic_association'} -> MolecularMixture\n",
      "PhenotypicFeature - {'disrupted_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> MolecularMixture\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> MolecularMixture\n",
      "PhenotypicFeature - {'exacerbates'} -> MolecularMixture\n",
      "PhenotypicFeature - {'increases_response_to'} -> MolecularMixture\n",
      "PhenotypicFeature - {'regulated_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_output'} -> MolecularMixture\n",
      "PhenotypicFeature - {'response_decreased_by'} -> MolecularMixture\n",
      "PhenotypicFeature - {'disrupts'} -> MolecularMixture\n",
      "PhenotypicFeature - {'produces'} -> MolecularMixture\n",
      "PhenotypicFeature - {'located_in'} -> MolecularMixture\n",
      "PhenotypicFeature - {'occurs_in'} -> MolecularMixture\n",
      "PhenotypicFeature - {'close_match'} -> MolecularMixture\n",
      "PhenotypicFeature - {'coexists_with'} -> MolecularMixture\n",
      "PhenotypicFeature - {'has_part'} -> MolecularMixture\n",
      "MolecularMixture - {'correlated_with'} -> Publication\n",
      "MolecularMixture - {'treats'} -> Publication\n",
      "MolecularMixture - {'subclass_of'} -> Publication\n",
      "MolecularMixture - {'located_in'} -> Publication\n",
      "MolecularMixture - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->NamedThing->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> NamedThing\n",
      "PhenotypicFeature - {'subclass_of'} -> NamedThing\n",
      "PhenotypicFeature - {'caused_by'} -> NamedThing\n",
      "PhenotypicFeature - {'has_participant'} -> NamedThing\n",
      "PhenotypicFeature - {'has_part'} -> NamedThing\n",
      "PhenotypicFeature - {'overlaps'} -> NamedThing\n",
      "PhenotypicFeature - {'part_of'} -> NamedThing\n",
      "PhenotypicFeature - {'preceded_by'} -> NamedThing\n",
      "PhenotypicFeature - {'related_to'} -> NamedThing\n",
      "PhenotypicFeature - {'temporally_related_to'} -> NamedThing\n",
      "PhenotypicFeature - {'enabled_by'} -> NamedThing\n",
      "PhenotypicFeature - {'has_catalyst'} -> NamedThing\n",
      "PhenotypicFeature - {'decreases_response_to'} -> NamedThing\n",
      "PhenotypicFeature - {'associated_with'} -> NamedThing\n",
      "PhenotypicFeature - {'precedes'} -> NamedThing\n",
      "PhenotypicFeature - {'actively_involves'} -> NamedThing\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> NamedThing\n",
      "PhenotypicFeature - {'exacerbates'} -> NamedThing\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> NamedThing\n",
      "PhenotypicFeature - {'increases_response_to'} -> NamedThing\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> NamedThing\n",
      "PhenotypicFeature - {'expressed_in'} -> NamedThing\n",
      "PhenotypicFeature - {'has_output'} -> NamedThing\n",
      "PhenotypicFeature - {'affects'} -> NamedThing\n",
      "PhenotypicFeature - {'similar_to'} -> NamedThing\n",
      "PhenotypicFeature - {'affects_response_to'} -> NamedThing\n",
      "PhenotypicFeature - {'located_in'} -> NamedThing\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> NamedThing\n",
      "PhenotypicFeature - {'disrupts'} -> NamedThing\n",
      "PhenotypicFeature - {'affected_by'} -> NamedThing\n",
      "PhenotypicFeature - {'has_substrate'} -> NamedThing\n",
      "PhenotypicFeature - {'interacts_with'} -> NamedThing\n",
      "PhenotypicFeature - {'ameliorates'} -> NamedThing\n",
      "PhenotypicFeature - {'treats'} -> NamedThing\n",
      "PhenotypicFeature - {'regulates'} -> NamedThing\n",
      "PhenotypicFeature - {'has_capability'} -> NamedThing\n",
      "PhenotypicFeature - {'participates_in'} -> NamedThing\n",
      "PhenotypicFeature - {'has_input'} -> NamedThing\n",
      "PhenotypicFeature - {'prevents'} -> NamedThing\n",
      "PhenotypicFeature - {'colocalizes_with'} -> NamedThing\n",
      "PhenotypicFeature - {'contributes_to'} -> NamedThing\n",
      "PhenotypicFeature - {'produces'} -> NamedThing\n",
      "PhenotypicFeature - {'predisposes'} -> NamedThing\n",
      "PhenotypicFeature - {'occurs_in'} -> NamedThing\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> NamedThing\n",
      "PhenotypicFeature - {'coexists_with'} -> NamedThing\n",
      "PhenotypicFeature - {'close_match'} -> NamedThing\n",
      "PhenotypicFeature - {'manifestation_of'} -> NamedThing\n",
      "PhenotypicFeature - {'contraindicated_for'} -> NamedThing\n",
      "PhenotypicFeature - {'diagnoses'} -> NamedThing\n",
      "PhenotypicFeature - {'derives_from'} -> NamedThing\n",
      "PhenotypicFeature - {'causes'} -> NamedThing\n",
      "PhenotypicFeature - {'in_taxon'} -> NamedThing\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> NamedThing\n",
      "PhenotypicFeature - {'disease_has_location'} -> NamedThing\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> NamedThing\n",
      "PhenotypicFeature - {'actively_involved_in'} -> NamedThing\n",
      "PhenotypicFeature - {'correlated_with'} -> NamedThing\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> NamedThing\n",
      "PhenotypicFeature - {'has_phenotype'} -> NamedThing\n",
      "PhenotypicFeature - {'biomarker_for'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Publication\n",
      "NamedThing - {'related_to'} -> Publication\n",
      "NamedThing - {'is_sequence_variant_of'} -> Publication\n",
      "NamedThing - {'associated_with'} -> Publication\n",
      "NamedThing - {'has_phenotype'} -> Publication\n",
      "NamedThing - {'disrupts'} -> Publication\n",
      "NamedThing - {'biomarker_for'} -> Publication\n",
      "NamedThing - {'correlated_with'} -> Publication\n",
      "NamedThing - {'gene_associated_with_condition'} -> Publication\n",
      "NamedThing - {'has_part'} -> Publication\n",
      "NamedThing - {'located_in'} -> Publication\n",
      "NamedThing - {'colocalizes_with'} -> Publication\n",
      "NamedThing - {'interacts_with'} -> Publication\n",
      "NamedThing - {'produces'} -> Publication\n",
      "NamedThing - {'prevents'} -> Publication\n",
      "NamedThing - {'diagnoses'} -> Publication\n",
      "NamedThing - {'predisposes'} -> Publication\n",
      "NamedThing - {'precedes'} -> Publication\n",
      "NamedThing - {'occurs_in'} -> Publication\n",
      "NamedThing - {'affects'} -> Publication\n",
      "NamedThing - {'close_match'} -> Publication\n",
      "NamedThing - {'manifestation_of'} -> Publication\n",
      "NamedThing - {'exacerbates'} -> Publication\n",
      "NamedThing - {'coexists_with'} -> Publication\n",
      "NamedThing - {'regulates'} -> Publication\n",
      "NamedThing - {'has_input'} -> Publication\n",
      "NamedThing - {'physically_interacts_with'} -> Publication\n",
      "NamedThing - {'treats'} -> Publication\n",
      "NamedThing - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->OrganismTaxon->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'contribution_from'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'treated_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'affected_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'has_biomarker'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'affects'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'related_to'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'subclass_of'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'response_decreased_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'disrupts'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'correlated_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'interacts_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'ameliorates'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'treats'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'regulates'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'response_increased_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'part_of'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'participates_in'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'contributes_to'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'caused_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'has_participant'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'decreases_response_to'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'associated_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'disrupted_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'exacerbates'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'increases_response_to'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'regulated_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'expressed_in'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'response_affected_by'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'similar_to'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'overlaps'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'affects_response_to'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'causes'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'located_in'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'disease_has_location'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'has_phenotype'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'biomarker_for'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'produces'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'has_part'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'prevents'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'predisposes'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'precedes'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'occurs_in'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'manifestation_of'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'coexists_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'close_match'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'diagnoses'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'has_input'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'derives_from'} -> OrganismTaxon\n",
      "PhenotypicFeature - {'in_taxon'} -> OrganismTaxon\n",
      "OrganismTaxon - {'regulates'} -> Publication\n",
      "OrganismTaxon - {'gene_associated_with_condition'} -> Publication\n",
      "OrganismTaxon - {'located_in'} -> Publication\n",
      "OrganismTaxon - {'interacts_with'} -> Publication\n",
      "OrganismTaxon - {'produces'} -> Publication\n",
      "OrganismTaxon - {'related_to'} -> Publication\n",
      "OrganismTaxon - {'has_part'} -> Publication\n",
      "OrganismTaxon - {'predisposes'} -> Publication\n",
      "OrganismTaxon - {'occurs_in'} -> Publication\n",
      "OrganismTaxon - {'affects'} -> Publication\n",
      "OrganismTaxon - {'manifestation_of'} -> Publication\n",
      "OrganismTaxon - {'exacerbates'} -> Publication\n",
      "OrganismTaxon - {'coexists_with'} -> Publication\n",
      "OrganismTaxon - {'close_match'} -> Publication\n",
      "OrganismTaxon - {'subclass_of'} -> Publication\n",
      "OrganismTaxon - {'physically_interacts_with'} -> Publication\n",
      "OrganismTaxon - {'treats'} -> Publication\n",
      "OrganismTaxon - {'has_input'} -> Publication\n",
      "OrganismTaxon - {'causes'} -> Publication\n",
      "OrganismTaxon - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Polypeptide->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'treated_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'contribution_from'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_biomarker'} -> Polypeptide\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'decreases_abundance_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'increases_abundance_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_predisposing_factor'} -> Polypeptide\n",
      "PhenotypicFeature - {'affected_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'entity_positively_regulated_by_entity'} -> Polypeptide\n",
      "PhenotypicFeature - {'associated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'prevented_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'disrupted_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'caused_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'related_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'affects'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_substrate'} -> Polypeptide\n",
      "PhenotypicFeature - {'participates_in'} -> Polypeptide\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_input'} -> Polypeptide\n",
      "PhenotypicFeature - {'preceded_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'enabled_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_participant'} -> Polypeptide\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_part'} -> Polypeptide\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'precedes'} -> Polypeptide\n",
      "PhenotypicFeature - {'interacts_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'ameliorates'} -> Polypeptide\n",
      "PhenotypicFeature - {'regulates'} -> Polypeptide\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_capability'} -> Polypeptide\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'actively_involves'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_catalyst'} -> Polypeptide\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'response_affected_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'coexpressed_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'similar_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'subclass_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'overlaps'} -> Polypeptide\n",
      "PhenotypicFeature - {'biomarker_for'} -> Polypeptide\n",
      "PhenotypicFeature - {'affects_response_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Polypeptide\n",
      "PhenotypicFeature - {'exacerbates'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Polypeptide\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'increases_response_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'regulated_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'has_output'} -> Polypeptide\n",
      "PhenotypicFeature - {'response_decreased_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'disrupts'} -> Polypeptide\n",
      "PhenotypicFeature - {'correlated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'treats'} -> Polypeptide\n",
      "PhenotypicFeature - {'response_increased_by'} -> Polypeptide\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'part_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'located_in'} -> Polypeptide\n",
      "PhenotypicFeature - {'produces'} -> Polypeptide\n",
      "PhenotypicFeature - {'colocalizes_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'contributes_to'} -> Polypeptide\n",
      "PhenotypicFeature - {'predisposes'} -> Polypeptide\n",
      "PhenotypicFeature - {'occurs_in'} -> Polypeptide\n",
      "PhenotypicFeature - {'manifestation_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> Polypeptide\n",
      "PhenotypicFeature - {'coexists_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'close_match'} -> Polypeptide\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Polypeptide\n",
      "PhenotypicFeature - {'diagnoses'} -> Polypeptide\n",
      "PhenotypicFeature - {'derives_from'} -> Polypeptide\n",
      "PhenotypicFeature - {'causes'} -> Polypeptide\n",
      "PhenotypicFeature - {'disease_has_location'} -> Polypeptide\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> Polypeptide\n",
      "Polypeptide - {'disrupts'} -> Publication\n",
      "Polypeptide - {'associated_with'} -> Publication\n",
      "Polypeptide - {'gene_associated_with_condition'} -> Publication\n",
      "Polypeptide - {'located_in'} -> Publication\n",
      "Polypeptide - {'regulates'} -> Publication\n",
      "Polypeptide - {'biomarker_for'} -> Publication\n",
      "Polypeptide - {'produces'} -> Publication\n",
      "Polypeptide - {'has_part'} -> Publication\n",
      "Polypeptide - {'physically_interacts_with'} -> Publication\n",
      "Polypeptide - {'colocalizes_with'} -> Publication\n",
      "Polypeptide - {'predisposes'} -> Publication\n",
      "Polypeptide - {'related_to'} -> Publication\n",
      "Polypeptide - {'prevents'} -> Publication\n",
      "Polypeptide - {'precedes'} -> Publication\n",
      "Polypeptide - {'close_match'} -> Publication\n",
      "Polypeptide - {'subclass_of'} -> Publication\n",
      "Polypeptide - {'interacts_with'} -> Publication\n",
      "Polypeptide - {'affects'} -> Publication\n",
      "Polypeptide - {'treats'} -> Publication\n",
      "Polypeptide - {'occurs_in'} -> Publication\n",
      "Polypeptide - {'exacerbates'} -> Publication\n",
      "Polypeptide - {'causes'} -> Publication\n",
      "Polypeptide - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->PhysiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'manifestation_of'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'correlated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'associated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'causes'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'decreases_response_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'ameliorates'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'genetic_association'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'disrupted_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'exacerbates'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'increases_response_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'regulated_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'response_affected_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'coexpressed_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'affects'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'coexists_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'similar_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'related_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'overlaps'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'biomarker_for'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'affects_response_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'treated_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'response_decreased_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'disrupts'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'affected_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'interacts_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'treats'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'regulates'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'response_increased_by'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'has_biomarker'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'participates_in'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'contributes_to'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'has_participant'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'diagnoses'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'has_input'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'gene_product_of'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'has_phenotype'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'located_in'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'has_part'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'prevents'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'enables'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'produces'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'predisposes'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'occurs_in'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'precedes'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'close_match'} -> PhysiologicalProcess\n",
      "PhenotypicFeature - {'subclass_of'} -> PhysiologicalProcess\n",
      "PhysiologicalProcess - {'correlated_with'} -> Publication\n",
      "PhysiologicalProcess - {'has_input'} -> Publication\n",
      "PhysiologicalProcess - {'causes'} -> Publication\n",
      "PhysiologicalProcess - {'has_output'} -> Publication\n",
      "PhysiologicalProcess - {'disrupts'} -> Publication\n",
      "PhysiologicalProcess - {'associated_with'} -> Publication\n",
      "PhysiologicalProcess - {'located_in'} -> Publication\n",
      "PhysiologicalProcess - {'produces'} -> Publication\n",
      "PhysiologicalProcess - {'predisposes'} -> Publication\n",
      "PhysiologicalProcess - {'precedes'} -> Publication\n",
      "PhysiologicalProcess - {'related_to'} -> Publication\n",
      "PhysiologicalProcess - {'occurs_in'} -> Publication\n",
      "PhysiologicalProcess - {'affects'} -> Publication\n",
      "PhysiologicalProcess - {'coexists_with'} -> Publication\n",
      "PhysiologicalProcess - {'close_match'} -> Publication\n",
      "PhysiologicalProcess - {'has_participant'} -> Publication\n",
      "PhysiologicalProcess - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->PathologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'manifestation_of'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'has_predisposing_factor'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'coexists_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'caused_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'treated_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'exacerbates'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'increases_response_to'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'regulated_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'response_affected_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'coexpressed_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'affects'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'related_to'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'affects_response_to'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'treats'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'causes'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'response_decreased_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'disrupts'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'correlated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'affected_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'predisposes'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'ameliorates'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'regulates'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'response_increased_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'has_biomarker'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'contributes_to'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'decreases_response_to'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'associated_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'biomarker_for'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'disrupted_by'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'occurs_in'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'precedes'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'close_match'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'has_participant'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'subclass_of'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'contraindicated_for'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'diagnoses'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'has_input'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'has_phenotype'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'located_in'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'interacts_with'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'produces'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'has_part'} -> PathologicalProcess\n",
      "PhenotypicFeature - {'prevents'} -> PathologicalProcess\n",
      "PathologicalProcess - {'manifestation_of'} -> Publication\n",
      "PathologicalProcess - {'occurs_in'} -> Publication\n",
      "PathologicalProcess - {'coexists_with'} -> Publication\n",
      "PathologicalProcess - {'exacerbates'} -> Publication\n",
      "PathologicalProcess - {'disrupts'} -> Publication\n",
      "PathologicalProcess - {'located_in'} -> Publication\n",
      "PathologicalProcess - {'regulates'} -> Publication\n",
      "PathologicalProcess - {'treats'} -> Publication\n",
      "PathologicalProcess - {'precedes'} -> Publication\n",
      "PathologicalProcess - {'causes'} -> Publication\n",
      "PathologicalProcess - {'close_match'} -> Publication\n",
      "PathologicalProcess - {'associated_with'} -> Publication\n",
      "PathologicalProcess - {'subclass_of'} -> Publication\n",
      "PathologicalProcess - {'produces'} -> Publication\n",
      "PathologicalProcess - {'predisposes'} -> Publication\n",
      "PathologicalProcess - {'related_to'} -> Publication\n",
      "PathologicalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->GeneticInheritance->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'subclass_of'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'superclass_of'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'regulates'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'manifestation_of'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'close_match'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'causes'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'has_input'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'precedes'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'affects'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'associated_with'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'coexists_with'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'disrupts'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'related_to'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'has_part'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'diagnoses'} -> GeneticInheritance\n",
      "PhenotypicFeature - {'located_in'} -> GeneticInheritance\n",
      "GeneticInheritance - {'occurs_in'} -> Publication\n",
      "GeneticInheritance - {'has_input'} -> Publication\n",
      "GeneticInheritance - {'located_in'} -> Publication\n",
      "GeneticInheritance - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->DiseaseOrPhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'subclass_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_part'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'overlaps'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'part_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'related_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'affects_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'associated_with_sensitivity_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'preceded_by'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'associated_with_increased_likelihood_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'disrupts'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'resistance_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'affected_by'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'ameliorates'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'treats'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'regulates'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'sensitivity_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'positively_correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_capability'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_biomarker'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'participates_in'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'decreased_likelihood_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'actively_involves'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_participant'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'decreases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'precedes'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'biomarker_for'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'increased_likelihood_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'associated_with_decreased_likelihood_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_together_in_literature_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'exacerbates'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'increases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'temporally_related_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'associated_with_resistance_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'coexpressed_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'negatively_correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'affects'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'similar_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'disease_has_location'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'actively_involved_in'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'gene_product_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_phenotype'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'located_in'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'produces'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'prevents'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'contributes_to'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'predisposes'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'occurs_in'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'manifestation_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'coexists_with'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'close_match'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'contraindicated_for'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'diagnoses'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'has_input'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'derives_from'} -> DiseaseOrPhenotypicFeature\n",
      "PhenotypicFeature - {'causes'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->BiologicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'subclass_of'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'caused_by'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_part'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'overlaps'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'part_of'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'preceded_by'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'related_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'treats'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'regulates'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_capability'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'participates_in'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_input'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'enabled_by'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_participant'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_catalyst'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'associated_with'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'precedes'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'genetic_association'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'actively_involves'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'increases_response_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'expressed_in'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_output'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'affects'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'similar_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'affects_response_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'located_in'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'disrupts'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'affected_by'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_substrate'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'ameliorates'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'colocalizes_with'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'contributes_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'produces'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'predisposes'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'prevents'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'close_match'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'manifestation_of'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'coexists_with'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'chemically_similar_to'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'contraindicated_for'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'diagnoses'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'disease_has_basis_in'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'derives_from'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'causes'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'in_taxon'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'disease_has_location'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'gene_product_of'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'has_phenotype'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'biomarker_for'} -> BiologicalEntity\n",
      "PhenotypicFeature - {'correlated_with'} -> BiologicalEntity\n",
      "BiologicalEntity - {'physically_interacts_with'} -> Publication\n",
      "BiologicalEntity - {'treats'} -> Publication\n",
      "BiologicalEntity - {'has_input'} -> Publication\n",
      "BiologicalEntity - {'causes'} -> Publication\n",
      "BiologicalEntity - {'disrupts'} -> Publication\n",
      "BiologicalEntity - {'is_sequence_variant_of'} -> Publication\n",
      "BiologicalEntity - {'has_phenotype'} -> Publication\n",
      "BiologicalEntity - {'biomarker_for'} -> Publication\n",
      "BiologicalEntity - {'associated_with'} -> Publication\n",
      "BiologicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalEntity - {'has_part'} -> Publication\n",
      "BiologicalEntity - {'located_in'} -> Publication\n",
      "BiologicalEntity - {'colocalizes_with'} -> Publication\n",
      "BiologicalEntity - {'interacts_with'} -> Publication\n",
      "BiologicalEntity - {'produces'} -> Publication\n",
      "BiologicalEntity - {'predisposes'} -> Publication\n",
      "BiologicalEntity - {'related_to'} -> Publication\n",
      "BiologicalEntity - {'prevents'} -> Publication\n",
      "BiologicalEntity - {'manifestation_of'} -> Publication\n",
      "BiologicalEntity - {'precedes'} -> Publication\n",
      "BiologicalEntity - {'occurs_in'} -> Publication\n",
      "BiologicalEntity - {'affects'} -> Publication\n",
      "BiologicalEntity - {'close_match'} -> Publication\n",
      "BiologicalEntity - {'exacerbates'} -> Publication\n",
      "BiologicalEntity - {'coexists_with'} -> Publication\n",
      "BiologicalEntity - {'has_participant'} -> Publication\n",
      "BiologicalEntity - {'subclass_of'} -> Publication\n",
      "BiologicalEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Entity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'subclass_of'} -> Entity\n",
      "PhenotypicFeature - {'caused_by'} -> Entity\n",
      "PhenotypicFeature - {'has_part'} -> Entity\n",
      "PhenotypicFeature - {'has_participant'} -> Entity\n",
      "PhenotypicFeature - {'overlaps'} -> Entity\n",
      "PhenotypicFeature - {'part_of'} -> Entity\n",
      "PhenotypicFeature - {'preceded_by'} -> Entity\n",
      "PhenotypicFeature - {'related_to'} -> Entity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Entity\n",
      "PhenotypicFeature - {'interacts_with'} -> Entity\n",
      "PhenotypicFeature - {'ameliorates'} -> Entity\n",
      "PhenotypicFeature - {'treats'} -> Entity\n",
      "PhenotypicFeature - {'regulates'} -> Entity\n",
      "PhenotypicFeature - {'has_capability'} -> Entity\n",
      "PhenotypicFeature - {'participates_in'} -> Entity\n",
      "PhenotypicFeature - {'has_input'} -> Entity\n",
      "PhenotypicFeature - {'enabled_by'} -> Entity\n",
      "PhenotypicFeature - {'has_catalyst'} -> Entity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Entity\n",
      "PhenotypicFeature - {'associated_with'} -> Entity\n",
      "PhenotypicFeature - {'precedes'} -> Entity\n",
      "PhenotypicFeature - {'actively_involves'} -> Entity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Entity\n",
      "PhenotypicFeature - {'exacerbates'} -> Entity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Entity\n",
      "PhenotypicFeature - {'increases_response_to'} -> Entity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Entity\n",
      "PhenotypicFeature - {'expressed_in'} -> Entity\n",
      "PhenotypicFeature - {'has_output'} -> Entity\n",
      "PhenotypicFeature - {'affects'} -> Entity\n",
      "PhenotypicFeature - {'similar_to'} -> Entity\n",
      "PhenotypicFeature - {'affects_response_to'} -> Entity\n",
      "PhenotypicFeature - {'located_in'} -> Entity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Entity\n",
      "PhenotypicFeature - {'disrupts'} -> Entity\n",
      "PhenotypicFeature - {'affected_by'} -> Entity\n",
      "PhenotypicFeature - {'has_substrate'} -> Entity\n",
      "Entity - {'subclass_of'} -> Publication\n",
      "Entity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->PopulationOfIndividualOrganisms->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'subclass_of'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'diagnoses'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'treats'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'has_input'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'associated_with'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'located_in'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'produces'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'interacts_with'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'predisposes'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'related_to'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'occurs_in'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'affects'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'coexists_with'} -> PopulationOfIndividualOrganisms\n",
      "PhenotypicFeature - {'close_match'} -> PopulationOfIndividualOrganisms\n",
      "PopulationOfIndividualOrganisms - {'correlated_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_part'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'produces'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'related_to'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'occurs_in'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'affects'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'close_match'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'subclass_of'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'interacts_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'treats'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_input'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ChemicalExposure->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> ChemicalExposure\n",
      "ChemicalExposure - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->EnvironmentalExposure->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> EnvironmentalExposure\n",
      "EnvironmentalExposure - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->InformationContentEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'associated_with'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'affects'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'similar_to'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'related_to'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'overlaps'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'affected_by'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'participates_in'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'has_participant'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'disease_has_location'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'actively_involved_in'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'has_phenotype'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'located_in'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'biomarker_for'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'produces'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'has_part'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'prevents'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'contributes_to'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'predisposes'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'precedes'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'manifestation_of'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'coexists_with'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'close_match'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'regulates'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'subclass_of'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'contraindicated_for'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'diagnoses'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'treats'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'has_input'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'derives_from'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'causes'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> InformationContentEntity\n",
      "PhenotypicFeature - {'disrupts'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Publication\n",
      "InformationContentEntity - {'subclass_of'} -> Publication\n",
      "InformationContentEntity - {'related_to'} -> Publication\n",
      "InformationContentEntity - {'disrupts'} -> Publication\n",
      "InformationContentEntity - {'associated_with'} -> Publication\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Publication\n",
      "InformationContentEntity - {'regulates'} -> Publication\n",
      "InformationContentEntity - {'biomarker_for'} -> Publication\n",
      "InformationContentEntity - {'produces'} -> Publication\n",
      "InformationContentEntity - {'has_part'} -> Publication\n",
      "InformationContentEntity - {'located_in'} -> Publication\n",
      "InformationContentEntity - {'predisposes'} -> Publication\n",
      "InformationContentEntity - {'prevents'} -> Publication\n",
      "InformationContentEntity - {'affects'} -> Publication\n",
      "InformationContentEntity - {'manifestation_of'} -> Publication\n",
      "InformationContentEntity - {'precedes'} -> Publication\n",
      "InformationContentEntity - {'close_match'} -> Publication\n",
      "InformationContentEntity - {'interacts_with'} -> Publication\n",
      "InformationContentEntity - {'diagnoses'} -> Publication\n",
      "InformationContentEntity - {'treats'} -> Publication\n",
      "InformationContentEntity - {'occurs_in'} -> Publication\n",
      "InformationContentEntity - {'has_input'} -> Publication\n",
      "InformationContentEntity - {'exacerbates'} -> Publication\n",
      "InformationContentEntity - {'causes'} -> Publication\n",
      "InformationContentEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ClinicalIntervention->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'causes'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'disrupts'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'associated_with'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'has_phenotype'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'biomarker_for'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'has_part'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'located_in'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'interacts_with'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'produces'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'predisposes'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'related_to'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'prevents'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'occurs_in'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'precedes'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'affects'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'close_match'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'has_participant'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'manifestation_of'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'exacerbates'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'coexists_with'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'subclass_of'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'regulates'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'diagnoses'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'treats'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'has_input'} -> ClinicalIntervention\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> ClinicalIntervention\n",
      "ClinicalIntervention - {'correlated_with'} -> Publication\n",
      "ClinicalIntervention - {'regulates'} -> Publication\n",
      "ClinicalIntervention - {'diagnoses'} -> Publication\n",
      "ClinicalIntervention - {'treats'} -> Publication\n",
      "ClinicalIntervention - {'has_input'} -> Publication\n",
      "ClinicalIntervention - {'causes'} -> Publication\n",
      "ClinicalIntervention - {'disrupts'} -> Publication\n",
      "ClinicalIntervention - {'associated_with'} -> Publication\n",
      "ClinicalIntervention - {'has_part'} -> Publication\n",
      "ClinicalIntervention - {'located_in'} -> Publication\n",
      "ClinicalIntervention - {'predisposes'} -> Publication\n",
      "ClinicalIntervention - {'related_to'} -> Publication\n",
      "ClinicalIntervention - {'prevents'} -> Publication\n",
      "ClinicalIntervention - {'manifestation_of'} -> Publication\n",
      "ClinicalIntervention - {'precedes'} -> Publication\n",
      "ClinicalIntervention - {'occurs_in'} -> Publication\n",
      "ClinicalIntervention - {'affects'} -> Publication\n",
      "ClinicalIntervention - {'close_match'} -> Publication\n",
      "ClinicalIntervention - {'exacerbates'} -> Publication\n",
      "ClinicalIntervention - {'coexists_with'} -> Publication\n",
      "ClinicalIntervention - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Device->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Device\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Device\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Device\n",
      "PhenotypicFeature - {'affects'} -> Device\n",
      "PhenotypicFeature - {'similar_to'} -> Device\n",
      "PhenotypicFeature - {'related_to'} -> Device\n",
      "PhenotypicFeature - {'overlaps'} -> Device\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Device\n",
      "PhenotypicFeature - {'affected_by'} -> Device\n",
      "PhenotypicFeature - {'interacts_with'} -> Device\n",
      "PhenotypicFeature - {'participates_in'} -> Device\n",
      "PhenotypicFeature - {'has_participant'} -> Device\n",
      "PhenotypicFeature - {'associated_with'} -> Device\n",
      "PhenotypicFeature - {'diagnoses'} -> Device\n",
      "PhenotypicFeature - {'treats'} -> Device\n",
      "PhenotypicFeature - {'has_input'} -> Device\n",
      "PhenotypicFeature - {'derives_from'} -> Device\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Device\n",
      "PhenotypicFeature - {'causes'} -> Device\n",
      "PhenotypicFeature - {'disease_has_location'} -> Device\n",
      "PhenotypicFeature - {'disrupts'} -> Device\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Device\n",
      "PhenotypicFeature - {'has_phenotype'} -> Device\n",
      "PhenotypicFeature - {'located_in'} -> Device\n",
      "PhenotypicFeature - {'has_part'} -> Device\n",
      "PhenotypicFeature - {'prevents'} -> Device\n",
      "PhenotypicFeature - {'produces'} -> Device\n",
      "PhenotypicFeature - {'predisposes'} -> Device\n",
      "PhenotypicFeature - {'occurs_in'} -> Device\n",
      "PhenotypicFeature - {'exacerbates'} -> Device\n",
      "PhenotypicFeature - {'precedes'} -> Device\n",
      "PhenotypicFeature - {'close_match'} -> Device\n",
      "PhenotypicFeature - {'manifestation_of'} -> Device\n",
      "PhenotypicFeature - {'coexists_with'} -> Device\n",
      "PhenotypicFeature - {'regulates'} -> Device\n",
      "PhenotypicFeature - {'subclass_of'} -> Device\n",
      "Device - {'subclass_of'} -> Publication\n",
      "Device - {'diagnoses'} -> Publication\n",
      "Device - {'related_to'} -> Publication\n",
      "Device - {'occurs_in'} -> Publication\n",
      "Device - {'affects'} -> Publication\n",
      "Device - {'treats'} -> Publication\n",
      "Device - {'has_input'} -> Publication\n",
      "Device - {'coexists_with'} -> Publication\n",
      "Device - {'regulates'} -> Publication\n",
      "Device - {'disrupts'} -> Publication\n",
      "Device - {'biomarker_for'} -> Publication\n",
      "Device - {'gene_associated_with_condition'} -> Publication\n",
      "Device - {'has_part'} -> Publication\n",
      "Device - {'located_in'} -> Publication\n",
      "Device - {'causes'} -> Publication\n",
      "Device - {'precedes'} -> Publication\n",
      "Device - {'close_match'} -> Publication\n",
      "Device - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ClinicalAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'overlaps'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'associated_with'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'affects'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'similar_to'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'related_to'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'genetic_association'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'affected_by'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'participates_in'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'has_participant'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'interacts_with'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'disrupts'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'diagnoses'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'located_in'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'regulates'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'manifestation_of'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'close_match'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'predisposes'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'prevents'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'causes'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'precedes'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'exacerbates'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'subclass_of'} -> ClinicalAttribute\n",
      "PhenotypicFeature - {'coexists_with'} -> ClinicalAttribute\n",
      "ClinicalAttribute - {'correlated_with'} -> Publication\n",
      "ClinicalAttribute - {'has_input'} -> Publication\n",
      "ClinicalAttribute - {'subclass_of'} -> Publication\n",
      "ClinicalAttribute - {'related_to'} -> Publication\n",
      "ClinicalAttribute - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Annotation->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Annotation\n",
      "PhenotypicFeature - {'has_part'} -> Annotation\n",
      "PhenotypicFeature - {'overlaps'} -> Annotation\n",
      "PhenotypicFeature - {'part_of'} -> Annotation\n",
      "PhenotypicFeature - {'related_to'} -> Annotation\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Annotation\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Annotation\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Annotation\n",
      "PhenotypicFeature - {'exacerbates'} -> Annotation\n",
      "PhenotypicFeature - {'increases_response_to'} -> Annotation\n",
      "PhenotypicFeature - {'disrupts'} -> Annotation\n",
      "PhenotypicFeature - {'affected_by'} -> Annotation\n",
      "PhenotypicFeature - {'interacts_with'} -> Annotation\n",
      "PhenotypicFeature - {'treats'} -> Annotation\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> Annotation\n",
      "PhenotypicFeature - {'regulates'} -> Annotation\n",
      "PhenotypicFeature - {'has_capability'} -> Annotation\n",
      "PhenotypicFeature - {'participates_in'} -> Annotation\n",
      "PhenotypicFeature - {'preceded_by'} -> Annotation\n",
      "PhenotypicFeature - {'has_participant'} -> Annotation\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Annotation\n",
      "PhenotypicFeature - {'associated_with'} -> Annotation\n",
      "PhenotypicFeature - {'precedes'} -> Annotation\n",
      "PhenotypicFeature - {'ameliorates'} -> Annotation\n",
      "PhenotypicFeature - {'actively_involves'} -> Annotation\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Annotation\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Annotation\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Annotation\n",
      "PhenotypicFeature - {'affects'} -> Annotation\n",
      "PhenotypicFeature - {'similar_to'} -> Annotation\n",
      "PhenotypicFeature - {'affects_response_to'} -> Annotation\n",
      "PhenotypicFeature - {'genetic_association'} -> Annotation\n",
      "Annotation - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Activity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Activity\n",
      "PhenotypicFeature - {'participates_in'} -> Activity\n",
      "PhenotypicFeature - {'associated_with'} -> Activity\n",
      "PhenotypicFeature - {'overlaps'} -> Activity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Activity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Activity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Activity\n",
      "PhenotypicFeature - {'affects'} -> Activity\n",
      "PhenotypicFeature - {'similar_to'} -> Activity\n",
      "PhenotypicFeature - {'related_to'} -> Activity\n",
      "PhenotypicFeature - {'affected_by'} -> Activity\n",
      "PhenotypicFeature - {'treats'} -> Activity\n",
      "PhenotypicFeature - {'has_participant'} -> Activity\n",
      "PhenotypicFeature - {'interacts_with'} -> Activity\n",
      "PhenotypicFeature - {'ameliorates'} -> Activity\n",
      "PhenotypicFeature - {'biomarker_for'} -> Activity\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Activity\n",
      "PhenotypicFeature - {'has_part'} -> Activity\n",
      "PhenotypicFeature - {'located_in'} -> Activity\n",
      "PhenotypicFeature - {'produces'} -> Activity\n",
      "PhenotypicFeature - {'prevents'} -> Activity\n",
      "PhenotypicFeature - {'contributes_to'} -> Activity\n",
      "PhenotypicFeature - {'predisposes'} -> Activity\n",
      "PhenotypicFeature - {'precedes'} -> Activity\n",
      "PhenotypicFeature - {'occurs_in'} -> Activity\n",
      "PhenotypicFeature - {'close_match'} -> Activity\n",
      "PhenotypicFeature - {'manifestation_of'} -> Activity\n",
      "PhenotypicFeature - {'exacerbates'} -> Activity\n",
      "PhenotypicFeature - {'coexists_with'} -> Activity\n",
      "PhenotypicFeature - {'subclass_of'} -> Activity\n",
      "PhenotypicFeature - {'regulates'} -> Activity\n",
      "PhenotypicFeature - {'contraindicated_for'} -> Activity\n",
      "PhenotypicFeature - {'diagnoses'} -> Activity\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Activity\n",
      "PhenotypicFeature - {'has_input'} -> Activity\n",
      "PhenotypicFeature - {'causes'} -> Activity\n",
      "PhenotypicFeature - {'disrupts'} -> Activity\n",
      "PhenotypicFeature - {'actively_involved_in'} -> Activity\n",
      "PhenotypicFeature - {'has_phenotype'} -> Activity\n",
      "Activity - {'correlated_with'} -> Publication\n",
      "Activity - {'causes'} -> Publication\n",
      "Activity - {'disrupts'} -> Publication\n",
      "Activity - {'subclass_of'} -> Publication\n",
      "Activity - {'located_in'} -> Publication\n",
      "Activity - {'interacts_with'} -> Publication\n",
      "Activity - {'produces'} -> Publication\n",
      "Activity - {'has_part'} -> Publication\n",
      "Activity - {'diagnoses'} -> Publication\n",
      "Activity - {'prevents'} -> Publication\n",
      "Activity - {'predisposes'} -> Publication\n",
      "Activity - {'precedes'} -> Publication\n",
      "Activity - {'related_to'} -> Publication\n",
      "Activity - {'occurs_in'} -> Publication\n",
      "Activity - {'affects'} -> Publication\n",
      "Activity - {'coexists_with'} -> Publication\n",
      "Activity - {'close_match'} -> Publication\n",
      "Activity - {'regulates'} -> Publication\n",
      "Activity - {'treats'} -> Publication\n",
      "Activity - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Publication\n",
      "PhenotypicFeature - {'occurs_in'} -> Publication\n",
      "PhenotypicFeature - {'affects'} -> Publication\n",
      "PhenotypicFeature - {'close_match'} -> Publication\n",
      "PhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "PhenotypicFeature - {'exacerbates'} -> Publication\n",
      "PhenotypicFeature - {'coexists_with'} -> Publication\n",
      "PhenotypicFeature - {'subclass_of'} -> Publication\n",
      "PhenotypicFeature - {'regulates'} -> Publication\n",
      "PhenotypicFeature - {'diagnoses'} -> Publication\n",
      "PhenotypicFeature - {'treats'} -> Publication\n",
      "PhenotypicFeature - {'has_input'} -> Publication\n",
      "PhenotypicFeature - {'causes'} -> Publication\n",
      "PhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "PhenotypicFeature - {'disrupts'} -> Publication\n",
      "PhenotypicFeature - {'associated_with'} -> Publication\n",
      "PhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "PhenotypicFeature - {'has_part'} -> Publication\n",
      "PhenotypicFeature - {'located_in'} -> Publication\n",
      "PhenotypicFeature - {'interacts_with'} -> Publication\n",
      "PhenotypicFeature - {'produces'} -> Publication\n",
      "PhenotypicFeature - {'prevents'} -> Publication\n",
      "PhenotypicFeature - {'contributes_to'} -> Publication\n",
      "PhenotypicFeature - {'predisposes'} -> Publication\n",
      "PhenotypicFeature - {'precedes'} -> Publication\n",
      "PhenotypicFeature - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->GenomicEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'caused_by'} -> GenomicEntity\n",
      "PhenotypicFeature - {'related_to'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> GenomicEntity\n",
      "PhenotypicFeature - {'increases_response_to'} -> GenomicEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> GenomicEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_output'} -> GenomicEntity\n",
      "PhenotypicFeature - {'affects'} -> GenomicEntity\n",
      "PhenotypicFeature - {'disrupts'} -> GenomicEntity\n",
      "PhenotypicFeature - {'similar_to'} -> GenomicEntity\n",
      "PhenotypicFeature - {'affected_by'} -> GenomicEntity\n",
      "PhenotypicFeature - {'treats'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_input'} -> GenomicEntity\n",
      "PhenotypicFeature - {'preceded_by'} -> GenomicEntity\n",
      "PhenotypicFeature - {'enabled_by'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_participant'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_part'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_substrate'} -> GenomicEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> GenomicEntity\n",
      "PhenotypicFeature - {'ameliorates'} -> GenomicEntity\n",
      "PhenotypicFeature - {'regulates'} -> GenomicEntity\n",
      "PhenotypicFeature - {'part_of'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_capability'} -> GenomicEntity\n",
      "PhenotypicFeature - {'participates_in'} -> GenomicEntity\n",
      "PhenotypicFeature - {'actively_involves'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_catalyst'} -> GenomicEntity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> GenomicEntity\n",
      "PhenotypicFeature - {'associated_with'} -> GenomicEntity\n",
      "PhenotypicFeature - {'precedes'} -> GenomicEntity\n",
      "PhenotypicFeature - {'overlaps'} -> GenomicEntity\n",
      "PhenotypicFeature - {'affects_response_to'} -> GenomicEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> GenomicEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> GenomicEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> GenomicEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> GenomicEntity\n",
      "PhenotypicFeature - {'close_match'} -> GenomicEntity\n",
      "PhenotypicFeature - {'subclass_of'} -> GenomicEntity\n",
      "PhenotypicFeature - {'produces'} -> GenomicEntity\n",
      "PhenotypicFeature - {'located_in'} -> GenomicEntity\n",
      "GenomicEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->NucleicAcidEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'caused_by'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'related_to'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'preceded_by'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'enabled_by'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_participant'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_part'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_substrate'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'ameliorates'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'regulates'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'part_of'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_capability'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'actively_involves'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_catalyst'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'associated_with'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'affects'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'similar_to'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'precedes'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'overlaps'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'affects_response_to'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'increases_response_to'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_output'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'disrupts'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'affected_by'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'treats'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'participates_in'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'has_input'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'located_in'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'contributes_to'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'produces'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'predisposes'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'close_match'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'manifestation_of'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'coexists_with'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'subclass_of'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'diagnoses'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'derives_from'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> NucleicAcidEntity\n",
      "PhenotypicFeature - {'causes'} -> NucleicAcidEntity\n",
      "NucleicAcidEntity - {'gene_associated_with_condition'} -> Publication\n",
      "NucleicAcidEntity - {'located_in'} -> Publication\n",
      "NucleicAcidEntity - {'related_to'} -> Publication\n",
      "NucleicAcidEntity - {'close_match'} -> Publication\n",
      "NucleicAcidEntity - {'subclass_of'} -> Publication\n",
      "NucleicAcidEntity - {'interacts_with'} -> Publication\n",
      "NucleicAcidEntity - {'occurs_in'} -> Publication\n",
      "NucleicAcidEntity - {'affects'} -> Publication\n",
      "NucleicAcidEntity - {'treats'} -> Publication\n",
      "NucleicAcidEntity - {'causes'} -> Publication\n",
      "NucleicAcidEntity - {'coexists_with'} -> Publication\n",
      "NucleicAcidEntity - {'disrupts'} -> Publication\n",
      "NucleicAcidEntity - {'associated_with'} -> Publication\n",
      "NucleicAcidEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->OrganismalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'caused_by'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'related_to'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'affects_response_to'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'located_in'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'preceded_by'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'disrupts'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'affected_by'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_substrate'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'ameliorates'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'treats'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'regulates'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'part_of'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_capability'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'participates_in'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'actively_involves'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_input'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'enabled_by'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_participant'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_catalyst'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'associated_with'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_part'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'precedes'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'exacerbates'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'increases_response_to'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'expressed_in'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'has_output'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'affects'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'similar_to'} -> OrganismalEntity\n",
      "PhenotypicFeature - {'overlaps'} -> OrganismalEntity\n",
      "OrganismalEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->OrganismAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'has_part'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'overlaps'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'part_of'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'related_to'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'precedes'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'interacts_with'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'ameliorates'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'regulates'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'has_capability'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'actively_involves'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'temporally_related_to'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'associated_with'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'affects'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'similar_to'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'affects_response_to'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'genetic_association'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'exacerbates'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'increases_response_to'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'disrupts'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'affected_by'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'treats'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'participates_in'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'preceded_by'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'has_participant'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'decreases_response_to'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'located_in'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'occurs_in'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'manifestation_of'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'close_match'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'causes'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'predisposes'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'coexists_with'} -> OrganismAttribute\n",
      "PhenotypicFeature - {'subclass_of'} -> OrganismAttribute\n",
      "OrganismAttribute - {'related_to'} -> Publication\n",
      "OrganismAttribute - {'produces'} -> Publication\n",
      "OrganismAttribute - {'has_input'} -> Publication\n",
      "OrganismAttribute - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->BiologicalProcessOrActivity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'preceded_by'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'related_to'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'temporally_related_to'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_catalyst'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'affects'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'similar_to'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'overlaps'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'affects_response_to'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'exacerbates'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'increases_response_to'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_output'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'disrupts'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'affected_by'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'interacts_with'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'treats'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'regulates'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'part_of'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_capability'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'participates_in'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_input'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'enabled_by'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_participant'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'decreases_response_to'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'associated_with'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_part'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'has_substrate'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'precedes'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'ameliorates'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'caused_by'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'actively_involves'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'causes'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'enables'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'manifestation_of'} -> BiologicalProcessOrActivity\n",
      "PhenotypicFeature - {'coexists_with'} -> BiologicalProcessOrActivity\n",
      "BiologicalProcessOrActivity - {'occurs_in'} -> Publication\n",
      "BiologicalProcessOrActivity - {'affects'} -> Publication\n",
      "BiologicalProcessOrActivity - {'has_participant'} -> Publication\n",
      "BiologicalProcessOrActivity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Food->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'affected_by'} -> Food\n",
      "PhenotypicFeature - {'is_ameliorated_by'} -> Food\n",
      "PhenotypicFeature - {'response_increased_by'} -> Food\n",
      "PhenotypicFeature - {'caused_by'} -> Food\n",
      "PhenotypicFeature - {'is_exacerbated_by'} -> Food\n",
      "PhenotypicFeature - {'disrupted_by'} -> Food\n",
      "PhenotypicFeature - {'regulated_by'} -> Food\n",
      "PhenotypicFeature - {'response_affected_by'} -> Food\n",
      "PhenotypicFeature - {'related_to'} -> Food\n",
      "PhenotypicFeature - {'treated_by'} -> Food\n",
      "PhenotypicFeature - {'response_decreased_by'} -> Food\n",
      "PhenotypicFeature - {'diagnoses'} -> Food\n",
      "PhenotypicFeature - {'treats'} -> Food\n",
      "PhenotypicFeature - {'has_input'} -> Food\n",
      "PhenotypicFeature - {'causes'} -> Food\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Food\n",
      "PhenotypicFeature - {'has_output'} -> Food\n",
      "PhenotypicFeature - {'associated_with'} -> Food\n",
      "PhenotypicFeature - {'has_phenotype'} -> Food\n",
      "PhenotypicFeature - {'located_in'} -> Food\n",
      "PhenotypicFeature - {'biomarker_for'} -> Food\n",
      "PhenotypicFeature - {'has_part'} -> Food\n",
      "PhenotypicFeature - {'interacts_with'} -> Food\n",
      "PhenotypicFeature - {'produces'} -> Food\n",
      "PhenotypicFeature - {'predisposes'} -> Food\n",
      "PhenotypicFeature - {'occurs_in'} -> Food\n",
      "PhenotypicFeature - {'exacerbates'} -> Food\n",
      "PhenotypicFeature - {'precedes'} -> Food\n",
      "PhenotypicFeature - {'affects'} -> Food\n",
      "PhenotypicFeature - {'coexists_with'} -> Food\n",
      "PhenotypicFeature - {'close_match'} -> Food\n",
      "PhenotypicFeature - {'has_participant'} -> Food\n",
      "PhenotypicFeature - {'manifestation_of'} -> Food\n",
      "PhenotypicFeature - {'regulates'} -> Food\n",
      "PhenotypicFeature - {'subclass_of'} -> Food\n",
      "PhenotypicFeature - {'derives_from'} -> Food\n",
      "Food - {'regulates'} -> Publication\n",
      "Food - {'associated_with'} -> Publication\n",
      "Food - {'located_in'} -> Publication\n",
      "Food - {'predisposes'} -> Publication\n",
      "Food - {'related_to'} -> Publication\n",
      "Food - {'occurs_in'} -> Publication\n",
      "Food - {'affects'} -> Publication\n",
      "Food - {'precedes'} -> Publication\n",
      "Food - {'subclass_of'} -> Publication\n",
      "Food - {'treats'} -> Publication\n",
      "Food - {'causes'} -> Publication\n",
      "Food - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->NoncodingRNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'affected_by'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_substrate'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'interacts_with'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_capability'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'participates_in'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_input'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'enabled_by'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_participant'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_catalyst'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'associated_with'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'actively_involves'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'affects'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'similar_to'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'related_to'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'overlaps'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_output'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'regulates'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'has_part'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'produces'} -> NoncodingRNAProduct\n",
      "PhenotypicFeature - {'coexists_with'} -> NoncodingRNAProduct\n",
      "NoncodingRNAProduct - {'affects'} -> Publication\n",
      "NoncodingRNAProduct - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->RNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'affects'} -> RNAProduct\n",
      "PhenotypicFeature - {'similar_to'} -> RNAProduct\n",
      "PhenotypicFeature - {'related_to'} -> RNAProduct\n",
      "PhenotypicFeature - {'overlaps'} -> RNAProduct\n",
      "PhenotypicFeature - {'affects_response_to'} -> RNAProduct\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_output'} -> RNAProduct\n",
      "PhenotypicFeature - {'disrupts'} -> RNAProduct\n",
      "PhenotypicFeature - {'affected_by'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_substrate'} -> RNAProduct\n",
      "PhenotypicFeature - {'interacts_with'} -> RNAProduct\n",
      "PhenotypicFeature - {'treats'} -> RNAProduct\n",
      "PhenotypicFeature - {'regulates'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_capability'} -> RNAProduct\n",
      "PhenotypicFeature - {'participates_in'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_input'} -> RNAProduct\n",
      "PhenotypicFeature - {'enabled_by'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_participant'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_catalyst'} -> RNAProduct\n",
      "PhenotypicFeature - {'decreases_response_to'} -> RNAProduct\n",
      "PhenotypicFeature - {'associated_with'} -> RNAProduct\n",
      "PhenotypicFeature - {'ameliorates'} -> RNAProduct\n",
      "PhenotypicFeature - {'actively_involves'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> RNAProduct\n",
      "PhenotypicFeature - {'exacerbates'} -> RNAProduct\n",
      "PhenotypicFeature - {'increases_response_to'} -> RNAProduct\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> RNAProduct\n",
      "PhenotypicFeature - {'coexists_with'} -> RNAProduct\n",
      "PhenotypicFeature - {'subclass_of'} -> RNAProduct\n",
      "PhenotypicFeature - {'located_in'} -> RNAProduct\n",
      "PhenotypicFeature - {'has_part'} -> RNAProduct\n",
      "PhenotypicFeature - {'produces'} -> RNAProduct\n",
      "RNAProduct - {'located_in'} -> Publication\n",
      "RNAProduct - {'regulates'} -> Publication\n",
      "RNAProduct - {'related_to'} -> Publication\n",
      "RNAProduct - {'affects'} -> Publication\n",
      "RNAProduct - {'subclass_of'} -> Publication\n",
      "RNAProduct - {'disrupts'} -> Publication\n",
      "RNAProduct - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->EnvironmentalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'actively_involves'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'associated_with'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'affects'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'related_to'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'precedes'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'overlaps'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'affects_response_to'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'exacerbates'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'increases_response_to'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'temporally_related_to'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'disrupts'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'similar_to'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'affected_by'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'treats'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'participates_in'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'preceded_by'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'has_participant'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'decreases_response_to'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'has_part'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'interacts_with'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'ameliorates'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'regulates'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'part_of'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'has_capability'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'close_match'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'coexists_with'} -> EnvironmentalProcess\n",
      "PhenotypicFeature - {'subclass_of'} -> EnvironmentalProcess\n",
      "EnvironmentalProcess - {'subclass_of'} -> Publication\n",
      "EnvironmentalProcess - {'related_to'} -> Publication\n",
      "EnvironmentalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Transcript->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'exacerbates'} -> Transcript\n",
      "PhenotypicFeature - {'treats'} -> Transcript\n",
      "PhenotypicFeature - {'participates_in'} -> Transcript\n",
      "PhenotypicFeature - {'has_input'} -> Transcript\n",
      "PhenotypicFeature - {'enabled_by'} -> Transcript\n",
      "PhenotypicFeature - {'has_participant'} -> Transcript\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Transcript\n",
      "PhenotypicFeature - {'has_substrate'} -> Transcript\n",
      "PhenotypicFeature - {'interacts_with'} -> Transcript\n",
      "PhenotypicFeature - {'ameliorates'} -> Transcript\n",
      "PhenotypicFeature - {'regulates'} -> Transcript\n",
      "PhenotypicFeature - {'has_capability'} -> Transcript\n",
      "PhenotypicFeature - {'actively_involves'} -> Transcript\n",
      "PhenotypicFeature - {'has_catalyst'} -> Transcript\n",
      "PhenotypicFeature - {'associated_with'} -> Transcript\n",
      "PhenotypicFeature - {'affects'} -> Transcript\n",
      "PhenotypicFeature - {'similar_to'} -> Transcript\n",
      "PhenotypicFeature - {'related_to'} -> Transcript\n",
      "PhenotypicFeature - {'overlaps'} -> Transcript\n",
      "PhenotypicFeature - {'affects_response_to'} -> Transcript\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Transcript\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Transcript\n",
      "PhenotypicFeature - {'increases_response_to'} -> Transcript\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Transcript\n",
      "PhenotypicFeature - {'has_output'} -> Transcript\n",
      "PhenotypicFeature - {'disrupts'} -> Transcript\n",
      "PhenotypicFeature - {'affected_by'} -> Transcript\n",
      "PhenotypicFeature - {'has_part'} -> Transcript\n",
      "PhenotypicFeature - {'produces'} -> Transcript\n",
      "PhenotypicFeature - {'coexists_with'} -> Transcript\n",
      "PhenotypicFeature - {'subclass_of'} -> Transcript\n",
      "Transcript - {'translates_to'} -> Publication\n",
      "Transcript - {'regulates'} -> Publication\n",
      "Transcript - {'transcribed_from'} -> Publication\n",
      "Transcript - {'associated_with'} -> Publication\n",
      "Transcript - {'located_in'} -> Publication\n",
      "Transcript - {'subclass_of'} -> Publication\n",
      "Transcript - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Phenomenon->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'related_to'} -> Phenomenon\n",
      "PhenotypicFeature - {'physically_interacts_with'} -> Phenomenon\n",
      "PhenotypicFeature - {'diagnoses'} -> Phenomenon\n",
      "PhenotypicFeature - {'treats'} -> Phenomenon\n",
      "PhenotypicFeature - {'has_input'} -> Phenomenon\n",
      "PhenotypicFeature - {'causes'} -> Phenomenon\n",
      "PhenotypicFeature - {'disrupts'} -> Phenomenon\n",
      "PhenotypicFeature - {'associated_with'} -> Phenomenon\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Phenomenon\n",
      "PhenotypicFeature - {'has_phenotype'} -> Phenomenon\n",
      "PhenotypicFeature - {'located_in'} -> Phenomenon\n",
      "PhenotypicFeature - {'biomarker_for'} -> Phenomenon\n",
      "PhenotypicFeature - {'interacts_with'} -> Phenomenon\n",
      "PhenotypicFeature - {'produces'} -> Phenomenon\n",
      "PhenotypicFeature - {'has_part'} -> Phenomenon\n",
      "PhenotypicFeature - {'prevents'} -> Phenomenon\n",
      "PhenotypicFeature - {'predisposes'} -> Phenomenon\n",
      "PhenotypicFeature - {'precedes'} -> Phenomenon\n",
      "PhenotypicFeature - {'occurs_in'} -> Phenomenon\n",
      "PhenotypicFeature - {'affects'} -> Phenomenon\n",
      "PhenotypicFeature - {'manifestation_of'} -> Phenomenon\n",
      "PhenotypicFeature - {'exacerbates'} -> Phenomenon\n",
      "PhenotypicFeature - {'coexists_with'} -> Phenomenon\n",
      "PhenotypicFeature - {'close_match'} -> Phenomenon\n",
      "PhenotypicFeature - {'has_participant'} -> Phenomenon\n",
      "PhenotypicFeature - {'regulates'} -> Phenomenon\n",
      "PhenotypicFeature - {'subclass_of'} -> Phenomenon\n",
      "Phenomenon - {'associated_with'} -> Publication\n",
      "Phenomenon - {'subclass_of'} -> Publication\n",
      "Phenomenon - {'interacts_with'} -> Publication\n",
      "Phenomenon - {'produces'} -> Publication\n",
      "Phenomenon - {'related_to'} -> Publication\n",
      "Phenomenon - {'has_part'} -> Publication\n",
      "Phenomenon - {'diagnoses'} -> Publication\n",
      "Phenomenon - {'occurs_in'} -> Publication\n",
      "Phenomenon - {'affects'} -> Publication\n",
      "Phenomenon - {'coexists_with'} -> Publication\n",
      "Phenomenon - {'close_match'} -> Publication\n",
      "Phenomenon - {'disrupts'} -> Publication\n",
      "Phenomenon - {'regulates'} -> Publication\n",
      "Phenomenon - {'located_in'} -> Publication\n",
      "Phenomenon - {'has_input'} -> Publication\n",
      "Phenomenon - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->IndividualOrganism->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'has_substrate'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'interacts_with'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'has_capability'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'participates_in'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'actively_involves'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'has_catalyst'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'associated_with'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'overlaps'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'has_output'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'affects'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'similar_to'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'related_to'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'affected_by'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'has_input'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'enabled_by'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'has_participant'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'regulates'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'subclass_of'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'causes'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'disrupts'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'located_in'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'has_part'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'occurs_in'} -> IndividualOrganism\n",
      "PhenotypicFeature - {'close_match'} -> IndividualOrganism\n",
      "IndividualOrganism - {'close_match'} -> Publication\n",
      "IndividualOrganism - {'subclass_of'} -> Publication\n",
      "IndividualOrganism - {'occurs_in'} -> Publication\n",
      "IndividualOrganism - {'located_in'} -> Publication\n",
      "IndividualOrganism - {'related_to'} -> Publication\n",
      "IndividualOrganism - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Treatment->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Treatment\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Treatment\n",
      "PhenotypicFeature - {'affected_by'} -> Treatment\n",
      "PhenotypicFeature - {'participates_in'} -> Treatment\n",
      "PhenotypicFeature - {'has_participant'} -> Treatment\n",
      "PhenotypicFeature - {'interacts_with'} -> Treatment\n",
      "PhenotypicFeature - {'associated_with'} -> Treatment\n",
      "PhenotypicFeature - {'affects'} -> Treatment\n",
      "PhenotypicFeature - {'similar_to'} -> Treatment\n",
      "PhenotypicFeature - {'related_to'} -> Treatment\n",
      "PhenotypicFeature - {'overlaps'} -> Treatment\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Treatment\n",
      "PhenotypicFeature - {'has_part'} -> Treatment\n",
      "PhenotypicFeature - {'contributes_to'} -> Treatment\n",
      "PhenotypicFeature - {'precedes'} -> Treatment\n",
      "PhenotypicFeature - {'manifestation_of'} -> Treatment\n",
      "PhenotypicFeature - {'coexists_with'} -> Treatment\n",
      "PhenotypicFeature - {'close_match'} -> Treatment\n",
      "PhenotypicFeature - {'subclass_of'} -> Treatment\n",
      "PhenotypicFeature - {'treats'} -> Treatment\n",
      "Treatment - {'subclass_of'} -> Publication\n",
      "Treatment - {'occurs_in'} -> Publication\n",
      "Treatment - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Behavior->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'exacerbates'} -> Behavior\n",
      "PhenotypicFeature - {'has_active_ingredient'} -> Behavior\n",
      "PhenotypicFeature - {'increases_response_to'} -> Behavior\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> Behavior\n",
      "PhenotypicFeature - {'temporally_related_to'} -> Behavior\n",
      "PhenotypicFeature - {'disrupts'} -> Behavior\n",
      "PhenotypicFeature - {'affected_by'} -> Behavior\n",
      "PhenotypicFeature - {'treats'} -> Behavior\n",
      "PhenotypicFeature - {'condition_associated_with_gene'} -> Behavior\n",
      "PhenotypicFeature - {'participates_in'} -> Behavior\n",
      "PhenotypicFeature - {'preceded_by'} -> Behavior\n",
      "PhenotypicFeature - {'has_participant'} -> Behavior\n",
      "PhenotypicFeature - {'decreases_response_to'} -> Behavior\n",
      "PhenotypicFeature - {'has_part'} -> Behavior\n",
      "PhenotypicFeature - {'interacts_with'} -> Behavior\n",
      "PhenotypicFeature - {'ameliorates'} -> Behavior\n",
      "PhenotypicFeature - {'regulates'} -> Behavior\n",
      "PhenotypicFeature - {'part_of'} -> Behavior\n",
      "PhenotypicFeature - {'has_capability'} -> Behavior\n",
      "PhenotypicFeature - {'actively_involves'} -> Behavior\n",
      "PhenotypicFeature - {'associated_with'} -> Behavior\n",
      "PhenotypicFeature - {'affects'} -> Behavior\n",
      "PhenotypicFeature - {'similar_to'} -> Behavior\n",
      "PhenotypicFeature - {'related_to'} -> Behavior\n",
      "PhenotypicFeature - {'precedes'} -> Behavior\n",
      "PhenotypicFeature - {'overlaps'} -> Behavior\n",
      "PhenotypicFeature - {'affects_response_to'} -> Behavior\n",
      "PhenotypicFeature - {'genetic_association'} -> Behavior\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> Behavior\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> Behavior\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Behavior\n",
      "PhenotypicFeature - {'has_input'} -> Behavior\n",
      "PhenotypicFeature - {'causes'} -> Behavior\n",
      "PhenotypicFeature - {'has_phenotype'} -> Behavior\n",
      "PhenotypicFeature - {'prevents'} -> Behavior\n",
      "PhenotypicFeature - {'predisposes'} -> Behavior\n",
      "PhenotypicFeature - {'occurs_in'} -> Behavior\n",
      "PhenotypicFeature - {'close_match'} -> Behavior\n",
      "PhenotypicFeature - {'manifestation_of'} -> Behavior\n",
      "PhenotypicFeature - {'coexists_with'} -> Behavior\n",
      "PhenotypicFeature - {'subclass_of'} -> Behavior\n",
      "PhenotypicFeature - {'diagnoses'} -> Behavior\n",
      "Behavior - {'treats'} -> Publication\n",
      "Behavior - {'has_input'} -> Publication\n",
      "Behavior - {'subclass_of'} -> Publication\n",
      "Behavior - {'located_in'} -> Publication\n",
      "Behavior - {'interacts_with'} -> Publication\n",
      "Behavior - {'predisposes'} -> Publication\n",
      "Behavior - {'related_to'} -> Publication\n",
      "Behavior - {'occurs_in'} -> Publication\n",
      "Behavior - {'affects'} -> Publication\n",
      "Behavior - {'coexists_with'} -> Publication\n",
      "Behavior - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->GeneFamily->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'participates_in'} -> GeneFamily\n",
      "PhenotypicFeature - {'has_participant'} -> GeneFamily\n",
      "PhenotypicFeature - {'associated_with'} -> GeneFamily\n",
      "PhenotypicFeature - {'has_upstream_actor'} -> GeneFamily\n",
      "PhenotypicFeature - {'primarily_composed_of'} -> GeneFamily\n",
      "PhenotypicFeature - {'affects'} -> GeneFamily\n",
      "PhenotypicFeature - {'similar_to'} -> GeneFamily\n",
      "PhenotypicFeature - {'related_to'} -> GeneFamily\n",
      "PhenotypicFeature - {'overlaps'} -> GeneFamily\n",
      "PhenotypicFeature - {'composed_primarily_of'} -> GeneFamily\n",
      "PhenotypicFeature - {'affected_by'} -> GeneFamily\n",
      "PhenotypicFeature - {'interacts_with'} -> GeneFamily\n",
      "PhenotypicFeature - {'directly_physically_interacts_with'} -> GeneFamily\n",
      "PhenotypicFeature - {'located_in'} -> GeneFamily\n",
      "PhenotypicFeature - {'has_part'} -> GeneFamily\n",
      "PhenotypicFeature - {'produces'} -> GeneFamily\n",
      "PhenotypicFeature - {'occurs_in'} -> GeneFamily\n",
      "PhenotypicFeature - {'is_sequence_variant_of'} -> GeneFamily\n",
      "PhenotypicFeature - {'coexists_with'} -> GeneFamily\n",
      "PhenotypicFeature - {'subclass_of'} -> GeneFamily\n",
      "PhenotypicFeature - {'regulates'} -> GeneFamily\n",
      "PhenotypicFeature - {'has_input'} -> GeneFamily\n",
      "PhenotypicFeature - {'derives_from'} -> GeneFamily\n",
      "GeneFamily - {'treats'} -> Publication\n",
      "GeneFamily - {'causes'} -> Publication\n",
      "GeneFamily - {'associated_with'} -> Publication\n",
      "GeneFamily - {'disrupts'} -> Publication\n",
      "GeneFamily - {'gene_associated_with_condition'} -> Publication\n",
      "GeneFamily - {'located_in'} -> Publication\n",
      "GeneFamily - {'regulates'} -> Publication\n",
      "GeneFamily - {'predisposes'} -> Publication\n",
      "GeneFamily - {'occurs_in'} -> Publication\n",
      "GeneFamily - {'affects'} -> Publication\n",
      "GeneFamily - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->CellLine->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'ameliorates'} -> CellLine\n",
      "PhenotypicFeature - {'affects'} -> CellLine\n",
      "PhenotypicFeature - {'related_to'} -> CellLine\n",
      "PhenotypicFeature - {'treats'} -> CellLine\n",
      "PhenotypicFeature - {'has_input'} -> CellLine\n",
      "PhenotypicFeature - {'derives_from'} -> CellLine\n",
      "PhenotypicFeature - {'disease_has_location'} -> CellLine\n",
      "PhenotypicFeature - {'disrupts'} -> CellLine\n",
      "PhenotypicFeature - {'associated_with'} -> CellLine\n",
      "PhenotypicFeature - {'located_in'} -> CellLine\n",
      "PhenotypicFeature - {'interacts_with'} -> CellLine\n",
      "PhenotypicFeature - {'produces'} -> CellLine\n",
      "PhenotypicFeature - {'expressed_in'} -> CellLine\n",
      "PhenotypicFeature - {'has_part'} -> CellLine\n",
      "PhenotypicFeature - {'occurs_in'} -> CellLine\n",
      "PhenotypicFeature - {'coexists_with'} -> CellLine\n",
      "PhenotypicFeature - {'close_match'} -> CellLine\n",
      "PhenotypicFeature - {'has_participant'} -> CellLine\n",
      "PhenotypicFeature - {'regulates'} -> CellLine\n",
      "PhenotypicFeature - {'subclass_of'} -> CellLine\n",
      "PhenotypicFeature - {'diagnoses'} -> CellLine\n",
      "CellLine - {'gene_associated_with_condition'} -> Publication\n",
      "CellLine - {'located_in'} -> Publication\n",
      "CellLine - {'produces'} -> Publication\n",
      "CellLine - {'related_to'} -> Publication\n",
      "CellLine - {'affects'} -> Publication\n",
      "CellLine - {'close_match'} -> Publication\n",
      "CellLine - {'subclass_of'} -> Publication\n",
      "CellLine - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->BehavioralFeature->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'contraindicated_for'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'diagnoses'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'treats'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'has_input'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'causes'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'disrupts'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'associated_with'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'has_phenotype'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'has_part'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'prevents'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'contributes_to'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'interacts_with'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'predisposes'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'related_to'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'occurs_in'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'exacerbates'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'precedes'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'affects'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'coexists_with'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'close_match'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'manifestation_of'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'regulates'} -> BehavioralFeature\n",
      "PhenotypicFeature - {'subclass_of'} -> BehavioralFeature\n",
      "BehavioralFeature - {'subclass_of'} -> Publication\n",
      "BehavioralFeature - {'produces'} -> Publication\n",
      "BehavioralFeature - {'related_to'} -> Publication\n",
      "BehavioralFeature - {'occurs_in'} -> Publication\n",
      "BehavioralFeature - {'affects'} -> Publication\n",
      "BehavioralFeature - {'manifestation_of'} -> Publication\n",
      "BehavioralFeature - {'coexists_with'} -> Publication\n",
      "BehavioralFeature - {'disrupts'} -> Publication\n",
      "BehavioralFeature - {'located_in'} -> Publication\n",
      "BehavioralFeature - {'predisposes'} -> Publication\n",
      "BehavioralFeature - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Agent->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'disrupts'} -> Agent\n",
      "PhenotypicFeature - {'associated_with'} -> Agent\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Agent\n",
      "PhenotypicFeature - {'has_phenotype'} -> Agent\n",
      "PhenotypicFeature - {'located_in'} -> Agent\n",
      "PhenotypicFeature - {'biomarker_for'} -> Agent\n",
      "PhenotypicFeature - {'interacts_with'} -> Agent\n",
      "PhenotypicFeature - {'produces'} -> Agent\n",
      "PhenotypicFeature - {'has_part'} -> Agent\n",
      "PhenotypicFeature - {'prevents'} -> Agent\n",
      "PhenotypicFeature - {'predisposes'} -> Agent\n",
      "PhenotypicFeature - {'related_to'} -> Agent\n",
      "PhenotypicFeature - {'occurs_in'} -> Agent\n",
      "PhenotypicFeature - {'affects'} -> Agent\n",
      "PhenotypicFeature - {'manifestation_of'} -> Agent\n",
      "PhenotypicFeature - {'exacerbates'} -> Agent\n",
      "PhenotypicFeature - {'precedes'} -> Agent\n",
      "PhenotypicFeature - {'coexists_with'} -> Agent\n",
      "PhenotypicFeature - {'close_match'} -> Agent\n",
      "PhenotypicFeature - {'regulates'} -> Agent\n",
      "PhenotypicFeature - {'subclass_of'} -> Agent\n",
      "PhenotypicFeature - {'diagnoses'} -> Agent\n",
      "PhenotypicFeature - {'treats'} -> Agent\n",
      "PhenotypicFeature - {'has_input'} -> Agent\n",
      "PhenotypicFeature - {'causes'} -> Agent\n",
      "Agent - {'has_part'} -> Publication\n",
      "Agent - {'interacts_with'} -> Publication\n",
      "Agent - {'produces'} -> Publication\n",
      "Agent - {'predisposes'} -> Publication\n",
      "Agent - {'related_to'} -> Publication\n",
      "Agent - {'prevents'} -> Publication\n",
      "Agent - {'precedes'} -> Publication\n",
      "Agent - {'occurs_in'} -> Publication\n",
      "Agent - {'affects'} -> Publication\n",
      "Agent - {'exacerbates'} -> Publication\n",
      "Agent - {'coexists_with'} -> Publication\n",
      "Agent - {'close_match'} -> Publication\n",
      "Agent - {'subclass_of'} -> Publication\n",
      "Agent - {'regulates'} -> Publication\n",
      "Agent - {'diagnoses'} -> Publication\n",
      "Agent - {'treats'} -> Publication\n",
      "Agent - {'has_input'} -> Publication\n",
      "Agent - {'causes'} -> Publication\n",
      "Agent - {'located_in'} -> Publication\n",
      "Agent - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->LifeStage->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'produces'} -> LifeStage\n",
      "PhenotypicFeature - {'related_to'} -> LifeStage\n",
      "PhenotypicFeature - {'occurs_in'} -> LifeStage\n",
      "PhenotypicFeature - {'affects'} -> LifeStage\n",
      "PhenotypicFeature - {'close_match'} -> LifeStage\n",
      "PhenotypicFeature - {'subclass_of'} -> LifeStage\n",
      "PhenotypicFeature - {'treats'} -> LifeStage\n",
      "PhenotypicFeature - {'located_in'} -> LifeStage\n",
      "LifeStage - {'subclass_of'} -> Publication\n",
      "LifeStage - {'has_part'} -> Publication\n",
      "LifeStage - {'interacts_with'} -> Publication\n",
      "LifeStage - {'produces'} -> Publication\n",
      "LifeStage - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Vitamin->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'interacts_with'} -> Vitamin\n",
      "PhenotypicFeature - {'produces'} -> Vitamin\n",
      "PhenotypicFeature - {'related_to'} -> Vitamin\n",
      "PhenotypicFeature - {'close_match'} -> Vitamin\n",
      "PhenotypicFeature - {'coexists_with'} -> Vitamin\n",
      "PhenotypicFeature - {'has_participant'} -> Vitamin\n",
      "PhenotypicFeature - {'regulates'} -> Vitamin\n",
      "PhenotypicFeature - {'has_input'} -> Vitamin\n",
      "PhenotypicFeature - {'associated_with'} -> Vitamin\n",
      "PhenotypicFeature - {'has_part'} -> Vitamin\n",
      "Vitamin - {'regulates'} -> Publication\n",
      "Vitamin - {'treats'} -> Publication\n",
      "Vitamin - {'disrupts'} -> Publication\n",
      "Vitamin - {'subclass_of'} -> Publication\n",
      "Vitamin - {'related_to'} -> Publication\n",
      "Vitamin - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ProteinDomain->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'has_part'} -> ProteinDomain\n",
      "PhenotypicFeature - {'located_in'} -> ProteinDomain\n",
      "PhenotypicFeature - {'interacts_with'} -> ProteinDomain\n",
      "PhenotypicFeature - {'produces'} -> ProteinDomain\n",
      "PhenotypicFeature - {'related_to'} -> ProteinDomain\n",
      "PhenotypicFeature - {'occurs_in'} -> ProteinDomain\n",
      "PhenotypicFeature - {'affects'} -> ProteinDomain\n",
      "PhenotypicFeature - {'coexists_with'} -> ProteinDomain\n",
      "PhenotypicFeature - {'regulates'} -> ProteinDomain\n",
      "PhenotypicFeature - {'disrupts'} -> ProteinDomain\n",
      "PhenotypicFeature - {'associated_with'} -> ProteinDomain\n",
      "ProteinDomain - {'associated_with'} -> Publication\n",
      "ProteinDomain - {'located_in'} -> Publication\n",
      "ProteinDomain - {'related_to'} -> Publication\n",
      "ProteinDomain - {'occurs_in'} -> Publication\n",
      "ProteinDomain - {'affects'} -> Publication\n",
      "ProteinDomain - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Cohort->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'treats'} -> Cohort\n",
      "PhenotypicFeature - {'has_input'} -> Cohort\n",
      "PhenotypicFeature - {'associated_with'} -> Cohort\n",
      "PhenotypicFeature - {'located_in'} -> Cohort\n",
      "PhenotypicFeature - {'interacts_with'} -> Cohort\n",
      "PhenotypicFeature - {'produces'} -> Cohort\n",
      "PhenotypicFeature - {'predisposes'} -> Cohort\n",
      "PhenotypicFeature - {'related_to'} -> Cohort\n",
      "PhenotypicFeature - {'occurs_in'} -> Cohort\n",
      "PhenotypicFeature - {'affects'} -> Cohort\n",
      "PhenotypicFeature - {'close_match'} -> Cohort\n",
      "PhenotypicFeature - {'coexists_with'} -> Cohort\n",
      "PhenotypicFeature - {'has_participant'} -> Cohort\n",
      "PhenotypicFeature - {'subclass_of'} -> Cohort\n",
      "PhenotypicFeature - {'diagnoses'} -> Cohort\n",
      "Cohort - {'diagnoses'} -> Publication\n",
      "Cohort - {'related_to'} -> Publication\n",
      "Cohort - {'treats'} -> Publication\n",
      "Cohort - {'has_input'} -> Publication\n",
      "Cohort - {'occurs_in'} -> Publication\n",
      "Cohort - {'has_part'} -> Publication\n",
      "Cohort - {'close_match'} -> Publication\n",
      "Cohort - {'subclass_of'} -> Publication\n",
      "Cohort - {'interacts_with'} -> Publication\n",
      "Cohort - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->PhysicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'exacerbates'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'precedes'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'affects'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'coexists_with'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'close_match'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'has_participant'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'manifestation_of'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'regulates'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'subclass_of'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'contraindicated_for'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'diagnoses'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'treats'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'has_input'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'derives_from'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'causes'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'disrupts'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'associated_with'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'has_phenotype'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'located_in'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'has_part'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'prevents'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'interacts_with'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'produces'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'predisposes'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'related_to'} -> PhysicalEntity\n",
      "PhenotypicFeature - {'occurs_in'} -> PhysicalEntity\n",
      "PhysicalEntity - {'subclass_of'} -> Publication\n",
      "PhysicalEntity - {'has_part'} -> Publication\n",
      "PhysicalEntity - {'interacts_with'} -> Publication\n",
      "PhysicalEntity - {'produces'} -> Publication\n",
      "PhysicalEntity - {'predisposes'} -> Publication\n",
      "PhysicalEntity - {'related_to'} -> Publication\n",
      "PhysicalEntity - {'occurs_in'} -> Publication\n",
      "PhysicalEntity - {'affects'} -> Publication\n",
      "PhysicalEntity - {'close_match'} -> Publication\n",
      "PhysicalEntity - {'coexists_with'} -> Publication\n",
      "PhysicalEntity - {'regulates'} -> Publication\n",
      "PhysicalEntity - {'located_in'} -> Publication\n",
      "PhysicalEntity - {'treats'} -> Publication\n",
      "PhysicalEntity - {'has_input'} -> Publication\n",
      "PhysicalEntity - {'causes'} -> Publication\n",
      "PhysicalEntity - {'disrupts'} -> Publication\n",
      "PhysicalEntity - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->MaterialSample->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'causes'} -> MaterialSample\n",
      "PhenotypicFeature - {'disrupts'} -> MaterialSample\n",
      "PhenotypicFeature - {'associated_with'} -> MaterialSample\n",
      "PhenotypicFeature - {'located_in'} -> MaterialSample\n",
      "PhenotypicFeature - {'interacts_with'} -> MaterialSample\n",
      "PhenotypicFeature - {'produces'} -> MaterialSample\n",
      "PhenotypicFeature - {'has_part'} -> MaterialSample\n",
      "PhenotypicFeature - {'predisposes'} -> MaterialSample\n",
      "PhenotypicFeature - {'related_to'} -> MaterialSample\n",
      "PhenotypicFeature - {'occurs_in'} -> MaterialSample\n",
      "PhenotypicFeature - {'affects'} -> MaterialSample\n",
      "PhenotypicFeature - {'manifestation_of'} -> MaterialSample\n",
      "PhenotypicFeature - {'exacerbates'} -> MaterialSample\n",
      "PhenotypicFeature - {'precedes'} -> MaterialSample\n",
      "PhenotypicFeature - {'coexists_with'} -> MaterialSample\n",
      "PhenotypicFeature - {'close_match'} -> MaterialSample\n",
      "PhenotypicFeature - {'regulates'} -> MaterialSample\n",
      "PhenotypicFeature - {'subclass_of'} -> MaterialSample\n",
      "PhenotypicFeature - {'diagnoses'} -> MaterialSample\n",
      "PhenotypicFeature - {'treats'} -> MaterialSample\n",
      "MaterialSample - {'located_in'} -> Publication\n",
      "MaterialSample - {'associated_with'} -> Publication\n",
      "MaterialSample - {'subclass_of'} -> Publication\n",
      "MaterialSample - {'has_part'} -> Publication\n",
      "MaterialSample - {'related_to'} -> Publication\n",
      "MaterialSample - {'occurs_in'} -> Publication\n",
      "MaterialSample - {'affects'} -> Publication\n",
      "MaterialSample - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->MicroRNA->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'associated_with'} -> MicroRNA\n",
      "PhenotypicFeature - {'located_in'} -> MicroRNA\n",
      "PhenotypicFeature - {'has_part'} -> MicroRNA\n",
      "PhenotypicFeature - {'interacts_with'} -> MicroRNA\n",
      "PhenotypicFeature - {'produces'} -> MicroRNA\n",
      "PhenotypicFeature - {'related_to'} -> MicroRNA\n",
      "PhenotypicFeature - {'occurs_in'} -> MicroRNA\n",
      "PhenotypicFeature - {'affects'} -> MicroRNA\n",
      "PhenotypicFeature - {'coexists_with'} -> MicroRNA\n",
      "MicroRNA - {'causes'} -> Publication\n",
      "MicroRNA - {'disrupts'} -> Publication\n",
      "MicroRNA - {'associated_with'} -> Publication\n",
      "MicroRNA - {'gene_associated_with_condition'} -> Publication\n",
      "MicroRNA - {'subclass_of'} -> Publication\n",
      "MicroRNA - {'predisposes'} -> Publication\n",
      "MicroRNA - {'located_in'} -> Publication\n",
      "MicroRNA - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->GeographicLocation->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'has_input'} -> GeographicLocation\n",
      "PhenotypicFeature - {'associated_with'} -> GeographicLocation\n",
      "PhenotypicFeature - {'located_in'} -> GeographicLocation\n",
      "PhenotypicFeature - {'produces'} -> GeographicLocation\n",
      "PhenotypicFeature - {'has_part'} -> GeographicLocation\n",
      "PhenotypicFeature - {'interacts_with'} -> GeographicLocation\n",
      "PhenotypicFeature - {'related_to'} -> GeographicLocation\n",
      "PhenotypicFeature - {'occurs_in'} -> GeographicLocation\n",
      "PhenotypicFeature - {'affects'} -> GeographicLocation\n",
      "PhenotypicFeature - {'coexists_with'} -> GeographicLocation\n",
      "PhenotypicFeature - {'close_match'} -> GeographicLocation\n",
      "PhenotypicFeature - {'has_participant'} -> GeographicLocation\n",
      "PhenotypicFeature - {'subclass_of'} -> GeographicLocation\n",
      "GeographicLocation - {'subclass_of'} -> Publication\n",
      "GeographicLocation - {'related_to'} -> Publication\n",
      "GeographicLocation - {'gene_associated_with_condition'} -> Publication\n",
      "GeographicLocation - {'located_in'} -> Publication\n",
      "GeographicLocation - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->ProteinFamily->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'regulates'} -> ProteinFamily\n",
      "PhenotypicFeature - {'associated_with'} -> ProteinFamily\n",
      "PhenotypicFeature - {'located_in'} -> ProteinFamily\n",
      "PhenotypicFeature - {'has_part'} -> ProteinFamily\n",
      "PhenotypicFeature - {'interacts_with'} -> ProteinFamily\n",
      "PhenotypicFeature - {'produces'} -> ProteinFamily\n",
      "PhenotypicFeature - {'related_to'} -> ProteinFamily\n",
      "PhenotypicFeature - {'occurs_in'} -> ProteinFamily\n",
      "PhenotypicFeature - {'affects'} -> ProteinFamily\n",
      "PhenotypicFeature - {'close_match'} -> ProteinFamily\n",
      "PhenotypicFeature - {'has_participant'} -> ProteinFamily\n",
      "PhenotypicFeature - {'coexists_with'} -> ProteinFamily\n",
      "ProteinFamily - {'associated_with'} -> Publication\n",
      "ProteinFamily - {'subclass_of'} -> Publication\n",
      "ProteinFamily - {'gene_associated_with_condition'} -> Publication\n",
      "ProteinFamily - {'predisposes'} -> Publication\n",
      "ProteinFamily - {'related_to'} -> Publication\n",
      "ProteinFamily - {'affects'} -> Publication\n",
      "ProteinFamily - {'disrupts'} -> Publication\n",
      "ProteinFamily - {'located_in'} -> Publication\n",
      "ProteinFamily - {'regulates'} -> Publication\n",
      "ProteinFamily - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->PhenotypicFeature->Event->Publication\n",
      "InformationResource - {'associated_with'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'related_to'} -> Event\n",
      "PhenotypicFeature - {'exacerbates'} -> Event\n",
      "PhenotypicFeature - {'manifestation_of'} -> Event\n",
      "PhenotypicFeature - {'coexists_with'} -> Event\n",
      "PhenotypicFeature - {'subclass_of'} -> Event\n",
      "PhenotypicFeature - {'diagnoses'} -> Event\n",
      "PhenotypicFeature - {'has_input'} -> Event\n",
      "PhenotypicFeature - {'close_match'} -> Event\n",
      "Event - {'occurs_in'} -> Publication\n",
      "Event - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->BiologicalEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_input'} -> BiologicalEntity\n",
      "Behavior - {'subclass_of'} -> BiologicalEntity\n",
      "Behavior - {'caused_by'} -> BiologicalEntity\n",
      "Behavior - {'has_part'} -> BiologicalEntity\n",
      "Behavior - {'has_participant'} -> BiologicalEntity\n",
      "Behavior - {'overlaps'} -> BiologicalEntity\n",
      "Behavior - {'part_of'} -> BiologicalEntity\n",
      "Behavior - {'preceded_by'} -> BiologicalEntity\n",
      "Behavior - {'related_to'} -> BiologicalEntity\n",
      "Behavior - {'temporally_related_to'} -> BiologicalEntity\n",
      "Behavior - {'has_upstream_actor'} -> BiologicalEntity\n",
      "Behavior - {'gene_associated_with_condition'} -> BiologicalEntity\n",
      "Behavior - {'has_output'} -> BiologicalEntity\n",
      "Behavior - {'affected_by'} -> BiologicalEntity\n",
      "Behavior - {'condition_associated_with_gene'} -> BiologicalEntity\n",
      "Behavior - {'has_capability'} -> BiologicalEntity\n",
      "Behavior - {'participates_in'} -> BiologicalEntity\n",
      "Behavior - {'enabled_by'} -> BiologicalEntity\n",
      "Behavior - {'has_substrate'} -> BiologicalEntity\n",
      "Behavior - {'precedes'} -> BiologicalEntity\n",
      "Behavior - {'interacts_with'} -> BiologicalEntity\n",
      "Behavior - {'actively_involves'} -> BiologicalEntity\n",
      "Behavior - {'has_active_ingredient'} -> BiologicalEntity\n",
      "Behavior - {'has_catalyst'} -> BiologicalEntity\n",
      "Behavior - {'primarily_composed_of'} -> BiologicalEntity\n",
      "Behavior - {'associated_with'} -> BiologicalEntity\n",
      "Behavior - {'affects'} -> BiologicalEntity\n",
      "Behavior - {'similar_to'} -> BiologicalEntity\n",
      "Behavior - {'genetic_association'} -> BiologicalEntity\n",
      "Behavior - {'composed_primarily_of'} -> BiologicalEntity\n",
      "Behavior - {'prevents'} -> BiologicalEntity\n",
      "Behavior - {'occurs_in'} -> BiologicalEntity\n",
      "Behavior - {'close_match'} -> BiologicalEntity\n",
      "Behavior - {'manifestation_of'} -> BiologicalEntity\n",
      "Behavior - {'exacerbates'} -> BiologicalEntity\n",
      "Behavior - {'coexists_with'} -> BiologicalEntity\n",
      "Behavior - {'regulates'} -> BiologicalEntity\n",
      "Behavior - {'treats'} -> BiologicalEntity\n",
      "Behavior - {'causes'} -> BiologicalEntity\n",
      "Behavior - {'located_in'} -> BiologicalEntity\n",
      "Behavior - {'produces'} -> BiologicalEntity\n",
      "Behavior - {'predisposes'} -> BiologicalEntity\n",
      "BiologicalEntity - {'physically_interacts_with'} -> Publication\n",
      "BiologicalEntity - {'treats'} -> Publication\n",
      "BiologicalEntity - {'has_input'} -> Publication\n",
      "BiologicalEntity - {'causes'} -> Publication\n",
      "BiologicalEntity - {'disrupts'} -> Publication\n",
      "BiologicalEntity - {'is_sequence_variant_of'} -> Publication\n",
      "BiologicalEntity - {'has_phenotype'} -> Publication\n",
      "BiologicalEntity - {'biomarker_for'} -> Publication\n",
      "BiologicalEntity - {'associated_with'} -> Publication\n",
      "BiologicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalEntity - {'has_part'} -> Publication\n",
      "BiologicalEntity - {'located_in'} -> Publication\n",
      "BiologicalEntity - {'colocalizes_with'} -> Publication\n",
      "BiologicalEntity - {'interacts_with'} -> Publication\n",
      "BiologicalEntity - {'produces'} -> Publication\n",
      "BiologicalEntity - {'predisposes'} -> Publication\n",
      "BiologicalEntity - {'related_to'} -> Publication\n",
      "BiologicalEntity - {'prevents'} -> Publication\n",
      "BiologicalEntity - {'manifestation_of'} -> Publication\n",
      "BiologicalEntity - {'precedes'} -> Publication\n",
      "BiologicalEntity - {'occurs_in'} -> Publication\n",
      "BiologicalEntity - {'affects'} -> Publication\n",
      "BiologicalEntity - {'close_match'} -> Publication\n",
      "BiologicalEntity - {'exacerbates'} -> Publication\n",
      "BiologicalEntity - {'coexists_with'} -> Publication\n",
      "BiologicalEntity - {'has_participant'} -> Publication\n",
      "BiologicalEntity - {'subclass_of'} -> Publication\n",
      "BiologicalEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->SmallMolecule->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'related_to'} -> SmallMolecule\n",
      "Behavior - {'caused_by'} -> SmallMolecule\n",
      "Behavior - {'primarily_composed_of'} -> SmallMolecule\n",
      "Behavior - {'affected_by'} -> SmallMolecule\n",
      "Behavior - {'participates_in'} -> SmallMolecule\n",
      "Behavior - {'has_participant'} -> SmallMolecule\n",
      "Behavior - {'interacts_with'} -> SmallMolecule\n",
      "Behavior - {'associated_with'} -> SmallMolecule\n",
      "Behavior - {'affects'} -> SmallMolecule\n",
      "Behavior - {'similar_to'} -> SmallMolecule\n",
      "Behavior - {'overlaps'} -> SmallMolecule\n",
      "Behavior - {'composed_primarily_of'} -> SmallMolecule\n",
      "Behavior - {'has_upstream_actor'} -> SmallMolecule\n",
      "Behavior - {'produces'} -> SmallMolecule\n",
      "Behavior - {'predisposes'} -> SmallMolecule\n",
      "Behavior - {'prevents'} -> SmallMolecule\n",
      "Behavior - {'occurs_in'} -> SmallMolecule\n",
      "Behavior - {'manifestation_of'} -> SmallMolecule\n",
      "Behavior - {'coexists_with'} -> SmallMolecule\n",
      "Behavior - {'regulates'} -> SmallMolecule\n",
      "Behavior - {'treats'} -> SmallMolecule\n",
      "Behavior - {'has_input'} -> SmallMolecule\n",
      "Behavior - {'in_taxon'} -> SmallMolecule\n",
      "Behavior - {'located_in'} -> SmallMolecule\n",
      "Behavior - {'has_part'} -> SmallMolecule\n",
      "Behavior - {'close_match'} -> SmallMolecule\n",
      "SmallMolecule - {'correlated_with'} -> Publication\n",
      "SmallMolecule - {'disrupts'} -> Publication\n",
      "SmallMolecule - {'gene_associated_with_condition'} -> Publication\n",
      "SmallMolecule - {'located_in'} -> Publication\n",
      "SmallMolecule - {'regulates'} -> Publication\n",
      "SmallMolecule - {'has_part'} -> Publication\n",
      "SmallMolecule - {'physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'predisposes'} -> Publication\n",
      "SmallMolecule - {'related_to'} -> Publication\n",
      "SmallMolecule - {'causes'} -> Publication\n",
      "SmallMolecule - {'directly_physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'precedes'} -> Publication\n",
      "SmallMolecule - {'close_match'} -> Publication\n",
      "SmallMolecule - {'associated_with'} -> Publication\n",
      "SmallMolecule - {'has_metabolite'} -> Publication\n",
      "SmallMolecule - {'subclass_of'} -> Publication\n",
      "SmallMolecule - {'interacts_with'} -> Publication\n",
      "SmallMolecule - {'treats'} -> Publication\n",
      "SmallMolecule - {'has_input'} -> Publication\n",
      "SmallMolecule - {'occurs_in'} -> Publication\n",
      "SmallMolecule - {'affects'} -> Publication\n",
      "SmallMolecule - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->MolecularMixture->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'caused_by'} -> MolecularMixture\n",
      "Behavior - {'composed_primarily_of'} -> MolecularMixture\n",
      "Behavior - {'has_upstream_actor'} -> MolecularMixture\n",
      "Behavior - {'primarily_composed_of'} -> MolecularMixture\n",
      "Behavior - {'temporally_related_to'} -> MolecularMixture\n",
      "Behavior - {'has_output'} -> MolecularMixture\n",
      "Behavior - {'affects'} -> MolecularMixture\n",
      "Behavior - {'similar_to'} -> MolecularMixture\n",
      "Behavior - {'affected_by'} -> MolecularMixture\n",
      "Behavior - {'condition_associated_with_gene'} -> MolecularMixture\n",
      "Behavior - {'gene_associated_with_condition'} -> MolecularMixture\n",
      "Behavior - {'has_input'} -> MolecularMixture\n",
      "Behavior - {'preceded_by'} -> MolecularMixture\n",
      "Behavior - {'enabled_by'} -> MolecularMixture\n",
      "Behavior - {'has_participant'} -> MolecularMixture\n",
      "Behavior - {'has_substrate'} -> MolecularMixture\n",
      "Behavior - {'interacts_with'} -> MolecularMixture\n",
      "Behavior - {'has_capability'} -> MolecularMixture\n",
      "Behavior - {'participates_in'} -> MolecularMixture\n",
      "Behavior - {'actively_involves'} -> MolecularMixture\n",
      "Behavior - {'has_catalyst'} -> MolecularMixture\n",
      "Behavior - {'associated_with'} -> MolecularMixture\n",
      "Behavior - {'related_to'} -> MolecularMixture\n",
      "Behavior - {'precedes'} -> MolecularMixture\n",
      "Behavior - {'overlaps'} -> MolecularMixture\n",
      "Behavior - {'genetic_association'} -> MolecularMixture\n",
      "MolecularMixture - {'correlated_with'} -> Publication\n",
      "MolecularMixture - {'treats'} -> Publication\n",
      "MolecularMixture - {'subclass_of'} -> Publication\n",
      "MolecularMixture - {'located_in'} -> Publication\n",
      "MolecularMixture - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->BiologicalProcess->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'subclass_of'} -> BiologicalProcess\n",
      "Behavior - {'has_part'} -> BiologicalProcess\n",
      "Behavior - {'overlaps'} -> BiologicalProcess\n",
      "Behavior - {'part_of'} -> BiologicalProcess\n",
      "Behavior - {'preceded_by'} -> BiologicalProcess\n",
      "Behavior - {'related_to'} -> BiologicalProcess\n",
      "Behavior - {'temporally_related_to'} -> BiologicalProcess\n",
      "Behavior - {'associated_with'} -> BiologicalProcess\n",
      "Behavior - {'affects'} -> BiologicalProcess\n",
      "Behavior - {'similar_to'} -> BiologicalProcess\n",
      "Behavior - {'composed_primarily_of'} -> BiologicalProcess\n",
      "Behavior - {'has_upstream_actor'} -> BiologicalProcess\n",
      "Behavior - {'has_output'} -> BiologicalProcess\n",
      "Behavior - {'affected_by'} -> BiologicalProcess\n",
      "Behavior - {'has_capability'} -> BiologicalProcess\n",
      "Behavior - {'participates_in'} -> BiologicalProcess\n",
      "Behavior - {'has_input'} -> BiologicalProcess\n",
      "Behavior - {'enabled_by'} -> BiologicalProcess\n",
      "Behavior - {'has_participant'} -> BiologicalProcess\n",
      "Behavior - {'has_substrate'} -> BiologicalProcess\n",
      "Behavior - {'precedes'} -> BiologicalProcess\n",
      "Behavior - {'interacts_with'} -> BiologicalProcess\n",
      "Behavior - {'caused_by'} -> BiologicalProcess\n",
      "Behavior - {'actively_involves'} -> BiologicalProcess\n",
      "Behavior - {'has_active_ingredient'} -> BiologicalProcess\n",
      "Behavior - {'has_catalyst'} -> BiologicalProcess\n",
      "Behavior - {'primarily_composed_of'} -> BiologicalProcess\n",
      "Behavior - {'disrupts'} -> BiologicalProcess\n",
      "Behavior - {'predisposes'} -> BiologicalProcess\n",
      "Behavior - {'prevents'} -> BiologicalProcess\n",
      "Behavior - {'manifestation_of'} -> BiologicalProcess\n",
      "Behavior - {'coexists_with'} -> BiologicalProcess\n",
      "Behavior - {'close_match'} -> BiologicalProcess\n",
      "Behavior - {'regulates'} -> BiologicalProcess\n",
      "Behavior - {'treats'} -> BiologicalProcess\n",
      "Behavior - {'causes'} -> BiologicalProcess\n",
      "BiologicalProcess - {'correlated_with'} -> Publication\n",
      "BiologicalProcess - {'disrupts'} -> Publication\n",
      "BiologicalProcess - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalProcess - {'regulates'} -> Publication\n",
      "BiologicalProcess - {'biomarker_for'} -> Publication\n",
      "BiologicalProcess - {'has_part'} -> Publication\n",
      "BiologicalProcess - {'located_in'} -> Publication\n",
      "BiologicalProcess - {'predisposes'} -> Publication\n",
      "BiologicalProcess - {'precedes'} -> Publication\n",
      "BiologicalProcess - {'prevents'} -> Publication\n",
      "BiologicalProcess - {'causes'} -> Publication\n",
      "BiologicalProcess - {'close_match'} -> Publication\n",
      "BiologicalProcess - {'associated_with'} -> Publication\n",
      "BiologicalProcess - {'produces'} -> Publication\n",
      "BiologicalProcess - {'has_participant'} -> Publication\n",
      "BiologicalProcess - {'subclass_of'} -> Publication\n",
      "BiologicalProcess - {'related_to'} -> Publication\n",
      "BiologicalProcess - {'affects'} -> Publication\n",
      "BiologicalProcess - {'treats'} -> Publication\n",
      "BiologicalProcess - {'manifestation_of'} -> Publication\n",
      "BiologicalProcess - {'has_input'} -> Publication\n",
      "BiologicalProcess - {'has_output'} -> Publication\n",
      "BiologicalProcess - {'occurs_in'} -> Publication\n",
      "BiologicalProcess - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->BiologicalProcessOrActivity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'subclass_of'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_part'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'overlaps'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'part_of'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'preceded_by'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'related_to'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'temporally_related_to'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_substrate'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'interacts_with'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_capability'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'participates_in'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'caused_by'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'actively_involves'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_catalyst'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'associated_with'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'precedes'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'composed_primarily_of'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_upstream_actor'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_active_ingredient'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'primarily_composed_of'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_output'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'affects'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'similar_to'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'affected_by'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_input'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'enabled_by'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'has_participant'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'manifestation_of'} -> BiologicalProcessOrActivity\n",
      "Behavior - {'coexists_with'} -> BiologicalProcessOrActivity\n",
      "BiologicalProcessOrActivity - {'occurs_in'} -> Publication\n",
      "BiologicalProcessOrActivity - {'affects'} -> Publication\n",
      "BiologicalProcessOrActivity - {'has_participant'} -> Publication\n",
      "BiologicalProcessOrActivity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->NamedThing->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'subclass_of'} -> NamedThing\n",
      "Behavior - {'caused_by'} -> NamedThing\n",
      "Behavior - {'has_participant'} -> NamedThing\n",
      "Behavior - {'has_part'} -> NamedThing\n",
      "Behavior - {'overlaps'} -> NamedThing\n",
      "Behavior - {'part_of'} -> NamedThing\n",
      "Behavior - {'preceded_by'} -> NamedThing\n",
      "Behavior - {'related_to'} -> NamedThing\n",
      "Behavior - {'temporally_related_to'} -> NamedThing\n",
      "Behavior - {'affects'} -> NamedThing\n",
      "Behavior - {'affected_by'} -> NamedThing\n",
      "Behavior - {'has_capability'} -> NamedThing\n",
      "Behavior - {'participates_in'} -> NamedThing\n",
      "Behavior - {'has_input'} -> NamedThing\n",
      "Behavior - {'enabled_by'} -> NamedThing\n",
      "Behavior - {'has_substrate'} -> NamedThing\n",
      "Behavior - {'precedes'} -> NamedThing\n",
      "Behavior - {'interacts_with'} -> NamedThing\n",
      "Behavior - {'actively_involves'} -> NamedThing\n",
      "Behavior - {'has_active_ingredient'} -> NamedThing\n",
      "Behavior - {'has_catalyst'} -> NamedThing\n",
      "Behavior - {'primarily_composed_of'} -> NamedThing\n",
      "Behavior - {'associated_with'} -> NamedThing\n",
      "Behavior - {'similar_to'} -> NamedThing\n",
      "Behavior - {'composed_primarily_of'} -> NamedThing\n",
      "Behavior - {'has_upstream_actor'} -> NamedThing\n",
      "Behavior - {'has_output'} -> NamedThing\n",
      "Behavior - {'prevents'} -> NamedThing\n",
      "Behavior - {'diagnoses'} -> NamedThing\n",
      "Behavior - {'occurs_in'} -> NamedThing\n",
      "Behavior - {'close_match'} -> NamedThing\n",
      "Behavior - {'manifestation_of'} -> NamedThing\n",
      "Behavior - {'exacerbates'} -> NamedThing\n",
      "Behavior - {'coexists_with'} -> NamedThing\n",
      "Behavior - {'regulates'} -> NamedThing\n",
      "Behavior - {'treats'} -> NamedThing\n",
      "Behavior - {'causes'} -> NamedThing\n",
      "Behavior - {'disrupts'} -> NamedThing\n",
      "Behavior - {'located_in'} -> NamedThing\n",
      "Behavior - {'produces'} -> NamedThing\n",
      "Behavior - {'predisposes'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Publication\n",
      "NamedThing - {'related_to'} -> Publication\n",
      "NamedThing - {'is_sequence_variant_of'} -> Publication\n",
      "NamedThing - {'associated_with'} -> Publication\n",
      "NamedThing - {'has_phenotype'} -> Publication\n",
      "NamedThing - {'disrupts'} -> Publication\n",
      "NamedThing - {'biomarker_for'} -> Publication\n",
      "NamedThing - {'correlated_with'} -> Publication\n",
      "NamedThing - {'gene_associated_with_condition'} -> Publication\n",
      "NamedThing - {'has_part'} -> Publication\n",
      "NamedThing - {'located_in'} -> Publication\n",
      "NamedThing - {'colocalizes_with'} -> Publication\n",
      "NamedThing - {'interacts_with'} -> Publication\n",
      "NamedThing - {'produces'} -> Publication\n",
      "NamedThing - {'prevents'} -> Publication\n",
      "NamedThing - {'diagnoses'} -> Publication\n",
      "NamedThing - {'predisposes'} -> Publication\n",
      "NamedThing - {'precedes'} -> Publication\n",
      "NamedThing - {'occurs_in'} -> Publication\n",
      "NamedThing - {'affects'} -> Publication\n",
      "NamedThing - {'close_match'} -> Publication\n",
      "NamedThing - {'manifestation_of'} -> Publication\n",
      "NamedThing - {'exacerbates'} -> Publication\n",
      "NamedThing - {'coexists_with'} -> Publication\n",
      "NamedThing - {'regulates'} -> Publication\n",
      "NamedThing - {'has_input'} -> Publication\n",
      "NamedThing - {'physically_interacts_with'} -> Publication\n",
      "NamedThing - {'treats'} -> Publication\n",
      "NamedThing - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Entity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'subclass_of'} -> Entity\n",
      "Behavior - {'caused_by'} -> Entity\n",
      "Behavior - {'has_part'} -> Entity\n",
      "Behavior - {'has_participant'} -> Entity\n",
      "Behavior - {'overlaps'} -> Entity\n",
      "Behavior - {'part_of'} -> Entity\n",
      "Behavior - {'preceded_by'} -> Entity\n",
      "Behavior - {'related_to'} -> Entity\n",
      "Behavior - {'temporally_related_to'} -> Entity\n",
      "Behavior - {'participates_in'} -> Entity\n",
      "Behavior - {'composed_primarily_of'} -> Entity\n",
      "Behavior - {'has_upstream_actor'} -> Entity\n",
      "Behavior - {'has_output'} -> Entity\n",
      "Behavior - {'affected_by'} -> Entity\n",
      "Behavior - {'has_capability'} -> Entity\n",
      "Behavior - {'has_input'} -> Entity\n",
      "Behavior - {'enabled_by'} -> Entity\n",
      "Behavior - {'has_substrate'} -> Entity\n",
      "Behavior - {'precedes'} -> Entity\n",
      "Behavior - {'interacts_with'} -> Entity\n",
      "Behavior - {'actively_involves'} -> Entity\n",
      "Behavior - {'has_active_ingredient'} -> Entity\n",
      "Behavior - {'has_catalyst'} -> Entity\n",
      "Behavior - {'primarily_composed_of'} -> Entity\n",
      "Behavior - {'associated_with'} -> Entity\n",
      "Behavior - {'affects'} -> Entity\n",
      "Behavior - {'similar_to'} -> Entity\n",
      "Entity - {'subclass_of'} -> Publication\n",
      "Entity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Protein->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'genetic_association'} -> Protein\n",
      "Behavior - {'related_to'} -> Protein\n",
      "Behavior - {'overlaps'} -> Protein\n",
      "Behavior - {'composed_primarily_of'} -> Protein\n",
      "Behavior - {'has_upstream_actor'} -> Protein\n",
      "Behavior - {'gene_associated_with_condition'} -> Protein\n",
      "Behavior - {'has_output'} -> Protein\n",
      "Behavior - {'affected_by'} -> Protein\n",
      "Behavior - {'interacts_with'} -> Protein\n",
      "Behavior - {'condition_associated_with_gene'} -> Protein\n",
      "Behavior - {'part_of'} -> Protein\n",
      "Behavior - {'has_capability'} -> Protein\n",
      "Behavior - {'participates_in'} -> Protein\n",
      "Behavior - {'has_input'} -> Protein\n",
      "Behavior - {'preceded_by'} -> Protein\n",
      "Behavior - {'enabled_by'} -> Protein\n",
      "Behavior - {'has_participant'} -> Protein\n",
      "Behavior - {'associated_with'} -> Protein\n",
      "Behavior - {'has_part'} -> Protein\n",
      "Behavior - {'has_substrate'} -> Protein\n",
      "Behavior - {'precedes'} -> Protein\n",
      "Behavior - {'caused_by'} -> Protein\n",
      "Behavior - {'actively_involves'} -> Protein\n",
      "Behavior - {'has_active_ingredient'} -> Protein\n",
      "Behavior - {'has_catalyst'} -> Protein\n",
      "Behavior - {'primarily_composed_of'} -> Protein\n",
      "Behavior - {'temporally_related_to'} -> Protein\n",
      "Behavior - {'affects'} -> Protein\n",
      "Behavior - {'similar_to'} -> Protein\n",
      "Behavior - {'regulates'} -> Protein\n",
      "Behavior - {'treats'} -> Protein\n",
      "Behavior - {'causes'} -> Protein\n",
      "Behavior - {'disrupts'} -> Protein\n",
      "Behavior - {'gene_product_of'} -> Protein\n",
      "Behavior - {'subclass_of'} -> Protein\n",
      "Behavior - {'located_in'} -> Protein\n",
      "Behavior - {'produces'} -> Protein\n",
      "Behavior - {'predisposes'} -> Protein\n",
      "Behavior - {'prevents'} -> Protein\n",
      "Behavior - {'occurs_in'} -> Protein\n",
      "Behavior - {'manifestation_of'} -> Protein\n",
      "Behavior - {'coexists_with'} -> Protein\n",
      "Behavior - {'close_match'} -> Protein\n",
      "Protein - {'correlated_with'} -> Publication\n",
      "Protein - {'gene_product_of'} -> Publication\n",
      "Protein - {'associated_with'} -> Publication\n",
      "Protein - {'gene_associated_with_condition'} -> Publication\n",
      "Protein - {'located_in'} -> Publication\n",
      "Protein - {'regulates'} -> Publication\n",
      "Protein - {'biomarker_for'} -> Publication\n",
      "Protein - {'expressed_in'} -> Publication\n",
      "Protein - {'has_part'} -> Publication\n",
      "Protein - {'colocalizes_with'} -> Publication\n",
      "Protein - {'produces'} -> Publication\n",
      "Protein - {'predisposes'} -> Publication\n",
      "Protein - {'related_to'} -> Publication\n",
      "Protein - {'prevents'} -> Publication\n",
      "Protein - {'occurs_in'} -> Publication\n",
      "Protein - {'precedes'} -> Publication\n",
      "Protein - {'affects'} -> Publication\n",
      "Protein - {'close_match'} -> Publication\n",
      "Protein - {'subclass_of'} -> Publication\n",
      "Protein - {'interacts_with'} -> Publication\n",
      "Protein - {'treats'} -> Publication\n",
      "Protein - {'has_input'} -> Publication\n",
      "Protein - {'causes'} -> Publication\n",
      "Protein - {'physically_interacts_with'} -> Publication\n",
      "Protein - {'exacerbates'} -> Publication\n",
      "Protein - {'coexists_with'} -> Publication\n",
      "Protein - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->AnatomicalEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'caused_by'} -> AnatomicalEntity\n",
      "Behavior - {'has_participant'} -> AnatomicalEntity\n",
      "Behavior - {'related_to'} -> AnatomicalEntity\n",
      "Behavior - {'part_of'} -> AnatomicalEntity\n",
      "Behavior - {'has_capability'} -> AnatomicalEntity\n",
      "Behavior - {'participates_in'} -> AnatomicalEntity\n",
      "Behavior - {'has_input'} -> AnatomicalEntity\n",
      "Behavior - {'enabled_by'} -> AnatomicalEntity\n",
      "Behavior - {'has_catalyst'} -> AnatomicalEntity\n",
      "Behavior - {'associated_with'} -> AnatomicalEntity\n",
      "Behavior - {'has_part'} -> AnatomicalEntity\n",
      "Behavior - {'precedes'} -> AnatomicalEntity\n",
      "Behavior - {'genetic_association'} -> AnatomicalEntity\n",
      "Behavior - {'actively_involves'} -> AnatomicalEntity\n",
      "Behavior - {'has_upstream_actor'} -> AnatomicalEntity\n",
      "Behavior - {'has_active_ingredient'} -> AnatomicalEntity\n",
      "Behavior - {'primarily_composed_of'} -> AnatomicalEntity\n",
      "Behavior - {'temporally_related_to'} -> AnatomicalEntity\n",
      "Behavior - {'affects'} -> AnatomicalEntity\n",
      "Behavior - {'similar_to'} -> AnatomicalEntity\n",
      "Behavior - {'overlaps'} -> AnatomicalEntity\n",
      "Behavior - {'condition_associated_with_gene'} -> AnatomicalEntity\n",
      "Behavior - {'composed_primarily_of'} -> AnatomicalEntity\n",
      "Behavior - {'gene_associated_with_condition'} -> AnatomicalEntity\n",
      "Behavior - {'preceded_by'} -> AnatomicalEntity\n",
      "Behavior - {'has_output'} -> AnatomicalEntity\n",
      "Behavior - {'affected_by'} -> AnatomicalEntity\n",
      "Behavior - {'has_substrate'} -> AnatomicalEntity\n",
      "Behavior - {'interacts_with'} -> AnatomicalEntity\n",
      "Behavior - {'coexists_with'} -> AnatomicalEntity\n",
      "Behavior - {'regulates'} -> AnatomicalEntity\n",
      "Behavior - {'treats'} -> AnatomicalEntity\n",
      "Behavior - {'causes'} -> AnatomicalEntity\n",
      "Behavior - {'disrupts'} -> AnatomicalEntity\n",
      "Behavior - {'subclass_of'} -> AnatomicalEntity\n",
      "Behavior - {'located_in'} -> AnatomicalEntity\n",
      "Behavior - {'produces'} -> AnatomicalEntity\n",
      "Behavior - {'predisposes'} -> AnatomicalEntity\n",
      "Behavior - {'occurs_in'} -> AnatomicalEntity\n",
      "AnatomicalEntity - {'develops_from'} -> Publication\n",
      "AnatomicalEntity - {'interacts_with'} -> Publication\n",
      "AnatomicalEntity - {'produces'} -> Publication\n",
      "AnatomicalEntity - {'has_part'} -> Publication\n",
      "AnatomicalEntity - {'diagnoses'} -> Publication\n",
      "AnatomicalEntity - {'related_to'} -> Publication\n",
      "AnatomicalEntity - {'affects'} -> Publication\n",
      "AnatomicalEntity - {'manifestation_of'} -> Publication\n",
      "AnatomicalEntity - {'has_input'} -> Publication\n",
      "AnatomicalEntity - {'occurs_in'} -> Publication\n",
      "AnatomicalEntity - {'coexists_with'} -> Publication\n",
      "AnatomicalEntity - {'close_match'} -> Publication\n",
      "AnatomicalEntity - {'disrupts'} -> Publication\n",
      "AnatomicalEntity - {'located_in'} -> Publication\n",
      "AnatomicalEntity - {'regulates'} -> Publication\n",
      "AnatomicalEntity - {'treats'} -> Publication\n",
      "AnatomicalEntity - {'causes'} -> Publication\n",
      "AnatomicalEntity - {'associated_with'} -> Publication\n",
      "AnatomicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "AnatomicalEntity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->ChemicalEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'caused_by'} -> ChemicalEntity\n",
      "Behavior - {'has_participant'} -> ChemicalEntity\n",
      "Behavior - {'related_to'} -> ChemicalEntity\n",
      "Behavior - {'has_upstream_actor'} -> ChemicalEntity\n",
      "Behavior - {'has_active_ingredient'} -> ChemicalEntity\n",
      "Behavior - {'primarily_composed_of'} -> ChemicalEntity\n",
      "Behavior - {'temporally_related_to'} -> ChemicalEntity\n",
      "Behavior - {'has_output'} -> ChemicalEntity\n",
      "Behavior - {'affects'} -> ChemicalEntity\n",
      "Behavior - {'similar_to'} -> ChemicalEntity\n",
      "Behavior - {'overlaps'} -> ChemicalEntity\n",
      "Behavior - {'composed_primarily_of'} -> ChemicalEntity\n",
      "Behavior - {'preceded_by'} -> ChemicalEntity\n",
      "Behavior - {'affected_by'} -> ChemicalEntity\n",
      "Behavior - {'has_substrate'} -> ChemicalEntity\n",
      "Behavior - {'interacts_with'} -> ChemicalEntity\n",
      "Behavior - {'part_of'} -> ChemicalEntity\n",
      "Behavior - {'has_capability'} -> ChemicalEntity\n",
      "Behavior - {'participates_in'} -> ChemicalEntity\n",
      "Behavior - {'actively_involves'} -> ChemicalEntity\n",
      "Behavior - {'has_input'} -> ChemicalEntity\n",
      "Behavior - {'enabled_by'} -> ChemicalEntity\n",
      "Behavior - {'has_catalyst'} -> ChemicalEntity\n",
      "Behavior - {'associated_with'} -> ChemicalEntity\n",
      "Behavior - {'has_part'} -> ChemicalEntity\n",
      "Behavior - {'precedes'} -> ChemicalEntity\n",
      "Behavior - {'subclass_of'} -> ChemicalEntity\n",
      "Behavior - {'located_in'} -> ChemicalEntity\n",
      "Behavior - {'produces'} -> ChemicalEntity\n",
      "Behavior - {'predisposes'} -> ChemicalEntity\n",
      "Behavior - {'prevents'} -> ChemicalEntity\n",
      "Behavior - {'occurs_in'} -> ChemicalEntity\n",
      "Behavior - {'manifestation_of'} -> ChemicalEntity\n",
      "Behavior - {'coexists_with'} -> ChemicalEntity\n",
      "Behavior - {'close_match'} -> ChemicalEntity\n",
      "Behavior - {'regulates'} -> ChemicalEntity\n",
      "Behavior - {'treats'} -> ChemicalEntity\n",
      "ChemicalEntity - {'correlated_with'} -> Publication\n",
      "ChemicalEntity - {'disrupts'} -> Publication\n",
      "ChemicalEntity - {'associated_with'} -> Publication\n",
      "ChemicalEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "ChemicalEntity - {'regulates'} -> Publication\n",
      "ChemicalEntity - {'located_in'} -> Publication\n",
      "ChemicalEntity - {'predisposes'} -> Publication\n",
      "ChemicalEntity - {'related_to'} -> Publication\n",
      "ChemicalEntity - {'has_part'} -> Publication\n",
      "ChemicalEntity - {'prevents'} -> Publication\n",
      "ChemicalEntity - {'affects'} -> Publication\n",
      "ChemicalEntity - {'precedes'} -> Publication\n",
      "ChemicalEntity - {'occurs_in'} -> Publication\n",
      "ChemicalEntity - {'has_metabolite'} -> Publication\n",
      "ChemicalEntity - {'close_match'} -> Publication\n",
      "ChemicalEntity - {'subclass_of'} -> Publication\n",
      "ChemicalEntity - {'interacts_with'} -> Publication\n",
      "ChemicalEntity - {'treats'} -> Publication\n",
      "ChemicalEntity - {'physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'has_input'} -> Publication\n",
      "ChemicalEntity - {'causes'} -> Publication\n",
      "ChemicalEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->OrganismalEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'caused_by'} -> OrganismalEntity\n",
      "Behavior - {'has_participant'} -> OrganismalEntity\n",
      "Behavior - {'related_to'} -> OrganismalEntity\n",
      "Behavior - {'actively_involves'} -> OrganismalEntity\n",
      "Behavior - {'has_active_ingredient'} -> OrganismalEntity\n",
      "Behavior - {'has_catalyst'} -> OrganismalEntity\n",
      "Behavior - {'primarily_composed_of'} -> OrganismalEntity\n",
      "Behavior - {'temporally_related_to'} -> OrganismalEntity\n",
      "Behavior - {'associated_with'} -> OrganismalEntity\n",
      "Behavior - {'affects'} -> OrganismalEntity\n",
      "Behavior - {'similar_to'} -> OrganismalEntity\n",
      "Behavior - {'overlaps'} -> OrganismalEntity\n",
      "Behavior - {'composed_primarily_of'} -> OrganismalEntity\n",
      "Behavior - {'has_upstream_actor'} -> OrganismalEntity\n",
      "Behavior - {'has_output'} -> OrganismalEntity\n",
      "Behavior - {'affected_by'} -> OrganismalEntity\n",
      "Behavior - {'part_of'} -> OrganismalEntity\n",
      "Behavior - {'has_capability'} -> OrganismalEntity\n",
      "Behavior - {'participates_in'} -> OrganismalEntity\n",
      "Behavior - {'has_input'} -> OrganismalEntity\n",
      "Behavior - {'preceded_by'} -> OrganismalEntity\n",
      "Behavior - {'enabled_by'} -> OrganismalEntity\n",
      "Behavior - {'has_part'} -> OrganismalEntity\n",
      "Behavior - {'has_substrate'} -> OrganismalEntity\n",
      "Behavior - {'precedes'} -> OrganismalEntity\n",
      "Behavior - {'interacts_with'} -> OrganismalEntity\n",
      "OrganismalEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->MolecularActivity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_part'} -> MolecularActivity\n",
      "Behavior - {'overlaps'} -> MolecularActivity\n",
      "Behavior - {'preceded_by'} -> MolecularActivity\n",
      "Behavior - {'related_to'} -> MolecularActivity\n",
      "Behavior - {'temporally_related_to'} -> MolecularActivity\n",
      "Behavior - {'affects'} -> MolecularActivity\n",
      "Behavior - {'similar_to'} -> MolecularActivity\n",
      "Behavior - {'composed_primarily_of'} -> MolecularActivity\n",
      "Behavior - {'has_output'} -> MolecularActivity\n",
      "Behavior - {'affected_by'} -> MolecularActivity\n",
      "Behavior - {'has_substrate'} -> MolecularActivity\n",
      "Behavior - {'interacts_with'} -> MolecularActivity\n",
      "Behavior - {'part_of'} -> MolecularActivity\n",
      "Behavior - {'has_capability'} -> MolecularActivity\n",
      "Behavior - {'participates_in'} -> MolecularActivity\n",
      "Behavior - {'caused_by'} -> MolecularActivity\n",
      "Behavior - {'has_input'} -> MolecularActivity\n",
      "Behavior - {'enabled_by'} -> MolecularActivity\n",
      "Behavior - {'has_participant'} -> MolecularActivity\n",
      "Behavior - {'has_catalyst'} -> MolecularActivity\n",
      "Behavior - {'associated_with'} -> MolecularActivity\n",
      "Behavior - {'precedes'} -> MolecularActivity\n",
      "Behavior - {'actively_involves'} -> MolecularActivity\n",
      "Behavior - {'has_upstream_actor'} -> MolecularActivity\n",
      "Behavior - {'has_active_ingredient'} -> MolecularActivity\n",
      "Behavior - {'primarily_composed_of'} -> MolecularActivity\n",
      "Behavior - {'subclass_of'} -> MolecularActivity\n",
      "MolecularActivity - {'occurs_in'} -> Publication\n",
      "MolecularActivity - {'treats'} -> Publication\n",
      "MolecularActivity - {'has_input'} -> Publication\n",
      "MolecularActivity - {'has_output'} -> Publication\n",
      "MolecularActivity - {'coexists_with'} -> Publication\n",
      "MolecularActivity - {'regulates'} -> Publication\n",
      "MolecularActivity - {'disrupts'} -> Publication\n",
      "MolecularActivity - {'associated_with'} -> Publication\n",
      "MolecularActivity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularActivity - {'located_in'} -> Publication\n",
      "MolecularActivity - {'produces'} -> Publication\n",
      "MolecularActivity - {'related_to'} -> Publication\n",
      "MolecularActivity - {'affects'} -> Publication\n",
      "MolecularActivity - {'has_participant'} -> Publication\n",
      "MolecularActivity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Pathway->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_part'} -> Pathway\n",
      "Behavior - {'overlaps'} -> Pathway\n",
      "Behavior - {'preceded_by'} -> Pathway\n",
      "Behavior - {'related_to'} -> Pathway\n",
      "Behavior - {'temporally_related_to'} -> Pathway\n",
      "Behavior - {'composed_primarily_of'} -> Pathway\n",
      "Behavior - {'has_upstream_actor'} -> Pathway\n",
      "Behavior - {'has_active_ingredient'} -> Pathway\n",
      "Behavior - {'primarily_composed_of'} -> Pathway\n",
      "Behavior - {'affected_by'} -> Pathway\n",
      "Behavior - {'participates_in'} -> Pathway\n",
      "Behavior - {'has_participant'} -> Pathway\n",
      "Behavior - {'interacts_with'} -> Pathway\n",
      "Behavior - {'part_of'} -> Pathway\n",
      "Behavior - {'has_capability'} -> Pathway\n",
      "Behavior - {'caused_by'} -> Pathway\n",
      "Behavior - {'actively_involves'} -> Pathway\n",
      "Behavior - {'associated_with'} -> Pathway\n",
      "Behavior - {'affects'} -> Pathway\n",
      "Behavior - {'similar_to'} -> Pathway\n",
      "Behavior - {'precedes'} -> Pathway\n",
      "Behavior - {'subclass_of'} -> Pathway\n",
      "Behavior - {'predisposes'} -> Pathway\n",
      "Behavior - {'prevents'} -> Pathway\n",
      "Behavior - {'manifestation_of'} -> Pathway\n",
      "Behavior - {'exacerbates'} -> Pathway\n",
      "Behavior - {'coexists_with'} -> Pathway\n",
      "Behavior - {'treats'} -> Pathway\n",
      "Behavior - {'causes'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Publication\n",
      "Pathway - {'regulates'} -> Publication\n",
      "Pathway - {'treats'} -> Publication\n",
      "Pathway - {'has_input'} -> Publication\n",
      "Pathway - {'causes'} -> Publication\n",
      "Pathway - {'associated_with'} -> Publication\n",
      "Pathway - {'gene_associated_with_condition'} -> Publication\n",
      "Pathway - {'disrupts'} -> Publication\n",
      "Pathway - {'located_in'} -> Publication\n",
      "Pathway - {'produces'} -> Publication\n",
      "Pathway - {'predisposes'} -> Publication\n",
      "Pathway - {'related_to'} -> Publication\n",
      "Pathway - {'precedes'} -> Publication\n",
      "Pathway - {'occurs_in'} -> Publication\n",
      "Pathway - {'affects'} -> Publication\n",
      "Pathway - {'manifestation_of'} -> Publication\n",
      "Pathway - {'coexists_with'} -> Publication\n",
      "Pathway - {'close_match'} -> Publication\n",
      "Pathway - {'has_participant'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Gene->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'genetic_association'} -> Gene\n",
      "Behavior - {'affected_by'} -> Gene\n",
      "Behavior - {'condition_associated_with_gene'} -> Gene\n",
      "Behavior - {'part_of'} -> Gene\n",
      "Behavior - {'participates_in'} -> Gene\n",
      "Behavior - {'has_input'} -> Gene\n",
      "Behavior - {'preceded_by'} -> Gene\n",
      "Behavior - {'enabled_by'} -> Gene\n",
      "Behavior - {'has_participant'} -> Gene\n",
      "Behavior - {'has_part'} -> Gene\n",
      "Behavior - {'has_substrate'} -> Gene\n",
      "Behavior - {'precedes'} -> Gene\n",
      "Behavior - {'interacts_with'} -> Gene\n",
      "Behavior - {'has_capability'} -> Gene\n",
      "Behavior - {'caused_by'} -> Gene\n",
      "Behavior - {'actively_involves'} -> Gene\n",
      "Behavior - {'has_catalyst'} -> Gene\n",
      "Behavior - {'temporally_related_to'} -> Gene\n",
      "Behavior - {'associated_with'} -> Gene\n",
      "Behavior - {'affects'} -> Gene\n",
      "Behavior - {'similar_to'} -> Gene\n",
      "Behavior - {'related_to'} -> Gene\n",
      "Behavior - {'overlaps'} -> Gene\n",
      "Behavior - {'composed_primarily_of'} -> Gene\n",
      "Behavior - {'has_upstream_actor'} -> Gene\n",
      "Behavior - {'gene_associated_with_condition'} -> Gene\n",
      "Behavior - {'has_active_ingredient'} -> Gene\n",
      "Behavior - {'primarily_composed_of'} -> Gene\n",
      "Behavior - {'has_output'} -> Gene\n",
      "Behavior - {'regulates'} -> Gene\n",
      "Behavior - {'treats'} -> Gene\n",
      "Behavior - {'causes'} -> Gene\n",
      "Behavior - {'disrupts'} -> Gene\n",
      "Behavior - {'gene_product_of'} -> Gene\n",
      "Behavior - {'located_in'} -> Gene\n",
      "Behavior - {'produces'} -> Gene\n",
      "Behavior - {'predisposes'} -> Gene\n",
      "Behavior - {'subclass_of'} -> Gene\n",
      "Behavior - {'prevents'} -> Gene\n",
      "Behavior - {'occurs_in'} -> Gene\n",
      "Behavior - {'manifestation_of'} -> Gene\n",
      "Behavior - {'coexists_with'} -> Gene\n",
      "Behavior - {'close_match'} -> Gene\n",
      "Gene - {'correlated_with'} -> Publication\n",
      "Gene - {'prevents'} -> Publication\n",
      "Gene - {'treats'} -> Publication\n",
      "Gene - {'has_input'} -> Publication\n",
      "Gene - {'related_to'} -> Publication\n",
      "Gene - {'occurs_in'} -> Publication\n",
      "Gene - {'affects'} -> Publication\n",
      "Gene - {'exacerbates'} -> Publication\n",
      "Gene - {'coexists_with'} -> Publication\n",
      "Gene - {'disrupts'} -> Publication\n",
      "Gene - {'gene_associated_with_condition'} -> Publication\n",
      "Gene - {'located_in'} -> Publication\n",
      "Gene - {'regulates'} -> Publication\n",
      "Gene - {'biomarker_for'} -> Publication\n",
      "Gene - {'expressed_in'} -> Publication\n",
      "Gene - {'physically_interacts_with'} -> Publication\n",
      "Gene - {'predisposes'} -> Publication\n",
      "Gene - {'causes'} -> Publication\n",
      "Gene - {'precedes'} -> Publication\n",
      "Gene - {'gene_product_of'} -> Publication\n",
      "Gene - {'close_match'} -> Publication\n",
      "Gene - {'associated_with'} -> Publication\n",
      "Gene - {'subclass_of'} -> Publication\n",
      "Gene - {'has_part'} -> Publication\n",
      "Gene - {'colocalizes_with'} -> Publication\n",
      "Gene - {'interacts_with'} -> Publication\n",
      "Gene - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->DiseaseOrPhenotypicFeature->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'actively_involves'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'has_active_ingredient'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'primarily_composed_of'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'temporally_related_to'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'affects'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'similar_to'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'related_to'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'overlaps'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'composed_primarily_of'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'has_upstream_actor'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'affected_by'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'part_of'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'has_capability'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'participates_in'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'preceded_by'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'has_participant'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'has_part'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'precedes'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'disrupts'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'located_in'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'subclass_of'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'prevents'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'predisposes'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'diagnoses'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'occurs_in'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'exacerbates'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'coexists_with'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'close_match'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'manifestation_of'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'regulates'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'treats'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'has_input'} -> DiseaseOrPhenotypicFeature\n",
      "Behavior - {'causes'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Transcript->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'actively_involves'} -> Transcript\n",
      "Behavior - {'has_upstream_actor'} -> Transcript\n",
      "Behavior - {'primarily_composed_of'} -> Transcript\n",
      "Behavior - {'affects'} -> Transcript\n",
      "Behavior - {'similar_to'} -> Transcript\n",
      "Behavior - {'related_to'} -> Transcript\n",
      "Behavior - {'overlaps'} -> Transcript\n",
      "Behavior - {'composed_primarily_of'} -> Transcript\n",
      "Behavior - {'has_output'} -> Transcript\n",
      "Behavior - {'affected_by'} -> Transcript\n",
      "Behavior - {'has_substrate'} -> Transcript\n",
      "Behavior - {'interacts_with'} -> Transcript\n",
      "Behavior - {'has_capability'} -> Transcript\n",
      "Behavior - {'participates_in'} -> Transcript\n",
      "Behavior - {'has_input'} -> Transcript\n",
      "Behavior - {'enabled_by'} -> Transcript\n",
      "Behavior - {'has_participant'} -> Transcript\n",
      "Behavior - {'has_catalyst'} -> Transcript\n",
      "Behavior - {'associated_with'} -> Transcript\n",
      "Transcript - {'translates_to'} -> Publication\n",
      "Transcript - {'regulates'} -> Publication\n",
      "Transcript - {'transcribed_from'} -> Publication\n",
      "Transcript - {'associated_with'} -> Publication\n",
      "Transcript - {'located_in'} -> Publication\n",
      "Transcript - {'subclass_of'} -> Publication\n",
      "Transcript - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Disease->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'caused_by'} -> Disease\n",
      "Behavior - {'has_active_ingredient'} -> Disease\n",
      "Behavior - {'primarily_composed_of'} -> Disease\n",
      "Behavior - {'temporally_related_to'} -> Disease\n",
      "Behavior - {'affects'} -> Disease\n",
      "Behavior - {'similar_to'} -> Disease\n",
      "Behavior - {'related_to'} -> Disease\n",
      "Behavior - {'overlaps'} -> Disease\n",
      "Behavior - {'composed_primarily_of'} -> Disease\n",
      "Behavior - {'has_upstream_actor'} -> Disease\n",
      "Behavior - {'gene_associated_with_condition'} -> Disease\n",
      "Behavior - {'affected_by'} -> Disease\n",
      "Behavior - {'interacts_with'} -> Disease\n",
      "Behavior - {'condition_associated_with_gene'} -> Disease\n",
      "Behavior - {'part_of'} -> Disease\n",
      "Behavior - {'participates_in'} -> Disease\n",
      "Behavior - {'has_participant'} -> Disease\n",
      "Behavior - {'associated_with'} -> Disease\n",
      "Behavior - {'has_part'} -> Disease\n",
      "Behavior - {'precedes'} -> Disease\n",
      "Behavior - {'genetic_association'} -> Disease\n",
      "Behavior - {'regulates'} -> Disease\n",
      "Behavior - {'treats'} -> Disease\n",
      "Behavior - {'has_input'} -> Disease\n",
      "Behavior - {'causes'} -> Disease\n",
      "Behavior - {'disrupts'} -> Disease\n",
      "Behavior - {'located_in'} -> Disease\n",
      "Behavior - {'predisposes'} -> Disease\n",
      "Behavior - {'subclass_of'} -> Disease\n",
      "Behavior - {'prevents'} -> Disease\n",
      "Behavior - {'diagnoses'} -> Disease\n",
      "Behavior - {'occurs_in'} -> Disease\n",
      "Behavior - {'manifestation_of'} -> Disease\n",
      "Behavior - {'exacerbates'} -> Disease\n",
      "Behavior - {'coexists_with'} -> Disease\n",
      "Behavior - {'close_match'} -> Disease\n",
      "Disease - {'correlated_with'} -> Publication\n",
      "Disease - {'associated_with'} -> Publication\n",
      "Disease - {'gene_associated_with_condition'} -> Publication\n",
      "Disease - {'subclass_of'} -> Publication\n",
      "Disease - {'has_part'} -> Publication\n",
      "Disease - {'interacts_with'} -> Publication\n",
      "Disease - {'produces'} -> Publication\n",
      "Disease - {'predisposes'} -> Publication\n",
      "Disease - {'related_to'} -> Publication\n",
      "Disease - {'prevents'} -> Publication\n",
      "Disease - {'diagnoses'} -> Publication\n",
      "Disease - {'contributes_to'} -> Publication\n",
      "Disease - {'manifestation_of'} -> Publication\n",
      "Disease - {'occurs_in'} -> Publication\n",
      "Disease - {'affects'} -> Publication\n",
      "Disease - {'has_input'} -> Publication\n",
      "Disease - {'exacerbates'} -> Publication\n",
      "Disease - {'coexists_with'} -> Publication\n",
      "Disease - {'close_match'} -> Publication\n",
      "Disease - {'disrupts'} -> Publication\n",
      "Disease - {'regulates'} -> Publication\n",
      "Disease - {'biomarker_for'} -> Publication\n",
      "Disease - {'located_in'} -> Publication\n",
      "Disease - {'treats'} -> Publication\n",
      "Disease - {'causes'} -> Publication\n",
      "Disease - {'disease_has_location'} -> Publication\n",
      "Disease - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->ClinicalAttribute->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'gene_associated_with_condition'} -> ClinicalAttribute\n",
      "Behavior - {'affected_by'} -> ClinicalAttribute\n",
      "Behavior - {'interacts_with'} -> ClinicalAttribute\n",
      "Behavior - {'participates_in'} -> ClinicalAttribute\n",
      "Behavior - {'has_participant'} -> ClinicalAttribute\n",
      "Behavior - {'associated_with'} -> ClinicalAttribute\n",
      "Behavior - {'genetic_association'} -> ClinicalAttribute\n",
      "Behavior - {'has_upstream_actor'} -> ClinicalAttribute\n",
      "Behavior - {'primarily_composed_of'} -> ClinicalAttribute\n",
      "Behavior - {'affects'} -> ClinicalAttribute\n",
      "Behavior - {'similar_to'} -> ClinicalAttribute\n",
      "Behavior - {'related_to'} -> ClinicalAttribute\n",
      "Behavior - {'overlaps'} -> ClinicalAttribute\n",
      "Behavior - {'condition_associated_with_gene'} -> ClinicalAttribute\n",
      "Behavior - {'composed_primarily_of'} -> ClinicalAttribute\n",
      "Behavior - {'subclass_of'} -> ClinicalAttribute\n",
      "Behavior - {'predisposes'} -> ClinicalAttribute\n",
      "Behavior - {'precedes'} -> ClinicalAttribute\n",
      "Behavior - {'coexists_with'} -> ClinicalAttribute\n",
      "Behavior - {'treats'} -> ClinicalAttribute\n",
      "ClinicalAttribute - {'correlated_with'} -> Publication\n",
      "ClinicalAttribute - {'has_input'} -> Publication\n",
      "ClinicalAttribute - {'subclass_of'} -> Publication\n",
      "ClinicalAttribute - {'related_to'} -> Publication\n",
      "ClinicalAttribute - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->InformationContentEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'overlaps'} -> InformationContentEntity\n",
      "Behavior - {'composed_primarily_of'} -> InformationContentEntity\n",
      "Behavior - {'affected_by'} -> InformationContentEntity\n",
      "Behavior - {'interacts_with'} -> InformationContentEntity\n",
      "Behavior - {'participates_in'} -> InformationContentEntity\n",
      "Behavior - {'has_participant'} -> InformationContentEntity\n",
      "Behavior - {'associated_with'} -> InformationContentEntity\n",
      "Behavior - {'has_upstream_actor'} -> InformationContentEntity\n",
      "Behavior - {'primarily_composed_of'} -> InformationContentEntity\n",
      "Behavior - {'affects'} -> InformationContentEntity\n",
      "Behavior - {'similar_to'} -> InformationContentEntity\n",
      "Behavior - {'related_to'} -> InformationContentEntity\n",
      "Behavior - {'located_in'} -> InformationContentEntity\n",
      "Behavior - {'produces'} -> InformationContentEntity\n",
      "Behavior - {'predisposes'} -> InformationContentEntity\n",
      "Behavior - {'subclass_of'} -> InformationContentEntity\n",
      "Behavior - {'has_part'} -> InformationContentEntity\n",
      "Behavior - {'prevents'} -> InformationContentEntity\n",
      "Behavior - {'occurs_in'} -> InformationContentEntity\n",
      "Behavior - {'exacerbates'} -> InformationContentEntity\n",
      "Behavior - {'precedes'} -> InformationContentEntity\n",
      "Behavior - {'coexists_with'} -> InformationContentEntity\n",
      "Behavior - {'close_match'} -> InformationContentEntity\n",
      "Behavior - {'manifestation_of'} -> InformationContentEntity\n",
      "Behavior - {'regulates'} -> InformationContentEntity\n",
      "Behavior - {'treats'} -> InformationContentEntity\n",
      "Behavior - {'has_input'} -> InformationContentEntity\n",
      "Behavior - {'causes'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Publication\n",
      "InformationContentEntity - {'subclass_of'} -> Publication\n",
      "InformationContentEntity - {'related_to'} -> Publication\n",
      "InformationContentEntity - {'disrupts'} -> Publication\n",
      "InformationContentEntity - {'associated_with'} -> Publication\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Publication\n",
      "InformationContentEntity - {'regulates'} -> Publication\n",
      "InformationContentEntity - {'biomarker_for'} -> Publication\n",
      "InformationContentEntity - {'produces'} -> Publication\n",
      "InformationContentEntity - {'has_part'} -> Publication\n",
      "InformationContentEntity - {'located_in'} -> Publication\n",
      "InformationContentEntity - {'predisposes'} -> Publication\n",
      "InformationContentEntity - {'prevents'} -> Publication\n",
      "InformationContentEntity - {'affects'} -> Publication\n",
      "InformationContentEntity - {'manifestation_of'} -> Publication\n",
      "InformationContentEntity - {'precedes'} -> Publication\n",
      "InformationContentEntity - {'close_match'} -> Publication\n",
      "InformationContentEntity - {'interacts_with'} -> Publication\n",
      "InformationContentEntity - {'diagnoses'} -> Publication\n",
      "InformationContentEntity - {'treats'} -> Publication\n",
      "InformationContentEntity - {'occurs_in'} -> Publication\n",
      "InformationContentEntity - {'has_input'} -> Publication\n",
      "InformationContentEntity - {'exacerbates'} -> Publication\n",
      "InformationContentEntity - {'causes'} -> Publication\n",
      "InformationContentEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->PhysiologicalProcess->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'affected_by'} -> PhysiologicalProcess\n",
      "Behavior - {'condition_associated_with_gene'} -> PhysiologicalProcess\n",
      "Behavior - {'gene_associated_with_condition'} -> PhysiologicalProcess\n",
      "Behavior - {'has_participant'} -> PhysiologicalProcess\n",
      "Behavior - {'interacts_with'} -> PhysiologicalProcess\n",
      "Behavior - {'participates_in'} -> PhysiologicalProcess\n",
      "Behavior - {'associated_with'} -> PhysiologicalProcess\n",
      "Behavior - {'related_to'} -> PhysiologicalProcess\n",
      "Behavior - {'overlaps'} -> PhysiologicalProcess\n",
      "Behavior - {'genetic_association'} -> PhysiologicalProcess\n",
      "Behavior - {'composed_primarily_of'} -> PhysiologicalProcess\n",
      "Behavior - {'has_upstream_actor'} -> PhysiologicalProcess\n",
      "Behavior - {'primarily_composed_of'} -> PhysiologicalProcess\n",
      "Behavior - {'affects'} -> PhysiologicalProcess\n",
      "Behavior - {'similar_to'} -> PhysiologicalProcess\n",
      "Behavior - {'has_input'} -> PhysiologicalProcess\n",
      "Behavior - {'causes'} -> PhysiologicalProcess\n",
      "Behavior - {'disrupts'} -> PhysiologicalProcess\n",
      "Behavior - {'subclass_of'} -> PhysiologicalProcess\n",
      "Behavior - {'has_part'} -> PhysiologicalProcess\n",
      "Behavior - {'predisposes'} -> PhysiologicalProcess\n",
      "Behavior - {'prevents'} -> PhysiologicalProcess\n",
      "Behavior - {'precedes'} -> PhysiologicalProcess\n",
      "Behavior - {'close_match'} -> PhysiologicalProcess\n",
      "Behavior - {'manifestation_of'} -> PhysiologicalProcess\n",
      "Behavior - {'coexists_with'} -> PhysiologicalProcess\n",
      "Behavior - {'regulates'} -> PhysiologicalProcess\n",
      "Behavior - {'treats'} -> PhysiologicalProcess\n",
      "PhysiologicalProcess - {'correlated_with'} -> Publication\n",
      "PhysiologicalProcess - {'has_input'} -> Publication\n",
      "PhysiologicalProcess - {'causes'} -> Publication\n",
      "PhysiologicalProcess - {'has_output'} -> Publication\n",
      "PhysiologicalProcess - {'disrupts'} -> Publication\n",
      "PhysiologicalProcess - {'associated_with'} -> Publication\n",
      "PhysiologicalProcess - {'located_in'} -> Publication\n",
      "PhysiologicalProcess - {'produces'} -> Publication\n",
      "PhysiologicalProcess - {'predisposes'} -> Publication\n",
      "PhysiologicalProcess - {'precedes'} -> Publication\n",
      "PhysiologicalProcess - {'related_to'} -> Publication\n",
      "PhysiologicalProcess - {'occurs_in'} -> Publication\n",
      "PhysiologicalProcess - {'affects'} -> Publication\n",
      "PhysiologicalProcess - {'coexists_with'} -> Publication\n",
      "PhysiologicalProcess - {'close_match'} -> Publication\n",
      "PhysiologicalProcess - {'has_participant'} -> Publication\n",
      "PhysiologicalProcess - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Polypeptide->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_active_ingredient'} -> Polypeptide\n",
      "Behavior - {'primarily_composed_of'} -> Polypeptide\n",
      "Behavior - {'temporally_related_to'} -> Polypeptide\n",
      "Behavior - {'has_output'} -> Polypeptide\n",
      "Behavior - {'affects'} -> Polypeptide\n",
      "Behavior - {'similar_to'} -> Polypeptide\n",
      "Behavior - {'related_to'} -> Polypeptide\n",
      "Behavior - {'overlaps'} -> Polypeptide\n",
      "Behavior - {'composed_primarily_of'} -> Polypeptide\n",
      "Behavior - {'preceded_by'} -> Polypeptide\n",
      "Behavior - {'enabled_by'} -> Polypeptide\n",
      "Behavior - {'affected_by'} -> Polypeptide\n",
      "Behavior - {'has_substrate'} -> Polypeptide\n",
      "Behavior - {'interacts_with'} -> Polypeptide\n",
      "Behavior - {'part_of'} -> Polypeptide\n",
      "Behavior - {'has_capability'} -> Polypeptide\n",
      "Behavior - {'participates_in'} -> Polypeptide\n",
      "Behavior - {'caused_by'} -> Polypeptide\n",
      "Behavior - {'actively_involves'} -> Polypeptide\n",
      "Behavior - {'has_input'} -> Polypeptide\n",
      "Behavior - {'has_participant'} -> Polypeptide\n",
      "Behavior - {'has_catalyst'} -> Polypeptide\n",
      "Behavior - {'associated_with'} -> Polypeptide\n",
      "Behavior - {'has_part'} -> Polypeptide\n",
      "Behavior - {'precedes'} -> Polypeptide\n",
      "Behavior - {'has_upstream_actor'} -> Polypeptide\n",
      "Behavior - {'coexists_with'} -> Polypeptide\n",
      "Behavior - {'located_in'} -> Polypeptide\n",
      "Behavior - {'produces'} -> Polypeptide\n",
      "Polypeptide - {'disrupts'} -> Publication\n",
      "Polypeptide - {'associated_with'} -> Publication\n",
      "Polypeptide - {'gene_associated_with_condition'} -> Publication\n",
      "Polypeptide - {'located_in'} -> Publication\n",
      "Polypeptide - {'regulates'} -> Publication\n",
      "Polypeptide - {'biomarker_for'} -> Publication\n",
      "Polypeptide - {'produces'} -> Publication\n",
      "Polypeptide - {'has_part'} -> Publication\n",
      "Polypeptide - {'physically_interacts_with'} -> Publication\n",
      "Polypeptide - {'colocalizes_with'} -> Publication\n",
      "Polypeptide - {'predisposes'} -> Publication\n",
      "Polypeptide - {'related_to'} -> Publication\n",
      "Polypeptide - {'prevents'} -> Publication\n",
      "Polypeptide - {'precedes'} -> Publication\n",
      "Polypeptide - {'close_match'} -> Publication\n",
      "Polypeptide - {'subclass_of'} -> Publication\n",
      "Polypeptide - {'interacts_with'} -> Publication\n",
      "Polypeptide - {'affects'} -> Publication\n",
      "Polypeptide - {'treats'} -> Publication\n",
      "Polypeptide - {'occurs_in'} -> Publication\n",
      "Polypeptide - {'exacerbates'} -> Publication\n",
      "Polypeptide - {'causes'} -> Publication\n",
      "Polypeptide - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->RNAProduct->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'affected_by'} -> RNAProduct\n",
      "Behavior - {'has_input'} -> RNAProduct\n",
      "Behavior - {'enabled_by'} -> RNAProduct\n",
      "Behavior - {'has_participant'} -> RNAProduct\n",
      "Behavior - {'has_substrate'} -> RNAProduct\n",
      "Behavior - {'interacts_with'} -> RNAProduct\n",
      "Behavior - {'has_capability'} -> RNAProduct\n",
      "Behavior - {'participates_in'} -> RNAProduct\n",
      "Behavior - {'actively_involves'} -> RNAProduct\n",
      "Behavior - {'has_catalyst'} -> RNAProduct\n",
      "Behavior - {'associated_with'} -> RNAProduct\n",
      "Behavior - {'related_to'} -> RNAProduct\n",
      "Behavior - {'overlaps'} -> RNAProduct\n",
      "Behavior - {'composed_primarily_of'} -> RNAProduct\n",
      "Behavior - {'has_upstream_actor'} -> RNAProduct\n",
      "Behavior - {'primarily_composed_of'} -> RNAProduct\n",
      "Behavior - {'has_output'} -> RNAProduct\n",
      "Behavior - {'affects'} -> RNAProduct\n",
      "Behavior - {'similar_to'} -> RNAProduct\n",
      "RNAProduct - {'located_in'} -> Publication\n",
      "RNAProduct - {'regulates'} -> Publication\n",
      "RNAProduct - {'related_to'} -> Publication\n",
      "RNAProduct - {'affects'} -> Publication\n",
      "RNAProduct - {'subclass_of'} -> Publication\n",
      "RNAProduct - {'disrupts'} -> Publication\n",
      "RNAProduct - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->EnvironmentalProcess->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'primarily_composed_of'} -> EnvironmentalProcess\n",
      "Behavior - {'temporally_related_to'} -> EnvironmentalProcess\n",
      "Behavior - {'affects'} -> EnvironmentalProcess\n",
      "Behavior - {'similar_to'} -> EnvironmentalProcess\n",
      "Behavior - {'related_to'} -> EnvironmentalProcess\n",
      "Behavior - {'overlaps'} -> EnvironmentalProcess\n",
      "Behavior - {'composed_primarily_of'} -> EnvironmentalProcess\n",
      "Behavior - {'has_upstream_actor'} -> EnvironmentalProcess\n",
      "Behavior - {'affected_by'} -> EnvironmentalProcess\n",
      "Behavior - {'interacts_with'} -> EnvironmentalProcess\n",
      "Behavior - {'part_of'} -> EnvironmentalProcess\n",
      "Behavior - {'has_capability'} -> EnvironmentalProcess\n",
      "Behavior - {'participates_in'} -> EnvironmentalProcess\n",
      "Behavior - {'preceded_by'} -> EnvironmentalProcess\n",
      "Behavior - {'has_participant'} -> EnvironmentalProcess\n",
      "Behavior - {'associated_with'} -> EnvironmentalProcess\n",
      "Behavior - {'has_part'} -> EnvironmentalProcess\n",
      "Behavior - {'precedes'} -> EnvironmentalProcess\n",
      "Behavior - {'actively_involves'} -> EnvironmentalProcess\n",
      "Behavior - {'has_active_ingredient'} -> EnvironmentalProcess\n",
      "EnvironmentalProcess - {'subclass_of'} -> Publication\n",
      "EnvironmentalProcess - {'related_to'} -> Publication\n",
      "EnvironmentalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Device->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_participant'} -> Device\n",
      "Behavior - {'interacts_with'} -> Device\n",
      "Behavior - {'primarily_composed_of'} -> Device\n",
      "Behavior - {'associated_with'} -> Device\n",
      "Behavior - {'affects'} -> Device\n",
      "Behavior - {'similar_to'} -> Device\n",
      "Behavior - {'related_to'} -> Device\n",
      "Behavior - {'overlaps'} -> Device\n",
      "Behavior - {'composed_primarily_of'} -> Device\n",
      "Behavior - {'has_upstream_actor'} -> Device\n",
      "Behavior - {'affected_by'} -> Device\n",
      "Behavior - {'participates_in'} -> Device\n",
      "Behavior - {'has_input'} -> Device\n",
      "Behavior - {'subclass_of'} -> Device\n",
      "Behavior - {'has_part'} -> Device\n",
      "Behavior - {'located_in'} -> Device\n",
      "Behavior - {'produces'} -> Device\n",
      "Behavior - {'predisposes'} -> Device\n",
      "Behavior - {'occurs_in'} -> Device\n",
      "Behavior - {'close_match'} -> Device\n",
      "Behavior - {'coexists_with'} -> Device\n",
      "Behavior - {'regulates'} -> Device\n",
      "Device - {'subclass_of'} -> Publication\n",
      "Device - {'diagnoses'} -> Publication\n",
      "Device - {'related_to'} -> Publication\n",
      "Device - {'occurs_in'} -> Publication\n",
      "Device - {'affects'} -> Publication\n",
      "Device - {'treats'} -> Publication\n",
      "Device - {'has_input'} -> Publication\n",
      "Device - {'coexists_with'} -> Publication\n",
      "Device - {'regulates'} -> Publication\n",
      "Device - {'disrupts'} -> Publication\n",
      "Device - {'biomarker_for'} -> Publication\n",
      "Device - {'gene_associated_with_condition'} -> Publication\n",
      "Device - {'has_part'} -> Publication\n",
      "Device - {'located_in'} -> Publication\n",
      "Device - {'causes'} -> Publication\n",
      "Device - {'precedes'} -> Publication\n",
      "Device - {'close_match'} -> Publication\n",
      "Device - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->OrganismTaxon->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'affects'} -> OrganismTaxon\n",
      "Behavior - {'similar_to'} -> OrganismTaxon\n",
      "Behavior - {'related_to'} -> OrganismTaxon\n",
      "Behavior - {'overlaps'} -> OrganismTaxon\n",
      "Behavior - {'composed_primarily_of'} -> OrganismTaxon\n",
      "Behavior - {'has_upstream_actor'} -> OrganismTaxon\n",
      "Behavior - {'affected_by'} -> OrganismTaxon\n",
      "Behavior - {'participates_in'} -> OrganismTaxon\n",
      "Behavior - {'has_participant'} -> OrganismTaxon\n",
      "Behavior - {'interacts_with'} -> OrganismTaxon\n",
      "Behavior - {'primarily_composed_of'} -> OrganismTaxon\n",
      "Behavior - {'associated_with'} -> OrganismTaxon\n",
      "Behavior - {'located_in'} -> OrganismTaxon\n",
      "Behavior - {'predisposes'} -> OrganismTaxon\n",
      "Behavior - {'subclass_of'} -> OrganismTaxon\n",
      "Behavior - {'has_part'} -> OrganismTaxon\n",
      "Behavior - {'prevents'} -> OrganismTaxon\n",
      "Behavior - {'occurs_in'} -> OrganismTaxon\n",
      "Behavior - {'coexists_with'} -> OrganismTaxon\n",
      "Behavior - {'close_match'} -> OrganismTaxon\n",
      "Behavior - {'manifestation_of'} -> OrganismTaxon\n",
      "Behavior - {'treats'} -> OrganismTaxon\n",
      "Behavior - {'has_input'} -> OrganismTaxon\n",
      "Behavior - {'in_taxon'} -> OrganismTaxon\n",
      "Behavior - {'causes'} -> OrganismTaxon\n",
      "OrganismTaxon - {'regulates'} -> Publication\n",
      "OrganismTaxon - {'gene_associated_with_condition'} -> Publication\n",
      "OrganismTaxon - {'located_in'} -> Publication\n",
      "OrganismTaxon - {'interacts_with'} -> Publication\n",
      "OrganismTaxon - {'produces'} -> Publication\n",
      "OrganismTaxon - {'related_to'} -> Publication\n",
      "OrganismTaxon - {'has_part'} -> Publication\n",
      "OrganismTaxon - {'predisposes'} -> Publication\n",
      "OrganismTaxon - {'occurs_in'} -> Publication\n",
      "OrganismTaxon - {'affects'} -> Publication\n",
      "OrganismTaxon - {'manifestation_of'} -> Publication\n",
      "OrganismTaxon - {'exacerbates'} -> Publication\n",
      "OrganismTaxon - {'coexists_with'} -> Publication\n",
      "OrganismTaxon - {'close_match'} -> Publication\n",
      "OrganismTaxon - {'subclass_of'} -> Publication\n",
      "OrganismTaxon - {'physically_interacts_with'} -> Publication\n",
      "OrganismTaxon - {'treats'} -> Publication\n",
      "OrganismTaxon - {'has_input'} -> Publication\n",
      "OrganismTaxon - {'causes'} -> Publication\n",
      "OrganismTaxon - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Cell->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'overlaps'} -> Cell\n",
      "Behavior - {'genetic_association'} -> Cell\n",
      "Behavior - {'composed_primarily_of'} -> Cell\n",
      "Behavior - {'has_upstream_actor'} -> Cell\n",
      "Behavior - {'gene_associated_with_condition'} -> Cell\n",
      "Behavior - {'has_active_ingredient'} -> Cell\n",
      "Behavior - {'primarily_composed_of'} -> Cell\n",
      "Behavior - {'has_output'} -> Cell\n",
      "Behavior - {'affected_by'} -> Cell\n",
      "Behavior - {'condition_associated_with_gene'} -> Cell\n",
      "Behavior - {'part_of'} -> Cell\n",
      "Behavior - {'participates_in'} -> Cell\n",
      "Behavior - {'has_input'} -> Cell\n",
      "Behavior - {'preceded_by'} -> Cell\n",
      "Behavior - {'enabled_by'} -> Cell\n",
      "Behavior - {'has_participant'} -> Cell\n",
      "Behavior - {'has_part'} -> Cell\n",
      "Behavior - {'has_substrate'} -> Cell\n",
      "Behavior - {'precedes'} -> Cell\n",
      "Behavior - {'interacts_with'} -> Cell\n",
      "Behavior - {'has_capability'} -> Cell\n",
      "Behavior - {'actively_involves'} -> Cell\n",
      "Behavior - {'has_catalyst'} -> Cell\n",
      "Behavior - {'temporally_related_to'} -> Cell\n",
      "Behavior - {'associated_with'} -> Cell\n",
      "Behavior - {'affects'} -> Cell\n",
      "Behavior - {'similar_to'} -> Cell\n",
      "Behavior - {'related_to'} -> Cell\n",
      "Behavior - {'located_in'} -> Cell\n",
      "Behavior - {'predisposes'} -> Cell\n",
      "Behavior - {'coexists_with'} -> Cell\n",
      "Behavior - {'regulates'} -> Cell\n",
      "Behavior - {'disrupts'} -> Cell\n",
      "Behavior - {'occurs_in'} -> Cell\n",
      "Cell - {'subclass_of'} -> Publication\n",
      "Cell - {'has_part'} -> Publication\n",
      "Cell - {'produces'} -> Publication\n",
      "Cell - {'related_to'} -> Publication\n",
      "Cell - {'occurs_in'} -> Publication\n",
      "Cell - {'affects'} -> Publication\n",
      "Cell - {'coexists_with'} -> Publication\n",
      "Cell - {'close_match'} -> Publication\n",
      "Cell - {'located_in'} -> Publication\n",
      "Cell - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Activity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'affected_by'} -> Activity\n",
      "Behavior - {'interacts_with'} -> Activity\n",
      "Behavior - {'participates_in'} -> Activity\n",
      "Behavior - {'has_participant'} -> Activity\n",
      "Behavior - {'associated_with'} -> Activity\n",
      "Behavior - {'has_upstream_actor'} -> Activity\n",
      "Behavior - {'primarily_composed_of'} -> Activity\n",
      "Behavior - {'affects'} -> Activity\n",
      "Behavior - {'similar_to'} -> Activity\n",
      "Behavior - {'related_to'} -> Activity\n",
      "Behavior - {'overlaps'} -> Activity\n",
      "Behavior - {'composed_primarily_of'} -> Activity\n",
      "Behavior - {'predisposes'} -> Activity\n",
      "Behavior - {'has_part'} -> Activity\n",
      "Behavior - {'prevents'} -> Activity\n",
      "Behavior - {'precedes'} -> Activity\n",
      "Behavior - {'occurs_in'} -> Activity\n",
      "Behavior - {'manifestation_of'} -> Activity\n",
      "Behavior - {'exacerbates'} -> Activity\n",
      "Behavior - {'coexists_with'} -> Activity\n",
      "Behavior - {'close_match'} -> Activity\n",
      "Behavior - {'regulates'} -> Activity\n",
      "Behavior - {'treats'} -> Activity\n",
      "Behavior - {'has_input'} -> Activity\n",
      "Behavior - {'causes'} -> Activity\n",
      "Behavior - {'actively_involved_in'} -> Activity\n",
      "Behavior - {'subclass_of'} -> Activity\n",
      "Activity - {'correlated_with'} -> Publication\n",
      "Activity - {'causes'} -> Publication\n",
      "Activity - {'disrupts'} -> Publication\n",
      "Activity - {'subclass_of'} -> Publication\n",
      "Activity - {'located_in'} -> Publication\n",
      "Activity - {'interacts_with'} -> Publication\n",
      "Activity - {'produces'} -> Publication\n",
      "Activity - {'has_part'} -> Publication\n",
      "Activity - {'diagnoses'} -> Publication\n",
      "Activity - {'prevents'} -> Publication\n",
      "Activity - {'predisposes'} -> Publication\n",
      "Activity - {'precedes'} -> Publication\n",
      "Activity - {'related_to'} -> Publication\n",
      "Activity - {'occurs_in'} -> Publication\n",
      "Activity - {'affects'} -> Publication\n",
      "Activity - {'coexists_with'} -> Publication\n",
      "Activity - {'close_match'} -> Publication\n",
      "Activity - {'regulates'} -> Publication\n",
      "Activity - {'treats'} -> Publication\n",
      "Activity - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Annotation->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'associated_with'} -> Annotation\n",
      "Behavior - {'related_to'} -> Annotation\n",
      "Behavior - {'precedes'} -> Annotation\n",
      "Behavior - {'overlaps'} -> Annotation\n",
      "Behavior - {'genetic_association'} -> Annotation\n",
      "Behavior - {'composed_primarily_of'} -> Annotation\n",
      "Behavior - {'has_upstream_actor'} -> Annotation\n",
      "Behavior - {'has_active_ingredient'} -> Annotation\n",
      "Behavior - {'primarily_composed_of'} -> Annotation\n",
      "Behavior - {'temporally_related_to'} -> Annotation\n",
      "Behavior - {'affects'} -> Annotation\n",
      "Behavior - {'similar_to'} -> Annotation\n",
      "Behavior - {'affected_by'} -> Annotation\n",
      "Behavior - {'condition_associated_with_gene'} -> Annotation\n",
      "Behavior - {'gene_associated_with_condition'} -> Annotation\n",
      "Behavior - {'preceded_by'} -> Annotation\n",
      "Behavior - {'has_participant'} -> Annotation\n",
      "Behavior - {'has_part'} -> Annotation\n",
      "Behavior - {'interacts_with'} -> Annotation\n",
      "Behavior - {'part_of'} -> Annotation\n",
      "Behavior - {'has_capability'} -> Annotation\n",
      "Behavior - {'participates_in'} -> Annotation\n",
      "Behavior - {'actively_involves'} -> Annotation\n",
      "Annotation - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->OrganismAttribute->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'genetic_association'} -> OrganismAttribute\n",
      "Behavior - {'actively_involves'} -> OrganismAttribute\n",
      "Behavior - {'has_upstream_actor'} -> OrganismAttribute\n",
      "Behavior - {'has_active_ingredient'} -> OrganismAttribute\n",
      "Behavior - {'primarily_composed_of'} -> OrganismAttribute\n",
      "Behavior - {'temporally_related_to'} -> OrganismAttribute\n",
      "Behavior - {'affects'} -> OrganismAttribute\n",
      "Behavior - {'similar_to'} -> OrganismAttribute\n",
      "Behavior - {'related_to'} -> OrganismAttribute\n",
      "Behavior - {'overlaps'} -> OrganismAttribute\n",
      "Behavior - {'composed_primarily_of'} -> OrganismAttribute\n",
      "Behavior - {'gene_associated_with_condition'} -> OrganismAttribute\n",
      "Behavior - {'affected_by'} -> OrganismAttribute\n",
      "Behavior - {'interacts_with'} -> OrganismAttribute\n",
      "Behavior - {'condition_associated_with_gene'} -> OrganismAttribute\n",
      "Behavior - {'part_of'} -> OrganismAttribute\n",
      "Behavior - {'has_capability'} -> OrganismAttribute\n",
      "Behavior - {'participates_in'} -> OrganismAttribute\n",
      "Behavior - {'preceded_by'} -> OrganismAttribute\n",
      "Behavior - {'has_participant'} -> OrganismAttribute\n",
      "Behavior - {'associated_with'} -> OrganismAttribute\n",
      "Behavior - {'has_part'} -> OrganismAttribute\n",
      "Behavior - {'precedes'} -> OrganismAttribute\n",
      "Behavior - {'occurs_in'} -> OrganismAttribute\n",
      "Behavior - {'treats'} -> OrganismAttribute\n",
      "Behavior - {'subclass_of'} -> OrganismAttribute\n",
      "OrganismAttribute - {'related_to'} -> Publication\n",
      "OrganismAttribute - {'produces'} -> Publication\n",
      "OrganismAttribute - {'has_input'} -> Publication\n",
      "OrganismAttribute - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Drug->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_capability'} -> Drug\n",
      "Behavior - {'participates_in'} -> Drug\n",
      "Behavior - {'has_input'} -> Drug\n",
      "Behavior - {'enabled_by'} -> Drug\n",
      "Behavior - {'has_participant'} -> Drug\n",
      "Behavior - {'has_catalyst'} -> Drug\n",
      "Behavior - {'associated_with'} -> Drug\n",
      "Behavior - {'precedes'} -> Drug\n",
      "Behavior - {'actively_involves'} -> Drug\n",
      "Behavior - {'has_upstream_actor'} -> Drug\n",
      "Behavior - {'primarily_composed_of'} -> Drug\n",
      "Behavior - {'temporally_related_to'} -> Drug\n",
      "Behavior - {'affects'} -> Drug\n",
      "Behavior - {'similar_to'} -> Drug\n",
      "Behavior - {'related_to'} -> Drug\n",
      "Behavior - {'overlaps'} -> Drug\n",
      "Behavior - {'composed_primarily_of'} -> Drug\n",
      "Behavior - {'preceded_by'} -> Drug\n",
      "Behavior - {'has_output'} -> Drug\n",
      "Behavior - {'affected_by'} -> Drug\n",
      "Behavior - {'has_substrate'} -> Drug\n",
      "Behavior - {'interacts_with'} -> Drug\n",
      "Behavior - {'located_in'} -> Drug\n",
      "Behavior - {'produces'} -> Drug\n",
      "Behavior - {'predisposes'} -> Drug\n",
      "Behavior - {'prevents'} -> Drug\n",
      "Behavior - {'occurs_in'} -> Drug\n",
      "Behavior - {'close_match'} -> Drug\n",
      "Behavior - {'manifestation_of'} -> Drug\n",
      "Behavior - {'coexists_with'} -> Drug\n",
      "Behavior - {'regulates'} -> Drug\n",
      "Behavior - {'treats'} -> Drug\n",
      "Behavior - {'subclass_of'} -> Drug\n",
      "Behavior - {'has_part'} -> Drug\n",
      "Drug - {'correlated_with'} -> Publication\n",
      "Drug - {'regulates'} -> Publication\n",
      "Drug - {'treats'} -> Publication\n",
      "Drug - {'has_input'} -> Publication\n",
      "Drug - {'physically_interacts_with'} -> Publication\n",
      "Drug - {'causes'} -> Publication\n",
      "Drug - {'directly_physically_interacts_with'} -> Publication\n",
      "Drug - {'disrupts'} -> Publication\n",
      "Drug - {'associated_with'} -> Publication\n",
      "Drug - {'gene_associated_with_condition'} -> Publication\n",
      "Drug - {'located_in'} -> Publication\n",
      "Drug - {'has_part'} -> Publication\n",
      "Drug - {'prevents'} -> Publication\n",
      "Drug - {'colocalizes_with'} -> Publication\n",
      "Drug - {'interacts_with'} -> Publication\n",
      "Drug - {'predisposes'} -> Publication\n",
      "Drug - {'related_to'} -> Publication\n",
      "Drug - {'occurs_in'} -> Publication\n",
      "Drug - {'precedes'} -> Publication\n",
      "Drug - {'affects'} -> Publication\n",
      "Drug - {'close_match'} -> Publication\n",
      "Drug - {'subclass_of'} -> Publication\n",
      "Drug - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Phenomenon->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'related_to'} -> Phenomenon\n",
      "Behavior - {'has_input'} -> Phenomenon\n",
      "Behavior - {'causes'} -> Phenomenon\n",
      "Behavior - {'disrupts'} -> Phenomenon\n",
      "Behavior - {'associated_with'} -> Phenomenon\n",
      "Behavior - {'predisposes'} -> Phenomenon\n",
      "Behavior - {'subclass_of'} -> Phenomenon\n",
      "Behavior - {'has_part'} -> Phenomenon\n",
      "Behavior - {'prevents'} -> Phenomenon\n",
      "Behavior - {'occurs_in'} -> Phenomenon\n",
      "Behavior - {'affects'} -> Phenomenon\n",
      "Behavior - {'precedes'} -> Phenomenon\n",
      "Behavior - {'coexists_with'} -> Phenomenon\n",
      "Behavior - {'close_match'} -> Phenomenon\n",
      "Behavior - {'has_participant'} -> Phenomenon\n",
      "Behavior - {'manifestation_of'} -> Phenomenon\n",
      "Behavior - {'regulates'} -> Phenomenon\n",
      "Behavior - {'treats'} -> Phenomenon\n",
      "Phenomenon - {'associated_with'} -> Publication\n",
      "Phenomenon - {'subclass_of'} -> Publication\n",
      "Phenomenon - {'interacts_with'} -> Publication\n",
      "Phenomenon - {'produces'} -> Publication\n",
      "Phenomenon - {'related_to'} -> Publication\n",
      "Phenomenon - {'has_part'} -> Publication\n",
      "Phenomenon - {'diagnoses'} -> Publication\n",
      "Phenomenon - {'occurs_in'} -> Publication\n",
      "Phenomenon - {'affects'} -> Publication\n",
      "Phenomenon - {'coexists_with'} -> Publication\n",
      "Phenomenon - {'close_match'} -> Publication\n",
      "Phenomenon - {'disrupts'} -> Publication\n",
      "Phenomenon - {'regulates'} -> Publication\n",
      "Phenomenon - {'located_in'} -> Publication\n",
      "Phenomenon - {'has_input'} -> Publication\n",
      "Phenomenon - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Procedure->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_upstream_actor'} -> Procedure\n",
      "Behavior - {'primarily_composed_of'} -> Procedure\n",
      "Behavior - {'affects'} -> Procedure\n",
      "Behavior - {'similar_to'} -> Procedure\n",
      "Behavior - {'related_to'} -> Procedure\n",
      "Behavior - {'overlaps'} -> Procedure\n",
      "Behavior - {'composed_primarily_of'} -> Procedure\n",
      "Behavior - {'affected_by'} -> Procedure\n",
      "Behavior - {'interacts_with'} -> Procedure\n",
      "Behavior - {'participates_in'} -> Procedure\n",
      "Behavior - {'has_participant'} -> Procedure\n",
      "Behavior - {'associated_with'} -> Procedure\n",
      "Behavior - {'prevents'} -> Procedure\n",
      "Behavior - {'precedes'} -> Procedure\n",
      "Behavior - {'occurs_in'} -> Procedure\n",
      "Behavior - {'coexists_with'} -> Procedure\n",
      "Behavior - {'close_match'} -> Procedure\n",
      "Behavior - {'regulates'} -> Procedure\n",
      "Behavior - {'treats'} -> Procedure\n",
      "Behavior - {'has_input'} -> Procedure\n",
      "Behavior - {'causes'} -> Procedure\n",
      "Behavior - {'subclass_of'} -> Procedure\n",
      "Behavior - {'predisposes'} -> Procedure\n",
      "Procedure - {'correlated_with'} -> Publication\n",
      "Procedure - {'manifestation_of'} -> Publication\n",
      "Procedure - {'close_match'} -> Publication\n",
      "Procedure - {'subclass_of'} -> Publication\n",
      "Procedure - {'interacts_with'} -> Publication\n",
      "Procedure - {'diagnoses'} -> Publication\n",
      "Procedure - {'occurs_in'} -> Publication\n",
      "Procedure - {'treats'} -> Publication\n",
      "Procedure - {'has_input'} -> Publication\n",
      "Procedure - {'exacerbates'} -> Publication\n",
      "Procedure - {'causes'} -> Publication\n",
      "Procedure - {'coexists_with'} -> Publication\n",
      "Procedure - {'disrupts'} -> Publication\n",
      "Procedure - {'associated_with'} -> Publication\n",
      "Procedure - {'regulates'} -> Publication\n",
      "Procedure - {'produces'} -> Publication\n",
      "Procedure - {'located_in'} -> Publication\n",
      "Procedure - {'predisposes'} -> Publication\n",
      "Procedure - {'related_to'} -> Publication\n",
      "Procedure - {'has_part'} -> Publication\n",
      "Procedure - {'prevents'} -> Publication\n",
      "Procedure - {'precedes'} -> Publication\n",
      "Procedure - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->PhenotypicFeature->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_active_ingredient'} -> PhenotypicFeature\n",
      "Behavior - {'primarily_composed_of'} -> PhenotypicFeature\n",
      "Behavior - {'temporally_related_to'} -> PhenotypicFeature\n",
      "Behavior - {'affects'} -> PhenotypicFeature\n",
      "Behavior - {'similar_to'} -> PhenotypicFeature\n",
      "Behavior - {'related_to'} -> PhenotypicFeature\n",
      "Behavior - {'overlaps'} -> PhenotypicFeature\n",
      "Behavior - {'condition_associated_with_gene'} -> PhenotypicFeature\n",
      "Behavior - {'composed_primarily_of'} -> PhenotypicFeature\n",
      "Behavior - {'gene_associated_with_condition'} -> PhenotypicFeature\n",
      "Behavior - {'preceded_by'} -> PhenotypicFeature\n",
      "Behavior - {'affected_by'} -> PhenotypicFeature\n",
      "Behavior - {'interacts_with'} -> PhenotypicFeature\n",
      "Behavior - {'part_of'} -> PhenotypicFeature\n",
      "Behavior - {'has_capability'} -> PhenotypicFeature\n",
      "Behavior - {'participates_in'} -> PhenotypicFeature\n",
      "Behavior - {'has_participant'} -> PhenotypicFeature\n",
      "Behavior - {'associated_with'} -> PhenotypicFeature\n",
      "Behavior - {'has_part'} -> PhenotypicFeature\n",
      "Behavior - {'precedes'} -> PhenotypicFeature\n",
      "Behavior - {'genetic_association'} -> PhenotypicFeature\n",
      "Behavior - {'actively_involves'} -> PhenotypicFeature\n",
      "Behavior - {'has_upstream_actor'} -> PhenotypicFeature\n",
      "Behavior - {'treats'} -> PhenotypicFeature\n",
      "Behavior - {'has_input'} -> PhenotypicFeature\n",
      "Behavior - {'causes'} -> PhenotypicFeature\n",
      "Behavior - {'disrupts'} -> PhenotypicFeature\n",
      "Behavior - {'subclass_of'} -> PhenotypicFeature\n",
      "Behavior - {'located_in'} -> PhenotypicFeature\n",
      "Behavior - {'predisposes'} -> PhenotypicFeature\n",
      "Behavior - {'prevents'} -> PhenotypicFeature\n",
      "Behavior - {'diagnoses'} -> PhenotypicFeature\n",
      "Behavior - {'occurs_in'} -> PhenotypicFeature\n",
      "Behavior - {'close_match'} -> PhenotypicFeature\n",
      "Behavior - {'manifestation_of'} -> PhenotypicFeature\n",
      "Behavior - {'exacerbates'} -> PhenotypicFeature\n",
      "Behavior - {'coexists_with'} -> PhenotypicFeature\n",
      "Behavior - {'regulates'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Publication\n",
      "PhenotypicFeature - {'occurs_in'} -> Publication\n",
      "PhenotypicFeature - {'affects'} -> Publication\n",
      "PhenotypicFeature - {'close_match'} -> Publication\n",
      "PhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "PhenotypicFeature - {'exacerbates'} -> Publication\n",
      "PhenotypicFeature - {'coexists_with'} -> Publication\n",
      "PhenotypicFeature - {'subclass_of'} -> Publication\n",
      "PhenotypicFeature - {'regulates'} -> Publication\n",
      "PhenotypicFeature - {'diagnoses'} -> Publication\n",
      "PhenotypicFeature - {'treats'} -> Publication\n",
      "PhenotypicFeature - {'has_input'} -> Publication\n",
      "PhenotypicFeature - {'causes'} -> Publication\n",
      "PhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "PhenotypicFeature - {'disrupts'} -> Publication\n",
      "PhenotypicFeature - {'associated_with'} -> Publication\n",
      "PhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "PhenotypicFeature - {'has_part'} -> Publication\n",
      "PhenotypicFeature - {'located_in'} -> Publication\n",
      "PhenotypicFeature - {'interacts_with'} -> Publication\n",
      "PhenotypicFeature - {'produces'} -> Publication\n",
      "PhenotypicFeature - {'prevents'} -> Publication\n",
      "PhenotypicFeature - {'contributes_to'} -> Publication\n",
      "PhenotypicFeature - {'predisposes'} -> Publication\n",
      "PhenotypicFeature - {'precedes'} -> Publication\n",
      "PhenotypicFeature - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->IndividualOrganism->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'overlaps'} -> IndividualOrganism\n",
      "Behavior - {'composed_primarily_of'} -> IndividualOrganism\n",
      "Behavior - {'has_upstream_actor'} -> IndividualOrganism\n",
      "Behavior - {'has_output'} -> IndividualOrganism\n",
      "Behavior - {'affected_by'} -> IndividualOrganism\n",
      "Behavior - {'participates_in'} -> IndividualOrganism\n",
      "Behavior - {'has_input'} -> IndividualOrganism\n",
      "Behavior - {'enabled_by'} -> IndividualOrganism\n",
      "Behavior - {'has_participant'} -> IndividualOrganism\n",
      "Behavior - {'has_substrate'} -> IndividualOrganism\n",
      "Behavior - {'interacts_with'} -> IndividualOrganism\n",
      "Behavior - {'has_capability'} -> IndividualOrganism\n",
      "Behavior - {'actively_involves'} -> IndividualOrganism\n",
      "Behavior - {'has_catalyst'} -> IndividualOrganism\n",
      "Behavior - {'primarily_composed_of'} -> IndividualOrganism\n",
      "Behavior - {'associated_with'} -> IndividualOrganism\n",
      "Behavior - {'affects'} -> IndividualOrganism\n",
      "Behavior - {'similar_to'} -> IndividualOrganism\n",
      "Behavior - {'related_to'} -> IndividualOrganism\n",
      "Behavior - {'subclass_of'} -> IndividualOrganism\n",
      "Behavior - {'has_part'} -> IndividualOrganism\n",
      "Behavior - {'occurs_in'} -> IndividualOrganism\n",
      "IndividualOrganism - {'close_match'} -> Publication\n",
      "IndividualOrganism - {'subclass_of'} -> Publication\n",
      "IndividualOrganism - {'occurs_in'} -> Publication\n",
      "IndividualOrganism - {'located_in'} -> Publication\n",
      "IndividualOrganism - {'related_to'} -> Publication\n",
      "IndividualOrganism - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->CellularComponent->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'affected_by'} -> CellularComponent\n",
      "Behavior - {'has_substrate'} -> CellularComponent\n",
      "Behavior - {'interacts_with'} -> CellularComponent\n",
      "Behavior - {'part_of'} -> CellularComponent\n",
      "Behavior - {'has_capability'} -> CellularComponent\n",
      "Behavior - {'participates_in'} -> CellularComponent\n",
      "Behavior - {'caused_by'} -> CellularComponent\n",
      "Behavior - {'has_input'} -> CellularComponent\n",
      "Behavior - {'enabled_by'} -> CellularComponent\n",
      "Behavior - {'has_participant'} -> CellularComponent\n",
      "Behavior - {'has_catalyst'} -> CellularComponent\n",
      "Behavior - {'associated_with'} -> CellularComponent\n",
      "Behavior - {'has_part'} -> CellularComponent\n",
      "Behavior - {'precedes'} -> CellularComponent\n",
      "Behavior - {'genetic_association'} -> CellularComponent\n",
      "Behavior - {'actively_involves'} -> CellularComponent\n",
      "Behavior - {'has_upstream_actor'} -> CellularComponent\n",
      "Behavior - {'has_active_ingredient'} -> CellularComponent\n",
      "Behavior - {'primarily_composed_of'} -> CellularComponent\n",
      "Behavior - {'temporally_related_to'} -> CellularComponent\n",
      "Behavior - {'affects'} -> CellularComponent\n",
      "Behavior - {'similar_to'} -> CellularComponent\n",
      "Behavior - {'related_to'} -> CellularComponent\n",
      "Behavior - {'overlaps'} -> CellularComponent\n",
      "Behavior - {'condition_associated_with_gene'} -> CellularComponent\n",
      "Behavior - {'composed_primarily_of'} -> CellularComponent\n",
      "Behavior - {'gene_associated_with_condition'} -> CellularComponent\n",
      "Behavior - {'preceded_by'} -> CellularComponent\n",
      "Behavior - {'has_output'} -> CellularComponent\n",
      "Behavior - {'located_in'} -> CellularComponent\n",
      "Behavior - {'predisposes'} -> CellularComponent\n",
      "Behavior - {'coexists_with'} -> CellularComponent\n",
      "CellularComponent - {'subclass_of'} -> Publication\n",
      "CellularComponent - {'produces'} -> Publication\n",
      "CellularComponent - {'has_part'} -> Publication\n",
      "CellularComponent - {'related_to'} -> Publication\n",
      "CellularComponent - {'occurs_in'} -> Publication\n",
      "CellularComponent - {'affects'} -> Publication\n",
      "CellularComponent - {'regulates'} -> Publication\n",
      "CellularComponent - {'disrupts'} -> Publication\n",
      "CellularComponent - {'located_in'} -> Publication\n",
      "CellularComponent - {'associated_with'} -> Publication\n",
      "CellularComponent - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->MolecularEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'part_of'} -> MolecularEntity\n",
      "Behavior - {'has_capability'} -> MolecularEntity\n",
      "Behavior - {'participates_in'} -> MolecularEntity\n",
      "Behavior - {'has_input'} -> MolecularEntity\n",
      "Behavior - {'preceded_by'} -> MolecularEntity\n",
      "Behavior - {'enabled_by'} -> MolecularEntity\n",
      "Behavior - {'has_participant'} -> MolecularEntity\n",
      "Behavior - {'associated_with'} -> MolecularEntity\n",
      "Behavior - {'has_part'} -> MolecularEntity\n",
      "Behavior - {'has_substrate'} -> MolecularEntity\n",
      "Behavior - {'precedes'} -> MolecularEntity\n",
      "Behavior - {'caused_by'} -> MolecularEntity\n",
      "Behavior - {'actively_involves'} -> MolecularEntity\n",
      "Behavior - {'has_active_ingredient'} -> MolecularEntity\n",
      "Behavior - {'has_catalyst'} -> MolecularEntity\n",
      "Behavior - {'primarily_composed_of'} -> MolecularEntity\n",
      "Behavior - {'temporally_related_to'} -> MolecularEntity\n",
      "Behavior - {'affects'} -> MolecularEntity\n",
      "Behavior - {'similar_to'} -> MolecularEntity\n",
      "Behavior - {'related_to'} -> MolecularEntity\n",
      "Behavior - {'overlaps'} -> MolecularEntity\n",
      "Behavior - {'composed_primarily_of'} -> MolecularEntity\n",
      "Behavior - {'has_upstream_actor'} -> MolecularEntity\n",
      "Behavior - {'has_output'} -> MolecularEntity\n",
      "Behavior - {'affected_by'} -> MolecularEntity\n",
      "Behavior - {'interacts_with'} -> MolecularEntity\n",
      "Behavior - {'produces'} -> MolecularEntity\n",
      "Behavior - {'occurs_in'} -> MolecularEntity\n",
      "Behavior - {'coexists_with'} -> MolecularEntity\n",
      "Behavior - {'regulates'} -> MolecularEntity\n",
      "Behavior - {'located_in'} -> MolecularEntity\n",
      "Behavior - {'subclass_of'} -> MolecularEntity\n",
      "MolecularEntity - {'causes'} -> Publication\n",
      "MolecularEntity - {'associated_with'} -> Publication\n",
      "MolecularEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "MolecularEntity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularEntity - {'regulates'} -> Publication\n",
      "MolecularEntity - {'disrupts'} -> Publication\n",
      "MolecularEntity - {'located_in'} -> Publication\n",
      "MolecularEntity - {'biomarker_for'} -> Publication\n",
      "MolecularEntity - {'interacts_with'} -> Publication\n",
      "MolecularEntity - {'predisposes'} -> Publication\n",
      "MolecularEntity - {'related_to'} -> Publication\n",
      "MolecularEntity - {'has_part'} -> Publication\n",
      "MolecularEntity - {'occurs_in'} -> Publication\n",
      "MolecularEntity - {'affects'} -> Publication\n",
      "MolecularEntity - {'precedes'} -> Publication\n",
      "MolecularEntity - {'coexists_with'} -> Publication\n",
      "MolecularEntity - {'close_match'} -> Publication\n",
      "MolecularEntity - {'subclass_of'} -> Publication\n",
      "MolecularEntity - {'treats'} -> Publication\n",
      "MolecularEntity - {'physically_interacts_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->GrossAnatomicalStructure->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'similar_to'} -> GrossAnatomicalStructure\n",
      "Behavior - {'related_to'} -> GrossAnatomicalStructure\n",
      "Behavior - {'associated_with'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_part'} -> GrossAnatomicalStructure\n",
      "Behavior - {'precedes'} -> GrossAnatomicalStructure\n",
      "Behavior - {'actively_involves'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_upstream_actor'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_active_ingredient'} -> GrossAnatomicalStructure\n",
      "Behavior - {'primarily_composed_of'} -> GrossAnatomicalStructure\n",
      "Behavior - {'temporally_related_to'} -> GrossAnatomicalStructure\n",
      "Behavior - {'affects'} -> GrossAnatomicalStructure\n",
      "Behavior - {'overlaps'} -> GrossAnatomicalStructure\n",
      "Behavior - {'composed_primarily_of'} -> GrossAnatomicalStructure\n",
      "Behavior - {'preceded_by'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_output'} -> GrossAnatomicalStructure\n",
      "Behavior - {'affected_by'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_substrate'} -> GrossAnatomicalStructure\n",
      "Behavior - {'interacts_with'} -> GrossAnatomicalStructure\n",
      "Behavior - {'part_of'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_capability'} -> GrossAnatomicalStructure\n",
      "Behavior - {'participates_in'} -> GrossAnatomicalStructure\n",
      "Behavior - {'caused_by'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_input'} -> GrossAnatomicalStructure\n",
      "Behavior - {'enabled_by'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_participant'} -> GrossAnatomicalStructure\n",
      "Behavior - {'has_catalyst'} -> GrossAnatomicalStructure\n",
      "Behavior - {'subclass_of'} -> GrossAnatomicalStructure\n",
      "Behavior - {'located_in'} -> GrossAnatomicalStructure\n",
      "Behavior - {'produces'} -> GrossAnatomicalStructure\n",
      "Behavior - {'occurs_in'} -> GrossAnatomicalStructure\n",
      "Behavior - {'treats'} -> GrossAnatomicalStructure\n",
      "GrossAnatomicalStructure - {'affects'} -> Publication\n",
      "GrossAnatomicalStructure - {'coexists_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'close_match'} -> Publication\n",
      "GrossAnatomicalStructure - {'develops_from'} -> Publication\n",
      "GrossAnatomicalStructure - {'regulates'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_input'} -> Publication\n",
      "GrossAnatomicalStructure - {'associated_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'located_in'} -> Publication\n",
      "GrossAnatomicalStructure - {'subclass_of'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_part'} -> Publication\n",
      "GrossAnatomicalStructure - {'produces'} -> Publication\n",
      "GrossAnatomicalStructure - {'related_to'} -> Publication\n",
      "GrossAnatomicalStructure - {'occurs_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->PathologicalProcess->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'associated_with'} -> PathologicalProcess\n",
      "Behavior - {'affects'} -> PathologicalProcess\n",
      "Behavior - {'related_to'} -> PathologicalProcess\n",
      "Behavior - {'coexists_with'} -> PathologicalProcess\n",
      "Behavior - {'close_match'} -> PathologicalProcess\n",
      "Behavior - {'manifestation_of'} -> PathologicalProcess\n",
      "Behavior - {'treats'} -> PathologicalProcess\n",
      "Behavior - {'causes'} -> PathologicalProcess\n",
      "Behavior - {'disrupts'} -> PathologicalProcess\n",
      "Behavior - {'predisposes'} -> PathologicalProcess\n",
      "Behavior - {'subclass_of'} -> PathologicalProcess\n",
      "Behavior - {'prevents'} -> PathologicalProcess\n",
      "Behavior - {'exacerbates'} -> PathologicalProcess\n",
      "Behavior - {'precedes'} -> PathologicalProcess\n",
      "PathologicalProcess - {'manifestation_of'} -> Publication\n",
      "PathologicalProcess - {'occurs_in'} -> Publication\n",
      "PathologicalProcess - {'coexists_with'} -> Publication\n",
      "PathologicalProcess - {'exacerbates'} -> Publication\n",
      "PathologicalProcess - {'disrupts'} -> Publication\n",
      "PathologicalProcess - {'located_in'} -> Publication\n",
      "PathologicalProcess - {'regulates'} -> Publication\n",
      "PathologicalProcess - {'treats'} -> Publication\n",
      "PathologicalProcess - {'precedes'} -> Publication\n",
      "PathologicalProcess - {'causes'} -> Publication\n",
      "PathologicalProcess - {'close_match'} -> Publication\n",
      "PathologicalProcess - {'associated_with'} -> Publication\n",
      "PathologicalProcess - {'subclass_of'} -> Publication\n",
      "PathologicalProcess - {'produces'} -> Publication\n",
      "PathologicalProcess - {'predisposes'} -> Publication\n",
      "PathologicalProcess - {'related_to'} -> Publication\n",
      "PathologicalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->GeneFamily->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_upstream_actor'} -> GeneFamily\n",
      "Behavior - {'affected_by'} -> GeneFamily\n",
      "Behavior - {'participates_in'} -> GeneFamily\n",
      "Behavior - {'has_participant'} -> GeneFamily\n",
      "Behavior - {'interacts_with'} -> GeneFamily\n",
      "Behavior - {'primarily_composed_of'} -> GeneFamily\n",
      "Behavior - {'associated_with'} -> GeneFamily\n",
      "Behavior - {'affects'} -> GeneFamily\n",
      "Behavior - {'similar_to'} -> GeneFamily\n",
      "Behavior - {'related_to'} -> GeneFamily\n",
      "Behavior - {'overlaps'} -> GeneFamily\n",
      "Behavior - {'composed_primarily_of'} -> GeneFamily\n",
      "Behavior - {'produces'} -> GeneFamily\n",
      "GeneFamily - {'treats'} -> Publication\n",
      "GeneFamily - {'causes'} -> Publication\n",
      "GeneFamily - {'associated_with'} -> Publication\n",
      "GeneFamily - {'disrupts'} -> Publication\n",
      "GeneFamily - {'gene_associated_with_condition'} -> Publication\n",
      "GeneFamily - {'located_in'} -> Publication\n",
      "GeneFamily - {'regulates'} -> Publication\n",
      "GeneFamily - {'predisposes'} -> Publication\n",
      "GeneFamily - {'occurs_in'} -> Publication\n",
      "GeneFamily - {'affects'} -> Publication\n",
      "GeneFamily - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->GenomicEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'enabled_by'} -> GenomicEntity\n",
      "Behavior - {'has_part'} -> GenomicEntity\n",
      "Behavior - {'precedes'} -> GenomicEntity\n",
      "Behavior - {'actively_involves'} -> GenomicEntity\n",
      "Behavior - {'has_upstream_actor'} -> GenomicEntity\n",
      "Behavior - {'has_active_ingredient'} -> GenomicEntity\n",
      "Behavior - {'primarily_composed_of'} -> GenomicEntity\n",
      "Behavior - {'temporally_related_to'} -> GenomicEntity\n",
      "Behavior - {'affects'} -> GenomicEntity\n",
      "Behavior - {'similar_to'} -> GenomicEntity\n",
      "Behavior - {'related_to'} -> GenomicEntity\n",
      "Behavior - {'overlaps'} -> GenomicEntity\n",
      "Behavior - {'composed_primarily_of'} -> GenomicEntity\n",
      "Behavior - {'preceded_by'} -> GenomicEntity\n",
      "Behavior - {'has_output'} -> GenomicEntity\n",
      "Behavior - {'affected_by'} -> GenomicEntity\n",
      "Behavior - {'has_substrate'} -> GenomicEntity\n",
      "Behavior - {'interacts_with'} -> GenomicEntity\n",
      "Behavior - {'part_of'} -> GenomicEntity\n",
      "Behavior - {'has_capability'} -> GenomicEntity\n",
      "Behavior - {'participates_in'} -> GenomicEntity\n",
      "Behavior - {'caused_by'} -> GenomicEntity\n",
      "Behavior - {'has_input'} -> GenomicEntity\n",
      "Behavior - {'has_participant'} -> GenomicEntity\n",
      "Behavior - {'has_catalyst'} -> GenomicEntity\n",
      "Behavior - {'associated_with'} -> GenomicEntity\n",
      "GenomicEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->CellLine->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'affects'} -> CellLine\n",
      "Behavior - {'related_to'} -> CellLine\n",
      "Behavior - {'disrupts'} -> CellLine\n",
      "Behavior - {'located_in'} -> CellLine\n",
      "Behavior - {'has_part'} -> CellLine\n",
      "Behavior - {'has_participant'} -> CellLine\n",
      "Behavior - {'regulates'} -> CellLine\n",
      "CellLine - {'gene_associated_with_condition'} -> Publication\n",
      "CellLine - {'located_in'} -> Publication\n",
      "CellLine - {'produces'} -> Publication\n",
      "CellLine - {'related_to'} -> Publication\n",
      "CellLine - {'affects'} -> Publication\n",
      "CellLine - {'close_match'} -> Publication\n",
      "CellLine - {'subclass_of'} -> Publication\n",
      "CellLine - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->ChemicalMixture->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'actively_involves'} -> ChemicalMixture\n",
      "Behavior - {'has_upstream_actor'} -> ChemicalMixture\n",
      "Behavior - {'primarily_composed_of'} -> ChemicalMixture\n",
      "Behavior - {'temporally_related_to'} -> ChemicalMixture\n",
      "Behavior - {'affects'} -> ChemicalMixture\n",
      "Behavior - {'similar_to'} -> ChemicalMixture\n",
      "Behavior - {'related_to'} -> ChemicalMixture\n",
      "Behavior - {'overlaps'} -> ChemicalMixture\n",
      "Behavior - {'composed_primarily_of'} -> ChemicalMixture\n",
      "Behavior - {'preceded_by'} -> ChemicalMixture\n",
      "Behavior - {'has_output'} -> ChemicalMixture\n",
      "Behavior - {'affected_by'} -> ChemicalMixture\n",
      "Behavior - {'has_substrate'} -> ChemicalMixture\n",
      "Behavior - {'interacts_with'} -> ChemicalMixture\n",
      "Behavior - {'has_capability'} -> ChemicalMixture\n",
      "Behavior - {'participates_in'} -> ChemicalMixture\n",
      "Behavior - {'has_input'} -> ChemicalMixture\n",
      "Behavior - {'enabled_by'} -> ChemicalMixture\n",
      "Behavior - {'has_participant'} -> ChemicalMixture\n",
      "Behavior - {'has_catalyst'} -> ChemicalMixture\n",
      "Behavior - {'associated_with'} -> ChemicalMixture\n",
      "Behavior - {'precedes'} -> ChemicalMixture\n",
      "ChemicalMixture - {'related_to'} -> Publication\n",
      "ChemicalMixture - {'affects'} -> Publication\n",
      "ChemicalMixture - {'subclass_of'} -> Publication\n",
      "ChemicalMixture - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->NucleicAcidEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'overlaps'} -> NucleicAcidEntity\n",
      "Behavior - {'composed_primarily_of'} -> NucleicAcidEntity\n",
      "Behavior - {'preceded_by'} -> NucleicAcidEntity\n",
      "Behavior - {'affected_by'} -> NucleicAcidEntity\n",
      "Behavior - {'has_substrate'} -> NucleicAcidEntity\n",
      "Behavior - {'interacts_with'} -> NucleicAcidEntity\n",
      "Behavior - {'part_of'} -> NucleicAcidEntity\n",
      "Behavior - {'has_capability'} -> NucleicAcidEntity\n",
      "Behavior - {'participates_in'} -> NucleicAcidEntity\n",
      "Behavior - {'caused_by'} -> NucleicAcidEntity\n",
      "Behavior - {'has_input'} -> NucleicAcidEntity\n",
      "Behavior - {'enabled_by'} -> NucleicAcidEntity\n",
      "Behavior - {'has_participant'} -> NucleicAcidEntity\n",
      "Behavior - {'has_catalyst'} -> NucleicAcidEntity\n",
      "Behavior - {'associated_with'} -> NucleicAcidEntity\n",
      "Behavior - {'has_part'} -> NucleicAcidEntity\n",
      "Behavior - {'precedes'} -> NucleicAcidEntity\n",
      "Behavior - {'actively_involves'} -> NucleicAcidEntity\n",
      "Behavior - {'has_upstream_actor'} -> NucleicAcidEntity\n",
      "Behavior - {'has_active_ingredient'} -> NucleicAcidEntity\n",
      "Behavior - {'primarily_composed_of'} -> NucleicAcidEntity\n",
      "Behavior - {'temporally_related_to'} -> NucleicAcidEntity\n",
      "Behavior - {'has_output'} -> NucleicAcidEntity\n",
      "Behavior - {'affects'} -> NucleicAcidEntity\n",
      "Behavior - {'similar_to'} -> NucleicAcidEntity\n",
      "Behavior - {'related_to'} -> NucleicAcidEntity\n",
      "Behavior - {'located_in'} -> NucleicAcidEntity\n",
      "NucleicAcidEntity - {'gene_associated_with_condition'} -> Publication\n",
      "NucleicAcidEntity - {'located_in'} -> Publication\n",
      "NucleicAcidEntity - {'related_to'} -> Publication\n",
      "NucleicAcidEntity - {'close_match'} -> Publication\n",
      "NucleicAcidEntity - {'subclass_of'} -> Publication\n",
      "NucleicAcidEntity - {'interacts_with'} -> Publication\n",
      "NucleicAcidEntity - {'occurs_in'} -> Publication\n",
      "NucleicAcidEntity - {'affects'} -> Publication\n",
      "NucleicAcidEntity - {'treats'} -> Publication\n",
      "NucleicAcidEntity - {'causes'} -> Publication\n",
      "NucleicAcidEntity - {'coexists_with'} -> Publication\n",
      "NucleicAcidEntity - {'disrupts'} -> Publication\n",
      "NucleicAcidEntity - {'associated_with'} -> Publication\n",
      "NucleicAcidEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->ComplexMolecularMixture->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'associated_with'} -> ComplexMolecularMixture\n",
      "Behavior - {'related_to'} -> ComplexMolecularMixture\n",
      "Behavior - {'overlaps'} -> ComplexMolecularMixture\n",
      "Behavior - {'composed_primarily_of'} -> ComplexMolecularMixture\n",
      "Behavior - {'has_upstream_actor'} -> ComplexMolecularMixture\n",
      "Behavior - {'primarily_composed_of'} -> ComplexMolecularMixture\n",
      "Behavior - {'affects'} -> ComplexMolecularMixture\n",
      "Behavior - {'similar_to'} -> ComplexMolecularMixture\n",
      "Behavior - {'affected_by'} -> ComplexMolecularMixture\n",
      "Behavior - {'participates_in'} -> ComplexMolecularMixture\n",
      "Behavior - {'has_participant'} -> ComplexMolecularMixture\n",
      "Behavior - {'interacts_with'} -> ComplexMolecularMixture\n",
      "Behavior - {'has_input'} -> ComplexMolecularMixture\n",
      "ComplexMolecularMixture - {'correlated_with'} -> Publication\n",
      "ComplexMolecularMixture - {'subclass_of'} -> Publication\n",
      "ComplexMolecularMixture - {'treats'} -> Publication\n",
      "ComplexMolecularMixture - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->NoncodingRNAProduct->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'composed_primarily_of'} -> NoncodingRNAProduct\n",
      "Behavior - {'has_upstream_actor'} -> NoncodingRNAProduct\n",
      "Behavior - {'primarily_composed_of'} -> NoncodingRNAProduct\n",
      "Behavior - {'has_output'} -> NoncodingRNAProduct\n",
      "Behavior - {'affects'} -> NoncodingRNAProduct\n",
      "Behavior - {'similar_to'} -> NoncodingRNAProduct\n",
      "Behavior - {'affected_by'} -> NoncodingRNAProduct\n",
      "Behavior - {'has_input'} -> NoncodingRNAProduct\n",
      "Behavior - {'enabled_by'} -> NoncodingRNAProduct\n",
      "Behavior - {'has_participant'} -> NoncodingRNAProduct\n",
      "Behavior - {'has_substrate'} -> NoncodingRNAProduct\n",
      "Behavior - {'interacts_with'} -> NoncodingRNAProduct\n",
      "Behavior - {'has_capability'} -> NoncodingRNAProduct\n",
      "Behavior - {'participates_in'} -> NoncodingRNAProduct\n",
      "Behavior - {'actively_involves'} -> NoncodingRNAProduct\n",
      "Behavior - {'has_catalyst'} -> NoncodingRNAProduct\n",
      "Behavior - {'associated_with'} -> NoncodingRNAProduct\n",
      "Behavior - {'related_to'} -> NoncodingRNAProduct\n",
      "Behavior - {'overlaps'} -> NoncodingRNAProduct\n",
      "NoncodingRNAProduct - {'affects'} -> Publication\n",
      "NoncodingRNAProduct - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Food->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_input'} -> Food\n",
      "Behavior - {'related_to'} -> Food\n",
      "Behavior - {'occurs_in'} -> Food\n",
      "Behavior - {'affects'} -> Food\n",
      "Behavior - {'has_participant'} -> Food\n",
      "Food - {'regulates'} -> Publication\n",
      "Food - {'associated_with'} -> Publication\n",
      "Food - {'located_in'} -> Publication\n",
      "Food - {'predisposes'} -> Publication\n",
      "Food - {'related_to'} -> Publication\n",
      "Food - {'occurs_in'} -> Publication\n",
      "Food - {'affects'} -> Publication\n",
      "Food - {'precedes'} -> Publication\n",
      "Food - {'subclass_of'} -> Publication\n",
      "Food - {'treats'} -> Publication\n",
      "Food - {'causes'} -> Publication\n",
      "Food - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->ProteinDomain->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_part'} -> ProteinDomain\n",
      "ProteinDomain - {'associated_with'} -> Publication\n",
      "ProteinDomain - {'located_in'} -> Publication\n",
      "ProteinDomain - {'related_to'} -> Publication\n",
      "ProteinDomain - {'occurs_in'} -> Publication\n",
      "ProteinDomain - {'affects'} -> Publication\n",
      "ProteinDomain - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->PhysicalEntity->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'manifestation_of'} -> PhysicalEntity\n",
      "Behavior - {'exacerbates'} -> PhysicalEntity\n",
      "Behavior - {'coexists_with'} -> PhysicalEntity\n",
      "Behavior - {'has_input'} -> PhysicalEntity\n",
      "Behavior - {'treats'} -> PhysicalEntity\n",
      "Behavior - {'causes'} -> PhysicalEntity\n",
      "Behavior - {'has_part'} -> PhysicalEntity\n",
      "Behavior - {'located_in'} -> PhysicalEntity\n",
      "Behavior - {'predisposes'} -> PhysicalEntity\n",
      "Behavior - {'related_to'} -> PhysicalEntity\n",
      "Behavior - {'prevents'} -> PhysicalEntity\n",
      "Behavior - {'precedes'} -> PhysicalEntity\n",
      "Behavior - {'occurs_in'} -> PhysicalEntity\n",
      "Behavior - {'affects'} -> PhysicalEntity\n",
      "PhysicalEntity - {'subclass_of'} -> Publication\n",
      "PhysicalEntity - {'has_part'} -> Publication\n",
      "PhysicalEntity - {'interacts_with'} -> Publication\n",
      "PhysicalEntity - {'produces'} -> Publication\n",
      "PhysicalEntity - {'predisposes'} -> Publication\n",
      "PhysicalEntity - {'related_to'} -> Publication\n",
      "PhysicalEntity - {'occurs_in'} -> Publication\n",
      "PhysicalEntity - {'affects'} -> Publication\n",
      "PhysicalEntity - {'close_match'} -> Publication\n",
      "PhysicalEntity - {'coexists_with'} -> Publication\n",
      "PhysicalEntity - {'regulates'} -> Publication\n",
      "PhysicalEntity - {'located_in'} -> Publication\n",
      "PhysicalEntity - {'treats'} -> Publication\n",
      "PhysicalEntity - {'has_input'} -> Publication\n",
      "PhysicalEntity - {'causes'} -> Publication\n",
      "PhysicalEntity - {'disrupts'} -> Publication\n",
      "PhysicalEntity - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Agent->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'predisposes'} -> Agent\n",
      "Behavior - {'subclass_of'} -> Agent\n",
      "Behavior - {'prevents'} -> Agent\n",
      "Behavior - {'related_to'} -> Agent\n",
      "Behavior - {'occurs_in'} -> Agent\n",
      "Behavior - {'precedes'} -> Agent\n",
      "Behavior - {'affects'} -> Agent\n",
      "Behavior - {'coexists_with'} -> Agent\n",
      "Behavior - {'close_match'} -> Agent\n",
      "Behavior - {'regulates'} -> Agent\n",
      "Behavior - {'treats'} -> Agent\n",
      "Behavior - {'has_input'} -> Agent\n",
      "Behavior - {'causes'} -> Agent\n",
      "Behavior - {'associated_with'} -> Agent\n",
      "Behavior - {'located_in'} -> Agent\n",
      "Agent - {'has_part'} -> Publication\n",
      "Agent - {'interacts_with'} -> Publication\n",
      "Agent - {'produces'} -> Publication\n",
      "Agent - {'predisposes'} -> Publication\n",
      "Agent - {'related_to'} -> Publication\n",
      "Agent - {'prevents'} -> Publication\n",
      "Agent - {'precedes'} -> Publication\n",
      "Agent - {'occurs_in'} -> Publication\n",
      "Agent - {'affects'} -> Publication\n",
      "Agent - {'exacerbates'} -> Publication\n",
      "Agent - {'coexists_with'} -> Publication\n",
      "Agent - {'close_match'} -> Publication\n",
      "Agent - {'subclass_of'} -> Publication\n",
      "Agent - {'regulates'} -> Publication\n",
      "Agent - {'diagnoses'} -> Publication\n",
      "Agent - {'treats'} -> Publication\n",
      "Agent - {'has_input'} -> Publication\n",
      "Agent - {'causes'} -> Publication\n",
      "Agent - {'located_in'} -> Publication\n",
      "Agent - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Event->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'subclass_of'} -> Event\n",
      "Behavior - {'related_to'} -> Event\n",
      "Event - {'occurs_in'} -> Publication\n",
      "Event - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->LifeStage->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'occurs_in'} -> LifeStage\n",
      "Behavior - {'affects'} -> LifeStage\n",
      "Behavior - {'treats'} -> LifeStage\n",
      "LifeStage - {'subclass_of'} -> Publication\n",
      "LifeStage - {'has_part'} -> Publication\n",
      "LifeStage - {'interacts_with'} -> Publication\n",
      "LifeStage - {'produces'} -> Publication\n",
      "LifeStage - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->BehavioralFeature->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'causes'} -> BehavioralFeature\n",
      "Behavior - {'associated_with'} -> BehavioralFeature\n",
      "Behavior - {'subclass_of'} -> BehavioralFeature\n",
      "Behavior - {'interacts_with'} -> BehavioralFeature\n",
      "Behavior - {'predisposes'} -> BehavioralFeature\n",
      "Behavior - {'related_to'} -> BehavioralFeature\n",
      "Behavior - {'prevents'} -> BehavioralFeature\n",
      "Behavior - {'precedes'} -> BehavioralFeature\n",
      "Behavior - {'affects'} -> BehavioralFeature\n",
      "Behavior - {'close_match'} -> BehavioralFeature\n",
      "Behavior - {'manifestation_of'} -> BehavioralFeature\n",
      "Behavior - {'exacerbates'} -> BehavioralFeature\n",
      "Behavior - {'coexists_with'} -> BehavioralFeature\n",
      "Behavior - {'treats'} -> BehavioralFeature\n",
      "Behavior - {'has_input'} -> BehavioralFeature\n",
      "BehavioralFeature - {'subclass_of'} -> Publication\n",
      "BehavioralFeature - {'produces'} -> Publication\n",
      "BehavioralFeature - {'related_to'} -> Publication\n",
      "BehavioralFeature - {'occurs_in'} -> Publication\n",
      "BehavioralFeature - {'affects'} -> Publication\n",
      "BehavioralFeature - {'manifestation_of'} -> Publication\n",
      "BehavioralFeature - {'coexists_with'} -> Publication\n",
      "BehavioralFeature - {'disrupts'} -> Publication\n",
      "BehavioralFeature - {'located_in'} -> Publication\n",
      "BehavioralFeature - {'predisposes'} -> Publication\n",
      "BehavioralFeature - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->PopulationOfIndividualOrganisms->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'treats'} -> PopulationOfIndividualOrganisms\n",
      "Behavior - {'has_input'} -> PopulationOfIndividualOrganisms\n",
      "Behavior - {'subclass_of'} -> PopulationOfIndividualOrganisms\n",
      "Behavior - {'related_to'} -> PopulationOfIndividualOrganisms\n",
      "Behavior - {'occurs_in'} -> PopulationOfIndividualOrganisms\n",
      "Behavior - {'affects'} -> PopulationOfIndividualOrganisms\n",
      "Behavior - {'close_match'} -> PopulationOfIndividualOrganisms\n",
      "PopulationOfIndividualOrganisms - {'correlated_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_part'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'produces'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'related_to'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'occurs_in'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'affects'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'close_match'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'subclass_of'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'interacts_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'treats'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_input'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->ProteinFamily->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_input'} -> ProteinFamily\n",
      "Behavior - {'produces'} -> ProteinFamily\n",
      "ProteinFamily - {'associated_with'} -> Publication\n",
      "ProteinFamily - {'subclass_of'} -> Publication\n",
      "ProteinFamily - {'gene_associated_with_condition'} -> Publication\n",
      "ProteinFamily - {'predisposes'} -> Publication\n",
      "ProteinFamily - {'related_to'} -> Publication\n",
      "ProteinFamily - {'affects'} -> Publication\n",
      "ProteinFamily - {'disrupts'} -> Publication\n",
      "ProteinFamily - {'located_in'} -> Publication\n",
      "ProteinFamily - {'regulates'} -> Publication\n",
      "ProteinFamily - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->ClinicalIntervention->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'subclass_of'} -> ClinicalIntervention\n",
      "Behavior - {'predisposes'} -> ClinicalIntervention\n",
      "Behavior - {'related_to'} -> ClinicalIntervention\n",
      "Behavior - {'precedes'} -> ClinicalIntervention\n",
      "Behavior - {'occurs_in'} -> ClinicalIntervention\n",
      "Behavior - {'affects'} -> ClinicalIntervention\n",
      "Behavior - {'close_match'} -> ClinicalIntervention\n",
      "Behavior - {'coexists_with'} -> ClinicalIntervention\n",
      "Behavior - {'regulates'} -> ClinicalIntervention\n",
      "Behavior - {'treats'} -> ClinicalIntervention\n",
      "Behavior - {'has_input'} -> ClinicalIntervention\n",
      "Behavior - {'causes'} -> ClinicalIntervention\n",
      "ClinicalIntervention - {'correlated_with'} -> Publication\n",
      "ClinicalIntervention - {'regulates'} -> Publication\n",
      "ClinicalIntervention - {'diagnoses'} -> Publication\n",
      "ClinicalIntervention - {'treats'} -> Publication\n",
      "ClinicalIntervention - {'has_input'} -> Publication\n",
      "ClinicalIntervention - {'causes'} -> Publication\n",
      "ClinicalIntervention - {'disrupts'} -> Publication\n",
      "ClinicalIntervention - {'associated_with'} -> Publication\n",
      "ClinicalIntervention - {'has_part'} -> Publication\n",
      "ClinicalIntervention - {'located_in'} -> Publication\n",
      "ClinicalIntervention - {'predisposes'} -> Publication\n",
      "ClinicalIntervention - {'related_to'} -> Publication\n",
      "ClinicalIntervention - {'prevents'} -> Publication\n",
      "ClinicalIntervention - {'manifestation_of'} -> Publication\n",
      "ClinicalIntervention - {'precedes'} -> Publication\n",
      "ClinicalIntervention - {'occurs_in'} -> Publication\n",
      "ClinicalIntervention - {'affects'} -> Publication\n",
      "ClinicalIntervention - {'close_match'} -> Publication\n",
      "ClinicalIntervention - {'exacerbates'} -> Publication\n",
      "ClinicalIntervention - {'coexists_with'} -> Publication\n",
      "ClinicalIntervention - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Cohort->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'subclass_of'} -> Cohort\n",
      "Behavior - {'interacts_with'} -> Cohort\n",
      "Behavior - {'produces'} -> Cohort\n",
      "Behavior - {'related_to'} -> Cohort\n",
      "Behavior - {'occurs_in'} -> Cohort\n",
      "Behavior - {'affects'} -> Cohort\n",
      "Behavior - {'close_match'} -> Cohort\n",
      "Behavior - {'has_participant'} -> Cohort\n",
      "Behavior - {'treats'} -> Cohort\n",
      "Behavior - {'has_input'} -> Cohort\n",
      "Cohort - {'diagnoses'} -> Publication\n",
      "Cohort - {'related_to'} -> Publication\n",
      "Cohort - {'treats'} -> Publication\n",
      "Cohort - {'has_input'} -> Publication\n",
      "Cohort - {'occurs_in'} -> Publication\n",
      "Cohort - {'has_part'} -> Publication\n",
      "Cohort - {'close_match'} -> Publication\n",
      "Cohort - {'subclass_of'} -> Publication\n",
      "Cohort - {'interacts_with'} -> Publication\n",
      "Cohort - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'treats'} -> Publication\n",
      "Behavior - {'has_input'} -> Publication\n",
      "Behavior - {'subclass_of'} -> Publication\n",
      "Behavior - {'located_in'} -> Publication\n",
      "Behavior - {'interacts_with'} -> Publication\n",
      "Behavior - {'predisposes'} -> Publication\n",
      "Behavior - {'related_to'} -> Publication\n",
      "Behavior - {'occurs_in'} -> Publication\n",
      "Behavior - {'affects'} -> Publication\n",
      "Behavior - {'coexists_with'} -> Publication\n",
      "Behavior - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->GeographicLocation->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'related_to'} -> GeographicLocation\n",
      "Behavior - {'has_input'} -> GeographicLocation\n",
      "GeographicLocation - {'subclass_of'} -> Publication\n",
      "GeographicLocation - {'related_to'} -> Publication\n",
      "GeographicLocation - {'gene_associated_with_condition'} -> Publication\n",
      "GeographicLocation - {'located_in'} -> Publication\n",
      "GeographicLocation - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->Vitamin->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'has_input'} -> Vitamin\n",
      "Vitamin - {'regulates'} -> Publication\n",
      "Vitamin - {'treats'} -> Publication\n",
      "Vitamin - {'disrupts'} -> Publication\n",
      "Vitamin - {'subclass_of'} -> Publication\n",
      "Vitamin - {'related_to'} -> Publication\n",
      "Vitamin - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->MaterialSample->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'affects'} -> MaterialSample\n",
      "MaterialSample - {'located_in'} -> Publication\n",
      "MaterialSample - {'associated_with'} -> Publication\n",
      "MaterialSample - {'subclass_of'} -> Publication\n",
      "MaterialSample - {'has_part'} -> Publication\n",
      "MaterialSample - {'related_to'} -> Publication\n",
      "MaterialSample - {'occurs_in'} -> Publication\n",
      "MaterialSample - {'affects'} -> Publication\n",
      "MaterialSample - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Behavior->GeneticInheritance->Publication\n",
      "InformationResource - {'related_to'} -> Behavior\n",
      "Behavior - {'precedes'} -> GeneticInheritance\n",
      "Behavior - {'affects'} -> GeneticInheritance\n",
      "Behavior - {'coexists_with'} -> GeneticInheritance\n",
      "GeneticInheritance - {'occurs_in'} -> Publication\n",
      "GeneticInheritance - {'has_input'} -> Publication\n",
      "GeneticInheritance - {'located_in'} -> Publication\n",
      "GeneticInheritance - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->PopulationOfIndividualOrganisms->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'interacts_with'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'produces'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'predisposes'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'related_to'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'diagnoses'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'occurs_in'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'has_input'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'affects'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'close_match'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'coexists_with'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'located_in'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'treats'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'associated_with'} -> PopulationOfIndividualOrganisms\n",
      "Disease - {'subclass_of'} -> PopulationOfIndividualOrganisms\n",
      "PopulationOfIndividualOrganisms - {'correlated_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_part'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'produces'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'related_to'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'occurs_in'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'affects'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'close_match'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'subclass_of'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'interacts_with'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'treats'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'has_input'} -> Publication\n",
      "PopulationOfIndividualOrganisms - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->EnvironmentalExposure->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> EnvironmentalExposure\n",
      "EnvironmentalExposure - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Drug->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Drug\n",
      "Disease - {'positively_correlated_with'} -> Drug\n",
      "Disease - {'negatively_correlated_with'} -> Drug\n",
      "Disease - {'treated_by'} -> Drug\n",
      "Disease - {'affected_by'} -> Drug\n",
      "Disease - {'prevented_by'} -> Drug\n",
      "Disease - {'caused_by'} -> Drug\n",
      "Disease - {'actively_involves'} -> Drug\n",
      "Disease - {'has_catalyst'} -> Drug\n",
      "Disease - {'primarily_composed_of'} -> Drug\n",
      "Disease - {'temporally_related_to'} -> Drug\n",
      "Disease - {'response_affected_by'} -> Drug\n",
      "Disease - {'associated_with'} -> Drug\n",
      "Disease - {'affects'} -> Drug\n",
      "Disease - {'similar_to'} -> Drug\n",
      "Disease - {'related_to'} -> Drug\n",
      "Disease - {'overlaps'} -> Drug\n",
      "Disease - {'disrupted_by'} -> Drug\n",
      "Disease - {'composed_primarily_of'} -> Drug\n",
      "Disease - {'has_upstream_actor'} -> Drug\n",
      "Disease - {'regulated_by'} -> Drug\n",
      "Disease - {'has_output'} -> Drug\n",
      "Disease - {'response_decreased_by'} -> Drug\n",
      "Disease - {'treats'} -> Drug\n",
      "Disease - {'is_ameliorated_by'} -> Drug\n",
      "Disease - {'response_increased_by'} -> Drug\n",
      "Disease - {'has_capability'} -> Drug\n",
      "Disease - {'participates_in'} -> Drug\n",
      "Disease - {'has_input'} -> Drug\n",
      "Disease - {'preceded_by'} -> Drug\n",
      "Disease - {'enabled_by'} -> Drug\n",
      "Disease - {'has_participant'} -> Drug\n",
      "Disease - {'is_exacerbated_by'} -> Drug\n",
      "Disease - {'has_substrate'} -> Drug\n",
      "Disease - {'interacts_with'} -> Drug\n",
      "Disease - {'ameliorates'} -> Drug\n",
      "Disease - {'biomarker_for'} -> Drug\n",
      "Disease - {'chemically_similar_to'} -> Drug\n",
      "Disease - {'physically_interacts_with'} -> Drug\n",
      "Disease - {'disease_has_location'} -> Drug\n",
      "Disease - {'derives_from'} -> Drug\n",
      "Disease - {'causes'} -> Drug\n",
      "Disease - {'directly_physically_interacts_with'} -> Drug\n",
      "Disease - {'precedes'} -> Drug\n",
      "Disease - {'subclass_of'} -> Drug\n",
      "Disease - {'gene_associated_with_condition'} -> Drug\n",
      "Disease - {'has_part'} -> Drug\n",
      "Disease - {'colocalizes_with'} -> Drug\n",
      "Disease - {'produces'} -> Drug\n",
      "Disease - {'predisposes'} -> Drug\n",
      "Disease - {'prevents'} -> Drug\n",
      "Disease - {'diagnoses'} -> Drug\n",
      "Disease - {'contributes_to'} -> Drug\n",
      "Disease - {'manifestation_of'} -> Drug\n",
      "Disease - {'occurs_in'} -> Drug\n",
      "Disease - {'close_match'} -> Drug\n",
      "Disease - {'exacerbates'} -> Drug\n",
      "Disease - {'coexists_with'} -> Drug\n",
      "Disease - {'has_phenotype'} -> Drug\n",
      "Disease - {'disrupts'} -> Drug\n",
      "Disease - {'located_in'} -> Drug\n",
      "Disease - {'regulates'} -> Drug\n",
      "Drug - {'correlated_with'} -> Publication\n",
      "Drug - {'regulates'} -> Publication\n",
      "Drug - {'treats'} -> Publication\n",
      "Drug - {'has_input'} -> Publication\n",
      "Drug - {'physically_interacts_with'} -> Publication\n",
      "Drug - {'causes'} -> Publication\n",
      "Drug - {'directly_physically_interacts_with'} -> Publication\n",
      "Drug - {'disrupts'} -> Publication\n",
      "Drug - {'associated_with'} -> Publication\n",
      "Drug - {'gene_associated_with_condition'} -> Publication\n",
      "Drug - {'located_in'} -> Publication\n",
      "Drug - {'has_part'} -> Publication\n",
      "Drug - {'prevents'} -> Publication\n",
      "Drug - {'colocalizes_with'} -> Publication\n",
      "Drug - {'interacts_with'} -> Publication\n",
      "Drug - {'predisposes'} -> Publication\n",
      "Drug - {'related_to'} -> Publication\n",
      "Drug - {'occurs_in'} -> Publication\n",
      "Drug - {'precedes'} -> Publication\n",
      "Drug - {'affects'} -> Publication\n",
      "Drug - {'close_match'} -> Publication\n",
      "Drug - {'subclass_of'} -> Publication\n",
      "Drug - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ChemicalExposure->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> ChemicalExposure\n",
      "ChemicalExposure - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Activity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Activity\n",
      "Disease - {'has_participant'} -> Activity\n",
      "Disease - {'interacts_with'} -> Activity\n",
      "Disease - {'ameliorates'} -> Activity\n",
      "Disease - {'associated_with'} -> Activity\n",
      "Disease - {'affects'} -> Activity\n",
      "Disease - {'similar_to'} -> Activity\n",
      "Disease - {'related_to'} -> Activity\n",
      "Disease - {'overlaps'} -> Activity\n",
      "Disease - {'composed_primarily_of'} -> Activity\n",
      "Disease - {'has_upstream_actor'} -> Activity\n",
      "Disease - {'primarily_composed_of'} -> Activity\n",
      "Disease - {'affected_by'} -> Activity\n",
      "Disease - {'treats'} -> Activity\n",
      "Disease - {'participates_in'} -> Activity\n",
      "Disease - {'located_in'} -> Activity\n",
      "Disease - {'physically_interacts_with'} -> Activity\n",
      "Disease - {'causes'} -> Activity\n",
      "Disease - {'actively_involved_in'} -> Activity\n",
      "Disease - {'precedes'} -> Activity\n",
      "Disease - {'gene_associated_with_condition'} -> Activity\n",
      "Disease - {'subclass_of'} -> Activity\n",
      "Disease - {'produces'} -> Activity\n",
      "Disease - {'predisposes'} -> Activity\n",
      "Disease - {'has_part'} -> Activity\n",
      "Disease - {'prevents'} -> Activity\n",
      "Disease - {'contributes_to'} -> Activity\n",
      "Disease - {'diagnoses'} -> Activity\n",
      "Disease - {'occurs_in'} -> Activity\n",
      "Disease - {'manifestation_of'} -> Activity\n",
      "Disease - {'has_input'} -> Activity\n",
      "Disease - {'exacerbates'} -> Activity\n",
      "Disease - {'coexists_with'} -> Activity\n",
      "Disease - {'close_match'} -> Activity\n",
      "Disease - {'disrupts'} -> Activity\n",
      "Disease - {'has_phenotype'} -> Activity\n",
      "Disease - {'regulates'} -> Activity\n",
      "Disease - {'contraindicated_for'} -> Activity\n",
      "Disease - {'biomarker_for'} -> Activity\n",
      "Activity - {'correlated_with'} -> Publication\n",
      "Activity - {'causes'} -> Publication\n",
      "Activity - {'disrupts'} -> Publication\n",
      "Activity - {'subclass_of'} -> Publication\n",
      "Activity - {'located_in'} -> Publication\n",
      "Activity - {'interacts_with'} -> Publication\n",
      "Activity - {'produces'} -> Publication\n",
      "Activity - {'has_part'} -> Publication\n",
      "Activity - {'diagnoses'} -> Publication\n",
      "Activity - {'prevents'} -> Publication\n",
      "Activity - {'predisposes'} -> Publication\n",
      "Activity - {'precedes'} -> Publication\n",
      "Activity - {'related_to'} -> Publication\n",
      "Activity - {'occurs_in'} -> Publication\n",
      "Activity - {'affects'} -> Publication\n",
      "Activity - {'coexists_with'} -> Publication\n",
      "Activity - {'close_match'} -> Publication\n",
      "Activity - {'regulates'} -> Publication\n",
      "Activity - {'treats'} -> Publication\n",
      "Activity - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ClinicalAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> ClinicalAttribute\n",
      "Disease - {'primarily_composed_of'} -> ClinicalAttribute\n",
      "Disease - {'affects'} -> ClinicalAttribute\n",
      "Disease - {'similar_to'} -> ClinicalAttribute\n",
      "Disease - {'related_to'} -> ClinicalAttribute\n",
      "Disease - {'overlaps'} -> ClinicalAttribute\n",
      "Disease - {'condition_associated_with_gene'} -> ClinicalAttribute\n",
      "Disease - {'composed_primarily_of'} -> ClinicalAttribute\n",
      "Disease - {'gene_associated_with_condition'} -> ClinicalAttribute\n",
      "Disease - {'affected_by'} -> ClinicalAttribute\n",
      "Disease - {'interacts_with'} -> ClinicalAttribute\n",
      "Disease - {'causes'} -> ClinicalAttribute\n",
      "Disease - {'participates_in'} -> ClinicalAttribute\n",
      "Disease - {'contributes_to'} -> ClinicalAttribute\n",
      "Disease - {'has_participant'} -> ClinicalAttribute\n",
      "Disease - {'associated_with'} -> ClinicalAttribute\n",
      "Disease - {'genetic_association'} -> ClinicalAttribute\n",
      "Disease - {'has_upstream_actor'} -> ClinicalAttribute\n",
      "Disease - {'coexists_with'} -> ClinicalAttribute\n",
      "Disease - {'manifestation_of'} -> ClinicalAttribute\n",
      "Disease - {'disrupts'} -> ClinicalAttribute\n",
      "Disease - {'diagnoses'} -> ClinicalAttribute\n",
      "Disease - {'located_in'} -> ClinicalAttribute\n",
      "Disease - {'subclass_of'} -> ClinicalAttribute\n",
      "Disease - {'regulates'} -> ClinicalAttribute\n",
      "Disease - {'prevents'} -> ClinicalAttribute\n",
      "Disease - {'close_match'} -> ClinicalAttribute\n",
      "Disease - {'precedes'} -> ClinicalAttribute\n",
      "Disease - {'predisposes'} -> ClinicalAttribute\n",
      "Disease - {'exacerbates'} -> ClinicalAttribute\n",
      "ClinicalAttribute - {'correlated_with'} -> Publication\n",
      "ClinicalAttribute - {'has_input'} -> Publication\n",
      "ClinicalAttribute - {'subclass_of'} -> Publication\n",
      "ClinicalAttribute - {'related_to'} -> Publication\n",
      "ClinicalAttribute - {'located_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Annotation->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Annotation\n",
      "Disease - {'has_part'} -> Annotation\n",
      "Disease - {'subclass_of'} -> Annotation\n",
      "Disease - {'related_to'} -> Annotation\n",
      "Disease - {'exacerbates'} -> Annotation\n",
      "Disease - {'has_participant'} -> Annotation\n",
      "Disease - {'interacts_with'} -> Annotation\n",
      "Disease - {'ameliorates'} -> Annotation\n",
      "Disease - {'regulates'} -> Annotation\n",
      "Disease - {'part_of'} -> Annotation\n",
      "Disease - {'participates_in'} -> Annotation\n",
      "Disease - {'contributes_to'} -> Annotation\n",
      "Disease - {'decreases_response_to'} -> Annotation\n",
      "Disease - {'associated_with'} -> Annotation\n",
      "Disease - {'overlaps'} -> Annotation\n",
      "Disease - {'affects_response_to'} -> Annotation\n",
      "Disease - {'genetic_association'} -> Annotation\n",
      "Disease - {'composed_primarily_of'} -> Annotation\n",
      "Disease - {'has_upstream_actor'} -> Annotation\n",
      "Disease - {'has_active_ingredient'} -> Annotation\n",
      "Disease - {'increases_response_to'} -> Annotation\n",
      "Disease - {'primarily_composed_of'} -> Annotation\n",
      "Disease - {'temporally_related_to'} -> Annotation\n",
      "Disease - {'affects'} -> Annotation\n",
      "Disease - {'disrupts'} -> Annotation\n",
      "Disease - {'similar_to'} -> Annotation\n",
      "Disease - {'affected_by'} -> Annotation\n",
      "Disease - {'treats'} -> Annotation\n",
      "Disease - {'condition_associated_with_gene'} -> Annotation\n",
      "Disease - {'causes'} -> Annotation\n",
      "Disease - {'gene_associated_with_condition'} -> Annotation\n",
      "Disease - {'preceded_by'} -> Annotation\n",
      "Annotation - {'correlated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->PhysiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> PhysiologicalProcess\n",
      "Disease - {'disrupts'} -> PhysiologicalProcess\n",
      "Disease - {'precedes'} -> PhysiologicalProcess\n",
      "Disease - {'affects'} -> PhysiologicalProcess\n",
      "Disease - {'manifestation_of'} -> PhysiologicalProcess\n",
      "Disease - {'coexists_with'} -> PhysiologicalProcess\n",
      "Disease - {'associated_with'} -> PhysiologicalProcess\n",
      "Disease - {'contains_process'} -> PhysiologicalProcess\n",
      "Disease - {'causes'} -> PhysiologicalProcess\n",
      "Disease - {'affected_by'} -> PhysiologicalProcess\n",
      "Disease - {'affects_response_to'} -> PhysiologicalProcess\n",
      "Disease - {'genetic_association'} -> PhysiologicalProcess\n",
      "Disease - {'entity_positively_regulates_entity'} -> PhysiologicalProcess\n",
      "Disease - {'disrupted_by'} -> PhysiologicalProcess\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> PhysiologicalProcess\n",
      "Disease - {'composed_primarily_of'} -> PhysiologicalProcess\n",
      "Disease - {'has_upstream_actor'} -> PhysiologicalProcess\n",
      "Disease - {'treated_by'} -> PhysiologicalProcess\n",
      "Disease - {'gene_associated_with_condition'} -> PhysiologicalProcess\n",
      "Disease - {'exacerbates'} -> PhysiologicalProcess\n",
      "Disease - {'associated_with_sensitivity_to'} -> PhysiologicalProcess\n",
      "Disease - {'increases_response_to'} -> PhysiologicalProcess\n",
      "Disease - {'primarily_composed_of'} -> PhysiologicalProcess\n",
      "Disease - {'regulated_by'} -> PhysiologicalProcess\n",
      "Disease - {'associated_with_resistance_to'} -> PhysiologicalProcess\n",
      "Disease - {'response_affected_by'} -> PhysiologicalProcess\n",
      "Disease - {'response_decreased_by'} -> PhysiologicalProcess\n",
      "Disease - {'treats'} -> PhysiologicalProcess\n",
      "Disease - {'condition_associated_with_gene'} -> PhysiologicalProcess\n",
      "Disease - {'is_ameliorated_by'} -> PhysiologicalProcess\n",
      "Disease - {'sensitivity_associated_with'} -> PhysiologicalProcess\n",
      "Disease - {'has_biomarker'} -> PhysiologicalProcess\n",
      "Disease - {'participates_in'} -> PhysiologicalProcess\n",
      "Disease - {'decreased_likelihood_associated_with'} -> PhysiologicalProcess\n",
      "Disease - {'contributes_to'} -> PhysiologicalProcess\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> PhysiologicalProcess\n",
      "Disease - {'has_participant'} -> PhysiologicalProcess\n",
      "Disease - {'decreases_response_to'} -> PhysiologicalProcess\n",
      "Disease - {'resistance_associated_with'} -> PhysiologicalProcess\n",
      "Disease - {'is_exacerbated_by'} -> PhysiologicalProcess\n",
      "Disease - {'interacts_with'} -> PhysiologicalProcess\n",
      "Disease - {'ameliorates'} -> PhysiologicalProcess\n",
      "Disease - {'regulates'} -> PhysiologicalProcess\n",
      "Disease - {'response_increased_by'} -> PhysiologicalProcess\n",
      "Disease - {'positively_correlated_with'} -> PhysiologicalProcess\n",
      "Disease - {'caused_by'} -> PhysiologicalProcess\n",
      "Disease - {'increased_likelihood_associated_with'} -> PhysiologicalProcess\n",
      "Disease - {'occurs_together_in_literature_with'} -> PhysiologicalProcess\n",
      "Disease - {'temporally_related_to'} -> PhysiologicalProcess\n",
      "Disease - {'coexpressed_with'} -> PhysiologicalProcess\n",
      "Disease - {'negatively_correlated_with'} -> PhysiologicalProcess\n",
      "Disease - {'similar_to'} -> PhysiologicalProcess\n",
      "Disease - {'related_to'} -> PhysiologicalProcess\n",
      "Disease - {'overlaps'} -> PhysiologicalProcess\n",
      "Disease - {'biomarker_for'} -> PhysiologicalProcess\n",
      "Disease - {'prevents'} -> PhysiologicalProcess\n",
      "Disease - {'diagnoses'} -> PhysiologicalProcess\n",
      "Disease - {'enables'} -> PhysiologicalProcess\n",
      "Disease - {'occurs_in'} -> PhysiologicalProcess\n",
      "Disease - {'has_input'} -> PhysiologicalProcess\n",
      "Disease - {'close_match'} -> PhysiologicalProcess\n",
      "Disease - {'has_phenotype'} -> PhysiologicalProcess\n",
      "Disease - {'located_in'} -> PhysiologicalProcess\n",
      "Disease - {'is_sequence_variant_of'} -> PhysiologicalProcess\n",
      "Disease - {'gene_product_of'} -> PhysiologicalProcess\n",
      "Disease - {'subclass_of'} -> PhysiologicalProcess\n",
      "Disease - {'has_part'} -> PhysiologicalProcess\n",
      "Disease - {'produces'} -> PhysiologicalProcess\n",
      "Disease - {'predisposes'} -> PhysiologicalProcess\n",
      "PhysiologicalProcess - {'correlated_with'} -> Publication\n",
      "PhysiologicalProcess - {'has_input'} -> Publication\n",
      "PhysiologicalProcess - {'causes'} -> Publication\n",
      "PhysiologicalProcess - {'has_output'} -> Publication\n",
      "PhysiologicalProcess - {'disrupts'} -> Publication\n",
      "PhysiologicalProcess - {'associated_with'} -> Publication\n",
      "PhysiologicalProcess - {'located_in'} -> Publication\n",
      "PhysiologicalProcess - {'produces'} -> Publication\n",
      "PhysiologicalProcess - {'predisposes'} -> Publication\n",
      "PhysiologicalProcess - {'precedes'} -> Publication\n",
      "PhysiologicalProcess - {'related_to'} -> Publication\n",
      "PhysiologicalProcess - {'occurs_in'} -> Publication\n",
      "PhysiologicalProcess - {'affects'} -> Publication\n",
      "PhysiologicalProcess - {'coexists_with'} -> Publication\n",
      "PhysiologicalProcess - {'close_match'} -> Publication\n",
      "PhysiologicalProcess - {'has_participant'} -> Publication\n",
      "PhysiologicalProcess - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->InformationContentEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> InformationContentEntity\n",
      "Disease - {'affected_by'} -> InformationContentEntity\n",
      "Disease - {'interacts_with'} -> InformationContentEntity\n",
      "Disease - {'causes'} -> InformationContentEntity\n",
      "Disease - {'participates_in'} -> InformationContentEntity\n",
      "Disease - {'contributes_to'} -> InformationContentEntity\n",
      "Disease - {'has_participant'} -> InformationContentEntity\n",
      "Disease - {'associated_with'} -> InformationContentEntity\n",
      "Disease - {'primarily_composed_of'} -> InformationContentEntity\n",
      "Disease - {'affects'} -> InformationContentEntity\n",
      "Disease - {'similar_to'} -> InformationContentEntity\n",
      "Disease - {'related_to'} -> InformationContentEntity\n",
      "Disease - {'overlaps'} -> InformationContentEntity\n",
      "Disease - {'composed_primarily_of'} -> InformationContentEntity\n",
      "Disease - {'has_upstream_actor'} -> InformationContentEntity\n",
      "Disease - {'has_phenotype'} -> InformationContentEntity\n",
      "Disease - {'regulates'} -> InformationContentEntity\n",
      "Disease - {'located_in'} -> InformationContentEntity\n",
      "Disease - {'contraindicated_for'} -> InformationContentEntity\n",
      "Disease - {'biomarker_for'} -> InformationContentEntity\n",
      "Disease - {'treats'} -> InformationContentEntity\n",
      "Disease - {'derives_from'} -> InformationContentEntity\n",
      "Disease - {'physically_interacts_with'} -> InformationContentEntity\n",
      "Disease - {'disease_has_location'} -> InformationContentEntity\n",
      "Disease - {'actively_involved_in'} -> InformationContentEntity\n",
      "Disease - {'temporally_related_to'} -> InformationContentEntity\n",
      "Disease - {'directly_physically_interacts_with'} -> InformationContentEntity\n",
      "Disease - {'precedes'} -> InformationContentEntity\n",
      "Disease - {'gene_associated_with_condition'} -> InformationContentEntity\n",
      "Disease - {'subclass_of'} -> InformationContentEntity\n",
      "Disease - {'predisposes'} -> InformationContentEntity\n",
      "Disease - {'has_part'} -> InformationContentEntity\n",
      "Disease - {'prevents'} -> InformationContentEntity\n",
      "Disease - {'produces'} -> InformationContentEntity\n",
      "Disease - {'diagnoses'} -> InformationContentEntity\n",
      "Disease - {'occurs_in'} -> InformationContentEntity\n",
      "Disease - {'manifestation_of'} -> InformationContentEntity\n",
      "Disease - {'has_input'} -> InformationContentEntity\n",
      "Disease - {'exacerbates'} -> InformationContentEntity\n",
      "Disease - {'coexists_with'} -> InformationContentEntity\n",
      "Disease - {'close_match'} -> InformationContentEntity\n",
      "Disease - {'disrupts'} -> InformationContentEntity\n",
      "InformationContentEntity - {'correlated_with'} -> Publication\n",
      "InformationContentEntity - {'subclass_of'} -> Publication\n",
      "InformationContentEntity - {'related_to'} -> Publication\n",
      "InformationContentEntity - {'disrupts'} -> Publication\n",
      "InformationContentEntity - {'associated_with'} -> Publication\n",
      "InformationContentEntity - {'gene_associated_with_condition'} -> Publication\n",
      "InformationContentEntity - {'regulates'} -> Publication\n",
      "InformationContentEntity - {'biomarker_for'} -> Publication\n",
      "InformationContentEntity - {'produces'} -> Publication\n",
      "InformationContentEntity - {'has_part'} -> Publication\n",
      "InformationContentEntity - {'located_in'} -> Publication\n",
      "InformationContentEntity - {'predisposes'} -> Publication\n",
      "InformationContentEntity - {'prevents'} -> Publication\n",
      "InformationContentEntity - {'affects'} -> Publication\n",
      "InformationContentEntity - {'manifestation_of'} -> Publication\n",
      "InformationContentEntity - {'precedes'} -> Publication\n",
      "InformationContentEntity - {'close_match'} -> Publication\n",
      "InformationContentEntity - {'interacts_with'} -> Publication\n",
      "InformationContentEntity - {'diagnoses'} -> Publication\n",
      "InformationContentEntity - {'treats'} -> Publication\n",
      "InformationContentEntity - {'occurs_in'} -> Publication\n",
      "InformationContentEntity - {'has_input'} -> Publication\n",
      "InformationContentEntity - {'exacerbates'} -> Publication\n",
      "InformationContentEntity - {'causes'} -> Publication\n",
      "InformationContentEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Procedure->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Procedure\n",
      "Disease - {'positively_correlated_with'} -> Procedure\n",
      "Disease - {'negatively_correlated_with'} -> Procedure\n",
      "Disease - {'caused_by'} -> Procedure\n",
      "Disease - {'treated_by'} -> Procedure\n",
      "Disease - {'prevented_by'} -> Procedure\n",
      "Disease - {'preceded_by'} -> Procedure\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> Procedure\n",
      "Disease - {'has_predisposing_factor'} -> Procedure\n",
      "Disease - {'affected_by'} -> Procedure\n",
      "Disease - {'decreased_likelihood_associated_with'} -> Procedure\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> Procedure\n",
      "Disease - {'increased_likelihood_associated_with'} -> Procedure\n",
      "Disease - {'is_diagnosed_by'} -> Procedure\n",
      "Disease - {'is_exacerbated_by'} -> Procedure\n",
      "Disease - {'participates_in'} -> Procedure\n",
      "Disease - {'contributes_to'} -> Procedure\n",
      "Disease - {'has_participant'} -> Procedure\n",
      "Disease - {'associated_with'} -> Procedure\n",
      "Disease - {'disrupted_by'} -> Procedure\n",
      "Disease - {'has_upstream_actor'} -> Procedure\n",
      "Disease - {'primarily_composed_of'} -> Procedure\n",
      "Disease - {'regulated_by'} -> Procedure\n",
      "Disease - {'temporally_related_to'} -> Procedure\n",
      "Disease - {'response_affected_by'} -> Procedure\n",
      "Disease - {'affects'} -> Procedure\n",
      "Disease - {'similar_to'} -> Procedure\n",
      "Disease - {'related_to'} -> Procedure\n",
      "Disease - {'overlaps'} -> Procedure\n",
      "Disease - {'causes'} -> Procedure\n",
      "Disease - {'composed_primarily_of'} -> Procedure\n",
      "Disease - {'response_decreased_by'} -> Procedure\n",
      "Disease - {'interacts_with'} -> Procedure\n",
      "Disease - {'ameliorates'} -> Procedure\n",
      "Disease - {'treats'} -> Procedure\n",
      "Disease - {'is_ameliorated_by'} -> Procedure\n",
      "Disease - {'response_increased_by'} -> Procedure\n",
      "Disease - {'located_in'} -> Procedure\n",
      "Disease - {'physically_interacts_with'} -> Procedure\n",
      "Disease - {'derives_from'} -> Procedure\n",
      "Disease - {'precedes'} -> Procedure\n",
      "Disease - {'is_sequence_variant_of'} -> Procedure\n",
      "Disease - {'gene_associated_with_condition'} -> Procedure\n",
      "Disease - {'subclass_of'} -> Procedure\n",
      "Disease - {'produces'} -> Procedure\n",
      "Disease - {'predisposes'} -> Procedure\n",
      "Disease - {'has_part'} -> Procedure\n",
      "Disease - {'prevents'} -> Procedure\n",
      "Disease - {'diagnoses'} -> Procedure\n",
      "Disease - {'occurs_in'} -> Procedure\n",
      "Disease - {'manifestation_of'} -> Procedure\n",
      "Disease - {'has_input'} -> Procedure\n",
      "Disease - {'exacerbates'} -> Procedure\n",
      "Disease - {'coexists_with'} -> Procedure\n",
      "Disease - {'close_match'} -> Procedure\n",
      "Disease - {'disrupts'} -> Procedure\n",
      "Disease - {'has_phenotype'} -> Procedure\n",
      "Disease - {'regulates'} -> Procedure\n",
      "Disease - {'biomarker_for'} -> Procedure\n",
      "Procedure - {'correlated_with'} -> Publication\n",
      "Procedure - {'manifestation_of'} -> Publication\n",
      "Procedure - {'close_match'} -> Publication\n",
      "Procedure - {'subclass_of'} -> Publication\n",
      "Procedure - {'interacts_with'} -> Publication\n",
      "Procedure - {'diagnoses'} -> Publication\n",
      "Procedure - {'occurs_in'} -> Publication\n",
      "Procedure - {'treats'} -> Publication\n",
      "Procedure - {'has_input'} -> Publication\n",
      "Procedure - {'exacerbates'} -> Publication\n",
      "Procedure - {'causes'} -> Publication\n",
      "Procedure - {'coexists_with'} -> Publication\n",
      "Procedure - {'disrupts'} -> Publication\n",
      "Procedure - {'associated_with'} -> Publication\n",
      "Procedure - {'regulates'} -> Publication\n",
      "Procedure - {'produces'} -> Publication\n",
      "Procedure - {'located_in'} -> Publication\n",
      "Procedure - {'predisposes'} -> Publication\n",
      "Procedure - {'related_to'} -> Publication\n",
      "Procedure - {'has_part'} -> Publication\n",
      "Procedure - {'prevents'} -> Publication\n",
      "Procedure - {'precedes'} -> Publication\n",
      "Procedure - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Publication\n",
      "Disease - {'associated_with'} -> Publication\n",
      "Disease - {'gene_associated_with_condition'} -> Publication\n",
      "Disease - {'subclass_of'} -> Publication\n",
      "Disease - {'has_part'} -> Publication\n",
      "Disease - {'interacts_with'} -> Publication\n",
      "Disease - {'produces'} -> Publication\n",
      "Disease - {'predisposes'} -> Publication\n",
      "Disease - {'related_to'} -> Publication\n",
      "Disease - {'prevents'} -> Publication\n",
      "Disease - {'diagnoses'} -> Publication\n",
      "Disease - {'contributes_to'} -> Publication\n",
      "Disease - {'manifestation_of'} -> Publication\n",
      "Disease - {'occurs_in'} -> Publication\n",
      "Disease - {'affects'} -> Publication\n",
      "Disease - {'has_input'} -> Publication\n",
      "Disease - {'exacerbates'} -> Publication\n",
      "Disease - {'coexists_with'} -> Publication\n",
      "Disease - {'close_match'} -> Publication\n",
      "Disease - {'disrupts'} -> Publication\n",
      "Disease - {'regulates'} -> Publication\n",
      "Disease - {'biomarker_for'} -> Publication\n",
      "Disease - {'located_in'} -> Publication\n",
      "Disease - {'treats'} -> Publication\n",
      "Disease - {'causes'} -> Publication\n",
      "Disease - {'disease_has_location'} -> Publication\n",
      "Disease - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ComplexMolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> ComplexMolecularMixture\n",
      "Disease - {'occurs_together_in_literature_with'} -> ComplexMolecularMixture\n",
      "Disease - {'has_biomarker'} -> ComplexMolecularMixture\n",
      "Disease - {'affected_by'} -> ComplexMolecularMixture\n",
      "Disease - {'is_ameliorated_by'} -> ComplexMolecularMixture\n",
      "Disease - {'contribution_from'} -> ComplexMolecularMixture\n",
      "Disease - {'treated_by'} -> ComplexMolecularMixture\n",
      "Disease - {'positively_correlated_with'} -> ComplexMolecularMixture\n",
      "Disease - {'negatively_correlated_with'} -> ComplexMolecularMixture\n",
      "Disease - {'has_real_world_evidence_of_association_with'} -> ComplexMolecularMixture\n",
      "Disease - {'affects_response_to'} -> ComplexMolecularMixture\n",
      "Disease - {'disrupted_by'} -> ComplexMolecularMixture\n",
      "Disease - {'composed_primarily_of'} -> ComplexMolecularMixture\n",
      "Disease - {'has_upstream_actor'} -> ComplexMolecularMixture\n",
      "Disease - {'has_active_ingredient'} -> ComplexMolecularMixture\n",
      "Disease - {'increases_response_to'} -> ComplexMolecularMixture\n",
      "Disease - {'primarily_composed_of'} -> ComplexMolecularMixture\n",
      "Disease - {'regulated_by'} -> ComplexMolecularMixture\n",
      "Disease - {'response_affected_by'} -> ComplexMolecularMixture\n",
      "Disease - {'affects'} -> ComplexMolecularMixture\n",
      "Disease - {'similar_to'} -> ComplexMolecularMixture\n",
      "Disease - {'has_participant'} -> ComplexMolecularMixture\n",
      "Disease - {'response_decreased_by'} -> ComplexMolecularMixture\n",
      "Disease - {'has_part'} -> ComplexMolecularMixture\n",
      "Disease - {'is_exacerbated_by'} -> ComplexMolecularMixture\n",
      "Disease - {'interacts_with'} -> ComplexMolecularMixture\n",
      "Disease - {'response_increased_by'} -> ComplexMolecularMixture\n",
      "Disease - {'participates_in'} -> ComplexMolecularMixture\n",
      "Disease - {'decreases_response_to'} -> ComplexMolecularMixture\n",
      "Disease - {'associated_with'} -> ComplexMolecularMixture\n",
      "Disease - {'related_to'} -> ComplexMolecularMixture\n",
      "Disease - {'overlaps'} -> ComplexMolecularMixture\n",
      "Disease - {'produces'} -> ComplexMolecularMixture\n",
      "Disease - {'subclass_of'} -> ComplexMolecularMixture\n",
      "Disease - {'regulates'} -> ComplexMolecularMixture\n",
      "Disease - {'close_match'} -> ComplexMolecularMixture\n",
      "Disease - {'has_input'} -> ComplexMolecularMixture\n",
      "Disease - {'coexists_with'} -> ComplexMolecularMixture\n",
      "ComplexMolecularMixture - {'correlated_with'} -> Publication\n",
      "ComplexMolecularMixture - {'subclass_of'} -> Publication\n",
      "ComplexMolecularMixture - {'treats'} -> Publication\n",
      "ComplexMolecularMixture - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ClinicalIntervention->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> ClinicalIntervention\n",
      "Disease - {'coexists_with'} -> ClinicalIntervention\n",
      "Disease - {'close_match'} -> ClinicalIntervention\n",
      "Disease - {'has_participant'} -> ClinicalIntervention\n",
      "Disease - {'has_phenotype'} -> ClinicalIntervention\n",
      "Disease - {'disrupts'} -> ClinicalIntervention\n",
      "Disease - {'regulates'} -> ClinicalIntervention\n",
      "Disease - {'biomarker_for'} -> ClinicalIntervention\n",
      "Disease - {'located_in'} -> ClinicalIntervention\n",
      "Disease - {'physically_interacts_with'} -> ClinicalIntervention\n",
      "Disease - {'treats'} -> ClinicalIntervention\n",
      "Disease - {'causes'} -> ClinicalIntervention\n",
      "Disease - {'precedes'} -> ClinicalIntervention\n",
      "Disease - {'associated_with'} -> ClinicalIntervention\n",
      "Disease - {'gene_associated_with_condition'} -> ClinicalIntervention\n",
      "Disease - {'subclass_of'} -> ClinicalIntervention\n",
      "Disease - {'has_part'} -> ClinicalIntervention\n",
      "Disease - {'interacts_with'} -> ClinicalIntervention\n",
      "Disease - {'produces'} -> ClinicalIntervention\n",
      "Disease - {'predisposes'} -> ClinicalIntervention\n",
      "Disease - {'related_to'} -> ClinicalIntervention\n",
      "Disease - {'prevents'} -> ClinicalIntervention\n",
      "Disease - {'diagnoses'} -> ClinicalIntervention\n",
      "Disease - {'manifestation_of'} -> ClinicalIntervention\n",
      "Disease - {'has_input'} -> ClinicalIntervention\n",
      "Disease - {'occurs_in'} -> ClinicalIntervention\n",
      "Disease - {'affects'} -> ClinicalIntervention\n",
      "Disease - {'exacerbates'} -> ClinicalIntervention\n",
      "ClinicalIntervention - {'correlated_with'} -> Publication\n",
      "ClinicalIntervention - {'regulates'} -> Publication\n",
      "ClinicalIntervention - {'diagnoses'} -> Publication\n",
      "ClinicalIntervention - {'treats'} -> Publication\n",
      "ClinicalIntervention - {'has_input'} -> Publication\n",
      "ClinicalIntervention - {'causes'} -> Publication\n",
      "ClinicalIntervention - {'disrupts'} -> Publication\n",
      "ClinicalIntervention - {'associated_with'} -> Publication\n",
      "ClinicalIntervention - {'has_part'} -> Publication\n",
      "ClinicalIntervention - {'located_in'} -> Publication\n",
      "ClinicalIntervention - {'predisposes'} -> Publication\n",
      "ClinicalIntervention - {'related_to'} -> Publication\n",
      "ClinicalIntervention - {'prevents'} -> Publication\n",
      "ClinicalIntervention - {'manifestation_of'} -> Publication\n",
      "ClinicalIntervention - {'precedes'} -> Publication\n",
      "ClinicalIntervention - {'occurs_in'} -> Publication\n",
      "ClinicalIntervention - {'affects'} -> Publication\n",
      "ClinicalIntervention - {'close_match'} -> Publication\n",
      "ClinicalIntervention - {'exacerbates'} -> Publication\n",
      "ClinicalIntervention - {'coexists_with'} -> Publication\n",
      "ClinicalIntervention - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->BiologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> BiologicalProcess\n",
      "Disease - {'occurs_together_in_literature_with'} -> BiologicalProcess\n",
      "Disease - {'overlaps'} -> BiologicalProcess\n",
      "Disease - {'participates_in'} -> BiologicalProcess\n",
      "Disease - {'preceded_by'} -> BiologicalProcess\n",
      "Disease - {'has_real_world_evidence_of_association_with'} -> BiologicalProcess\n",
      "Disease - {'causes'} -> BiologicalProcess\n",
      "Disease - {'disrupts'} -> BiologicalProcess\n",
      "Disease - {'related_to'} -> BiologicalProcess\n",
      "Disease - {'has_participant'} -> BiologicalProcess\n",
      "Disease - {'caused_by'} -> BiologicalProcess\n",
      "Disease - {'superclass_of'} -> BiologicalProcess\n",
      "Disease - {'subclass_of'} -> BiologicalProcess\n",
      "Disease - {'increases_abundance_of'} -> BiologicalProcess\n",
      "Disease - {'has_part'} -> BiologicalProcess\n",
      "Disease - {'affects'} -> BiologicalProcess\n",
      "Disease - {'is_input_of'} -> BiologicalProcess\n",
      "Disease - {'decreases_response_to'} -> BiologicalProcess\n",
      "Disease - {'enables'} -> BiologicalProcess\n",
      "Disease - {'has_substrate'} -> BiologicalProcess\n",
      "Disease - {'interacts_with'} -> BiologicalProcess\n",
      "Disease - {'ameliorates'} -> BiologicalProcess\n",
      "Disease - {'capable_of'} -> BiologicalProcess\n",
      "Disease - {'actively_involves'} -> BiologicalProcess\n",
      "Disease - {'has_active_ingredient'} -> BiologicalProcess\n",
      "Disease - {'has_catalyst'} -> BiologicalProcess\n",
      "Disease - {'primarily_composed_of'} -> BiologicalProcess\n",
      "Disease - {'temporally_related_to'} -> BiologicalProcess\n",
      "Disease - {'actively_involved_in'} -> BiologicalProcess\n",
      "Disease - {'associated_with'} -> BiologicalProcess\n",
      "Disease - {'similar_to'} -> BiologicalProcess\n",
      "Disease - {'catalyzes'} -> BiologicalProcess\n",
      "Disease - {'is_output_of'} -> BiologicalProcess\n",
      "Disease - {'affects_response_to'} -> BiologicalProcess\n",
      "Disease - {'entity_positively_regulates_entity'} -> BiologicalProcess\n",
      "Disease - {'composed_primarily_of'} -> BiologicalProcess\n",
      "Disease - {'has_upstream_actor'} -> BiologicalProcess\n",
      "Disease - {'treated_by'} -> BiologicalProcess\n",
      "Disease - {'exacerbates'} -> BiologicalProcess\n",
      "Disease - {'increases_response_to'} -> BiologicalProcess\n",
      "Disease - {'has_output'} -> BiologicalProcess\n",
      "Disease - {'affected_by'} -> BiologicalProcess\n",
      "Disease - {'treats'} -> BiologicalProcess\n",
      "Disease - {'is_ameliorated_by'} -> BiologicalProcess\n",
      "Disease - {'regulates'} -> BiologicalProcess\n",
      "Disease - {'is_substrate_of'} -> BiologicalProcess\n",
      "Disease - {'part_of'} -> BiologicalProcess\n",
      "Disease - {'has_capability'} -> BiologicalProcess\n",
      "Disease - {'contributes_to'} -> BiologicalProcess\n",
      "Disease - {'has_input'} -> BiologicalProcess\n",
      "Disease - {'enabled_by'} -> BiologicalProcess\n",
      "Disease - {'coexists_with'} -> BiologicalProcess\n",
      "Disease - {'close_match'} -> BiologicalProcess\n",
      "Disease - {'has_phenotype'} -> BiologicalProcess\n",
      "Disease - {'located_in'} -> BiologicalProcess\n",
      "Disease - {'physically_interacts_with'} -> BiologicalProcess\n",
      "Disease - {'precedes'} -> BiologicalProcess\n",
      "Disease - {'directly_physically_interacts_with'} -> BiologicalProcess\n",
      "Disease - {'gene_associated_with_condition'} -> BiologicalProcess\n",
      "Disease - {'produces'} -> BiologicalProcess\n",
      "Disease - {'predisposes'} -> BiologicalProcess\n",
      "Disease - {'prevents'} -> BiologicalProcess\n",
      "Disease - {'diagnoses'} -> BiologicalProcess\n",
      "Disease - {'colocalizes_with'} -> BiologicalProcess\n",
      "Disease - {'occurs_in'} -> BiologicalProcess\n",
      "Disease - {'manifestation_of'} -> BiologicalProcess\n",
      "BiologicalProcess - {'correlated_with'} -> Publication\n",
      "BiologicalProcess - {'disrupts'} -> Publication\n",
      "BiologicalProcess - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalProcess - {'regulates'} -> Publication\n",
      "BiologicalProcess - {'biomarker_for'} -> Publication\n",
      "BiologicalProcess - {'has_part'} -> Publication\n",
      "BiologicalProcess - {'located_in'} -> Publication\n",
      "BiologicalProcess - {'predisposes'} -> Publication\n",
      "BiologicalProcess - {'precedes'} -> Publication\n",
      "BiologicalProcess - {'prevents'} -> Publication\n",
      "BiologicalProcess - {'causes'} -> Publication\n",
      "BiologicalProcess - {'close_match'} -> Publication\n",
      "BiologicalProcess - {'associated_with'} -> Publication\n",
      "BiologicalProcess - {'produces'} -> Publication\n",
      "BiologicalProcess - {'has_participant'} -> Publication\n",
      "BiologicalProcess - {'subclass_of'} -> Publication\n",
      "BiologicalProcess - {'related_to'} -> Publication\n",
      "BiologicalProcess - {'affects'} -> Publication\n",
      "BiologicalProcess - {'treats'} -> Publication\n",
      "BiologicalProcess - {'manifestation_of'} -> Publication\n",
      "BiologicalProcess - {'has_input'} -> Publication\n",
      "BiologicalProcess - {'has_output'} -> Publication\n",
      "BiologicalProcess - {'occurs_in'} -> Publication\n",
      "BiologicalProcess - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Protein->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Protein\n",
      "Disease - {'treated_by'} -> Protein\n",
      "Disease - {'occurs_together_in_literature_with'} -> Protein\n",
      "Disease - {'has_biomarker'} -> Protein\n",
      "Disease - {'entity_negatively_regulates_entity'} -> Protein\n",
      "Disease - {'produces'} -> Protein\n",
      "Disease - {'entity_positively_regulates_entity'} -> Protein\n",
      "Disease - {'contribution_from'} -> Protein\n",
      "Disease - {'phenotype_of'} -> Protein\n",
      "Disease - {'related_to'} -> Protein\n",
      "Disease - {'disrupts'} -> Protein\n",
      "Disease - {'genetic_association'} -> Protein\n",
      "Disease - {'causes'} -> Protein\n",
      "Disease - {'regulates'} -> Protein\n",
      "Disease - {'has_participant'} -> Protein\n",
      "Disease - {'associated_with'} -> Protein\n",
      "Disease - {'decreases_abundance_of'} -> Protein\n",
      "Disease - {'is_ameliorated_by'} -> Protein\n",
      "Disease - {'caused_by'} -> Protein\n",
      "Disease - {'increases_abundance_of'} -> Protein\n",
      "Disease - {'affects'} -> Protein\n",
      "Disease - {'similar_to'} -> Protein\n",
      "Disease - {'overlaps'} -> Protein\n",
      "Disease - {'biomarker_for'} -> Protein\n",
      "Disease - {'affects_response_to'} -> Protein\n",
      "Disease - {'disrupted_by'} -> Protein\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> Protein\n",
      "Disease - {'composed_primarily_of'} -> Protein\n",
      "Disease - {'has_upstream_actor'} -> Protein\n",
      "Disease - {'exacerbates'} -> Protein\n",
      "Disease - {'has_active_ingredient'} -> Protein\n",
      "Disease - {'associated_with_sensitivity_to'} -> Protein\n",
      "Disease - {'increases_response_to'} -> Protein\n",
      "Disease - {'primarily_composed_of'} -> Protein\n",
      "Disease - {'regulated_by'} -> Protein\n",
      "Disease - {'temporally_related_to'} -> Protein\n",
      "Disease - {'associated_with_resistance_to'} -> Protein\n",
      "Disease - {'actively_involved_in'} -> Protein\n",
      "Disease - {'has_output'} -> Protein\n",
      "Disease - {'response_affected_by'} -> Protein\n",
      "Disease - {'response_decreased_by'} -> Protein\n",
      "Disease - {'is_output_of'} -> Protein\n",
      "Disease - {'affected_by'} -> Protein\n",
      "Disease - {'entity_positively_regulated_by_entity'} -> Protein\n",
      "Disease - {'treats'} -> Protein\n",
      "Disease - {'condition_associated_with_gene'} -> Protein\n",
      "Disease - {'sensitivity_associated_with'} -> Protein\n",
      "Disease - {'participates_in'} -> Protein\n",
      "Disease - {'decreased_likelihood_associated_with'} -> Protein\n",
      "Disease - {'contributes_to'} -> Protein\n",
      "Disease - {'gene_associated_with_condition'} -> Protein\n",
      "Disease - {'has_input'} -> Protein\n",
      "Disease - {'preceded_by'} -> Protein\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> Protein\n",
      "Disease - {'enabled_by'} -> Protein\n",
      "Disease - {'superclass_of'} -> Protein\n",
      "Disease - {'decreases_response_to'} -> Protein\n",
      "Disease - {'resistance_associated_with'} -> Protein\n",
      "Disease - {'has_part'} -> Protein\n",
      "Disease - {'is_exacerbated_by'} -> Protein\n",
      "Disease - {'has_substrate'} -> Protein\n",
      "Disease - {'interacts_with'} -> Protein\n",
      "Disease - {'ameliorates'} -> Protein\n",
      "Disease - {'response_increased_by'} -> Protein\n",
      "Disease - {'positively_correlated_with'} -> Protein\n",
      "Disease - {'part_of'} -> Protein\n",
      "Disease - {'has_capability'} -> Protein\n",
      "Disease - {'capable_of'} -> Protein\n",
      "Disease - {'increased_likelihood_associated_with'} -> Protein\n",
      "Disease - {'actively_involves'} -> Protein\n",
      "Disease - {'has_catalyst'} -> Protein\n",
      "Disease - {'coexpressed_with'} -> Protein\n",
      "Disease - {'negatively_correlated_with'} -> Protein\n",
      "Disease - {'gene_product_of'} -> Protein\n",
      "Disease - {'subclass_of'} -> Protein\n",
      "Disease - {'predisposes'} -> Protein\n",
      "Disease - {'prevents'} -> Protein\n",
      "Disease - {'colocalizes_with'} -> Protein\n",
      "Disease - {'diagnoses'} -> Protein\n",
      "Disease - {'occurs_in'} -> Protein\n",
      "Disease - {'manifestation_of'} -> Protein\n",
      "Disease - {'coexists_with'} -> Protein\n",
      "Disease - {'close_match'} -> Protein\n",
      "Disease - {'has_phenotype'} -> Protein\n",
      "Disease - {'located_in'} -> Protein\n",
      "Disease - {'physically_interacts_with'} -> Protein\n",
      "Disease - {'derives_from'} -> Protein\n",
      "Disease - {'disease_has_location'} -> Protein\n",
      "Disease - {'precedes'} -> Protein\n",
      "Disease - {'is_sequence_variant_of'} -> Protein\n",
      "Disease - {'directly_physically_interacts_with'} -> Protein\n",
      "Protein - {'correlated_with'} -> Publication\n",
      "Protein - {'gene_product_of'} -> Publication\n",
      "Protein - {'associated_with'} -> Publication\n",
      "Protein - {'gene_associated_with_condition'} -> Publication\n",
      "Protein - {'located_in'} -> Publication\n",
      "Protein - {'regulates'} -> Publication\n",
      "Protein - {'biomarker_for'} -> Publication\n",
      "Protein - {'expressed_in'} -> Publication\n",
      "Protein - {'has_part'} -> Publication\n",
      "Protein - {'colocalizes_with'} -> Publication\n",
      "Protein - {'produces'} -> Publication\n",
      "Protein - {'predisposes'} -> Publication\n",
      "Protein - {'related_to'} -> Publication\n",
      "Protein - {'prevents'} -> Publication\n",
      "Protein - {'occurs_in'} -> Publication\n",
      "Protein - {'precedes'} -> Publication\n",
      "Protein - {'affects'} -> Publication\n",
      "Protein - {'close_match'} -> Publication\n",
      "Protein - {'subclass_of'} -> Publication\n",
      "Protein - {'interacts_with'} -> Publication\n",
      "Protein - {'treats'} -> Publication\n",
      "Protein - {'has_input'} -> Publication\n",
      "Protein - {'causes'} -> Publication\n",
      "Protein - {'physically_interacts_with'} -> Publication\n",
      "Protein - {'exacerbates'} -> Publication\n",
      "Protein - {'coexists_with'} -> Publication\n",
      "Protein - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Gene->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Gene\n",
      "Disease - {'treated_by'} -> Gene\n",
      "Disease - {'affected_by'} -> Gene\n",
      "Disease - {'phenotype_of'} -> Gene\n",
      "Disease - {'decreases_expression_of'} -> Gene\n",
      "Disease - {'prevented_by'} -> Gene\n",
      "Disease - {'entity_negatively_regulates_entity'} -> Gene\n",
      "Disease - {'contribution_from'} -> Gene\n",
      "Disease - {'has_predisposing_factor'} -> Gene\n",
      "Disease - {'has_biomarker'} -> Gene\n",
      "Disease - {'entity_positively_regulates_entity'} -> Gene\n",
      "Disease - {'has_part'} -> Gene\n",
      "Disease - {'decreases_abundance_of'} -> Gene\n",
      "Disease - {'condition_associated_with_gene'} -> Gene\n",
      "Disease - {'risk_affected_by'} -> Gene\n",
      "Disease - {'affects_risk_for'} -> Gene\n",
      "Disease - {'affects'} -> Gene\n",
      "Disease - {'regulates'} -> Gene\n",
      "Disease - {'caused_by'} -> Gene\n",
      "Disease - {'entity_positively_regulated_by_entity'} -> Gene\n",
      "Disease - {'produces'} -> Gene\n",
      "Disease - {'increases_expression_of'} -> Gene\n",
      "Disease - {'associated_with'} -> Gene\n",
      "Disease - {'occurs_together_in_literature_with'} -> Gene\n",
      "Disease - {'disrupted_by'} -> Gene\n",
      "Disease - {'genetic_association'} -> Gene\n",
      "Disease - {'related_to'} -> Gene\n",
      "Disease - {'disrupts'} -> Gene\n",
      "Disease - {'has_participant'} -> Gene\n",
      "Disease - {'causes'} -> Gene\n",
      "Disease - {'associated_with_sensitivity_to'} -> Gene\n",
      "Disease - {'has_substrate'} -> Gene\n",
      "Disease - {'interacts_with'} -> Gene\n",
      "Disease - {'ameliorates'} -> Gene\n",
      "Disease - {'response_increased_by'} -> Gene\n",
      "Disease - {'positively_correlated_with'} -> Gene\n",
      "Disease - {'has_capability'} -> Gene\n",
      "Disease - {'increased_likelihood_associated_with'} -> Gene\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> Gene\n",
      "Disease - {'actively_involves'} -> Gene\n",
      "Disease - {'has_catalyst'} -> Gene\n",
      "Disease - {'temporally_related_to'} -> Gene\n",
      "Disease - {'coexpressed_with'} -> Gene\n",
      "Disease - {'negatively_correlated_with'} -> Gene\n",
      "Disease - {'similar_to'} -> Gene\n",
      "Disease - {'overlaps'} -> Gene\n",
      "Disease - {'biomarker_for'} -> Gene\n",
      "Disease - {'affects_response_to'} -> Gene\n",
      "Disease - {'composed_primarily_of'} -> Gene\n",
      "Disease - {'has_upstream_actor'} -> Gene\n",
      "Disease - {'gene_associated_with_condition'} -> Gene\n",
      "Disease - {'exacerbates'} -> Gene\n",
      "Disease - {'has_active_ingredient'} -> Gene\n",
      "Disease - {'increases_response_to'} -> Gene\n",
      "Disease - {'primarily_composed_of'} -> Gene\n",
      "Disease - {'regulated_by'} -> Gene\n",
      "Disease - {'associated_with_resistance_to'} -> Gene\n",
      "Disease - {'has_output'} -> Gene\n",
      "Disease - {'response_affected_by'} -> Gene\n",
      "Disease - {'response_decreased_by'} -> Gene\n",
      "Disease - {'increases_abundance_of'} -> Gene\n",
      "Disease - {'treats'} -> Gene\n",
      "Disease - {'is_ameliorated_by'} -> Gene\n",
      "Disease - {'sensitivity_associated_with'} -> Gene\n",
      "Disease - {'part_of'} -> Gene\n",
      "Disease - {'participates_in'} -> Gene\n",
      "Disease - {'decreased_likelihood_associated_with'} -> Gene\n",
      "Disease - {'contributes_to'} -> Gene\n",
      "Disease - {'has_input'} -> Gene\n",
      "Disease - {'preceded_by'} -> Gene\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> Gene\n",
      "Disease - {'enabled_by'} -> Gene\n",
      "Disease - {'decreases_response_to'} -> Gene\n",
      "Disease - {'resistance_associated_with'} -> Gene\n",
      "Disease - {'is_exacerbated_by'} -> Gene\n",
      "Disease - {'subclass_of'} -> Gene\n",
      "Disease - {'predisposes'} -> Gene\n",
      "Disease - {'prevents'} -> Gene\n",
      "Disease - {'colocalizes_with'} -> Gene\n",
      "Disease - {'diagnoses'} -> Gene\n",
      "Disease - {'occurs_in'} -> Gene\n",
      "Disease - {'manifestation_of'} -> Gene\n",
      "Disease - {'coexists_with'} -> Gene\n",
      "Disease - {'close_match'} -> Gene\n",
      "Disease - {'has_phenotype'} -> Gene\n",
      "Disease - {'capable_of'} -> Gene\n",
      "Disease - {'located_in'} -> Gene\n",
      "Disease - {'derives_from'} -> Gene\n",
      "Disease - {'physically_interacts_with'} -> Gene\n",
      "Disease - {'disease_has_location'} -> Gene\n",
      "Disease - {'precedes'} -> Gene\n",
      "Disease - {'is_sequence_variant_of'} -> Gene\n",
      "Disease - {'directly_physically_interacts_with'} -> Gene\n",
      "Disease - {'gene_product_of'} -> Gene\n",
      "Gene - {'correlated_with'} -> Publication\n",
      "Gene - {'prevents'} -> Publication\n",
      "Gene - {'treats'} -> Publication\n",
      "Gene - {'has_input'} -> Publication\n",
      "Gene - {'related_to'} -> Publication\n",
      "Gene - {'occurs_in'} -> Publication\n",
      "Gene - {'affects'} -> Publication\n",
      "Gene - {'exacerbates'} -> Publication\n",
      "Gene - {'coexists_with'} -> Publication\n",
      "Gene - {'disrupts'} -> Publication\n",
      "Gene - {'gene_associated_with_condition'} -> Publication\n",
      "Gene - {'located_in'} -> Publication\n",
      "Gene - {'regulates'} -> Publication\n",
      "Gene - {'biomarker_for'} -> Publication\n",
      "Gene - {'expressed_in'} -> Publication\n",
      "Gene - {'physically_interacts_with'} -> Publication\n",
      "Gene - {'predisposes'} -> Publication\n",
      "Gene - {'causes'} -> Publication\n",
      "Gene - {'precedes'} -> Publication\n",
      "Gene - {'gene_product_of'} -> Publication\n",
      "Gene - {'close_match'} -> Publication\n",
      "Gene - {'associated_with'} -> Publication\n",
      "Gene - {'subclass_of'} -> Publication\n",
      "Gene - {'has_part'} -> Publication\n",
      "Gene - {'colocalizes_with'} -> Publication\n",
      "Gene - {'interacts_with'} -> Publication\n",
      "Gene - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Device->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'correlated_with'} -> Device\n",
      "Disease - {'affected_by'} -> Device\n",
      "Disease - {'participates_in'} -> Device\n",
      "Disease - {'has_participant'} -> Device\n",
      "Disease - {'interacts_with'} -> Device\n",
      "Disease - {'associated_with'} -> Device\n",
      "Disease - {'affects'} -> Device\n",
      "Disease - {'similar_to'} -> Device\n",
      "Disease - {'related_to'} -> Device\n",
      "Disease - {'overlaps'} -> Device\n",
      "Disease - {'composed_primarily_of'} -> Device\n",
      "Disease - {'has_upstream_actor'} -> Device\n",
      "Disease - {'primarily_composed_of'} -> Device\n",
      "Disease - {'prevents'} -> Device\n",
      "Disease - {'diagnoses'} -> Device\n",
      "Disease - {'manifestation_of'} -> Device\n",
      "Disease - {'has_input'} -> Device\n",
      "Disease - {'occurs_in'} -> Device\n",
      "Disease - {'close_match'} -> Device\n",
      "Disease - {'exacerbates'} -> Device\n",
      "Disease - {'coexists_with'} -> Device\n",
      "Disease - {'has_phenotype'} -> Device\n",
      "Disease - {'disrupts'} -> Device\n",
      "Disease - {'located_in'} -> Device\n",
      "Disease - {'regulates'} -> Device\n",
      "Disease - {'physically_interacts_with'} -> Device\n",
      "Disease - {'treats'} -> Device\n",
      "Disease - {'derives_from'} -> Device\n",
      "Disease - {'causes'} -> Device\n",
      "Disease - {'disease_has_location'} -> Device\n",
      "Disease - {'precedes'} -> Device\n",
      "Disease - {'subclass_of'} -> Device\n",
      "Disease - {'gene_associated_with_condition'} -> Device\n",
      "Disease - {'has_part'} -> Device\n",
      "Disease - {'produces'} -> Device\n",
      "Disease - {'predisposes'} -> Device\n",
      "Device - {'subclass_of'} -> Publication\n",
      "Device - {'diagnoses'} -> Publication\n",
      "Device - {'related_to'} -> Publication\n",
      "Device - {'occurs_in'} -> Publication\n",
      "Device - {'affects'} -> Publication\n",
      "Device - {'treats'} -> Publication\n",
      "Device - {'has_input'} -> Publication\n",
      "Device - {'coexists_with'} -> Publication\n",
      "Device - {'regulates'} -> Publication\n",
      "Device - {'disrupts'} -> Publication\n",
      "Device - {'biomarker_for'} -> Publication\n",
      "Device - {'gene_associated_with_condition'} -> Publication\n",
      "Device - {'has_part'} -> Publication\n",
      "Device - {'located_in'} -> Publication\n",
      "Device - {'causes'} -> Publication\n",
      "Device - {'precedes'} -> Publication\n",
      "Device - {'close_match'} -> Publication\n",
      "Device - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->PhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'positively_correlated_with'} -> PhenotypicFeature\n",
      "Disease - {'negatively_correlated_with'} -> PhenotypicFeature\n",
      "Disease - {'occurs_together_in_literature_with'} -> PhenotypicFeature\n",
      "Disease - {'superclass_of'} -> PhenotypicFeature\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> PhenotypicFeature\n",
      "Disease - {'decreased_likelihood_associated_with'} -> PhenotypicFeature\n",
      "Disease - {'causes'} -> PhenotypicFeature\n",
      "Disease - {'affected_by'} -> PhenotypicFeature\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> PhenotypicFeature\n",
      "Disease - {'caused_by'} -> PhenotypicFeature\n",
      "Disease - {'has_predisposing_factor'} -> PhenotypicFeature\n",
      "Disease - {'part_of'} -> PhenotypicFeature\n",
      "Disease - {'has_real_world_evidence_of_association_with'} -> PhenotypicFeature\n",
      "Disease - {'coexists_with'} -> PhenotypicFeature\n",
      "Disease - {'related_to'} -> PhenotypicFeature\n",
      "Disease - {'has_mode_of_inheritance'} -> PhenotypicFeature\n",
      "Disease - {'has_phenotype'} -> PhenotypicFeature\n",
      "Disease - {'associated_with'} -> PhenotypicFeature\n",
      "Disease - {'correlated_with'} -> PhenotypicFeature\n",
      "Disease - {'overlaps'} -> PhenotypicFeature\n",
      "Disease - {'phenotype_of'} -> PhenotypicFeature\n",
      "Disease - {'increases_abundance_of'} -> PhenotypicFeature\n",
      "Disease - {'has_manifestation'} -> PhenotypicFeature\n",
      "Disease - {'increased_likelihood_associated_with'} -> PhenotypicFeature\n",
      "Disease - {'is_diagnosed_by'} -> PhenotypicFeature\n",
      "Disease - {'subclass_of'} -> PhenotypicFeature\n",
      "Disease - {'has_part'} -> PhenotypicFeature\n",
      "Disease - {'affects'} -> PhenotypicFeature\n",
      "Disease - {'gene_associated_with_condition'} -> PhenotypicFeature\n",
      "Disease - {'decreases_response_to'} -> PhenotypicFeature\n",
      "Disease - {'is_exacerbated_by'} -> PhenotypicFeature\n",
      "Disease - {'predisposes'} -> PhenotypicFeature\n",
      "Disease - {'ameliorates'} -> PhenotypicFeature\n",
      "Disease - {'genetic_association'} -> PhenotypicFeature\n",
      "Disease - {'disrupted_by'} -> PhenotypicFeature\n",
      "Disease - {'has_upstream_actor'} -> PhenotypicFeature\n",
      "Disease - {'exacerbates'} -> PhenotypicFeature\n",
      "Disease - {'has_active_ingredient'} -> PhenotypicFeature\n",
      "Disease - {'primarily_composed_of'} -> PhenotypicFeature\n",
      "Disease - {'regulated_by'} -> PhenotypicFeature\n",
      "Disease - {'temporally_related_to'} -> PhenotypicFeature\n",
      "Disease - {'associated_with_resistance_to'} -> PhenotypicFeature\n",
      "Disease - {'response_affected_by'} -> PhenotypicFeature\n",
      "Disease - {'coexpressed_with'} -> PhenotypicFeature\n",
      "Disease - {'similar_to'} -> PhenotypicFeature\n",
      "Disease - {'biomarker_for'} -> PhenotypicFeature\n",
      "Disease - {'affects_response_to'} -> PhenotypicFeature\n",
      "Disease - {'condition_associated_with_gene'} -> PhenotypicFeature\n",
      "Disease - {'entity_positively_regulates_entity'} -> PhenotypicFeature\n",
      "Disease - {'composed_primarily_of'} -> PhenotypicFeature\n",
      "Disease - {'treated_by'} -> PhenotypicFeature\n",
      "Disease - {'associated_with_sensitivity_to'} -> PhenotypicFeature\n",
      "Disease - {'increases_response_to'} -> PhenotypicFeature\n",
      "Disease - {'preceded_by'} -> PhenotypicFeature\n",
      "Disease - {'response_decreased_by'} -> PhenotypicFeature\n",
      "Disease - {'disrupts'} -> PhenotypicFeature\n",
      "Disease - {'resistance_associated_with'} -> PhenotypicFeature\n",
      "Disease - {'interacts_with'} -> PhenotypicFeature\n",
      "Disease - {'treats'} -> PhenotypicFeature\n",
      "Disease - {'is_ameliorated_by'} -> PhenotypicFeature\n",
      "Disease - {'regulates'} -> PhenotypicFeature\n",
      "Disease - {'response_increased_by'} -> PhenotypicFeature\n",
      "Disease - {'sensitivity_associated_with'} -> PhenotypicFeature\n",
      "Disease - {'has_biomarker'} -> PhenotypicFeature\n",
      "Disease - {'participates_in'} -> PhenotypicFeature\n",
      "Disease - {'contributes_to'} -> PhenotypicFeature\n",
      "Disease - {'has_participant'} -> PhenotypicFeature\n",
      "Disease - {'produces'} -> PhenotypicFeature\n",
      "Disease - {'prevents'} -> PhenotypicFeature\n",
      "Disease - {'diagnoses'} -> PhenotypicFeature\n",
      "Disease - {'manifestation_of'} -> PhenotypicFeature\n",
      "Disease - {'has_input'} -> PhenotypicFeature\n",
      "Disease - {'occurs_in'} -> PhenotypicFeature\n",
      "Disease - {'close_match'} -> PhenotypicFeature\n",
      "Disease - {'contraindicated_for'} -> PhenotypicFeature\n",
      "Disease - {'located_in'} -> PhenotypicFeature\n",
      "Disease - {'derives_from'} -> PhenotypicFeature\n",
      "Disease - {'disease_has_location'} -> PhenotypicFeature\n",
      "Disease - {'actively_involved_in'} -> PhenotypicFeature\n",
      "Disease - {'precedes'} -> PhenotypicFeature\n",
      "Disease - {'is_sequence_variant_of'} -> PhenotypicFeature\n",
      "Disease - {'gene_product_of'} -> PhenotypicFeature\n",
      "PhenotypicFeature - {'correlated_with'} -> Publication\n",
      "PhenotypicFeature - {'occurs_in'} -> Publication\n",
      "PhenotypicFeature - {'affects'} -> Publication\n",
      "PhenotypicFeature - {'close_match'} -> Publication\n",
      "PhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "PhenotypicFeature - {'exacerbates'} -> Publication\n",
      "PhenotypicFeature - {'coexists_with'} -> Publication\n",
      "PhenotypicFeature - {'subclass_of'} -> Publication\n",
      "PhenotypicFeature - {'regulates'} -> Publication\n",
      "PhenotypicFeature - {'diagnoses'} -> Publication\n",
      "PhenotypicFeature - {'treats'} -> Publication\n",
      "PhenotypicFeature - {'has_input'} -> Publication\n",
      "PhenotypicFeature - {'causes'} -> Publication\n",
      "PhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "PhenotypicFeature - {'disrupts'} -> Publication\n",
      "PhenotypicFeature - {'associated_with'} -> Publication\n",
      "PhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "PhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "PhenotypicFeature - {'has_part'} -> Publication\n",
      "PhenotypicFeature - {'located_in'} -> Publication\n",
      "PhenotypicFeature - {'interacts_with'} -> Publication\n",
      "PhenotypicFeature - {'produces'} -> Publication\n",
      "PhenotypicFeature - {'prevents'} -> Publication\n",
      "PhenotypicFeature - {'contributes_to'} -> Publication\n",
      "PhenotypicFeature - {'predisposes'} -> Publication\n",
      "PhenotypicFeature - {'precedes'} -> Publication\n",
      "PhenotypicFeature - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->SmallMolecule->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'is_side_effect_of'} -> SmallMolecule\n",
      "Disease - {'treated_by'} -> SmallMolecule\n",
      "Disease - {'location of'} -> SmallMolecule\n",
      "Disease - {'occurs_together_in_literature_with'} -> SmallMolecule\n",
      "Disease - {'causes'} -> SmallMolecule\n",
      "Disease - {'contribution_from'} -> SmallMolecule\n",
      "Disease - {'has_contraindication'} -> SmallMolecule\n",
      "Disease - {'entity_positively_regulated_by_entity'} -> SmallMolecule\n",
      "Disease - {'affected_by'} -> SmallMolecule\n",
      "Disease - {'has_biomarker'} -> SmallMolecule\n",
      "Disease - {'is_ameliorated_by'} -> SmallMolecule\n",
      "Disease - {'decreases_abundance_of'} -> SmallMolecule\n",
      "Disease - {'increased_likelihood_associated_with'} -> SmallMolecule\n",
      "Disease - {'associated_with'} -> SmallMolecule\n",
      "Disease - {'has_part'} -> SmallMolecule\n",
      "Disease - {'decreased_likelihood_associated_with'} -> SmallMolecule\n",
      "Disease - {'produces'} -> SmallMolecule\n",
      "Disease - {'related_to'} -> SmallMolecule\n",
      "Disease - {'caused_by'} -> SmallMolecule\n",
      "Disease - {'positively_correlated_with'} -> SmallMolecule\n",
      "Disease - {'negatively_correlated_with'} -> SmallMolecule\n",
      "Disease - {'increases_abundance_of'} -> SmallMolecule\n",
      "Disease - {'has_participant'} -> SmallMolecule\n",
      "Disease - {'adverse_event_caused_by'} -> SmallMolecule\n",
      "Disease - {'has_predisposing_factor'} -> SmallMolecule\n",
      "Disease - {'is_exacerbated_by'} -> SmallMolecule\n",
      "Disease - {'has_real_world_evidence_of_association_with'} -> SmallMolecule\n",
      "Disease - {'prevented_by'} -> SmallMolecule\n",
      "Disease - {'disrupted_by'} -> SmallMolecule\n",
      "Disease - {'correlated_with'} -> SmallMolecule\n",
      "Disease - {'affects'} -> SmallMolecule\n",
      "Disease - {'response_decreased_by'} -> SmallMolecule\n",
      "Disease - {'disrupts'} -> SmallMolecule\n",
      "Disease - {'is_input_of'} -> SmallMolecule\n",
      "Disease - {'resistance_associated_with'} -> SmallMolecule\n",
      "Disease - {'has_substrate'} -> SmallMolecule\n",
      "Disease - {'interacts_with'} -> SmallMolecule\n",
      "Disease - {'ameliorates'} -> SmallMolecule\n",
      "Disease - {'treats'} -> SmallMolecule\n",
      "Disease - {'regulates'} -> SmallMolecule\n",
      "Disease - {'response_increased_by'} -> SmallMolecule\n",
      "Disease - {'sensitivity_associated_with'} -> SmallMolecule\n",
      "Disease - {'is_substrate_of'} -> SmallMolecule\n",
      "Disease - {'part_of'} -> SmallMolecule\n",
      "Disease - {'has_capability'} -> SmallMolecule\n",
      "Disease - {'participates_in'} -> SmallMolecule\n",
      "Disease - {'capable_of'} -> SmallMolecule\n",
      "Disease - {'contributes_to'} -> SmallMolecule\n",
      "Disease - {'actively_involves'} -> SmallMolecule\n",
      "Disease - {'has_input'} -> SmallMolecule\n",
      "Disease - {'enabled_by'} -> SmallMolecule\n",
      "Disease - {'has_catalyst'} -> SmallMolecule\n",
      "Disease - {'entity_negatively_regulates_entity'} -> SmallMolecule\n",
      "Disease - {'decreases_response_to'} -> SmallMolecule\n",
      "Disease - {'enables'} -> SmallMolecule\n",
      "Disease - {'catalyzes'} -> SmallMolecule\n",
      "Disease - {'biomarker_for'} -> SmallMolecule\n",
      "Disease - {'entity_positively_regulates_entity'} -> SmallMolecule\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> SmallMolecule\n",
      "Disease - {'has_upstream_actor'} -> SmallMolecule\n",
      "Disease - {'exacerbates'} -> SmallMolecule\n",
      "Disease - {'has_active_ingredient'} -> SmallMolecule\n",
      "Disease - {'increases_response_to'} -> SmallMolecule\n",
      "Disease - {'primarily_composed_of'} -> SmallMolecule\n",
      "Disease - {'regulated_by'} -> SmallMolecule\n",
      "Disease - {'temporally_related_to'} -> SmallMolecule\n",
      "Disease - {'associated_with_resistance_to'} -> SmallMolecule\n",
      "Disease - {'actively_involved_in'} -> SmallMolecule\n",
      "Disease - {'has_output'} -> SmallMolecule\n",
      "Disease - {'response_affected_by'} -> SmallMolecule\n",
      "Disease - {'coexpressed_with'} -> SmallMolecule\n",
      "Disease - {'similar_to'} -> SmallMolecule\n",
      "Disease - {'is_output_of'} -> SmallMolecule\n",
      "Disease - {'overlaps'} -> SmallMolecule\n",
      "Disease - {'affects_response_to'} -> SmallMolecule\n",
      "Disease - {'composed_primarily_of'} -> SmallMolecule\n",
      "Disease - {'associated_with_sensitivity_to'} -> SmallMolecule\n",
      "Disease - {'preceded_by'} -> SmallMolecule\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> SmallMolecule\n",
      "Disease - {'adverse_event_of'} -> SmallMolecule\n",
      "Disease - {'derives_from'} -> SmallMolecule\n",
      "Disease - {'in_taxon'} -> SmallMolecule\n",
      "Disease - {'physically_interacts_with'} -> SmallMolecule\n",
      "Disease - {'disease_has_location'} -> SmallMolecule\n",
      "Disease - {'directly_physically_interacts_with'} -> SmallMolecule\n",
      "Disease - {'precedes'} -> SmallMolecule\n",
      "Disease - {'gene_associated_with_condition'} -> SmallMolecule\n",
      "Disease - {'subclass_of'} -> SmallMolecule\n",
      "Disease - {'colocalizes_with'} -> SmallMolecule\n",
      "Disease - {'predisposes'} -> SmallMolecule\n",
      "Disease - {'prevents'} -> SmallMolecule\n",
      "Disease - {'diagnoses'} -> SmallMolecule\n",
      "Disease - {'occurs_in'} -> SmallMolecule\n",
      "Disease - {'disease_has_basis_in'} -> SmallMolecule\n",
      "Disease - {'manifestation_of'} -> SmallMolecule\n",
      "Disease - {'close_match'} -> SmallMolecule\n",
      "Disease - {'coexists_with'} -> SmallMolecule\n",
      "Disease - {'has_phenotype'} -> SmallMolecule\n",
      "Disease - {'chemically_similar_to'} -> SmallMolecule\n",
      "Disease - {'located_in'} -> SmallMolecule\n",
      "SmallMolecule - {'correlated_with'} -> Publication\n",
      "SmallMolecule - {'disrupts'} -> Publication\n",
      "SmallMolecule - {'gene_associated_with_condition'} -> Publication\n",
      "SmallMolecule - {'located_in'} -> Publication\n",
      "SmallMolecule - {'regulates'} -> Publication\n",
      "SmallMolecule - {'has_part'} -> Publication\n",
      "SmallMolecule - {'physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'predisposes'} -> Publication\n",
      "SmallMolecule - {'related_to'} -> Publication\n",
      "SmallMolecule - {'causes'} -> Publication\n",
      "SmallMolecule - {'directly_physically_interacts_with'} -> Publication\n",
      "SmallMolecule - {'precedes'} -> Publication\n",
      "SmallMolecule - {'close_match'} -> Publication\n",
      "SmallMolecule - {'associated_with'} -> Publication\n",
      "SmallMolecule - {'has_metabolite'} -> Publication\n",
      "SmallMolecule - {'subclass_of'} -> Publication\n",
      "SmallMolecule - {'interacts_with'} -> Publication\n",
      "SmallMolecule - {'treats'} -> Publication\n",
      "SmallMolecule - {'has_input'} -> Publication\n",
      "SmallMolecule - {'occurs_in'} -> Publication\n",
      "SmallMolecule - {'affects'} -> Publication\n",
      "SmallMolecule - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Cell->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'treated_by'} -> Cell\n",
      "Disease - {'occurs_together_in_literature_with'} -> Cell\n",
      "Disease - {'superclass_of'} -> Cell\n",
      "Disease - {'located_in'} -> Cell\n",
      "Disease - {'causes'} -> Cell\n",
      "Disease - {'part_of'} -> Cell\n",
      "Disease - {'correlated_with'} -> Cell\n",
      "Disease - {'decreases_abundance_of'} -> Cell\n",
      "Disease - {'caused_by'} -> Cell\n",
      "Disease - {'has_capability'} -> Cell\n",
      "Disease - {'increases_abundance_of'} -> Cell\n",
      "Disease - {'subclass_of'} -> Cell\n",
      "Disease - {'related_to'} -> Cell\n",
      "Disease - {'associated_with'} -> Cell\n",
      "Disease - {'has_participant'} -> Cell\n",
      "Disease - {'affects'} -> Cell\n",
      "Disease - {'disrupts'} -> Cell\n",
      "Disease - {'capable_of'} -> Cell\n",
      "Disease - {'affects_response_to'} -> Cell\n",
      "Disease - {'treats'} -> Cell\n",
      "Disease - {'condition_associated_with_gene'} -> Cell\n",
      "Disease - {'composed_primarily_of'} -> Cell\n",
      "Disease - {'decreased_likelihood_associated_with'} -> Cell\n",
      "Disease - {'gene_associated_with_condition'} -> Cell\n",
      "Disease - {'has_input'} -> Cell\n",
      "Disease - {'preceded_by'} -> Cell\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> Cell\n",
      "Disease - {'enabled_by'} -> Cell\n",
      "Disease - {'response_decreased_by'} -> Cell\n",
      "Disease - {'is_input_of'} -> Cell\n",
      "Disease - {'resistance_associated_with'} -> Cell\n",
      "Disease - {'affected_by'} -> Cell\n",
      "Disease - {'has_substrate'} -> Cell\n",
      "Disease - {'interacts_with'} -> Cell\n",
      "Disease - {'ameliorates'} -> Cell\n",
      "Disease - {'is_ameliorated_by'} -> Cell\n",
      "Disease - {'regulates'} -> Cell\n",
      "Disease - {'response_increased_by'} -> Cell\n",
      "Disease - {'sensitivity_associated_with'} -> Cell\n",
      "Disease - {'is_substrate_of'} -> Cell\n",
      "Disease - {'positively_correlated_with'} -> Cell\n",
      "Disease - {'has_biomarker'} -> Cell\n",
      "Disease - {'participates_in'} -> Cell\n",
      "Disease - {'contributes_to'} -> Cell\n",
      "Disease - {'increased_likelihood_associated_with'} -> Cell\n",
      "Disease - {'actively_involves'} -> Cell\n",
      "Disease - {'has_catalyst'} -> Cell\n",
      "Disease - {'expressed_in'} -> Cell\n",
      "Disease - {'decreases_response_to'} -> Cell\n",
      "Disease - {'enables'} -> Cell\n",
      "Disease - {'has_part'} -> Cell\n",
      "Disease - {'is_exacerbated_by'} -> Cell\n",
      "Disease - {'catalyzes'} -> Cell\n",
      "Disease - {'overlaps'} -> Cell\n",
      "Disease - {'biomarker_for'} -> Cell\n",
      "Disease - {'contribution_from'} -> Cell\n",
      "Disease - {'genetic_association'} -> Cell\n",
      "Disease - {'entity_positively_regulates_entity'} -> Cell\n",
      "Disease - {'disrupted_by'} -> Cell\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> Cell\n",
      "Disease - {'has_upstream_actor'} -> Cell\n",
      "Disease - {'exacerbates'} -> Cell\n",
      "Disease - {'has_active_ingredient'} -> Cell\n",
      "Disease - {'associated_with_sensitivity_to'} -> Cell\n",
      "Disease - {'increases_response_to'} -> Cell\n",
      "Disease - {'primarily_composed_of'} -> Cell\n",
      "Disease - {'regulated_by'} -> Cell\n",
      "Disease - {'temporally_related_to'} -> Cell\n",
      "Disease - {'associated_with_resistance_to'} -> Cell\n",
      "Disease - {'actively_involved_in'} -> Cell\n",
      "Disease - {'has_output'} -> Cell\n",
      "Disease - {'response_affected_by'} -> Cell\n",
      "Disease - {'coexpressed_with'} -> Cell\n",
      "Disease - {'negatively_correlated_with'} -> Cell\n",
      "Disease - {'similar_to'} -> Cell\n",
      "Disease - {'is_output_of'} -> Cell\n",
      "Disease - {'derives_from'} -> Cell\n",
      "Disease - {'disease_has_location'} -> Cell\n",
      "Disease - {'precedes'} -> Cell\n",
      "Disease - {'produces'} -> Cell\n",
      "Disease - {'predisposes'} -> Cell\n",
      "Disease - {'prevents'} -> Cell\n",
      "Disease - {'diagnoses'} -> Cell\n",
      "Disease - {'occurs_in'} -> Cell\n",
      "Disease - {'manifestation_of'} -> Cell\n",
      "Disease - {'close_match'} -> Cell\n",
      "Disease - {'coexists_with'} -> Cell\n",
      "Disease - {'has_phenotype'} -> Cell\n",
      "Cell - {'subclass_of'} -> Publication\n",
      "Cell - {'has_part'} -> Publication\n",
      "Cell - {'produces'} -> Publication\n",
      "Cell - {'related_to'} -> Publication\n",
      "Cell - {'occurs_in'} -> Publication\n",
      "Cell - {'affects'} -> Publication\n",
      "Cell - {'coexists_with'} -> Publication\n",
      "Cell - {'close_match'} -> Publication\n",
      "Cell - {'located_in'} -> Publication\n",
      "Cell - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->MolecularEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'treated_by'} -> MolecularEntity\n",
      "Disease - {'related_to'} -> MolecularEntity\n",
      "Disease - {'caused_by'} -> MolecularEntity\n",
      "Disease - {'has_output'} -> MolecularEntity\n",
      "Disease - {'actively_involves'} -> MolecularEntity\n",
      "Disease - {'has_catalyst'} -> MolecularEntity\n",
      "Disease - {'temporally_related_to'} -> MolecularEntity\n",
      "Disease - {'associated_with'} -> MolecularEntity\n",
      "Disease - {'affects'} -> MolecularEntity\n",
      "Disease - {'similar_to'} -> MolecularEntity\n",
      "Disease - {'overlaps'} -> MolecularEntity\n",
      "Disease - {'affects_response_to'} -> MolecularEntity\n",
      "Disease - {'entity_positively_regulates_entity'} -> MolecularEntity\n",
      "Disease - {'disrupted_by'} -> MolecularEntity\n",
      "Disease - {'composed_primarily_of'} -> MolecularEntity\n",
      "Disease - {'has_upstream_actor'} -> MolecularEntity\n",
      "Disease - {'exacerbates'} -> MolecularEntity\n",
      "Disease - {'has_active_ingredient'} -> MolecularEntity\n",
      "Disease - {'increases_response_to'} -> MolecularEntity\n",
      "Disease - {'primarily_composed_of'} -> MolecularEntity\n",
      "Disease - {'regulated_by'} -> MolecularEntity\n",
      "Disease - {'response_affected_by'} -> MolecularEntity\n",
      "Disease - {'response_decreased_by'} -> MolecularEntity\n",
      "Disease - {'disrupts'} -> MolecularEntity\n",
      "Disease - {'affected_by'} -> MolecularEntity\n",
      "Disease - {'treats'} -> MolecularEntity\n",
      "Disease - {'is_ameliorated_by'} -> MolecularEntity\n",
      "Disease - {'part_of'} -> MolecularEntity\n",
      "Disease - {'participates_in'} -> MolecularEntity\n",
      "Disease - {'has_input'} -> MolecularEntity\n",
      "Disease - {'preceded_by'} -> MolecularEntity\n",
      "Disease - {'enabled_by'} -> MolecularEntity\n",
      "Disease - {'has_participant'} -> MolecularEntity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> MolecularEntity\n",
      "Disease - {'decreases_response_to'} -> MolecularEntity\n",
      "Disease - {'has_part'} -> MolecularEntity\n",
      "Disease - {'is_exacerbated_by'} -> MolecularEntity\n",
      "Disease - {'has_substrate'} -> MolecularEntity\n",
      "Disease - {'interacts_with'} -> MolecularEntity\n",
      "Disease - {'ameliorates'} -> MolecularEntity\n",
      "Disease - {'regulates'} -> MolecularEntity\n",
      "Disease - {'response_increased_by'} -> MolecularEntity\n",
      "Disease - {'has_capability'} -> MolecularEntity\n",
      "Disease - {'derives_from'} -> MolecularEntity\n",
      "Disease - {'physically_interacts_with'} -> MolecularEntity\n",
      "Disease - {'disease_has_location'} -> MolecularEntity\n",
      "Disease - {'causes'} -> MolecularEntity\n",
      "Disease - {'directly_physically_interacts_with'} -> MolecularEntity\n",
      "Disease - {'precedes'} -> MolecularEntity\n",
      "Disease - {'is_sequence_variant_of'} -> MolecularEntity\n",
      "Disease - {'subclass_of'} -> MolecularEntity\n",
      "Disease - {'colocalizes_with'} -> MolecularEntity\n",
      "Disease - {'disease_has_basis_in'} -> MolecularEntity\n",
      "Disease - {'contributes_to'} -> MolecularEntity\n",
      "Disease - {'produces'} -> MolecularEntity\n",
      "Disease - {'predisposes'} -> MolecularEntity\n",
      "Disease - {'diagnoses'} -> MolecularEntity\n",
      "Disease - {'occurs_in'} -> MolecularEntity\n",
      "Disease - {'manifestation_of'} -> MolecularEntity\n",
      "Disease - {'close_match'} -> MolecularEntity\n",
      "Disease - {'coexists_with'} -> MolecularEntity\n",
      "Disease - {'chemically_similar_to'} -> MolecularEntity\n",
      "Disease - {'located_in'} -> MolecularEntity\n",
      "Disease - {'biomarker_for'} -> MolecularEntity\n",
      "MolecularEntity - {'causes'} -> Publication\n",
      "MolecularEntity - {'associated_with'} -> Publication\n",
      "MolecularEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "MolecularEntity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularEntity - {'regulates'} -> Publication\n",
      "MolecularEntity - {'disrupts'} -> Publication\n",
      "MolecularEntity - {'located_in'} -> Publication\n",
      "MolecularEntity - {'biomarker_for'} -> Publication\n",
      "MolecularEntity - {'interacts_with'} -> Publication\n",
      "MolecularEntity - {'predisposes'} -> Publication\n",
      "MolecularEntity - {'related_to'} -> Publication\n",
      "MolecularEntity - {'has_part'} -> Publication\n",
      "MolecularEntity - {'occurs_in'} -> Publication\n",
      "MolecularEntity - {'affects'} -> Publication\n",
      "MolecularEntity - {'precedes'} -> Publication\n",
      "MolecularEntity - {'coexists_with'} -> Publication\n",
      "MolecularEntity - {'close_match'} -> Publication\n",
      "MolecularEntity - {'subclass_of'} -> Publication\n",
      "MolecularEntity - {'treats'} -> Publication\n",
      "MolecularEntity - {'physically_interacts_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ChemicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'treated_by'} -> ChemicalEntity\n",
      "Disease - {'occurs_together_in_literature_with'} -> ChemicalEntity\n",
      "Disease - {'is_ameliorated_by'} -> ChemicalEntity\n",
      "Disease - {'has_biomarker'} -> ChemicalEntity\n",
      "Disease - {'disrupted_by'} -> ChemicalEntity\n",
      "Disease - {'has_contraindication'} -> ChemicalEntity\n",
      "Disease - {'contribution_from'} -> ChemicalEntity\n",
      "Disease - {'affected_by'} -> ChemicalEntity\n",
      "Disease - {'has_predisposing_factor'} -> ChemicalEntity\n",
      "Disease - {'increases_abundance_of'} -> ChemicalEntity\n",
      "Disease - {'entity_positively_regulated_by_entity'} -> ChemicalEntity\n",
      "Disease - {'has_real_world_evidence_of_association_with'} -> ChemicalEntity\n",
      "Disease - {'produces'} -> ChemicalEntity\n",
      "Disease - {'decreases_abundance_of'} -> ChemicalEntity\n",
      "Disease - {'prevented_by'} -> ChemicalEntity\n",
      "Disease - {'related_to'} -> ChemicalEntity\n",
      "Disease - {'caused_by'} -> ChemicalEntity\n",
      "Disease - {'positively_correlated_with'} -> ChemicalEntity\n",
      "Disease - {'negatively_correlated_with'} -> ChemicalEntity\n",
      "Disease - {'associated_with'} -> ChemicalEntity\n",
      "Disease - {'causes'} -> ChemicalEntity\n",
      "Disease - {'has_participant'} -> ChemicalEntity\n",
      "Disease - {'correlated_with'} -> ChemicalEntity\n",
      "Disease - {'affects'} -> ChemicalEntity\n",
      "Disease - {'composed_primarily_of'} -> ChemicalEntity\n",
      "Disease - {'exacerbates'} -> ChemicalEntity\n",
      "Disease - {'associated_with_sensitivity_to'} -> ChemicalEntity\n",
      "Disease - {'increases_response_to'} -> ChemicalEntity\n",
      "Disease - {'associated_with_resistance_to'} -> ChemicalEntity\n",
      "Disease - {'has_output'} -> ChemicalEntity\n",
      "Disease - {'response_decreased_by'} -> ChemicalEntity\n",
      "Disease - {'disrupts'} -> ChemicalEntity\n",
      "Disease - {'resistance_associated_with'} -> ChemicalEntity\n",
      "Disease - {'has_substrate'} -> ChemicalEntity\n",
      "Disease - {'interacts_with'} -> ChemicalEntity\n",
      "Disease - {'treats'} -> ChemicalEntity\n",
      "Disease - {'regulates'} -> ChemicalEntity\n",
      "Disease - {'response_increased_by'} -> ChemicalEntity\n",
      "Disease - {'sensitivity_associated_with'} -> ChemicalEntity\n",
      "Disease - {'is_substrate_of'} -> ChemicalEntity\n",
      "Disease - {'part_of'} -> ChemicalEntity\n",
      "Disease - {'has_capability'} -> ChemicalEntity\n",
      "Disease - {'participates_in'} -> ChemicalEntity\n",
      "Disease - {'decreased_likelihood_associated_with'} -> ChemicalEntity\n",
      "Disease - {'contributes_to'} -> ChemicalEntity\n",
      "Disease - {'has_input'} -> ChemicalEntity\n",
      "Disease - {'preceded_by'} -> ChemicalEntity\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> ChemicalEntity\n",
      "Disease - {'enabled_by'} -> ChemicalEntity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> ChemicalEntity\n",
      "Disease - {'is_input_of'} -> ChemicalEntity\n",
      "Disease - {'decreases_response_to'} -> ChemicalEntity\n",
      "Disease - {'enables'} -> ChemicalEntity\n",
      "Disease - {'has_part'} -> ChemicalEntity\n",
      "Disease - {'is_exacerbated_by'} -> ChemicalEntity\n",
      "Disease - {'ameliorates'} -> ChemicalEntity\n",
      "Disease - {'capable_of'} -> ChemicalEntity\n",
      "Disease - {'increased_likelihood_associated_with'} -> ChemicalEntity\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> ChemicalEntity\n",
      "Disease - {'actively_involves'} -> ChemicalEntity\n",
      "Disease - {'has_upstream_actor'} -> ChemicalEntity\n",
      "Disease - {'has_active_ingredient'} -> ChemicalEntity\n",
      "Disease - {'has_catalyst'} -> ChemicalEntity\n",
      "Disease - {'primarily_composed_of'} -> ChemicalEntity\n",
      "Disease - {'regulated_by'} -> ChemicalEntity\n",
      "Disease - {'temporally_related_to'} -> ChemicalEntity\n",
      "Disease - {'actively_involved_in'} -> ChemicalEntity\n",
      "Disease - {'response_affected_by'} -> ChemicalEntity\n",
      "Disease - {'coexpressed_with'} -> ChemicalEntity\n",
      "Disease - {'similar_to'} -> ChemicalEntity\n",
      "Disease - {'catalyzes'} -> ChemicalEntity\n",
      "Disease - {'subclass_of'} -> ChemicalEntity\n",
      "Disease - {'is_output_of'} -> ChemicalEntity\n",
      "Disease - {'overlaps'} -> ChemicalEntity\n",
      "Disease - {'biomarker_for'} -> ChemicalEntity\n",
      "Disease - {'affects_response_to'} -> ChemicalEntity\n",
      "Disease - {'entity_positively_regulates_entity'} -> ChemicalEntity\n",
      "Disease - {'has_phenotype'} -> ChemicalEntity\n",
      "Disease - {'chemically_similar_to'} -> ChemicalEntity\n",
      "Disease - {'in_taxon'} -> ChemicalEntity\n",
      "Disease - {'located_in'} -> ChemicalEntity\n",
      "Disease - {'physically_interacts_with'} -> ChemicalEntity\n",
      "Disease - {'derives_from'} -> ChemicalEntity\n",
      "Disease - {'disease_has_location'} -> ChemicalEntity\n",
      "Disease - {'precedes'} -> ChemicalEntity\n",
      "Disease - {'directly_physically_interacts_with'} -> ChemicalEntity\n",
      "Disease - {'gene_associated_with_condition'} -> ChemicalEntity\n",
      "Disease - {'predisposes'} -> ChemicalEntity\n",
      "Disease - {'prevents'} -> ChemicalEntity\n",
      "Disease - {'colocalizes_with'} -> ChemicalEntity\n",
      "Disease - {'disease_has_basis_in'} -> ChemicalEntity\n",
      "Disease - {'diagnoses'} -> ChemicalEntity\n",
      "Disease - {'occurs_in'} -> ChemicalEntity\n",
      "Disease - {'manifestation_of'} -> ChemicalEntity\n",
      "Disease - {'coexists_with'} -> ChemicalEntity\n",
      "Disease - {'close_match'} -> ChemicalEntity\n",
      "Disease - {'is_sequence_variant_of'} -> ChemicalEntity\n",
      "ChemicalEntity - {'correlated_with'} -> Publication\n",
      "ChemicalEntity - {'disrupts'} -> Publication\n",
      "ChemicalEntity - {'associated_with'} -> Publication\n",
      "ChemicalEntity - {'directly_physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "ChemicalEntity - {'regulates'} -> Publication\n",
      "ChemicalEntity - {'located_in'} -> Publication\n",
      "ChemicalEntity - {'predisposes'} -> Publication\n",
      "ChemicalEntity - {'related_to'} -> Publication\n",
      "ChemicalEntity - {'has_part'} -> Publication\n",
      "ChemicalEntity - {'prevents'} -> Publication\n",
      "ChemicalEntity - {'affects'} -> Publication\n",
      "ChemicalEntity - {'precedes'} -> Publication\n",
      "ChemicalEntity - {'occurs_in'} -> Publication\n",
      "ChemicalEntity - {'has_metabolite'} -> Publication\n",
      "ChemicalEntity - {'close_match'} -> Publication\n",
      "ChemicalEntity - {'subclass_of'} -> Publication\n",
      "ChemicalEntity - {'interacts_with'} -> Publication\n",
      "ChemicalEntity - {'treats'} -> Publication\n",
      "ChemicalEntity - {'physically_interacts_with'} -> Publication\n",
      "ChemicalEntity - {'has_input'} -> Publication\n",
      "ChemicalEntity - {'causes'} -> Publication\n",
      "ChemicalEntity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Pathway->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'genetic_association'} -> Pathway\n",
      "Disease - {'occurs_together_in_literature_with'} -> Pathway\n",
      "Disease - {'preceded_by'} -> Pathway\n",
      "Disease - {'superclass_of'} -> Pathway\n",
      "Disease - {'actively_involves'} -> Pathway\n",
      "Disease - {'caused_by'} -> Pathway\n",
      "Disease - {'overlaps'} -> Pathway\n",
      "Disease - {'disrupts'} -> Pathway\n",
      "Disease - {'related_to'} -> Pathway\n",
      "Disease - {'has_participant'} -> Pathway\n",
      "Disease - {'causes'} -> Pathway\n",
      "Disease - {'has_part'} -> Pathway\n",
      "Disease - {'participates_in'} -> Pathway\n",
      "Disease - {'contributes_to'} -> Pathway\n",
      "Disease - {'gene_associated_with_condition'} -> Pathway\n",
      "Disease - {'decreases_response_to'} -> Pathway\n",
      "Disease - {'interacts_with'} -> Pathway\n",
      "Disease - {'ameliorates'} -> Pathway\n",
      "Disease - {'regulates'} -> Pathway\n",
      "Disease - {'part_of'} -> Pathway\n",
      "Disease - {'has_capability'} -> Pathway\n",
      "Disease - {'associated_with'} -> Pathway\n",
      "Disease - {'affects'} -> Pathway\n",
      "Disease - {'affects_response_to'} -> Pathway\n",
      "Disease - {'entity_positively_regulates_entity'} -> Pathway\n",
      "Disease - {'composed_primarily_of'} -> Pathway\n",
      "Disease - {'has_upstream_actor'} -> Pathway\n",
      "Disease - {'exacerbates'} -> Pathway\n",
      "Disease - {'has_active_ingredient'} -> Pathway\n",
      "Disease - {'increases_response_to'} -> Pathway\n",
      "Disease - {'primarily_composed_of'} -> Pathway\n",
      "Disease - {'temporally_related_to'} -> Pathway\n",
      "Disease - {'similar_to'} -> Pathway\n",
      "Disease - {'affected_by'} -> Pathway\n",
      "Disease - {'treats'} -> Pathway\n",
      "Disease - {'condition_associated_with_gene'} -> Pathway\n",
      "Disease - {'located_in'} -> Pathway\n",
      "Disease - {'contraindicated_for'} -> Pathway\n",
      "Disease - {'biomarker_for'} -> Pathway\n",
      "Disease - {'disease_has_location'} -> Pathway\n",
      "Disease - {'precedes'} -> Pathway\n",
      "Disease - {'subclass_of'} -> Pathway\n",
      "Disease - {'correlated_with'} -> Pathway\n",
      "Disease - {'colocalizes_with'} -> Pathway\n",
      "Disease - {'produces'} -> Pathway\n",
      "Disease - {'predisposes'} -> Pathway\n",
      "Disease - {'prevents'} -> Pathway\n",
      "Disease - {'diagnoses'} -> Pathway\n",
      "Disease - {'manifestation_of'} -> Pathway\n",
      "Disease - {'has_input'} -> Pathway\n",
      "Disease - {'occurs_in'} -> Pathway\n",
      "Disease - {'close_match'} -> Pathway\n",
      "Disease - {'coexists_with'} -> Pathway\n",
      "Disease - {'has_phenotype'} -> Pathway\n",
      "Pathway - {'subclass_of'} -> Publication\n",
      "Pathway - {'regulates'} -> Publication\n",
      "Pathway - {'treats'} -> Publication\n",
      "Pathway - {'has_input'} -> Publication\n",
      "Pathway - {'causes'} -> Publication\n",
      "Pathway - {'associated_with'} -> Publication\n",
      "Pathway - {'gene_associated_with_condition'} -> Publication\n",
      "Pathway - {'disrupts'} -> Publication\n",
      "Pathway - {'located_in'} -> Publication\n",
      "Pathway - {'produces'} -> Publication\n",
      "Pathway - {'predisposes'} -> Publication\n",
      "Pathway - {'related_to'} -> Publication\n",
      "Pathway - {'precedes'} -> Publication\n",
      "Pathway - {'occurs_in'} -> Publication\n",
      "Pathway - {'affects'} -> Publication\n",
      "Pathway - {'manifestation_of'} -> Publication\n",
      "Pathway - {'coexists_with'} -> Publication\n",
      "Pathway - {'close_match'} -> Publication\n",
      "Pathway - {'has_participant'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->AnatomicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> AnatomicalEntity\n",
      "Disease - {'superclass_of'} -> AnatomicalEntity\n",
      "Disease - {'contribution_from'} -> AnatomicalEntity\n",
      "Disease - {'has_biomarker'} -> AnatomicalEntity\n",
      "Disease - {'treated_by'} -> AnatomicalEntity\n",
      "Disease - {'is_ameliorated_by'} -> AnatomicalEntity\n",
      "Disease - {'decreases_abundance_of'} -> AnatomicalEntity\n",
      "Disease - {'increases_abundance_of'} -> AnatomicalEntity\n",
      "Disease - {'related_to'} -> AnatomicalEntity\n",
      "Disease - {'caused_by'} -> AnatomicalEntity\n",
      "Disease - {'part_of'} -> AnatomicalEntity\n",
      "Disease - {'causes'} -> AnatomicalEntity\n",
      "Disease - {'located_in'} -> AnatomicalEntity\n",
      "Disease - {'subclass_of'} -> AnatomicalEntity\n",
      "Disease - {'has_part'} -> AnatomicalEntity\n",
      "Disease - {'has_participant'} -> AnatomicalEntity\n",
      "Disease - {'affects'} -> AnatomicalEntity\n",
      "Disease - {'disrupts'} -> AnatomicalEntity\n",
      "Disease - {'preceded_by'} -> AnatomicalEntity\n",
      "Disease - {'composed_primarily_of'} -> AnatomicalEntity\n",
      "Disease - {'gene_associated_with_condition'} -> AnatomicalEntity\n",
      "Disease - {'has_output'} -> AnatomicalEntity\n",
      "Disease - {'correlated_with'} -> AnatomicalEntity\n",
      "Disease - {'affected_by'} -> AnatomicalEntity\n",
      "Disease - {'has_substrate'} -> AnatomicalEntity\n",
      "Disease - {'interacts_with'} -> AnatomicalEntity\n",
      "Disease - {'treats'} -> AnatomicalEntity\n",
      "Disease - {'regulates'} -> AnatomicalEntity\n",
      "Disease - {'is_substrate_of'} -> AnatomicalEntity\n",
      "Disease - {'has_capability'} -> AnatomicalEntity\n",
      "Disease - {'participates_in'} -> AnatomicalEntity\n",
      "Disease - {'contributes_to'} -> AnatomicalEntity\n",
      "Disease - {'has_input'} -> AnatomicalEntity\n",
      "Disease - {'enabled_by'} -> AnatomicalEntity\n",
      "Disease - {'has_catalyst'} -> AnatomicalEntity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> AnatomicalEntity\n",
      "Disease - {'is_input_of'} -> AnatomicalEntity\n",
      "Disease - {'decreases_response_to'} -> AnatomicalEntity\n",
      "Disease - {'associated_with'} -> AnatomicalEntity\n",
      "Disease - {'enables'} -> AnatomicalEntity\n",
      "Disease - {'ameliorates'} -> AnatomicalEntity\n",
      "Disease - {'genetic_association'} -> AnatomicalEntity\n",
      "Disease - {'capable_of'} -> AnatomicalEntity\n",
      "Disease - {'actively_involves'} -> AnatomicalEntity\n",
      "Disease - {'has_upstream_actor'} -> AnatomicalEntity\n",
      "Disease - {'exacerbates'} -> AnatomicalEntity\n",
      "Disease - {'has_active_ingredient'} -> AnatomicalEntity\n",
      "Disease - {'increases_response_to'} -> AnatomicalEntity\n",
      "Disease - {'primarily_composed_of'} -> AnatomicalEntity\n",
      "Disease - {'expressed_in'} -> AnatomicalEntity\n",
      "Disease - {'temporally_related_to'} -> AnatomicalEntity\n",
      "Disease - {'actively_involved_in'} -> AnatomicalEntity\n",
      "Disease - {'similar_to'} -> AnatomicalEntity\n",
      "Disease - {'catalyzes'} -> AnatomicalEntity\n",
      "Disease - {'is_output_of'} -> AnatomicalEntity\n",
      "Disease - {'overlaps'} -> AnatomicalEntity\n",
      "Disease - {'biomarker_for'} -> AnatomicalEntity\n",
      "Disease - {'affects_response_to'} -> AnatomicalEntity\n",
      "Disease - {'condition_associated_with_gene'} -> AnatomicalEntity\n",
      "Disease - {'entity_positively_regulates_entity'} -> AnatomicalEntity\n",
      "Disease - {'directly_physically_interacts_with'} -> AnatomicalEntity\n",
      "Disease - {'develops_from'} -> AnatomicalEntity\n",
      "Disease - {'colocalizes_with'} -> AnatomicalEntity\n",
      "Disease - {'produces'} -> AnatomicalEntity\n",
      "Disease - {'predisposes'} -> AnatomicalEntity\n",
      "Disease - {'prevents'} -> AnatomicalEntity\n",
      "Disease - {'diagnoses'} -> AnatomicalEntity\n",
      "Disease - {'disease_has_basis_in'} -> AnatomicalEntity\n",
      "Disease - {'manifestation_of'} -> AnatomicalEntity\n",
      "Disease - {'occurs_in'} -> AnatomicalEntity\n",
      "Disease - {'close_match'} -> AnatomicalEntity\n",
      "Disease - {'coexists_with'} -> AnatomicalEntity\n",
      "Disease - {'has_phenotype'} -> AnatomicalEntity\n",
      "Disease - {'derives_from'} -> AnatomicalEntity\n",
      "Disease - {'disease_has_location'} -> AnatomicalEntity\n",
      "Disease - {'precedes'} -> AnatomicalEntity\n",
      "AnatomicalEntity - {'develops_from'} -> Publication\n",
      "AnatomicalEntity - {'interacts_with'} -> Publication\n",
      "AnatomicalEntity - {'produces'} -> Publication\n",
      "AnatomicalEntity - {'has_part'} -> Publication\n",
      "AnatomicalEntity - {'diagnoses'} -> Publication\n",
      "AnatomicalEntity - {'related_to'} -> Publication\n",
      "AnatomicalEntity - {'affects'} -> Publication\n",
      "AnatomicalEntity - {'manifestation_of'} -> Publication\n",
      "AnatomicalEntity - {'has_input'} -> Publication\n",
      "AnatomicalEntity - {'occurs_in'} -> Publication\n",
      "AnatomicalEntity - {'coexists_with'} -> Publication\n",
      "AnatomicalEntity - {'close_match'} -> Publication\n",
      "AnatomicalEntity - {'disrupts'} -> Publication\n",
      "AnatomicalEntity - {'located_in'} -> Publication\n",
      "AnatomicalEntity - {'regulates'} -> Publication\n",
      "AnatomicalEntity - {'treats'} -> Publication\n",
      "AnatomicalEntity - {'causes'} -> Publication\n",
      "AnatomicalEntity - {'associated_with'} -> Publication\n",
      "AnatomicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "AnatomicalEntity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->CellularComponent->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> CellularComponent\n",
      "Disease - {'disrupts'} -> CellularComponent\n",
      "Disease - {'located_in'} -> CellularComponent\n",
      "Disease - {'has_part'} -> CellularComponent\n",
      "Disease - {'decreases_abundance_of'} -> CellularComponent\n",
      "Disease - {'related_to'} -> CellularComponent\n",
      "Disease - {'has_participant'} -> CellularComponent\n",
      "Disease - {'caused_by'} -> CellularComponent\n",
      "Disease - {'increases_abundance_of'} -> CellularComponent\n",
      "Disease - {'causes'} -> CellularComponent\n",
      "Disease - {'has_capability'} -> CellularComponent\n",
      "Disease - {'affects'} -> CellularComponent\n",
      "Disease - {'positively_correlated_with'} -> CellularComponent\n",
      "Disease - {'capable_of'} -> CellularComponent\n",
      "Disease - {'actively_involves'} -> CellularComponent\n",
      "Disease - {'has_active_ingredient'} -> CellularComponent\n",
      "Disease - {'has_catalyst'} -> CellularComponent\n",
      "Disease - {'primarily_composed_of'} -> CellularComponent\n",
      "Disease - {'expressed_in'} -> CellularComponent\n",
      "Disease - {'temporally_related_to'} -> CellularComponent\n",
      "Disease - {'actively_involved_in'} -> CellularComponent\n",
      "Disease - {'response_affected_by'} -> CellularComponent\n",
      "Disease - {'associated_with'} -> CellularComponent\n",
      "Disease - {'similar_to'} -> CellularComponent\n",
      "Disease - {'catalyzes'} -> CellularComponent\n",
      "Disease - {'subclass_of'} -> CellularComponent\n",
      "Disease - {'is_output_of'} -> CellularComponent\n",
      "Disease - {'overlaps'} -> CellularComponent\n",
      "Disease - {'biomarker_for'} -> CellularComponent\n",
      "Disease - {'affects_response_to'} -> CellularComponent\n",
      "Disease - {'genetic_association'} -> CellularComponent\n",
      "Disease - {'entity_positively_regulates_entity'} -> CellularComponent\n",
      "Disease - {'disrupted_by'} -> CellularComponent\n",
      "Disease - {'composed_primarily_of'} -> CellularComponent\n",
      "Disease - {'has_upstream_actor'} -> CellularComponent\n",
      "Disease - {'treated_by'} -> CellularComponent\n",
      "Disease - {'gene_associated_with_condition'} -> CellularComponent\n",
      "Disease - {'exacerbates'} -> CellularComponent\n",
      "Disease - {'increases_response_to'} -> CellularComponent\n",
      "Disease - {'regulated_by'} -> CellularComponent\n",
      "Disease - {'has_output'} -> CellularComponent\n",
      "Disease - {'response_decreased_by'} -> CellularComponent\n",
      "Disease - {'correlated_with'} -> CellularComponent\n",
      "Disease - {'affected_by'} -> CellularComponent\n",
      "Disease - {'interacts_with'} -> CellularComponent\n",
      "Disease - {'treats'} -> CellularComponent\n",
      "Disease - {'condition_associated_with_gene'} -> CellularComponent\n",
      "Disease - {'is_ameliorated_by'} -> CellularComponent\n",
      "Disease - {'regulates'} -> CellularComponent\n",
      "Disease - {'response_increased_by'} -> CellularComponent\n",
      "Disease - {'is_substrate_of'} -> CellularComponent\n",
      "Disease - {'part_of'} -> CellularComponent\n",
      "Disease - {'participates_in'} -> CellularComponent\n",
      "Disease - {'contributes_to'} -> CellularComponent\n",
      "Disease - {'has_input'} -> CellularComponent\n",
      "Disease - {'preceded_by'} -> CellularComponent\n",
      "Disease - {'enabled_by'} -> CellularComponent\n",
      "Disease - {'entity_negatively_regulates_entity'} -> CellularComponent\n",
      "Disease - {'superclass_of'} -> CellularComponent\n",
      "Disease - {'is_input_of'} -> CellularComponent\n",
      "Disease - {'decreases_response_to'} -> CellularComponent\n",
      "Disease - {'enables'} -> CellularComponent\n",
      "Disease - {'is_exacerbated_by'} -> CellularComponent\n",
      "Disease - {'has_substrate'} -> CellularComponent\n",
      "Disease - {'ameliorates'} -> CellularComponent\n",
      "Disease - {'coexists_with'} -> CellularComponent\n",
      "Disease - {'close_match'} -> CellularComponent\n",
      "Disease - {'disease_has_location'} -> CellularComponent\n",
      "Disease - {'precedes'} -> CellularComponent\n",
      "Disease - {'produces'} -> CellularComponent\n",
      "Disease - {'predisposes'} -> CellularComponent\n",
      "Disease - {'prevents'} -> CellularComponent\n",
      "Disease - {'colocalizes_with'} -> CellularComponent\n",
      "Disease - {'diagnoses'} -> CellularComponent\n",
      "Disease - {'occurs_in'} -> CellularComponent\n",
      "Disease - {'manifestation_of'} -> CellularComponent\n",
      "CellularComponent - {'subclass_of'} -> Publication\n",
      "CellularComponent - {'produces'} -> Publication\n",
      "CellularComponent - {'has_part'} -> Publication\n",
      "CellularComponent - {'related_to'} -> Publication\n",
      "CellularComponent - {'occurs_in'} -> Publication\n",
      "CellularComponent - {'affects'} -> Publication\n",
      "CellularComponent - {'regulates'} -> Publication\n",
      "CellularComponent - {'disrupts'} -> Publication\n",
      "CellularComponent - {'located_in'} -> Publication\n",
      "CellularComponent - {'associated_with'} -> Publication\n",
      "CellularComponent - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ChemicalMixture->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> ChemicalMixture\n",
      "Disease - {'causes'} -> ChemicalMixture\n",
      "Disease - {'caused_by'} -> ChemicalMixture\n",
      "Disease - {'has_real_world_evidence_of_association_with'} -> ChemicalMixture\n",
      "Disease - {'affects'} -> ChemicalMixture\n",
      "Disease - {'has_input'} -> ChemicalMixture\n",
      "Disease - {'enabled_by'} -> ChemicalMixture\n",
      "Disease - {'has_participant'} -> ChemicalMixture\n",
      "Disease - {'has_catalyst'} -> ChemicalMixture\n",
      "Disease - {'is_input_of'} -> ChemicalMixture\n",
      "Disease - {'decreases_response_to'} -> ChemicalMixture\n",
      "Disease - {'associated_with'} -> ChemicalMixture\n",
      "Disease - {'enables'} -> ChemicalMixture\n",
      "Disease - {'has_part'} -> ChemicalMixture\n",
      "Disease - {'ameliorates'} -> ChemicalMixture\n",
      "Disease - {'capable_of'} -> ChemicalMixture\n",
      "Disease - {'actively_involves'} -> ChemicalMixture\n",
      "Disease - {'has_upstream_actor'} -> ChemicalMixture\n",
      "Disease - {'exacerbates'} -> ChemicalMixture\n",
      "Disease - {'has_active_ingredient'} -> ChemicalMixture\n",
      "Disease - {'primarily_composed_of'} -> ChemicalMixture\n",
      "Disease - {'temporally_related_to'} -> ChemicalMixture\n",
      "Disease - {'actively_involved_in'} -> ChemicalMixture\n",
      "Disease - {'similar_to'} -> ChemicalMixture\n",
      "Disease - {'catalyzes'} -> ChemicalMixture\n",
      "Disease - {'related_to'} -> ChemicalMixture\n",
      "Disease - {'is_output_of'} -> ChemicalMixture\n",
      "Disease - {'overlaps'} -> ChemicalMixture\n",
      "Disease - {'affects_response_to'} -> ChemicalMixture\n",
      "Disease - {'composed_primarily_of'} -> ChemicalMixture\n",
      "Disease - {'increases_response_to'} -> ChemicalMixture\n",
      "Disease - {'preceded_by'} -> ChemicalMixture\n",
      "Disease - {'has_output'} -> ChemicalMixture\n",
      "Disease - {'disrupts'} -> ChemicalMixture\n",
      "Disease - {'affected_by'} -> ChemicalMixture\n",
      "Disease - {'has_substrate'} -> ChemicalMixture\n",
      "Disease - {'interacts_with'} -> ChemicalMixture\n",
      "Disease - {'treats'} -> ChemicalMixture\n",
      "Disease - {'regulates'} -> ChemicalMixture\n",
      "Disease - {'is_substrate_of'} -> ChemicalMixture\n",
      "Disease - {'has_capability'} -> ChemicalMixture\n",
      "Disease - {'participates_in'} -> ChemicalMixture\n",
      "Disease - {'contributes_to'} -> ChemicalMixture\n",
      "Disease - {'subclass_of'} -> ChemicalMixture\n",
      "Disease - {'produces'} -> ChemicalMixture\n",
      "Disease - {'coexists_with'} -> ChemicalMixture\n",
      "ChemicalMixture - {'related_to'} -> Publication\n",
      "ChemicalMixture - {'affects'} -> Publication\n",
      "ChemicalMixture - {'subclass_of'} -> Publication\n",
      "ChemicalMixture - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->GrossAnatomicalStructure->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> GrossAnatomicalStructure\n",
      "Disease - {'part_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'located_in'} -> GrossAnatomicalStructure\n",
      "Disease - {'caused_by'} -> GrossAnatomicalStructure\n",
      "Disease - {'superclass_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'disrupts'} -> GrossAnatomicalStructure\n",
      "Disease - {'related_to'} -> GrossAnatomicalStructure\n",
      "Disease - {'increases_abundance_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'causes'} -> GrossAnatomicalStructure\n",
      "Disease - {'subclass_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_part'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_participant'} -> GrossAnatomicalStructure\n",
      "Disease - {'affects'} -> GrossAnatomicalStructure\n",
      "Disease - {'affected_by'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_substrate'} -> GrossAnatomicalStructure\n",
      "Disease - {'interacts_with'} -> GrossAnatomicalStructure\n",
      "Disease - {'treats'} -> GrossAnatomicalStructure\n",
      "Disease - {'regulates'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_capability'} -> GrossAnatomicalStructure\n",
      "Disease - {'participates_in'} -> GrossAnatomicalStructure\n",
      "Disease - {'contributes_to'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_input'} -> GrossAnatomicalStructure\n",
      "Disease - {'enabled_by'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_catalyst'} -> GrossAnatomicalStructure\n",
      "Disease - {'decreases_response_to'} -> GrossAnatomicalStructure\n",
      "Disease - {'associated_with'} -> GrossAnatomicalStructure\n",
      "Disease - {'ameliorates'} -> GrossAnatomicalStructure\n",
      "Disease - {'actively_involves'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_upstream_actor'} -> GrossAnatomicalStructure\n",
      "Disease - {'exacerbates'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_active_ingredient'} -> GrossAnatomicalStructure\n",
      "Disease - {'increases_response_to'} -> GrossAnatomicalStructure\n",
      "Disease - {'primarily_composed_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'expressed_in'} -> GrossAnatomicalStructure\n",
      "Disease - {'temporally_related_to'} -> GrossAnatomicalStructure\n",
      "Disease - {'similar_to'} -> GrossAnatomicalStructure\n",
      "Disease - {'is_output_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'overlaps'} -> GrossAnatomicalStructure\n",
      "Disease - {'affects_response_to'} -> GrossAnatomicalStructure\n",
      "Disease - {'entity_positively_regulates_entity'} -> GrossAnatomicalStructure\n",
      "Disease - {'composed_primarily_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'preceded_by'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_output'} -> GrossAnatomicalStructure\n",
      "Disease - {'diagnoses'} -> GrossAnatomicalStructure\n",
      "Disease - {'occurs_in'} -> GrossAnatomicalStructure\n",
      "Disease - {'manifestation_of'} -> GrossAnatomicalStructure\n",
      "Disease - {'coexists_with'} -> GrossAnatomicalStructure\n",
      "Disease - {'close_match'} -> GrossAnatomicalStructure\n",
      "Disease - {'has_phenotype'} -> GrossAnatomicalStructure\n",
      "Disease - {'biomarker_for'} -> GrossAnatomicalStructure\n",
      "Disease - {'derives_from'} -> GrossAnatomicalStructure\n",
      "Disease - {'disease_has_location'} -> GrossAnatomicalStructure\n",
      "Disease - {'actively_involved_in'} -> GrossAnatomicalStructure\n",
      "Disease - {'precedes'} -> GrossAnatomicalStructure\n",
      "Disease - {'develops_from'} -> GrossAnatomicalStructure\n",
      "Disease - {'gene_associated_with_condition'} -> GrossAnatomicalStructure\n",
      "Disease - {'produces'} -> GrossAnatomicalStructure\n",
      "Disease - {'predisposes'} -> GrossAnatomicalStructure\n",
      "Disease - {'prevents'} -> GrossAnatomicalStructure\n",
      "Disease - {'disease_has_basis_in'} -> GrossAnatomicalStructure\n",
      "GrossAnatomicalStructure - {'affects'} -> Publication\n",
      "GrossAnatomicalStructure - {'coexists_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'close_match'} -> Publication\n",
      "GrossAnatomicalStructure - {'develops_from'} -> Publication\n",
      "GrossAnatomicalStructure - {'regulates'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_input'} -> Publication\n",
      "GrossAnatomicalStructure - {'associated_with'} -> Publication\n",
      "GrossAnatomicalStructure - {'located_in'} -> Publication\n",
      "GrossAnatomicalStructure - {'subclass_of'} -> Publication\n",
      "GrossAnatomicalStructure - {'has_part'} -> Publication\n",
      "GrossAnatomicalStructure - {'produces'} -> Publication\n",
      "GrossAnatomicalStructure - {'related_to'} -> Publication\n",
      "GrossAnatomicalStructure - {'occurs_in'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->MolecularActivity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> MolecularActivity\n",
      "Disease - {'caused_by'} -> MolecularActivity\n",
      "Disease - {'overlaps'} -> MolecularActivity\n",
      "Disease - {'associated_with'} -> MolecularActivity\n",
      "Disease - {'disrupts'} -> MolecularActivity\n",
      "Disease - {'related_to'} -> MolecularActivity\n",
      "Disease - {'has_participant'} -> MolecularActivity\n",
      "Disease - {'has_part'} -> MolecularActivity\n",
      "Disease - {'coexists_with'} -> MolecularActivity\n",
      "Disease - {'causes'} -> MolecularActivity\n",
      "Disease - {'affects'} -> MolecularActivity\n",
      "Disease - {'correlated_with'} -> MolecularActivity\n",
      "Disease - {'is_output_of'} -> MolecularActivity\n",
      "Disease - {'affected_by'} -> MolecularActivity\n",
      "Disease - {'treats'} -> MolecularActivity\n",
      "Disease - {'is_ameliorated_by'} -> MolecularActivity\n",
      "Disease - {'response_increased_by'} -> MolecularActivity\n",
      "Disease - {'sensitivity_associated_with'} -> MolecularActivity\n",
      "Disease - {'part_of'} -> MolecularActivity\n",
      "Disease - {'has_biomarker'} -> MolecularActivity\n",
      "Disease - {'participates_in'} -> MolecularActivity\n",
      "Disease - {'decreased_likelihood_associated_with'} -> MolecularActivity\n",
      "Disease - {'contributes_to'} -> MolecularActivity\n",
      "Disease - {'has_input'} -> MolecularActivity\n",
      "Disease - {'preceded_by'} -> MolecularActivity\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> MolecularActivity\n",
      "Disease - {'enabled_by'} -> MolecularActivity\n",
      "Disease - {'is_input_of'} -> MolecularActivity\n",
      "Disease - {'decreases_response_to'} -> MolecularActivity\n",
      "Disease - {'enables'} -> MolecularActivity\n",
      "Disease - {'resistance_associated_with'} -> MolecularActivity\n",
      "Disease - {'is_exacerbated_by'} -> MolecularActivity\n",
      "Disease - {'has_substrate'} -> MolecularActivity\n",
      "Disease - {'interacts_with'} -> MolecularActivity\n",
      "Disease - {'ameliorates'} -> MolecularActivity\n",
      "Disease - {'regulates'} -> MolecularActivity\n",
      "Disease - {'is_substrate_of'} -> MolecularActivity\n",
      "Disease - {'positively_correlated_with'} -> MolecularActivity\n",
      "Disease - {'has_capability'} -> MolecularActivity\n",
      "Disease - {'capable_of'} -> MolecularActivity\n",
      "Disease - {'increased_likelihood_associated_with'} -> MolecularActivity\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> MolecularActivity\n",
      "Disease - {'actively_involves'} -> MolecularActivity\n",
      "Disease - {'has_catalyst'} -> MolecularActivity\n",
      "Disease - {'temporally_related_to'} -> MolecularActivity\n",
      "Disease - {'actively_involved_in'} -> MolecularActivity\n",
      "Disease - {'response_affected_by'} -> MolecularActivity\n",
      "Disease - {'coexpressed_with'} -> MolecularActivity\n",
      "Disease - {'negatively_correlated_with'} -> MolecularActivity\n",
      "Disease - {'similar_to'} -> MolecularActivity\n",
      "Disease - {'catalyzes'} -> MolecularActivity\n",
      "Disease - {'biomarker_for'} -> MolecularActivity\n",
      "Disease - {'affects_response_to'} -> MolecularActivity\n",
      "Disease - {'entity_positively_regulates_entity'} -> MolecularActivity\n",
      "Disease - {'disrupted_by'} -> MolecularActivity\n",
      "Disease - {'composed_primarily_of'} -> MolecularActivity\n",
      "Disease - {'has_upstream_actor'} -> MolecularActivity\n",
      "Disease - {'treated_by'} -> MolecularActivity\n",
      "Disease - {'exacerbates'} -> MolecularActivity\n",
      "Disease - {'has_active_ingredient'} -> MolecularActivity\n",
      "Disease - {'associated_with_sensitivity_to'} -> MolecularActivity\n",
      "Disease - {'increases_response_to'} -> MolecularActivity\n",
      "Disease - {'primarily_composed_of'} -> MolecularActivity\n",
      "Disease - {'regulated_by'} -> MolecularActivity\n",
      "Disease - {'associated_with_resistance_to'} -> MolecularActivity\n",
      "Disease - {'has_output'} -> MolecularActivity\n",
      "Disease - {'response_decreased_by'} -> MolecularActivity\n",
      "Disease - {'precedes'} -> MolecularActivity\n",
      "Disease - {'subclass_of'} -> MolecularActivity\n",
      "Disease - {'produces'} -> MolecularActivity\n",
      "Disease - {'occurs_in'} -> MolecularActivity\n",
      "Disease - {'manifestation_of'} -> MolecularActivity\n",
      "Disease - {'close_match'} -> MolecularActivity\n",
      "Disease - {'located_in'} -> MolecularActivity\n",
      "MolecularActivity - {'occurs_in'} -> Publication\n",
      "MolecularActivity - {'treats'} -> Publication\n",
      "MolecularActivity - {'has_input'} -> Publication\n",
      "MolecularActivity - {'has_output'} -> Publication\n",
      "MolecularActivity - {'coexists_with'} -> Publication\n",
      "MolecularActivity - {'regulates'} -> Publication\n",
      "MolecularActivity - {'disrupts'} -> Publication\n",
      "MolecularActivity - {'associated_with'} -> Publication\n",
      "MolecularActivity - {'gene_associated_with_condition'} -> Publication\n",
      "MolecularActivity - {'located_in'} -> Publication\n",
      "MolecularActivity - {'produces'} -> Publication\n",
      "MolecularActivity - {'related_to'} -> Publication\n",
      "MolecularActivity - {'affects'} -> Publication\n",
      "MolecularActivity - {'has_participant'} -> Publication\n",
      "MolecularActivity - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->MolecularMixture->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> MolecularMixture\n",
      "Disease - {'treated_by'} -> MolecularMixture\n",
      "Disease - {'has_contraindication'} -> MolecularMixture\n",
      "Disease - {'affected_by'} -> MolecularMixture\n",
      "Disease - {'contribution_from'} -> MolecularMixture\n",
      "Disease - {'is_ameliorated_by'} -> MolecularMixture\n",
      "Disease - {'caused_by'} -> MolecularMixture\n",
      "Disease - {'related_to'} -> MolecularMixture\n",
      "Disease - {'has_biomarker'} -> MolecularMixture\n",
      "Disease - {'causes'} -> MolecularMixture\n",
      "Disease - {'positively_correlated_with'} -> MolecularMixture\n",
      "Disease - {'negatively_correlated_with'} -> MolecularMixture\n",
      "Disease - {'affects'} -> MolecularMixture\n",
      "Disease - {'has_real_world_evidence_of_association_with'} -> MolecularMixture\n",
      "Disease - {'correlated_with'} -> MolecularMixture\n",
      "Disease - {'composed_primarily_of'} -> MolecularMixture\n",
      "Disease - {'gene_associated_with_condition'} -> MolecularMixture\n",
      "Disease - {'preceded_by'} -> MolecularMixture\n",
      "Disease - {'has_output'} -> MolecularMixture\n",
      "Disease - {'response_decreased_by'} -> MolecularMixture\n",
      "Disease - {'disrupts'} -> MolecularMixture\n",
      "Disease - {'has_substrate'} -> MolecularMixture\n",
      "Disease - {'interacts_with'} -> MolecularMixture\n",
      "Disease - {'ameliorates'} -> MolecularMixture\n",
      "Disease - {'treats'} -> MolecularMixture\n",
      "Disease - {'regulates'} -> MolecularMixture\n",
      "Disease - {'response_increased_by'} -> MolecularMixture\n",
      "Disease - {'is_substrate_of'} -> MolecularMixture\n",
      "Disease - {'part_of'} -> MolecularMixture\n",
      "Disease - {'has_capability'} -> MolecularMixture\n",
      "Disease - {'participates_in'} -> MolecularMixture\n",
      "Disease - {'contributes_to'} -> MolecularMixture\n",
      "Disease - {'has_input'} -> MolecularMixture\n",
      "Disease - {'enabled_by'} -> MolecularMixture\n",
      "Disease - {'has_participant'} -> MolecularMixture\n",
      "Disease - {'has_catalyst'} -> MolecularMixture\n",
      "Disease - {'is_input_of'} -> MolecularMixture\n",
      "Disease - {'decreases_response_to'} -> MolecularMixture\n",
      "Disease - {'associated_with'} -> MolecularMixture\n",
      "Disease - {'enables'} -> MolecularMixture\n",
      "Disease - {'has_part'} -> MolecularMixture\n",
      "Disease - {'is_exacerbated_by'} -> MolecularMixture\n",
      "Disease - {'catalyzes'} -> MolecularMixture\n",
      "Disease - {'genetic_association'} -> MolecularMixture\n",
      "Disease - {'capable_of'} -> MolecularMixture\n",
      "Disease - {'disrupted_by'} -> MolecularMixture\n",
      "Disease - {'actively_involves'} -> MolecularMixture\n",
      "Disease - {'has_upstream_actor'} -> MolecularMixture\n",
      "Disease - {'exacerbates'} -> MolecularMixture\n",
      "Disease - {'has_active_ingredient'} -> MolecularMixture\n",
      "Disease - {'increases_response_to'} -> MolecularMixture\n",
      "Disease - {'primarily_composed_of'} -> MolecularMixture\n",
      "Disease - {'regulated_by'} -> MolecularMixture\n",
      "Disease - {'temporally_related_to'} -> MolecularMixture\n",
      "Disease - {'actively_involved_in'} -> MolecularMixture\n",
      "Disease - {'response_affected_by'} -> MolecularMixture\n",
      "Disease - {'coexpressed_with'} -> MolecularMixture\n",
      "Disease - {'increases_abundance_of'} -> MolecularMixture\n",
      "Disease - {'similar_to'} -> MolecularMixture\n",
      "Disease - {'subclass_of'} -> MolecularMixture\n",
      "Disease - {'is_output_of'} -> MolecularMixture\n",
      "Disease - {'overlaps'} -> MolecularMixture\n",
      "Disease - {'biomarker_for'} -> MolecularMixture\n",
      "Disease - {'affects_response_to'} -> MolecularMixture\n",
      "Disease - {'condition_associated_with_gene'} -> MolecularMixture\n",
      "Disease - {'produces'} -> MolecularMixture\n",
      "Disease - {'located_in'} -> MolecularMixture\n",
      "Disease - {'occurs_in'} -> MolecularMixture\n",
      "Disease - {'close_match'} -> MolecularMixture\n",
      "Disease - {'coexists_with'} -> MolecularMixture\n",
      "MolecularMixture - {'correlated_with'} -> Publication\n",
      "MolecularMixture - {'treats'} -> Publication\n",
      "MolecularMixture - {'subclass_of'} -> Publication\n",
      "MolecularMixture - {'located_in'} -> Publication\n",
      "MolecularMixture - {'precedes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->NamedThing->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> NamedThing\n",
      "Disease - {'subclass_of'} -> NamedThing\n",
      "Disease - {'related_to'} -> NamedThing\n",
      "Disease - {'participates_in'} -> NamedThing\n",
      "Disease - {'has_substrate'} -> NamedThing\n",
      "Disease - {'interacts_with'} -> NamedThing\n",
      "Disease - {'ameliorates'} -> NamedThing\n",
      "Disease - {'regulates'} -> NamedThing\n",
      "Disease - {'is_substrate_of'} -> NamedThing\n",
      "Disease - {'part_of'} -> NamedThing\n",
      "Disease - {'has_capability'} -> NamedThing\n",
      "Disease - {'capable_of'} -> NamedThing\n",
      "Disease - {'caused_by'} -> NamedThing\n",
      "Disease - {'actively_involves'} -> NamedThing\n",
      "Disease - {'has_catalyst'} -> NamedThing\n",
      "Disease - {'expressed_in'} -> NamedThing\n",
      "Disease - {'associated_with'} -> NamedThing\n",
      "Disease - {'enables'} -> NamedThing\n",
      "Disease - {'affects'} -> NamedThing\n",
      "Disease - {'catalyzes'} -> NamedThing\n",
      "Disease - {'overlaps'} -> NamedThing\n",
      "Disease - {'affects_response_to'} -> NamedThing\n",
      "Disease - {'entity_positively_regulates_entity'} -> NamedThing\n",
      "Disease - {'located_in'} -> NamedThing\n",
      "Disease - {'composed_primarily_of'} -> NamedThing\n",
      "Disease - {'has_upstream_actor'} -> NamedThing\n",
      "Disease - {'exacerbates'} -> NamedThing\n",
      "Disease - {'has_active_ingredient'} -> NamedThing\n",
      "Disease - {'increases_response_to'} -> NamedThing\n",
      "Disease - {'primarily_composed_of'} -> NamedThing\n",
      "Disease - {'temporally_related_to'} -> NamedThing\n",
      "Disease - {'actively_involved_in'} -> NamedThing\n",
      "Disease - {'has_output'} -> NamedThing\n",
      "Disease - {'disrupts'} -> NamedThing\n",
      "Disease - {'similar_to'} -> NamedThing\n",
      "Disease - {'is_output_of'} -> NamedThing\n",
      "Disease - {'affected_by'} -> NamedThing\n",
      "Disease - {'treats'} -> NamedThing\n",
      "Disease - {'causes'} -> NamedThing\n",
      "Disease - {'contributes_to'} -> NamedThing\n",
      "Disease - {'has_input'} -> NamedThing\n",
      "Disease - {'preceded_by'} -> NamedThing\n",
      "Disease - {'enabled_by'} -> NamedThing\n",
      "Disease - {'has_participant'} -> NamedThing\n",
      "Disease - {'entity_negatively_regulates_entity'} -> NamedThing\n",
      "Disease - {'is_input_of'} -> NamedThing\n",
      "Disease - {'decreases_response_to'} -> NamedThing\n",
      "Disease - {'has_part'} -> NamedThing\n",
      "Disease - {'physically_interacts_with'} -> NamedThing\n",
      "Disease - {'disease_has_location'} -> NamedThing\n",
      "Disease - {'directly_physically_interacts_with'} -> NamedThing\n",
      "Disease - {'precedes'} -> NamedThing\n",
      "Disease - {'is_sequence_variant_of'} -> NamedThing\n",
      "Disease - {'gene_associated_with_condition'} -> NamedThing\n",
      "Disease - {'correlated_with'} -> NamedThing\n",
      "Disease - {'colocalizes_with'} -> NamedThing\n",
      "Disease - {'produces'} -> NamedThing\n",
      "Disease - {'predisposes'} -> NamedThing\n",
      "Disease - {'prevents'} -> NamedThing\n",
      "Disease - {'diagnoses'} -> NamedThing\n",
      "Disease - {'occurs_in'} -> NamedThing\n",
      "Disease - {'manifestation_of'} -> NamedThing\n",
      "Disease - {'close_match'} -> NamedThing\n",
      "Disease - {'coexists_with'} -> NamedThing\n",
      "Disease - {'has_phenotype'} -> NamedThing\n",
      "Disease - {'contraindicated_for'} -> NamedThing\n",
      "Disease - {'biomarker_for'} -> NamedThing\n",
      "Disease - {'derives_from'} -> NamedThing\n",
      "Disease - {'in_taxon'} -> NamedThing\n",
      "NamedThing - {'subclass_of'} -> Publication\n",
      "NamedThing - {'related_to'} -> Publication\n",
      "NamedThing - {'is_sequence_variant_of'} -> Publication\n",
      "NamedThing - {'associated_with'} -> Publication\n",
      "NamedThing - {'has_phenotype'} -> Publication\n",
      "NamedThing - {'disrupts'} -> Publication\n",
      "NamedThing - {'biomarker_for'} -> Publication\n",
      "NamedThing - {'correlated_with'} -> Publication\n",
      "NamedThing - {'gene_associated_with_condition'} -> Publication\n",
      "NamedThing - {'has_part'} -> Publication\n",
      "NamedThing - {'located_in'} -> Publication\n",
      "NamedThing - {'colocalizes_with'} -> Publication\n",
      "NamedThing - {'interacts_with'} -> Publication\n",
      "NamedThing - {'produces'} -> Publication\n",
      "NamedThing - {'prevents'} -> Publication\n",
      "NamedThing - {'diagnoses'} -> Publication\n",
      "NamedThing - {'predisposes'} -> Publication\n",
      "NamedThing - {'precedes'} -> Publication\n",
      "NamedThing - {'occurs_in'} -> Publication\n",
      "NamedThing - {'affects'} -> Publication\n",
      "NamedThing - {'close_match'} -> Publication\n",
      "NamedThing - {'manifestation_of'} -> Publication\n",
      "NamedThing - {'exacerbates'} -> Publication\n",
      "NamedThing - {'coexists_with'} -> Publication\n",
      "NamedThing - {'regulates'} -> Publication\n",
      "NamedThing - {'has_input'} -> Publication\n",
      "NamedThing - {'physically_interacts_with'} -> Publication\n",
      "NamedThing - {'treats'} -> Publication\n",
      "NamedThing - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->OrganismTaxon->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> OrganismTaxon\n",
      "Disease - {'treated_by'} -> OrganismTaxon\n",
      "Disease - {'subclass_of'} -> OrganismTaxon\n",
      "Disease - {'has_biomarker'} -> OrganismTaxon\n",
      "Disease - {'contribution_from'} -> OrganismTaxon\n",
      "Disease - {'affected_by'} -> OrganismTaxon\n",
      "Disease - {'positively_correlated_with'} -> OrganismTaxon\n",
      "Disease - {'negatively_correlated_with'} -> OrganismTaxon\n",
      "Disease - {'is_ameliorated_by'} -> OrganismTaxon\n",
      "Disease - {'caused_by'} -> OrganismTaxon\n",
      "Disease - {'causes'} -> OrganismTaxon\n",
      "Disease - {'part_of'} -> OrganismTaxon\n",
      "Disease - {'has_participant'} -> OrganismTaxon\n",
      "Disease - {'located_in'} -> OrganismTaxon\n",
      "Disease - {'related_to'} -> OrganismTaxon\n",
      "Disease - {'in_taxon'} -> OrganismTaxon\n",
      "Disease - {'affects'} -> OrganismTaxon\n",
      "Disease - {'disrupts'} -> OrganismTaxon\n",
      "Disease - {'treats'} -> OrganismTaxon\n",
      "Disease - {'participates_in'} -> OrganismTaxon\n",
      "Disease - {'contributes_to'} -> OrganismTaxon\n",
      "Disease - {'has_input'} -> OrganismTaxon\n",
      "Disease - {'enabled_by'} -> OrganismTaxon\n",
      "Disease - {'decreases_response_to'} -> OrganismTaxon\n",
      "Disease - {'has_part'} -> OrganismTaxon\n",
      "Disease - {'correlated_with'} -> OrganismTaxon\n",
      "Disease - {'is_exacerbated_by'} -> OrganismTaxon\n",
      "Disease - {'has_substrate'} -> OrganismTaxon\n",
      "Disease - {'interacts_with'} -> OrganismTaxon\n",
      "Disease - {'ameliorates'} -> OrganismTaxon\n",
      "Disease - {'regulates'} -> OrganismTaxon\n",
      "Disease - {'response_increased_by'} -> OrganismTaxon\n",
      "Disease - {'has_capability'} -> OrganismTaxon\n",
      "Disease - {'actively_involves'} -> OrganismTaxon\n",
      "Disease - {'has_catalyst'} -> OrganismTaxon\n",
      "Disease - {'expressed_in'} -> OrganismTaxon\n",
      "Disease - {'associated_with'} -> OrganismTaxon\n",
      "Disease - {'overlaps'} -> OrganismTaxon\n",
      "Disease - {'affects_response_to'} -> OrganismTaxon\n",
      "Disease - {'disrupted_by'} -> OrganismTaxon\n",
      "Disease - {'composed_primarily_of'} -> OrganismTaxon\n",
      "Disease - {'has_upstream_actor'} -> OrganismTaxon\n",
      "Disease - {'exacerbates'} -> OrganismTaxon\n",
      "Disease - {'has_active_ingredient'} -> OrganismTaxon\n",
      "Disease - {'increases_response_to'} -> OrganismTaxon\n",
      "Disease - {'primarily_composed_of'} -> OrganismTaxon\n",
      "Disease - {'regulated_by'} -> OrganismTaxon\n",
      "Disease - {'has_output'} -> OrganismTaxon\n",
      "Disease - {'response_affected_by'} -> OrganismTaxon\n",
      "Disease - {'response_decreased_by'} -> OrganismTaxon\n",
      "Disease - {'similar_to'} -> OrganismTaxon\n",
      "Disease - {'has_phenotype'} -> OrganismTaxon\n",
      "Disease - {'biomarker_for'} -> OrganismTaxon\n",
      "Disease - {'derives_from'} -> OrganismTaxon\n",
      "Disease - {'physically_interacts_with'} -> OrganismTaxon\n",
      "Disease - {'disease_has_location'} -> OrganismTaxon\n",
      "Disease - {'precedes'} -> OrganismTaxon\n",
      "Disease - {'gene_associated_with_condition'} -> OrganismTaxon\n",
      "Disease - {'predisposes'} -> OrganismTaxon\n",
      "Disease - {'prevents'} -> OrganismTaxon\n",
      "Disease - {'produces'} -> OrganismTaxon\n",
      "Disease - {'diagnoses'} -> OrganismTaxon\n",
      "Disease - {'occurs_in'} -> OrganismTaxon\n",
      "Disease - {'manifestation_of'} -> OrganismTaxon\n",
      "Disease - {'coexists_with'} -> OrganismTaxon\n",
      "Disease - {'close_match'} -> OrganismTaxon\n",
      "OrganismTaxon - {'regulates'} -> Publication\n",
      "OrganismTaxon - {'gene_associated_with_condition'} -> Publication\n",
      "OrganismTaxon - {'located_in'} -> Publication\n",
      "OrganismTaxon - {'interacts_with'} -> Publication\n",
      "OrganismTaxon - {'produces'} -> Publication\n",
      "OrganismTaxon - {'related_to'} -> Publication\n",
      "OrganismTaxon - {'has_part'} -> Publication\n",
      "OrganismTaxon - {'predisposes'} -> Publication\n",
      "OrganismTaxon - {'occurs_in'} -> Publication\n",
      "OrganismTaxon - {'affects'} -> Publication\n",
      "OrganismTaxon - {'manifestation_of'} -> Publication\n",
      "OrganismTaxon - {'exacerbates'} -> Publication\n",
      "OrganismTaxon - {'coexists_with'} -> Publication\n",
      "OrganismTaxon - {'close_match'} -> Publication\n",
      "OrganismTaxon - {'subclass_of'} -> Publication\n",
      "OrganismTaxon - {'physically_interacts_with'} -> Publication\n",
      "OrganismTaxon - {'treats'} -> Publication\n",
      "OrganismTaxon - {'has_input'} -> Publication\n",
      "OrganismTaxon - {'causes'} -> Publication\n",
      "OrganismTaxon - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Polypeptide->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'occurs_together_in_literature_with'} -> Polypeptide\n",
      "Disease - {'caused_by'} -> Polypeptide\n",
      "Disease - {'produces'} -> Polypeptide\n",
      "Disease - {'is_ameliorated_by'} -> Polypeptide\n",
      "Disease - {'prevented_by'} -> Polypeptide\n",
      "Disease - {'disrupted_by'} -> Polypeptide\n",
      "Disease - {'treated_by'} -> Polypeptide\n",
      "Disease - {'affected_by'} -> Polypeptide\n",
      "Disease - {'has_biomarker'} -> Polypeptide\n",
      "Disease - {'contribution_from'} -> Polypeptide\n",
      "Disease - {'decreases_abundance_of'} -> Polypeptide\n",
      "Disease - {'is_exacerbated_by'} -> Polypeptide\n",
      "Disease - {'entity_positively_regulated_by_entity'} -> Polypeptide\n",
      "Disease - {'has_predisposing_factor'} -> Polypeptide\n",
      "Disease - {'associated_with'} -> Polypeptide\n",
      "Disease - {'increases_abundance_of'} -> Polypeptide\n",
      "Disease - {'has_part'} -> Polypeptide\n",
      "Disease - {'affects'} -> Polypeptide\n",
      "Disease - {'has_participant'} -> Polypeptide\n",
      "Disease - {'entity_negatively_regulates_entity'} -> Polypeptide\n",
      "Disease - {'decreases_response_to'} -> Polypeptide\n",
      "Disease - {'enables'} -> Polypeptide\n",
      "Disease - {'catalyzes'} -> Polypeptide\n",
      "Disease - {'biomarker_for'} -> Polypeptide\n",
      "Disease - {'has_catalyst'} -> Polypeptide\n",
      "Disease - {'entity_positively_regulates_entity'} -> Polypeptide\n",
      "Disease - {'increased_likelihood_associated_with'} -> Polypeptide\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> Polypeptide\n",
      "Disease - {'has_upstream_actor'} -> Polypeptide\n",
      "Disease - {'exacerbates'} -> Polypeptide\n",
      "Disease - {'has_active_ingredient'} -> Polypeptide\n",
      "Disease - {'increases_response_to'} -> Polypeptide\n",
      "Disease - {'primarily_composed_of'} -> Polypeptide\n",
      "Disease - {'regulated_by'} -> Polypeptide\n",
      "Disease - {'temporally_related_to'} -> Polypeptide\n",
      "Disease - {'associated_with_resistance_to'} -> Polypeptide\n",
      "Disease - {'actively_involved_in'} -> Polypeptide\n",
      "Disease - {'has_output'} -> Polypeptide\n",
      "Disease - {'response_affected_by'} -> Polypeptide\n",
      "Disease - {'coexpressed_with'} -> Polypeptide\n",
      "Disease - {'negatively_correlated_with'} -> Polypeptide\n",
      "Disease - {'similar_to'} -> Polypeptide\n",
      "Disease - {'related_to'} -> Polypeptide\n",
      "Disease - {'is_output_of'} -> Polypeptide\n",
      "Disease - {'overlaps'} -> Polypeptide\n",
      "Disease - {'affects_response_to'} -> Polypeptide\n",
      "Disease - {'causes'} -> Polypeptide\n",
      "Disease - {'composed_primarily_of'} -> Polypeptide\n",
      "Disease - {'associated_with_sensitivity_to'} -> Polypeptide\n",
      "Disease - {'preceded_by'} -> Polypeptide\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> Polypeptide\n",
      "Disease - {'response_decreased_by'} -> Polypeptide\n",
      "Disease - {'disrupts'} -> Polypeptide\n",
      "Disease - {'is_input_of'} -> Polypeptide\n",
      "Disease - {'resistance_associated_with'} -> Polypeptide\n",
      "Disease - {'correlated_with'} -> Polypeptide\n",
      "Disease - {'has_substrate'} -> Polypeptide\n",
      "Disease - {'interacts_with'} -> Polypeptide\n",
      "Disease - {'ameliorates'} -> Polypeptide\n",
      "Disease - {'treats'} -> Polypeptide\n",
      "Disease - {'regulates'} -> Polypeptide\n",
      "Disease - {'response_increased_by'} -> Polypeptide\n",
      "Disease - {'sensitivity_associated_with'} -> Polypeptide\n",
      "Disease - {'is_substrate_of'} -> Polypeptide\n",
      "Disease - {'positively_correlated_with'} -> Polypeptide\n",
      "Disease - {'part_of'} -> Polypeptide\n",
      "Disease - {'has_capability'} -> Polypeptide\n",
      "Disease - {'participates_in'} -> Polypeptide\n",
      "Disease - {'decreased_likelihood_associated_with'} -> Polypeptide\n",
      "Disease - {'capable_of'} -> Polypeptide\n",
      "Disease - {'contributes_to'} -> Polypeptide\n",
      "Disease - {'actively_involves'} -> Polypeptide\n",
      "Disease - {'has_input'} -> Polypeptide\n",
      "Disease - {'enabled_by'} -> Polypeptide\n",
      "Disease - {'located_in'} -> Polypeptide\n",
      "Disease - {'physically_interacts_with'} -> Polypeptide\n",
      "Disease - {'derives_from'} -> Polypeptide\n",
      "Disease - {'disease_has_location'} -> Polypeptide\n",
      "Disease - {'precedes'} -> Polypeptide\n",
      "Disease - {'is_sequence_variant_of'} -> Polypeptide\n",
      "Disease - {'directly_physically_interacts_with'} -> Polypeptide\n",
      "Disease - {'subclass_of'} -> Polypeptide\n",
      "Disease - {'predisposes'} -> Polypeptide\n",
      "Disease - {'colocalizes_with'} -> Polypeptide\n",
      "Disease - {'diagnoses'} -> Polypeptide\n",
      "Disease - {'occurs_in'} -> Polypeptide\n",
      "Disease - {'manifestation_of'} -> Polypeptide\n",
      "Disease - {'coexists_with'} -> Polypeptide\n",
      "Disease - {'close_match'} -> Polypeptide\n",
      "Polypeptide - {'disrupts'} -> Publication\n",
      "Polypeptide - {'associated_with'} -> Publication\n",
      "Polypeptide - {'gene_associated_with_condition'} -> Publication\n",
      "Polypeptide - {'located_in'} -> Publication\n",
      "Polypeptide - {'regulates'} -> Publication\n",
      "Polypeptide - {'biomarker_for'} -> Publication\n",
      "Polypeptide - {'produces'} -> Publication\n",
      "Polypeptide - {'has_part'} -> Publication\n",
      "Polypeptide - {'physically_interacts_with'} -> Publication\n",
      "Polypeptide - {'colocalizes_with'} -> Publication\n",
      "Polypeptide - {'predisposes'} -> Publication\n",
      "Polypeptide - {'related_to'} -> Publication\n",
      "Polypeptide - {'prevents'} -> Publication\n",
      "Polypeptide - {'precedes'} -> Publication\n",
      "Polypeptide - {'close_match'} -> Publication\n",
      "Polypeptide - {'subclass_of'} -> Publication\n",
      "Polypeptide - {'interacts_with'} -> Publication\n",
      "Polypeptide - {'affects'} -> Publication\n",
      "Polypeptide - {'treats'} -> Publication\n",
      "Polypeptide - {'occurs_in'} -> Publication\n",
      "Polypeptide - {'exacerbates'} -> Publication\n",
      "Polypeptide - {'causes'} -> Publication\n",
      "Polypeptide - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->BiologicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affects'} -> BiologicalEntity\n",
      "Disease - {'subclass_of'} -> BiologicalEntity\n",
      "Disease - {'related_to'} -> BiologicalEntity\n",
      "Disease - {'has_output'} -> BiologicalEntity\n",
      "Disease - {'has_active_ingredient'} -> BiologicalEntity\n",
      "Disease - {'has_catalyst'} -> BiologicalEntity\n",
      "Disease - {'primarily_composed_of'} -> BiologicalEntity\n",
      "Disease - {'expressed_in'} -> BiologicalEntity\n",
      "Disease - {'temporally_related_to'} -> BiologicalEntity\n",
      "Disease - {'actively_involved_in'} -> BiologicalEntity\n",
      "Disease - {'associated_with'} -> BiologicalEntity\n",
      "Disease - {'similar_to'} -> BiologicalEntity\n",
      "Disease - {'catalyzes'} -> BiologicalEntity\n",
      "Disease - {'overlaps'} -> BiologicalEntity\n",
      "Disease - {'affects_response_to'} -> BiologicalEntity\n",
      "Disease - {'genetic_association'} -> BiologicalEntity\n",
      "Disease - {'entity_positively_regulates_entity'} -> BiologicalEntity\n",
      "Disease - {'located_in'} -> BiologicalEntity\n",
      "Disease - {'composed_primarily_of'} -> BiologicalEntity\n",
      "Disease - {'has_upstream_actor'} -> BiologicalEntity\n",
      "Disease - {'gene_associated_with_condition'} -> BiologicalEntity\n",
      "Disease - {'exacerbates'} -> BiologicalEntity\n",
      "Disease - {'increases_response_to'} -> BiologicalEntity\n",
      "Disease - {'disrupts'} -> BiologicalEntity\n",
      "Disease - {'is_output_of'} -> BiologicalEntity\n",
      "Disease - {'affected_by'} -> BiologicalEntity\n",
      "Disease - {'treats'} -> BiologicalEntity\n",
      "Disease - {'condition_associated_with_gene'} -> BiologicalEntity\n",
      "Disease - {'causes'} -> BiologicalEntity\n",
      "Disease - {'is_substrate_of'} -> BiologicalEntity\n",
      "Disease - {'part_of'} -> BiologicalEntity\n",
      "Disease - {'has_capability'} -> BiologicalEntity\n",
      "Disease - {'participates_in'} -> BiologicalEntity\n",
      "Disease - {'contributes_to'} -> BiologicalEntity\n",
      "Disease - {'has_input'} -> BiologicalEntity\n",
      "Disease - {'preceded_by'} -> BiologicalEntity\n",
      "Disease - {'enabled_by'} -> BiologicalEntity\n",
      "Disease - {'has_participant'} -> BiologicalEntity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> BiologicalEntity\n",
      "Disease - {'is_input_of'} -> BiologicalEntity\n",
      "Disease - {'decreases_response_to'} -> BiologicalEntity\n",
      "Disease - {'enables'} -> BiologicalEntity\n",
      "Disease - {'has_part'} -> BiologicalEntity\n",
      "Disease - {'has_substrate'} -> BiologicalEntity\n",
      "Disease - {'interacts_with'} -> BiologicalEntity\n",
      "Disease - {'ameliorates'} -> BiologicalEntity\n",
      "Disease - {'regulates'} -> BiologicalEntity\n",
      "Disease - {'capable_of'} -> BiologicalEntity\n",
      "Disease - {'caused_by'} -> BiologicalEntity\n",
      "Disease - {'actively_involves'} -> BiologicalEntity\n",
      "Disease - {'derives_from'} -> BiologicalEntity\n",
      "Disease - {'disease_has_location'} -> BiologicalEntity\n",
      "Disease - {'precedes'} -> BiologicalEntity\n",
      "Disease - {'is_sequence_variant_of'} -> BiologicalEntity\n",
      "Disease - {'gene_product_of'} -> BiologicalEntity\n",
      "Disease - {'directly_physically_interacts_with'} -> BiologicalEntity\n",
      "Disease - {'correlated_with'} -> BiologicalEntity\n",
      "Disease - {'colocalizes_with'} -> BiologicalEntity\n",
      "Disease - {'produces'} -> BiologicalEntity\n",
      "Disease - {'predisposes'} -> BiologicalEntity\n",
      "Disease - {'prevents'} -> BiologicalEntity\n",
      "Disease - {'diagnoses'} -> BiologicalEntity\n",
      "Disease - {'disease_has_basis_in'} -> BiologicalEntity\n",
      "Disease - {'manifestation_of'} -> BiologicalEntity\n",
      "Disease - {'occurs_in'} -> BiologicalEntity\n",
      "Disease - {'coexists_with'} -> BiologicalEntity\n",
      "Disease - {'close_match'} -> BiologicalEntity\n",
      "Disease - {'has_phenotype'} -> BiologicalEntity\n",
      "Disease - {'contraindicated_for'} -> BiologicalEntity\n",
      "Disease - {'biomarker_for'} -> BiologicalEntity\n",
      "Disease - {'chemically_similar_to'} -> BiologicalEntity\n",
      "Disease - {'in_taxon'} -> BiologicalEntity\n",
      "Disease - {'physically_interacts_with'} -> BiologicalEntity\n",
      "BiologicalEntity - {'physically_interacts_with'} -> Publication\n",
      "BiologicalEntity - {'treats'} -> Publication\n",
      "BiologicalEntity - {'has_input'} -> Publication\n",
      "BiologicalEntity - {'causes'} -> Publication\n",
      "BiologicalEntity - {'disrupts'} -> Publication\n",
      "BiologicalEntity - {'is_sequence_variant_of'} -> Publication\n",
      "BiologicalEntity - {'has_phenotype'} -> Publication\n",
      "BiologicalEntity - {'biomarker_for'} -> Publication\n",
      "BiologicalEntity - {'associated_with'} -> Publication\n",
      "BiologicalEntity - {'gene_associated_with_condition'} -> Publication\n",
      "BiologicalEntity - {'has_part'} -> Publication\n",
      "BiologicalEntity - {'located_in'} -> Publication\n",
      "BiologicalEntity - {'colocalizes_with'} -> Publication\n",
      "BiologicalEntity - {'interacts_with'} -> Publication\n",
      "BiologicalEntity - {'produces'} -> Publication\n",
      "BiologicalEntity - {'predisposes'} -> Publication\n",
      "BiologicalEntity - {'related_to'} -> Publication\n",
      "BiologicalEntity - {'prevents'} -> Publication\n",
      "BiologicalEntity - {'manifestation_of'} -> Publication\n",
      "BiologicalEntity - {'precedes'} -> Publication\n",
      "BiologicalEntity - {'occurs_in'} -> Publication\n",
      "BiologicalEntity - {'affects'} -> Publication\n",
      "BiologicalEntity - {'close_match'} -> Publication\n",
      "BiologicalEntity - {'exacerbates'} -> Publication\n",
      "BiologicalEntity - {'coexists_with'} -> Publication\n",
      "BiologicalEntity - {'has_participant'} -> Publication\n",
      "BiologicalEntity - {'subclass_of'} -> Publication\n",
      "BiologicalEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Treatment->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affects'} -> Treatment\n",
      "Disease - {'related_to'} -> Treatment\n",
      "Disease - {'affected_by'} -> Treatment\n",
      "Disease - {'interacts_with'} -> Treatment\n",
      "Disease - {'participates_in'} -> Treatment\n",
      "Disease - {'has_participant'} -> Treatment\n",
      "Disease - {'associated_with'} -> Treatment\n",
      "Disease - {'has_upstream_actor'} -> Treatment\n",
      "Disease - {'primarily_composed_of'} -> Treatment\n",
      "Disease - {'similar_to'} -> Treatment\n",
      "Disease - {'overlaps'} -> Treatment\n",
      "Disease - {'composed_primarily_of'} -> Treatment\n",
      "Disease - {'treats'} -> Treatment\n",
      "Disease - {'precedes'} -> Treatment\n",
      "Disease - {'subclass_of'} -> Treatment\n",
      "Disease - {'has_part'} -> Treatment\n",
      "Disease - {'contributes_to'} -> Treatment\n",
      "Disease - {'manifestation_of'} -> Treatment\n",
      "Disease - {'coexists_with'} -> Treatment\n",
      "Disease - {'close_match'} -> Treatment\n",
      "Treatment - {'subclass_of'} -> Publication\n",
      "Treatment - {'occurs_in'} -> Publication\n",
      "Treatment - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->NucleicAcidEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affects'} -> NucleicAcidEntity\n",
      "Disease - {'affects_response_to'} -> NucleicAcidEntity\n",
      "Disease - {'entity_positively_regulates_entity'} -> NucleicAcidEntity\n",
      "Disease - {'actively_involves'} -> NucleicAcidEntity\n",
      "Disease - {'has_upstream_actor'} -> NucleicAcidEntity\n",
      "Disease - {'exacerbates'} -> NucleicAcidEntity\n",
      "Disease - {'has_active_ingredient'} -> NucleicAcidEntity\n",
      "Disease - {'increases_response_to'} -> NucleicAcidEntity\n",
      "Disease - {'primarily_composed_of'} -> NucleicAcidEntity\n",
      "Disease - {'temporally_related_to'} -> NucleicAcidEntity\n",
      "Disease - {'similar_to'} -> NucleicAcidEntity\n",
      "Disease - {'related_to'} -> NucleicAcidEntity\n",
      "Disease - {'overlaps'} -> NucleicAcidEntity\n",
      "Disease - {'composed_primarily_of'} -> NucleicAcidEntity\n",
      "Disease - {'preceded_by'} -> NucleicAcidEntity\n",
      "Disease - {'has_output'} -> NucleicAcidEntity\n",
      "Disease - {'disrupts'} -> NucleicAcidEntity\n",
      "Disease - {'affected_by'} -> NucleicAcidEntity\n",
      "Disease - {'has_substrate'} -> NucleicAcidEntity\n",
      "Disease - {'interacts_with'} -> NucleicAcidEntity\n",
      "Disease - {'ameliorates'} -> NucleicAcidEntity\n",
      "Disease - {'treats'} -> NucleicAcidEntity\n",
      "Disease - {'regulates'} -> NucleicAcidEntity\n",
      "Disease - {'part_of'} -> NucleicAcidEntity\n",
      "Disease - {'has_capability'} -> NucleicAcidEntity\n",
      "Disease - {'participates_in'} -> NucleicAcidEntity\n",
      "Disease - {'caused_by'} -> NucleicAcidEntity\n",
      "Disease - {'has_input'} -> NucleicAcidEntity\n",
      "Disease - {'enabled_by'} -> NucleicAcidEntity\n",
      "Disease - {'has_participant'} -> NucleicAcidEntity\n",
      "Disease - {'has_catalyst'} -> NucleicAcidEntity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> NucleicAcidEntity\n",
      "Disease - {'decreases_response_to'} -> NucleicAcidEntity\n",
      "Disease - {'associated_with'} -> NucleicAcidEntity\n",
      "Disease - {'has_part'} -> NucleicAcidEntity\n",
      "Disease - {'located_in'} -> NucleicAcidEntity\n",
      "Disease - {'physically_interacts_with'} -> NucleicAcidEntity\n",
      "Disease - {'derives_from'} -> NucleicAcidEntity\n",
      "Disease - {'causes'} -> NucleicAcidEntity\n",
      "Disease - {'precedes'} -> NucleicAcidEntity\n",
      "Disease - {'subclass_of'} -> NucleicAcidEntity\n",
      "Disease - {'produces'} -> NucleicAcidEntity\n",
      "Disease - {'predisposes'} -> NucleicAcidEntity\n",
      "Disease - {'diagnoses'} -> NucleicAcidEntity\n",
      "Disease - {'contributes_to'} -> NucleicAcidEntity\n",
      "Disease - {'manifestation_of'} -> NucleicAcidEntity\n",
      "Disease - {'occurs_in'} -> NucleicAcidEntity\n",
      "Disease - {'close_match'} -> NucleicAcidEntity\n",
      "Disease - {'coexists_with'} -> NucleicAcidEntity\n",
      "NucleicAcidEntity - {'gene_associated_with_condition'} -> Publication\n",
      "NucleicAcidEntity - {'located_in'} -> Publication\n",
      "NucleicAcidEntity - {'related_to'} -> Publication\n",
      "NucleicAcidEntity - {'close_match'} -> Publication\n",
      "NucleicAcidEntity - {'subclass_of'} -> Publication\n",
      "NucleicAcidEntity - {'interacts_with'} -> Publication\n",
      "NucleicAcidEntity - {'occurs_in'} -> Publication\n",
      "NucleicAcidEntity - {'affects'} -> Publication\n",
      "NucleicAcidEntity - {'treats'} -> Publication\n",
      "NucleicAcidEntity - {'causes'} -> Publication\n",
      "NucleicAcidEntity - {'coexists_with'} -> Publication\n",
      "NucleicAcidEntity - {'disrupts'} -> Publication\n",
      "NucleicAcidEntity - {'associated_with'} -> Publication\n",
      "NucleicAcidEntity - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Transcript->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affects'} -> Transcript\n",
      "Disease - {'enabled_by'} -> Transcript\n",
      "Disease - {'has_participant'} -> Transcript\n",
      "Disease - {'has_catalyst'} -> Transcript\n",
      "Disease - {'decreases_response_to'} -> Transcript\n",
      "Disease - {'associated_with'} -> Transcript\n",
      "Disease - {'actively_involves'} -> Transcript\n",
      "Disease - {'has_upstream_actor'} -> Transcript\n",
      "Disease - {'exacerbates'} -> Transcript\n",
      "Disease - {'increases_response_to'} -> Transcript\n",
      "Disease - {'primarily_composed_of'} -> Transcript\n",
      "Disease - {'has_output'} -> Transcript\n",
      "Disease - {'similar_to'} -> Transcript\n",
      "Disease - {'related_to'} -> Transcript\n",
      "Disease - {'overlaps'} -> Transcript\n",
      "Disease - {'affects_response_to'} -> Transcript\n",
      "Disease - {'composed_primarily_of'} -> Transcript\n",
      "Disease - {'disrupts'} -> Transcript\n",
      "Disease - {'affected_by'} -> Transcript\n",
      "Disease - {'has_substrate'} -> Transcript\n",
      "Disease - {'interacts_with'} -> Transcript\n",
      "Disease - {'ameliorates'} -> Transcript\n",
      "Disease - {'treats'} -> Transcript\n",
      "Disease - {'regulates'} -> Transcript\n",
      "Disease - {'has_capability'} -> Transcript\n",
      "Disease - {'participates_in'} -> Transcript\n",
      "Disease - {'has_input'} -> Transcript\n",
      "Disease - {'subclass_of'} -> Transcript\n",
      "Disease - {'produces'} -> Transcript\n",
      "Disease - {'has_part'} -> Transcript\n",
      "Disease - {'coexists_with'} -> Transcript\n",
      "Transcript - {'translates_to'} -> Publication\n",
      "Transcript - {'regulates'} -> Publication\n",
      "Transcript - {'transcribed_from'} -> Publication\n",
      "Transcript - {'associated_with'} -> Publication\n",
      "Transcript - {'located_in'} -> Publication\n",
      "Transcript - {'subclass_of'} -> Publication\n",
      "Transcript - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->PathologicalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'causes'} -> PathologicalProcess\n",
      "Disease - {'preceded_by'} -> PathologicalProcess\n",
      "Disease - {'exacerbates'} -> PathologicalProcess\n",
      "Disease - {'has_predisposing_factor'} -> PathologicalProcess\n",
      "Disease - {'caused_by'} -> PathologicalProcess\n",
      "Disease - {'precedes'} -> PathologicalProcess\n",
      "Disease - {'has_manifestation'} -> PathologicalProcess\n",
      "Disease - {'associated_with'} -> PathologicalProcess\n",
      "Disease - {'coexists_with'} -> PathologicalProcess\n",
      "Disease - {'occurs_in'} -> PathologicalProcess\n",
      "Disease - {'predisposes'} -> PathologicalProcess\n",
      "Disease - {'affects'} -> PathologicalProcess\n",
      "Disease - {'manifestation_of'} -> PathologicalProcess\n",
      "Disease - {'affected_by'} -> PathologicalProcess\n",
      "Disease - {'is_exacerbated_by'} -> PathologicalProcess\n",
      "Disease - {'disrupts'} -> PathologicalProcess\n",
      "Disease - {'resistance_associated_with'} -> PathologicalProcess\n",
      "Disease - {'correlated_with'} -> PathologicalProcess\n",
      "Disease - {'treats'} -> PathologicalProcess\n",
      "Disease - {'is_ameliorated_by'} -> PathologicalProcess\n",
      "Disease - {'regulates'} -> PathologicalProcess\n",
      "Disease - {'response_increased_by'} -> PathologicalProcess\n",
      "Disease - {'sensitivity_associated_with'} -> PathologicalProcess\n",
      "Disease - {'has_biomarker'} -> PathologicalProcess\n",
      "Disease - {'decreased_likelihood_associated_with'} -> PathologicalProcess\n",
      "Disease - {'contributes_to'} -> PathologicalProcess\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> PathologicalProcess\n",
      "Disease - {'decreases_response_to'} -> PathologicalProcess\n",
      "Disease - {'ameliorates'} -> PathologicalProcess\n",
      "Disease - {'positively_correlated_with'} -> PathologicalProcess\n",
      "Disease - {'disrupted_by'} -> PathologicalProcess\n",
      "Disease - {'increased_likelihood_associated_with'} -> PathologicalProcess\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> PathologicalProcess\n",
      "Disease - {'occurs_together_in_literature_with'} -> PathologicalProcess\n",
      "Disease - {'temporally_related_to'} -> PathologicalProcess\n",
      "Disease - {'response_affected_by'} -> PathologicalProcess\n",
      "Disease - {'coexpressed_with'} -> PathologicalProcess\n",
      "Disease - {'negatively_correlated_with'} -> PathologicalProcess\n",
      "Disease - {'related_to'} -> PathologicalProcess\n",
      "Disease - {'biomarker_for'} -> PathologicalProcess\n",
      "Disease - {'affects_response_to'} -> PathologicalProcess\n",
      "Disease - {'entity_positively_regulates_entity'} -> PathologicalProcess\n",
      "Disease - {'treated_by'} -> PathologicalProcess\n",
      "Disease - {'associated_with_sensitivity_to'} -> PathologicalProcess\n",
      "Disease - {'increases_response_to'} -> PathologicalProcess\n",
      "Disease - {'regulated_by'} -> PathologicalProcess\n",
      "Disease - {'associated_with_resistance_to'} -> PathologicalProcess\n",
      "Disease - {'response_decreased_by'} -> PathologicalProcess\n",
      "Disease - {'gene_associated_with_condition'} -> PathologicalProcess\n",
      "Disease - {'subclass_of'} -> PathologicalProcess\n",
      "Disease - {'has_part'} -> PathologicalProcess\n",
      "Disease - {'prevents'} -> PathologicalProcess\n",
      "Disease - {'interacts_with'} -> PathologicalProcess\n",
      "Disease - {'produces'} -> PathologicalProcess\n",
      "Disease - {'diagnoses'} -> PathologicalProcess\n",
      "Disease - {'has_input'} -> PathologicalProcess\n",
      "Disease - {'close_match'} -> PathologicalProcess\n",
      "Disease - {'has_participant'} -> PathologicalProcess\n",
      "Disease - {'has_phenotype'} -> PathologicalProcess\n",
      "Disease - {'located_in'} -> PathologicalProcess\n",
      "Disease - {'contraindicated_for'} -> PathologicalProcess\n",
      "PathologicalProcess - {'manifestation_of'} -> Publication\n",
      "PathologicalProcess - {'occurs_in'} -> Publication\n",
      "PathologicalProcess - {'coexists_with'} -> Publication\n",
      "PathologicalProcess - {'exacerbates'} -> Publication\n",
      "PathologicalProcess - {'disrupts'} -> Publication\n",
      "PathologicalProcess - {'located_in'} -> Publication\n",
      "PathologicalProcess - {'regulates'} -> Publication\n",
      "PathologicalProcess - {'treats'} -> Publication\n",
      "PathologicalProcess - {'precedes'} -> Publication\n",
      "PathologicalProcess - {'causes'} -> Publication\n",
      "PathologicalProcess - {'close_match'} -> Publication\n",
      "PathologicalProcess - {'associated_with'} -> Publication\n",
      "PathologicalProcess - {'subclass_of'} -> Publication\n",
      "PathologicalProcess - {'produces'} -> Publication\n",
      "PathologicalProcess - {'predisposes'} -> Publication\n",
      "PathologicalProcess - {'related_to'} -> Publication\n",
      "PathologicalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->GeneticInheritance->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'related_to'} -> GeneticInheritance\n",
      "Disease - {'has_mode_of_inheritance'} -> GeneticInheritance\n",
      "Disease - {'diagnoses'} -> GeneticInheritance\n",
      "Disease - {'located_in'} -> GeneticInheritance\n",
      "Disease - {'subclass_of'} -> GeneticInheritance\n",
      "Disease - {'regulates'} -> GeneticInheritance\n",
      "Disease - {'has_part'} -> GeneticInheritance\n",
      "Disease - {'close_match'} -> GeneticInheritance\n",
      "Disease - {'precedes'} -> GeneticInheritance\n",
      "Disease - {'causes'} -> GeneticInheritance\n",
      "Disease - {'has_input'} -> GeneticInheritance\n",
      "Disease - {'affects'} -> GeneticInheritance\n",
      "Disease - {'coexists_with'} -> GeneticInheritance\n",
      "Disease - {'associated_with'} -> GeneticInheritance\n",
      "Disease - {'manifestation_of'} -> GeneticInheritance\n",
      "Disease - {'disrupts'} -> GeneticInheritance\n",
      "GeneticInheritance - {'occurs_in'} -> Publication\n",
      "GeneticInheritance - {'has_input'} -> Publication\n",
      "GeneticInheritance - {'located_in'} -> Publication\n",
      "GeneticInheritance - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->DiseaseOrPhenotypicFeature->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'subclass_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'related_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'composed_primarily_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'has_upstream_actor'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'exacerbates'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'has_active_ingredient'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'associated_with_sensitivity_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'increases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'primarily_composed_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'temporally_related_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'associated_with_resistance_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'disrupts'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'affected_by'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'treats'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'sensitivity_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'has_biomarker'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'participates_in'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'decreased_likelihood_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'preceded_by'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'has_participant'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'decreases_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'resistance_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'has_part'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'interacts_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'ameliorates'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'regulates'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'positively_correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'part_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'increased_likelihood_associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'occurs_together_in_literature_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'coexpressed_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'associated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'negatively_correlated_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'affects'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'similar_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'overlaps'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'biomarker_for'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'affects_response_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'coexists_with'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'has_phenotype'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'located_in'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'contraindicated_for'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'derives_from'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'disease_has_location'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'causes'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'actively_involved_in'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'precedes'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'is_sequence_variant_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'gene_product_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'gene_associated_with_condition'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'prevents'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'contributes_to'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'produces'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'predisposes'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'diagnoses'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'occurs_in'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'manifestation_of'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'has_input'} -> DiseaseOrPhenotypicFeature\n",
      "Disease - {'close_match'} -> DiseaseOrPhenotypicFeature\n",
      "DiseaseOrPhenotypicFeature - {'predisposes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'related_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'occurs_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'affects'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'manifestation_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_input'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'exacerbates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'coexists_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'close_match'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'gene_associated_with_condition'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disrupts'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'biomarker_for'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'located_in'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'regulates'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'contributes_to'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'treats'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'disease_has_location'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'precedes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'causes'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'associated_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'subclass_of'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'has_part'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'interacts_with'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'produces'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'diagnoses'} -> Publication\n",
      "DiseaseOrPhenotypicFeature - {'prevents'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->OrganismAttribute->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'subclass_of'} -> OrganismAttribute\n",
      "Disease - {'related_to'} -> OrganismAttribute\n",
      "Disease - {'participates_in'} -> OrganismAttribute\n",
      "Disease - {'preceded_by'} -> OrganismAttribute\n",
      "Disease - {'has_participant'} -> OrganismAttribute\n",
      "Disease - {'decreases_response_to'} -> OrganismAttribute\n",
      "Disease - {'associated_with'} -> OrganismAttribute\n",
      "Disease - {'has_part'} -> OrganismAttribute\n",
      "Disease - {'ameliorates'} -> OrganismAttribute\n",
      "Disease - {'genetic_association'} -> OrganismAttribute\n",
      "Disease - {'has_upstream_actor'} -> OrganismAttribute\n",
      "Disease - {'has_active_ingredient'} -> OrganismAttribute\n",
      "Disease - {'primarily_composed_of'} -> OrganismAttribute\n",
      "Disease - {'temporally_related_to'} -> OrganismAttribute\n",
      "Disease - {'affects'} -> OrganismAttribute\n",
      "Disease - {'similar_to'} -> OrganismAttribute\n",
      "Disease - {'overlaps'} -> OrganismAttribute\n",
      "Disease - {'affects_response_to'} -> OrganismAttribute\n",
      "Disease - {'composed_primarily_of'} -> OrganismAttribute\n",
      "Disease - {'gene_associated_with_condition'} -> OrganismAttribute\n",
      "Disease - {'exacerbates'} -> OrganismAttribute\n",
      "Disease - {'increases_response_to'} -> OrganismAttribute\n",
      "Disease - {'disrupts'} -> OrganismAttribute\n",
      "Disease - {'affected_by'} -> OrganismAttribute\n",
      "Disease - {'interacts_with'} -> OrganismAttribute\n",
      "Disease - {'treats'} -> OrganismAttribute\n",
      "Disease - {'condition_associated_with_gene'} -> OrganismAttribute\n",
      "Disease - {'regulates'} -> OrganismAttribute\n",
      "Disease - {'part_of'} -> OrganismAttribute\n",
      "Disease - {'manifestation_of'} -> OrganismAttribute\n",
      "Disease - {'located_in'} -> OrganismAttribute\n",
      "Disease - {'occurs_in'} -> OrganismAttribute\n",
      "Disease - {'close_match'} -> OrganismAttribute\n",
      "Disease - {'predisposes'} -> OrganismAttribute\n",
      "Disease - {'causes'} -> OrganismAttribute\n",
      "Disease - {'precedes'} -> OrganismAttribute\n",
      "Disease - {'coexists_with'} -> OrganismAttribute\n",
      "OrganismAttribute - {'related_to'} -> Publication\n",
      "OrganismAttribute - {'produces'} -> Publication\n",
      "OrganismAttribute - {'has_input'} -> Publication\n",
      "OrganismAttribute - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Entity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'subclass_of'} -> Entity\n",
      "Disease - {'related_to'} -> Entity\n",
      "Disease - {'part_of'} -> Entity\n",
      "Disease - {'participates_in'} -> Entity\n",
      "Disease - {'has_catalyst'} -> Entity\n",
      "Disease - {'expressed_in'} -> Entity\n",
      "Disease - {'temporally_related_to'} -> Entity\n",
      "Disease - {'actively_involved_in'} -> Entity\n",
      "Disease - {'associated_with'} -> Entity\n",
      "Disease - {'affects'} -> Entity\n",
      "Disease - {'similar_to'} -> Entity\n",
      "Disease - {'catalyzes'} -> Entity\n",
      "Disease - {'overlaps'} -> Entity\n",
      "Disease - {'affects_response_to'} -> Entity\n",
      "Disease - {'entity_positively_regulates_entity'} -> Entity\n",
      "Disease - {'located_in'} -> Entity\n",
      "Disease - {'composed_primarily_of'} -> Entity\n",
      "Disease - {'has_upstream_actor'} -> Entity\n",
      "Disease - {'exacerbates'} -> Entity\n",
      "Disease - {'has_active_ingredient'} -> Entity\n",
      "Disease - {'increases_response_to'} -> Entity\n",
      "Disease - {'primarily_composed_of'} -> Entity\n",
      "Disease - {'has_output'} -> Entity\n",
      "Disease - {'disrupts'} -> Entity\n",
      "Disease - {'is_output_of'} -> Entity\n",
      "Disease - {'affected_by'} -> Entity\n",
      "Disease - {'treats'} -> Entity\n",
      "Disease - {'causes'} -> Entity\n",
      "Disease - {'contributes_to'} -> Entity\n",
      "Disease - {'has_input'} -> Entity\n",
      "Disease - {'preceded_by'} -> Entity\n",
      "Disease - {'enabled_by'} -> Entity\n",
      "Disease - {'has_participant'} -> Entity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> Entity\n",
      "Disease - {'is_input_of'} -> Entity\n",
      "Disease - {'decreases_response_to'} -> Entity\n",
      "Disease - {'enables'} -> Entity\n",
      "Disease - {'has_part'} -> Entity\n",
      "Disease - {'has_substrate'} -> Entity\n",
      "Disease - {'interacts_with'} -> Entity\n",
      "Disease - {'ameliorates'} -> Entity\n",
      "Disease - {'regulates'} -> Entity\n",
      "Disease - {'is_substrate_of'} -> Entity\n",
      "Disease - {'has_capability'} -> Entity\n",
      "Disease - {'capable_of'} -> Entity\n",
      "Disease - {'caused_by'} -> Entity\n",
      "Disease - {'actively_involves'} -> Entity\n",
      "Entity - {'subclass_of'} -> Publication\n",
      "Entity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Food->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affected_by'} -> Food\n",
      "Disease - {'has_predisposing_factor'} -> Food\n",
      "Disease - {'caused_by'} -> Food\n",
      "Disease - {'associated_with_sensitivity_to'} -> Food\n",
      "Disease - {'associated_with_increased_likelihood_of'} -> Food\n",
      "Disease - {'response_decreased_by'} -> Food\n",
      "Disease - {'resistance_associated_with'} -> Food\n",
      "Disease - {'correlated_with'} -> Food\n",
      "Disease - {'is_ameliorated_by'} -> Food\n",
      "Disease - {'response_increased_by'} -> Food\n",
      "Disease - {'sensitivity_associated_with'} -> Food\n",
      "Disease - {'positively_correlated_with'} -> Food\n",
      "Disease - {'has_biomarker'} -> Food\n",
      "Disease - {'decreased_likelihood_associated_with'} -> Food\n",
      "Disease - {'associated_with'} -> Food\n",
      "Disease - {'is_exacerbated_by'} -> Food\n",
      "Disease - {'biomarker_for'} -> Food\n",
      "Disease - {'disrupted_by'} -> Food\n",
      "Disease - {'increased_likelihood_associated_with'} -> Food\n",
      "Disease - {'associated_with_decreased_likelihood_of'} -> Food\n",
      "Disease - {'occurs_together_in_literature_with'} -> Food\n",
      "Disease - {'treated_by'} -> Food\n",
      "Disease - {'regulated_by'} -> Food\n",
      "Disease - {'associated_with_resistance_to'} -> Food\n",
      "Disease - {'response_affected_by'} -> Food\n",
      "Disease - {'coexpressed_with'} -> Food\n",
      "Disease - {'negatively_correlated_with'} -> Food\n",
      "Disease - {'related_to'} -> Food\n",
      "Disease - {'diagnoses'} -> Food\n",
      "Disease - {'occurs_in'} -> Food\n",
      "Disease - {'manifestation_of'} -> Food\n",
      "Disease - {'has_input'} -> Food\n",
      "Disease - {'has_output'} -> Food\n",
      "Disease - {'affects'} -> Food\n",
      "Disease - {'close_match'} -> Food\n",
      "Disease - {'has_participant'} -> Food\n",
      "Disease - {'exacerbates'} -> Food\n",
      "Disease - {'coexists_with'} -> Food\n",
      "Disease - {'has_phenotype'} -> Food\n",
      "Disease - {'located_in'} -> Food\n",
      "Disease - {'regulates'} -> Food\n",
      "Disease - {'treats'} -> Food\n",
      "Disease - {'physically_interacts_with'} -> Food\n",
      "Disease - {'causes'} -> Food\n",
      "Disease - {'precedes'} -> Food\n",
      "Disease - {'subclass_of'} -> Food\n",
      "Disease - {'has_part'} -> Food\n",
      "Disease - {'interacts_with'} -> Food\n",
      "Disease - {'produces'} -> Food\n",
      "Disease - {'predisposes'} -> Food\n",
      "Disease - {'derives_from'} -> Food\n",
      "Food - {'regulates'} -> Publication\n",
      "Food - {'associated_with'} -> Publication\n",
      "Food - {'located_in'} -> Publication\n",
      "Food - {'predisposes'} -> Publication\n",
      "Food - {'related_to'} -> Publication\n",
      "Food - {'occurs_in'} -> Publication\n",
      "Food - {'affects'} -> Publication\n",
      "Food - {'precedes'} -> Publication\n",
      "Food - {'subclass_of'} -> Publication\n",
      "Food - {'treats'} -> Publication\n",
      "Food - {'causes'} -> Publication\n",
      "Food - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->NoncodingRNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'has_input'} -> NoncodingRNAProduct\n",
      "Disease - {'enabled_by'} -> NoncodingRNAProduct\n",
      "Disease - {'has_participant'} -> NoncodingRNAProduct\n",
      "Disease - {'has_substrate'} -> NoncodingRNAProduct\n",
      "Disease - {'interacts_with'} -> NoncodingRNAProduct\n",
      "Disease - {'has_capability'} -> NoncodingRNAProduct\n",
      "Disease - {'participates_in'} -> NoncodingRNAProduct\n",
      "Disease - {'actively_involves'} -> NoncodingRNAProduct\n",
      "Disease - {'has_catalyst'} -> NoncodingRNAProduct\n",
      "Disease - {'associated_with'} -> NoncodingRNAProduct\n",
      "Disease - {'overlaps'} -> NoncodingRNAProduct\n",
      "Disease - {'composed_primarily_of'} -> NoncodingRNAProduct\n",
      "Disease - {'has_upstream_actor'} -> NoncodingRNAProduct\n",
      "Disease - {'primarily_composed_of'} -> NoncodingRNAProduct\n",
      "Disease - {'has_output'} -> NoncodingRNAProduct\n",
      "Disease - {'affects'} -> NoncodingRNAProduct\n",
      "Disease - {'similar_to'} -> NoncodingRNAProduct\n",
      "Disease - {'related_to'} -> NoncodingRNAProduct\n",
      "Disease - {'affected_by'} -> NoncodingRNAProduct\n",
      "Disease - {'has_part'} -> NoncodingRNAProduct\n",
      "Disease - {'produces'} -> NoncodingRNAProduct\n",
      "Disease - {'coexists_with'} -> NoncodingRNAProduct\n",
      "Disease - {'regulates'} -> NoncodingRNAProduct\n",
      "NoncodingRNAProduct - {'affects'} -> Publication\n",
      "NoncodingRNAProduct - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->RNAProduct->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'overlaps'} -> RNAProduct\n",
      "Disease - {'has_upstream_actor'} -> RNAProduct\n",
      "Disease - {'exacerbates'} -> RNAProduct\n",
      "Disease - {'increases_response_to'} -> RNAProduct\n",
      "Disease - {'primarily_composed_of'} -> RNAProduct\n",
      "Disease - {'has_output'} -> RNAProduct\n",
      "Disease - {'affects'} -> RNAProduct\n",
      "Disease - {'disrupts'} -> RNAProduct\n",
      "Disease - {'similar_to'} -> RNAProduct\n",
      "Disease - {'related_to'} -> RNAProduct\n",
      "Disease - {'affected_by'} -> RNAProduct\n",
      "Disease - {'affects_response_to'} -> RNAProduct\n",
      "Disease - {'treats'} -> RNAProduct\n",
      "Disease - {'composed_primarily_of'} -> RNAProduct\n",
      "Disease - {'has_input'} -> RNAProduct\n",
      "Disease - {'enabled_by'} -> RNAProduct\n",
      "Disease - {'has_participant'} -> RNAProduct\n",
      "Disease - {'has_substrate'} -> RNAProduct\n",
      "Disease - {'interacts_with'} -> RNAProduct\n",
      "Disease - {'ameliorates'} -> RNAProduct\n",
      "Disease - {'regulates'} -> RNAProduct\n",
      "Disease - {'has_capability'} -> RNAProduct\n",
      "Disease - {'participates_in'} -> RNAProduct\n",
      "Disease - {'actively_involves'} -> RNAProduct\n",
      "Disease - {'has_catalyst'} -> RNAProduct\n",
      "Disease - {'decreases_response_to'} -> RNAProduct\n",
      "Disease - {'associated_with'} -> RNAProduct\n",
      "Disease - {'subclass_of'} -> RNAProduct\n",
      "Disease - {'has_part'} -> RNAProduct\n",
      "Disease - {'produces'} -> RNAProduct\n",
      "Disease - {'coexists_with'} -> RNAProduct\n",
      "Disease - {'located_in'} -> RNAProduct\n",
      "RNAProduct - {'located_in'} -> Publication\n",
      "RNAProduct - {'regulates'} -> Publication\n",
      "RNAProduct - {'related_to'} -> Publication\n",
      "RNAProduct - {'affects'} -> Publication\n",
      "RNAProduct - {'subclass_of'} -> Publication\n",
      "RNAProduct - {'disrupts'} -> Publication\n",
      "RNAProduct - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->EnvironmentalProcess->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'decreases_response_to'} -> EnvironmentalProcess\n",
      "Disease - {'has_part'} -> EnvironmentalProcess\n",
      "Disease - {'ameliorates'} -> EnvironmentalProcess\n",
      "Disease - {'actively_involves'} -> EnvironmentalProcess\n",
      "Disease - {'has_active_ingredient'} -> EnvironmentalProcess\n",
      "Disease - {'primarily_composed_of'} -> EnvironmentalProcess\n",
      "Disease - {'temporally_related_to'} -> EnvironmentalProcess\n",
      "Disease - {'associated_with'} -> EnvironmentalProcess\n",
      "Disease - {'affects'} -> EnvironmentalProcess\n",
      "Disease - {'similar_to'} -> EnvironmentalProcess\n",
      "Disease - {'related_to'} -> EnvironmentalProcess\n",
      "Disease - {'overlaps'} -> EnvironmentalProcess\n",
      "Disease - {'affects_response_to'} -> EnvironmentalProcess\n",
      "Disease - {'composed_primarily_of'} -> EnvironmentalProcess\n",
      "Disease - {'has_upstream_actor'} -> EnvironmentalProcess\n",
      "Disease - {'exacerbates'} -> EnvironmentalProcess\n",
      "Disease - {'increases_response_to'} -> EnvironmentalProcess\n",
      "Disease - {'disrupts'} -> EnvironmentalProcess\n",
      "Disease - {'affected_by'} -> EnvironmentalProcess\n",
      "Disease - {'interacts_with'} -> EnvironmentalProcess\n",
      "Disease - {'treats'} -> EnvironmentalProcess\n",
      "Disease - {'regulates'} -> EnvironmentalProcess\n",
      "Disease - {'part_of'} -> EnvironmentalProcess\n",
      "Disease - {'has_capability'} -> EnvironmentalProcess\n",
      "Disease - {'participates_in'} -> EnvironmentalProcess\n",
      "Disease - {'preceded_by'} -> EnvironmentalProcess\n",
      "Disease - {'has_participant'} -> EnvironmentalProcess\n",
      "Disease - {'subclass_of'} -> EnvironmentalProcess\n",
      "Disease - {'coexists_with'} -> EnvironmentalProcess\n",
      "Disease - {'close_match'} -> EnvironmentalProcess\n",
      "EnvironmentalProcess - {'subclass_of'} -> Publication\n",
      "EnvironmentalProcess - {'related_to'} -> Publication\n",
      "EnvironmentalProcess - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->OrganismalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'capable_of'} -> OrganismalEntity\n",
      "Disease - {'has_input'} -> OrganismalEntity\n",
      "Disease - {'preceded_by'} -> OrganismalEntity\n",
      "Disease - {'enabled_by'} -> OrganismalEntity\n",
      "Disease - {'has_participant'} -> OrganismalEntity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> OrganismalEntity\n",
      "Disease - {'is_input_of'} -> OrganismalEntity\n",
      "Disease - {'decreases_response_to'} -> OrganismalEntity\n",
      "Disease - {'enables'} -> OrganismalEntity\n",
      "Disease - {'has_part'} -> OrganismalEntity\n",
      "Disease - {'has_substrate'} -> OrganismalEntity\n",
      "Disease - {'interacts_with'} -> OrganismalEntity\n",
      "Disease - {'ameliorates'} -> OrganismalEntity\n",
      "Disease - {'caused_by'} -> OrganismalEntity\n",
      "Disease - {'actively_involves'} -> OrganismalEntity\n",
      "Disease - {'has_active_ingredient'} -> OrganismalEntity\n",
      "Disease - {'has_catalyst'} -> OrganismalEntity\n",
      "Disease - {'primarily_composed_of'} -> OrganismalEntity\n",
      "Disease - {'expressed_in'} -> OrganismalEntity\n",
      "Disease - {'temporally_related_to'} -> OrganismalEntity\n",
      "Disease - {'actively_involved_in'} -> OrganismalEntity\n",
      "Disease - {'associated_with'} -> OrganismalEntity\n",
      "Disease - {'affects'} -> OrganismalEntity\n",
      "Disease - {'similar_to'} -> OrganismalEntity\n",
      "Disease - {'catalyzes'} -> OrganismalEntity\n",
      "Disease - {'related_to'} -> OrganismalEntity\n",
      "Disease - {'overlaps'} -> OrganismalEntity\n",
      "Disease - {'affects_response_to'} -> OrganismalEntity\n",
      "Disease - {'entity_positively_regulates_entity'} -> OrganismalEntity\n",
      "Disease - {'located_in'} -> OrganismalEntity\n",
      "Disease - {'composed_primarily_of'} -> OrganismalEntity\n",
      "Disease - {'has_upstream_actor'} -> OrganismalEntity\n",
      "Disease - {'exacerbates'} -> OrganismalEntity\n",
      "Disease - {'increases_response_to'} -> OrganismalEntity\n",
      "Disease - {'has_output'} -> OrganismalEntity\n",
      "Disease - {'disrupts'} -> OrganismalEntity\n",
      "Disease - {'is_output_of'} -> OrganismalEntity\n",
      "Disease - {'affected_by'} -> OrganismalEntity\n",
      "Disease - {'treats'} -> OrganismalEntity\n",
      "Disease - {'regulates'} -> OrganismalEntity\n",
      "Disease - {'causes'} -> OrganismalEntity\n",
      "Disease - {'is_substrate_of'} -> OrganismalEntity\n",
      "Disease - {'part_of'} -> OrganismalEntity\n",
      "Disease - {'has_capability'} -> OrganismalEntity\n",
      "Disease - {'participates_in'} -> OrganismalEntity\n",
      "Disease - {'contributes_to'} -> OrganismalEntity\n",
      "OrganismalEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Phenomenon->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'related_to'} -> Phenomenon\n",
      "Disease - {'causes'} -> Phenomenon\n",
      "Disease - {'contributes_to'} -> Phenomenon\n",
      "Disease - {'diagnoses'} -> Phenomenon\n",
      "Disease - {'occurs_in'} -> Phenomenon\n",
      "Disease - {'manifestation_of'} -> Phenomenon\n",
      "Disease - {'has_input'} -> Phenomenon\n",
      "Disease - {'exacerbates'} -> Phenomenon\n",
      "Disease - {'affects'} -> Phenomenon\n",
      "Disease - {'coexists_with'} -> Phenomenon\n",
      "Disease - {'close_match'} -> Phenomenon\n",
      "Disease - {'has_participant'} -> Phenomenon\n",
      "Disease - {'has_phenotype'} -> Phenomenon\n",
      "Disease - {'regulates'} -> Phenomenon\n",
      "Disease - {'disrupts'} -> Phenomenon\n",
      "Disease - {'located_in'} -> Phenomenon\n",
      "Disease - {'biomarker_for'} -> Phenomenon\n",
      "Disease - {'treats'} -> Phenomenon\n",
      "Disease - {'physically_interacts_with'} -> Phenomenon\n",
      "Disease - {'precedes'} -> Phenomenon\n",
      "Disease - {'associated_with'} -> Phenomenon\n",
      "Disease - {'gene_associated_with_condition'} -> Phenomenon\n",
      "Disease - {'subclass_of'} -> Phenomenon\n",
      "Disease - {'predisposes'} -> Phenomenon\n",
      "Disease - {'has_part'} -> Phenomenon\n",
      "Disease - {'prevents'} -> Phenomenon\n",
      "Disease - {'interacts_with'} -> Phenomenon\n",
      "Disease - {'produces'} -> Phenomenon\n",
      "Phenomenon - {'associated_with'} -> Publication\n",
      "Phenomenon - {'subclass_of'} -> Publication\n",
      "Phenomenon - {'interacts_with'} -> Publication\n",
      "Phenomenon - {'produces'} -> Publication\n",
      "Phenomenon - {'related_to'} -> Publication\n",
      "Phenomenon - {'has_part'} -> Publication\n",
      "Phenomenon - {'diagnoses'} -> Publication\n",
      "Phenomenon - {'occurs_in'} -> Publication\n",
      "Phenomenon - {'affects'} -> Publication\n",
      "Phenomenon - {'coexists_with'} -> Publication\n",
      "Phenomenon - {'close_match'} -> Publication\n",
      "Phenomenon - {'disrupts'} -> Publication\n",
      "Phenomenon - {'regulates'} -> Publication\n",
      "Phenomenon - {'located_in'} -> Publication\n",
      "Phenomenon - {'has_input'} -> Publication\n",
      "Phenomenon - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Behavior->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affected_by'} -> Behavior\n",
      "Disease - {'interacts_with'} -> Behavior\n",
      "Disease - {'ameliorates'} -> Behavior\n",
      "Disease - {'treats'} -> Behavior\n",
      "Disease - {'regulates'} -> Behavior\n",
      "Disease - {'part_of'} -> Behavior\n",
      "Disease - {'has_capability'} -> Behavior\n",
      "Disease - {'participates_in'} -> Behavior\n",
      "Disease - {'contributes_to'} -> Behavior\n",
      "Disease - {'has_participant'} -> Behavior\n",
      "Disease - {'decreases_response_to'} -> Behavior\n",
      "Disease - {'associated_with'} -> Behavior\n",
      "Disease - {'has_part'} -> Behavior\n",
      "Disease - {'genetic_association'} -> Behavior\n",
      "Disease - {'actively_involves'} -> Behavior\n",
      "Disease - {'has_upstream_actor'} -> Behavior\n",
      "Disease - {'exacerbates'} -> Behavior\n",
      "Disease - {'has_active_ingredient'} -> Behavior\n",
      "Disease - {'increases_response_to'} -> Behavior\n",
      "Disease - {'primarily_composed_of'} -> Behavior\n",
      "Disease - {'temporally_related_to'} -> Behavior\n",
      "Disease - {'affects'} -> Behavior\n",
      "Disease - {'similar_to'} -> Behavior\n",
      "Disease - {'related_to'} -> Behavior\n",
      "Disease - {'overlaps'} -> Behavior\n",
      "Disease - {'affects_response_to'} -> Behavior\n",
      "Disease - {'condition_associated_with_gene'} -> Behavior\n",
      "Disease - {'causes'} -> Behavior\n",
      "Disease - {'composed_primarily_of'} -> Behavior\n",
      "Disease - {'gene_associated_with_condition'} -> Behavior\n",
      "Disease - {'preceded_by'} -> Behavior\n",
      "Disease - {'disrupts'} -> Behavior\n",
      "Disease - {'occurs_in'} -> Behavior\n",
      "Disease - {'close_match'} -> Behavior\n",
      "Disease - {'coexists_with'} -> Behavior\n",
      "Disease - {'has_phenotype'} -> Behavior\n",
      "Disease - {'precedes'} -> Behavior\n",
      "Disease - {'subclass_of'} -> Behavior\n",
      "Disease - {'predisposes'} -> Behavior\n",
      "Disease - {'prevents'} -> Behavior\n",
      "Disease - {'diagnoses'} -> Behavior\n",
      "Disease - {'manifestation_of'} -> Behavior\n",
      "Disease - {'has_input'} -> Behavior\n",
      "Behavior - {'treats'} -> Publication\n",
      "Behavior - {'has_input'} -> Publication\n",
      "Behavior - {'subclass_of'} -> Publication\n",
      "Behavior - {'located_in'} -> Publication\n",
      "Behavior - {'interacts_with'} -> Publication\n",
      "Behavior - {'predisposes'} -> Publication\n",
      "Behavior - {'related_to'} -> Publication\n",
      "Behavior - {'occurs_in'} -> Publication\n",
      "Behavior - {'affects'} -> Publication\n",
      "Behavior - {'coexists_with'} -> Publication\n",
      "Behavior - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->BiologicalProcessOrActivity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'enables'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_part'} -> BiologicalProcessOrActivity\n",
      "Disease - {'catalyzes'} -> BiologicalProcessOrActivity\n",
      "Disease - {'capable_of'} -> BiologicalProcessOrActivity\n",
      "Disease - {'entity_positively_regulates_entity'} -> BiologicalProcessOrActivity\n",
      "Disease - {'actively_involves'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_upstream_actor'} -> BiologicalProcessOrActivity\n",
      "Disease - {'exacerbates'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_active_ingredient'} -> BiologicalProcessOrActivity\n",
      "Disease - {'increases_response_to'} -> BiologicalProcessOrActivity\n",
      "Disease - {'primarily_composed_of'} -> BiologicalProcessOrActivity\n",
      "Disease - {'temporally_related_to'} -> BiologicalProcessOrActivity\n",
      "Disease - {'actively_involved_in'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_output'} -> BiologicalProcessOrActivity\n",
      "Disease - {'affects'} -> BiologicalProcessOrActivity\n",
      "Disease - {'similar_to'} -> BiologicalProcessOrActivity\n",
      "Disease - {'related_to'} -> BiologicalProcessOrActivity\n",
      "Disease - {'is_output_of'} -> BiologicalProcessOrActivity\n",
      "Disease - {'overlaps'} -> BiologicalProcessOrActivity\n",
      "Disease - {'affects_response_to'} -> BiologicalProcessOrActivity\n",
      "Disease - {'causes'} -> BiologicalProcessOrActivity\n",
      "Disease - {'composed_primarily_of'} -> BiologicalProcessOrActivity\n",
      "Disease - {'preceded_by'} -> BiologicalProcessOrActivity\n",
      "Disease - {'disrupts'} -> BiologicalProcessOrActivity\n",
      "Disease - {'affected_by'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_substrate'} -> BiologicalProcessOrActivity\n",
      "Disease - {'interacts_with'} -> BiologicalProcessOrActivity\n",
      "Disease - {'ameliorates'} -> BiologicalProcessOrActivity\n",
      "Disease - {'treats'} -> BiologicalProcessOrActivity\n",
      "Disease - {'regulates'} -> BiologicalProcessOrActivity\n",
      "Disease - {'is_substrate_of'} -> BiologicalProcessOrActivity\n",
      "Disease - {'part_of'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_capability'} -> BiologicalProcessOrActivity\n",
      "Disease - {'participates_in'} -> BiologicalProcessOrActivity\n",
      "Disease - {'contributes_to'} -> BiologicalProcessOrActivity\n",
      "Disease - {'caused_by'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_input'} -> BiologicalProcessOrActivity\n",
      "Disease - {'enabled_by'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_participant'} -> BiologicalProcessOrActivity\n",
      "Disease - {'has_catalyst'} -> BiologicalProcessOrActivity\n",
      "Disease - {'is_input_of'} -> BiologicalProcessOrActivity\n",
      "Disease - {'decreases_response_to'} -> BiologicalProcessOrActivity\n",
      "Disease - {'associated_with'} -> BiologicalProcessOrActivity\n",
      "Disease - {'coexists_with'} -> BiologicalProcessOrActivity\n",
      "Disease - {'precedes'} -> BiologicalProcessOrActivity\n",
      "Disease - {'manifestation_of'} -> BiologicalProcessOrActivity\n",
      "BiologicalProcessOrActivity - {'occurs_in'} -> Publication\n",
      "BiologicalProcessOrActivity - {'affects'} -> Publication\n",
      "BiologicalProcessOrActivity - {'has_participant'} -> Publication\n",
      "BiologicalProcessOrActivity - {'coexists_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->GeneFamily->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'has_catalyst'} -> GeneFamily\n",
      "Disease - {'associated_with'} -> GeneFamily\n",
      "Disease - {'affects'} -> GeneFamily\n",
      "Disease - {'similar_to'} -> GeneFamily\n",
      "Disease - {'related_to'} -> GeneFamily\n",
      "Disease - {'overlaps'} -> GeneFamily\n",
      "Disease - {'composed_primarily_of'} -> GeneFamily\n",
      "Disease - {'has_upstream_actor'} -> GeneFamily\n",
      "Disease - {'has_active_ingredient'} -> GeneFamily\n",
      "Disease - {'primarily_composed_of'} -> GeneFamily\n",
      "Disease - {'has_output'} -> GeneFamily\n",
      "Disease - {'affected_by'} -> GeneFamily\n",
      "Disease - {'participates_in'} -> GeneFamily\n",
      "Disease - {'has_input'} -> GeneFamily\n",
      "Disease - {'enabled_by'} -> GeneFamily\n",
      "Disease - {'has_participant'} -> GeneFamily\n",
      "Disease - {'has_part'} -> GeneFamily\n",
      "Disease - {'has_substrate'} -> GeneFamily\n",
      "Disease - {'interacts_with'} -> GeneFamily\n",
      "Disease - {'has_capability'} -> GeneFamily\n",
      "Disease - {'actively_involves'} -> GeneFamily\n",
      "Disease - {'located_in'} -> GeneFamily\n",
      "Disease - {'regulates'} -> GeneFamily\n",
      "Disease - {'derives_from'} -> GeneFamily\n",
      "Disease - {'is_sequence_variant_of'} -> GeneFamily\n",
      "Disease - {'directly_physically_interacts_with'} -> GeneFamily\n",
      "Disease - {'subclass_of'} -> GeneFamily\n",
      "Disease - {'produces'} -> GeneFamily\n",
      "Disease - {'occurs_in'} -> GeneFamily\n",
      "Disease - {'coexists_with'} -> GeneFamily\n",
      "GeneFamily - {'treats'} -> Publication\n",
      "GeneFamily - {'causes'} -> Publication\n",
      "GeneFamily - {'associated_with'} -> Publication\n",
      "GeneFamily - {'disrupts'} -> Publication\n",
      "GeneFamily - {'gene_associated_with_condition'} -> Publication\n",
      "GeneFamily - {'located_in'} -> Publication\n",
      "GeneFamily - {'regulates'} -> Publication\n",
      "GeneFamily - {'predisposes'} -> Publication\n",
      "GeneFamily - {'occurs_in'} -> Publication\n",
      "GeneFamily - {'affects'} -> Publication\n",
      "GeneFamily - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->GenomicEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affected_by'} -> GenomicEntity\n",
      "Disease - {'has_substrate'} -> GenomicEntity\n",
      "Disease - {'interacts_with'} -> GenomicEntity\n",
      "Disease - {'treats'} -> GenomicEntity\n",
      "Disease - {'regulates'} -> GenomicEntity\n",
      "Disease - {'part_of'} -> GenomicEntity\n",
      "Disease - {'has_capability'} -> GenomicEntity\n",
      "Disease - {'participates_in'} -> GenomicEntity\n",
      "Disease - {'caused_by'} -> GenomicEntity\n",
      "Disease - {'has_input'} -> GenomicEntity\n",
      "Disease - {'preceded_by'} -> GenomicEntity\n",
      "Disease - {'enabled_by'} -> GenomicEntity\n",
      "Disease - {'has_participant'} -> GenomicEntity\n",
      "Disease - {'has_catalyst'} -> GenomicEntity\n",
      "Disease - {'entity_negatively_regulates_entity'} -> GenomicEntity\n",
      "Disease - {'decreases_response_to'} -> GenomicEntity\n",
      "Disease - {'associated_with'} -> GenomicEntity\n",
      "Disease - {'has_part'} -> GenomicEntity\n",
      "Disease - {'ameliorates'} -> GenomicEntity\n",
      "Disease - {'actively_involves'} -> GenomicEntity\n",
      "Disease - {'has_upstream_actor'} -> GenomicEntity\n",
      "Disease - {'exacerbates'} -> GenomicEntity\n",
      "Disease - {'has_active_ingredient'} -> GenomicEntity\n",
      "Disease - {'primarily_composed_of'} -> GenomicEntity\n",
      "Disease - {'temporally_related_to'} -> GenomicEntity\n",
      "Disease - {'affects'} -> GenomicEntity\n",
      "Disease - {'similar_to'} -> GenomicEntity\n",
      "Disease - {'related_to'} -> GenomicEntity\n",
      "Disease - {'overlaps'} -> GenomicEntity\n",
      "Disease - {'affects_response_to'} -> GenomicEntity\n",
      "Disease - {'entity_positively_regulates_entity'} -> GenomicEntity\n",
      "Disease - {'composed_primarily_of'} -> GenomicEntity\n",
      "Disease - {'increases_response_to'} -> GenomicEntity\n",
      "Disease - {'has_output'} -> GenomicEntity\n",
      "Disease - {'disrupts'} -> GenomicEntity\n",
      "Disease - {'produces'} -> GenomicEntity\n",
      "Disease - {'located_in'} -> GenomicEntity\n",
      "Disease - {'subclass_of'} -> GenomicEntity\n",
      "Disease - {'occurs_in'} -> GenomicEntity\n",
      "Disease - {'close_match'} -> GenomicEntity\n",
      "GenomicEntity - {'related_to'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->IndividualOrganism->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'participates_in'} -> IndividualOrganism\n",
      "Disease - {'has_input'} -> IndividualOrganism\n",
      "Disease - {'enabled_by'} -> IndividualOrganism\n",
      "Disease - {'has_participant'} -> IndividualOrganism\n",
      "Disease - {'has_substrate'} -> IndividualOrganism\n",
      "Disease - {'interacts_with'} -> IndividualOrganism\n",
      "Disease - {'has_capability'} -> IndividualOrganism\n",
      "Disease - {'actively_involves'} -> IndividualOrganism\n",
      "Disease - {'has_catalyst'} -> IndividualOrganism\n",
      "Disease - {'associated_with'} -> IndividualOrganism\n",
      "Disease - {'affects'} -> IndividualOrganism\n",
      "Disease - {'similar_to'} -> IndividualOrganism\n",
      "Disease - {'related_to'} -> IndividualOrganism\n",
      "Disease - {'overlaps'} -> IndividualOrganism\n",
      "Disease - {'composed_primarily_of'} -> IndividualOrganism\n",
      "Disease - {'has_upstream_actor'} -> IndividualOrganism\n",
      "Disease - {'primarily_composed_of'} -> IndividualOrganism\n",
      "Disease - {'has_output'} -> IndividualOrganism\n",
      "Disease - {'affected_by'} -> IndividualOrganism\n",
      "Disease - {'has_part'} -> IndividualOrganism\n",
      "Disease - {'occurs_in'} -> IndividualOrganism\n",
      "Disease - {'close_match'} -> IndividualOrganism\n",
      "Disease - {'disrupts'} -> IndividualOrganism\n",
      "Disease - {'regulates'} -> IndividualOrganism\n",
      "Disease - {'located_in'} -> IndividualOrganism\n",
      "Disease - {'causes'} -> IndividualOrganism\n",
      "Disease - {'subclass_of'} -> IndividualOrganism\n",
      "IndividualOrganism - {'close_match'} -> Publication\n",
      "IndividualOrganism - {'subclass_of'} -> Publication\n",
      "IndividualOrganism - {'occurs_in'} -> Publication\n",
      "IndividualOrganism - {'located_in'} -> Publication\n",
      "IndividualOrganism - {'related_to'} -> Publication\n",
      "IndividualOrganism - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->CellLine->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'ameliorates'} -> CellLine\n",
      "Disease - {'affects'} -> CellLine\n",
      "Disease - {'related_to'} -> CellLine\n",
      "Disease - {'treats'} -> CellLine\n",
      "Disease - {'close_match'} -> CellLine\n",
      "Disease - {'has_participant'} -> CellLine\n",
      "Disease - {'coexists_with'} -> CellLine\n",
      "Disease - {'regulates'} -> CellLine\n",
      "Disease - {'disrupts'} -> CellLine\n",
      "Disease - {'expressed_in'} -> CellLine\n",
      "Disease - {'located_in'} -> CellLine\n",
      "Disease - {'derives_from'} -> CellLine\n",
      "Disease - {'disease_has_location'} -> CellLine\n",
      "Disease - {'associated_with'} -> CellLine\n",
      "Disease - {'subclass_of'} -> CellLine\n",
      "Disease - {'has_part'} -> CellLine\n",
      "Disease - {'interacts_with'} -> CellLine\n",
      "Disease - {'produces'} -> CellLine\n",
      "Disease - {'diagnoses'} -> CellLine\n",
      "Disease - {'occurs_in'} -> CellLine\n",
      "Disease - {'has_input'} -> CellLine\n",
      "CellLine - {'gene_associated_with_condition'} -> Publication\n",
      "CellLine - {'located_in'} -> Publication\n",
      "CellLine - {'produces'} -> Publication\n",
      "CellLine - {'related_to'} -> Publication\n",
      "CellLine - {'affects'} -> Publication\n",
      "CellLine - {'close_match'} -> Publication\n",
      "CellLine - {'subclass_of'} -> Publication\n",
      "CellLine - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->BehavioralFeature->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'contributes_to'} -> BehavioralFeature\n",
      "Disease - {'manifestation_of'} -> BehavioralFeature\n",
      "Disease - {'has_input'} -> BehavioralFeature\n",
      "Disease - {'occurs_in'} -> BehavioralFeature\n",
      "Disease - {'affects'} -> BehavioralFeature\n",
      "Disease - {'close_match'} -> BehavioralFeature\n",
      "Disease - {'exacerbates'} -> BehavioralFeature\n",
      "Disease - {'coexists_with'} -> BehavioralFeature\n",
      "Disease - {'has_phenotype'} -> BehavioralFeature\n",
      "Disease - {'disrupts'} -> BehavioralFeature\n",
      "Disease - {'regulates'} -> BehavioralFeature\n",
      "Disease - {'contraindicated_for'} -> BehavioralFeature\n",
      "Disease - {'treats'} -> BehavioralFeature\n",
      "Disease - {'causes'} -> BehavioralFeature\n",
      "Disease - {'precedes'} -> BehavioralFeature\n",
      "Disease - {'associated_with'} -> BehavioralFeature\n",
      "Disease - {'subclass_of'} -> BehavioralFeature\n",
      "Disease - {'gene_associated_with_condition'} -> BehavioralFeature\n",
      "Disease - {'has_part'} -> BehavioralFeature\n",
      "Disease - {'interacts_with'} -> BehavioralFeature\n",
      "Disease - {'predisposes'} -> BehavioralFeature\n",
      "Disease - {'related_to'} -> BehavioralFeature\n",
      "Disease - {'prevents'} -> BehavioralFeature\n",
      "Disease - {'diagnoses'} -> BehavioralFeature\n",
      "BehavioralFeature - {'subclass_of'} -> Publication\n",
      "BehavioralFeature - {'produces'} -> Publication\n",
      "BehavioralFeature - {'related_to'} -> Publication\n",
      "BehavioralFeature - {'occurs_in'} -> Publication\n",
      "BehavioralFeature - {'affects'} -> Publication\n",
      "BehavioralFeature - {'manifestation_of'} -> Publication\n",
      "BehavioralFeature - {'coexists_with'} -> Publication\n",
      "BehavioralFeature - {'disrupts'} -> Publication\n",
      "BehavioralFeature - {'located_in'} -> Publication\n",
      "BehavioralFeature - {'predisposes'} -> Publication\n",
      "BehavioralFeature - {'causes'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Cohort->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'has_input'} -> Cohort\n",
      "Disease - {'coexists_with'} -> Cohort\n",
      "Disease - {'close_match'} -> Cohort\n",
      "Disease - {'has_participant'} -> Cohort\n",
      "Disease - {'located_in'} -> Cohort\n",
      "Disease - {'treats'} -> Cohort\n",
      "Disease - {'associated_with'} -> Cohort\n",
      "Disease - {'subclass_of'} -> Cohort\n",
      "Disease - {'interacts_with'} -> Cohort\n",
      "Disease - {'produces'} -> Cohort\n",
      "Disease - {'predisposes'} -> Cohort\n",
      "Disease - {'related_to'} -> Cohort\n",
      "Disease - {'occurs_in'} -> Cohort\n",
      "Disease - {'affects'} -> Cohort\n",
      "Disease - {'diagnoses'} -> Cohort\n",
      "Cohort - {'diagnoses'} -> Publication\n",
      "Cohort - {'related_to'} -> Publication\n",
      "Cohort - {'treats'} -> Publication\n",
      "Cohort - {'has_input'} -> Publication\n",
      "Cohort - {'occurs_in'} -> Publication\n",
      "Cohort - {'has_part'} -> Publication\n",
      "Cohort - {'close_match'} -> Publication\n",
      "Cohort - {'subclass_of'} -> Publication\n",
      "Cohort - {'interacts_with'} -> Publication\n",
      "Cohort - {'produces'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->GeographicLocation->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'has_input'} -> GeographicLocation\n",
      "Disease - {'affects'} -> GeographicLocation\n",
      "Disease - {'close_match'} -> GeographicLocation\n",
      "Disease - {'has_participant'} -> GeographicLocation\n",
      "Disease - {'coexists_with'} -> GeographicLocation\n",
      "Disease - {'located_in'} -> GeographicLocation\n",
      "Disease - {'associated_with'} -> GeographicLocation\n",
      "Disease - {'subclass_of'} -> GeographicLocation\n",
      "Disease - {'has_part'} -> GeographicLocation\n",
      "Disease - {'interacts_with'} -> GeographicLocation\n",
      "Disease - {'produces'} -> GeographicLocation\n",
      "Disease - {'related_to'} -> GeographicLocation\n",
      "Disease - {'occurs_in'} -> GeographicLocation\n",
      "GeographicLocation - {'subclass_of'} -> Publication\n",
      "GeographicLocation - {'related_to'} -> Publication\n",
      "GeographicLocation - {'gene_associated_with_condition'} -> Publication\n",
      "GeographicLocation - {'located_in'} -> Publication\n",
      "GeographicLocation - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->MaterialSample->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'affects'} -> MaterialSample\n",
      "Disease - {'coexists_with'} -> MaterialSample\n",
      "Disease - {'close_match'} -> MaterialSample\n",
      "Disease - {'regulates'} -> MaterialSample\n",
      "Disease - {'disrupts'} -> MaterialSample\n",
      "Disease - {'located_in'} -> MaterialSample\n",
      "Disease - {'treats'} -> MaterialSample\n",
      "Disease - {'causes'} -> MaterialSample\n",
      "Disease - {'precedes'} -> MaterialSample\n",
      "Disease - {'associated_with'} -> MaterialSample\n",
      "Disease - {'subclass_of'} -> MaterialSample\n",
      "Disease - {'has_part'} -> MaterialSample\n",
      "Disease - {'interacts_with'} -> MaterialSample\n",
      "Disease - {'produces'} -> MaterialSample\n",
      "Disease - {'predisposes'} -> MaterialSample\n",
      "Disease - {'related_to'} -> MaterialSample\n",
      "Disease - {'diagnoses'} -> MaterialSample\n",
      "Disease - {'occurs_in'} -> MaterialSample\n",
      "Disease - {'manifestation_of'} -> MaterialSample\n",
      "Disease - {'exacerbates'} -> MaterialSample\n",
      "MaterialSample - {'located_in'} -> Publication\n",
      "MaterialSample - {'associated_with'} -> Publication\n",
      "MaterialSample - {'subclass_of'} -> Publication\n",
      "MaterialSample - {'has_part'} -> Publication\n",
      "MaterialSample - {'related_to'} -> Publication\n",
      "MaterialSample - {'occurs_in'} -> Publication\n",
      "MaterialSample - {'affects'} -> Publication\n",
      "MaterialSample - {'close_match'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ProteinFamily->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'interacts_with'} -> ProteinFamily\n",
      "Disease - {'produces'} -> ProteinFamily\n",
      "Disease - {'related_to'} -> ProteinFamily\n",
      "Disease - {'occurs_in'} -> ProteinFamily\n",
      "Disease - {'affects'} -> ProteinFamily\n",
      "Disease - {'close_match'} -> ProteinFamily\n",
      "Disease - {'coexists_with'} -> ProteinFamily\n",
      "Disease - {'has_participant'} -> ProteinFamily\n",
      "Disease - {'regulates'} -> ProteinFamily\n",
      "Disease - {'located_in'} -> ProteinFamily\n",
      "Disease - {'associated_with'} -> ProteinFamily\n",
      "Disease - {'has_part'} -> ProteinFamily\n",
      "ProteinFamily - {'associated_with'} -> Publication\n",
      "ProteinFamily - {'subclass_of'} -> Publication\n",
      "ProteinFamily - {'gene_associated_with_condition'} -> Publication\n",
      "ProteinFamily - {'predisposes'} -> Publication\n",
      "ProteinFamily - {'related_to'} -> Publication\n",
      "ProteinFamily - {'affects'} -> Publication\n",
      "ProteinFamily - {'disrupts'} -> Publication\n",
      "ProteinFamily - {'located_in'} -> Publication\n",
      "ProteinFamily - {'regulates'} -> Publication\n",
      "ProteinFamily - {'treats'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Agent->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'located_in'} -> Agent\n",
      "Disease - {'biomarker_for'} -> Agent\n",
      "Disease - {'treats'} -> Agent\n",
      "Disease - {'causes'} -> Agent\n",
      "Disease - {'precedes'} -> Agent\n",
      "Disease - {'associated_with'} -> Agent\n",
      "Disease - {'gene_associated_with_condition'} -> Agent\n",
      "Disease - {'subclass_of'} -> Agent\n",
      "Disease - {'has_part'} -> Agent\n",
      "Disease - {'prevents'} -> Agent\n",
      "Disease - {'interacts_with'} -> Agent\n",
      "Disease - {'produces'} -> Agent\n",
      "Disease - {'predisposes'} -> Agent\n",
      "Disease - {'related_to'} -> Agent\n",
      "Disease - {'diagnoses'} -> Agent\n",
      "Disease - {'occurs_in'} -> Agent\n",
      "Disease - {'manifestation_of'} -> Agent\n",
      "Disease - {'has_input'} -> Agent\n",
      "Disease - {'exacerbates'} -> Agent\n",
      "Disease - {'affects'} -> Agent\n",
      "Disease - {'coexists_with'} -> Agent\n",
      "Disease - {'close_match'} -> Agent\n",
      "Disease - {'has_phenotype'} -> Agent\n",
      "Disease - {'regulates'} -> Agent\n",
      "Disease - {'disrupts'} -> Agent\n",
      "Agent - {'has_part'} -> Publication\n",
      "Agent - {'interacts_with'} -> Publication\n",
      "Agent - {'produces'} -> Publication\n",
      "Agent - {'predisposes'} -> Publication\n",
      "Agent - {'related_to'} -> Publication\n",
      "Agent - {'prevents'} -> Publication\n",
      "Agent - {'precedes'} -> Publication\n",
      "Agent - {'occurs_in'} -> Publication\n",
      "Agent - {'affects'} -> Publication\n",
      "Agent - {'exacerbates'} -> Publication\n",
      "Agent - {'coexists_with'} -> Publication\n",
      "Agent - {'close_match'} -> Publication\n",
      "Agent - {'subclass_of'} -> Publication\n",
      "Agent - {'regulates'} -> Publication\n",
      "Agent - {'diagnoses'} -> Publication\n",
      "Agent - {'treats'} -> Publication\n",
      "Agent - {'has_input'} -> Publication\n",
      "Agent - {'causes'} -> Publication\n",
      "Agent - {'located_in'} -> Publication\n",
      "Agent - {'disrupts'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->MicroRNA->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'located_in'} -> MicroRNA\n",
      "Disease - {'associated_with'} -> MicroRNA\n",
      "Disease - {'has_part'} -> MicroRNA\n",
      "Disease - {'interacts_with'} -> MicroRNA\n",
      "Disease - {'produces'} -> MicroRNA\n",
      "Disease - {'related_to'} -> MicroRNA\n",
      "Disease - {'occurs_in'} -> MicroRNA\n",
      "Disease - {'affects'} -> MicroRNA\n",
      "Disease - {'coexists_with'} -> MicroRNA\n",
      "MicroRNA - {'causes'} -> Publication\n",
      "MicroRNA - {'disrupts'} -> Publication\n",
      "MicroRNA - {'associated_with'} -> Publication\n",
      "MicroRNA - {'gene_associated_with_condition'} -> Publication\n",
      "MicroRNA - {'subclass_of'} -> Publication\n",
      "MicroRNA - {'predisposes'} -> Publication\n",
      "MicroRNA - {'located_in'} -> Publication\n",
      "MicroRNA - {'regulates'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->PhysicalEntity->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'associated_with'} -> PhysicalEntity\n",
      "Disease - {'subclass_of'} -> PhysicalEntity\n",
      "Disease - {'gene_associated_with_condition'} -> PhysicalEntity\n",
      "Disease - {'has_part'} -> PhysicalEntity\n",
      "Disease - {'interacts_with'} -> PhysicalEntity\n",
      "Disease - {'produces'} -> PhysicalEntity\n",
      "Disease - {'predisposes'} -> PhysicalEntity\n",
      "Disease - {'related_to'} -> PhysicalEntity\n",
      "Disease - {'prevents'} -> PhysicalEntity\n",
      "Disease - {'diagnoses'} -> PhysicalEntity\n",
      "Disease - {'manifestation_of'} -> PhysicalEntity\n",
      "Disease - {'has_input'} -> PhysicalEntity\n",
      "Disease - {'occurs_in'} -> PhysicalEntity\n",
      "Disease - {'affects'} -> PhysicalEntity\n",
      "Disease - {'close_match'} -> PhysicalEntity\n",
      "Disease - {'exacerbates'} -> PhysicalEntity\n",
      "Disease - {'coexists_with'} -> PhysicalEntity\n",
      "Disease - {'has_participant'} -> PhysicalEntity\n",
      "Disease - {'has_phenotype'} -> PhysicalEntity\n",
      "Disease - {'disrupts'} -> PhysicalEntity\n",
      "Disease - {'located_in'} -> PhysicalEntity\n",
      "Disease - {'regulates'} -> PhysicalEntity\n",
      "Disease - {'contraindicated_for'} -> PhysicalEntity\n",
      "Disease - {'treats'} -> PhysicalEntity\n",
      "Disease - {'derives_from'} -> PhysicalEntity\n",
      "Disease - {'causes'} -> PhysicalEntity\n",
      "Disease - {'precedes'} -> PhysicalEntity\n",
      "PhysicalEntity - {'subclass_of'} -> Publication\n",
      "PhysicalEntity - {'has_part'} -> Publication\n",
      "PhysicalEntity - {'interacts_with'} -> Publication\n",
      "PhysicalEntity - {'produces'} -> Publication\n",
      "PhysicalEntity - {'predisposes'} -> Publication\n",
      "PhysicalEntity - {'related_to'} -> Publication\n",
      "PhysicalEntity - {'occurs_in'} -> Publication\n",
      "PhysicalEntity - {'affects'} -> Publication\n",
      "PhysicalEntity - {'close_match'} -> Publication\n",
      "PhysicalEntity - {'coexists_with'} -> Publication\n",
      "PhysicalEntity - {'regulates'} -> Publication\n",
      "PhysicalEntity - {'located_in'} -> Publication\n",
      "PhysicalEntity - {'treats'} -> Publication\n",
      "PhysicalEntity - {'has_input'} -> Publication\n",
      "PhysicalEntity - {'causes'} -> Publication\n",
      "PhysicalEntity - {'disrupts'} -> Publication\n",
      "PhysicalEntity - {'associated_with'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Vitamin->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'associated_with'} -> Vitamin\n",
      "Disease - {'interacts_with'} -> Vitamin\n",
      "Disease - {'produces'} -> Vitamin\n",
      "Disease - {'related_to'} -> Vitamin\n",
      "Disease - {'has_part'} -> Vitamin\n",
      "Disease - {'has_input'} -> Vitamin\n",
      "Disease - {'coexists_with'} -> Vitamin\n",
      "Disease - {'close_match'} -> Vitamin\n",
      "Disease - {'has_participant'} -> Vitamin\n",
      "Disease - {'regulates'} -> Vitamin\n",
      "Vitamin - {'regulates'} -> Publication\n",
      "Vitamin - {'treats'} -> Publication\n",
      "Vitamin - {'disrupts'} -> Publication\n",
      "Vitamin - {'subclass_of'} -> Publication\n",
      "Vitamin - {'related_to'} -> Publication\n",
      "Vitamin - {'affects'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->LifeStage->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'subclass_of'} -> LifeStage\n",
      "Disease - {'produces'} -> LifeStage\n",
      "Disease - {'related_to'} -> LifeStage\n",
      "Disease - {'occurs_in'} -> LifeStage\n",
      "Disease - {'affects'} -> LifeStage\n",
      "Disease - {'close_match'} -> LifeStage\n",
      "Disease - {'located_in'} -> LifeStage\n",
      "Disease - {'treats'} -> LifeStage\n",
      "LifeStage - {'subclass_of'} -> Publication\n",
      "LifeStage - {'has_part'} -> Publication\n",
      "LifeStage - {'interacts_with'} -> Publication\n",
      "LifeStage - {'produces'} -> Publication\n",
      "LifeStage - {'has_input'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->ProteinDomain->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'associated_with'} -> ProteinDomain\n",
      "Disease - {'interacts_with'} -> ProteinDomain\n",
      "Disease - {'produces'} -> ProteinDomain\n",
      "Disease - {'related_to'} -> ProteinDomain\n",
      "Disease - {'has_part'} -> ProteinDomain\n",
      "Disease - {'occurs_in'} -> ProteinDomain\n",
      "Disease - {'affects'} -> ProteinDomain\n",
      "Disease - {'coexists_with'} -> ProteinDomain\n",
      "Disease - {'disrupts'} -> ProteinDomain\n",
      "Disease - {'regulates'} -> ProteinDomain\n",
      "Disease - {'located_in'} -> ProteinDomain\n",
      "ProteinDomain - {'associated_with'} -> Publication\n",
      "ProteinDomain - {'located_in'} -> Publication\n",
      "ProteinDomain - {'related_to'} -> Publication\n",
      "ProteinDomain - {'occurs_in'} -> Publication\n",
      "ProteinDomain - {'affects'} -> Publication\n",
      "ProteinDomain - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Disease->Event->Publication\n",
      "InformationResource - {'associated_with'} -> Disease\n",
      "Disease - {'subclass_of'} -> Event\n",
      "Disease - {'related_to'} -> Event\n",
      "Disease - {'diagnoses'} -> Event\n",
      "Disease - {'manifestation_of'} -> Event\n",
      "Disease - {'has_input'} -> Event\n",
      "Disease - {'exacerbates'} -> Event\n",
      "Disease - {'coexists_with'} -> Event\n",
      "Disease - {'close_match'} -> Event\n",
      "Event - {'occurs_in'} -> Publication\n",
      "Event - {'subclass_of'} -> Publication\n",
      "\n",
      "\n",
      "PATH: InformationResource->Publication\n",
      "InformationResource - {'subclass_of'} -> Publication\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for p in paths:\n",
    "    print(f\"PATH: {'->'.join(p['path'])}\")\n",
    "    for e in p['edges']:\n",
    "        print(f\"{e['subject']} - { {e['predicate']} } -> {e['object']}\")\n",
    "    print(\"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['InformationResource', 'NamedThing', 'AnatomicalEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'BiologicalProcess', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Cell', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'CellularComponent', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'ChemicalEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'ChemicalMixture', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'ComplexMolecularMixture', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Disease', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'GrossAnatomicalStructure', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'MolecularActivity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'MolecularMixture', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'OrganismTaxon', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Pathway', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'PhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Polypeptide', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Protein', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'SmallMolecule', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'CellLine', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Entity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'InformationContentEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'NoncodingRNAProduct', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Drug', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'EnvironmentalProcess', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Transcript', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Behavior', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'BiologicalProcessOrActivity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'MicroRNA', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'PathologicalProcess', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'RNAProduct', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Exon', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'NucleicAcidEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'LifeStage', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'DiseaseOrPhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Gene', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'MaterialSample', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'PopulationOfIndividualOrganisms', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'MolecularEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'OrganismalEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'BiologicalEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'OrganismAttribute', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'PhysicalEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'GenomicEntity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Annotation', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'IndividualOrganism', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'ClinicalAttribute', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'PhysiologicalProcess', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Phenomenon', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Procedure', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'GeneFamily', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Activity', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Device', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'ProteinFamily', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'BehavioralFeature', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Vitamin', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'ClinicalIntervention', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Food', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Cohort', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'GeographicLocation', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Agent', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'ProteinDomain', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Event', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'Treatment', 'Publication']\n",
      "['InformationResource', 'NamedThing', 'GeneticInheritance', 'Publication']\n",
      "['InformationResource', 'Pathway', 'SmallMolecule', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Disease', 'Publication']\n",
      "['InformationResource', 'Pathway', 'PhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'Pathway', 'AnatomicalEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'BiologicalProcess', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Cell', 'Publication']\n",
      "['InformationResource', 'Pathway', 'CellularComponent', 'Publication']\n",
      "['InformationResource', 'Pathway', 'ChemicalEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'ChemicalMixture', 'Publication']\n",
      "['InformationResource', 'Pathway', 'ComplexMolecularMixture', 'Publication']\n",
      "['InformationResource', 'Pathway', 'GrossAnatomicalStructure', 'Publication']\n",
      "['InformationResource', 'Pathway', 'MolecularActivity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'MolecularMixture', 'Publication']\n",
      "['InformationResource', 'Pathway', 'NamedThing', 'Publication']\n",
      "['InformationResource', 'Pathway', 'OrganismTaxon', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Polypeptide', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Protein', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Transcript', 'Publication']\n",
      "['InformationResource', 'Pathway', 'PhysiologicalProcess', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Procedure', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Drug', 'Publication']\n",
      "['InformationResource', 'Pathway', 'GeneFamily', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Gene', 'Publication']\n",
      "['InformationResource', 'Pathway', 'MolecularEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'BiologicalProcessOrActivity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'BiologicalEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Entity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'EnvironmentalProcess', 'Publication']\n",
      "['InformationResource', 'Pathway', 'OrganismalEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Behavior', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Annotation', 'Publication']\n",
      "['InformationResource', 'Pathway', 'DiseaseOrPhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'Pathway', 'OrganismAttribute', 'Publication']\n",
      "['InformationResource', 'Pathway', 'GenomicEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'NucleicAcidEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'RNAProduct', 'Publication']\n",
      "['InformationResource', 'Pathway', 'ClinicalAttribute', 'Publication']\n",
      "['InformationResource', 'Pathway', 'NoncodingRNAProduct', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Device', 'Publication']\n",
      "['InformationResource', 'Pathway', 'InformationContentEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Phenomenon', 'Publication']\n",
      "['InformationResource', 'Pathway', 'IndividualOrganism', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Treatment', 'Publication']\n",
      "['InformationResource', 'Pathway', 'PathologicalProcess', 'Publication']\n",
      "['InformationResource', 'Pathway', 'CellLine', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Activity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Food', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Agent', 'Publication']\n",
      "['InformationResource', 'Pathway', 'LifeStage', 'Publication']\n",
      "['InformationResource', 'Pathway', 'MicroRNA', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Vitamin', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Publication']\n",
      "['InformationResource', 'Pathway', 'ClinicalIntervention', 'Publication']\n",
      "['InformationResource', 'Pathway', 'BehavioralFeature', 'Publication']\n",
      "['InformationResource', 'Pathway', 'ProteinFamily', 'Publication']\n",
      "['InformationResource', 'Pathway', 'PhysicalEntity', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Cohort', 'Publication']\n",
      "['InformationResource', 'Pathway', 'PopulationOfIndividualOrganisms', 'Publication']\n",
      "['InformationResource', 'Pathway', 'MaterialSample', 'Publication']\n",
      "['InformationResource', 'Pathway', 'ProteinDomain', 'Publication']\n",
      "['InformationResource', 'Pathway', 'GeographicLocation', 'Publication']\n",
      "['InformationResource', 'Pathway', 'Event', 'Publication']\n",
      "['InformationResource', 'Pathway', 'GeneticInheritance', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'SmallMolecule', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Protein', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'ChemicalEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'MolecularEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Disease', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'PhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'OrganismAttribute', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'BiologicalEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Annotation', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'NamedThing', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Entity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'AnatomicalEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Gene', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'GenomicEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'NucleicAcidEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'OrganismalEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Polypeptide', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'BiologicalProcess', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'BiologicalProcessOrActivity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'MolecularActivity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Pathway', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'CellularComponent', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'CellLine', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Transcript', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'ClinicalAttribute', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'PhysiologicalProcess', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'ComplexMolecularMixture', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'NoncodingRNAProduct', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'RNAProduct', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'EnvironmentalProcess', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'InformationContentEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Phenomenon', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Procedure', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'IndividualOrganism', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Behavior', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'GrossAnatomicalStructure', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'PathologicalProcess', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'OrganismTaxon', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'MolecularMixture', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'GeneFamily', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Cell', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'ChemicalMixture', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Activity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Drug', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'PhysicalEntity', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Agent', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Device', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Vitamin', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'PopulationOfIndividualOrganisms', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'ClinicalIntervention', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'LifeStage', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'MaterialSample', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'ProteinFamily', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'BehavioralFeature', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'ProteinDomain', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Cohort', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Food', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'MicroRNA', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Treatment', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'GeographicLocation', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'Event', 'Publication']\n",
      "['InformationResource', 'DiseaseOrPhenotypicFeature', 'GeneticInheritance', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'SmallMolecule', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Drug', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ChemicalExposure', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Disease', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'PhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'EnvironmentalExposure', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Activity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ClinicalIntervention', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'PopulationOfIndividualOrganisms', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ChemicalEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ComplexMolecularMixture', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'BiologicalProcess', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'PhysiologicalProcess', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Protein', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Device', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Gene', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Procedure', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ClinicalAttribute', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Annotation', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'MolecularMixture', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'NamedThing', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Entity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'MolecularActivity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Behavior', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'CellularComponent', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'MolecularEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'GrossAnatomicalStructure', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'PathologicalProcess', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'GenomicEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'CellLine', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ChemicalMixture', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'NucleicAcidEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Pathway', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'DiseaseOrPhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'AnatomicalEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Polypeptide', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'EnvironmentalProcess', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'BiologicalProcessOrActivity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'OrganismalEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'BiologicalEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'OrganismTaxon', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Cell', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'OrganismAttribute', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Phenomenon', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'BehavioralFeature', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Agent', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'MaterialSample', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'MicroRNA', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'PhysicalEntity', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Vitamin', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ProteinFamily', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Event', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'RNAProduct', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'GeographicLocation', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'LifeStage', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'GeneFamily', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'IndividualOrganism', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Treatment', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Food', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'NoncodingRNAProduct', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'ProteinDomain', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Transcript', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Cohort', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'Exon', 'Publication']\n",
      "['InformationResource', 'InformationContentEntity', 'GeneticInheritance', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Disease', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Drug', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Procedure', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'SmallMolecule', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ChemicalEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Cell', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Protein', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'MolecularEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Gene', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Pathway', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'AnatomicalEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'BiologicalProcess', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'CellularComponent', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ChemicalMixture', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ComplexMolecularMixture', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'GrossAnatomicalStructure', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'MolecularActivity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'MolecularMixture', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'NamedThing', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'OrganismTaxon', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Polypeptide', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'PhysiologicalProcess', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'PathologicalProcess', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'GeneticInheritance', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'DiseaseOrPhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'BiologicalEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Entity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'PopulationOfIndividualOrganisms', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ChemicalExposure', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'EnvironmentalExposure', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'InformationContentEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ClinicalIntervention', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Device', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ClinicalAttribute', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Annotation', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Activity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'GenomicEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'NucleicAcidEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'OrganismalEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'OrganismAttribute', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'BiologicalProcessOrActivity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Food', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'NoncodingRNAProduct', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'RNAProduct', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'EnvironmentalProcess', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Transcript', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Phenomenon', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'IndividualOrganism', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Treatment', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Behavior', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'GeneFamily', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'CellLine', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'BehavioralFeature', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Agent', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'LifeStage', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Vitamin', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ProteinDomain', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Cohort', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'PhysicalEntity', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'MaterialSample', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'MicroRNA', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'GeographicLocation', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'ProteinFamily', 'Publication']\n",
      "['InformationResource', 'PhenotypicFeature', 'Event', 'Publication']\n",
      "['InformationResource', 'Behavior', 'BiologicalEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'SmallMolecule', 'Publication']\n",
      "['InformationResource', 'Behavior', 'MolecularMixture', 'Publication']\n",
      "['InformationResource', 'Behavior', 'BiologicalProcess', 'Publication']\n",
      "['InformationResource', 'Behavior', 'BiologicalProcessOrActivity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'NamedThing', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Entity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Protein', 'Publication']\n",
      "['InformationResource', 'Behavior', 'AnatomicalEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'ChemicalEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'OrganismalEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'MolecularActivity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Pathway', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Gene', 'Publication']\n",
      "['InformationResource', 'Behavior', 'DiseaseOrPhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Transcript', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Disease', 'Publication']\n",
      "['InformationResource', 'Behavior', 'ClinicalAttribute', 'Publication']\n",
      "['InformationResource', 'Behavior', 'InformationContentEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'PhysiologicalProcess', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Polypeptide', 'Publication']\n",
      "['InformationResource', 'Behavior', 'RNAProduct', 'Publication']\n",
      "['InformationResource', 'Behavior', 'EnvironmentalProcess', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Device', 'Publication']\n",
      "['InformationResource', 'Behavior', 'OrganismTaxon', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Cell', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Activity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Annotation', 'Publication']\n",
      "['InformationResource', 'Behavior', 'OrganismAttribute', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Drug', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Phenomenon', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Procedure', 'Publication']\n",
      "['InformationResource', 'Behavior', 'PhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'Behavior', 'IndividualOrganism', 'Publication']\n",
      "['InformationResource', 'Behavior', 'CellularComponent', 'Publication']\n",
      "['InformationResource', 'Behavior', 'MolecularEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'GrossAnatomicalStructure', 'Publication']\n",
      "['InformationResource', 'Behavior', 'PathologicalProcess', 'Publication']\n",
      "['InformationResource', 'Behavior', 'GeneFamily', 'Publication']\n",
      "['InformationResource', 'Behavior', 'GenomicEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'CellLine', 'Publication']\n",
      "['InformationResource', 'Behavior', 'ChemicalMixture', 'Publication']\n",
      "['InformationResource', 'Behavior', 'NucleicAcidEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'ComplexMolecularMixture', 'Publication']\n",
      "['InformationResource', 'Behavior', 'NoncodingRNAProduct', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Food', 'Publication']\n",
      "['InformationResource', 'Behavior', 'ProteinDomain', 'Publication']\n",
      "['InformationResource', 'Behavior', 'PhysicalEntity', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Agent', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Event', 'Publication']\n",
      "['InformationResource', 'Behavior', 'LifeStage', 'Publication']\n",
      "['InformationResource', 'Behavior', 'BehavioralFeature', 'Publication']\n",
      "['InformationResource', 'Behavior', 'PopulationOfIndividualOrganisms', 'Publication']\n",
      "['InformationResource', 'Behavior', 'ProteinFamily', 'Publication']\n",
      "['InformationResource', 'Behavior', 'ClinicalIntervention', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Cohort', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Publication']\n",
      "['InformationResource', 'Behavior', 'GeographicLocation', 'Publication']\n",
      "['InformationResource', 'Behavior', 'Vitamin', 'Publication']\n",
      "['InformationResource', 'Behavior', 'MaterialSample', 'Publication']\n",
      "['InformationResource', 'Behavior', 'GeneticInheritance', 'Publication']\n",
      "['InformationResource', 'Disease', 'PopulationOfIndividualOrganisms', 'Publication']\n",
      "['InformationResource', 'Disease', 'EnvironmentalExposure', 'Publication']\n",
      "['InformationResource', 'Disease', 'Drug', 'Publication']\n",
      "['InformationResource', 'Disease', 'ChemicalExposure', 'Publication']\n",
      "['InformationResource', 'Disease', 'Activity', 'Publication']\n",
      "['InformationResource', 'Disease', 'ClinicalAttribute', 'Publication']\n",
      "['InformationResource', 'Disease', 'Annotation', 'Publication']\n",
      "['InformationResource', 'Disease', 'PhysiologicalProcess', 'Publication']\n",
      "['InformationResource', 'Disease', 'InformationContentEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'Procedure', 'Publication']\n",
      "['InformationResource', 'Disease', 'Publication']\n",
      "['InformationResource', 'Disease', 'ComplexMolecularMixture', 'Publication']\n",
      "['InformationResource', 'Disease', 'ClinicalIntervention', 'Publication']\n",
      "['InformationResource', 'Disease', 'BiologicalProcess', 'Publication']\n",
      "['InformationResource', 'Disease', 'Protein', 'Publication']\n",
      "['InformationResource', 'Disease', 'Gene', 'Publication']\n",
      "['InformationResource', 'Disease', 'Device', 'Publication']\n",
      "['InformationResource', 'Disease', 'PhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'Disease', 'SmallMolecule', 'Publication']\n",
      "['InformationResource', 'Disease', 'Cell', 'Publication']\n",
      "['InformationResource', 'Disease', 'MolecularEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'ChemicalEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'Pathway', 'Publication']\n",
      "['InformationResource', 'Disease', 'AnatomicalEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'CellularComponent', 'Publication']\n",
      "['InformationResource', 'Disease', 'ChemicalMixture', 'Publication']\n",
      "['InformationResource', 'Disease', 'GrossAnatomicalStructure', 'Publication']\n",
      "['InformationResource', 'Disease', 'MolecularActivity', 'Publication']\n",
      "['InformationResource', 'Disease', 'MolecularMixture', 'Publication']\n",
      "['InformationResource', 'Disease', 'NamedThing', 'Publication']\n",
      "['InformationResource', 'Disease', 'OrganismTaxon', 'Publication']\n",
      "['InformationResource', 'Disease', 'Polypeptide', 'Publication']\n",
      "['InformationResource', 'Disease', 'BiologicalEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'Treatment', 'Publication']\n",
      "['InformationResource', 'Disease', 'NucleicAcidEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'Transcript', 'Publication']\n",
      "['InformationResource', 'Disease', 'PathologicalProcess', 'Publication']\n",
      "['InformationResource', 'Disease', 'GeneticInheritance', 'Publication']\n",
      "['InformationResource', 'Disease', 'DiseaseOrPhenotypicFeature', 'Publication']\n",
      "['InformationResource', 'Disease', 'OrganismAttribute', 'Publication']\n",
      "['InformationResource', 'Disease', 'Entity', 'Publication']\n",
      "['InformationResource', 'Disease', 'Food', 'Publication']\n",
      "['InformationResource', 'Disease', 'NoncodingRNAProduct', 'Publication']\n",
      "['InformationResource', 'Disease', 'RNAProduct', 'Publication']\n",
      "['InformationResource', 'Disease', 'EnvironmentalProcess', 'Publication']\n",
      "['InformationResource', 'Disease', 'OrganismalEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'Phenomenon', 'Publication']\n",
      "['InformationResource', 'Disease', 'Behavior', 'Publication']\n",
      "['InformationResource', 'Disease', 'BiologicalProcessOrActivity', 'Publication']\n",
      "['InformationResource', 'Disease', 'GeneFamily', 'Publication']\n",
      "['InformationResource', 'Disease', 'GenomicEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'IndividualOrganism', 'Publication']\n",
      "['InformationResource', 'Disease', 'CellLine', 'Publication']\n",
      "['InformationResource', 'Disease', 'BehavioralFeature', 'Publication']\n",
      "['InformationResource', 'Disease', 'Cohort', 'Publication']\n",
      "['InformationResource', 'Disease', 'GeographicLocation', 'Publication']\n",
      "['InformationResource', 'Disease', 'MaterialSample', 'Publication']\n",
      "['InformationResource', 'Disease', 'ProteinFamily', 'Publication']\n",
      "['InformationResource', 'Disease', 'Agent', 'Publication']\n",
      "['InformationResource', 'Disease', 'MicroRNA', 'Publication']\n",
      "['InformationResource', 'Disease', 'PhysicalEntity', 'Publication']\n",
      "['InformationResource', 'Disease', 'Vitamin', 'Publication']\n",
      "['InformationResource', 'Disease', 'LifeStage', 'Publication']\n",
      "['InformationResource', 'Disease', 'ProteinDomain', 'Publication']\n",
      "['InformationResource', 'Disease', 'Event', 'Publication']\n",
      "['InformationResource', 'Publication']\n"
     ]
    }
   ],
   "source": [
    "if nx.has_path(G, subject, object):\n",
    "    raw_paths = list(nx.all_simple_paths(G, source=subject, target=object, cutoff=cutoff))\n",
    "    for path in raw_paths:\n",
    "        print(path)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "---\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For the relations(predicates), if `node X` is connected to node Z through node Y, i.e X->Y->Z , will every relation(predicate) that connects Y->Z , or X->Y, will those relations all connect X->Z?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "---"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
